<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATCS</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natcs">NATCS - 115</h2>
<ul>
<li><details>
<summary>
(2025). Confidential computing for population-scale genome-wide association studies with SECRET-GWAS. <em>NATCS</em>, <em>5</em>(9), 825-835. (<a href='https://doi.org/10.1038/s43588-025-00856-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic data from a single institution lacks global diversity representation, especially for rare variants and diseases. Confidential computing can enable collaborative genome-wide association studies (GWAS) without compromising privacy or accuracy. However, due to limited secure memory space and performance overheads, previous solutions fail to support widely used regression methods. Here we present SECRET-GWAS—a rapid, privacy-preserving, population-scale, collaborative GWAS tool. We discuss several system optimizations, including streaming, batching, data parallelization and reducing trusted hardware overheads to efficiently scale linear and logistic regression to over a thousand processor cores on an Intel SGX-based cloud platform. In addition, we protect SECRET-GWAS against several hardware side-channel attacks. SECRET-GWAS is an open-source tool and works with the widely used Hail genomic analysis framework. Our experiments on Azure’s Confidential Computing platform demonstrate that SECRET-GWAS enables multivariate linear and logistic regression GWAS queries on population-scale datasets from ten independent sources in just 4.5 and 29 minutes, respectively. Secure collaborative genome-wide association studies (GWAS) with population-scale datasets address gaps in genomic data. This work proposes SECRET-GWAS and system optimizations that overcome resource constraints and exploit parallelism, while maintaining privacy and accuracy.},
  archive      = {J_NATCS},
  author       = {Rosenblum, Jonah and Dong, Juechu and Narayanasamy, Satish},
  doi          = {10.1038/s43588-025-00856-z},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {825-835},
  shortjournal = {Nat. Comput. Sci},
  title        = {Confidential computing for population-scale genome-wide association studies with SECRET-GWAS},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Analog in-memory computing attention mechanism for fast and energy-efficient large language models. <em>NATCS</em>, <em>5</em>(9), 813-824. (<a href='https://doi.org/10.1038/s43588-025-00854-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transformer networks, driven by self-attention, are central to large language models. In generative transformers, self-attention uses cache memory to store token projections, avoiding recomputation at each time step. However, graphics processing unit (GPU)-stored projections must be loaded into static random-access memory for each new generation step, causing latency and energy bottlenecks. Here we present a custom self-attention in-memory computing architecture based on emerging charge-based memories called gain cells, which can be efficiently written to store new tokens during sequence generation and enable parallel analog dot-product computation required for self-attention. However, the analog gain-cell circuits introduce non-idealities and constraints preventing the direct mapping of pre-trained models. To circumvent this problem, we design an initialization algorithm achieving text-processing performance comparable to GPT-2 without training from scratch. Our architecture reduces attention latency and energy consumption by up to two and four orders of magnitude, respectively, compared with GPUs, marking a substantial step toward ultrafast, low-power generative transformers. Leveraging in-memory computing with emerging gain-cell devices, the authors accelerate attention—a core mechanism in large language models. They train a 1.5-billion-parameter model, achieving up to a 70,000-fold reduction in energy consumption and a 100-fold speed-up compared with GPUs.},
  archive      = {J_NATCS},
  author       = {Leroux, Nathan and Manea, Paul-Philipp and Sudarshan, Chirag and Finkbeiner, Jan and Siegel, Sebastian and Strachan, John Paul and Neftci, Emre},
  doi          = {10.1038/s43588-025-00854-1},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {813-824},
  shortjournal = {Nat. Comput. Sci},
  title        = {Analog in-memory computing attention mechanism for fast and energy-efficient large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep active optimization for complex systems. <em>NATCS</em>, <em>5</em>(9), 801-812. (<a href='https://doi.org/10.1038/s43588-025-00858-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inferring optimal solutions from limited data is considered the ultimate goal in scientific discovery. Artificial intelligence offers a promising avenue to greatly accelerate this process. Existing methods often depend on large datasets, strong assumptions about objective functions, and classic machine learning techniques, restricting their effectiveness to low-dimensional or data-rich problems. Here we introduce an optimization pipeline that can effectively tackle complex, high-dimensional problems with limited data. This approach utilizes a deep neural surrogate to iteratively find optimal solutions and introduces additional mechanisms to avoid local optima, thereby minimizing the required samples. Our method finds superior solutions in problems with up to 2,000 dimensions, whereas existing approaches are confined to 100 dimensions and need considerably more data. It excels across varied real-world systems, outperforming current algorithms and enabling efficient knowledge discovery. Although focused on scientific problems, its benefits extend to numerous quantitative fields, paving the way for advanced self-driving laboratories. This study introduces a deep active optimization pipeline that effectively tackles high-dimensional, complex problems with limited data. The approach minimizes sample size and surpasses existing methods, achieving optimal solutions in up to 2,000 dimensions.},
  archive      = {J_NATCS},
  author       = {Wei, Ye and Peng, Bo and Xie, Ruiwen and Chen, Yangtao and Qin, Yu and Wen, Peng and Bauer, Stefan and Tung, Po-Yen and Raabe, Dierk},
  doi          = {10.1038/s43588-025-00858-x},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {801-812},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deep active optimization for complex systems},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Digital twin for chemical science: A case study on water interactions on the ag(111) surface. <em>NATCS</em>, <em>5</em>(9), 793-800. (<a href='https://doi.org/10.1038/s43588-025-00857-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Directly visualizing chemical trajectories offers insights into catalysis, gas-phase reactions and photoinduced dynamics. Tracking the transformation of chemical species is best achieved by coupling theory and experiment. Here we developed Digital Twin for Chemical Science (DTCS) v.01, which integrates theory, experiment and their bidirectional feedback loops into a unified platform for chemical characterization. DTCS addresses a core question: given a set of experimental conditions, what is the expected outcome and why? It consists of a forward solver that takes a chemical reaction network and predicts spectra under experimental conditions, and an inverse solver that infers kinetics from measured spectra. We applied DTCS to ambient-pressure X-ray photoelectron spectroscopy measurements of the Ag–H2O interface as an example. This approach enables real-time knowledge extraction and guides experiments until a stopping condition is met based on accuracy and degeneracy. As a step toward autonomous chemical characterization, DTCS provides mechanistic knowledge in a verified, standardized manner. Interpreting spectroscopic data in real time remains a challenge in chemical characterization. Here a digital twin framework is developed that links first-principles theory and experimental data via a bidirectional feedback loop, enabling on-the-fly decision-making and insights into reaction mechanisms based on measured spectra during chemical experiments.},
  archive      = {J_NATCS},
  author       = {Qian, Jin and Jana, Asmita and Menon, Siddarth and Bogdan, Andrew E. and Hamlyn, Rebecca and Mahl, Johannes and Crumlin, Ethan J.},
  doi          = {10.1038/s43588-025-00857-y},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {793-800},
  shortjournal = {Nat. Comput. Sci},
  title        = {Digital twin for chemical science: A case study on water interactions on the ag(111) surface},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SurFF: A foundation model for surface exposure and morphology across intermetallic crystals. <em>NATCS</em>, <em>5</em>(9), 782-792. (<a href='https://doi.org/10.1038/s43588-025-00839-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With approximately 90% of industrial reactions occurring on surfaces, the role of heterogeneous catalysts is paramount. Currently, accurate surface exposure prediction is vital for heterogeneous catalyst design, but it is hindered by the high costs of experimental and computational methods. Here we introduce a foundation force-field-based model for predicting surface exposure and synthesizability (SurFF) across intermetallic crystals, which are essential materials for heterogeneous catalysts. We created a comprehensive intermetallic surface database using an active learning method and high-throughput density functional theory calculations, encompassing 12,553 unique surfaces and 344,200 single points. SurFF achieves density-functional-theory-level precision with a prediction error of 3 meV Å−2 and enables large-scale surface exposure prediction with a 105-fold acceleration. Validation against computational and experimental data both show strong alignment. We applied SurFF for large-scale predictions of surface energy and Wulff shapes for over 6,000 intermetallic crystals, providing valuable data for the community. A foundation machine learning model, SurFF, enables DFT-accurate predictions of surface energies and morphologies in intermetallic catalysts, achieving over 105-fold acceleration for high-throughput materials screening.},
  archive      = {J_NATCS},
  author       = {Yin, Jun and Chen, Honghao and Qiu, Jiangjie and Li, Wentao and He, Peng and Li, Jiali and Karimi, Iftekhar A. and Lan, Xiaocheng and Wang, Tiefeng and Wang, Xiaonan},
  doi          = {10.1038/s43588-025-00839-0},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {782-792},
  shortjournal = {Nat. Comput. Sci},
  title        = {SurFF: A foundation model for surface exposure and morphology across intermetallic crystals},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Exploiting pleiotropy to enhance variant discovery with functional false discovery rates. <em>NATCS</em>, <em>5</em>(9), 769-781. (<a href='https://doi.org/10.1038/s43588-025-00852-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cost of recruiting participants for genome-wide association studies (GWASs) can limit sample sizes and hinder the discovery of genetic variants. Here we introduce the surrogate functional false discovery rate (sfFDR) framework that integrates summary statistics of related traits to increase power. The sfFDR framework provides estimates of FDR quantities such as the functional local FDR and q value, and uses these estimates to derive a functional P value for type I error rate control and a functional local Bayes’ factor for post-GWAS analyses. Compared with a standard analysis, sfFDR substantially increased power (equivalent to a 52% increase in sample size) in a study of obesity-related traits from the UK Biobank and discovered eight additional lead SNPs near genes linked to immune-related responses in a rare disease GWAS of eosinophilic granulomatosis with polyangiitis. Collectively, these results highlight the utility of exploiting related traits in both small and large studies. This study introduces a cost-effective strategy called surrogate functional false discovery rates to increase power in genome-wide association studies by leveraging genetic correlations (or pleiotropy) between related traits.},
  archive      = {J_NATCS},
  author       = {Bass, Andrew J. and Wallace, Chris},
  doi          = {10.1038/s43588-025-00852-3},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {769-781},
  shortjournal = {Nat. Comput. Sci},
  title        = {Exploiting pleiotropy to enhance variant discovery with functional false discovery rates},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Virtual brain twins for stimulation in epilepsy. <em>NATCS</em>, <em>5</em>(9), 754-768. (<a href='https://doi.org/10.1038/s43588-025-00841-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Estimating the epileptogenic zone network (EZN) is an important part of the diagnosis of drug-resistant focal epilepsy and has a pivotal role in treatment and intervention. Virtual brain twins provide a modeling method for personalized diagnosis and treatment. They integrate patient-specific brain topography with structural connectivity from anatomical neuroimaging such as magnetic resonance imaging, and dynamic activity from functional recordings such as electroencephalography (EEG) and stereo-EEG (SEEG). Seizures show rich spatial and temporal features in functional recordings, which can be exploited to estimate the EZN. Stimulation-induced seizures can provide important and complementary information. Here we consider invasive SEEG stimulation and non-invasive temporal interference stimulation as a complementary approach. This paper offers a high-resolution virtual brain twin framework for EZN diagnosis based on stimulation-induced seizures. It provides an important methodological and conceptual basis to make the transition from invasive to non-invasive diagnosis and treatment of drug-resistant focal epilepsy. A high-resolution virtual brain twin approach is proposed using stimulation-induced seizures to estimate the epileptogenic network, offering a step toward non-invasive diagnosis and treatment of drug-resistant focal epilepsy.},
  archive      = {J_NATCS},
  author       = {Wang, Huifang E. and Dollomaja, Borana and Triebkorn, Paul and Duma, Gian Marco and Williamson, Adam and Makhalova, Julia and Lemarechal, Jean-Didier and Bartolomei, Fabrice and Jirsa, Viktor},
  doi          = {10.1038/s43588-025-00841-6},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {754-768},
  shortjournal = {Nat. Comput. Sci},
  title        = {Virtual brain twins for stimulation in epilepsy},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). On the compatibility of generative AI and generative linguistics. <em>NATCS</em>, <em>5</em>(9), 745-753. (<a href='https://doi.org/10.1038/s43588-025-00861-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chomsky’s generative linguistics has made substantial contributions to cognitive science and symbolic artificial intelligence. With the rise of neural language models, however, the compatibility between generative artificial intelligence and generative linguistics has come under debate. Here we outline three ways in which generative artificial intelligence aligns with and supports the core ideas of generative linguistics. In turn, generative linguistics can provide criteria to evaluate and improve neural language models as models of human language and cognition. This Perspective discusses that generative AI aligns with generative linguistics by showing that neural language models (NLMs) are formal generative models. Furthermore, generative linguistics offers a framework for evaluating and improving NLMs.},
  archive      = {J_NATCS},
  author       = {Portelance, Eva and Jasbi, Masoud},
  doi          = {10.1038/s43588-025-00861-2},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {745-753},
  shortjournal = {Nat. Comput. Sci},
  title        = {On the compatibility of generative AI and generative linguistics},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Arti-‘fickle’ intelligence: Using LLMs as a tool for inference in the political and social sciences. <em>NATCS</em>, <em>5</em>(9), 737-744. (<a href='https://doi.org/10.1038/s43588-025-00843-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To promote the scientific use of large language models (LLMs), we suggest that researchers in the political and social sciences refocus on the scientific goal of inference. We suggest that this refocus will improve the accumulation of shared scientific knowledge about these tools and their uses in the social sciences. We discuss the challenges and opportunities related to scientific inference with LLMs, using validation of model output as an illustrative case for discussion. We then propose a set of guidelines related to establishing the failure and success of LLMs when completing particular tasks and discuss how to make inferences from these observations. Large language models are increasingly important in social science research. The authors provide guidance on how best to validate and use these models as rigorous tools to further scientific inference.},
  archive      = {J_NATCS},
  author       = {Argyle, Lisa P. and Busby, Ethan C. and Gubler, Joshua R. and Hepner, Bryce and Lyman, Alex and Wingate, David},
  doi          = {10.1038/s43588-025-00843-4},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {737-744},
  shortjournal = {Nat. Comput. Sci},
  title        = {Arti-‘fickle’ intelligence: Using LLMs as a tool for inference in the political and social sciences},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Urban planning in the era of large language models. <em>NATCS</em>, <em>5</em>(9), 727-736. (<a href='https://doi.org/10.1038/s43588-025-00846-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {City plans are the product of integrating human creativity with emerging technologies, which continuously evolve and reshape urban morphology and environments. Here we argue that large language models hold large untapped potential in addressing the growing complexities of urban planning and enabling a more holistic, innovative and responsive approach to city design. By harnessing their advanced generation and simulation capabilities, large language models can contribute as an intelligent assistant for human planners in synthesizing conceptual ideas, generating urban designs and evaluating the outcomes of planning efforts. Large language models remain largely unexplored is the design of cities. In this Perspective, the authors discuss the potential opportunities brought by these models in assisting urban planning.},
  archive      = {J_NATCS},
  author       = {Zheng, Yu and Xu, Fengli and Lin, Yuming and Santi, Paolo and Ratti, Carlo and Wang, Qi R. and Li, Yong},
  doi          = {10.1038/s43588-025-00846-1},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {727-736},
  shortjournal = {Nat. Comput. Sci},
  title        = {Urban planning in the era of large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rethinking chemical research in the age of large language models. <em>NATCS</em>, <em>5</em>(9), 715-726. (<a href='https://doi.org/10.1038/s43588-025-00811-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs) offer opportunities for advancing chemical research, including planning, optimization, data analysis, automation and knowledge management. Deploying LLMs in active environments, where they interact with tools and data, can greatly enhance their capabilities. However, challenges remain in evaluating their performance and addressing ethical issues such as reproducibility, data privacy and bias. Here we discuss ongoing and potential integrations of LLMs in chemical research, highlighting existing challenges to guide the effective use of LLMs as active scientific partners. This Perspective highlights the potential integrations of large language models (LLMs) in chemical research and provides guidance on the effective use of LLMs as research partners, noting the ethical and performance-based challenges that must be addressed moving forward.},
  archive      = {J_NATCS},
  author       = {MacKnight, Robert and Boiko, Daniil A. and Regio, Jose Emilio and Gallegos, Liliana C. and Neukomm, Théo A. and Gomes, Gabe},
  doi          = {10.1038/s43588-025-00811-y},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {715-726},
  shortjournal = {Nat. Comput. Sci},
  title        = {Rethinking chemical research in the age of large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A digital twin that interprets and refines chemical mechanisms. <em>NATCS</em>, <em>5</em>(9), 713-714. (<a href='https://doi.org/10.1038/s43588-025-00859-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An integrated platform, Digital Twin for Chemical Science (DTCS), is developed to connect first-principles theory with spectroscopic measurements through a bidirectional feedback loop. By predicting and refining chemical reaction mechanisms before, during and after experiments, DTCS enables the interpretation of spectra and supports real-time decision-making in chemical characterization.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00859-w},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {713-714},
  shortjournal = {Nat. Comput. Sci},
  title        = {A digital twin that interprets and refines chemical mechanisms},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Overcoming computational bottlenecks in large language models through analog in-memory computing. <em>NATCS</em>, <em>5</em>(9), 711-712. (<a href='https://doi.org/10.1038/s43588-025-00860-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study demonstrates the potential of using in-memory computing architecture for implementing large language models for an improved computational efficiency in both time and energy while maintaining a high accuracy.},
  archive      = {J_NATCS},
  author       = {Lin, Yudeng and Tang, Jianshi},
  doi          = {10.1038/s43588-025-00860-3},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {711-712},
  shortjournal = {Nat. Comput. Sci},
  title        = {Overcoming computational bottlenecks in large language models through analog in-memory computing},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Neuromorphic principles in self-attention hardware for efficient transformers. <em>NATCS</em>, <em>5</em>(9), 708-710. (<a href='https://doi.org/10.1038/s43588-025-00868-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Strong barriers remain between neuromorphic engineering and machine learning, especially with regard to recent large language models (LLMs) and transformers. This Comment makes the case that neuromorphic engineering may hold the keys to more efficient inference with transformer-like models.},
  archive      = {J_NATCS},
  author       = {Leroux, Nathan and Finkbeiner, Jan and Neftci, Emre},
  doi          = {10.1038/s43588-025-00868-9},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {708-710},
  shortjournal = {Nat. Comput. Sci},
  title        = {Neuromorphic principles in self-attention hardware for efficient transformers},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Using LLMs to advance the cognitive science of collectives. <em>NATCS</em>, <em>5</em>(9), 704-707. (<a href='https://doi.org/10.1038/s43588-025-00848-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs) are already transforming the study of individual cognition, but their application to studying collective cognition has been underexplored. We lay out how LLMs may be able to address the complexity that has hindered the study of collectives and raise possible risks that warrant new methods.},
  archive      = {J_NATCS},
  author       = {Sucholutsky, Ilia and Collins, Katherine M. and Jacoby, Nori and Thompson, Bill D. and Hawkins, Robert D.},
  doi          = {10.1038/s43588-025-00848-z},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {704-707},
  shortjournal = {Nat. Comput. Sci},
  title        = {Using LLMs to advance the cognitive science of collectives},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Threats to scientific software from over-reliance on AI code assistants. <em>NATCS</em>, <em>5</em>(9), 701-703. (<a href='https://doi.org/10.1038/s43588-025-00845-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The adoption of generative artificial intelligence (AI) code assistants in scientific software development is promising, but user studies across an array of programming contexts suggest that programmers are at risk of over-reliance on these tools, leading them to accept undetected errors in generated code. Scientific software may be particularly vulnerable to such errors because most research code is untested and scientists are undertrained in software development skills. This Comment outlines the factors that place scientific code at risk and suggests directions for research groups, educators, publishers and funders to counter these liabilities.},
  archive      = {J_NATCS},
  author       = {O’Brien, Gabrielle},
  doi          = {10.1038/s43588-025-00845-2},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {701-703},
  shortjournal = {Nat. Comput. Sci},
  title        = {Threats to scientific software from over-reliance on AI code assistants},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Computational challenges arising in algorithmic fairness and health equity with generative AI. <em>NATCS</em>, <em>5</em>(9), 698-700. (<a href='https://doi.org/10.1038/s43588-025-00806-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The use of generative artificial intelligence (AI) in healthcare is advancing, but understanding its potential challenges for fairness and health equity is still in its early stages. This Comment investigates how to define fairness and measure it, and highlights research that can help address challenges in the field.},
  archive      = {J_NATCS},
  author       = {Suriyakumar, Vinith M. and Zink, Anna and Hightower, Maia and Ghassemi, Marzyeh and Beaulieu-Jones, Brett},
  doi          = {10.1038/s43588-025-00806-9},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {698-700},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational challenges arising in algorithmic fairness and health equity with generative AI},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The impact of language models on the humanities and vice versa. <em>NATCS</em>, <em>5</em>(9), 695-697. (<a href='https://doi.org/10.1038/s43588-025-00819-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many humanists are skeptical of language models and concerned about their effects on universities. However, researchers with a background in the humanities are also actively engaging with artificial intelligence — seeking not only to adopt language models as tools, but to steer them toward a more flexible, contextual representation of written culture.},
  archive      = {J_NATCS},
  author       = {Underwood, Ted},
  doi          = {10.1038/s43588-025-00819-4},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {695-697},
  shortjournal = {Nat. Comput. Sci},
  title        = {The impact of language models on the humanities and vice versa},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The other AI revolution: How the global south is building and repurposing language models that speak to billions. <em>NATCS</em>, <em>5</em>(9), 691-694. (<a href='https://doi.org/10.1038/s43588-025-00865-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While leading tech companies race to build ever-larger models, researchers in Brazil, India and Africa are using clever tricks to remix big labs’ LLMs to bring AI to billions of users.},
  archive      = {J_NATCS},
  author       = {Burgos, Pedro},
  doi          = {10.1038/s43588-025-00865-y},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {691-694},
  shortjournal = {Nat. Comput. Sci},
  title        = {The other AI revolution: How the global south is building and repurposing language models that speak to billions},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The rise of large language models. <em>NATCS</em>, <em>5</em>(9), 689-690. (<a href='https://doi.org/10.1038/s43588-025-00890-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This issue of Nature Computational Science features a Focus that highlights both the promises and perils of large language models, their emerging applications across diverse scientific domains, and the opportunities to overcome the challenges that lie ahead.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00890-x},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {689-690},
  shortjournal = {Nat. Comput. Sci},
  title        = {The rise of large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Privacy-preserving multicenter differential protein abundance analysis with FedProt. <em>NATCS</em>, <em>5</em>(8), 675-688. (<a href='https://doi.org/10.1038/s43588-025-00832-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantitative mass spectrometry has revolutionized proteomics by enabling simultaneous quantification of thousands of proteins. Pooling patient-derived data from multiple institutions enhances statistical power but raises serious privacy concerns. Here we introduce FedProt, the first privacy-preserving tool for collaborative differential protein abundance analysis of distributed data, which utilizes federated learning and additive secret sharing. In the absence of a multicenter patient-derived dataset for evaluation, we created two: one at five centers from E. coli experiments and one at three centers from human serum. Evaluations using these datasets confirm that FedProt achieves accuracy equivalent to the DEqMS method applied to pooled data, with completely negligible absolute differences no greater than 4 × 10−12. By contrast, −log10P computed by the most accurate meta-analysis methods diverged from the centralized analysis results by up to 25–26. In this Resource, the authors present FedProt, a tool that enables privacy-preserving, federated differential protein abundance analysis across multiple institutions. Its results match the results of centralized analysis, enabling secure, collaborative proteomics without sensitive data sharing.},
  archive      = {J_NATCS},
  author       = {Burankova, Yuliya and Abele, Miriam and Bakhtiari, Mohammad and von Toerne, Christine and Barth, Teresa K. and Schweizer, Lisa and Giesbertz, Pieter and Schmidt, Johannes R. and Kalkhof, Stefan and Müller-Deile, Janina and van Veelen, Peter A. and Mohammed, Yassene and Hammer, Elke and Arend, Lis and Adamowicz, Klaudia and Laske, Tanja and Hartebrodt, Anne and Frisch, Tobias and Meng, Chen and Matschinske, Julian and Späth, Julian and Röttger, Richard and Schwämmle, Veit and Hauck, Stefanie M. and Lichtenthaler, Stefan F. and Imhof, Axel and Mann, Matthias and Ludwig, Christina and Kuster, Bernhard and Baumbach, Jan and Zolotareva, Olga},
  doi          = {10.1038/s43588-025-00832-7},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {675-688},
  shortjournal = {Nat. Comput. Sci},
  title        = {Privacy-preserving multicenter differential protein abundance analysis with FedProt},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A comprehensive mapping of zeolite–template chemical space. <em>NATCS</em>, <em>5</em>(8), 661-674. (<a href='https://doi.org/10.1038/s43588-025-00842-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Zeolites are industrially important catalysts and adsorbents, typically synthesized using specific molecules known as organic structure-directing agents (OSDAs). The templating effect of the OSDAs is pivotal in determining the zeolite polymorph formed and its physicochemical properties. However, de novo design of selective OSDAs is challenging because of the diversity and size of the zeolite–OSDA chemical space. Here we present ZeoBind, a computational workflow powered by machine learning that enables an exhaustive exploration of the OSDA space. We design predictive tasks that capture zeolite–molecule matching, train predictive models for these tasks on hundreds of thousands of datapoints and curate a library of 2.3 million synthetically accessible, hypothetical OSDA-like molecules enumerated from commercially available precursors. We use ZeoBind to screen nearly 500 million zeolite–molecule pairs and identified and experimentally validated two new OSDAs that template zeolites with novel compositions. The scale of the OSDA library, along with the open-access tools and data, has the potential to accelerate OSDA design for zeolite synthesis. ZeoBind is developed for high-throughput molecule screening in zeolite synthesis. Here 2.3 million organic structure-directing agents are enumerated and predictive models for binding affinity are developed; the screening is experimentally validated for two zeolites.},
  archive      = {J_NATCS},
  author       = {Xie, Mingrou and Schwalbe-Koda, Daniel and Semanate-Esquivel, Yolanda Marcela and Bello-Jurado, Estefanía and Hoffman, Alexander and Santiago-Reyes, Omar and Paris, Cecilia and Moliner, Manuel and Gómez-Bombarelli, Rafael},
  doi          = {10.1038/s43588-025-00842-5},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {661-674},
  shortjournal = {Nat. Comput. Sci},
  title        = {A comprehensive mapping of zeolite–template chemical space},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). RNA–ligand interaction scoring via data perturbation and augmentation modeling. <em>NATCS</em>, <em>5</em>(8), 648-660. (<a href='https://doi.org/10.1038/s43588-025-00820-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite recent advances in RNA-targeting drug discovery, the development of data-driven deep learning models remains challenging owing to limited validated RNA–small molecule interaction data and scarce known RNA structures. In this context, we introduce RNAsmol, a sequence-based deep learning framework that incorporates data perturbation with augmentation, graph-based molecular feature representation and attention-based feature fusion modules to predict RNA–small molecule interactions. RNAsmol employs perturbation strategies to balance the bias between the true negative and unknown interaction space, thereby elucidating the intrinsic binding patterns between RNA and small molecules. The resulting model demonstrates accurate predictions of the binding between RNA and small molecules, outperforming other methods in ten-fold cross-validation, unseen evaluation and decoy evaluation. Moreover, we use case studies to visualize molecular binding profiles and the distribution of learned weights, providing interpretable insights into RNAsmol’s predictions. In particular, without requiring structural input, RNAsmol can generate reliable predictions and be adapted to various drug design scenarios. RNAsmol is a sequence-based deep learning framework for predicting RNA–small molecule interactions, integrating perturbation and augmentation to achieve robust predictions under limited data and adaptability to diverse drug discovery scenarios.},
  archive      = {J_NATCS},
  author       = {Ma, Hongli and Gao, Letian and Jin, Yunfan and Ma, Jianwei and Bai, Yilan and Liu, Xiaofan and Bao, Pengfei and Liu, Ke and Xu, Zhenjiang Zech and Lu, Zhi John},
  doi          = {10.1038/s43588-025-00820-x},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {648-660},
  shortjournal = {Nat. Comput. Sci},
  title        = {RNA–ligand interaction scoring via data perturbation and augmentation modeling},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Investigating the volume and diversity of data needed for generalizable antibody–antigen ΔΔG prediction. <em>NATCS</em>, <em>5</em>(8), 635-647. (<a href='https://doi.org/10.1038/s43588-025-00823-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antibody–antigen binding affinity lies at the heart of therapeutic antibody development: efficacy is guided by specific binding and control of affinity. Here we present Graphinity, an equivariant graph neural network architecture built directly from antibody–antigen structures that achieves test Pearson’s correlations of up to 0.87 on experimental change in binding affinity (ΔΔG) prediction. However, our model, like previous methods, appears to be overtraining on the few hundred experimental data points available and performance is not robust to train–test cut-offs. To investigate the amount and type of data required to generalizably predict ΔΔG, we built synthetic datasets of nearly 1 million FoldX-generated and &gt;20,000 Rosetta Flex ddG-generated ΔΔG values. Our results indicate that there are currently insufficient experimental data to accurately and robustly predict ΔΔG, with orders of magnitude more likely needed. Dataset size is not the only consideration; diversity is also an important factor for model predictiveness. These findings provide a lower bound on data requirements to inform future method development and data collection efforts. Predicting the effects of mutations on antibody–antigen binding is a key challenge in therapeutic antibody development. Orders of magnitude more data will be needed to unlock accurate, robust prediction.},
  archive      = {J_NATCS},
  author       = {Hummer, Alissa M. and Schneider, Constantin and Chinery, Lewis and Deane, Charlotte M.},
  doi          = {10.1038/s43588-025-00823-8},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {635-647},
  shortjournal = {Nat. Comput. Sci},
  title        = {Investigating the volume and diversity of data needed for generalizable antibody–antigen ΔΔG prediction},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A large-scale replication of scenario-based experiments in psychology and management using large language models. <em>NATCS</em>, <em>5</em>(8), 627-634. (<a href='https://doi.org/10.1038/s43588-025-00840-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We conducted a large-scale study replicating 156 psychological experiments from top social science journals using three state-of-the-art large language models (LLMs). Our results reveal that, while LLMs demonstrated high replication rates for main effects (73–81%) and moderate to strong success with interaction effects (46–63%), they consistently produced larger effect sizes than human studies. Notably, LLMs showed significantly lower replication rates for studies involving socially sensitive topics such as race, gender and ethics. When original studies reported null findings, LLMs produced significant results at remarkably high rates (68–83%); while this could reflect cleaner data with less noise, it also suggests potential risks of effect size overestimation. Our results demonstrate both the promises and the challenges of LLMs in psychological research: while LLMs are efficient tools for pilot testing and rapid hypothesis validation, enriching rather than replacing traditional human-participant studies, they require more nuanced interpretation and human validation for complex social phenomena and culturally sensitive research questions. Researchers replicated 156 psychological experiments using three large language models (LLMs) instead of human participants. LLMs achieved 73–81% replication rates but showed amplified effect sizes and challenges with socially sensitive topics.},
  archive      = {J_NATCS},
  author       = {Cui, Ziyan and Li, Ning and Zhou, Huaikang},
  doi          = {10.1038/s43588-025-00840-7},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {627-634},
  shortjournal = {Nat. Comput. Sci},
  title        = {A large-scale replication of scenario-based experiments in psychology and management using large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A formally exact method for high-throughput absolute binding-free-energy calculations. <em>NATCS</em>, <em>5</em>(8), 621-626. (<a href='https://doi.org/10.1038/s43588-025-00821-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we introduce a high-throughput, formally exact method for absolute binding-free-energy calculations that enhances computational efficiency and accuracy. At the core of this method is a thermodynamic cycle that minimizes protein ligand relative motion, thereby reducing system perturbations and driving a fourfold gain in efficiency over the traditional double-decoupling method. By combining this strategy with double-wide sampling and hydrogen-mass repartitioning algorithms, the efficiency is further boosted to eightfold. The presented method is applied to 45 diverse protein–ligand complexes. For 34 complexes with validated force-field accuracy, our method achieves an average unsigned error of less than 1 kcal mol−1 and a hysteresis below 0.5 kcal mol−1, showcasing exceptional reliability. Moreover, it efficiently manages flexible peptide ligands through a potential-of-mean-force calculation, adding less than 5% extra simulation time. For 11 challenging cases, the presented method also shows an improvement compared with previously published results. Put together, this method has potential for advancing research in physical, biological and medicinal chemistry. A thermodynamic cycle is proposed that limits protein–ligand relative motion and boosts the efficiency of absolute binding-free-energy calculations four- to eightfold, without compromising theoretical rigor.},
  archive      = {J_NATCS},
  author       = {Bian, Hengwei and Shao, Xueguang and Chipot, Christophe and Cai, Wensheng and Fu, Haohao},
  doi          = {10.1038/s43588-025-00821-w},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {621-626},
  shortjournal = {Nat. Comput. Sci},
  title        = {A formally exact method for high-throughput absolute binding-free-energy calculations},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantifying batch effects for individual genes in single-cell data. <em>NATCS</em>, <em>5</em>(8), 612-620. (<a href='https://doi.org/10.1038/s43588-025-00824-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Batch effects substantially impede the comparison of multiple single-cell experiment batches. Existing methods for batch effect removal and quantification primarily emphasize cell alignment across batches, often overlooking gene-level batch effects. Here we introduce group technical effects (GTE)—a quantitative metric to assess batch effects on individual genes. Using GTE, we show that batch effects unevenly impact genes within the dataset. A portion of highly batch-sensitive genes (HBGs) differ between datasets and dominate the batch effects, whereas non-HBGs exhibit low batch effects. We demonstrate that as few as three HBGs are sufficient to introduce substantial batch effects. Our method also enables the assessment of cell-level batch effects, outperforming existing batch effect quantification methods. We also observe that biologically similar cell types undergo similar batch effects, informing the development of data integration strategies. The GTE method is versatile and applicable to various single-cell omics data types. Group technical effects, a metric to quantify gene-level batch effects in single-cell data, is proposed. The analysis based on group technical effects reveals the unbalanced batch effects across genes.},
  archive      = {J_NATCS},
  author       = {Zhou, Yang and Sheng, Qiongyu and Wang, Guohua and Xu, Li and Jin, Shuilin},
  doi          = {10.1038/s43588-025-00824-7},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {612-620},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantifying batch effects for individual genes in single-cell data},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unbalanced gene-level batch effects in single-cell data. <em>NATCS</em>, <em>5</em>(8), 610-611. (<a href='https://doi.org/10.1038/s43588-025-00829-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed group technical effects (GTE) as a quantitative metric for evaluating gene-level batch effects in single-cell data. It identifies highly batch-sensitive genes — the primary contributors to batch effects — that vary across datasets, and whose removal effectively mitigates the batch effects.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00829-2},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {610-611},
  shortjournal = {Nat. Comput. Sci},
  title        = {Unbalanced gene-level batch effects in single-cell data},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Diving deep into zeolite space. <em>NATCS</em>, <em>5</em>(8), 608-609. (<a href='https://doi.org/10.1038/s43588-025-00831-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study proposed ZeoBind, an AI-accelerated workflow enabling the discovery and experimental verification of hits within chemical spaces containing hundreds of millions of zeolites.},
  archive      = {J_NATCS},
  author       = {Balcells, David},
  doi          = {10.1038/s43588-025-00831-8},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {608-609},
  shortjournal = {Nat. Comput. Sci},
  title        = {Diving deep into zeolite space},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). What’s so hard about RNA-targeting drug discovery?. <em>NATCS</em>, <em>5</em>(8), 606-607. (<a href='https://doi.org/10.1038/s43588-025-00853-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An artificial intelligence model leverages language models and molecular perturbations to tackle the data scarcity problem in the RNA drug world.},
  archive      = {J_NATCS},
  author       = {Oliver, Carlos and Waldispühl, Jérôme},
  doi          = {10.1038/s43588-025-00853-2},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {606-607},
  shortjournal = {Nat. Comput. Sci},
  title        = {What’s so hard about RNA-targeting drug discovery?},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Experimenting on AI can (sometimes) teach us about ourselves. <em>NATCS</em>, <em>5</em>(8), 604-605. (<a href='https://doi.org/10.1038/s43588-025-00838-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study sought to replicate published experimental research using large language models, finding that human behavior is replicated surprisingly well overall, but deviates in important ways that could lead social scientists astray.},
  archive      = {J_NATCS},
  author       = {van Loon, Austin and Heidenry, Zoe},
  doi          = {10.1038/s43588-025-00838-1},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {604-605},
  shortjournal = {Nat. Comput. Sci},
  title        = {Experimenting on AI can (sometimes) teach us about ourselves},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pushing the frontiers in computational chemistry. <em>NATCS</em>, <em>5</em>(8), 603. (<a href='https://doi.org/10.1038/s43588-025-00862-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We reflect on ongoing topics and discussions at conferences in the field of computational chemistry.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00862-1},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {603},
  shortjournal = {Nat. Comput. Sci},
  title        = {Pushing the frontiers in computational chemistry},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Iterative variational learning of committor-consistent transition pathways using artificial neural networks. <em>NATCS</em>, <em>5</em>(7), 592-602. (<a href='https://doi.org/10.1038/s43588-025-00828-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Discovering transition pathways that are physically meaningful and committor-consistent has long been a challenge in studying rare events in complex systems. Here we introduce a neural network-based strategy that learns simultaneously the committor function and the associated committor-consistent string, offering an unprecedented view of transition processes. Built on the committor time-correlation function, this method operates across diverse dynamical regimes, and extends beyond traditional approaches relying on infinitesimal time-lag approximations, valid only in the overdamped diffusive limit. It also distinguishes multiple competing pathways, crucial for understanding complex biomolecular transformations. Demonstrated on benchmark potentials and biological systems such as peptide isomerization and protein-model folding, this approach robustly reproduces established dynamics, rate constants and transition mechanisms. Its adaptability to collective variables and resilience across neural architectures make it a powerful and versatile tool for enhanced-sampling simulations of rare events, enabling insights into the intricate landscapes of biomolecular systems. A neural network approach grounded in transition-path theory is shown to uncover committor-consistent transition pathways, resolving competing mechanisms across dynamical regimes and advancing the modeling of rare events in biomolecular systems.},
  archive      = {J_NATCS},
  author       = {Megías, Alberto and Contreras Arredondo, Sergio and Chen, Cheng Giuseppe and Tang, Chenyu and Roux, Benoît and Chipot, Christophe},
  doi          = {10.1038/s43588-025-00828-3},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {592-602},
  shortjournal = {Nat. Comput. Sci},
  title        = {Iterative variational learning of committor-consistent transition pathways using artificial neural networks},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Everything everywhere all at once: A probability-based enhanced sampling approach to rare events. <em>NATCS</em>, <em>5</em>(7), 582-591. (<a href='https://doi.org/10.1038/s43588-025-00799-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The problem of studying rare events is central to many areas of computer simulations. We recently proposed an approach to solving this problem that involves computing the committor function, showing how it can be iteratively computed in a variational way while efficiently sampling the transition state ensemble. Here we greatly improve this procedure by combining it with a metadynamics-like enhanced sampling approach in which a logarithmic function of the committor is used as a collective variable. This procedure leads to an accurate sampling of the free energy surface in which transition states and metastable basins are studied with the same thoroughness. We show that our approach can be used in cases with the possibility of competing reactive paths and metastable intermediates. In addition, we demonstrate how physical insights can be obtained from the optimized committor model and the sampled data, thus providing a full characterization of the rare event under study. A single semi-automatic enhanced sampling method for rare events, based on machine-learned committor functions, allows simultaneous sampling of reactive events, calculation of free energy and understanding of transition states.},
  archive      = {J_NATCS},
  author       = {Trizio, Enrico and Kang, Peilin and Parrinello, Michele},
  doi          = {10.1038/s43588-025-00799-5},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {582-591},
  shortjournal = {Nat. Comput. Sci},
  title        = {Everything everywhere all at once: A probability-based enhanced sampling approach to rare events},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Best practices for multi-fidelity bayesian optimization in materials and molecular research. <em>NATCS</em>, <em>5</em>(7), 572-581. (<a href='https://doi.org/10.1038/s43588-025-00822-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multi-fidelity Bayesian optimization (MFBO) is a promising framework to speed up materials and molecular discovery as sources of information of different accuracies are at hand at increasing cost. Despite its potential use in chemical tasks, there is a lack of systematic evaluation of the many parameters playing a role in MFBO. Here we provide guidelines and recommendations to decide when to use MFBO in experimental settings. We investigate MFBO methods applied to molecules and materials problems. First, we test two different families of acquisition functions in two synthetic problems and study the effect of the informativeness and cost of the approximate function. We use our implementation and guidelines to benchmark three real discovery problems and compare them against their single-fidelity counterparts. Our results may help guide future efforts to implement MFBO as a routine tool in the chemical sciences. Multi-fidelity Bayesian optimization methods are studied on molecular and material discovery tasks, and guidelines are provided to recommend cheaper and informative low-fidelity sources when using this technique in experimental settings.},
  archive      = {J_NATCS},
  author       = {Sabanza-Gil, Víctor and Barbano, Riccardo and Pacheco Gutiérrez, Daniel and Luterbacher, Jeremy S. and Hernández-Lobato, José Miguel and Schwaller, Philippe and Roch, Loïc},
  doi          = {10.1038/s43588-025-00822-9},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {572-581},
  shortjournal = {Nat. Comput. Sci},
  title        = {Best practices for multi-fidelity bayesian optimization in materials and molecular research},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Memory kernel minimization-based neural networks for discovering slow collective variables of biomolecular dynamics. <em>NATCS</em>, <em>5</em>(7), 562-571. (<a href='https://doi.org/10.1038/s43588-025-00815-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying collective variables (CVs) that accurately capture the slowest timescales of protein conformational changes is crucial to comprehend numerous biological processes. Here we introduce memory kernel minimization-based neural networks (MEMnets), a deep learning framework that accurately identifies the slow CVs of biomolecular dynamics. Unlike popular CV-identification methods, which typically assume Markovian dynamics, MEMnets is built on the integrative generalized master equation theory, which incorporates non-Markovian dynamics by encoding them in a memory kernel for continuous CVs. The key innovation of MEMnets is the identification of optimal CVs by minimizing the upper bound for the time-integrated memory kernels through parallel encoder networks. We demonstrate that MEMnets effectively identifies slow CVs involved in the folding of the FIP35 WW domain, revealing two parallel folding pathways. In addition, we illustrate MEMnets’ robust numerical stability in identifying meaningful CVs in large biomolecular dynamic systems with limited sampling by applying it to the clamp opening of bacterial RNA polymerase, a much more complex conformational change. This study presents MEMnets, an approach that integrates statistical mechanics with deep learning to identify the slowest collective variables for biomolecular dynamics. MEMnets effectively captures memory effects, enabling advanced time-series analysis.},
  archive      = {J_NATCS},
  author       = {Liu, Bojun and Cao, Siqin and Boysen, Jordan G. and Xue, Mingyi and Huang, Xuhui},
  doi          = {10.1038/s43588-025-00815-8},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {562-571},
  shortjournal = {Nat. Comput. Sci},
  title        = {Memory kernel minimization-based neural networks for discovering slow collective variables of biomolecular dynamics},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicting adverse drug reactions for combination pharmacotherapy with cross-scale associative learning via attention modules. <em>NATCS</em>, <em>5</em>(7), 547-561. (<a href='https://doi.org/10.1038/s43588-025-00816-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rapid emergence of combination pharmacotherapies offers substantial therapeutic advantages but also poses risks of adverse drug reactions (ADRs). The accurate prediction of ADRs with interpretable computational methods is crucial for clinical medication management, drug development and precision medicine. Machine-learning and recently developed deep learning architectures struggle to effectively elucidate the key protein–protein interactions underlying ADRs from an organ perspective and to explicitly represent ADR associations. Here we propose OrganADR, an associative learning-enhanced model to predict ADRs at the organ level for emerging combination pharmacotherapy. It incorporates ADR information at the organ level, drug information at the molecular level and network-based biomedical knowledge into integrated representations with multi-interpretable modules. Evaluation across 15 organs demonstrates that OrganADR not only achieves state-of-the-art performance but also delivers both interpretable insights at the organ level and network-based perspectives. Overall, OrganADR represents a useful tool for cross-scale biomedical information integration and could be used to prevent ADRs during clinical precision medicine. An associative-learning-enhanced model, called OrganADR, is developed to predict adverse drug reactions at the organ level. OrganADR could be applied in emerging combination pharmacotherapies and improve medication safety.},
  archive      = {J_NATCS},
  author       = {Li, Boyang and Qi, Yifan and Li, Bo and Li, Xiaoqiong},
  doi          = {10.1038/s43588-025-00816-7},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {547-561},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting adverse drug reactions for combination pharmacotherapy with cross-scale associative learning via attention modules},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Inter-individual and inter-site neural code conversion without shared stimuli. <em>NATCS</em>, <em>5</em>(7), 534-546. (<a href='https://doi.org/10.1038/s43588-025-00826-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inter-individual variability in fine-grained functional topographies poses challenges for scalable data analysis and modeling. Functional alignment techniques can help mitigate these individual differences but they typically require paired brain data with the same stimuli between individuals, which are often unavailable. Here we present a neural code conversion method that overcomes this constraint by optimizing conversion parameters based on the discrepancy between the stimulus contents represented by original and converted brain activity patterns. This approach, combined with hierarchical features of deep neural networks as latent content representations, achieves conversion accuracies that are comparable with methods using shared stimuli. The converted brain activity from a source subject can be accurately decoded using the target’s pre-trained decoders, producing high-quality visual image reconstructions that rival within-individual decoding, even with data across different sites and limited training samples. Our approach offers a promising framework for scalable neural data analysis and modeling and a foundation for brain-to-brain communication. A neural code conversion method is introduced using deep neural network representations to align brain data across individuals without shared stimuli. The approach enables accurate inter-individual brain decoding and visual image reconstruction across sites.},
  archive      = {J_NATCS},
  author       = {Wang, Haibao and Ho, Jun Kai and Cheng, Fan L. and Aoki, Shuntaro C. and Muraki, Yusuke and Tanaka, Misato and Park, Jong-Yun and Kamitani, Yukiyasu},
  doi          = {10.1038/s43588-025-00826-5},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {534-546},
  shortjournal = {Nat. Comput. Sci},
  title        = {Inter-individual and inter-site neural code conversion without shared stimuli},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cost-effective instruction learning for pathology vision and language analysis. <em>NATCS</em>, <em>5</em>(7), 524-533. (<a href='https://doi.org/10.1038/s43588-025-00818-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The advent of vision–language models fosters interactive conversations between artificial intelligence-enabled models and humans. However, applying these models in the clinic faces challenges related to large-scale training data as well as financial and computational resources. Here we propose CLOVER, a cost-effective instruction learning framework for conversational pathology. CLOVER trains a lightweight module and uses instruction tuning while freezing the parameters of the large language model. Instead of using costly GPT-4, we propose well-designed prompts on GPT-3.5 for building generation-based instructions, emphasizing the utility of pathological knowledge derived from the Internet source. We construct a high-quality set of template-based instructions in the context of digital pathology. Using two benchmark datasets, our findings reveal the strength of hybrid-form, pathological visual question–answer instructions. CLOVER outperforms baselines that possess 37 times more training parameters and exhibits few-shot capacity on an external clinical dataset. CLOVER could thus accelerate the adoption of rapid conversational applications in digital pathology. Training foundation models often requires a costly budget and excessive computational resources. In this study, a low-cost instruction learning framework is proposed that could enable the rapid adoption of visual-language pathology applications.},
  archive      = {J_NATCS},
  author       = {Chen, Kaitao and Liu, Mianxin and Yan, Fang and Ma, Lei and Shi, Xiaoming and Wang, Lilong and Wang, Xiaosong and Zhu, Lifeng and Wang, Zhe and Zhou, Mu and Zhang, Shaoting},
  doi          = {10.1038/s43588-025-00818-5},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {524-533},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cost-effective instruction learning for pathology vision and language analysis},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A committor-based method to uniformly sample rare reactive events. <em>NATCS</em>, <em>5</em>(7), 522-523. (<a href='https://doi.org/10.1038/s43588-025-00825-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Enhanced sampling methods aim to simulate rare physical and chemical reactive processes involving transitions between long-lived states. Existing methods often disproportionally sample either metastable or transition states. A machine-learning approach combines the strengths of these two cases to characterize entire rare events with the same thoroughness in a single calculation.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00825-6},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {522-523},
  shortjournal = {Nat. Comput. Sci},
  title        = {A committor-based method to uniformly sample rare reactive events},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Learning committor-consistent collective variables. <em>NATCS</em>, <em>5</em>(7), 520-521. (<a href='https://doi.org/10.1038/s43588-025-00834-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An artificial neural network-based strategy is developed to learn committor-consistent transition pathways, providing insight into rare events in biomolecular systems.},
  archive      = {J_NATCS},
  author       = {Fröhlking, Thorben and Aureli, Simone and Gervasio, Francesco Luigi},
  doi          = {10.1038/s43588-025-00834-5},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {520-521},
  shortjournal = {Nat. Comput. Sci},
  title        = {Learning committor-consistent collective variables},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Guidelines for multi-fidelity bayesian optimization of molecules and materials. <em>NATCS</em>, <em>5</em>(7), 518-519. (<a href='https://doi.org/10.1038/s43588-025-00833-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study provides intuition and guidelines for deciding whether to incorporate cheaper, lower-fidelity experiments into a closed-loop search for molecules and materials with desired properties.},
  archive      = {J_NATCS},
  author       = {Ke, Qia and Simon, Cory M.},
  doi          = {10.1038/s43588-025-00833-6},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {518-519},
  shortjournal = {Nat. Comput. Sci},
  title        = {Guidelines for multi-fidelity bayesian optimization of molecules and materials},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing neural decoding with deep learning. <em>NATCS</em>, <em>5</em>(7), 516-517. (<a href='https://doi.org/10.1038/s43588-025-00837-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study introduces a neural code conversion method that aligns brain activity across individuals without shared stimuli, using deep neural network-derived features to match stimulus content.},
  archive      = {J_NATCS},
  author       = {Feilong, Ma and Zhang, Yuqi},
  doi          = {10.1038/s43588-025-00837-2},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {516-517},
  shortjournal = {Nat. Comput. Sci},
  title        = {Advancing neural decoding with deep learning},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). On writing accessible computational science papers. <em>NATCS</em>, <em>5</em>(7), 515. (<a href='https://doi.org/10.1038/s43588-025-00847-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We provide our recommendations on effectively writing computational science manuscripts.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00847-0},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {515},
  shortjournal = {Nat. Comput. Sci},
  title        = {On writing accessible computational science papers},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unified deep learning framework for many-body quantum chemistry via green’s functions. <em>NATCS</em>, <em>5</em>(6), 502-513. (<a href='https://doi.org/10.1038/s43588-025-00810-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum many-body methods provide a systematic route to computing electronic properties of molecules and materials, but high computational costs restrict their use in large-scale applications. Owing to the complexity in many-electron wavefunctions, machine learning models capable of capturing fundamental many-body physics remain limited. Here we present a deep learning framework targeting the many-body Green’s function, which unifies predictions of electronic properties in ground and excited states, while offering physical insights into many-electron correlation effects. By learning the many-body perturbation theory or coupled-cluster self-energy from mean-field features, our graph neural network achieves competitive performance in predicting one- and two-particle excitations and quantities derivable from a one-particle density matrix. We demonstrate its high data efficiency and good transferability across chemical species, system sizes, molecular conformations and correlation strengths in bond breaking, through multiple molecular and nanomaterial benchmarks. This work opens up opportunities for utilizing machine learning to solve many-electron problems. A data-efficient deep learning model developed to predict ground-state and photophysical properties of molecules and nanomaterials by learning many-body Green’s functions achieves an accuracy surpassing the state-of-the-art density functional theory calculations.},
  archive      = {J_NATCS},
  author       = {Venturella, Christian and Li, Jiachen and Hillenbrand, Christopher and Leyva Peralta, Ximena and Liu, Jessica and Zhu, Tianyu},
  doi          = {10.1038/s43588-025-00810-z},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {502-513},
  shortjournal = {Nat. Comput. Sci},
  title        = {Unified deep learning framework for many-body quantum chemistry via green’s functions},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The critical role of persistent disruption in advancing science. <em>NATCS</em>, <em>5</em>(6), 492-501. (<a href='https://doi.org/10.1038/s43588-025-00808-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Disruptive innovation is an important feature of scientific research. However, increasing evidence in recent years shows that highly disruptive papers are not necessarily milestone works in science and may even receive very few citations. To understand the mechanisms leading to such phenomena, we develop a link disruption metric that quantifies the disruptiveness of each citation link. This metric allows us to investigate disruption at both the reference and citation levels, enabling the development of a two-dimensional framework to evaluate the persistence of disruption caused by a given paper. Surprisingly, we find that papers with high reference disruption can have high citation disruption, meaning that a paper that disrupts previous papers may itself be further disrupted by its later citing papers. We find that persistently disruptive papers (disruptive papers that are not disrupted by citing papers) are more likely to be recognized as award-winning papers and receive high numbers of citations. Finally, we find that papers of larger teams and papers in recent years, though found to have weaker disruption, are more likely to have stronger persistent disruption once they disrupt previous papers. This study introduces a concept of persistent disruption, which marks papers that disrupt previous work but remain undisrupted later, signaling lasting impact. Milestone papers are found to be consistently associated with persistent disruption.},
  archive      = {J_NATCS},
  author       = {Deng, Nan and Gu, Xifeng and Fan, Ying and Havlin, Shlomo and Zeng, An},
  doi          = {10.1038/s43588-025-00808-7},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {492-501},
  shortjournal = {Nat. Comput. Sci},
  title        = {The critical role of persistent disruption in advancing science},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gender and racial diversity socialization in science. <em>NATCS</em>, <em>5</em>(6), 481-491. (<a href='https://doi.org/10.1038/s43588-025-00795-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientific collaboration networks are a form of unequally distributed social capital that shapes both researcher job placement and long-term research productivity and prominence. However, the role of collaboration networks in shaping the gender and racial diversity of the scientific workforce remains unclear. Here we propose a computational null model to investigate the degree to which early-career scientific collaborators with representationally diverse cohorts of scholars are associated with forming or participating in more diverse research groups as established researchers. When testing this hypothesis using two large-scale, longitudinal datasets on scientific collaborations, we find that the gender and racial diversity in a researcher’s early-career collaboration environment is strongly associated with the diversity of their collaborators in their established period. This diversity-association effect is particularly prominent for men. Coupled with gender and racial homophily between advisors and advisees, collaborator diversity represents a generational effect that partly explains why changes in representation within the scientific workforce tend to happen very slowly. A computational null model is proposed to study the gender and racial diversity-association effects in academia. Researchers’ early training in diverse environments is strongly correlated with nurturing diverse groups in the established period.},
  archive      = {J_NATCS},
  author       = {Li, Weihua and Zheng, Hongwei and Brand, Jennie E. and Clauset, Aaron},
  doi          = {10.1038/s43588-025-00795-9},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {481-491},
  shortjournal = {Nat. Comput. Sci},
  title        = {Gender and racial diversity socialization in science},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing real-time infectious disease forecasting using large language models. <em>NATCS</em>, <em>5</em>(6), 467-480. (<a href='https://doi.org/10.1038/s43588-025-00798-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Forecasting the short-term spread of an ongoing disease outbreak poses a challenge owing to the complexity of contributing factors, some of which can be characterized through interlinked, multi-modality variables, and the intersection of public policy and human behavior. Here we introduce PandemicLLM, a framework with multi-modal large language models (LLMs) that reformulates real-time forecasting of disease spread as a text-reasoning problem, with the ability to incorporate real-time, complex, non-numerical information. This approach, through an artificial intelligence–human cooperative prompt design and time-series representation learning, encodes multi-modal data for LLMs. The model is applied to the COVID-19 pandemic, and trained to utilize textual public health policies, genomic surveillance, spatial and epidemiological time-series data, and is tested across all 50 states of the United States for a duration of 19 months. PandemicLLM opens avenues for incorporating various pandemic-related data in heterogeneous formats and shows performance benefits over existing models. PandemicLLM adapts the large language model to predict disease trends by converting diverse disease-relevant data into text. It responds to new variants in real time, offering robust, interpretable forecasts for effective public health responses.},
  archive      = {J_NATCS},
  author       = {Du, Hongru and Zhao, Yang and Zhao, Jianan and Xu, Shaochong and Lin, Xihong and Chen, Yiran and Gardner, Lauren M. and Yang, Hao ‘Frank’},
  doi          = {10.1038/s43588-025-00798-6},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {467-480},
  shortjournal = {Nat. Comput. Sci},
  title        = {Advancing real-time infectious disease forecasting using large language models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bi-level identification of governing equations for nonlinear physical systems. <em>NATCS</em>, <em>5</em>(6), 456-466. (<a href='https://doi.org/10.1038/s43588-025-00804-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying governing equations from observational data is crucial for understanding nonlinear physical systems but remains challenging due to the risk of overfitting. Here we introduce the Bi-Level Identification of Equations (BILLIE) framework, which simultaneously discovers and validates equations using a hierarchical optimization strategy. The policy gradient algorithm of reinforcement learning is leveraged to achieve the bi-level optimization. We demonstrate BILLIE’s superior performance through comparisons with baseline methods in canonical nonlinear systems such as turbulent flows and three-body systems. Furthermore, we apply the BILLIE framework to discover RNA and protein velocity equations directly from single-cell sequencing data. The equations identified by BILLIE outperform empirical models in predicting cellular differentiation states, underscoring BILLIE’s potential to reveal fundamental physical laws across a wide range of scientific fields. In this study the authors introduce BILLIE, a bi-level framework for equation identification that decouples term selection and quantification, enhancing robustness and accuracy in modeling nonlinear systems. BILLIE outperforms existing methods in handling complex systems and imperfect data, as demonstrated through both simulations and real-world biological applications.},
  archive      = {J_NATCS},
  author       = {Li, Zeyu and Yuan, Huining and Han, Wang and Hou, Yimin and Li, Hongjue and Ding, Haidong and Jiang, Zhiguo and Yang, Lijun},
  doi          = {10.1038/s43588-025-00804-x},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {456-466},
  shortjournal = {Nat. Comput. Sci},
  title        = {Bi-level identification of governing equations for nonlinear physical systems},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Interpretable niche-based cell‒cell communication inference using multi-view graph neural networks. <em>NATCS</em>, <em>5</em>(6), 444-455. (<a href='https://doi.org/10.1038/s43588-025-00809-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell‒cell communication (CCC) is a fundamental biological process for the harmonious functioning of biological systems. Increasing evidence indicates that cells of the same type or cluster may exhibit different interaction patterns under varying niches, yet most prevailing methods perform CCC inference at the cell type or cluster level while disregarding niche heterogeneity. Here we introduce the Spatial Transcriptomics-based cell‒cell Communication And Subtype Exploration (STCase) tool, which can describe CCC events at the single-cell/spot level based on spatial transcriptomics (ST). STCase includes an interpretable multi-view graph neural network via CCC-aware attention to identify niches for each cell type and uncover niche-specific CCC events. We show that STCase outperforms state-of-the-art approaches and accurately captures reported immune-related CCC events in human bronchial glands. We also identify three distinct niches of oral squamous cell carcinoma that may be obscured by agglomerative methods, and discover niche-specific CCC events that could influence tumor prognosis. An interpretable tool called STCase is introduced to leverage a multi-view graph neural network based on cell‒cell communication (CCC)-aware attention to uncover functional niche-specific CCC events based on spatial transcriptomics data.},
  archive      = {J_NATCS},
  author       = {Qi, Juntian and Luo, Zhengchao and Li, Chuan-Yun and Wang, Jinzhuo and Ding, Wanqiu},
  doi          = {10.1038/s43588-025-00809-6},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {444-455},
  shortjournal = {Nat. Comput. Sci},
  title        = {Interpretable niche-based cell‒cell communication inference using multi-view graph neural networks},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How molecular changes impact brain states and whole-brain activity: A multiscale approach. <em>NATCS</em>, <em>5</em>(6), 442-443. (<a href='https://doi.org/10.1038/s43588-025-00813-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting how molecular changes affect brain activity is a challenge in neuroscience. We introduced a multiscale modeling approach to simulate these microscopic changes and how they impact macroscale brain activity. This approach predicted how the anesthetic action on synaptic receptors can lead to the transitions in macroscale brain activity observed empirically.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00813-w},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {442-443},
  shortjournal = {Nat. Comput. Sci},
  title        = {How molecular changes impact brain states and whole-brain activity: A multiscale approach},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). When disruption endures. <em>NATCS</em>, <em>5</em>(6), 440-441. (<a href='https://doi.org/10.1038/s43588-025-00812-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new framework disentangles the nature of disruption in science, revealing how rare but persistent breakthroughs shake the foundations of research fields while remaining central to future work.},
  archive      = {J_NATCS},
  author       = {Funk, Russell J. and Wu, Xiangting},
  doi          = {10.1038/s43588-025-00812-x},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {440-441},
  shortjournal = {Nat. Comput. Sci},
  title        = {When disruption endures},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Leveraging large language models for pandemic preparedness. <em>NATCS</em>, <em>5</em>(6), 438-439. (<a href='https://doi.org/10.1038/s43588-025-00805-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A framework with large language models is proposed to predict disease spread in real-time by incorporating complex, multi-modal information and using a artificial intelligence–human cooperative prompt design.},
  archive      = {J_NATCS},
  author       = {Dixit, Narendra M.},
  doi          = {10.1038/s43588-025-00805-w},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {438-439},
  shortjournal = {Nat. Comput. Sci},
  title        = {Leveraging large language models for pandemic preparedness},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Making titles and abstracts broadly accessible. <em>NATCS</em>, <em>5</em>(6), 437. (<a href='https://doi.org/10.1038/s43588-025-00835-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We discuss different strategies to improve the writing of titles and abstracts.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00835-4},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {437},
  shortjournal = {Nat. Comput. Sci},
  title        = {Making titles and abstracts broadly accessible},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Benchmarking the performance of quantum computing software for quantum circuit creation, manipulation and compilation. <em>NATCS</em>, <em>5</em>(5), 427-435. (<a href='https://doi.org/10.1038/s43588-025-00792-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present Benchpress, a benchmarking suite for evaluating the performance and range of functionality of multiple quantum computing software development kits. This suite consists of a collection of over 1,000 tests measuring key performance metrics for a wide variety of operations on quantum circuits composed of up to 930 qubits and $${\mathcal{O}}(1{0}^{6})$$ two-qubit gates, as well as an execution framework for running the tests over multiple quantum software packages in a unified manner. Here we give a detailed overview of the benchmark suite and its methodology and generate representative results over seven different quantum software packages. The flexibility of the Benchpress framework enables benchmarking that not only keeps pace with quantum hardware improvements but also can preemptively gauge the quantum circuit processing costs of future device architectures. In this Resource, the authors present an open-source extensible benchmark tool, Benchpress, for evaluating the performance of mainstream quantum computing software. Benchpress was demonstrated to perform over 1,000 tests with up to 930 qubits to compare the performance of quantum software, providing insight into how to best use current programming stacks.},
  archive      = {J_NATCS},
  author       = {Nation, Paul D. and Saki, Abdullah Ash and Brandhofer, Sebastian and Bello, Luciano and Garion, Shelly and Treinish, Matthew and Javadi-Abhari, Ali},
  doi          = {10.1038/s43588-025-00792-y},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {427-435},
  shortjournal = {Nat. Comput. Sci},
  title        = {Benchmarking the performance of quantum computing software for quantum circuit creation, manipulation and compilation},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Extending atomic decomposition and many-body representation with a chemistry-motivated approach to machine learning potentials. <em>NATCS</em>, <em>5</em>(5), 418-426. (<a href='https://doi.org/10.1038/s43588-025-00790-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most widely used machine learning potentials for condensed-phase applications rely on many-body permutationally invariant polynomial or atom-centered neural networks. However, these approaches face challenges in achieving chemical interpretability in atomistic energy decomposition and fully matching the computational efficiency of traditional force fields. Here we present a method that combines aspects of both approaches and balances accuracy and force-field-level speed. This method utilizes a monomer-centered representation, where the potential energy is decomposed into the sum of chemically meaningful monomeric energies. The structural descriptors of monomers are described by one-body and two-body effective interactions, enforced by appropriate sets of permutationally invariant polynomials as inputs to the feed-forward neural networks. Systematic assessments of models for gas-phase water trimer, liquid water, methane–water cluster and liquid carbon dioxide are performed. The improved accuracy, efficiency and flexibility of this method have promise for constructing accurate machine learning potentials and enabling large-scale quantum and classical simulations for complex molecular systems. A machine-learning-potential framework achieves an optimal balance of accuracy and efficiency through monomeric decomposition. Systematic evaluations highlight its potential in large-scale simulations of complex molecular systems.},
  archive      = {J_NATCS},
  author       = {Yu, Qi and Ma, Ruitao and Qu, Chen and Conte, Riccardo and Nandi, Apurba and Pandey, Priyanka and Houston, Paul L. and Zhang, Dong H. and Bowman, Joel M.},
  doi          = {10.1038/s43588-025-00790-0},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {418-426},
  shortjournal = {Nat. Comput. Sci},
  title        = {Extending atomic decomposition and many-body representation with a chemistry-motivated approach to machine learning potentials},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A computational approach to evaluate how molecular mechanisms impact large-scale brain activity. <em>NATCS</em>, <em>5</em>(5), 405-417. (<a href='https://doi.org/10.1038/s43588-025-00796-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Assessing the impact of pharmaceutical compounds on brain activity is a critical issue in contemporary neuroscience. Currently, no systematic approach exists for evaluating these effects in whole-brain models, which typically focus on macroscopic phenomena, while pharmaceutical interventions operate at the molecular scale. Here we address this issue by presenting a computational approach for brain simulations using biophysically grounded mean-field models that integrate membrane conductances and synaptic receptors, showcased in the example of anesthesia. We show that anesthetics targeting GABAA and NMDA receptors can switch brain activity to generalized slow-wave patterns, as observed experimentally in deep anesthesia. To validate our models, we demonstrate that these slow-wave states exhibit reduced responsiveness to external stimuli and functional connectivity constrained by anatomical connectivity, mirroring experimental findings in anesthetized states across species. Our approach, founded on mean-field models that incorporate molecular realism, provides a robust framework for understanding how molecular-level drug actions impact whole-brain dynamics. A computational framework is introduced to study how molecular changes impact brain activity, using biophysically grounded mean-field models to evaluate how drugs acting on synaptic receptors lead to emergent changes in large-scale brain activity.},
  archive      = {J_NATCS},
  author       = {Sacha, Maria and Tesler, Federico and Cofre, Rodrigo and Destexhe, Alain},
  doi          = {10.1038/s43588-025-00796-8},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {405-417},
  shortjournal = {Nat. Comput. Sci},
  title        = {A computational approach to evaluate how molecular mechanisms impact large-scale brain activity},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DNA data storage for biomedical images using HELIX. <em>NATCS</em>, <em>5</em>(5), 397-404. (<a href='https://doi.org/10.1038/s43588-025-00793-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deoxyribonucleic acid (DNA) data storage is expected to become a key medium for large-scale data. Biomedical data images typically require substantial storage space over extended periods, making them ideal candidates for DNA data storage. However, existing DNA data storage models are primarily designed for generic files and lack a comprehensive retrieval system for biomedical images. Here, to address this, we propose HELIX, a DNA-based storage system for biomedical images. HELIX introduces an image-compression algorithm tailored to the characteristics of biomedical images, achieving high compression rates and robust error tolerance. In addition, HELIX incorporates an error-correcting encoding algorithm that eliminates the need for indexing, enhancing storage density and decoding speed. We utilize a deep learning-based image repair algorithm for the predictive restoration of partially missing image blocks. In our in vitro experiments, we successfully stored two spatiotemporal genomics images. This sequencing process achieved 97.20% image quality at a depth of 7× coverage. A DNA-based storage system for biomedical images is proposed, combining compression, error correction and deep learning repair. It achieves 97.20% image quality at 7× coverage depth, demonstrating high compression and fault tolerance.},
  archive      = {J_NATCS},
  author       = {Qu, Guanjin and Yan, Zihui and Chen, Xin and Wu, Huaming},
  doi          = {10.1038/s43588-025-00793-x},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {397-404},
  shortjournal = {Nat. Comput. Sci},
  title        = {DNA data storage for biomedical images using HELIX},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evaluating and mitigating bias in AI-based medical text generation. <em>NATCS</em>, <em>5</em>(5), 388-396. (<a href='https://doi.org/10.1038/s43588-025-00789-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence (AI) systems, particularly those based on deep learning models, have increasingly achieved expert-level performance in medical applications. However, there is growing concern that such AI systems may reflect and amplify human bias, reducing the quality of their performance in historically underserved populations. The fairness issue has attracted considerable research interest in the medical imaging classification field, yet it remains understudied in the text-generation domain. In this study, we investigate the fairness problem in text generation within the medical field and observe substantial performance discrepancies across different races, sexes and age groups, including intersectional groups, various model scales and different evaluation metrics. To mitigate this fairness issue, we propose an algorithm that selectively optimizes those underserved groups to reduce bias. Our evaluations across multiple backbones, datasets and modalities demonstrate that our proposed algorithm enhances fairness in text generation without compromising overall performance. This study evaluates bias in AI-generated medical text, revealing disparities across race, sex and age. An optimization method is proposed to enhance fairness without compromising performance, offering a step toward more equitable AI in healthcare.},
  archive      = {J_NATCS},
  author       = {Chen, Xiuying and Wang, Tairan and Zhou, Juexiao and Song, Zirui and Gao, Xin and Zhang, Xiangliang},
  doi          = {10.1038/s43588-025-00789-7},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {388-396},
  shortjournal = {Nat. Comput. Sci},
  title        = {Evaluating and mitigating bias in AI-based medical text generation},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Probabilistic photonic computing for AI. <em>NATCS</em>, <em>5</em>(5), 377-387. (<a href='https://doi.org/10.1038/s43588-025-00800-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Probabilistic computing excels in approximating combinatorial problems and modeling uncertainty. However, using conventional deterministic hardware for probabilistic models is challenging: (pseudo) random number generation introduces computational overhead and additional data shuffling. Therefore, there is a pressing need for different probabilistic computing architectures that achieve low latencies with reasonable energy consumption. Physical computing offers a promising solution, as these systems do not rely on an abstract deterministic representation of data but directly encode the information in physical quantities, enabling inherent probabilistic architectures utilizing entropy sources. Photonic computing is a prominent variant of physical computing due to the large available bandwidth, several orthogonal degrees of freedom for data encoding and optimal properties for in-memory computing and parallel data transfer. Here, we highlight key developments in physical photonic computing and photonic random number generation. We further provide insights into the realization of probabilistic photonic processors and their impact on artificial intelligence systems and future challenges. Physical computing, particularly photonic computing, offers a promising alternative by directly encoding data in physical quantities, enabling efficient probabilistic computing. This Perspective discusses the challenges and opportunities in photonic probabilistic computing and its applications in artificial intelligence.},
  archive      = {J_NATCS},
  author       = {Brückerhoff-Plückelmann, Frank and Ovvyan, Anna P. and Varri, Akhil and Borras, Hendrik and Klein, Bernhard and Meyer, Lennart and Wright, C. David and Bhaskaran, Harish and Syed, Ghazi Sarwat and Sebastian, Abu and Fröning, Holger and Pernice, Wolfram},
  doi          = {10.1038/s43588-025-00800-1},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {377-387},
  shortjournal = {Nat. Comput. Sci},
  title        = {Probabilistic photonic computing for AI},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Leveraging generative models with periodicity-aware, invertible and invariant representations for crystalline materials design. <em>NATCS</em>, <em>5</em>(5), 365-376. (<a href='https://doi.org/10.1038/s43588-025-00797-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Designing periodicity-aware, invariant and invertible representations provides an opportunity for the inverse design of crystalline materials with desired properties by generative models. This objective requires optimizing representations and refining the architecture of generative models, yet its feasibility remains uncertain, given current progress in molecular inverse generation. In this Perspective, we highlight the progress of various methods for designing representations and generative schemes for crystalline materials, discuss the challenges in the field and propose a roadmap for future developments. The inverse design of functional crystalline materials via generative models is a rapidly growing field, but one that faces challenges in representation and generation architectures. This Perspective systematically examines these limitations and explores strategies for future improvement.},
  archive      = {J_NATCS},
  author       = {Wang, Zhilong and You, Fengqi},
  doi          = {10.1038/s43588-025-00797-7},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {365-376},
  shortjournal = {Nat. Comput. Sci},
  title        = {Leveraging generative models with periodicity-aware, invertible and invariant representations for crystalline materials design},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A quest toward comprehensive benchmarking of quantum computing software. <em>NATCS</em>, <em>5</em>(5), 363-364. (<a href='https://doi.org/10.1038/s43588-025-00803-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A comprehensive open-source benchmarking suite is presented. It can be used to evaluate the performance and functionality of various quantum software development kits for manipulating and compiling quantum circuits.},
  archive      = {J_NATCS},
  author       = {Zhu, Chenghong and Zhang, Lei and Wang, Xin},
  doi          = {10.1038/s43588-025-00803-y},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {363-364},
  shortjournal = {Nat. Comput. Sci},
  title        = {A quest toward comprehensive benchmarking of quantum computing software},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Toward fair AI-driven medical text generation. <em>NATCS</em>, <em>5</em>(5), 361-362. (<a href='https://doi.org/10.1038/s43588-025-00807-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study assesses bias in artificial intelligence (AI)-generated medical language to find differences in age, sex, and ethnicity. An optimization technique is proposed to improve fairness without sacrificing performance.},
  archive      = {J_NATCS},
  author       = {Zhang, Yumeng and Song, Jiangning},
  doi          = {10.1038/s43588-025-00807-8},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {361-362},
  shortjournal = {Nat. Comput. Sci},
  title        = {Toward fair AI-driven medical text generation},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Algorithms for reliable decision-making need causal reasoning. <em>NATCS</em>, <em>5</em>(5), 356-360. (<a href='https://doi.org/10.1038/s43588-025-00814-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Decision-making inherently involves cause–effect relationships that introduce causal challenges. We argue that reliable algorithms for decision-making need to build upon causal reasoning. Addressing these causal challenges requires explicit assumptions about the underlying causal structure to ensure identifiability and estimatability, which means that the computational methods must successfully align with decision-making objectives in real-world tasks.},
  archive      = {J_NATCS},
  author       = {Kern, Christoph and Fischer-Abaigar, Unai and Schweisthal, Jonas and Frauen, Dennis and Ghani, Rayid and Feuerriegel, Stefan and van der Schaar, Mihaela and Kreuter, Frauke},
  doi          = {10.1038/s43588-025-00814-9},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {356-360},
  shortjournal = {Nat. Comput. Sci},
  title        = {Algorithms for reliable decision-making need causal reasoning},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Making the most of presubmission enquiries. <em>NATCS</em>, <em>5</em>(5), 355. (<a href='https://doi.org/10.1038/s43588-025-00817-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We provide recommendations on how to write an effective presubmission enquiry.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00817-6},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {355},
  shortjournal = {Nat. Comput. Sci},
  title        = {Making the most of presubmission enquiries},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A dynamic block activation framework for continuum models. <em>NATCS</em>, <em>5</em>(4), 345-354. (<a href='https://doi.org/10.1038/s43588-025-00780-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Efficient utilization of massively parallel computing resources is crucial for advancing scientific understanding through complex simulations. However, existing adaptive methods often face challenges in implementation complexity and scalability on modern parallel hardware. Here we present dynamic block activation (DBA), an acceleration framework that can be applied to a broad range of continuum simulations by strategically allocating resources on the basis of the dynamic features of the physical model. By exploiting the hierarchical structure of parallel hardware and dynamically activating and deactivating computation blocks, DBA optimizes performance while maintaining accuracy. We demonstrate DBA’s effectiveness through solving representative models spanning multiple scientific fields, including materials science, biophysics and fluid dynamics, achieving 216–816 central processing unit core-equivalent speedups on a single graphics processing unit (GPU), up to fivefold acceleration compared with highly optimized GPU code and nearly perfect scalability up to 32 GPUs. By addressing common challenges, such as divergent memory access, and reducing programming burden, DBA offers a promising approach to fully leverage massively parallel systems across multiple scientific computing domains. An acceleration framework for a broad range of continuum models improves the performance of numerical simulations, without compromising accuracy, by selectively allocating computational resources to regions of interest.},
  archive      = {J_NATCS},
  author       = {Zhang, Ruoyao and Xia, Yang},
  doi          = {10.1038/s43588-025-00780-2},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {345-354},
  shortjournal = {Nat. Comput. Sci},
  title        = {A dynamic block activation framework for continuum models},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multimodal learning for mapping genotype–phenotype dynamics. <em>NATCS</em>, <em>5</em>(4), 333-344. (<a href='https://doi.org/10.1038/s43588-024-00765-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How complex phenotypes emerge from intricate gene expression patterns is a fundamental question in biology. Integrating high-content genotyping approaches such as single-cell RNA sequencing and advanced learning methods such as language models offers an opportunity for dissecting this complex relationship. Here we present a computational integrated genetics framework designed to analyze and interpret the high-dimensional landscape of genotypes and their associated phenotypes simultaneously. We applied this approach to develop a multimodal foundation model to explore the genotype–phenotype relationship manifold for human transcriptomics at the cellular level. Analyzing this joint manifold showed a refined resolution of cellular heterogeneity, uncovered potential cross-tissue biomarkers and provided contextualized embeddings to investigate the polyfunctionality of genes shown for the von Willebrand factor (VWF) gene in endothelial cells. Overall, this study advances our understanding of the dynamic interplay between gene expression and phenotypic manifestation and demonstrates the potential of integrated genetics in uncovering new dimensions of cellular function and complexity. A multimodal computational framework is proposed to integrate single-cell RNA sequencing data with phenotypic information to map complex genotype–phenotype relationships. This approach helps to refine cellular heterogeneity analysis, identify cross-tissue biomarkers and reveal polyfunctional characteristics of genes with cellular resolution.},
  archive      = {J_NATCS},
  author       = {Khodaee, Farhan and Zandie, Rohola and Edelman, Elazer R.},
  doi          = {10.1038/s43588-024-00765-7},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {333-344},
  shortjournal = {Nat. Comput. Sci},
  title        = {Multimodal learning for mapping genotype–phenotype dynamics},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Free-energy machine for combinatorial optimization. <em>NATCS</em>, <em>5</em>(4), 322-332. (<a href='https://doi.org/10.1038/s43588-025-00782-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Finding optimal solutions to combinatorial optimization problems (COPs) is pivotal in both scientific and industrial domains. Considerable efforts have been invested on developing accelerated methods utilizing sophisticated models and advanced computational hardware. However, the challenge remains to achieve both high efficiency and broad generality in problem-solving. Here we propose a general method, free-energy machine (FEM), based on the ideas of free-energy minimization in statistical physics, combined with automatic differentiation and gradient-based optimization in machine learning. FEM flexibly addresses various COPs within a unified framework and efficiently leverages parallel computational devices such as graphics processing units. We benchmark FEM on diverse COPs including maximum cut, balanced minimum cut and maximum k-satisfiability, scaled to millions of variables, across synthetic and real-world instances. The findings indicate that FEM remarkably outperforms state-of-the-art algorithms tailored for individual COP in both efficiency and efficacy, demonstrating the potential of combining statistical physics and machine learning for broad applications. This work introduces the free-energy machine (FEM), which combines statistical physics principles and machine learning techniques to solve various combinatorial optimization problems. FEM outperforms state-of-the-art algorithms in both efficiency and effectiveness across multiple combinatorial optimization problems, demonstrating its broad applicability.},
  archive      = {J_NATCS},
  author       = {Shen, Zi-Song and Pan, Feng and Wang, Yao and Men, Yi-Ding and Xu, Wen-Biao and Yung, Man-Hong and Zhang, Pan},
  doi          = {10.1038/s43588-025-00782-0},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {322-332},
  shortjournal = {Nat. Comput. Sci},
  title        = {Free-energy machine for combinatorial optimization},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Active twisting for adaptive droplet collection. <em>NATCS</em>, <em>5</em>(4), 313-321. (<a href='https://doi.org/10.1038/s43588-025-00786-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many xeric plant leaves exhibit bending and twisting morphology, which may contribute to their important biological and physical functions adapted to drought and desert conditions. Revealing the relationships between various morphologies and functionalities can inspire device designs for meeting increasingly stringent environmental requirements. Here, demonstrated on the biomimetic bilayer ribbons made of liquid crystal elastomers, we reveal that the stimulus-induced morphological evolution of bending, spiraling, twisting and various coupling states among them can be selectively achieved and precisely tuned by designing the director orientations in liquid crystal elastomer bilayers. The mathematical models and analytical solutions are developed to quantify the morphology selection and phase transition of these liquid crystal elastomer ribbons for material design, as confirmed by experiments. Moreover, we show that, under activation and control of external stimuli, the twisting configuration can be harnessed to effectively collect and guide the transportation of droplets, and enhance the structural stiffness for resisting wind blow and rainfall to achieve the optimal configuration for water collection. Our results reveal the interesting functions correlated with bending, spiraling and twisting morphologies widely present in the natural world, by providing fundamental insights into their shape transformation and controlling factors. This work also demonstrates a potential application with integrating morphogenesis–environment interactions into devices or equipments. Using theoretical and mechanical models, this study reveals that active twisting of most xeric plant leaves contributes to effective droplet collection and wind resistance in extreme environments. The twisting configuration is used to design a biomimetic plant cluster, enabling tunable water collection and transportation.},
  archive      = {J_NATCS},
  author       = {Yang, Yifan and Dai, Zhijun and Chen, Yuzhen and Xu, Fan},
  doi          = {10.1038/s43588-025-00786-w},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {313-321},
  shortjournal = {Nat. Comput. Sci},
  title        = {Active twisting for adaptive droplet collection},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rapid traversal of vast chemical space using machine learning-guided docking screens. <em>NATCS</em>, <em>5</em>(4), 301-312. (<a href='https://doi.org/10.1038/s43588-025-00777-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The accelerating growth of make-on-demand chemical libraries provides unprecedented opportunities to identify starting points for drug discovery with virtual screening. However, these multi-billion-scale libraries are challenging to screen, even for the fastest structure-based docking methods. Here we explore a strategy that combines machine learning and molecular docking to enable rapid virtual screening of databases containing billions of compounds. In our workflow, a classification algorithm is trained to identify top-scoring compounds based on molecular docking of 1 million compounds to the target protein. The conformal prediction framework is then used to make selections from the multi-billion-scale library, reducing the number of compounds to be scored by docking. The CatBoost classifier showed an optimal balance between speed and accuracy and was used to adapt the workflow for screens of ultralarge libraries. Application to a library of 3.5 billion compounds demonstrated that our protocol can reduce the computational cost of structure-based virtual screening by more than 1,000-fold. Experimental testing of predictions identified ligands of G protein-coupled receptors and demonstrated that our approach enables discovery of compounds with multi-target activity tailored for therapeutic effect. Combining conformal prediction machine learning with molecular docking, a method to efficiently screen multi-billion-scale libraries is developed, enabling the discovery of a dual-target ligand modulating the A2A adenosine and D2 dopamine receptors.},
  archive      = {J_NATCS},
  author       = {Luttens, Andreas and Cabeza de Vaca, Israel and Sparring, Leonard and Brea, José and Martínez, Antón Leandro and Kahlous, Nour Aldin and Radchenko, Dmytro S. and Moroz, Yurii S. and Loza, María Isabel and Norinder, Ulf and Carlsson, Jens},
  doi          = {10.1038/s43588-025-00777-x},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {301-312},
  shortjournal = {Nat. Comput. Sci},
  title        = {Rapid traversal of vast chemical space using machine learning-guided docking screens},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Toward a unified benchmark and framework for deep learning-based prediction of nuclear magnetic resonance chemical shifts. <em>NATCS</em>, <em>5</em>(4), 292-300. (<a href='https://doi.org/10.1038/s43588-025-00783-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The study of structure–spectrum relationships is essential for spectral interpretation, impacting structural elucidation and material design. Predicting spectra from molecular structures is challenging due to their complex relationships. Here we introduce NMRNet, a deep learning framework using the SE(3) Transformer for atomic environment modeling, following a pretraining and fine-tuning paradigm. To support the evaluation of nuclear magnetic resonance chemical shift prediction models, we have established a comprehensive benchmark based on previous research and databases, covering diverse chemical systems. Applying NMRNet to these benchmark datasets, we achieve competitive performance in both liquid-state and solid-state nuclear magnetic resonance datasets, demonstrating its robustness and practical utility in real-world scenarios. Our work helps to advance deep learning applications in analytical and structural chemistry. A deep learning framework (NMRNet) is developed to model atomic environments for predicting NMR chemical shifts. A benchmark dataset, nmrshiftdb2-2024, is also established to provide a standardized source for evaluating NMR models.},
  archive      = {J_NATCS},
  author       = {Xu, Fanjie and Guo, Wentao and Wang, Feng and Yao, Lin and Wang, Hongshuai and Tang, Fujie and Gao, Zhifeng and Zhang, Linfeng and E, Weinan and Tian, Zhong-Qun and Cheng, Jun},
  doi          = {10.1038/s43588-025-00783-z},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {292-300},
  shortjournal = {Nat. Comput. Sci},
  title        = {Toward a unified benchmark and framework for deep learning-based prediction of nuclear magnetic resonance chemical shifts},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Physics-based modeling in the new era of enzyme engineering. <em>NATCS</em>, <em>5</em>(4), 279-291. (<a href='https://doi.org/10.1038/s43588-025-00788-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Enzyme engineering is entering a new era characterized by the integration of computational strategies. While bioinformatics and artificial intelligence methods have been extensively applied to accelerate the screening of function-enhancing mutants, physics-based modeling methods, such as molecular mechanics and quantum mechanics, are essential complements in many objectives. In this Perspective, we highlight how physics-based modeling will help the field of computational enzyme engineering reach its full potential by exploring current developments, unmet challenges and emerging opportunities for tool development. This Perspective highlights the vital role of physics-based modeling in computational enzyme engineering, exploring key advances, challenges and future steps. By integrating machine learning, these approaches can enhance each other, unlocking the full potential of enzyme design and discovery.},
  archive      = {J_NATCS},
  author       = {Jurich, Christopher and Shao, Qianzhen and Ran, Xinchun and Yang, Zhongyue J.},
  doi          = {10.1038/s43588-025-00788-8},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {279-291},
  shortjournal = {Nat. Comput. Sci},
  title        = {Physics-based modeling in the new era of enzyme engineering},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Integrating statistical physics and machine learning for combinatorial optimization. <em>NATCS</em>, <em>5</em>(4), 277-278. (<a href='https://doi.org/10.1038/s43588-025-00794-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce free-energy machine (FEM), an efficient and general method for solving combinatorial optimization problems. FEM combines free-energy minimization from statistical physics with gradient-based optimization techniques in machine learning and utilizes parallel computation, outperforming state-of-the-art algorithms and showcasing the synergy of merging statistical physics with machine learning.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00794-w},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {277-278},
  shortjournal = {Nat. Comput. Sci},
  title        = {Integrating statistical physics and machine learning for combinatorial optimization},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Xeric plants inspire adaptable liquid crystal elastomers for water collection. <em>NATCS</em>, <em>5</em>(4), 275-276. (<a href='https://doi.org/10.1038/s43588-025-00801-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inspired by the morphologies of xeric plant leaves, we have developed biomimetic liquid crystal elastomer bilayers that can bend, spiral and twist. These adaptive shape morphing structures can twist to improve water collection efficiency and wind resistance, suggesting their potential application in adaptive water collection and directional transportation.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00801-0},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {275-276},
  shortjournal = {Nat. Comput. Sci},
  title        = {Xeric plants inspire adaptable liquid crystal elastomers for water collection},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Towards efficient and adaptive simulations for continuum physics. <em>NATCS</em>, <em>5</em>(4), 273-274. (<a href='https://doi.org/10.1038/s43588-025-00791-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The continuous drive for efficiency in high-performance computing has led to the development of new frameworks aimed at optimizing large-scale simulations. One such advancement is dynamic block activation, a method designed to significantly accelerate continuum models while making full use of modern computing architectures that combine central processing units and graphics processing units.},
  archive      = {J_NATCS},
  author       = {Scapin, Nicolò},
  doi          = {10.1038/s43588-025-00791-z},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {273-274},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards efficient and adaptive simulations for continuum physics},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Towards transparency and knowledge exchange in AI-assisted data analysis code generation. <em>NATCS</em>, <em>5</em>(4), 271-272. (<a href='https://doi.org/10.1038/s43588-025-00781-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Haase, Robert},
  doi          = {10.1038/s43588-025-00781-1},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {271-272},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards transparency and knowledge exchange in AI-assisted data analysis code generation},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Generative molecular design and discovery on the rise. <em>NATCS</em>, <em>5</em>(4), 269-270. (<a href='https://doi.org/10.1038/s43588-025-00802-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Computational Science is calling all researchers who develop and use generative models for molecular design and discovery to publish with us.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00802-z},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {269-270},
  shortjournal = {Nat. Comput. Sci},
  title        = {Generative molecular design and discovery on the rise},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficient crystal structure prediction based on the symmetry principle. <em>NATCS</em>, <em>5</em>(3), 255-267. (<a href='https://doi.org/10.1038/s43588-025-00775-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Crystal structure prediction (CSP) is an evolving field aimed at discerning crystal structures with minimal prior information. Despite the success of various CSP algorithms, their practical applicability remains circumscribed, particularly for large and complex systems. Here, to address this challenge, we show an evolutionary structure generator within the MAGUS (Machine Learning and Graph Theory Assisted Universal Structure Searcher) framework, inspired by the symmetry principle. This generator extracts both global and local features of explored crystal structures using group and graph theory. By integrating an on-the-fly space group miner and fragment reorganizer, augmented by symmetry-kept mutation, our approach generates higher-quality initial structures, reducing the computational costs of CSP tasks. Benchmarking tests show up to fourfold performance improvements. The method also proves valid in complex phosphorus allotrope systems. Furthermore, we apply our approach to the diamond–silicon (111)-(7 × 7) surface system, identifying up to 42 metastable structures within an 18 meV Å−2 energy range, demonstrating the efficacy of our approach in navigating challenging search spaces. This study presents a symmetry principle-biased crystal structure prediction scheme within the MAGUS framework, achieving up to a fourfold performance improvement compared with state-of-the-art methods.},
  archive      = {J_NATCS},
  author       = {Han, Yu and Ding, Chi and Wang, Junjie and Gao, Hao and Shi, Jiuyang and Yu, Shaobo and Jia, Qiuhan and Pan, Shuning and Sun, Jian},
  doi          = {10.1038/s43588-025-00775-z},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {255-267},
  shortjournal = {Nat. Comput. Sci},
  title        = {Efficient crystal structure prediction based on the symmetry principle},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Harnessing large language models for data-scarce learning of polymer properties. <em>NATCS</em>, <em>5</em>(3), 245-254. (<a href='https://doi.org/10.1038/s43588-025-00768-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs) bear promise as a fast and accurate material modeling paradigm for evaluation, analysis and design. Their vast number of trainable parameters necessitates a wealth of data to achieve accuracy and mitigate overfitting. However, experimental measurements are often limited and costly to obtain in sufficient quantities for fine-tuning. To this end, here we present a physics-based training pipeline that tackles the pathology of data scarcity. The core enabler is a physics-based modeling framework that generates a multitude of synthetic data to align the LLM to a physically consistent initial state before fine-tuning. Our framework features a two-phase training strategy: utilizing the large-in-amount but less accurate synthetic data for supervised pretraining, and fine-tuning the phase-1 model with limited experimental data. We empirically demonstrate that supervised pretraining is vital to obtaining accurate fine-tuned LLMs, via the lens of learning polymer flammability metrics where cone calorimeter data are sparse. A physics-based training pipeline is developed to help tackle the challenges of data scarcity. The framework aligns large language models to a physically consistent initial state that is fine-tuned for learning polymer properties.},
  archive      = {J_NATCS},
  author       = {Liu, Ning and Jafarzadeh, Siavash and Lattimer, Brian Y. and Ni, Shuna and Lua, Jim and Yu, Yue},
  doi          = {10.1038/s43588-025-00768-y},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {245-254},
  shortjournal = {Nat. Comput. Sci},
  title        = {Harnessing large language models for data-scarce learning of polymer properties},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Decoupled peak property learning for efficient and interpretable electronic circular dichroism spectrum prediction. <em>NATCS</em>, <em>5</em>(3), 234-244. (<a href='https://doi.org/10.1038/s43588-024-00757-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Electronic circular dichroism (ECD) spectra contain key information about molecular chirality by discriminating the absolute configurations of chiral molecules, which is crucial in asymmetric organic synthesis and the drug industry. However, existing predictive approaches lack the consideration of ECD spectra owing to the data scarcity and the limited interpretability to achieve trustworthy prediction. Here we establish a large-scale dataset for chiral molecular ECD spectra and propose ECDFormer for accurate and interpretable ECD spectrum prediction. ECDFormer decomposes ECD spectra into peak entities, uses the QFormer architecture to learn peak properties and renders peaks into spectra. Compared with spectrum sequence prediction methods, our decoupled peak prediction approach substantially enhances both accuracy and efficiency, improving the peak symbol accuracy from 37.3% to 72.7% and decreasing the time cost from an average of 4.6 central processing unit hours to 1.5 s. Moreover, ECDFormer demonstrated its ability to capture molecular orbital information directly from spectral data using the explainable peak-decoupling approach. Furthermore, ECDFormer proved to be equally proficient at predicting various types of spectrum, including infrared and mass spectroscopies, highlighting its substantial generalization capabilities. A large-scale electronic circular dichroism spectrum dataset is proposed and the ECDFormer framework is developed to achieve accurate and interpretable ECD spectrum prediction for natural products.},
  archive      = {J_NATCS},
  author       = {Li, Hao and Long, Da and Yuan, Li and Wang, Yu and Tian, Yonghong and Wang, Xinchang and Mo, Fanyang},
  doi          = {10.1038/s43588-024-00757-7},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {234-244},
  shortjournal = {Nat. Comput. Sci},
  title        = {Decoupled peak property learning for efficient and interpretable electronic circular dichroism spectrum prediction},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantifying associations between socio-spatial factors and cognitive development in the ABCD cohort. <em>NATCS</em>, <em>5</em>(3), 221-233. (<a href='https://doi.org/10.1038/s43588-025-00774-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the mounting demand for generative population models, their limited generalizability to underrepresented demographic groups hinders widespread adoption in real-world applications. Here we propose a diversity-aware population modeling framework that can guide targeted strategies in public health and education, by estimating subgroup-level effects and stratifying predictions to capture sociodemographic variability. We leverage Bayesian multilevel regression and post-stratification to systematically quantify inter-individual differences in the relationship between socioeconomic status and cognitive development. Post-stratification enhanced the interpretability of model predictions across underrepresented groups by incorporating US Census data to gain additional insights into smaller subgroups in the Adolescent Brain Cognitive Development Study. This ensured that predictions were not skewed by overly heterogeneous or homogeneous representations. Our analyses underscore the importance of combining Bayesian multilevel modeling with post-stratification to validate reliability and provide a more holistic explanation of sociodemographic disparities in our diversity-aware population modeling framework. The study proposes a diversity-aware population modeling framework that can guide targeted strategies in public health, using Bayesian multilevel regression and post-stratification to quantify sociodemographic disparities in cognitive development.},
  archive      = {J_NATCS},
  author       = {Osayande, Nicole and Marotta, Justin and Aggarwal, Shambhavi and Kopal, Jakub and Holmes, Avram and Yip, Sarah W. and Bzdok, Danilo},
  doi          = {10.1038/s43588-025-00774-0},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {221-233},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantifying associations between socio-spatial factors and cognitive development in the ABCD cohort},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Leveraging pharmacovigilance data to predict population-scale toxicity profiles of checkpoint inhibitor immunotherapy. <em>NATCS</em>, <em>5</em>(3), 207-220. (<a href='https://doi.org/10.1038/s43588-024-00748-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Immune checkpoint inhibitor (ICI) therapies have made considerable advances in cancer immunotherapy, but the complex and diverse spectrum of ICI-induced toxicities poses substantial challenges to treatment outcomes and computational analysis. Here we introduce DySPred, a dynamic graph convolutional network-based deep learning framework, to map and predict the toxicity profiles of ICIs at the population level by leveraging large-scale real-world pharmacovigilance data. DySPred accurately predicts toxicity risks across diverse demographic cohorts and cancer types, demonstrating resilience in small-sample scenarios and revealing toxicity trends over time. Furthermore, DySPred consistently aligns the toxicity-safety profiles of small-molecule antineoplastic agents with their drug-induced transcriptional alterations. Our study provides a versatile methodology for population-level profiling of ICI-induced toxicities, enabling proactive toxicity monitoring and timely tailoring of treatment and intervention strategies in the advancement of cancer immunotherapy. A deep learning framework is proposed with real-world pharmacovigilance data to predict population-scale toxicity profiles of checkpoint inhibitor immunotherapy, enabling proactive toxicity monitoring and timely tailoring of treatment.},
  archive      = {J_NATCS},
  author       = {Yan, Dongxue and Bao, Siqi and Zhang, Zicheng and Sun, Jie and Zhou, Meng},
  doi          = {10.1038/s43588-024-00748-8},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {207-220},
  shortjournal = {Nat. Comput. Sci},
  title        = {Leveraging pharmacovigilance data to predict population-scale toxicity profiles of checkpoint inhibitor immunotherapy},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comprehensive prediction and analysis of human protein essentiality based on a pretrained large language model. <em>NATCS</em>, <em>5</em>(3), 196-206. (<a href='https://doi.org/10.1038/s43588-024-00733-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human essential proteins (HEPs) are indispensable for individual viability and development. However, experimental methods to identify HEPs are often costly, time consuming and labor intensive. In addition, existing computational methods predict HEPs only at the cell line level, but HEPs vary across living human, cell line and animal models. Here we develop a sequence-based deep learning model, Protein Importance Calculator (PIC), by fine-tuning a pretrained protein language model. PIC not only substantially outperforms existing methods for predicting HEPs but also provides comprehensive prediction results across three levels: human, cell line and mouse. Furthermore, we define the protein essential score, derived from PIC, to quantify human protein essentiality and validate its effectiveness by a series of biological analyses. We also demonstrate the biomedical value of the protein essential score by identifying potential prognostic biomarkers for breast cancer and quantifying the essentiality of 617,462 human microproteins. This study introduces the Protein Importance Calculator (PIC), a deep learning model designed to predict human essential proteins (HEPs) crucial for survival and development. Unlike conventional methods, PIC offers a comprehensive assessment of HEPs across three levels: humans, cell lines and mice.},
  archive      = {J_NATCS},
  author       = {Kang, Boming and Fan, Rui and Cui, Chunmei and Cui, Qinghua},
  doi          = {10.1038/s43588-024-00733-1},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {196-206},
  shortjournal = {Nat. Comput. Sci},
  title        = {Comprehensive prediction and analysis of human protein essentiality based on a pretrained large language model},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Diversity-aware population modeling. <em>NATCS</em>, <em>5</em>(3), 194-195. (<a href='https://doi.org/10.1038/s43588-025-00787-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We propose a diversity-aware population modeling framework using Bayesian multilevel regression and post-stratification to quantify sociodemographic disparities in cognitive development. Our approach improved subgroup estimates, guiding targeted public health strategies and addressing biases in traditional models to support more equitable decision-making.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00787-9},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {194-195},
  shortjournal = {Nat. Comput. Sci},
  title        = {Diversity-aware population modeling},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boosting crystal structure prediction via symmetry. <em>NATCS</em>, <em>5</em>(3), 192-193. (<a href='https://doi.org/10.1038/s43588-025-00779-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting stable crystal structures for complex systems that involve multiple elements or a large number of atoms presents a formidable challenge in computational materials science. A recent study presents an efficient crystal-structure search method for this task, utilizing symmetry and graph theory.},
  archive      = {J_NATCS},
  author       = {Wang, Yanchao},
  doi          = {10.1038/s43588-025-00779-9},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {192-193},
  shortjournal = {Nat. Comput. Sci},
  title        = {Boosting crystal structure prediction via symmetry},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An organelle-agnostic image analysis tool. <em>NATCS</em>, <em>5</em>(3), 191. (<a href='https://doi.org/10.1038/s43588-025-00785-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Badri, Michelle},
  doi          = {10.1038/s43588-025-00785-x},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {191},
  shortjournal = {Nat. Comput. Sci},
  title        = {An organelle-agnostic image analysis tool},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). On the unknowable limits to prediction. <em>NATCS</em>, <em>5</em>(3), 188-190. (<a href='https://doi.org/10.1038/s43588-025-00776-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Yan, Jiani and Rahal, Charles},
  doi          = {10.1038/s43588-025-00776-y},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {188-190},
  shortjournal = {Nat. Comput. Sci},
  title        = {On the unknowable limits to prediction},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Celebrating a pioneer in bioinformatics. <em>NATCS</em>, <em>5</em>(3), 187. (<a href='https://doi.org/10.1038/s43588-025-00784-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In honor of the 100th birthday of Margaret Dayhoff, we spotlight her footprint in the field of bioinformatics.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00784-y},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {187},
  shortjournal = {Nat. Comput. Sci},
  title        = {Celebrating a pioneer in bioinformatics},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Spatial modeling algorithms for reactions and transport in biological cells. <em>NATCS</em>, <em>5</em>(2), 185. (<a href='https://doi.org/10.1038/s43588-025-00773-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Francis, Emmet A. and Laughlin, Justin G. and Dokken, Jørgen S. and Finsberg, Henrik N. T. and Lee, Christopher T. and Rognes, Marie E. and Rangamani, Padmini},
  doi          = {10.1038/s43588-025-00773-1},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {185},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Spatial modeling algorithms for reactions and transport in biological cells},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Approaching coupled-cluster accuracy for molecular electronic structures with multi-task learning. <em>NATCS</em>, <em>5</em>(2), 184. (<a href='https://doi.org/10.1038/s43588-025-00767-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Tang, Hao and Xiao, Brian and He, Wenhao and Subasic, Pero and Harutyunyan, Avetik R. and Wang, Yao and Liu, Fang and Xu, Haowei and Li, Ju},
  doi          = {10.1038/s43588-025-00767-z},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {184},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Approaching coupled-cluster accuracy for molecular electronic structures with multi-task learning},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A programmable environment for shape optimization and shapeshifting problems. <em>NATCS</em>, <em>5</em>(2), 170-183. (<a href='https://doi.org/10.1038/s43588-024-00749-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Soft materials underpin many domains of science and engineering, including soft robotics, structured fluids, and biological and particulate media. In response to applied mechanical, electromagnetic or chemical stimuli, such materials typically change shape, often dramatically. Predicting their structure is of great interest to facilitate design and mechanistic understanding, and can be cast as an optimization problem where a given energy function describing the physics of the material is minimized with respect to the shape of the domain and additional fields. However, shape-optimization problems are very challenging to solve, and there is a lack of suitable simulation tools that are both readily accessible and general in purpose. Here we present an open-source programmable environment, Morpho, and demonstrate its versatility by showcasing a range of applications from different areas of soft-matter physics: swelling hydrogels, complex fluids that form aspherical droplets, soap films and membranes, and filaments. This study introduces an extensible framework—Morpho—for shape optimization, enabling researchers to predict the structure of soft materials, such as complex fluids, gels, particulate and biological materials.},
  archive      = {J_NATCS},
  author       = {Joshi, Chaitanya and Hellstein, Daniel and Wennerholm, Cole and Downey, Eoghan and Hamilton, Emmett and Hocking, Samuel and Andrei, Anca S. and Adler, James H. and Atherton, Timothy J.},
  doi          = {10.1038/s43588-024-00749-7},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {170-183},
  shortjournal = {Nat. Comput. Sci},
  title        = {A programmable environment for shape optimization and shapeshifting problems},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A spatiotemporal style transfer algorithm for dynamic visual stimulus generation. <em>NATCS</em>, <em>5</em>(2), 155-169. (<a href='https://doi.org/10.1038/s43588-024-00746-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding how visual information is encoded in biological and artificial systems often requires the generation of appropriate stimuli to test specific hypotheses, but available methods for video generation are scarce. Here we introduce the spatiotemporal style transfer (STST) algorithm, a dynamic visual stimulus generation framework that allows the manipulation and synthesis of video stimuli for vision research. We show how stimuli can be generated that match the low-level spatiotemporal features of their natural counterparts, but lack their high-level semantic features, providing a useful tool to study object recognition. We used these stimuli to probe PredNet, a predictive coding deep network, and found that its next-frame predictions were not disrupted by the omission of high-level information, with human observers also confirming the preservation of low-level features and lack of high-level information in the generated stimuli. We also introduce a procedure for the independent spatiotemporal factorization of dynamic stimuli. Testing such factorized stimuli on humans and deep vision models suggests a spatial bias in how humans and deep vision models encode dynamic visual information. These results showcase potential applications of the STST algorithm as a versatile tool for dynamic stimulus generation in vision science. The spatiotemporal style transfer (STST) algorithm enables video generation by selectively manipulating the spatial and temporal features of natural videos, fostering vision science research in both biological and artificial systems.},
  archive      = {J_NATCS},
  author       = {Greco, Antonino and Siegel, Markus},
  doi          = {10.1038/s43588-024-00746-w},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {155-169},
  shortjournal = {Nat. Comput. Sci},
  title        = {A spatiotemporal style transfer algorithm for dynamic visual stimulus generation},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Approaching coupled-cluster accuracy for molecular electronic structures with multi-task learning. <em>NATCS</em>, <em>5</em>(2), 144-154. (<a href='https://doi.org/10.1038/s43588-024-00747-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning plays an important role in quantum chemistry, providing fast-to-evaluate predictive models for various properties of molecules; however, most existing machine learning models for molecular electronic properties use density functional theory (DFT) databases as ground truth in training, and their prediction accuracy cannot surpass that of DFT. In this work we developed a unified machine learning method for electronic structures of organic molecules using the gold-standard CCSD(T) calculations as training data. Tested on hydrocarbon molecules, our model outperforms DFT with several widely used hybrid and double-hybrid functionals in terms of both computational cost and prediction accuracy of various quantum chemical properties. We apply the model to aromatic compounds and semiconducting polymers, evaluating both ground- and excited-state properties. The results demonstrate the model’s accuracy and generalization capability to complex systems that cannot be calculated using CCSD(T)-level methods due to scaling. A multi-task deep learning method for molecular electronic structures, called MEHnet, is developed to predict various molecular properties in a unified framework, approaching chemical accuracy while exhibiting local DFT-level computational costs.},
  archive      = {J_NATCS},
  author       = {Tang, Hao and Xiao, Brian and He, Wenhao and Subasic, Pero and Harutyunyan, Avetik R. and Wang, Yao and Liu, Fang and Xu, Haowei and Li, Ju},
  doi          = {10.1038/s43588-024-00747-9},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {144-154},
  shortjournal = {Nat. Comput. Sci},
  title        = {Approaching coupled-cluster accuracy for molecular electronic structures with multi-task learning},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A statistical framework for multi-trait rare variant analysis in large-scale whole-genome sequencing studies. <em>NATCS</em>, <em>5</em>(2), 125-143. (<a href='https://doi.org/10.1038/s43588-024-00764-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale whole-genome sequencing (WGS) studies have improved our understanding of the contributions of coding and noncoding rare variants to complex human traits. Leveraging association effect sizes across multiple traits in WGS rare variant association analysis can improve statistical power over single-trait analysis, and also detect pleiotropic genes and regions. Existing multi-trait methods have limited ability to perform rare variant analysis of large-scale WGS data. We propose MultiSTAAR, a statistical framework and computationally scalable analytical pipeline for functionally informed multi-trait rare variant analysis in large-scale WGS studies. MultiSTAAR accounts for relatedness, population structure and correlation among phenotypes by jointly analyzing multiple traits, and further empowers rare variant association analysis by incorporating multiple functional annotations. We applied MultiSTAAR to jointly analyze three lipid traits in 61,838 multi-ethnic samples from the Trans-Omics for Precision Medicine (TOPMed) Program. We discovered and replicated new associations with lipid traits missed by single-trait analysis. MultiSTAAR provides a general and flexible statistical framework for functionally informed multi-trait rare variant analysis of biobank-scale sequencing studies by jointly analyzing multiple traits and incorporating annotation information.},
  archive      = {J_NATCS},
  author       = {Li, Xihao and Chen, Han and Selvaraj, Margaret Sunitha and Van Buren, Eric and Zhou, Hufeng and Wang, Yuxuan and Sun, Ryan and McCaw, Zachary R. and Yu, Zhi and Jiang, Min-Zhi and DiCorpo, Daniel and Gaynor, Sheila M. and Dey, Rounak and Arnett, Donna K. and Benjamin, Emelia J. and Bis, Joshua C. and Blangero, John and Boerwinkle, Eric and Bowden, Donald W. and Brody, Jennifer A. and Cade, Brian E. and Carson, April P. and Carlson, Jenna C. and Chami, Nathalie and Chen, Yii-Der Ida and Curran, Joanne E. and de Vries, Paul S. and Fornage, Myriam and Franceschini, Nora and Freedman, Barry I. and Gu, Charles and Heard-Costa, Nancy L. and He, Jiang and Hou, Lifang and Hung, Yi-Jen and Irvin, Marguerite R. and Kaplan, Robert C. and Kardia, Sharon L. R. and Kelly, Tanika N. and Konigsberg, Iain and Kooperberg, Charles and Kral, Brian G. and Li, Changwei and Li, Yun and Lin, Honghuang and Liu, Ching-Ti and Loos, Ruth J. F. and Mahaney, Michael C. and Martin, Lisa W. and Mathias, Rasika A. and Mitchell, Braxton D. and Montasser, May E. and Morrison, Alanna C. and Naseri, Take and North, Kari E. and Palmer, Nicholette D. and Peyser, Patricia A. and Psaty, Bruce M. and Redline, Susan and Reiner, Alexander P. and Rich, Stephen S. and Sitlani, Colleen M. and Smith, Jennifer A. and Taylor, Kent D. and Tiwari, Hemant K. and Vasan, Ramachandran S. and Viali, Satupa’itea and Wang, Zhe and Wessel, Jennifer and Yanek, Lisa R. and Yu, Bing and Dupuis, Josée and Meigs, James B. and Auer, Paul L. and Raffield, Laura M. and Manning, Alisa K. and Rice, Kenneth M. and Rotter, Jerome I. and Peloso, Gina M. and Natarajan, Pradeep and Li, Zilin and Liu, Zhonghua and Lin, Xihong},
  doi          = {10.1038/s43588-024-00764-8},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {125-143},
  shortjournal = {Nat. Comput. Sci},
  title        = {A statistical framework for multi-trait rare variant analysis in large-scale whole-genome sequencing studies},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enabling efficient analysis of biobank-scale data with genotype representation graphs. <em>NATCS</em>, <em>5</em>(2), 112-124. (<a href='https://doi.org/10.1038/s43588-024-00739-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational analysis of a large number of genomes requires a data structure that can represent the dataset compactly while also enabling efficient operations on variants and samples. However, encoding genetic data in existing tabular data structures and file formats has become costly and unsustainable. Here we introduce the genotype representation graph (GRG), a fully connected hierarchical data structure that losslessly encodes phased whole-genome polymorphisms. Exploiting variant-sharing across samples enables GRG to compress 200,000 UK Biobank phased human genomes to 5–26 gigabytes per chromosome, also enabling graph-traversal algorithms to reuse computed values in random access memory. Constructing and processing GRG files scales to a million whole genomes. Using allele frequencies and association effects as examples, we show that computation on GRG via graph traversal runs the fastest among all tested alternatives. GRG-based algorithms have the potential to increase the scalability and reduce the cost of analyzing large genomic datasets. The genotype representation graph (GRG) is a compact data structure that encodes 200,000 human genomes in just 5–26 gigabytes per chromosome. Computation on GRG via graph traversal greatly accelerates genome-wide analysis.},
  archive      = {J_NATCS},
  author       = {DeHaas, Drew and Pan, Ziqing and Wei, Xinzhu},
  doi          = {10.1038/s43588-024-00739-9},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {112-124},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enabling efficient analysis of biobank-scale data with genotype representation graphs},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicting emergence of crystals from amorphous precursors with deep learning potentials. <em>NATCS</em>, <em>5</em>(2), 105-111. (<a href='https://doi.org/10.1038/s43588-024-00752-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Crystallization of amorphous precursors into metastable crystals plays a fundamental role in the formation of new matter, from geological to biological processes in nature to the synthesis and development of new materials in the laboratory. Reliably predicting the outcome of such a process would enable new research directions in these areas, but has remained beyond the reach of molecular modeling or ab initio methods. Here we show that candidates for the crystallization products of amorphous precursors can be predicted in many inorganic systems by sampling the local structural motifs at the atomistic level using universal deep learning interatomic potentials. We show that this approach identifies, with high accuracy, the most likely crystal structures of the polymorphs that initially nucleate from amorphous precursors, across a diverse set of material systems, including polymorphic oxides, nitrides, carbides, fluorides, chlorides, chalcogenides and metal alloys. This study introduces a2c, a computational method that leverages machine learning and atomistic simulations to predict the most likely crystallization products upon annealing of amorphous precursors. The a2c tool was demonstrated on a variety of materials, including oxides, nitrides and metallic glasses, and can assist researchers in discovering synthesis pathways for materials design.},
  archive      = {J_NATCS},
  author       = {Aykol, Muratahan and Merchant, Amil and Batzner, Simon and Wei, Jennifer N. and Cubuk, Ekin Dogus},
  doi          = {10.1038/s43588-024-00752-y},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {105-111},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting emergence of crystals from amorphous precursors with deep learning potentials},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A new tool for shape and structure optimization of soft materials. <em>NATCS</em>, <em>5</em>(2), 103-104. (<a href='https://doi.org/10.1038/s43588-024-00754-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present Morpho, an extensible programmable environment that uses finite elements for shape optimization in soft matter. Given an energy functional that incorporates physical boundaries and effects such as elasticity and electromagnetism, together with additional constraints to be satisfied, Morpho predicts the optimized shape and structure adopted by the material.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00754-w},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {103-104},
  shortjournal = {Nat. Comput. Sci},
  title        = {A new tool for shape and structure optimization of soft materials},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MultiSTAAR delivers multi-trait rare variant analysis of biobank-scale sequencing data. <em>NATCS</em>, <em>5</em>(2), 101-102. (<a href='https://doi.org/10.1038/s43588-025-00766-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying pleiotropic associations for rare variants in multi-ethnic biobank-scale whole-genome sequencing data poses considerable challenges. This study introduced MultiSTAAR as a scalable and robust multi-trait rare variant analysis framework designed for both coding and noncoding regions by integrating multiple variant functional annotations and leveraging multivariate modeling across diverse phenotypes.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00766-0},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {101-102},
  shortjournal = {Nat. Comput. Sci},
  title        = {MultiSTAAR delivers multi-trait rare variant analysis of biobank-scale sequencing data},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Shedding light on spatial signal transduction in cells using computational simulations. <em>NATCS</em>, <em>5</em>(2), 99-100. (<a href='https://doi.org/10.1038/s43588-025-00772-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present Spatial Modeling Algorithms for Reactions and Transport (SMART), a software package that simulates spatiotemporally detailed biochemical reaction networks within realistic cellular and subcellular geometries. This paper highlights the use of SMART in several biological test cases including cellular mechanotransduction, calcium signaling in neurons and cardiomyocytes, and adenosine triphosphate synthesis.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00772-2},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {99-100},
  shortjournal = {Nat. Comput. Sci},
  title        = {Shedding light on spatial signal transduction in cells using computational simulations},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biologically inspired graphs to explore massive genetic datasets. <em>NATCS</em>, <em>5</em>(2), 97-98. (<a href='https://doi.org/10.1038/s43588-024-00763-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study proposes a data structure that addresses crucial challenges related to storage and computation of large genome databases.},
  archive      = {J_NATCS},
  author       = {Layer, Ryan M.},
  doi          = {10.1038/s43588-024-00763-9},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {97-98},
  shortjournal = {Nat. Comput. Sci},
  title        = {Biologically inspired graphs to explore massive genetic datasets},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enhancing synthesis prediction via machine learning. <em>NATCS</em>, <em>5</em>(2), 95-96. (<a href='https://doi.org/10.1038/s43588-025-00771-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying promising synthesis targets and designing routes to their synthesis is a grand challenge in chemistry and materials science. Recent work employing machine learning in combination with traditional approaches is opening new ways to address this truly Herculean task.},
  archive      = {J_NATCS},
  author       = {Schön, J. C.},
  doi          = {10.1038/s43588-025-00771-3},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {95-96},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enhancing synthesis prediction via machine learning},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Balancing autonomy and expertise in autonomous synthesis laboratories. <em>NATCS</em>, <em>5</em>(2), 92-94. (<a href='https://doi.org/10.1038/s43588-025-00769-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Autonomous synthesis laboratories promise to streamline the plan–make–measure–analyze iteration loop. Here, we comment on the barriers in the field, the promise of a human on-the-loop approach, and strategies for optimizing accessibility, accuracy, and efficiency of autonomous laboratories.},
  archive      = {J_NATCS},
  author       = {Liu, Xiaozhao and Ouyang, Bin and Zeng, Yan},
  doi          = {10.1038/s43588-025-00769-x},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {92-94},
  shortjournal = {Nat. Comput. Sci},
  title        = {Balancing autonomy and expertise in autonomous synthesis laboratories},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Based on the science, diversity matters. <em>NATCS</em>, <em>5</em>(2), 91. (<a href='https://doi.org/10.1038/s43588-025-00778-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We reflect on what science tells us about the importance of diversity.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00778-w},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {91},
  shortjournal = {Nat. Comput. Sci},
  title        = {Based on the science, diversity matters},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial modeling algorithms for reactions and transport in biological cells. <em>NATCS</em>, <em>5</em>(1), 76-89. (<a href='https://doi.org/10.1038/s43588-024-00745-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biological cells rely on precise spatiotemporal coordination of biochemical reactions to control their functions. Such cell signaling networks have been a common focus for mathematical models, but they remain challenging to simulate, particularly in realistic cell geometries. Here we present Spatial Modeling Algorithms for Reactions and Transport (SMART), a software package that takes in high-level user specifications about cell signaling networks and then assembles and solves the associated mathematical systems. SMART uses state-of-the-art finite element analysis, via the FEniCS Project software, to efficiently and accurately resolve cell signaling events over discretized cellular and subcellular geometries. We demonstrate its application to several different biological systems, including yes-associated protein (YAP)/PDZ-binding motif (TAZ) mechanotransduction, calcium signaling in neurons and cardiomyocytes, and ATP generation in mitochondria. Throughout, we utilize experimentally derived realistic cellular geometries represented by well-conditioned tetrahedral meshes. These scenarios demonstrate the applicability, flexibility, accuracy and efficiency of SMART across a range of temporal and spatial scales. Spatial Modeling Algorithms for Reactions and Transport (SMART) is a software package that allows users to simulate spatially resolved biochemical signaling networks within realistic geometries of cells and organelles.},
  archive      = {J_NATCS},
  author       = {Francis, Emmet A. and Laughlin, Justin G. and Dokken, Jørgen S. and Finsberg, Henrik N. T. and Lee, Christopher T. and Rognes, Marie E. and Rangamani, Padmini},
  doi          = {10.1038/s43588-024-00745-x},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {76-89},
  shortjournal = {Nat. Comput. Sci},
  title        = {Spatial modeling algorithms for reactions and transport in biological cells},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Generative language models exhibit social identity biases. <em>NATCS</em>, <em>5</em>(1), 65-75. (<a href='https://doi.org/10.1038/s43588-024-00741-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Social identity biases, particularly the tendency to favor one’s own group (ingroup solidarity) and derogate other groups (outgroup hostility), are deeply rooted in human psychology and social behavior. However, it is unknown if such biases are also present in artificial intelligence systems. Here we show that large language models (LLMs) exhibit patterns of social identity bias, similarly to humans. By administering sentence completion prompts to 77 different LLMs (for instance, ‘We are…’), we demonstrate that nearly all base models and some instruction-tuned and preference-tuned models display clear ingroup favoritism and outgroup derogation. These biases manifest both in controlled experimental settings and in naturalistic human–LLM conversations. However, we find that careful curation of training data and specialized fine-tuning can substantially reduce bias levels. These findings have important implications for developing more equitable artificial intelligence systems and highlight the urgent need to understand how human–LLM interactions might reinforce existing social biases. Researchers show that large language models exhibit social identity biases similar to humans, having favoritism toward ingroups and hostility toward outgroups. These biases persist across models, training data and real-world human–LLM conversations.},
  archive      = {J_NATCS},
  author       = {Hu, Tiancheng and Kyrychenko, Yara and Rathje, Steve and Collier, Nigel and van der Linden, Sander and Roozenbeek, Jon},
  doi          = {10.1038/s43588-024-00741-1},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {65-75},
  shortjournal = {Nat. Comput. Sci},
  title        = {Generative language models exhibit social identity biases},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A simulated annealing algorithm for randomizing weighted networks. <em>NATCS</em>, <em>5</em>(1), 48-64. (<a href='https://doi.org/10.1038/s43588-024-00735-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientific discovery in connectomics relies on network null models. The prominence of network features is conventionally evaluated against null distributions estimated using randomized networks. Modern imaging technologies provide an increasingly rich array of biologically meaningful edge weights. Despite the prevalence of weighted graph analysis in connectomics, randomization models that only preserve binary node degree remain most widely used. Here we propose a simulated annealing procedure for generating randomized networks that preserve weighted degree (strength) sequences. We show that the procedure outperforms other rewiring algorithms and generalizes to multiple network formats, including directed and signed networks, as well as diverse real-world networks. Throughout, we use morphospace representation to assess the sampling behavior of the algorithm and the variability of the resulting ensemble. Finally, we show that accurate strength preservation yields different inferences about brain network organization. Collectively, this work provides a simple but powerful method to analyze richly detailed next-generation connectomics datasets. This study proposes an algorithm for generating randomized networks that preserve the weighted degree sequence. The procedure outperforms standard rewiring algorithms and extends to multiple network types, including directed and signed networks.},
  archive      = {J_NATCS},
  author       = {Milisav, Filip and Bazinet, Vincent and Betzel, Richard F. and Misic, Bratislav},
  doi          = {10.1038/s43588-024-00735-z},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {48-64},
  shortjournal = {Nat. Comput. Sci},
  title        = {A simulated annealing algorithm for randomizing weighted networks},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Resistive memory-based zero-shot liquid state machine for multimodal event data learning. <em>NATCS</em>, <em>5</em>(1), 37-47. (<a href='https://doi.org/10.1038/s43588-024-00751-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human brain is a complex spiking neural network (SNN) capable of learning multimodal signals in a zero-shot manner by generalizing existing knowledge. Remarkably, it maintains minimal power consumption through event-based signal propagation. However, replicating the human brain in neuromorphic hardware presents both hardware and software challenges. Hardware limitations, such as the slowdown of Moore’s law and Von Neumann bottleneck, hinder the efficiency of digital computers. In addition, SNNs are characterized by their software training complexities. Here, to this end, we propose a hardware–software co-design on a 40 nm 256 kB in-memory computing macro that physically integrates a fixed and random liquid state machine SNN encoder with trainable artificial neural network projections. We showcase the zero-shot learning of multimodal events on the N-MNIST and N-TIDIGITS datasets, including visual and audio data association, as well as neural and visual data alignment for brain–machine interfaces. Our co-design achieves classification accuracy comparable to fully optimized software models, resulting in a 152.83- and 393.07-fold reduction in training costs compared with state-of-the-art spiking recurrent neural network-based contrastive learning and prototypical networks, and a 23.34- and 160-fold improvement in energy efficiency compared with cutting-edge digital hardware, respectively. These proof-of-principle prototypes demonstrate zero-shot multimodal events learning capability for emerging efficient and compact neuromorphic hardware. This study presents a neuromorphic computing platform capable of learning cross-modal, event-driven signals for efficient real-time knowledge generalization. It also achieves zero-shot transfer learning for multimodal data.},
  archive      = {J_NATCS},
  author       = {Lin, Ning and Wang, Shaocong and Li, Yi and Wang, Bo and Shi, Shuhui and He, Yangu and Zhang, Woyu and Yu, Yifei and Zhang, Yue and Zhang, Xinyuan and Wong, Kwunhang and Wang, Songqi and Chen, Xiaoming and Jiang, Hao and Zhang, Xumeng and Lin, Peng and Xu, Xiaoxin and Qi, Xiaojuan and Wang, Zhongrui and Shang, Dashan and Liu, Qi and Liu, Ming},
  doi          = {10.1038/s43588-024-00751-z},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {37-47},
  shortjournal = {Nat. Comput. Sci},
  title        = {Resistive memory-based zero-shot liquid state machine for multimodal event data learning},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep bayesian active learning using in-memory computing hardware. <em>NATCS</em>, <em>5</em>(1), 27-36. (<a href='https://doi.org/10.1038/s43588-024-00744-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Labeling data is a time-consuming, labor-intensive and costly procedure for many artificial intelligence tasks. Deep Bayesian active learning (DBAL) boosts labeling efficiency exponentially, substantially reducing costs. However, DBAL demands high-bandwidth data transfer and probabilistic computing, posing great challenges for conventional deterministic hardware. Here we propose a memristor stochastic gradient Langevin dynamics in situ learning method that uses the stochastic of memristor modulation to learn efficiency, enabling DBAL within the computation-in-memory (CIM) framework. To prove the feasibility and effectiveness of the proposed method, we implemented in-memory DBAL on a memristor-based stochastic CIM system and successfully demonstrated a robot’s skill learning task. The inherent stochastic characteristics of memristors allow a four-layer memristor Bayesian deep neural network to efficiently identify and learn from uncertain samples. Compared with cutting-edge conventional complementary metal-oxide-semiconductor-based hardware implementation, the stochastic CIM system achieves a remarkable 44% boost in speed and could conserve 153 times more energy. This study introduces an in-memory deep Bayesian active learning framework that uses the stochastic properties of memristors for in situ probabilistic computations. This framework can greatly improve the efficiency and speed of artificial intelligence learning tasks, as demonstrated with a robot skill-learning task.},
  archive      = {J_NATCS},
  author       = {Lin, Yudeng and Gao, Bin and Tang, Jianshi and Zhang, Qingtian and Qian, He and Wu, Huaqiang},
  doi          = {10.1038/s43588-024-00744-y},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {27-36},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deep bayesian active learning using in-memory computing hardware},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficient scaling of large language models with mixture of experts and 3D analog in-memory computing. <em>NATCS</em>, <em>5</em>(1), 13-26. (<a href='https://doi.org/10.1038/s43588-024-00753-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models (LLMs), with their remarkable generative capacities, have greatly impacted a range of fields, but they face scalability challenges due to their large parameter counts, which result in high costs for training and inference. The trend of increasing model sizes is exacerbating these challenges, particularly in terms of memory footprint, latency and energy consumption. Here we explore the deployment of ‘mixture of experts’ (MoEs) networks—networks that use conditional computing to keep computational demands low despite having many parameters—on three-dimensional (3D) non-volatile memory (NVM)-based analog in-memory computing (AIMC) hardware. When combined with the MoE architecture, this hardware, utilizing stacked NVM devices arranged in a crossbar array, offers a solution to the parameter-fetching bottleneck typical in traditional models deployed on conventional von-Neumann-based architectures. By simulating the deployment of MoEs on an abstract 3D AIMC system, we demonstrate that, due to their conditional compute mechanism, MoEs are inherently better suited to this hardware than conventional, dense model architectures. Our findings suggest that MoEs, in conjunction with emerging 3D NVM-based AIMC, can substantially reduce the inference costs of state-of-the-art LLMs, making them more accessible and energy-efficient. This study shows a viable pathway to the efficient deployment of state-of-the-art large language models using mixture of experts on 3D analog in-memory computing hardware.},
  archive      = {J_NATCS},
  author       = {Büchel, Julian and Vasilopoulos, Athanasios and Simon, William Andrew and Boybat, Irem and Tsai, HsinYu and Burr, Geoffrey W. and Castro, Hernan and Filipiak, Bill and Le Gallo, Manuel and Rahimi, Abbas and Narayanan, Vijay and Sebastian, Abu},
  doi          = {10.1038/s43588-024-00753-x},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {13-26},
  shortjournal = {Nat. Comput. Sci},
  title        = {Efficient scaling of large language models with mixture of experts and 3D analog in-memory computing},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Energy-efficient multimodal zero-shot learning using in-memory reservoir computing. <em>NATCS</em>, <em>5</em>(1), 11-12. (<a href='https://doi.org/10.1038/s43588-024-00762-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To achieve an advanced neuromorphic computing system with brain-like energy efficiency and generalization capabilities, we propose a hardware–software co-design of in-memory reservoir computing. This co-design integrates a liquid state machine-based encoder with artificial neural network projections on a hybrid analog–digital system, demonstrating zero-shot learning for multimodal event data.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00762-w},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {11-12},
  shortjournal = {Nat. Comput. Sci},
  title        = {Energy-efficient multimodal zero-shot learning using in-memory reservoir computing},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Large language models act as if they are part of a group. <em>NATCS</em>, <em>5</em>(1), 9-10. (<a href='https://doi.org/10.1038/s43588-024-00750-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An extensive audit of large language models reveals that numerous models mirror the ‘us versus them’ thinking seen in human behavior. These social prejudices are likely captured from the biased contents of the training data.},
  archive      = {J_NATCS},
  author       = {Savcisens, Germans},
  doi          = {10.1038/s43588-024-00750-0},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {9-10},
  shortjournal = {Nat. Comput. Sci},
  title        = {Large language models act as if they are part of a group},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Memristors enabling probabilistic AI at the edge. <em>NATCS</em>, <em>5</em>(1), 7-8. (<a href='https://doi.org/10.1038/s43588-024-00761-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By combining several probabilistic AI algorithms, a recent study demonstrates experimentally that the inherent noise and variation in memristor nanodevices can be exploited as features for energy-efficient on-chip learning.},
  archive      = {J_NATCS},
  author       = {Querlioz, Damien},
  doi          = {10.1038/s43588-024-00761-x},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {7-8},
  shortjournal = {Nat. Comput. Sci},
  title        = {Memristors enabling probabilistic AI at the edge},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficient large language model with analog in-memory computing. <em>NATCS</em>, <em>5</em>(1), 5-6. (<a href='https://doi.org/10.1038/s43588-024-00760-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study demonstrates through numerical simulations that implementing large language models based on sparse mixture-of-experts architectures on 3D in-memory computing technologies can substantially reduce energy consumption.},
  archive      = {J_NATCS},
  author       = {Subramoney, Anand},
  doi          = {10.1038/s43588-024-00760-y},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {5-6},
  shortjournal = {Nat. Comput. Sci},
  title        = {Efficient large language model with analog in-memory computing},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bridging generations and cultures in mathematics and computer science. <em>NATCS</em>, <em>5</em>(1), 3-4. (<a href='https://doi.org/10.1038/s43588-024-00756-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Dellow, Alyssa April and Razak, Fatimah Abdul},
  doi          = {10.1038/s43588-024-00756-8},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {3-4},
  shortjournal = {Nat. Comput. Sci},
  title        = {Bridging generations and cultures in mathematics and computer science},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boosting AI with neuromorphic computing. <em>NATCS</em>, <em>5</em>(1), 1-2. (<a href='https://doi.org/10.1038/s43588-025-00770-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We highlight the important role of neuromorphic computing in enhancing the power efficiency and performance of AI.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00770-4},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {1-2},
  shortjournal = {Nat. Comput. Sci},
  title        = {Boosting AI with neuromorphic computing},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
