<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>EVLETT</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="evlett">EVLETT - 45</h2>
<ul>
<li><details>
<summary>
(2025). Repeated evolution of reduced visual investment at the onset of ecological speciation in high-altitude heliconius butterflies. <em>EVLETT</em>, <em>9</em>(4), 502-510. (<a href='https://doi.org/10.1093/evlett/qraf017'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Colonization of new habitats is typically followed by divergent selection acting on traits that are immediately important for fitness. For example, differences between sensory environments are often associated with variation in sensory traits critical for navigation and foraging. However, the extent to which the initial response to novel sensory conditions is mediated by phenotypic plasticity, and the contribution of sensory or neural adaptation to early species divergence remains unclear. We took advantage of repeated cases of speciation in Heliconius butterflies with independent allopatric distributions in the west of the Colombian and Ecuadorian Andes. Using volumetric brain measurements, we analyzed patterns of investment in primary sensory processing areas of the brain across different localities and habitats. We find that a higher altitude species, Heliconius chestertonii , differs in levels of investment in visual and olfactory brain components compared with its lower altitude relative H. erato venus , mainly attributable to broad-sense heritable variation as inferred from comparisons between wild and common-garden-reared individuals. We provide evidence that this variation is consistent with divergent selection, and compare these shifts with those reported for another high-altitude species, H. himera , and its parapatric lowland counterpart, H. erato cyrbia , to demonstrate parallel reductions in the size of specific optic lobe neuropils. Conversely, for the antennal lobe, we detected different trait shifts in H. himera and H. chestertonii relative to their lowland H. erato neighbors. Overall, our findings add weight to the adaptive potential of neuroanatomical divergence related to sensory processing during early species formation. Repeated associations between trait variation and environmental shifts may indicate adaptation to local sources of natural selection. For instance, in sticklebacks, independent populations in separate freshwater habitats have evolved reduced armored plating, suggesting equivalent phenotypic responses to shared sources of natural selection. We compared independent cases of ecological divergence in Heliconius butterflies distributed along altitudinal gradients from sea level to mid-elevation in the west of the Colombian and Ecuadorian Andes. Shifts in altitude involve repeated, abrupt transitions from wet, large-leaved, warm forests to higher altitude scrub forests that are drier and more exposed. We tested hypotheses about the role of these ecological shifts in driving adaptive evolution in neuroanatomical traits during speciation. We showed that in Heliconius , independent changes in forest type have been accompanied by heritable, parallel patterns of divergence in investment in visual processing in the brain. We propose these differences likely facilitate species divergence in the face of ongoing gene flow.},
  archive      = {J_EVLETT},
  author       = {Rivas-Sánchez, David F and Morris, Jake and Salazar, Camilo and Pardo-Díaz, Carolina and Merrill, Richard M and Montgomery, Stephen H},
  doi          = {10.1093/evlett/qraf017},
  journal      = {Evolution Letters},
  month        = {8},
  number       = {4},
  pages        = {502-510},
  shortjournal = {Evol. Lett.},
  title        = {Repeated evolution of reduced visual investment at the onset of ecological speciation in high-altitude heliconius butterflies},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The evolution of reversible plasticity in stable environments. <em>EVLETT</em>, <em>9</em>(4), 491-501. (<a href='https://doi.org/10.1093/evlett/qraf015'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reversible plasticity, i.e., the ability to deconstruct phenotypic specializations based on environmental conditions, is widespread in nature. Despite its ubiquity, few mathematical models have explored the evolutionary selection pressures that favor trait reversibility. Therefore, many scenarios remain to be examined. In particular, existing theory has modeled trait development as an instantaneous process. These models do not capture the fact that trait development is often a constructive process, in which phenotypes incrementally adjust to local ecologies. Here, we present an optimality model of the evolution of reversible plasticity in which organisms build traits incrementally. In our model, organisms repeatedly sample cues to infer the environmental state—which can vary between generations but not within generations—and incrementally tailor their phenotypes to match their environments. Organisms also have the option to deconstruct phenotypic adjustments. We investigate two different modes of phenotypic deconstruction: Organisms can either deconstruct phenotypic adjustments incrementally or completely deconstruct all phenotypic adjustments in one time period. We highlight two results. First, early-life sensitive periods in construction precede mid-ontogeny sensitive periods in deconstruction. Intriguingly, although organisms typically only deconstruct toward the end of ontogeny, environmental cues in mid-ontogeny have the strongest impact on deconstruction. Second, in contrast to previous models, we find that reversibility often evolves in environments that are stable within generations. Thus, our model shows that reversibility does not require environmental change during development—as long as organisms are initially uncertain about environmental conditions. Our model provides new insights into the capacity for reversibility in species that have evolved in ontogenetically stable environments.},
  archive      = {J_EVLETT},
  author       = {Walasek, Nicole and Panchanathan, Karthik and Frankenhuis, Willem E},
  doi          = {10.1093/evlett/qraf015},
  journal      = {Evolution Letters},
  month        = {8},
  number       = {4},
  pages        = {491-501},
  shortjournal = {Evol. Lett.},
  title        = {The evolution of reversible plasticity in stable environments},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Continent-wide differentiation of fitness traits and patterns of climate adaptation among european populations of drosophila melanogaster. <em>EVLETT</em>, <em>9</em>(4), 473-490. (<a href='https://doi.org/10.1093/evlett/qraf014'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A particularly well-studied evolutionary model is the vinegar fly Drosophila melanogaster , a cosmopolitan insect of ancestral southern-central African origin. Recent work suggests that it expanded out of Africa ∼9,000 years ago, and spread from the Middle East into Europe ∼1,800 years ago. During its global expansion, this human commensal adapted to novel climate zones and habitats. Despite much work on phenotypic differentiation and adaptation on several continents (especially North America and Australia), typically in the context of latitudinal clines, little is known about phenotypic divergence among European populations. Here, we sought to provide a continent-wide study of phenotypic differentiation among European populations of D. melanogaster . In a consortium-wide phenomics effort, we assayed 16 fitness-related traits on a panel of 173 isofemale lines from 9 European populations, with the majority of traits measured by several groups using semi-standardized protocols. For most fitness-related traits, we found significant differentiation among populations on a continental scale. Despite inevitable differences in assay conditions among labs, the reproducibility and hence robustness of our measurements were overall remarkably good. Several fitness components (e.g., viability, development time) exhibited significant latitudinal or longitudinal clines, and populations differed markedly in multivariate trait structure. Notably, populations experiencing higher humidity/rainfall and lower maximum temperature showed higher viability, fertility, starvation resistance, and lifespan at the expense of lower heat-shock survival, suggesting a pattern of local adaptation. Our results indicate that derived populations of this tropical fly have been shaped by pervasive spatially varying multivariate selection and adaptation to different climates on the European continent.},
  archive      = {J_EVLETT},
  author       = {Durmaz Mitchell, Esra and Kerdaffrec, Envel and Harney, Ewan and Paulo, Tânia F and Veselinovic, Marija Savic and Tanaskovic, Marija and Tyukmaeva, Venera and de Arcaya, Teresa Abaurrea Fernandez and Aksoy, Cansu and Argyridou, Eliza and Bailly, Tiphaine P M and Can, Dogus and Cobanoglu, Ezgi and Cook, Nicola and Coşkun, Seda and Davidovic, Slobodan and Demir, Ekin and Dias, Tânia and Rasouli-Dogaheh, Somayeh and Duque, Pedro and Eric, Katarina and Eric, Pavle and Erickson, Priscilla and Filipovski, Filip and Fishman, Bettina and Glaser-Schmitt, Amanda and Goldfischer, August and Green, Llewellyn and Janillon, Sonia and Jelic, Mihailo and Kostic, Hristina and Kreiman, Lucas E and Kremer, Natacha and Lyrakis, Manolis and Maistrenko, Oleksandr M and Marti, Sapho-Lou and McGunnigle, Megan and Merenciano, Miriam and Mira, Mário S and Montbel, Vincent and Mouton, Laurence and Mukha, Dmitry V and Murali, Siddharth and Patenkovic, Aleksandra and Protsenko, Oleksandra and Putero, Florencia A and Reis, Micael and Roshina, Natalia V and Rybina, Olga Y and Schou, Mads F and Schowing, Thibault and Senkal, Senel Selin and Serga, Svitlana and Trieu, Virginie and Symonenko, Alexander V and Trostnikov, Mikhail V and Tsybul'ko, Evgenia A and van den Heuvel, Joost and van Waarde, David and Veselkina, Ekaterina R and Vieira, Cristina P and Wang, Xiaocui and Zandveld, Jelle and Abbott, Jessica and Billeter, Jean-Christophe and Colinet, Hervé and Ebrahimi, Mehregan and Gibert, Patricia and Hrcek, Jan and Kankare, Maaria and Kozeretska, Iryna and Loeschcke, Volker and Mensch, Julián and Onder, Banu Sebnem and Parsch, John and Pasyukova, Elena G and Stamenkovic-Radak, Marina and Tauber, Eran and Vieira, Cristina and Wegener, Christian and Hoedjes, Katja M and Zwaan, Bas J and Betancourt, Andrea J and Fricke, Claudia and Grath, Sonja and Posnien, Nico and Vieira, Jorge and Kapun, Martin and Schlötterer, Christian and Schmidt, Paul and Sucena, Élio and González, Josefa and Bergland, Alan and Ritchie, Michael G and Flatt, Thomas},
  doi          = {10.1093/evlett/qraf014},
  journal      = {Evolution Letters},
  month        = {8},
  number       = {4},
  pages        = {473-490},
  shortjournal = {Evol. Lett.},
  title        = {Continent-wide differentiation of fitness traits and patterns of climate adaptation among european populations of drosophila melanogaster},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Diverging arabidopsis populations quickly accumulate pollen-acting genetic incompatibilities. <em>EVLETT</em>, <em>9</em>(4), 461-472. (<a href='https://doi.org/10.1093/evlett/qraf013'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The process by which species diverge from one another, gradually accumulate genetic incompatibilities, and eventually reach full-fledged reproductive isolation is a key question in evolutionary biology. However, the nature of reproductive barriers, the pace at which they accumulate, and their genomic distribution remain poorly documented. The disruption of co-adapted epistatic interactions in hybrids and the accumulation of selfish genetic elements are proposed contributors to this process, and can lead to the distortion of the Mendelian segregation of the affected loci across the genome. In this study we detect and quantify segregation distortion across the genomes of crosses produced from a diverse sampling of Arabidopsis lyrata and A. halleri populations, 2 species at the early stages of speciation and that can still interbreed. We observe no distortion loci in crosses with geographically and genetically similar parents, but both their frequency of occurrence and their magnitude become highly variable in more distant crosses. We also observe that distorter loci evolve rapidly, as they occur not only in interspecific hybrids, but also in intraspecific hybrids produced by crossing individuals from 2 isolated regions. Finally, we identify both genome-wide nonindependence and 2 specific genomic regions on different chromosomes where opposite distortion effects are repeatedly observed across multiple F1 individuals, suggesting negative epistasis is a major contributor to the evolution of hybrid segregation distortion. Our study demonstrates that pollen-acting segregation distortion is ubiquitous, and may contribute not only to the ongoing reproductive isolation between A. halleri and A. lyrata , but also between recently diverged populations of the same species.},
  archive      = {J_EVLETT},
  author       = {Condon, Christopher and Carpentier, Fantin and Tabourin, Marie and Wozniak, Natalia and Takou, Margarita and Blassiau, Christelle and Kumar, Vinod and Pietzenuk, Björn and Habert, Rémi and De Meaux, Juliette and Krämer, Ute and Roux, Camille and Corbett-Detig, Russell and Castric, Vincent},
  doi          = {10.1093/evlett/qraf013},
  journal      = {Evolution Letters},
  month        = {8},
  number       = {4},
  pages        = {461-472},
  shortjournal = {Evol. Lett.},
  title        = {Diverging arabidopsis populations quickly accumulate pollen-acting genetic incompatibilities},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evolutionary trends in the emergence of skeletal cell types. <em>EVLETT</em>, <em>9</em>(4), 446-460. (<a href='https://doi.org/10.1093/evlett/qraf012'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell types are fundamental functional units of multicellular organisms. The evolutionary emergence of new cell types is underpinned by genetic changes, such as gene co-option and cis-regulatory evolution, that propel the assembly or rewiring of molecular networks and give rise to new cell types with specialized functions. Here, we integrate genomic phylostratigraphy with single-cell transcriptomics to explore the evolutionary trends in the assembly of the skeletal cell type-specific gene expression programs. In particular, we investigate how the emergence of lineage-specific genes contributed to this process. We show that osteoblasts and hypertrophic chondrocytes (HC) express evolutionary younger transcriptomes compared to immature chondrocytes that resemble the ancestral skeletogenic program. We demonstrate that the recruitment of lineage-specific genes resulted in subsequent elaboration and individuation of the ancestral chondrogenic gene expression program, propelling the emergence of osteoblasts and HC. Notably, osteoblasts show significant enrichment of vertebrate-specific genes, while HC is enriched in gnathostome-specific genes. By identifying the functional properties of the recruited genes, coupled with the recently discovered fossil evidence, our study challenges the long-standing view on the evolution of vertebrate skeletal structures by suggesting that endochondral ossification and chondrocyte hypertrophy may have already evolved in the last common ancestors of gnathostomes.},
  archive      = {J_EVLETT},
  author       = {Damatac, Amor and Koska, Sara and Ullrich, Kristian K and Domazet-Lošo, Tomislav and Klimovich, Alexander and Kaucká, Markéta},
  doi          = {10.1093/evlett/qraf012},
  journal      = {Evolution Letters},
  month        = {8},
  number       = {4},
  pages        = {446-460},
  shortjournal = {Evol. Lett.},
  title        = {Evolutionary trends in the emergence of skeletal cell types},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). K-mer-based diversity scales with population size proxies more than nucleotide diversity in a meta-analysis of 98 plant species. <em>EVLETT</em>, <em>9</em>(4), 434-445. (<a href='https://doi.org/10.1093/evlett/qraf011'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A key prediction of neutral theory is that the level of genetic diversity in a population should scale with population size. However, as was noted by Richard Lewontin in 1974 and reaffirmed by later studies, the slope of the population size-diversity relationship in nature is much weaker than expected under neutral theory. We hypothesize that one contributor to this paradox is that current methods relying on single nucleotide polymorphisms (SNPs) called from aligning short reads to a reference genome underestimate levels of genetic diversity in many species. As a first step to testing this idea, we calculated nucleotide diversity ( ⁠ π ⁠ ) and k -mer-based metrics of genetic diversity across 112 plant species, amounting to over 205 terabases of DNA sequencing data from 27,488 individuals. After excluding 14 species with low coverage or no variant sites called, we compared how different diversity metrics correlated with proxies of population size that account for both range size and population density variation across species. We found that our population size proxies scaled anywhere from about 3 to over 20 times faster with k -mer diversity than nucleotide diversity after adjusting for evolutionary history, mating system, life cycle habit, cultivation status, and invasiveness. The relationship between k -mer diversity and population size proxies also remains significant after correcting for genome size, whereas the analogous relationship for nucleotide diversity does not. These results are consistent with the possibility that variation not captured by common SNP-based analyses explains part of Lewontin’s paradox in plants, but larger scale pangenomic studies are needed to definitively address this question. Even after many revolutions in our ability to sequence and understand DNA, many important biological questions remain unsolved. One such problem is Lewontin’s paradox, named after Richard Lewontin who first described it in 1974. The core of the paradox is a simple idea: species with more individuals should be more genetically diverse. The reasoning is that more individuals means more replication of DNA, and thus more opportunities for mutation to create new variation. However, species that differ massively in population size often have similar diversity levels. Lewontin’s paradox has several potential, previously investigated mechanisms but what if one contributor is simply that our measurements of genetic diversity are off? Most studies estimate diversity by comparing sample genomes to a standard reference genome. While this approach is useful, it is impossible to measure variation in DNA that is not represented in the reference—a phenomenon known as reference bias. We estimate metrics of diversity that are free of reference-bias and re-investigate Lewontin’s paradox in plants. Overall, we find that reference-free diversity metrics scale more with population size, compared with the reference-biased approach. While it is unlikely that reference-bias fully explains Lewontin’s paradox and larger-scale pangenomic studies are needed, our analyses are consistent with reference-bias contributing to Lewontin’s paradox.},
  archive      = {J_EVLETT},
  author       = {Roberts, Miles D and Josephs, Emily B},
  doi          = {10.1093/evlett/qraf011},
  journal      = {Evolution Letters},
  month        = {8},
  number       = {4},
  pages        = {434-445},
  shortjournal = {Evol. Lett.},
  title        = {K-mer-based diversity scales with population size proxies more than nucleotide diversity in a meta-analysis of 98 plant species},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Replicate geographic transects across a hybrid zone reveal parallelism and differences in the genetic architecture of reproductive isolation. <em>EVLETT</em>, <em>9</em>(4), 421-433. (<a href='https://doi.org/10.1093/evlett/qraf009'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Determining the genetic architecture of traits involved in adaptation and speciation is one of the key components of understanding the evolutionary mechanisms behind biological diversification. Hybrid zones provide a unique opportunity to use genetic admixture to identify traits and loci contributing to partial reproductive barriers between taxa. Many studies have focused on the temporal dynamics of hybrid zones, but geographical variation in hybrid zones that span distinct ecological contexts has received less attention. We address this knowledge gap by analyzing hybridization and introgression between black-capped and Carolina chickadees in two geographically remote transects across their extensive hybrid zone, one located in eastern and one in central North America. Previous studies demonstrated that this hybrid zone is moving northward as a result of climate change but is staying consistently narrow due to selection against hybrids. In addition, the hybrid zone is moving ~5× slower in central North America compared to more eastern regions, reflecting continent-wide variation in the rate of climate change. We use whole genome sequencing of 259 individuals to assess whether variation in the rate of hybrid zone movement is reflected in patterns of hybridization and introgression, and which genes and genomic regions show consistently restricted introgression in distinct ecological contexts. Our results highlight substantial similarities between geographically remote transects and reveal large Z-linked chromosomal rearrangements that generate measurable differences in the degree of gene flow between transects. We further use simulations and analyses of climatic data to examine potential factors contributing to continental-scale nuances in selection pressures. We discuss our findings in the context of speciation mechanisms and the importance of sex chromosome inversions in chickadees and other species.},
  archive      = {J_EVLETT},
  author       = {Semenov, Georgy and Kenyon, Haley and Funk, Erik and Anderson, William and McQuillan, Michael and Spinelli, Joan and Russell, Austin and Martinez, Noel and Van Huynh, Alex and Alexander, Alana and Schweizer, Rena and Linck, Ethan and Cheviron, Zachary and Carling, Matt and Roth, Timothy and Robbins, Mark and Rice, Amber and Taylor, Scott},
  doi          = {10.1093/evlett/qraf009},
  journal      = {Evolution Letters},
  month        = {8},
  number       = {4},
  pages        = {421-433},
  shortjournal = {Evol. Lett.},
  title        = {Replicate geographic transects across a hybrid zone reveal parallelism and differences in the genetic architecture of reproductive isolation},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Drosophila melanogaster pigmentation demonstrates adaptive phenotypic parallelism over multiple spatiotemporal scales. <em>EVLETT</em>, <em>9</em>(4), 408-420. (<a href='https://doi.org/10.1093/evlett/qraf008'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Populations are capable of responding to environmental change over ecological timescales via adaptive tracking. However, the translation from patterns of allele frequency change to rapid adaptation of complex traits remains unresolved. We used abdominal pigmentation in Drosophila melanogaster as a model phenotype to address the nature, genetic architecture, and repeatability of rapid adaptation in the field. We show that D. melanogaster pigmentation evolves as a highly parallel and deterministic response to shared environmental variation across latitude and season in natural North American populations. We then experimentally evolved replicate, genetically diverse fly populations in field mesocosms to remove any confounding effects of demography and/or cryptic structure that may drive patterns in wild populations; we show that pigmentation rapidly responds, in parallel, in fewer than 15 generations. Thus, pigmentation evolves concordantly in response to spatial and temporal climatic axes. We next examined whether phenotypic differentiation was associated with allele frequency change at loci with established links to genetic variance in pigmentation in natural populations. We found that across all spatial and temporal scales, phenotypic patterns were associated with variation at pigmentation-related loci, and the sets of genes we identified at each scale were largely nonoverlapping. Therefore, our findings suggest that parallel phenotypic evolution is associated with distinct components of the polygenic architecture shifting across each environmental axis to produce redundant adaptive patterns.},
  archive      = {J_EVLETT},
  author       = {Berardi, Skyler and Rhodes, Jessica A and Berner, Mary Catherine and Greenblum, Sharon I and Bitter, Mark C and Behrman, Emily L and Betancourt, Nicolas J and Bergland, Alan O and Petrov, Dmitri A and Rajpurohit, Subhash and Schmidt, Paul},
  doi          = {10.1093/evlett/qraf008},
  journal      = {Evolution Letters},
  month        = {8},
  number       = {4},
  pages        = {408-420},
  shortjournal = {Evol. Lett.},
  title        = {Drosophila melanogaster pigmentation demonstrates adaptive phenotypic parallelism over multiple spatiotemporal scales},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicting fitness in future climates: Insights from temporally replicated field experiments in arabidopsis thaliana. <em>EVLETT</em>, <em>9</em>(4), 392-407. (<a href='https://doi.org/10.1093/evlett/qraf007'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organisms are already facing climate change. To understand and mitigate the negative effects of climate change on wild and cultivated species, recent research has focused on predicting the fitness of organisms or populations in future climates. The accuracy of these predictions is, however, seldom tested. To test such predictions, we grew a set of 800 genetic families of the annual plant Arabidopsis thaliana in the same field site located in Northern France for 2 consecutive years with contrasted climates. Despite observing, in both years, a clear association between fitness and climatic distance between our field site and the climate of origin of these genetic families, the diverse set of methods we used failed to accurately predict fitness from a year to another. This low accuracy can be explained by the fact that different climatic factors contributed to climate adaptation in different years, which impeded the definition of a meaningful climate descriptor across years. Our results also suggest that populations of A. thaliana from Northern France already suffer from an adaptational lag with respect to climate, and that vegetative growth seems to be a more important trait for climate adaptation than phenology. We discuss the implications of our results for predicting the fitness of wild organisms in future climates and for breeding programs.},
  archive      = {J_EVLETT},
  author       = {Villoutreix, Romain and Faure, Nathalie and Glorieux, Cédric and Roux, Fabrice},
  doi          = {10.1093/evlett/qraf007},
  journal      = {Evolution Letters},
  month        = {8},
  number       = {4},
  pages        = {392-407},
  shortjournal = {Evol. Lett.},
  title        = {Predicting fitness in future climates: Insights from temporally replicated field experiments in arabidopsis thaliana},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dynamics of infection and immunity over 50 years as marine stickleback adapt to freshwater. <em>EVLETT</em>, <em>9</em>(4), 383-391. (<a href='https://doi.org/10.1093/evlett/qraf016'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {When a species colonizes a new environment, it may encounter new parasites to which its immune system is poorly adapted. After an initial spike in infection rates in the naïve founder population, the host may subsequently evolve increased immunity, thereby reducing infection rates. Here, we present an example of this eco-evolutionary process in a population of threespine stickleback ( Gasterosteus aculeatus ) that was founded in Heisholt Quarry, a man-made quarry pond, in 1967. Marine stickleback rarely encounter Schistocephalus solidus tapeworms (which require freshwater to hatch), and so remain highly susceptible to infection. Initially, introduced marine fish were heavily infected by S. solidus . They exhibited low levels of fibrosis, a heritable immune trait that some genotypes activate in response to infection, thereby suppressing tapeworm growth and viability. By the 1990s, the Heisholt Quarry population exhibited high rates of fibrosis, which partly suppressed S. solidus infection. This increased immune response led to reduced infection rates, and the tapeworm was apparently extirpated by 2021. Because fibrosis has a strong genetic basis in other stickleback populations, we infer that the newly founded stickleback–parasite interaction exhibits an eco-evolutionary process of increased immunity that effectively reduced infection. The infection and immune dynamics documented here closely match those expected from a simple eco-evo dynamic model presented here.},
  archive      = {J_EVLETT},
  author       = {Sriramulu, Pranav and Schluter, Dolph and Bolnick, Daniel I},
  doi          = {10.1093/evlett/qraf016},
  journal      = {Evolution Letters},
  month        = {8},
  number       = {4},
  pages        = {383-391},
  shortjournal = {Evol. Lett.},
  title        = {Dynamics of infection and immunity over 50 years as marine stickleback adapt to freshwater},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editorial: Accountability, voice, and trust - Responsible use of GenAI in scientific publishing. <em>EVLETT</em>, <em>9</em>(4), 381-382. (<a href='https://doi.org/10.1093/evlett/qraf027'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_EVLETT},
  doi          = {10.1093/evlett/qraf027},
  journal      = {Evolution Letters},
  month        = {8},
  number       = {4},
  pages        = {381-382},
  shortjournal = {Evol. Lett.},
  title        = {Editorial: Accountability, voice, and trust - Responsible use of GenAI in scientific publishing},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Correction to: Positive correlations in susceptibility to a diverse panel of viruses across drosophilidae host species. <em>EVLETT</em>, <em>9</em>(3), 379-380. (<a href='https://doi.org/10.1093/evlett/qraf010'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_EVLETT},
  doi          = {10.1093/evlett/qraf010},
  journal      = {Evolution Letters},
  month        = {6},
  number       = {3},
  pages        = {379-380},
  shortjournal = {Evol. Lett.},
  title        = {Correction to: Positive correlations in susceptibility to a diverse panel of viruses across drosophilidae host species},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Landscape structure as a driver of eco-evolution in host–parasite systems. <em>EVLETT</em>, <em>9</em>(3), 367-378. (<a href='https://doi.org/10.1093/evlett/qraf003'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial network structure of biological systems drives ecology and evolution by distributing organisms and their genes. The ubiquitous host–parasite systems are no exception. However, past theoretical work has largely focused on simple spatial structures, such as grids, hampering the translation of theoretical predictions to real ecosystems. Thus, we develop an eco-evolutionary metapopulation model of host–parasite dynamics where hosts and parasites disperse through realistically complex spatial networks representing major biomes using river-like and terrestrial-like networks. We generate the testable prediction that parasite virulence, or how parasites harm their hosts, peaks at intermediate dispersal values in river-like systems while it increases with increasing dispersal in terrestrial-like systems. In river-like systems, virulence also reaches higher overall values. Moreover, we show that kin selection is the main driver of virulence evolution. Spatial networks generate characteristic patterns of parasite relatedness which drive differential virulence evolution. Finally, we show that accounting for virulence evolution allows us to predict the distribution of key epidemiological variables (e.g., parasite extinction risks) within spatial networks. Our study highlights how eco-evolutionary feedbacks can be understood in light of spatial network structure by linking network topology to classical evolutionary mechanisms such as kin selection. Networks are everywhere, be it the internet, trading, road, or pathogen transmission networks. In nature, organisms are usually embedded in landscapes: networks of habitats on land, like connected forest patches, or river networks for freshwater-living organisms. Using computer models, we study this natural level of complexity, interaction networks within spatial networks, focusing on hosts and parasites, one of the most common species interaction on earth. Imagine looking at a landscape and being able to say whether parasites are harmful or not. This is what our model allows us to do. We focus on virulence, the degree to which parasites harm hosts, and show that aquatic and terrestrial parasites generally differ in virulence evolution that affects the stability of host–parasite systems.},
  archive      = {J_EVLETT},
  author       = {Deshpande, Jhelam N and Dakos, Vasilis and Kaltz, Oliver and Fronhofer, Emanuel A},
  doi          = {10.1093/evlett/qraf003},
  journal      = {Evolution Letters},
  month        = {6},
  number       = {3},
  pages        = {367-378},
  shortjournal = {Evol. Lett.},
  title        = {Landscape structure as a driver of eco-evolution in host–parasite systems},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Do the “big four” orders of insects comprise evolutionarily significant higher taxa with coherent patterns of selection on protein-coding genes?. <em>EVLETT</em>, <em>9</em>(3), 355-366. (<a href='https://doi.org/10.1093/evlett/qraf005'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Species are often treated as evolutionarily significant units of diversity that reflect patterns of gene flow and selection. In contrast, higher taxa are mostly regarded as convenient labels for levels in the tree of life, which reflect evolutionary history if defined cladistically but are assumed to have no real significance for ongoing evolution. We test the alternative hypothesis that some higher taxa are evolutionarily significant units with coherent patterns of selection on their constituent species. Specifically, we ask whether the big 4 orders of holometabolous insects, namely Coleoptera, Diptera, Hymenoptera, and Lepidoptera, display divergent, but internally conserved patterns of selection acting on protein-coding genes. Analyzing orthologous genes from whole genome sequence data for multiple species per order, we find that, in most genes, selection on roughly one fifth of codons is conserved within each order but differs significantly among orders. The shift is associated with variation in GC content among orders, but primarily at codon 2nd positions hence due to selection rather than mutational or repair bias. Comparison of alternative models assigning different taxonomic levels (either more lumped or divided than orders) shows that best models always specify Hymenoptera and Lepidoptera as coherent units, whereas patterns of selection on protein-coding genes within Coleoptera and especially Diptera are better explained by subdividing them further. We hypothesise that some aspect of the general lifestyle, body plan or genetic makeup of orders (or of nested clades within Coleoptera and Diptera) leads to conserved patterns of selection across protein-coding genes within them, whereas constraints differ among them. The emergence of whole-genome data for broad and deep phylogenetic samples will allow this hypothesis of evolutionarily significant higher taxa versus more evenly dispersed shifts in selection across genes to be tested further. Species are usually considered a natural unit of biodiversity defined by shared evolutionary processes. Higher taxa such as families and orders, on the contrary, are not generally thought to be evolutionarily significant units (ESUs), but instead reflect arbitrary levels within the branching tree of life. Yet, there could be processes acting across higher levels that maintain coherence, such as selection to constrain divergence of certain traits within higher taxa. We explored this possibility by analyzing whole genome sequences of the four megadiverse orders of holometabolous insects (Lepidoptera, Coleoptera, Diptera, and Hymenoptera). We compared different models of selection on protein-coding genes and found that around one fifth of codon positions show conserved evolution within orders but shift among orders. Hymenoptera and Lepidoptera are always found as single coherent units in the best models, whereas patterns of selection in Diptera and Coleoptera are better explained when these orders are subdivided further. This result opens the possibility for certain higher taxa to be considered natural entities. The increasing amount of genomic data will allow for this concept to be tested further in other taxonomic groups and at larger scale.},
  archive      = {J_EVLETT},
  author       = {Février, Pierre J and Barraclough, Timothy G},
  doi          = {10.1093/evlett/qraf005},
  journal      = {Evolution Letters},
  month        = {6},
  number       = {3},
  pages        = {355-366},
  shortjournal = {Evol. Lett.},
  title        = {Do the “big four” orders of insects comprise evolutionarily significant higher taxa with coherent patterns of selection on protein-coding genes?},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enemy dispersal drives the diversity loss of bacterial victim populations at diversity cold spots. <em>EVLETT</em>, <em>9</em>(3), 346-354. (<a href='https://doi.org/10.1093/evlett/qraf004'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dispersal between heterogenous habitats is a major determinant of population diversification, and may often introduce new morphotypes in habitats where population diversity is low. Natural enemies are also key factors affecting the diversification of victim populations. Co-dispersal of enemies may induce local diversity loss at diversity cold spots as enemies from diversity hots pots are often more efficient in predation. Here, we experimentally tested this hypothesis using a model microbial system: Pseudomonas fluorescens and its lytic phage. The ancestral bacterium diversified at three resource levels across eight temperature gradients in the presence and absence of phages. Bacteria diversified into more morphotypes at higher temperatures and higher resource levels when phages are absent, and dispersal increased population local diversity at low-diversity habitats. The presence of phages removed the differences in morphological diversity among different temperatures or resource levels. In addition, the co-dispersal of enemies caused higher morphotype loss at lower-quality habitats where the local bacteria are of lower resistance. The simultaneous dispersal of enemies and victims may have crucial consequences for population persistence in edge habitats. Individual dispersal affects population diversification and the maintenance of population diversity. Research on dispersal has focused on single taxon while interspecific interactions such as natural enemies can also determine population diversity through ecological and evolutionary mechanisms. The dispersal of enemies between diversity hot spots and cold spots may induce diversity loss of victim populations at diversity cold spots because enemies from diversity hots pots are likely to impose higher predation pressure. Here, we tested this hypothesis by evolving Pseudomonas fluorescens and its lytic phage at different temperatures and resource levels. We found that broadly, dispersal increased the local diversity at low-quality habitats and enemies regulated the diversification process through top–down effects. We also observed that dispersal with enemies induced higher morphotype loss at lower temperatures and lower-resource levels, which may be a result of the higher predation efficiency of immigrated enemies. These findings show that the dispersal of interacting species shapes local population diversity.},
  archive      = {J_EVLETT},
  author       = {Chu, Xiao-Lin},
  doi          = {10.1093/evlett/qraf004},
  journal      = {Evolution Letters},
  month        = {6},
  number       = {3},
  pages        = {346-354},
  shortjournal = {Evol. Lett.},
  title        = {Enemy dispersal drives the diversity loss of bacterial victim populations at diversity cold spots},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Positive correlations in susceptibility to a diverse panel of viruses across drosophilidae host species. <em>EVLETT</em>, <em>9</em>(3), 335-345. (<a href='https://doi.org/10.1093/evlett/qraf002'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our ability to predict the emergence of novel viruses relies on there being generalizable patterns in the susceptibilities of hosts to novel infections. Studies investigating variation in susceptibility among host species have consistently shown that closely related hosts share similar susceptibilities to a given virus. However, the extent to which such phylogenetic patterns of susceptibility are correlated among diverse sets of viruses is unclear. Here, we investigate phylogenetic correlations in susceptibility among Drosophilidae hosts to a panel of 11 different invertebrate viruses, comprising 7 unique virus species, 6 unique families, and both RNA and DNA viruses. The susceptibility of hosts to each pair of viruses tested was either positively correlated across host species or did not show evidence of correlation. No negative correlations, indicative of evolutionary trade-offs in host susceptibility to different viruses, were detected between any virus pairs. The strength of correlations was generally higher in viruses of the same species and family, consistent with virus phylogenetic patterns in host infectivity. Our results suggest that generalized host susceptibility can result in positive correlations, even between highly diverged viruses, while specialized interactions with individual viruses cause a stepwise decrease in correlation strength between viruses from the within-species, to the within-family, and to the across-family level.},
  archive      = {J_EVLETT},
  author       = {Imrie, Ryan M and Wallace, Megan A and Longdon, Ben},
  doi          = {10.1093/evlett/qraf002},
  journal      = {Evolution Letters},
  month        = {6},
  number       = {3},
  pages        = {335-345},
  shortjournal = {Evol. Lett.},
  title        = {Positive correlations in susceptibility to a diverse panel of viruses across drosophilidae host species},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A male-biased sex ratio increases the opportunity for precopulatory sexual selection but does not change the bateman gradient. <em>EVLETT</em>, <em>9</em>(3), 324-334. (<a href='https://doi.org/10.1093/evlett/qraf001'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Theory predicts that the sex ratio within populations should influence the strength of sexual selection, and sex ratio is often used as a proxy for sexual selection. However, recent studies challenge this relationship. We manipulated adult sex ratios in Drosophila melanogaster to comprehensively investigate the relationship between sex ratio and sexual selection. Consistent with theory, we found stronger sexual selection in males than females and an increased variance in male reproductive success (the opportunity for selection) in male-biased sex ratios. In addition, males faced more intense sperm competition in male-biased sex ratios, although the structure of sexual networks was largely invariant to sex ratio. Despite this, we show that sex ratios did not influence sexual selection in males as measured by the Bateman gradient. We leverage randomized null models to reconcile these results and show that the higher male reproductive variance in male-biased sex ratios may be explained by random chance in mating, rather than competitive mechanisms. Our findings indicate that caution is warranted over the long-standing assumption that sex ratio bias is a good proxy for the strength of sexual selection. For decades, evolutionary theory has pinpointed biases in the ratio of males to females (i.e., sex ratios) as a primary factor driving differences in sexual selection between populations. Theory predicts that members of the over-represented sex should experience stronger sexual selection. However, studies have increasingly challenged the link between sex ratio bias and sexual selection. We experimentally manipulated sex ratios in Drosophila melanogaster to test their impact on sexual selection in males and females. Consistent with classical predictions, we show that male-biased sex ratios increased variance in male reproductive success and therefore accentuated the potential for sexual selection to act on males. However, contrary to classical predictions, sex ratios did not influence the strength of sexual selection on males to acquire more mates. We compare our data to simulated random patterns of mating to reconcile these contradictory results. We show that the higher male reproductive variance in male-biased sex ratios can be explained by random chance in reproduction, rather than competitive mechanisms. Our findings underline recent appeals for caution over the long-standing assumption that sex ratio bias is a good proxy for sexual selection.},
  archive      = {J_EVLETT},
  author       = {McDonald, Grant C and Edmunds, Danielle and Morimoto, Juliano and Wigby, Stuart and Perry, Jennifer C},
  doi          = {10.1093/evlett/qraf001},
  journal      = {Evolution Letters},
  month        = {6},
  number       = {3},
  pages        = {324-334},
  shortjournal = {Evol. Lett.},
  title        = {A male-biased sex ratio increases the opportunity for precopulatory sexual selection but does not change the bateman gradient},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Size in the city: Morphological differences between city and forest great tits have a genetic basis. <em>EVLETT</em>, <em>9</em>(3), 317-323. (<a href='https://doi.org/10.1093/evlett/qraf006'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Animals living in cities are smaller than their conspecifics from rural areas but whether such differences are caused by genetic differences or food constraints remains untested. We performed a multi-generation common garden study where we raised great tits ( Parus major ), originating from eggs collected from multiple Dutch cities and forests under the same conditions for two generations. Offspring from city birds had a smaller tarsus than forest birds in both generations, demonstrating that these morphological differences are genetic. Next, we tested whether size differences are an adaptation to the low food abundance when offspring are raised in the city. Third-generation birds of both origins were given food amounts mimicking being raised in forests or cities during the second part of their nestling development. While the treatment resulted in birds in the lower feeding frequency treatment to be smaller, city and forest birds responded the same way, suggesting that city birds do not cope better with reduced food availability. Our study shows that the smaller size of urban birds has a genetic basis and is not only caused by a plastic response to restricted resources in the urban environment. Our experiment does not provide evidence that these genetic differences have evolved as an adaptive response to a reduced food availability in cities. Cities are expanding at unprecedented rates and urban life is challenging for most organisms. In our study, we show that evolution has led urban great tits to be genetically smaller, but our experiment does not provide evidence that these genetic differences allow city birds to cope better than forest birds with a reduced food availability in cities.},
  archive      = {J_EVLETT},
  author       = {Tomotani, Barbara M and Couweleers, Mika and ten Brinke, Bram and Walboom, Anne and van Oers, Kees and Visser, Marcel E},
  doi          = {10.1093/evlett/qraf006},
  journal      = {Evolution Letters},
  month        = {6},
  number       = {3},
  pages        = {317-323},
  shortjournal = {Evol. Lett.},
  title        = {Size in the city: Morphological differences between city and forest great tits have a genetic basis},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mammalian retinal specializations for high acuity vision evolve in response to both foraging strategies and morphological constraints. <em>EVLETT</em>, <em>9</em>(2), 302. (<a href='https://doi.org/10.1093/evlett/qrae072'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Vision is a complex sensory system that requires coordination among cellular and morphological traits, and it remains unclear how functional relationships among traits interact with ecological selective pressures to shape the evolution of vision. Many species have specialized high visual acuity regions in the retina defined by patterns of ganglion cell density, which may evolve in response to ecological traits. For example, ganglion cell density can increase radially towards the center of the retina to form an area centralis, which is thought to improve acuity towards the center of the visual field in predators. Another example is the horizontal streak, where ganglion cells are dense in a horizontal pattern across the retina, which is thought to be beneficial in horizon-dominated habitats. At the morphological level, many have proposed that predation selects for high orbit convergence angles, or forward-facing eyes. We tested these hypotheses in a phylogenetic framework across eutherian mammals and found support for the association between the horizontal streak and horizon-dominated habitats. However, we did not find a significant association between orbit convergence and predation. We also tested if retinal specializations evolve in response to orbit convergence angles. We found that horizontal streaks were associated with side-facing eyes, potentially facilitating panoramic vision. Previous studies observed that some species with side-facing eyes have an area centralis shifted towards the temporal side of the retina, such that the high acuity region would project forward, but this relationship had not been tested quantitatively. We found that the temporal distance of the area centralis from the center of the retina was inversely correlated with orbit convergence, as predicted. Our work shows a strong relationship between orbit convergence and retinal specializations. We find support that both visual ecology and functional interactions among traits play important roles in the evolution of ocular traits across mammals.},
  archive      = {J_EVLETT},
  author       = {Kopania, Emily E K and Clark, Nathan L},
  doi          = {10.1093/evlett/qrae072},
  journal      = {Evolution Letters},
  month        = {4},
  number       = {2},
  pages        = {302},
  shortjournal = {Evol. Lett.},
  title        = {Mammalian retinal specializations for high acuity vision evolve in response to both foraging strategies and morphological constraints},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fine-scale spatial variation in fitness, inbreeding, and inbreeding depression in a wild ungulate. <em>EVLETT</em>, <em>9</em>(2), 292-301. (<a href='https://doi.org/10.1093/evlett/qrae073'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Environmental stress can exacerbate inbreeding depression by amplifying differences between inbred and outbred individuals. In wild populations, where the environment is often unpredictable and stress can be highly detrimental, the interplay between inbreeding depression and environmental variation is potentially important. Here, we investigate variation in inbreeding level, fitness and strength of inbreeding depression across a fine-scale geographic area (~12 km 2 ) in an individually monitored population of red deer ( Cervus elaphus ). We show that northern regions of the study area have lower birth weights, lower juvenile survival rates, and higher inbreeding coefficients. Such fine-scale differences in inbreeding coefficients could be caused by the mating system of red deer combined with female density variation. We then tested for an inbreeding depression-by-environment interaction (ID × E) in birth weight and juvenile survival, by fitting an interaction term between the inbreeding coefficient and geographic location. We find that inbreeding depression in juvenile survival is stronger in the harsher northern regions, indicating the presence of ID × E. We also highlight that the ability to infer ID × E is affected by the variation in inbreeding within each geographic region. Therefore, for future studies on ID × E in wild populations, we recommend first assessing whether inbreeding and traits vary spatially or temporally. Overall, this is one of only a handful of studies to find evidence for ID × E in a wild population—despite its prevalence in experimental systems—likely due to intense data demands or insufficient variation in environmental stress or inbreeding coefficients. The environment in which an animal lives can have a profound impact on its overall health and likelihood of surviving. In addition, it is well documented that inbred individuals have lower survival and reproduction, known as inbreeding depression. In small, wild populations, where the environment can be harsh and inbreeding sometimes cannot be avoided, these two factors can interact, meaning that in worse environmental conditions inbred individuals may be even more unlikely to survive. This phenomenon (referred to as ID × E) is understandably of great concern to conservationists, but much of the empirical evidence for it comes from laboratory or semi-natural populations. In this study, we use a long-term dataset from a wild population of red deer living on the Isle of Rum, Scotland, to investigate whether this interaction exists. Overall, we show that the northern regions of the study area have more severe inbreeding depression caused by the worse environmental conditions in comparison to the southern regions. Additionally, these regions have higher inbreeding coefficients, likely due to the mating system of red deer, which can promote inbreeding in regions of higher density. Our study is one of only a few examples of ID × E in a wild population and highlights the demanding data requirements in this field.},
  archive      = {J_EVLETT},
  author       = {Hewett, Anna M and Johnston, Susan E and Albery, Gregory F and Morris, Alison and Morris, Sean J and Pemberton, Josephine M},
  doi          = {10.1093/evlett/qrae073},
  journal      = {Evolution Letters},
  month        = {4},
  number       = {2},
  pages        = {292-301},
  shortjournal = {Evol. Lett.},
  title        = {Fine-scale spatial variation in fitness, inbreeding, and inbreeding depression in a wild ungulate},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reproductive output of old males is limited by seminal fluid, not sperm number. <em>EVLETT</em>, <em>9</em>(2), 282-291. (<a href='https://doi.org/10.1093/evlett/qrae071'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Male reproductive senescence is typically characterized by a decline in the number of sperm produced and transferred by old males, a phenomenon that may be exacerbated in polygynous species where males mate multiply. However, males also transfer seminal fluid to females, and little is known about its role in modulating male reproductive senescence. Here, we explore the contributions of sperm and seminal fluid towards male reproductive senescence in a series of sequential matings, using Drosophila melanogaster . As expected, old males produce fewer offspring than young males. However, this pattern is not driven by sperm limitation: old males have more sperm and transfer similar numbers to females, compared to young males. Instead, females storing fewer sperm of old males compared to that of young males, over a long term, drives male reproductive senescence. We are able to mitigate the age-related decline in male reproductive output by supplementing females with the seminal fluid of a young male, before she mates with an old male. Similarly, we alleviate the reduction in reproductive output across sequential matings by supplementing females with seminal fluid. Our findings highlight that seminal fluid, rather than sperm number, limits reproductive success in old or multiply mating males, highlighting its underappreciated role in reproductive aging.},
  archive      = {J_EVLETT},
  author       = {Sanghvi, Krish and Shandilya, Sucheta and Brown, Alana and Todorova, Biliana and Jahn, Martin and Gascoigne, Samuel J L and Camilleri, Tara-Lyn and Pizzari, Tommaso and Sepil, Irem},
  doi          = {10.1093/evlett/qrae071},
  journal      = {Evolution Letters},
  month        = {4},
  number       = {2},
  pages        = {282-291},
  shortjournal = {Evol. Lett.},
  title        = {Reproductive output of old males is limited by seminal fluid, not sperm number},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Co-regulation of cooperative and private traits by PsdR in pseudomonas aeruginosa. <em>EVLETT</em>, <em>9</em>(2), 273-281. (<a href='https://doi.org/10.1093/evlett/qrae067'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Social interactions profoundly shape the dynamics and functionality of microbial populations. However, mechanisms governing the regulation of cooperative or individual traits have remained elusive. Here, we investigated the regulatory mechanisms of social behaviors by characterizing the fitness of transcriptional regulator PsdR mutants in cooperating Pseudomonas aeruginosa populations. In a canonical model described previously, PsdR was shown to solely have a nonsocial role in adaptation of these populations by controlling the intracellular uptake and processing of dipeptides. In addition to these known private traits, we found that PsdR mutants also enhanced cooperation by increasing the production of quorum sensing (QS)-regulated public goods. Although private dipeptide utilization promotes individual absolute fitness, it only partially accounts for the growth advantage of PsdR mutants. The absence of the QS master regulator LasR delayed the appearance of PsdR variants in an evolution experiment. We also demonstrated that the growth fitness of PsdR mutants is determined by a combination of the QS-mediated cooperative trait and the dipeptide metabolism-related private trait. This dual trait is co-regulated by PsdR, leading to the rapid spread of PsdR variants throughout the population. In conclusion, we identified a new social model of co-regulating cooperative and private traits in PsdR variants, uncovering the social and nonsocial roles of this transcriptional regulator in cooperating bacterial populations. Our findings advance the fundamental understanding of bacterial social interactions and provide insights into population evolution, pathogen infection control and synthetic biotechnology. Pseudomonas aeruginosa is a leading cause of opportunistic acute and chronic infections in humans. The establishment and progression of P. aeruginosa infections are significantly influenced by the intricate social dynamics of its subpopulations. We uncover a new mechanism of social interaction in which cooperative and private traits are co-regulated by PsdR, a transcriptional repressor of the XRE-cupin family. Our investigation demonstrates that the inactivation of PsdR heightens quorum sensing (QS) activity with increased production of shared public goods and facilitates intracellular dipeptide utilization. Given that PsdR is a regulator governing both cooperative and private traits, our findings suggest that PsdR may serve as a potential target for manipulating social interactions within the P. aeruginosa population. This discovery provides a new perspective for the control of pathogen infection and opens doors to synthetic applications.},
  archive      = {J_EVLETT},
  author       = {Qiu, Huifang and Dandekar, Ajai A and Dai, Weijun},
  doi          = {10.1093/evlett/qrae067},
  journal      = {Evolution Letters},
  month        = {4},
  number       = {2},
  pages        = {273-281},
  shortjournal = {Evol. Lett.},
  title        = {Co-regulation of cooperative and private traits by PsdR in pseudomonas aeruginosa},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The maintenance of genetic polymorphism underlying sexually antagonistic traits. <em>EVLETT</em>, <em>9</em>(2), 259-272. (<a href='https://doi.org/10.1093/evlett/qrae059'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Selection often favors different trait values in males and females, leading to genetic conflicts between the sexes when traits have a shared genetic basis. Such sexual antagonism has been proposed to maintain genetic polymorphism. However, this notion is based on insights from population genetic models of single loci with fixed fitness effects. It is thus unclear how readily polymorphism emerges from sex-specific selection acting on continuous traits, where fitness effects arise from the genotype-phenotype map and the fitness landscape. Here, we model the evolution of a continuous trait that has a shared genetic basis but different optima in males and females, considering a wide variety of genetic architectures and fitness landscapes. For autosomal loci, the long-term maintenance of polymorphism requires strong conflict between males and females that generates uncharacteristic sex-specific fitness patterns. Instead, more plausible sex-specific fitness landscapes typically generate stabilizing selection leading to an evolutionarily stable state that consists of a single homozygous genotype. Except for sites tightly linked to the sex-determining region, our results indicate that genetic variation due to sexual antagonism should arise only rarely and often be transient, making these signatures challenging to detect in genomic data. Evolutionary biologists aim to understand how genetic variation is maintained among individuals of the same population. In species with separate sexes, a commonly proposed mechanism is sexually antagonistic selection. This occurs when the same genes underlie traits with different optima in each sex, resulting in a genetic tug-of-war as males and females favor different alleles at the same loci. We test this idea mathematically for traits influenced by many alleles and genes, such as quantitative phenotypes like height or body weight. Contrary to previous claims, we find that sexually antagonistic selection does not readily maintain genetic polymorphism—even transiently—and instead erodes it in most genomic regions. Our results indicate that sexual antagonism is unlikely to contribute to elevated within-population genetic variation and that sexually antagonistic genes will be difficult to characterize from genomic data.},
  archive      = {J_EVLETT},
  author       = {Flintham, Ewan and Savolainen, Vincent and Otto, Sarah P and Reuter, Max and Mullon, Charles},
  doi          = {10.1093/evlett/qrae059},
  journal      = {Evolution Letters},
  month        = {4},
  number       = {2},
  pages        = {259-272},
  shortjournal = {Evol. Lett.},
  title        = {The maintenance of genetic polymorphism underlying sexually antagonistic traits},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sex-specific immunocompetence: Resistance and tolerance can both be futile but not under the same circumstances. <em>EVLETT</em>, <em>9</em>(2), 249-258. (<a href='https://doi.org/10.1093/evlett/qrae066'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Immunocompetence evolution can involve a “resistance is futile” scenario if parasite encounter rates are so high that high investment in resistance only marginally delays infection. Here, we investigate two understudied aspects of “futility”: the mode of immunocompetence and sexual selection. First, immunocompetence is usefully categorized as reducing the rate of becoming infected (resistance) or reducing the negative fitness consequences of infection once it happened (tolerance). We compare the prospects of futility for resistance, tolerance, and their joint occurrence, showing that resistance futility arises with respect to parasite encounter rates, while tolerance futility arises with respect to parasite virulence. However, if the same host trait improves pleiotropically both resistance and tolerance, futility disappears altogether and immunity investment remains profitable when increasing parasite encounter rates, virulence, or both. Second, we examine how sexual selection strength impacts these findings. If one sex (typically males) is near the faster end of a fast-slow continuum of life histories, then life history patterns reflecting futility can evolve sex-specificity. The solutions often feature sexual dimorphism in immunocompetence, but not always in the direction of strong sexual selection yielding low immunity: sexual selection can select for faster and “sicker” lives, but if sexual selection also favors traits that impact parasite encounter rates, the results are strongly dependent on whether futility (along any axis) plays a role. Intuition suggests that investment in immunity is higher when hosts frequently encounter parasites. While there are examples that confirm this, in other cases, hosts have been shown to abandon immune defenses under high parasite pressure. We reconcile these findings by modeling the optimal host resource allocation towards immunity under varying parasite pressure and strength of sexual selection. Our results show two axes along which immunity investments are futile and should therefore be abandoned in favor of investing in reproduction: resisting infection becomes futile under high parasite abundance, while tolerating the harmful effects of infection is not beneficial under ever increasing parasitic virulence. However, investments of organisms that are capable of both resistance and tolerance mechanisms yield fitness payoffs also when parasites are highly virulent and abundant. This work highlights the impact of parasites and immune defenses on optimal immunity investment levels in hosts, an insight which also complements theory on sex-specific immunity.},
  archive      = {J_EVLETT},
  author       = {Brenninger, Franziska A and Kovalov, Viktor and Kokko, Hanna},
  doi          = {10.1093/evlett/qrae066},
  journal      = {Evolution Letters},
  month        = {4},
  number       = {2},
  pages        = {249-258},
  shortjournal = {Evol. Lett.},
  title        = {Sex-specific immunocompetence: Resistance and tolerance can both be futile but not under the same circumstances},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Testing the coordination hypothesis: Incompatibilities in aggregative development of an experimentally evolved social amoeba. <em>EVLETT</em>, <em>9</em>(2), 236-248. (<a href='https://doi.org/10.1093/evlett/qrae063'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multicellular organisms that form by aggregation of cells arguably do not achieve high levels of complexity. Conflict among the cells is a widely accepted explanation for this, but an alternative hypothesis is that mixing cells of different genotypes leads to failures of coordination, which we call the “coordination hypothesis.” We empirically tested the coordination hypothesis in the social amoeba Dictyostelium discoideum . We mixed D. discoideum clones that had evolved in isolation for generations and acquired mutations that have not been tested against each other by selection. To quantify the effect of incompatibilities, we measured performance in terms of the developmental traits of slug migration and spore production. Importantly, we mixed lines evolved from the same ancestor under conditions that would not select for the evolution of de novo kin recognition. Our results show no evidence of incompatibilities in four traits related to the coordinated movement of slugs toward light in the social amoeba. Spore production was higher than expected in mixtures, in apparent contradiction to the coordination hypothesis. However, we found support for coordination incompatibilities in an interaction between migration and spore production: in mixtures, fewer cells succeeded at both migrating and becoming spores. Most living things that we can see with our naked eyes develop from a single fertilized egg cell. Even dinosaurs hatched from eggs. Why are eggs so successful over the evolutionary history of life? One theory is eggs result in a multicellular body where the daughter cells are genetically identical, and therefore have nothing to gain from conflict with other cells. Another theory is that when individual cells acquire changes independently of other cells, they may become incompatible with each other. Such cells can be thought of as speaking different genetic languages, such that they lose the ability to communicate during a crucial process like the development of the body. Here we have tested this latter theory, which we call the “coordination hypothesis” in a species of social amoeba. Of several traits tested, one showed strong evidence of poorer coordination in mixtures, supporting the coordination hypothesis.},
  archive      = {J_EVLETT},
  author       = {Jahan, Israt and Scott, Trey J and Strassmann, Joan E and Queller, David C},
  doi          = {10.1093/evlett/qrae063},
  journal      = {Evolution Letters},
  month        = {4},
  number       = {2},
  pages        = {236-248},
  shortjournal = {Evol. Lett.},
  title        = {Testing the coordination hypothesis: Incompatibilities in aggregative development of an experimentally evolved social amoeba},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Phylogenomics resolves key relationships in rumex and uncovers a dynamic history of independently evolving sex chromosomes. <em>EVLETT</em>, <em>9</em>(2), 221-235. (<a href='https://doi.org/10.1093/evlett/qrae060'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sex chromosomes have evolved independently many times across eukaryotes. Despite a considerable body of literature on sex chromosome evolution, the causes and consequences of variation in their formation, degeneration, and turnover remain poorly understood. Chromosomal rearrangements are thought to play an important role in these processes by promoting or extending the suppression of recombination on sex chromosomes. Sex chromosome variation may also contribute to barriers to gene flow, limiting introgression among species. Comparative approaches in groups with sexual system variation can be valuable for understanding these questions. Rumex is a diverse genus of flowering plants harboring significant sexual system and karyotypic variation, including hermaphroditic and dioecious clades with XY (and XYY) sex chromosomes. Previous disagreement in the phylogenetic relationships among key species has rendered the history of sex chromosome evolution uncertain. Resolving this history is important for investigating the interplay of chromosomal rearrangements, introgression, and sex chromosome evolution in the genus. Here, we use new transcriptome assemblies from 11 species representing major clades in the genus, along with a whole-genome assembly generated for a key hermaphroditic species. Using phylogenomic approaches, we find evidence for the independent evolution of sex chromosomes across two major clades, and introgression from unsampled lineages likely predating the formation of sex chromosomes in the genus. Comparative genomic approaches revealed high rates of chromosomal rearrangement, especially in dioecious species, with evidence for a complex origin of the sex chromosomes through multiple chromosomal fusions. However, we found no evidence of elevated rates of fusion on the sex chromosomes in comparison with autosomes, providing no support for an adaptive hypothesis of sex chromosome expansion due to sexually antagonistic selection. Overall, our results highlight a complex history of karyotypic evolution in Rumex , raising questions about the role that chromosomal rearrangements might play in the evolution of large heteromorphic sex chromosomes. Sex chromosomes have arisen independently in different groups of organisms. The reasons why some groups of organisms evolve sex chromosomes while others do not, as well as the consequences of this variation, are uncertain. Chromosomal rearrangements (changes in the number and configuration of chromosomes) are thought to be an important factor in promoting sex chromosome formation by linking favorable combinations of genes. Once arisen, sex chromosomes may contribute to species formation by harboring genetic incompatibilities between populations. We investigated these questions using genetic data in Rumex , a group of flowering plants with diverse sexual systems and sex chromosome configurations. Analyses of evolutionary relationships suggested two independent origins of XY sex chromosomes, arising from hermaphroditic (male and female reproductive organs in the same individual) ancestors. We also found evidence for hybridization predating the formation of these sex chromosomes and among contemporary hermaphroditic species, consistent with the role of sex chromosomes in maintaining species barriers. Finally, rates of chromosomal rearrangement were generally high in species with sex chromosomes but were not elevated in the sex chromosomes specifically compared to other chromosome pairs. Overall, our results highlight the evolutionary consequences of repeated sex chromosome formation and raise questions about the role of chromosomal rearrangement in this process.},
  archive      = {J_EVLETT},
  author       = {Hibbins, Mark S and Rifkin, Joanna L and Choudhury, Baharul I and Voznesenska, Olena and Sacchi, Bianca and Yuan, Meng and Gong, Yunchen and Barrett, Spencer C H and Wright, Stephen I},
  doi          = {10.1093/evlett/qrae060},
  journal      = {Evolution Letters},
  month        = {4},
  number       = {2},
  pages        = {221-235},
  shortjournal = {Evol. Lett.},
  title        = {Phylogenomics resolves key relationships in rumex and uncovers a dynamic history of independently evolving sex chromosomes},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Heritability and age-dependent changes in genetic variation of telomere length in a wild house sparrow population. <em>EVLETT</em>, <em>9</em>(2), 209-220. (<a href='https://doi.org/10.1093/evlett/qrae055'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Telomere length (TL) and/or its rate of change are popular biomarkers of senescence, as telomere dynamics are linked with survival and lifespan. However, the evolutionary potential of telomere dynamics has received mixed support in natural populations. To better understand how telomere dynamics evolve, it is necessary to quantify genetic variation in TL and how such variation changes with age. Here, we analyzed 2,083 longitudinal samples from 1,225 individuals across 16 years, collected from a wild, insular house sparrow ( Passer domesticus ) population with complete life history and genetic relatedness data. Using a series of “animal” models, we confirmed that TL changes with age, reflecting senescence in this population. We found TL to be repeatable (14.0%, 95% CrI: 9.1%–19.9%) and heritable (12.3%, 95% CrI: 7.5%–18.2%); and detected a genotype-by-age interaction, meaning that genotypes differ in their rate of change of TL, and additive genetic variance increases at older ages. Our findings provide empirical evidence from a wild population that supports hypotheses explaining the evolution of senescence and highlight the importance of telomere dynamics as a key biomarker of body physiology for the evolution of senescence. Telomeres are protective stretches of DNA at the ends of chromosomes that shorten over time. telomere length (TL) and telomere shortening have been previously linked with survival, aging, and fitness, implying it has the potential to be under natural selection and to evolve. Therefore, estimating how much variation in telomere dynamics is contributed by genes and how this variation changes with age allows us to better understand the evolution of aging. Using long-term data from a wild population of house sparrows, we found that TL is heritable and thus has the potential to evolve. We also found that the rate of telomere change is also influenced by genes and that there was an increase in genetic variation of TL at older ages. This finding provides direct empirical evidence concerning TL that supports the evolutionary hypotheses of aging. Our study provides insights into the natural selection patterns that give rise to aging, supports evolutionary predictions for a fitness-related trait, and encourages further investigation into telomere dynamics as a biomarker of aging.},
  archive      = {J_EVLETT},
  author       = {Chik, Heung Ying Janet and Sibma, Aaron and Mannarelli, Maria-Elena and dos Remedios, Natalie and Simons, Mirre J P and Burke, Terry and Dugdale, Hannah L and Schroeder, Julia},
  doi          = {10.1093/evlett/qrae055},
  journal      = {Evolution Letters},
  month        = {4},
  number       = {2},
  pages        = {209-220},
  shortjournal = {Evol. Lett.},
  title        = {Heritability and age-dependent changes in genetic variation of telomere length in a wild house sparrow population},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Parallel and convergent evolution in genes underlying seasonal migration. <em>EVLETT</em>, <em>9</em>(2), 189-208. (<a href='https://doi.org/10.1093/evlett/qrae064'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Seasonal migration has fascinated scientists and natural historians for centuries. While the genetic basis of migration has been widely studied across different taxa, there is little consensus regarding which genomic regions play a role in the ability to migrate and whether they are similar across species. Here, we examine the genetic basis of intraspecific variation within and between distinct migratory phenotypes in a songbird. We focus on the Common Yellowthroat ( Geothlypis trichas ) as a model system because the polyphyletic origin of eastern and western clades across North America provides a strong framework for understanding the extent to which there has been parallel or convergent evolution in the genes associated with migratory behavior. First, we investigate genome-wide population genetic structure in the Common Yellowthroat in 196 individuals collected from 22 locations across breeding range. Then, to identify candidate genes involved in seasonal migration, we identify signals of putative selection in replicate comparisons between resident and migratory phenotypes within and between eastern and western clades. Overall, we find wide-spread support for parallel evolution at the genic level, particularly in genes that mediate biological timekeeping. However, we find little evidence of parallelism at the individual SNP level, supporting the idea that there are multiple genetic pathways involved in the modulation of migration. Migration is a complex behavior that has been switched multiple times throughout evolutionary history. However, it remains unclear whether the modulation of migration in different groups is due to parallel evolution, where the same genetic changes result in similar phenotypes, or convergent evolution, where different genetic changes result in the same phenotypes. Here we use genomic data across the breeding range of the Common Yellowthroat ( Geothlypis trichas ) to both investigate population genetic structure and identify candidate genes underlying the evolution of diverse migratory behaviors. We analyze genomic data from migratory and resident populations within the eastern and western groups with distinct evolutionary origins. Within three replicate migratory-resident comparisons, we use outlier analyses and selection scans to identify migration-linked candidate genes. We find evidence of parallel evolution at the genic level, particularly in genes linked to biological time keeping. However, we find little evidence of parallelism at the individual SNP level, supporting the idea that there are multiple pathways to the propensity of migratory behavior.},
  archive      = {J_EVLETT},
  author       = {Zamudio-Beltrán, Luz E and Bossu, Christen M and Bueno-Hernández, Alfredo A and Dunn, Peter O and Sly, Nicholas D and Rayne, Christine and Anderson, Eric C and Hernández-Baños, Blanca E and Ruegg, Kristen C},
  doi          = {10.1093/evlett/qrae064},
  journal      = {Evolution Letters},
  month        = {4},
  number       = {2},
  pages        = {189-208},
  shortjournal = {Evol. Lett.},
  title        = {Parallel and convergent evolution in genes underlying seasonal migration},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). What does infrared thermography tell us about the evolutionary potential of heat tolerance in endotherms?. <em>EVLETT</em>, <em>9</em>(2), 184-188. (<a href='https://doi.org/10.1093/evlett/qrae070'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Anthropogenic climate change affects wild animal populations through increasing average temperatures and more frequent extreme climatic events. Endotherms have evolved the capacity to regulate their body temperature but little is still known about how they can physiologically adapt to the pace of global warming. Adaptive responses would require that heat-tolerance mechanisms, such as the capacity to withstand high body temperatures and regulate evaporative water loss, exhibit sufficient heritable genetic variation for selection to act upon. Unfortunately, the quantitative genetics of these traits in endotherms remains poorly understood. In a recent study using infrared thermography (IRT) on semi-captive ostriches, Svensson et al., (Heritable variation in thermal profiles is associated with reproductive success in the world’s largest bird. Evolution Letters, 8(2), 200–211.) sought to address this knowledge gap by measuring relative heat exchange from the head and neck and assessing the link between among-individual variation in heat dissipation and reproductive fitness. We discuss how IRT serves as a valuable tool for non-invasive data collecting, highlighting its potential for field studies of the evolutionary potential of thermal tolerance. Nevertheless, interpreting IRT data is not as straightforward as it may seem and thus must be conducted carefully. For instance, body parts from which surface temperatures are measured need to be unequivocally identified as sources of dry heat exchange in order to inform on thermoregulation—something lacking in the mentioned study. Furthermore, there is still no conclusive evidence that surface temperatures reflect core body temperatures in endotherms. Critical underlying mechanisms of the heat response, such as evaporative cooling, must also be considered. Assumptions stemming from uncertain proxies of thermoregulation can obscure our understanding of the endothermic adaptation of heat-tolerance traits to rapid global warming. These considerations emphasize that, while IRT can be a valuable tool for developing quantitative genetic approaches to estimate the evolutionary potential of heat tolerance in endotherms—particularly for species most vulnerable to warming, its application warrants careful planning.},
  archive      = {J_EVLETT},
  author       = {Monge, Otto and Caro, Samuel P and Charmantier, Anne},
  doi          = {10.1093/evlett/qrae070},
  journal      = {Evolution Letters},
  month        = {4},
  number       = {2},
  pages        = {184-188},
  shortjournal = {Evol. Lett.},
  title        = {What does infrared thermography tell us about the evolutionary potential of heat tolerance in endotherms?},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A development-centric perspective on pace-of-life syndromes. <em>EVLETT</em>, <em>9</em>(2), 172-183. (<a href='https://doi.org/10.1093/evlett/qrae069'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organism responses to environmental change require coordinated changes across correlated traits, so-called syndromes. For example, animals differ in their “pace-of-life syndrome” (POLS); suites of correlated life-history, behavioral and physiological traits. But standard “gene-centric” evolutionary theory cannot explain why POLSs exist because it assumes that the expression of phenotypic traits of animals is determined by genotype-specified reaction norms; it ignores that developmental processes can bias the direction of evolution so that phenotypes no longer match genotype-by-environment interactions. Here we apply a development-centric perspective to derive new POLS hypotheses that can resolve the conflict that current POLS predictions fail to explain which species/populations are resilient to environmental change.},
  archive      = {J_EVLETT},
  author       = {Smallegange, Isabel M and Guenther, Anja},
  doi          = {10.1093/evlett/qrae069},
  journal      = {Evolution Letters},
  month        = {4},
  number       = {2},
  pages        = {172-183},
  shortjournal = {Evol. Lett.},
  title        = {A development-centric perspective on pace-of-life syndromes},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Hiding in plain sight: The y chromosome and its reinvigorated role in evolutionary processes. <em>EVLETT</em>, <em>9</em>(2), 165-171. (<a href='https://doi.org/10.1093/evlett/qrae065'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent methodological approaches have expanded our understanding of Y chromosome sequence, revealed unexpected Y diversity, and sparked a growing realization of its importance in evolutionary processes. To fully understand the diversity and importance of the Y chromosome, we suggest the need to move from a holotype Y chromosome sequence, based on a single individual and meant to represent the species, to a thorough understanding of Y chromosome haplotype diversity, its phenotypic implications, and its phylogeographic distribution. Additionally, the Y chromosome may play an important role in two key rules of speciation that have otherwise been attributed to the X, namely Haldane’s Rule and the Large-X Effect. Emerging genomic tools and analytical approaches are just now giving us the means to ask how important this small, often forgotten region of the genome is in evolutionary processes. Recent advances in sequencing and GWAS methods have led to a growing realization that Y chromosomes are unexpectedly diverse and are important in the genetic architecture of male phenotypic variation. We outline several new approaches that are useful in characterizing the Y chromosome. We also highlight several questions about the Y itself and its role in evolution that these methods now make possible.},
  archive      = {J_EVLETT},
  author       = {van der Bijl, Wouter and Mank, Judith E},
  doi          = {10.1093/evlett/qrae065},
  journal      = {Evolution Letters},
  month        = {4},
  number       = {2},
  pages        = {165-171},
  shortjournal = {Evol. Lett.},
  title        = {Hiding in plain sight: The y chromosome and its reinvigorated role in evolutionary processes},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Correction to: A shared developmental genetic basis for sexually antagonistic male and female adaptations in the toothed water strider. <em>EVLETT</em>, <em>9</em>(1), 163. (<a href='https://doi.org/10.1093/evlett/qrae068'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_EVLETT},
  doi          = {10.1093/evlett/qrae068},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {163},
  shortjournal = {Evol. Lett.},
  title        = {Correction to: A shared developmental genetic basis for sexually antagonistic male and female adaptations in the toothed water strider},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Shared genetic architecture links energy metabolism, behavior and starvation resistance along a power-endurance axis. <em>EVLETT</em>, <em>9</em>(1), 150-162. (<a href='https://doi.org/10.1093/evlett/qrae062'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Shared developmental, physiological, and molecular mechanisms can generate strong genetic covariances across suites of traits, constraining genetic variability, and evolvability to certain axes in multivariate trait space (“variational modules” or “syndromes”). Such trait suites will not only respond jointly to selection; they will also covary across populations that diverged from one another by genetic drift. We report evidence for such a genetically correlated trait suite that links traits related to energy metabolism along a “power-endurance” axis in Drosophila melanogaster. The “power” pole of the axis is characterized by high potential for energy generation and expenditure—high expression of glycolysis and TCA cycle genes, high abundance of mitochondria, and high spontaneous locomotor activity. The opposite “endurance” pole is characterized by high triglyceride (fat) reserves, locomotor endurance, and starvation resistance (and low values of traits associated with the “power” pole). This trait suite also aligns with the first principal component of metabolome; the “power” direction is characterized by low levels of trehalose (blood sugar) and high levels of some amino acids and their derivatives, including creatine, a compound known to facilitate energy production in muscles. Our evidence comes from six replicate “Selected” populations adapted to a nutrient-poor larval diet regime during 250 generations of experimental evolution and six “Control” populations evolved in parallel on a standard diet regime. We found that, within each of these experimental evolutionary regimes, the above traits strongly covaried along this “power-endurance” axis across replicate populations which diversified by drift, indicating a shared genetic architecture. The two evolutionary regimes also drove divergence along this axis, with Selected populations on average displaced towards the “power” direction compared to Controls. Aspects of this “power-endurance” axis resemble the “pace of life” syndrome and the “thrifty phenotype”; it may have evolved as part of a coordinated organismal response to nutritional conditions. While evolution is often looked at one trait at a time, there is increasing awareness that developmental, physiological, and cellular regulatory mechanisms integrate multiple traits in correlated traits suites or “syndromes,” constraining their ability to vary independently of one another. We report such a trait syndrome linking traits involved in energy generation, energy expenditure, and energy storage in fruit flies. We found that, among a set of genetically differentiated laboratory fly populations, those that show a high cellular potential for energy generation (based on gene expression and mitochondria abundance) are also characterized by a high rate of locomotor activity, but low endurance, low-fat reserves, and low resistance to starvation—and vice versa. This is accompanied by differences in metabolome profile consistent with a known physiological role of some compounds, such as trehalose (blood sugar in insects) and creatine. This concerted evolution of these diverse traits suggests that they share genetic architecture, likely reflecting regulatory mechanisms that evolved to ensure coordinated responses to varying nutritional conditions, and which now constrain traits from evolving independently. This implies that physiological mechanisms evolved in the past may canalize future evolution.},
  archive      = {J_EVLETT},
  author       = {Erkosar, Berra and Dupuis, Cindy and Savary, Loriane and Kawecki, Tadeusz J},
  doi          = {10.1093/evlett/qrae062},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {150-162},
  shortjournal = {Evol. Lett.},
  title        = {Shared genetic architecture links energy metabolism, behavior and starvation resistance along a power-endurance axis},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Does metabolic rate influence genome-wide amino acid composition in the course of animal evolution?. <em>EVLETT</em>, <em>9</em>(1), 137-149. (<a href='https://doi.org/10.1093/evlett/qrae061'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Natural selection is believed to shape amino acid usage of the proteome by minimizing the energy cost of protein biosynthesis. Although this hypothesis explains well the amino acid frequency (AA frequency ) difference among the 20 common amino acids within a given genome (species), whether it is applicable to cross-species difference remains to be inspected. Here, we proposed and tested a “metabolic rate hypothesis,” which suggests that metabolic rate impacts genome-wide AA frequency , considering that the energy allocated to protein biosynthesis is under selection pressure due to metabolic rate constraint. We performed integrated phylogenetic comparative analyses on proteomic sequence and metabolic rate data of 166 species covering 130 eumetazoan orders. We showed that resting metabolic rate (RMR) was significantly linked to AA frequency variation across animal lineages, with a contribution comparable to or greater than genomic traits such as GC content and codon usage bias. Consistent with the metabolic rate hypothesis, low-energy-cost amino acids are observed to be more likely at higher frequency in animal species with high (residual) metabolic rate. Correlated evolution of RMR and AA frequency was further inferred being driven by adaptation. The relationship between RMR and AA frequency varied greatly among amino acids, most likely reflecting a trade-off among various interacting factors. Overall, there exists no “one-size-fits-all” predictor for AA frequency , and integrated investigation of multilevel traits is indispensable for a fuller understanding of AA frequency variation and evolution in animal. Natural selection has previously been proposed to underlie compositional differences of amino acids within a given animal species: to minimize the energy cost of protein biosynthesis, selection should result in an increased abundance of energy-cheap amino acids and a decreased abundance of energy-expensive ones in the proteome. However, amino acid composition varies greatly among species, yet the underlying evolutionary processes are largely unclear. Here, we propose that interspecific variations in metabolic rate exert an effect on amino acid composition due to natural selection, which is referred to as the “metabolic rate hypothesis.” The present study intended to test this hypothesis by examining the relationship between molecular-level amino acid frequency and organism-level metabolic rate (as a proxy for energy cost) in an evolutionary framework. Phylogeny-based analysis revealed significant connection of these two traits, indicating they have evolved most likely in concert, and metabolic rate is an important confining factor to the compositional variation of amino acid in animal. Consistent with the metabolic rate hypothesis, low-energy-cost amino acids are observed to be more likely at higher frequency in animal species with high residual metabolic rate.},
  archive      = {J_EVLETT},
  author       = {Wang, Wei and Zhang, De-Xing},
  doi          = {10.1093/evlett/qrae061},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {137-149},
  shortjournal = {Evol. Lett.},
  title        = {Does metabolic rate influence genome-wide amino acid composition in the course of animal evolution?},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tempo and mode of winter diapause evolution in butterflies. <em>EVLETT</em>, <em>9</em>(1), 125-136. (<a href='https://doi.org/10.1093/evlett/qrae054'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantifying the tempo and mode via modern phylogenetic comparative methods can provide key insights into how selection and constraints shape trait evolution on a macroevolutionary time scale. Here, we elucidate the evolution of hibernation (winter) diapause, a complex and defining life-history trait that allows temporal escape from harsh winters in temperate regions for many insects, including our model system, butterflies. Butterflies can diapause in all major life stages, and the availability of global-scale phylogenies makes them an ideal model system for studying diapause evolution. First, using a thorough literature survey, we scored the developmental stage of hibernation diapause (egg, larva, pupa, adult) vs. absence of diapause. We find that larval diapause is most common, while pupal, egg, and adult diapause are relatively rare. Next, we determined that the loss of diapause occurred at a much higher rate and that gains primarily occurred from the non-diapause state. While ancestral state estimation at deeper nodes remained uncertain, we found consistent patterns for some families and strong evidence for extensive convergence in diapause evolution. Contrary to expectations, we find no support for increased gain of diapause during the Eocene–Oligocene glaciation (~35 million years ago). Overall, the evolution of diapause in butterflies has a complex history, has evolved convergently, and has likely predated the major glaciation event consistent with the deep history of diapause evolution in insects. This study advances our understanding of the evolution of a complex and important life-history trait and establishes a macroevolutionary foundation for future studies on the ultimate and proximate basis of diapause evolution. Animals living in seasonal environments, with harsh winters and mild summers in temperate regions and wet–dry seasons in the tropics, are confronted with alternating cycles of favorable and unfavorable seasons. While some migrate, many animals, especially insects, undergo hibernation diapause—a deep resting stage—during the unfavorable season to survive harsh winters. Hibernation diapause, thus, is a key evolutionary strategy for surviving in temperate environments. Butterflies, which can diapause in all life stages—egg, larva, pupa, and adult—serve as a model system for understanding diapause evolution. Through a comprehensive literature survey, we find that larval diapause is most common in temperate regions. Next, by employing statistical methods that allow inferring evolutionary patterns across millions of years, we show that hibernation diapause has evolved independently multiple times, and its loss occurs at a much higher rate than its gain. We also find that the major glacial cycle that led to the global cooling of the Earth around 35 million years ago did not influence diapause evolution. Our study sheds light on diapause evolution across ca. 100 million years, laying the foundations for future studies on ecological drivers and genetic mechanisms behind species-specific diapause strategies.},
  archive      = {J_EVLETT},
  author       = {Halali, Sridhar and Yapar, Etka and Wheat, Christopher W and Wahlberg, Niklas and Gotthard, Karl and Chazot, Nicolas and Nylin, Sören and Lehmann, Philipp},
  doi          = {10.1093/evlett/qrae054},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {125-136},
  shortjournal = {Evol. Lett.},
  title        = {Tempo and mode of winter diapause evolution in butterflies},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Same trait, different genes: Pelvic spine loss in three brook stickleback populations in alberta, canada. <em>EVLETT</em>, <em>9</em>(1), 115-124. (<a href='https://doi.org/10.1093/evlett/qrae053'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genetic basis of phenotypic or adaptive parallelism can reveal much about constraints on evolution. This study investigated the genetic basis of a canonically parallel trait: pelvic spine reduction in sticklebacks. Pelvic reduction has a highly parallel genetic basis in threespine stickleback in populations around the world, always involving a deletion of the pel1 enhancer of Pitx1 . We conducted a genome-wide association study to investigate the genetic basis of pelvic spine reduction in 3 populations of brook stickleback in Alberta, Canada. Pelvic reduction did not involve Pitx1 in any of the 3 populations. Instead, pelvic reduction in 1 population involved a mutation in an exon of Tbx4 , and it involved a mutation in an intron of Lmbr1 in the other two populations. Hence, the parallel phenotypic evolution of pelvic spine reduction across stickleback genera, and among brook stickleback populations, has a nonparallel genetic basis. This suggests that there may be redundancy in the genetic basis of this adaptive polymorphism, but it is not clear whether a lack of parallelism indicates a lack of constraint on the evolution of this adaptive trait. Whether different pleiotropic effects of different mutations have different fitness consequences or whether certain pelvic reduction mutations confer specific benefits in certain environments remains to be determined. In this study, we looked for the genetic basis of a well-studied trait in stickleback fish: the pelvic spines. This structure (i.e., the pelvic girdle and attached spines) has a shared developmental basis (and is homologous) to the pelvic bones and hind limbs of all tetrapods (including humans). We know from studying mice, fish, humans, and manatees that there are several genes that could affect the development of pelvic spines and hind limbs. In 1 species of stickleback, the threespine stickleback, however, a single gene called Pitx1 is always involved in the loss of pelvic spines in populations that have adapted to freshwater lakes. This replicated evolution of the same trait in the same environmental conditions is called parallel evolution. It is remarkable that Pitx1 is always the gene underlying this adaptive loss of spines in freshwater threespine stickleback populations. We were interested in whether this “genetic parallelism” extended to other species of stickleback that have also evolved the loss of pelvic spines. We looked at 3 populations of brook stickleback (which are never found in the ocean), each of which contains individuals with and without pelvic spines. We found that the Pitx1 genetic parallelism does not extend to brook stickleback, and, in fact, the genetic basis of pelvic spine loss differs between populations. In Muir Lake and Astotin Lake, pelvic spine loss results from a mutation in the Lmbr1 gene, and in Shunda Lake, pelvic spine loss results from a mutation in the Tbx4 gene.},
  archive      = {J_EVLETT},
  author       = {Mee, Jonathan A and Ly, Carolyn and Pigott, Grace C},
  doi          = {10.1093/evlett/qrae053},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {115-124},
  shortjournal = {Evol. Lett.},
  title        = {Same trait, different genes: Pelvic spine loss in three brook stickleback populations in alberta, canada},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Characterizing the evolution of defense in a tripartite marine symbiosis using adaptive dynamics. <em>EVLETT</em>, <em>9</em>(1), 105-114. (<a href='https://doi.org/10.1093/evlett/qrae052'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The evolution and maintenance of symbiotic systems remains a fascinating puzzle. While the coevolutionary dynamics of bipartite (host–symbiont) systems are well-studied, the dynamics of more complex systems have only recently garnered attention with increasing technological advances. We model a tripartite system inspired by the marine symbiotic relationship between the alga Bryopsis sp., its intracellular defensive bacterial symbiont “ Candidatus Endobryopsis kahalalidifaciens,” which produces a toxin that protects the alga against fish herbivores, and the sea-slug Elysia rufescens ( Zan et al., 2019 ), which is not deterred by the toxin. We disentangle the role of selection on different actors within this system by investigating evolutionary scenarios where defense evolves as (i) a host-controlled trait that reduces algal reproductive ability; (ii) a symbiont-controlled trait that impacts symbiont transmission; and (iii) a trait jointly controlled by both host and symbiont. Optimal investment in defensive toxins varies based on the characteristics of the host, symbiont, and sea slug; and evolutionary trajectories are modulated by trade-off shape, i.e., a strongly decelerating trade-off between defense and symbiont transmission can drive symbiont diversification via evolutionary branching. Increasing slug herbivory reduces host investment in defense to favor reproduction, while symbiont investment in defense first declines and then increases as host density declines to the degree that horizontal symbiont transmission is no longer beneficial. Increasing vertical transmission selects for reduced defense by the host when it evolves as a jointly controlled trait, as a result of investment by the symbiont. Our theoretical exploration of the evolution of defensive symbiosis in scenarios involving interactions with multiple herbivores provides a first window into the origin and maintenance of the Bryopsis sp. system, and adds another piece to the puzzle of the evolution of symbiotic systems. This study delves into a tripartite marine symbiosis between the algal host Bryopsis sp., and its protective bacterium Ca. Endobryopsis kahalalidifaciens , and the sea slug Elysia rufescens which feeds on bacterium-carrying algae. The bacterium produces defensive toxins that protect the algae from a range of herbivores. We developed a host–symbiont–herbivore model to study how this defensive relationship evolves in the presence of sea slugs which are not deterred by the toxins. We assumed that producing more toxins for the algal host means fewer resources for its reproduction, while for the bacterium, it means reduced ability to spread to new hosts. We discovered that the alga and bacterium’s investment in toxin production is influenced by the strength of such defense costs, the magnitude of herbivory by sea slugs and other environmental factors. As such, when producing toxins becomes too costly, both alga and bacterium reduce their investment in defense. However, under certain conditions, multiple bacterium populations that have different levels of defense can coexist. This research highlights the evolution and maintenance of complex symbiotic relationships, offering insights into how organisms work together to survive in the face of multiple threats.},
  archive      = {J_EVLETT},
  author       = {Singh, Prerna and Bruijning, Marjolein and Carver, Gavriela D and Donia, Mohamed S and Metcalf, Charlotte Jessica E},
  doi          = {10.1093/evlett/qrae052},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {105-114},
  shortjournal = {Evol. Lett.},
  title        = {Characterizing the evolution of defense in a tripartite marine symbiosis using adaptive dynamics},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Indirect genetic effects increase the heritable variation available to selection and are largest for behaviors: A meta-analysis. <em>EVLETT</em>, <em>9</em>(1), 89-104. (<a href='https://doi.org/10.1093/evlett/qrae051'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The evolutionary potential of traits is governed by the amount of heritable variation available to selection. While this is typically quantified based on genetic variation in a focal individual for its own traits (direct genetic effects, DGEs), when social interactions occur, genetic variation in interacting partners can influence a focal individual’s traits (indirect genetic effects, IGEs). Theory and studies on domesticated species have suggested IGEs can greatly impact evolutionary trajectories, but whether this is true more broadly remains unclear. Here, we perform a systematic review and meta-analysis to quantify the amount of trait variance explained by IGEs and the contribution of IGEs to predictions of adaptive potential. We identified 180 effect sizes from 47 studies across 21 species and found that, on average, IGEs of a single social partner account for a small but statistically significant amount of phenotypic variation (0.03). As IGEs affect the trait values of each interacting group member and due to a typically positive—although statistically nonsignificant—correlation with DGEs ( r DGE-IGE = 0.26), IGEs ultimately increase trait heritability substantially from 0.27 (narrow-sense heritability) to 0.45 (total heritable variance). This 66% average increase in heritability suggests IGEs can increase the amount of genetic variation available to selection. Furthermore, whilst showing considerable variation across studies, IGEs were most prominent for behaviors and, to a lesser extent, for reproduction and survival, in contrast to morphological, metabolic, physiological, and development traits. Our meta-analysis, therefore, shows that IGEs tend to enhance the evolutionary potential of traits, especially for those tightly related to interactions with other individuals, such as behavior and reproduction. Predicting evolutionary change is important for breeding better livestock and crops, understanding how biodiversity arises, and how populations respond to environmental change. Normally, these predictions are based on how the genetic variants in an organism influence its own traits (characteristics). However, when organisms socially interact, for instance, by fighting or cooperating, then the genes in one individual can influence the traits of others, therefore affecting the potential for evolutionary change. We compared 47 studies across 21 animal species and found that the effect of the genes of a single social partner is small but statistically significant, while the total contribution of social genetic effects to evolutionary potential is large. These effects are particularly important for the evolution of animals’ behaviors and reproductive traits but less so for other traits such as body size and physiology. We also found that because an individual can interact with many others and influence them all, social interactions can substantially increase the potential for a population to evolve from generation to generation. Our results show how social interactions can potentially alter the evolution of those traits known to respond to social interactions in comparison to standard expectations.},
  archive      = {J_EVLETT},
  author       = {Santostefano, Francesca and Moiron, Maria and Sánchez-Tójar, Alfredo and Fisher, David N},
  doi          = {10.1093/evlett/qrae051},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {89-104},
  shortjournal = {Evol. Lett.},
  title        = {Indirect genetic effects increase the heritable variation available to selection and are largest for behaviors: A meta-analysis},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fluctuating selection in a monkeyflower hybrid zone. <em>EVLETT</em>, <em>9</em>(1), 77-88. (<a href='https://doi.org/10.1093/evlett/qrae050'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While hybridization was viewed as a hindrance to adaptation and speciation by early evolutionary biologists, recent studies have demonstrated the importance of hybridization in facilitating evolutionary processes. However, it is still not well-known what role spatial and temporal variation in natural selection play in the maintenance of naturally occurring hybrid zones. To identify whether hybridization is adaptive between two closely related monkeyflower species, Mimulus guttatus and Mimulus laciniatus , we performed repeated reciprocal transplants between natural hybrid and pure species’ populations. We planted parental genotypes along with multiple experimental hybrid generations in a dry (2021) and extremely wet (2023) year in the Sierra Nevada, CA. By taking fine-scale environmental measurements, we found that the environment of the hybrid zone is more similar to M. laciniatus’s seasonally dry rocky outcrop habitat than M. guttatus’s moist meadows. In our transplants hybridization does not appear to be maintained by a consistent fitness advantage of hybrids over parental species in hybrid zones, but rather a lack of strong selection against hybrids. We also found higher fitness of the drought-adapted species, M. laciniatus, than M. guttatus in both species’ habitats, as well as phenotypic selection for M. laciniatus- like traits in the hybrid habitat in the dry year of our experiment. These findings suggest that in this system, hybridization might function to introduce drought-adapted traits and genes from M. laciniatus into M. guttatus , specifically in years with limited soil moisture. However, we also find evidence of genetic incompatibilities in second generation hybrids in the wetter year, which may balance a selective advantage of M. laciniatus introgression. Therefore, we find that hybridization in this system is both potentially adaptive and costly, and that the interaction of positive and negative selection likely determines patterns of gene flow between these Mimulus species. Early evolutionary biologists understood hybridization, or interbreeding between species, as limiting to adaptation. While recent studies have shown that hybridization is important for adaptation, much remains to be learned about the role of natural selection in maintaining hybridization. We use a repeated transplant experiment in dry and wet years with two closely related monkeyflower species, Mimulus guttatus and Mimulus laciniatus , and experimental hybrids, to identify how hybridization is maintained. By measuring environmental variables, we found that in some years the hybrid zone is more similar to M. laciniatus’s habitat than M. guttatus’s . We found that hybrids do equally well as parental species in hybrid zones. Additionally, the drought-adapted species, M. laciniatus, performed better than M. guttatus in both parental habitats, and there was selection for more M. laciniatus- like traits in the hybrid habitat. These results suggest that hybridization might introduce drought-adapted traits and genes from M. laciniatus in a dry year. In a wet year, first-generation hybrids performed better than advanced generation hybrids, possibly due to negative genetic interactions. In summary, we find that hybridization is beneficial and costly, and variation in environmental factors likely determines patterns of hybridization.},
  archive      = {J_EVLETT},
  author       = {Tataru, Diana and De Leon, Max and Dutton, Spencer and Machado Perez, Fidel and Rendahl, Alexander and Ferris, Kathleen G},
  doi          = {10.1093/evlett/qrae050},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {77-88},
  shortjournal = {Evol. Lett.},
  title        = {Fluctuating selection in a monkeyflower hybrid zone},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sex-specific expression of circadian rhythms enables allochronic speciation. <em>EVLETT</em>, <em>9</em>(1), 65-76. (<a href='https://doi.org/10.1093/evlett/qrae049'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Noctuid moths provide prime examples of species in various stages of allochronic speciation, where reproductive barriers are mediated by genetic divergence in daily or seasonal timing. Theory indicates that allochronic divergence might be one of the most plausible mechanisms of adaptive speciation, especially when timing is subject to divergent ecological selection. Here, we show that the validity of this theoretical expectation is entirely contingent on species characteristics of the mating system. Our analysis focuses on the moth Spodoptera frugiperda (Lepidoptera, Noctuidae), which occurs as two strains that differ in circadian reproductive activity. Unlike in generic models of assortative mating, where chronotypes diverge under mild assumptions, individual-based evolutionary simulations of the mating system and life cycle of S. frugiperda fail to recover allochronic diversification, even under conditions highly conducive to speciation. Instead, we observe that both chronotypes advance their activity schedule toward the early night, resulting in a rapid loss of allochronic variation. This outcome is caused by the fact that mating in S. frugiperda takes considerable time and potential mates are encountered sequentially, so that early males enjoy a systematic advantage. The undermining effect of male mate competition can be overcome when circadian genes evolve sex-specific expression, enabling early and late chronotypes to be maintained or even to diversify in sympatry. These results give new significance to sex differences in biological rhythms and suggest that species characteristics of the mating system and genetic architecture are key to understanding the scope for allochronic speciation across diverse species exhibiting variation in timing. Various species of moths exhibit differences in daily or seasonal reproductive activity patterns that create barriers to gene flow within populations. Over evolutionary time, progressive divergence in timing may culminate in the origin of separate species isolated by temporal differences. Theory suggests that such temporal isolation is one of the most plausible mechanisms of ecological speciation. Yet, here we model a case study of temporal isolation in the fall armyworm ( Spodoptera frugiperda ) and illustrate that divergence of reproductive timing is much less straightforward than anticipated. In our evolutionary simulation of the mating habits of the fall armyworm, timing differences between preexisting chronotypes erode rapidly due to a systematic advantage of early males: when mating takes substantial time, as in the fall armyworm, these males secure a larger fraction of the mating opportunities than males who are active late. We propose that circadian genes with sex-specific expression are key to overcoming this obstacle and show that timing differences between the sexes can evolve hand-in-hand with the diversification of mating times between progressively reproductively isolated populations. Our findings underscore the importance of sex differences in biological rhythms and species-characteristics of the mating system and shed new light on the conditions for allochronic diversification.},
  archive      = {J_EVLETT},
  author       = {van Doorn, G Sander and Schepers, Jens and Hut, Roelof A and Groot, Astrid T},
  doi          = {10.1093/evlett/qrae049},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {65-76},
  shortjournal = {Evol. Lett.},
  title        = {Sex-specific expression of circadian rhythms enables allochronic speciation},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Disentangling evolutionary, geometric and ecological components of the elevational gradient of diversity. <em>EVLETT</em>, <em>9</em>(1), 51-64. (<a href='https://doi.org/10.1093/evlett/qrae048'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the high importance and risk of mountain ecosystems in global biodiversity conservation, the mechanisms giving rise to and maintaining elevational biodiversity gradients are poorly understood, limiting predictions of future responses. Species richness peaks at lowlands for many taxa, which might be a consequence of mountain shape, reducing available area in highlands. For other taxa, diversity can be highest at mid elevations, suggesting the presence of mechanisms that counteract the influence of geometry. Here, we mechanistically investigate the role of mountain geometry (smaller at the peak) interaction with ecological niche width, diversification, and altitudinal dispersal to investigate the relative roles of these processes in shaping elevational biodiversity gradients. We simulated landscapes and lineages until species richness stop increasing and showed that the disproportionately large area of lowlands provides opportunity for higher species accumulation than any other elevation, even when available niche width and per-capita diversification rate are uniform across altitudes. Regardless of the underlying Elevational Diversity Gradient, altitudinal dispersal always plays a stronger role in maintaining highland than lowland diversity, due to unequal areas involved. To empirically test these predictions resulting from our model, we fit dynamic models of diversification and altitudinal dispersal to three mountainous endemic radiations whose species richness peaks in mid and high-elevation. We find that highland diversity is explained by increased diversification rates with elevation in Fijian bees, whereas niche availability is more likely to explain high altitude diversity in frailejon bushes and earless frogs, suggesting these clades are still growing. Our model and findings provide a new framework for distinguishing drivers of diversity dynamics on mountainsides and allow to detect the presence of clade-specific mechanisms underlying the geometry-diversity relationship. Understanding of these ecological and evolutionary forces can allow increased predictability of how ongoing land use and climate changes will impact future highland biodiversity. The evolutionary dynamics that occur in mountain regions are thought to influence global patterns of biodiversity. Yet, recent global temperature rise threatens these fragile and rapidly changing ecosystems, spurring a need to understand the interaction of biotic and abiotic factors in driving biodiversity dynamics along elevation. In particular, because area availability tends to decrease with elevation, the accumulation of biodiversity along gradients of elevation may be a consequence of any ecological or evolutionary process that is area-dependent. In this paper, by the means of extensive simulations, we generate new theoretical expectations on the interaction of mountain geometry with rates of altitudinal dispersal, evolutionary diversification, and ecological niche breadth. Along with this, we fit models to three mountainous endemic radiations to compare with the theoretical predictions of our models. This work is very timely as it provides a new framework for distinguishing drivers of diversity dynamics on mountainsides and allows us to detect the presence of clade-specific mechanisms underlying the geometry-diversity relationship.},
  archive      = {J_EVLETT},
  author       = {Herrera-Alsina, Leonel and Parvanova, Rossina and Guirguis, Jacinta and Bocedi, Greta and Trethowan, Liam and Lancaster, Lesley T and Travis, Justin M J},
  doi          = {10.1093/evlett/qrae048},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {51-64},
  shortjournal = {Evol. Lett.},
  title        = {Disentangling evolutionary, geometric and ecological components of the elevational gradient of diversity},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Selection on the vascular-remodeling BMPER gene is associated with altitudinal adaptation in an insular lizard. <em>EVLETT</em>, <em>9</em>(1), 41-50. (<a href='https://doi.org/10.1093/evlett/qrae047'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High altitude imposes several extreme constraints on life, such as low oxygen pressure and high levels of ultraviolet radiation, which require specialized adaptations. Many studies have focused on how endothermic vertebrates respond to these challenging environments, but there is still uncertainty on how ectotherms adapt to these conditions. Here, we used whole-genome sequencing of low-altitude (100–600 m) and high-altitude (3,550 m) populations of the wide-ranging Tenerife lizard Gallotia galloti to uncover signatures of selection for altitudinal adaptation. The studied populations show reduced differentiation, sharing similar patterns of genetic variation. Selective sweep mapping suggests that signatures of adaptation to high altitude are not widespread across the genome, clustering in a relatively small number of genomic regions. One of these regions contains BMPER , a gene involved with vascular remodeling, and that has been associated with hypoxia-induced angiogenic response. By genotyping samples across 2 altitudinal transects, we show that allele frequency changes at this locus are not gradual, but rather show a well-defined shift above ca. 1,900 m. Transcript and protein structure analyses on this gene suggest that putative selection likely acts on noncoding variation. These results underline how low oxygen pressure generates the most consistent selective constraint in high-altitude environments, to which vertebrates with vastly contrasting physiological profiles need to adapt in the context of ongoing climate change. Animals that inhabit high-altitude areas face a variety of extreme conditions that are not felt closer to sea level, including low availability of oxygen, high levels of ultraviolet radiation, and low temperatures, to which they are adapted. For endothermic vertebrates, like mammals and birds, multiple studies have revealed how genetic changes can promote this adaptive process. A classic example is that of human populations in Tibet, which possess a high frequency of genetic variants that enable them to cope with the reduced oxygen availability in the Tibetan plateau. In contrast, ectotherms such as reptiles and amphibians have received far less research attention, so the mechanisms that mediate their responses to high altitudes are largely unknown. To fill this gap, we studied the genome of a unique species—the Tenerife lizard that is endemic to the Canary Islands. Its distribution ranges across a steep altitudinal gradient in the island of Tenerife, where it can be found from sea level all the way to the summit of the Teide volcano, at ca. 3,700 m. By comparing all the genetic information from low-altitude and high-altitude populations, we found only a reduced number of regions of the genome that show evidence of having undergone natural selection at high altitudes. Among these, the genomic region containing the gene BMPER (implicated in blood vessel formation and whose expression is changed by low oxygen conditions) shows a strong pattern of selection, in particular changes in the predominant genetic variants in populations living above 1,900 m. These results underline how low oxygen availability is a key factor that controls the evolution of animals living at high altitudes and that the complex and multifaceted responses to these environments may require the integration of genetic and nongenetic mechanisms.},
  archive      = {J_EVLETT},
  author       = {Serén, Nina and Pinho, Catarina and Megía-Palma, Rodrigo and Aguilar, Prem and Žagar, Anamarija and Andrade, Pedro and Carretero, Miguel A},
  doi          = {10.1093/evlett/qrae047},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {41-50},
  shortjournal = {Evol. Lett.},
  title        = {Selection on the vascular-remodeling BMPER gene is associated with altitudinal adaptation in an insular lizard},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A classic key innovation constrains oral jaw functional diversification in fishes. <em>EVLETT</em>, <em>9</em>(1), 24-40. (<a href='https://doi.org/10.1093/evlett/qrae046'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modifications to the pharyngeal jaws—a prey processing system located posterior to the mouth cavity—are widely considered a key innovation that enhanced diversification within several prominent fish clades. Seen in cichlids, damselfishes, wrasses, and a few other lineages, these musculoskeletal alterations are believed to increase the evolutionary independence and, thus, the diversification of the oral and pharyngeal jaw systems. To test this classic hypothesis, we conducted comparative phylogenetic analyses to assess the effect of the pharyngeal novelty on the diversification of feeding morphology and kinematics across a taxonomically diverse sample of spiny-rayed fishes. We quantified movements of the oral jaws and other craniofacial structures from 689 suction-feeding strikes using high-speed videos collected from 228 species with and without the pharyngeal jaw novelty. Contradicting long-held predictions, we find significantly greater disparity across all traits and faster rates of oral jaw functional evolution in fishes without the specialized prey processing system. The modified pharyngeal jaw is undoubtedly a functional innovation as it enhances the strength of the prey processing system, facilitating exceptional transition rates to feeding on hard and tough prey. However, it also restricts the diversification of the feeding system, revealing that the impact of pharyngognathy is more nuanced than previously thought. In light of these and other recent findings, a reinterpretation of the macroevolutionary consequences of the pharyngeal jaw novelty is needed. Thousands of fish species inhabit diverse aquatic environments due, in part, to the evolution of functional innovations. Such traits enable novel interactions with the environment, often facilitating increased diversification within the lineages in which they occur. Modifications to the pharyngeal jaws—a second jaw system located behind the mouth cavity that primarily functions to process prey—are widely considered innovations as multiple musculoskeletal changes allow fishes to process harder and tougher prey items (e.g., mollusks, algae). These modifications to the prey processing apparatus are predicted to increase the independence of the oral and pharyngeal jaws, enabling greater morphological and functional diversification in both systems. To test this classic hypothesis, we examined the diversification of oral jaw morphology and feeding motions in fishes with the modified pharyngeal jaw (133 species representing 4 families) and those without it (95 species representing 39 families). Contrary to long-held predictions, fishes with a specialized prey processing system show less variance across all traits and slower rates of functional evolution. Thus, although the novelty enhances the functional capacity of the pharyngeal jaws, it is also associated with restricted diversification of the feeding system. Our results highlight the importance of examining the consequences of specialized traits within a comparative framework, as their presence may not be uniformly advantageous to the diversification of functional systems.},
  archive      = {J_EVLETT},
  author       = {Roberts-Hugghis, Alexus S and Martinez, Christopher M and Corn, Katherine A and Wainwright, Peter C},
  doi          = {10.1093/evlett/qrae046},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {24-40},
  shortjournal = {Evol. Lett.},
  title        = {A classic key innovation constrains oral jaw functional diversification in fishes},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A shared developmental genetic basis for sexually antagonistic male and female adaptations in the toothed water strider. <em>EVLETT</em>, <em>9</em>(1), 13-23. (<a href='https://doi.org/10.1093/evlett/qrae056'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sexual conflict can drive the divergence of male and female phenotypes and cross-species comparative analyses have documented patterns of correlated evolution of sex-specific traits that promote the evolutionary interests of the sexes. However, male–female coevolution can be highly dynamic, particularly if the male and female traits share an underlying genetic program. Here, we use water striders, a well-studied model system for sexually antagonistic coevolution, and ask whether sex-specific phenotypic adaptations covary across populations and whether they share a common developmental genetic basis. Using comparative analyses both at the population and species levels, we document an association between a derived male mate-grasping trait and a putative female antigrasping counteradaptation in the toothed water strider Gerris odontogaster . Interestingly, in several populations where males have partly lost their derived grasping trait, females have also reduced their antigrasping adaptation. We used RNAi to show that these male and female traits are both linked to a common developmental genetic program involving Hox- and sex-determination genes, despite the fact that they are different structures on different abdominal segments. Our work illustrates the dynamic nature of sexually antagonistic coevolution and suggests that the pleiotropic nature of developmental genetic programs can blur the distinction between inter- and intralocus genetic conflict. The evolutionary interests of males and females often diverge, particularly over mating interactions. This coevolutionary pattern leads to the evolution of adaptations and counteradaptations that further the evolutionary interest of each sex. How the phenotypic divergence of the sexes emerges at small time scales and despite the shared genome is not fully understood. We address these questions in the toothed water strider, Gerris odontogaster , a classical model for the study of sexual conflict. By analyzing multiple populations and species, we uncover strongly correlated evolution between male abdominal teeth that act to grasp the female and a counteradaptation in the females consisting of a concealed proctiger that is the target of the male trait. We also show, through manipulating gene expression, that the male and the female traits share, at least partly, an underlying developmental genetic program containing both Hox- and sex-determination genes. This study paves the way to testing the extent of overlap between intra- and interlocus genetic conflict.},
  archive      = {J_EVLETT},
  author       = {Pruvôt, Claudia and Armisén, David and Roux, Pascale and Arnqvist, Göran and Rowe, Locke and Husby, Arild and Khila, Abderrahman},
  doi          = {10.1093/evlett/qrae056},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {13-23},
  shortjournal = {Evol. Lett.},
  title        = {A shared developmental genetic basis for sexually antagonistic male and female adaptations in the toothed water strider},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic structure and common ancestry expose the dingo-dog hybrid myth. <em>EVLETT</em>, <em>9</em>(1), 1-12. (<a href='https://doi.org/10.1093/evlett/qrae057'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The evolutionary history of canids has been shown to be complex, with hybridization and domestication confounding our understanding of speciation among various canid lineages. The dingo is a recent canid lineage that was completely isolated from other canids for over 5000 years on the Australian mainland, but the introduction of domestic dogs in 1788 has placed doubt on its independence, with recent studies highlighting hybridization between dingoes and domestic dogs. Using genomic single nucleotide polymorphism data from 434 Australian canid samples, we explicitly test for introgression between closely related canid groups and dingoes. We found no evidence of introgression between dingoes and domestic dogs and show that previous work has likely mischaracterized shared ancestral genetic variation as evidence for hybridization. Further, New Guinea Singing Dogs are the only canid group that significantly shared genetic variation with dingoes, which fits with our understanding of previous phylogenetic analyses. Despite more recent sympatric distributions with dogs, dingoes have likely maintained their independence since their arrival in Australia, even in areas with high lethal control, indicating that their evolutionary trajectory is currently being conserved. The future conservation of the dingo lineage will require policies that promote coexistence pathways between humans and dingoes that protect rangeland systems and the dingoes’ evolutionary future. The dingo is an old lineage of dog that was isolated from other canids on the Australian mainland for over 5,000 years prior to the arrival of domestic dogs in 1788. Here, we explore the evolutionary relationships between the Australian dingo and other canids using genomic single nucleotide polymorphisms and a large sample dataset. We show that despite decades of previous research indicating widespread hybridization between dingoes and domestic dogs, we find no evidence of introgression and the Australian dingo still represents a unique lineage on a pathway towards speciation. These results have important implications for the future conservation of this evolutionary unique canid lineage, where lethal control and habitat destruction are the greatest threat to the dingo’s continued evolution.},
  archive      = {J_EVLETT},
  author       = {Weeks, Andrew R and Kriesner, Peter and Bartonicek, Nenad and van Rooyen, Anthony and Cairns, Kylie M and Ahrens, Collin W},
  doi          = {10.1093/evlett/qrae057},
  journal      = {Evolution Letters},
  month        = {2},
  number       = {1},
  pages        = {1-12},
  shortjournal = {Evol. Lett.},
  title        = {Genetic structure and common ancestry expose the dingo-dog hybrid myth},
  volume       = {9},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
