<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NG</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="ng">NG - 334</h2>
<ul>
<li><details>
<summary>
(2025). Publisher correction: Genome-wide association meta-analysis of childhood ADHD symptoms and diagnosis identifies new loci and potential effector genes. <em>NG</em>, <em>57</em>(10), 2604. (<a href='https://doi.org/10.1038/s41588-025-02383-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {van der Laan, Camiel M. and Ip, Hill F. and Schipper, Marijn and Hottenga, Jouke-Jan and St Pourcain, Beate and Zayats, Tetyana and Pool, René and Krapohl, Eva M. L. and Brikell, Isabell and Soler Artigas, María and Cabana-Domínguez, Judit and Llonga, Natalia and Nolte, Ilja M. and Bolhuis, Koen and Palviainen, Teemu and Zafarmand, Hadi and Gordon, Scott and Aliev, Fazil and Burt, S. Alexandra and Wang, Carol A. and Saunders, Gretchen and Karhunen, Ville and Adkins, Daniel E. and Border, Richard and Peterson, Roseann E. and Prinz, Joseph A. and Thiering, Elisabeth and Vilor-Tejedor, Natàlia and Ahluwalia, Tarunveer S. and Allegrini, Andrea and Rimfeld, Kaili and Chen, Qi and Lu, Yi and Martin, Joanna and Bosch, Rosa and Ramos-Quiroga, Josep Antoni and Neumann, Alexander and Ensink, Judith and Grasby, Katrina L. and Morosoli, José J. and Tong, Xiaoran and Marrington, Shelby and Scott, James G. and Shabalin, Andrey A. and Corley, Robin and Evans, Luke M. and Sugden, Karen and Alemany, Silvia and Sass, Lærke and Vinding, Rebecca and Ehli, Erik A. and Hagenbeek, Fiona A. and Derks, Eske M. and Larsson, Henrik and Snieder, Harold and Cecil, Charlotte and Whipp, Alyce M. and Korhonen, Tellervo and Vuoksimaa, Eero and Rose, Richard J. and Uitterlinden, André G. and Haavik, Jan and Harris, Jennifer R. and Helgeland, Øyvind and Johansson, Stefan and Knudsen, Gun Peggy S. and Njolstad, Pal Rasmus and Lu, Qing and Rodriguez, Alina and Henders, Anjali K. and Mamun, Abdullah and Najman, Jackob M. and Brown, Sandy and Hopfer, Christian and Krauter, Kenneth and Reynolds, Chandra A. and Smolen, Andrew and Stallings, Michael and Wadsworth, Sally and Wall, Tamara L. and Eaves, Lindon and Silberg, Judy L. and Miller, Allison and Havdahl, Alexandra and Llop, Sabrina and Lopez-Espinosa, Maria-Jose and Bønnelykke, Klaus and Sunyer, Jordi and Arseneault, Louise and Standl, Marie and Heinrich, Joachim and Boden, Joseph and Pearson, John and Horwood, John and Kennedy, Martin and Poulton, Richie and Maes, Hermine H. and Hewitt, John and Copeland, William E. and Middeldorp, Christel M. and Williams, Gail M. and Wray, Naomi and Järvelin, Marjo-Riitta and McGue, Matt and Iacono, William and Caspi, Avshalom and Moffitt, Terrie E. and Whitehouse, Andrew J. O. and Pennell, Craig E. and Klump, Kelly L. and Jiang, Chang and Dick, Danielle M. and Reichborn-Kjennerud, Ted and Martin, Nicholas G. and Medland, Sarah E. and Vrijkotte, Tanja and Kaprio, Jaakko and Tiemeier, Henning and Davey Smith, George and Hartman, Catharina A. and Oldehinkel, Albertine J. and Casas, Miquel and Ribasés, Marta and Lichtenstein, Paul and Lundström, Sebastian and Plomin, Robert and Bartels, Meike and Nivard, Michel G. and Boomsma, Dorret I.},
  doi          = {10.1038/s41588-025-02383-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2604},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Genome-wide association meta-analysis of childhood ADHD symptoms and diagnosis identifies new loci and potential effector genes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: The genomic landscape of gene-level structural variations in japanese and global soybean glycine max cultivars. <em>NG</em>, <em>57</em>(10), 2603. (<a href='https://doi.org/10.1038/s41588-025-02356-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Yano, Ryoichi and Li, Feng and Hiraga, Susumu and Takeshima, Ryoma and Kobayashi, Michie and Toda, Kyoko and Umehara, Yosuke and Kajiya-Kanegae, Hiromi and Iwata, Hiroyoshi and Kaga, Akito and Ishimoto, Masao},
  doi          = {10.1038/s41588-025-02356-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2603},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: The genomic landscape of gene-level structural variations in japanese and global soybean glycine max cultivars},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Why genomic diversity should not be framed by census alone. <em>NG</em>, <em>57</em>(10), 2603. (<a href='https://doi.org/10.1038/s41588-025-02357-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Corpas, Manuel and Guio, Heinner and Lopez-Correa, Catalina and Fatumo, Segun},
  doi          = {10.1038/s41588-025-02357-1},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2603},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Why genomic diversity should not be framed by census alone},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nucleotide dependency analysis of genomic language models detects functional elements. <em>NG</em>, <em>57</em>(10), 2589-2602. (<a href='https://doi.org/10.1038/s41588-025-02347-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deciphering how nucleotides in genomes encode regulatory instructions and molecular machines is a long-standing goal. Genomic language models (gLMs) implicitly capture functional elements and their organization from genomic sequences alone by modeling probabilities of each nucleotide given its sequence context. However, discovering functional genomic elements from gLMs has been challenging due to the lack of interpretable methods. Here we introduce nucleotide dependencies, which quantify how nucleotide substitutions at one genomic position affect the probabilities of nucleotides at other positions. We demonstrate that nucleotide dependencies are more effective at indicating the deleteriousness of genetic variants than alignment-based conservation and gLM reconstruction. Dependency analysis accurately detects regulatory motifs and highlights bases in contact within RNAs, including pseudoknots and tertiary structure contacts, revealing new, experimentally validated RNA structures. Finally, we leverage dependency maps to reveal critical limitations of several gLM architectures and training strategies. Altogether, nucleotide dependency analysis opens a new avenue for discovering and studying functional elements and their interactions in genomes. Mapping pairwise nucleotide dependencies by leveraging genomic language models highlights functional genomic elements and predicts deleterious genetic variants more effectively than alignment-based conservation metrics.},
  archive      = {J_NG},
  author       = {Tomaz da Silva, Pedro and Karollus, Alexander and Hingerl, Johannes and Galindez, Gihanna Sta. Teresa and Wagner, Nils and Hernandez-Alias, Xavier and Incarnato, Danny and Gagneur, Julien},
  doi          = {10.1038/s41588-025-02347-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2589-2602},
  shortjournal = {Nat. Genet.},
  title        = {Nucleotide dependency analysis of genomic language models detects functional elements},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic variants affecting RNA stability influence complex traits and disease risk. <em>NG</em>, <em>57</em>(10), 2578-2588. (<a href='https://doi.org/10.1038/s41588-025-02326-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression is modulated jointly by transcriptional regulation and messenger RNA stability, yet the latter is often overlooked in studies on genetic variants. Here, leveraging metabolic labeling data (Bru/BruChase-seq) and a new computational pipeline, RNAtracker, we categorize genes as allele-specific RNA stability (asRS) or allele-specific RNA transcription events. We identify more than 5,000 asRS variants among 665 genes across a panel of 11 human cell lines. These variants directly overlap conserved microRNA target regions and allele-specific RNA-binding protein sites, illuminating mechanisms through which stability is mediated. Furthermore, we identified causal asRS variants using a massively parallel screen (MapUTR) for variants that affect post-transcriptional mRNA abundance, as well as through CRISPR prime editing approaches. Notably, asRS genes were enriched significantly among a multitude of immune-related pathways and contribute to the risk of several immune system diseases. This work highlights RNA stability as a critical, yet understudied mechanism linking genetic variation and disease. RNAtracker is a computational pipeline that distinguishes variants associated with allele-specific RNA stability from those associated with allele-specific RNA transcription. Variants affecting RNA stability are enriched in immune-related genes and contribute to disease risk.},
  archive      = {J_NG},
  author       = {Huang, Elaine and Fu, Ting and Zhang, Ling and Yan, Guanao and Yamamoto, Ryo and Terrazas, Sari and Nguyen, Thuy Linh and Gonzalez-Figueroa, Carlos and Khanbabaei, Armen and Bahn, Jae Hoon and Varada, Rajagopal and Amoah, Kofi and Hervoso, Jonatan and Paulsen, Michelle T. and Magnuson, Brian and Ljungman, Mats and Li, Jingyi Jessica and Xiao, Xinshu},
  doi          = {10.1038/s41588-025-02326-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2578-2588},
  shortjournal = {Nat. Genet.},
  title        = {Genetic variants affecting RNA stability influence complex traits and disease risk},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Accelerated bayesian inference of population size history from recombining sequence data. <em>NG</em>, <em>57</em>(10), 2570-2577. (<a href='https://doi.org/10.1038/s41588-025-02323-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This study introduces population history learning by averaging sampled histories (PHLASH), a new method for inferring population history from whole-genome sequence data. It works by drawing random, low-dimensional projections of the coalescent intensity function from the posterior distribution of a pairwise sequentially Markovian coalescent-like model and averaging them together to form an accurate and adaptive estimator. On simulated data, PHLASH tends to be faster and have lower error than several competing methods, including SMC++, MSMC2 and FITCOAL. Moreover, it provides automatic uncertainty quantification and leads to new Bayesian testing procedures for detecting population structure and ancient bottlenecks. The key technical advance is a new algorithm for computing the score function (gradient of the log likelihood) of a coalescent hidden Markov model, which has the same computational cost as evaluating the log likelihood. PHLASH has been released as an easy-to-use Python software package and leverages graphics processing unit acceleration when available. Population history learning by averaging sampled histories is a new Bayesian method for estimating historical effective population size from recombining sequence data that offers improved speed and accuracy and enables uncertainty quantification and testing for ancient bottlenecks.},
  archive      = {J_NG},
  author       = {Terhorst, Jonathan},
  doi          = {10.1038/s41588-025-02323-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2570-2577},
  shortjournal = {Nat. Genet.},
  title        = {Accelerated bayesian inference of population size history from recombining sequence data},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bryophytes hold a larger gene family space than vascular plants. <em>NG</em>, <em>57</em>(10), 2562-2569. (<a href='https://doi.org/10.1038/s41588-025-02325-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After 500 million years of evolution, extant land plants compose the following two sister groups: the bryophytes and the vascular plants. Despite their small size and simple structure, bryophytes thrive in a wide variety of habitats, including extreme conditions. However, the genetic basis for their ecological adaptability and long-term survival is not well understood. A comprehensive super-pangenome analysis, incorporating 123 newly sequenced bryophyte genomes, reveals that bryophytes possess a substantially greater diversity of gene families than vascular plants. This includes a higher number of unique and lineage-specific gene families, originating from extensive new gene formation and continuous horizontal transfer of microbial genes over their long evolutionary history. The evolution of bryophytes’ rich and diverse genetic toolkit, which includes new physiological innovations like unique immune receptors, likely facilitated their spread across different biomes. These newly sequenced bryophyte genomes offer a valuable resource for exploring alternative evolutionary strategies for terrestrial success. A super-pangenome analysis incorporating 123 newly sequenced bryophyte genomes reveals that bryophytes exhibit a larger number of unique and lineage-specific gene families than vascular plants.},
  archive      = {J_NG},
  author       = {Dong, Shanshan and Wang, Sibo and Li, Linzhou and Yu, Jin and Zhang, Yongxia and Xue, Jia-Yu and Chen, Hengchi and Ma, Jianchao and Zeng, Yuying and Cai, Yuqing and Huang, Wei and Zhou, Xuping and Wu, Jiayi and Li, Jianyou and Yao, Yifeng and Hu, Ruoyang and Zhao, Tao and Villarreal A, Juan Carlos and Dirick, Leon and Liu, Li and Ignatov, Michael and Jin, Minghui and Ruan, Jue and He, Yikun and Wang, Haifeng and Xu, Bo and Rozzi, Ricardo and Wegrzyn, Jill and Stevenson, Dennis William and Renzaglia, Karen S. and Chen, Hongfeng and Zhang, Li and Zhang, Shouzhou and Mackenzie, Roy and Moreno, Javier E. and Melkonian, Michael and Wei, Tong and Gu, Ying and Xu, Xun and Rensing, Stefan A. and Huang, Jinling and Long, Manyuan and Goffinet, Bernard and Bowman, John L. and Van de Peer, Yves and Liu, Huan and Liu, Yang},
  doi          = {10.1038/s41588-025-02325-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2562-2569},
  shortjournal = {Nat. Genet.},
  title        = {Bryophytes hold a larger gene family space than vascular plants},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A multi-tissue single-cell expression atlas in cattle. <em>NG</em>, <em>57</em>(10), 2546-2561. (<a href='https://doi.org/10.1038/s41588-025-02329-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Systematic characterization of the molecular states of cells in livestock tissues is essential for understanding the cellular and genetic mechanisms underlying economically and ecologically important physiological traits. Here, as part of the Farm Animal Genotype-Tissue Expression (FarmGTEx) project, we describe a comprehensive reference map including 1,793,854 cells from 59 bovine tissues in calves and adult cattle, spanning both sexes, which reveals intra-tissue and inter-tissue cellular heterogeneity in gene expression, transcription factor regulation and intercellular communication. Integrative analysis with genetic variants that underpin bovine monogenic and complex traits uncovers cell types of relevance, such as spermatocytes, responsible for sperm motility and excitatory neurons for milk fat yield. Comparative analysis reveals similarities in gene expression between cattle and humans, allowing for the detection of relevant cell types to study human complex phenotypes. This Cattle Cell Atlas will serve as a key resource for cattle genetics and genomics, selective breeding and comparative biology. This study introduces the Cattle Cell Atlas, a single-cell expression resource including 1,793,854 cells from 59 tissues. Integrative analyses leveraging this atlas provide insights into the biology underlying bovine monogenic and complex traits.},
  archive      = {J_NG},
  author       = {Han, Bo and Li, Houcheng and Zheng, Weijie and Zhang, Qi and Chen, Ao and Zhu, Senlin and Shi, Tao and Wang, Fei and Zou, Dong and Song, Yu and Ye, Wen and Du, Aixia and Fu, Yihan and Jia, Minghui and Bai, Zhonghao and Yuan, Zhixiang and Liu, Wansheng and Tuo, Wenbin and Hope, Jayne C. and MacHugh, David E. and O’Grady, John F. and Madsen, Ole and Sahana, Goutam and Luo, Yonglun and Lin, Lin and Li, Congjun and Cai, Zexi and Li, Bingjie and Huang, Jinming and Liu, Lin and Zhang, Zhang and Ma, Zhu and Hou, Yali and Liu, George E. and Jiang, Yu and Sun, Hui-zeng and Fang, Lingzhao and Sun, Dongxiao},
  doi          = {10.1038/s41588-025-02329-5},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2546-2561},
  shortjournal = {Nat. Genet.},
  title        = {A multi-tissue single-cell expression atlas in cattle},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic and epigenetic screens in primary human t cells link candidate causal autoimmune variants to t cell networks. <em>NG</em>, <em>57</em>(10), 2536-2545. (<a href='https://doi.org/10.1038/s41588-025-02301-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic variants associated with autoimmune diseases are highly enriched within putative cis-regulatory regions of CD4+ T cells, suggesting that they could alter disease risk through changes in gene regulation. However, very few genetic variants have been shown to affect T cell gene expression or function. Here we tested &gt;18,000 autoimmune disease-associated variants for allele-specific effects on expression using massively parallel reporter assays in primary human CD4+ T cells. We find 545 variants that modulate expression in an allele-specific manner (emVars). Primary T cell emVars greatly enrich for likely causal variants, are mediated by common upstream pathways and their putative target genes are highly enriched within a lymphocyte activation network. Using bulk and single-cell CRISPR-interference screens, we confirm that emVar-containing T cell cis-regulatory elements modulate both known and previously unappreciated target genes that regulate T cell proliferation, providing plausible mechanisms by which these variants alter autoimmune disease risk. Massively parallel reporter assay in primary human CD4+ T cells and bulk and single-cell CRISPR-interference screens identify candidate causal variants linked to autoimmune disease risk that modulate T cell gene expression and proliferation.},
  archive      = {J_NG},
  author       = {Ho, Ching-Huang and Dippel, Maxwell A. and McQuade, Meghan S. and Nguyen, LeAnn P. and Mishra, Arpit and Pribitzer, Stephan and Hardy, Samantha and Chandok, Harshpreet and Chardon, Florence M. and McDiarmid, Troy A. and DeBerg, Hannah A. and Buckner, Jane H. and Shendure, Jay and de Boer, Carl G. and Guo, Michael H. and Tewhey, Ryan and Ray, John P.},
  doi          = {10.1038/s41588-025-02301-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2536-2545},
  shortjournal = {Nat. Genet.},
  title        = {Genetic and epigenetic screens in primary human t cells link candidate causal autoimmune variants to t cell networks},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dissecting the impact of transcription factor dose on cell reprogramming heterogeneity using scTF-seq. <em>NG</em>, <em>57</em>(10), 2522-2535. (<a href='https://doi.org/10.1038/s41588-025-02343-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reprogramming often yields heterogeneous cell fates, yet the underlying mechanisms remain poorly understood. To address this, we developed single-cell transcription factor sequencing (scTF-seq), a single-cell technique that induces barcoded, doxycycline-inducible TF overexpression and quantifies TF dose-dependent transcriptomic changes. Applied to mouse embryonic multipotent stromal cells, scTF-seq generated a gain-of-function atlas for 384 mouse TFs, identifying key regulators of lineage specification, cell cycle control and their interplay. Leveraging single-cell resolution, we uncovered how TF dose shapes reprogramming heterogeneity, revealing both dose-dependent and stochastic cell state transitions. We classified TFs into low-capacity and high-capacity groups, with the latter further subdivided by dose sensitivity. Combinatorial scTF-seq demonstrated that TF interactions can shift from synergistic to antagonistic depending on the relative dose. Altogether, scTF-seq enables the dissection of TF function, dose and cell fate control, providing a high-resolution framework to understand and predict reprogramming outcomes, advancing gene regulation research and the design of cell engineering strategies. This study introduces single-cell transcription factor (TF) sequencing, a single-cell barcoded and doxycycline-inducible TF overexpression approach that reveals dose-sensitive functional classes of TFs and cellular heterogeneity by mapping TF dose-dependent transcriptomic changes during the reprogramming of mouse embryonic multipotent stromal cells.},
  archive      = {J_NG},
  author       = {Liu, Wangjie and Saelens, Wouter and Rainer, Pernille and Biočanin, Marjan and Gardeux, Vincent and Gralak, Antoni Jakub and van Mierlo, Guido and Gebhart, Angelika and Russeil, Julie and Liu, Tingdang and Chen, Wanze and Deplancke, Bart},
  doi          = {10.1038/s41588-025-02343-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2522-2535},
  shortjournal = {Nat. Genet.},
  title        = {Dissecting the impact of transcription factor dose on cell reprogramming heterogeneity using scTF-seq},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DNA methylation influences human centromere positioning and function. <em>NG</em>, <em>57</em>(10), 2509-2521. (<a href='https://doi.org/10.1038/s41588-025-02324-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Maintaining the epigenetic identity of centromeres is essential to prevent genome instability. Centromeres are epigenetically defined by the histone H3 variant CENP-A. Prior work in human centromeres has shown that CENP-A is associated with regions of hypomethylated DNA located within large arrays of hypermethylated repeats, but the functional importance of these DNA methylation (DNAme) patterns remains poorly understood. To address this, we developed tools to perturb centromeric DNAme, revealing that it causally influences CENP-A positioning. We show that rapid loss of methylation results in increased binding of centromeric proteins and alterations in centromere architecture, leading to aneuploidy and reduced cell viability. We also demonstrate that gradual centromeric DNA demethylation prompts a process of cellular adaptation. Altogether, we find that DNAme causally influences CENP-A localization and centromere function, offering mechanistic insights into pathological alterations of centromeric DNAme. Genome-wide and targeted perturbation of DNA methylation at centromeres affects CENP-A positioning and centromere structure, resulting in aneuploidy and reduced cell viability.},
  archive      = {J_NG},
  author       = {Salinas-Luypaert, Catalina and Dubocanin, Danilo and Lee, Rosa Jooyoung and Andrade Ruiz, Lorena and Gamba, Riccardo and Grison, Marine and Velikovsky, Leonid and Angrisani, Annapaola and Scelfo, Andrea and Xu, Yuan and Dumont, Marie and Barra, Viviana and Wilhelm, Therese and Velasco, Guillaume and Losito, Marialucrezia and Wardenaar, René and Francastel, Claire and Foijer, Floris and Kops, Geert J. P. L. and Miga, Karen H. and Altemose, Nicolas and Fachinetti, Daniele},
  doi          = {10.1038/s41588-025-02324-w},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2509-2521},
  shortjournal = {Nat. Genet.},
  title        = {DNA methylation influences human centromere positioning and function},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell multi-omic and spatial profiling of esophageal squamous cell carcinoma reveals the immunosuppressive role of GPR116+ pericytes in cancer metastasis. <em>NG</em>, <em>57</em>(10), 2494-2508. (<a href='https://doi.org/10.1038/s41588-025-02341-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumor metastasis leads to most cancer deaths. However, how cellular diversity and dynamic cooperation within the tumor microenvironment contribute to metastasis remains poorly understood. Here we leverage single-cell multi-omics (16 samples, 117,169 cells) and spatial transcriptomics (five samples, 195,366 cells) to uncover the cellular and spatial architecture of esophageal squamous cell carcinoma (ESCC), and characterize an immunosuppressive GPR116+ pericyte subset promoting tumor metastasis and immunotherapy resistance. GPR116+ pericyte enrichment is transcriptionally regulated by PRRX1, evidenced by pericyte-specific Prrx1 knockout mice. Mechanistically, GPR116+ pericytes secrete EGFL6 to bind integrin β1 on cancer cells, activating the NF-κB pathway to facilitate metastasis. Serum EGFL6 serves as a noninvasive biomarker for the diagnosis and prognosis of several tumors. Blocking integrin β1 suppresses metastasis and improves immunotherapy response in animal models of ESCC. Collectively, we provide a spatially resolved landscape of the prometastatic tumor microenvironment in ESCC and highlight the biological and clinical importance of GPR116+ pericytes, proposing potential innovative therapeutic strategies for metastatic cancers. This study provides a single-cell and spatial atlas of the prometastatic tumor microenvironment in esophageal squamous cell carcinoma and characterizes the immunosuppressive function of GPR116+ pericytes in cancer metastasis and immune evasion.},
  archive      = {J_NG},
  author       = {Pei, Xiaoya and Liu, Zhichao and Tang, Lin and Zhang, Junpeng and He, Yan and Zhuang, Xiaowan and Song, Yuxiang and Peng, Shaocong and Zhao, Yahui and Huang, Xuanzhang and Xu, Yiwei and Li, Junyi and Peng, Liang and Shi, Xingyuan and Zhang, Fan and Liu, Chong and He, Mingliang and Dai, Wei and Wu, Liang and Law, Simon and Liu, Jinbao and Wang, Zhenning and Li, Zhigang and Xu, Wenwen and Liu, Zhihua and Li, Bin},
  doi          = {10.1038/s41588-025-02341-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2494-2508},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell multi-omic and spatial profiling of esophageal squamous cell carcinoma reveals the immunosuppressive role of GPR116+ pericytes in cancer metastasis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial signatures for predicting immunotherapy outcomes using multi-omics in non-small cell lung cancer. <em>NG</em>, <em>57</em>(10), 2482-2493. (<a href='https://doi.org/10.1038/s41588-025-02351-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Non-small cell lung cancer (NSCLC) shows variable responses to immunotherapy, highlighting the need for biomarkers to guide patient selection. We applied a spatial multi-omics approach to 234 advanced NSCLC patients treated with programmed death 1-based immunotherapy across three cohorts to identify biomarkers associated with outcome. Spatial proteomics (n = 67) and spatial compartment-based transcriptomics (n = 131) enabled profiling of the tumor immune microenvironment (TIME). Using spatial proteomics, we identified a resistance cell-type signature including proliferating tumor cells, granulocytes, vessels (hazard ratio (HR) = 3.8, P = 0.004) and a response signature, including M1/M2 macrophages and CD4 T cells (HR = 0.4, P = 0.019). We then generated a cell-to-gene resistance signature using spatial transcriptomics, which was predictive of poor outcomes (HR = 5.3, 2.2, 1.7 across Yale, University of Queensland and University of Athens cohorts), while a cell-to-gene response signature predicted favorable outcomes (HR = 0.22, 0.38 and 0.56, respectively). This framework enables robust TIME modeling and identifies biomarkers to support precision immunotherapy in NSCLC. This study uses spatial proteomic and spatial compartment-based whole-transcriptome profiling to develop predictive models for immunotherapy outcomes in non-small cell lung cancer.},
  archive      = {J_NG},
  author       = {Aung, Thazin N. and Monkman, James and Warrell, Jonathan and Vathiotis, Ioannis and Bates, Katherine M. and Gavrielatou, Niki and Trontzas, Ioannis P. and Tan, Chin Wee and Fernandez, Aileen I. and Moutafi, Myrto and O’ Byrne, Ken and Schalper, Kurt A. and Syrigos, Konstantinos and Herbst, Roy S. and Kulasinghe, Arutha and Rimm, David L.},
  doi          = {10.1038/s41588-025-02351-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2482-2493},
  shortjournal = {Nat. Genet.},
  title        = {Spatial signatures for predicting immunotherapy outcomes using multi-omics in non-small cell lung cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Targeting histone H2B acetylated enhanceosomes via p300/CBP degradation in prostate cancer. <em>NG</em>, <em>57</em>(10), 2468-2481. (<a href='https://doi.org/10.1038/s41588-025-02336-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prostate cancer is driven by oncogenic transcription factor enhanceosomes comprising chromatin and epigenetic regulators. The lysine acetyltransferases p300 and CREB-binding protein (CBP) are key cofactors that activate enhancers through histone acetylation. Here we identify p300/CBP-mediated multisite histone H2B N-terminal acetylation (H2BNTac) as a defining feature of oncogenic enhanceosomes in androgen receptor (AR)-positive prostate cancer. p300/CBP are essential for AR and ETS transcription factor ERG transcriptional activity, and their dual degradation eliminates H2BNTac and histone H3 lysine 27 acetylation at hyperactive enhancers, leading to stronger suppression of oncogenic transcription than targeting either paralog or bromodomain alone. Cytotoxicity profiling across &gt;900 cell lines revealed that tumors with high H2BNTac, including AR-positive prostate cancer, are selectively dependent on p300/CBP. In preclinical models, systemic p300/CBP degradation inhibited tumor growth, synergized with AR antagonists and showed no evident toxicity. These findings position H2BNTac as an epigenetic marker of enhancer addiction and establish dual p300/CBP degradation as a promising therapeutic strategy for enhancer-driven cancers. This study shows that p300/CBP-dependent H2B acetylation is crucial for maintaining transcription of oncogenic gene programs in androgen receptor-driven prostate cancer.},
  archive      = {J_NG},
  author       = {Luo, Jie and Chen, Zhixiang and Qiao, Yuanyuan and Tien, Jean Ching-Yi and Young, Eleanor and Mannan, Rahul and Mahapatra, Somnath and Bhattacharyya, Rupam and Xiao, Lanbo and He, Tongchen and Eyunni, Sanjana and Zhang, Yuping and Zheng, Yang and Su, Fengyun and Cao, Xuhong and Wang, Rui and Cheng, Yunhui and Seri, Rithvik and George, James and Shahine, Miriam and Miner, Stephanie J. and Rees, Matthew G. and Ronan, Melissa M. and Roth, Jennifer A. and Vaishampayan, Ulka and Wang, Mi and Wang, Shaomeng and Parolia, Abhijit and Chinnaiyan, Arul M.},
  doi          = {10.1038/s41588-025-02336-6},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2468-2481},
  shortjournal = {Nat. Genet.},
  title        = {Targeting histone H2B acetylated enhanceosomes via p300/CBP degradation in prostate cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rapid epigenomic classification of acute leukemia. <em>NG</em>, <em>57</em>(10), 2456-2467. (<a href='https://doi.org/10.1038/s41588-025-02321-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Acute leukemia requires precise molecular classification and urgent treatment. However, standard-of-care diagnostic tests are time-intensive and do not capture the full spectrum of acute leukemia heterogeneity. Here, we developed a framework to classify acute leukemia using genome-wide DNA methylation profiling. We first assembled a comprehensive reference cohort (n = 2,540 samples) and defined 38 methylation classes. Methylation-based classification matched standard-pathology lineage classification in most cases and revealed heterogeneity in addition to that captured by genetic categories. Using this reference, we developed a neural network (MARLIN; methylation- and AI-guided rapid leukemia subtype inference) for acute leukemia classification from sparse DNA methylation profiles. In retrospective cohorts profiled by nanopore sequencing, high-confidence predictions were concordant with conventional diagnoses in 25 out of 26 cases. Real-time MARLIN classification in patients with suspected acute leukemia provided accurate predictions in five out of five cases, which were typically generated within 2 h of sample receipt. In summary, we present a framework for rapid acute leukemia classification that complements and enhances standard-of-care diagnostics. The authors present a molecular classification of acute leukemia using 5-methylcytosine signatures, together with a neural network-based classifier for clinical use.},
  archive      = {J_NG},
  author       = {Steinicke, Til L. and Benfatto, Salvatore and Capilla-Guerra, Maria R. and Monteleone, Andre B. and Young, Jonathan H. and Shankar, Subha and Michaels, Phillip D. and Tsai, Harrison K. and Good, Jonathan D. and Kreso, Antonia and van Galen, Peter and Schliemann, Christoph and Chen, Evan C. and Griffin, Gabriel K. and Hovestadt, Volker},
  doi          = {10.1038/s41588-025-02321-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2456-2467},
  shortjournal = {Nat. Genet.},
  title        = {Rapid epigenomic classification of acute leukemia},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A genetic map of human metabolism across the allele frequency spectrum. <em>NG</em>, <em>57</em>(10), 2445-2455. (<a href='https://doi.org/10.1038/s41588-025-02355-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic studies of human metabolism have been limited in scale and allelic breadth. Here we provide a data-driven map of the genetic regulation of circulating small molecules and lipoprotein characteristics (249 traits) measured using proton nuclear magnetic resonance spectroscopy across the allele frequency spectrum in ~450,000 individuals. Trans-ancestral meta-analyses identify 29,824 locus–metabolite associations mapping to 753 regions with effects largely consistent between men and women and large ancestral groups represented in UK Biobank. We observe and classify extreme genetic pleiotropy, identify regulators of lipid metabolism, and assign effector genes at &gt;100 loci through rare-to-common allelic series. We propose roles for genes less established in metabolic control (for example, SIDT2), genes characterized by phenotypic heterogeneity (for example, APOA1) and genes with specific disease relevance (for example, VEGFA). Our study demonstrates the value of broad, large-scale metabolomic phenotyping to identify and characterize regulators of human metabolism. Genome-wide association analyses of 249 circulating small molecules and lipoprotein characteristics across the allele frequency spectrum in UK Biobank advance understanding of genes and pathways regulating human metabolism.},
  archive      = {J_NG},
  author       = {Zoodsma, Martijn and Beuchel, Carl and Yasmeen, Summaira and Kohleick, Leonhard and Nepal, Aakash and Koprulu, Mine and Kronenberg, Florian and Mayr, Manuel and Williamson, Alice and Pietzner, Maik and Langenberg, Claudia},
  doi          = {10.1038/s41588-025-02355-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2445-2455},
  shortjournal = {Nat. Genet.},
  title        = {A genetic map of human metabolism across the allele frequency spectrum},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Population-scale gene-based analysis of whole-genome sequencing provides insights into metabolic health. <em>NG</em>, <em>57</em>(10), 2436-2444. (<a href='https://doi.org/10.1038/s41588-025-02364-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In addition to its coverage of the noncoding genome, whole-genome sequencing (WGS) may better capture the coding genome than exome sequencing. Here we sought to exploit this and identify new rare, protein-coding variants associated with metabolic health in WGS data (n = 708,956) from the UK Biobank and All of Us studies. Identified genes highlight new biological mechanisms, including protein-truncating variants (PTVs) in the DNA double-strand break repair gene RIF1 that have a substantial effect on body mass index (2.66 kg m−2, s.e. 0.43, P = 3.7 × 10−10). UBR3 is an intriguing example where PTVs independently increase body mass index and type 2 diabetes risk. Furthermore, PTVs in IRS2 have a substantial effect on type 2 diabetes (odds ratio 6.4 (3.7–11.3), P = 9.9 × 10−14, 34% case prevalence among carriers) and were also associated with chronic kidney disease independent of diabetes status, suggesting an important role for IRS2 in maintaining renal health. Our study demonstrates that large-scale WGS provides new mechanistic insights into human metabolic phenotypes through improved capture of coding sequences. Analyses of whole-genome sequencing data from UK Biobank and All of Us identify rare variant burden signals associated with metabolic health, including effects of protein-truncating variants in IRS2 on type 2 diabetes and chronic kidney disease risk.},
  archive      = {J_NG},
  author       = {Zhao, Yajie and Lockhart, Sam and Liu, Jimmy and Li, Xihao and Cortes, Adrian and Hua, Xing and Gardner, Eugene J. and Kentistou, Katherine A. and Cañadas-Garre, Marisa and Fabian, Laurie and Ho, Karen and Timpson, Nicholas and Lo, Yancy and Davitte, Jonathan and Savage, David B. and Buser-Doepner, Carolyn and Ong, Ken K. and Zhang, Haoyu and Scott, Robert and O’Rahilly, Stephen and Perry, John R. B.},
  doi          = {10.1038/s41588-025-02364-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2436-2444},
  shortjournal = {Nat. Genet.},
  title        = {Population-scale gene-based analysis of whole-genome sequencing provides insights into metabolic health},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome-wide association meta-analysis of childhood ADHD symptoms and diagnosis identifies new loci and potential effector genes. <em>NG</em>, <em>57</em>(10), 2427-2435. (<a href='https://doi.org/10.1038/s41588-025-02295-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We performed a genome-wide association meta-analysis (GWAMA) of 290,134 attention-deficit/hyperactivity disorder (ADHD) symptom measures of 70,953 unique individuals from multiple raters, ages and instruments (ADHDSYMP). Next, we meta-analyzed the results with a study of ADHD diagnosis (ADHDOVERALL). ADHDSYMP returned no genome-wide significant variants. We show that the combined ADHDOVERALL GWAMA identified 39 independent loci, of which 17 were new. Using a recently developed gene-mapping method, Fine-mapped Locus Assessment Model of Effector genes, we identified 22 potential ADHD effector genes implicating several new biological processes and pathways. Moderate negative genetic correlations (rg &lt; −0.40) were observed with multiple cognitive traits. In three cohorts, polygenic scores (PGSs) based on ADHDOVERALL outperformed PGSs based on ADHD symptoms and diagnosis alone. Our findings support the notion that clinical ADHD is at the extreme end of a continuous liability that is indexed by ADHD symptoms. We show that including ADHD symptom counts helps to identify new genes implicated in ADHD. Genome-wide association meta-analyses of attention-deficit/hyperactivity disorder symptom measures and clinical diagnoses identify new risk loci, highlight putative effector genes and yield improved polygenic risk scores.},
  archive      = {J_NG},
  author       = {van der Laan, Camiel M. and Ip, Hill F. and Schipper, Marijn and Hottenga, Jouke-Jan and St Pourcain, Beate and Zayats, Tetyana and Pool, René and Krapohl, Eva M. L. and Brikell, Isabell and Soler Artigas, María and Cabana-Domínguez, Judit and Llonga, Natalia and Nolte, Ilja M. and Bolhuis, Koen and Palviainen, Teemu and Zafarmand, Hadi and Gordon, Scott and Aliev, Fazil and Burt, S. Alexandra and Wang, Carol A. and Saunders, Gretchen and Karhunen, Ville and Adkins, Daniel E. and Border, Richard and Peterson, Roseann E. and Prinz, Joseph A. and Thiering, Elisabeth and Vilor-Tejedor, Natàlia and Ahluwalia, Tarunveer S. and Allegrini, Andrea and Rimfeld, Kaili and Chen, Qi and Lu, Yi and Martin, Joanna and Bosch, Rosa and Ramos-Quiroga, Josep Antoni and Neumann, Alexander and Ensink, Judith and Grasby, Katrina L. and Morosoli, José J. and Tong, Xiaoran and Marrington, Shelby and Scott, James G. and Shabalin, Andrey A. and Corley, Robin and Evans, Luke M. and Sugden, Karen and Alemany, Silvia and Sass, Lærke and Vinding, Rebecca and Ehli, Erik A. and Hagenbeek, Fiona A. and Derks, Eske M. and Larsson, Henrik and Snieder, Harold and Cecil, Charlotte and Whipp, Alyce M. and Korhonen, Tellervo and Vuoksimaa, Eero and Rose, Richard J. and Uitterlinden, André G. and Haavik, Jan and Harris, Jennifer R. and Helgeland, Øyvind and Johansson, Stefan and Knudsen, Gun Peggy S. and Njolstad, Pal Rasmus and Lu, Qing and Rodriguez, Alina and Henders, Anjali K. and Mamun, Abdullah and Najman, Jackob M. and Brown, Sandy and Hopfer, Christian and Krauter, Kenneth and Reynolds, Chandra A. and Smolen, Andrew and Stallings, Michael and Wadsworth, Sally and Wall, Tamara L. and Eaves, Lindon and Silberg, Judy L. and Miller, Allison and Havdahl, Alexandra and Llop, Sabrina and Lopez-Espinosa, Maria-Jose and Bønnelykke, Klaus and Sunyer, Jordi and Arseneault, Louise and Standl, Marie and Heinrich, Joachim and Boden, Joseph and Pearson, John and Horwood, John and Kennedy, Martin and Poulton, Richie and Maes, Hermine H. and Hewitt, John and Copeland, William E. and Middeldorp, Christel M. and Williams, Gail M. and Wray, Naomi and Järvelin, Marjo-Riitta and McGue, Matt and Iacono, William and Caspi, Avshalom and Moffitt, Terrie E. and Whitehouse, Andrew J. O. and Pennell, Craig E. and Klump, Kelly L. and Jiang, Chang and Dick, Danielle M. and Reichborn-Kjennerud, Ted and Martin, Nicholas G. and Medland, Sarah E. and Vrijkotte, Tanja and Kaprio, Jaakko and Tiemeier, Henning and Davey Smith, George and Hartman, Catharina A. and Oldehinkel, Albertine J. and Casas, Miquel and Ribasés, Marta and Lichtenstein, Paul and Lundström, Sebastian and Plomin, Robert and Bartels, Meike and Nivard, Michel G. and Boomsma, Dorret I.},
  doi          = {10.1038/s41588-025-02295-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2427-2435},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association meta-analysis of childhood ADHD symptoms and diagnosis identifies new loci and potential effector genes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Limited overlap between genetic effects on disease susceptibility and disease survival. <em>NG</em>, <em>57</em>(10), 2418-2426. (<a href='https://doi.org/10.1038/s41588-025-02342-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding disease progression is of high biological and clinical interest. Unlike disease susceptibility, whose genetic basis has been abundantly studied, less is known about the genetics of disease progression and its overlap with disease susceptibility. Considering nine common diseases (ncases ranging from 11,980 to 124,682) across seven biobanks, we systematically compared genetic architectures of susceptibility and progression, defined as disease-specific mortality. We identified only one locus substantially associated with disease-specific mortality and showed that, at a similar sample size, more genome-wide significant loci can be identified in a genome-wide association study of disease susceptibility. Variants substantially affecting disease susceptibility were weakly or not associated with disease-specific mortality. Moreover, susceptibility polygenic scores (PGSs) were weak predictors of disease-specific mortality, while a PGS for general lifespan was substantially associated with disease-specific mortality for seven of nine diseases. We explored alternative definitions of disease progression and found that genetic signals for macrovascular complications in type 2 diabetes overlap with similar phenotypes in the general population; however, these effects are attenuated. Overall, our findings indicate limited similarity in genetic effects between disease susceptibility and disease-specific mortality, suggesting that larger sample sizes, different measures of progression, or the integration of related phenotypes from the general population is needed to identify the genetic underpinnings of disease progression. Systematic comparison of genome-wide association results for disease risk and disease-specific mortality for nine common diseases across seven biobanks finds limited overlap between genetic effects on disease susceptibility and survival.},
  archive      = {J_NG},
  author       = {Yang, Zhiyu and Pajuste, Fanny-Dhelia and Zguro, Kristina and Cheng, Yipeng and Kurant, Danielle E. and Eoli, Andrea and Wanner, Julian and Jermy, Bradley and Rämö, Joel and Kanoni, Stavroula and van Heel, David A. and Hayward, Caroline and Marioni, Riccardo E. and McCartney, Daniel L. and Renieri, Alessandra and Furini, Simone and Mägi, Reedik and Gusev, Alexander and Drineas, Petros and Paschou, Peristera and Heyne, Henrike and Ripatti, Samuli and Mars, Nina and Ganna, Andrea},
  doi          = {10.1038/s41588-025-02342-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2418-2426},
  shortjournal = {Nat. Genet.},
  title        = {Limited overlap between genetic effects on disease susceptibility and disease survival},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pan-UK biobank genome-wide association analyses enhance discovery and resolution of ancestry-enriched effects. <em>NG</em>, <em>57</em>(10), 2408-2417. (<a href='https://doi.org/10.1038/s41588-025-02335-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large biobanks, such as the UK Biobank (UKB), enable massive phenome by genome-wide association studies that elucidate genetic etiology of complex traits. However, people from diverse genetic ancestry groups are often excluded from association analyses due to concerns about population structure introducing false positive associations. Here we generate mixed model associations and meta-analyses across genetic ancestry groups, inclusive of a larger fraction of the UK Biobank than previous efforts, to produce freely available summary statistics for 7,266 traits. We build a quality control and analysis framework informed by genetic architecture. Overall, we identify 14,676 significant loci (P &lt; 5 × 10−8) in the meta-analysis that were not found in the EUR genetic ancestry group alone, including new associations, for example between CAMK2D and triglycerides. We also highlight associations from ancestry-enriched variation, including a known pleiotropic missense variant in G6PD associated with several biomarker traits. We release these results publicly alongside frequently asked questions that describe caveats for interpretation of results, enhancing available resources for interpretation of risk variants across diverse populations. Genome-wide analyses for 7,266 traits leveraging data from several genetic ancestry groups in UK Biobank identify new associations and enhance resources for interpreting risk variants across diverse populations.},
  archive      = {J_NG},
  author       = {Karczewski, Konrad J. and Gupta, Rahul and Kanai, Masahiro and Lu, Wenhan and Tsuo, Kristin and Wang, Ying and Walters, Raymond K. and Turley, Patrick and Callier, Shawneequa and Shah, Nirav N. and Baya, Nikolas and Palmer, Duncan S. and Goldstein, Jacqueline I. and Sarma, Gopal and Solomonson, Matthew and Cheng, Nathan and Bryant, Sam and Churchhouse, Claire and Cusick, Caroline M. and Poterba, Timothy and Compitello, John and King, Daniel and Zhou, Wei and Seed, Cotton and Finucane, Hilary K. and Daly, Mark J. and Neale, Benjamin M. and Atkinson, Elizabeth G. and Martin, Alicia R.},
  doi          = {10.1038/s41588-025-02335-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2408-2417},
  shortjournal = {Nat. Genet.},
  title        = {Pan-UK biobank genome-wide association analyses enhance discovery and resolution of ancestry-enriched effects},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Non-CG DNA methylation in animal genomes. <em>NG</em>, <em>57</em>(10), 2395-2407. (<a href='https://doi.org/10.1038/s41588-025-02303-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cytosine DNA methylation is widespread in animal genomes and occurs predominantly at CG dinucleotides (mCG). While the roles of mCG, such as in genomic imprinting and genome stability, are well established, non-CG DNA methylation (mCH) remains poorly understood. In most vertebrate tissues, roughly 80% of CGs are methylated, whereas mCH levels are generally low, typically ranging from 1% to 3%. In vertebrates, mCH is most prevalent in neural tissue, oocytes and embryonic stem cells and has been linked to neurodevelopmental disorders. Moreover, mCH appears to have a conserved role in regulating vertebrate neural genomes, and recent studies suggest that it has functions in the embryogenesis of teleost fish. Overall, mCH represents an intriguing emerging aspect of gene regulation with potential implications for cellular identity, repeat silencing and neural function. In this Review, we provide a critical overview of the patterning, mechanisms and functional implications of mCH in animals. This Review discusses noncanonical DNA methylation (mCH) in animal genomes and highlights the remaining need to clarify whether mCH represents a conserved regulatory layer or a lineage-specific epigenetic feature with distinct biological roles.},
  archive      = {J_NG},
  author       = {Brethouwer, Thirsa and de Mendoza, Alex and Bogdanovic, Ozren},
  doi          = {10.1038/s41588-025-02303-1},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2395-2407},
  shortjournal = {Nat. Genet.},
  title        = {Non-CG DNA methylation in animal genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing biological understanding of cellular senescence with computational multiomics. <em>NG</em>, <em>57</em>(10), 2381-2394. (<a href='https://doi.org/10.1038/s41588-025-02314-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular senescence is a complex biological process that plays a pathophysiological role in aging and age-related diseases. The biological understanding of senescence at the cellular and tissue levels remains incomplete due to the lack of specific biomarkers as well as the relative rarity of senescent cells, their phenotypic heterogeneity and dynamic features. This Review provides a comprehensive overview of multiomic approaches for the characterization and biological understanding of cellular senescence. The technical capability and challenges of each approach are discussed, and practical guidelines are provided for selecting tools for identifying, characterizing and spatially mapping senescent cells. The importance of computational analyses in multiomics research, including senescent cell identification, signature detection and interactions of senescent cells with microenvironments, is highlighted. Moreover, tissue-specific case studies and experimental design considerations for individual organs are presented. Finally, future directions and the potential impact of multiomic approaches on the biological understanding of cellular senescence are discussed. This Review discusses multiomic approaches for the characterization and biological understanding of cellular senescence, including detailed case studies on skeletal muscle and adipose tissue that highlight current outstanding issues in the field.},
  archive      = {J_NG},
  author       = {Li, Sheng and Agudelo Garcia, Paula A. and Aliferis, Constantin and Becich, Michael J. and Calyeca, Jazmin and Cosgrove, Benjamin D. and Elisseeff, Jennifer and Farzad, Negin and Fertig, Elana J. and Glass, Carolyn and Gu, Liangcai and Hu, Qianjiang and Ji, Zhicheng and Königshoff, Melanie and LeBrasseur, Nathan K. and Li, Dongmei and Ma, Anjun and Ma, Qin and Menon, Vilas and Mitchell, Jacob T. and Mora, Ana L. and Nagaraj, Sushma and Nelson, Andrew C. and Niedernhofer, Laura J. and Rojas, Mauricio and Taha, Hash Brown and Wang, Jinhua and Wang, Siyuan and Wu, Pei-Hsun and Xie, Jichun and Xu, Ming and Yu, Miao and Zhang, Xu and Zhao, Yue and Adams, Peter D. and Aguayo-Mazzucato, Cristina and Baker, Darren J. and Benz, Christopher and Bernlohr, David A. and Bueno, Marta and Chen, Jin and Childs, Bennett G. and Chuang, Jeffrey H. and Chung, Dongjun and Dileepan, Mythili and Ding, Li and Dong, Mingze and Duncan, Francesca and Enninful, Archibald and Flynn, William F. and Franco, Ana Catarina and Furman, David and Garovic, Vesna and Halene, Stephanie and Herman, Allison B. and Hertzel, Ann V. and Iwasaki, Kanako and Jeon, Hyeongseon and Kang, Jeon Woong and Karmakar, Shilpita and Kirkland, James L. and Korstanje, Ron and Kummerfeld, Erich and Lee, Jun Hee and Liu, Yang and Lu, Yao and Lugo-Martinez, Jose and Martini, Helene and Melov, Simon and Musi, Nicolas and Passos, João F. and Peters, Samuel T. and Rahman, Irfan and Ramasamy, Ramalakshmi and Rindone, Alexandra N. and Robbins, Paul D. and Robson, Paul and Rodriguez-Lopez, Jhonny and Rosas, Lorena and Rosenthal, Nadia and Schafer, Marissa J. and Schilling, Birgit and Schmidt, Elizabeth L. and Schneider, Kevin and Sengupta, Kaustav and Shu, Jian and So, Peter T. C. and Sun, Li and Tchkonia, Tamar and Teneche, Marcos G. and Vanegas, Natalia and Wang, Julia and Xie, Juan and Yin, Shanshan and Zhang, Ke and Zhu, Quan and Fan, Rong},
  doi          = {10.1038/s41588-025-02314-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2381-2394},
  shortjournal = {Nat. Genet.},
  title        = {Advancing biological understanding of cellular senescence with computational multiomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A standardized framework for circulating blood proteomics. <em>NG</em>, <em>57</em>(10), 2371-2380. (<a href='https://doi.org/10.1038/s41588-025-02319-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The circulating blood proteome holds immense potential for biomarker discovery and understanding disease mechanisms. Notable advances in mass spectrometry and affinity-based technologies have been made, but data integration across studies and platforms is hindered by the absence of unified analytical standards. This limitation impedes comprehensive exploration of human biology across diverse phenotypes and cohorts as well as the translation of findings into clinical applications. The disparities between datasets, stemming from a combination of factors related to differences in sample collection, pre-analytical handling, measurement methods and instrumentation, further complicate data integration. In this Perspective, we outline key challenges in blood-based proteomics and propose actionable strategies. Central to our recommendations are high-quality, technology-agnostic reference samples, which can bridge disparate datasets and enable robust cross-study comparisons. By fostering interconnected investigations across proteomic technologies, blood sample collections, clinical phenotypes and different populations, these references will accelerate the field and its translation. The field of blood proteomics faces an upsurge of data with the challenge of cross-study comparisons. This Perspective offers an in-depth analysis and proposes reference materials to enhance data integration and accelerate clinical translation.},
  archive      = {J_NG},
  author       = {Cai, Xue and Geyer, Philipp E. and Perez-Riverol, Yasset and Omenn, Gilbert S. and Dong, Lianhua and Winkler, Robert and Ahadi, Sara and Lössl, Philip and Yu, Xiaobo and Chang, Cheng and Ralser, Markus and Jimenez, Connie R. and Zhao, Yang and Chen, Yu-Ju and Poon, Terence C. W. and Bache, Nicolai and Shi, Leming and Dai, Xinhua and Wang, Ziyue and Zhu, Yi and Fang, Xiang and Schwenk, Jochen M. and Van Eyk, Jennifer E. and Völker, Uwe and Guo, Tiannan},
  doi          = {10.1038/s41588-025-02319-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2371-2380},
  shortjournal = {Nat. Genet.},
  title        = {A standardized framework for circulating blood proteomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The solve-RD solvathons as a pan-european interdisciplinary collaboration to diagnose patients with rare disease. <em>NG</em>, <em>57</em>(10), 2361-2370. (<a href='https://doi.org/10.1038/s41588-025-02290-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite advances in genomic diagnostics, the majority of individuals with rare diseases remain without a confirmed genetic diagnosis. The rapid emergence of advanced omics technologies, such as long-read genome sequencing, optical genome mapping and multiomic profiling, has improved diagnostic yield but also substantially increased analytical and interpretational complexity. Addressing this complexity requires systematic multidisciplinary collaboration, as recently demonstrated by targeted diagnostic workshops. Here, we highlight the experience of the Solve-RD consortium, a pan-European initiative, in implementing four structured workshops, termed ‘Solvathons’, as a regular and effective component of its operational workflow. We provide actionable insights, best practices and lessons learned for successful data integration, expert training and scalable collaborative diagnostics within large research consortia. This Perspective presents the Solve-RD Solvathon model, an innovative, pan-European framework uniting clinical and bioinformatics experts to diagnose rare diseases through integrative multiomics analysis and structured collaboration.},
  archive      = {J_NG},
  author       = {Yépez, Vicente A. and Demidov, German and Ellwanger, Kornelia and Laurie, Steven and Luknárová, Rebeka and Joseph Maran, Midhuna Immaculate and Hentrich, Thomas and Sagath, Lydia and van der Sanden, Bart and Astuti, Galuh and Neveling, Kornelia and Batlle-Masó, Laura and Beijer, Danique and Brechtmann, Felix and Caballero-Oteyza, Andrés and Dabad, Marc and Denommé-Pichon, Anne-Sophie and Doornbos, Cenna and Eddafir, Zakaria and Estévez-Arias, Berta and Kilicarslan, Ozge Aksel and Kolen, Ingrid H. M. and Kraß, Leon and Lohmann, Katja and Londhe, Shubhankar and López-Martín, Estrella and Maassen, Kars and Macken, William and Martínez-Delgado, Beatriz and Mei, Davide and Mertes, Christian and Minardi, Raffaella and Morsy, Heba and Mueller, Juliane S. and Natera-de Benito, Daniel and Nelson, Isabelle and Oud, Machteld M. and Paramonov, Ida and Picó, Daniel and Piscia, Davide and Polavarapu, Kiran and Raineri, Emanuele and Savarese, Marco and Smal, Noor and Steehouwer, Marloes and Steyaert, Wouter and Swertz, Morris A. and Thomsen, Mirja and Töpf, Ana and Van de Vondel, Liedewei and van der Vries, Gerben and Vitobello, Antonio and Wilke, Carlo and Zurek, Birte and t’ Hoen, Peter-Bram and Matalonga, Leslie and Vissers, Lisenka E. L. M. and Gilissen, Christian and Schulze-Hentrich, Julia and Beltran, Sergi and Esteve-Codina, Anna and Hoischen, Alexander and Gagneur, Julien and Graessner, Holm},
  doi          = {10.1038/s41588-025-02290-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2361-2370},
  shortjournal = {Nat. Genet.},
  title        = {The solve-RD solvathons as a pan-european interdisciplinary collaboration to diagnose patients with rare disease},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic perturbations link autoimmune risk variants to t cell function. <em>NG</em>, <em>57</em>(10), 2359-2360. (<a href='https://doi.org/10.1038/s41588-025-02302-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We used a massively parallel reporter assay (MPRA) to test thousands of autoimmune disease-associated genetic variants for allele-specific effects on gene expression in primary human T cells. Variants that altered gene expression within our MPRA were then linked to T cell regulatory networks and proliferation using bulk and single-cell CRISPR-interference screens.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02302-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2359-2360},
  shortjournal = {Nat. Genet.},
  title        = {Genetic perturbations link autoimmune risk variants to t cell function},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cracking the histone code for prostate cancer therapy. <em>NG</em>, <em>57</em>(10), 2357-2358. (<a href='https://doi.org/10.1038/s41588-025-02354-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this study, we uncover the critical role of p300/CBP-mediated histone H2B N terminus multisite lysine acetylation (H2BNTac) in defining oncogenic enhanceosomes in prostate cancer. Degradation of p300/CBP rapidly disables H2BNTac-marked oncogenic enhancers and represents a promising therapeutic strategy for enhancer-driven malignancies, including prostate cancer.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02354-4},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2357-2358},
  shortjournal = {Nat. Genet.},
  title        = {Cracking the histone code for prostate cancer therapy},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The dawn of bryophyte genomics is here. <em>NG</em>, <em>57</em>(10), 2355-2356. (<a href='https://doi.org/10.1038/s41588-025-02240-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bryophytes are a widespread group of land plants that occupy nearly all biomes, yet their genetics and evolutionary history have long remained underexplored. Now, a study that generates extensive genomic data for bryophytes highlights de novo gene formation and horizontal gene transfer as key forces that shape bryophyte diversity and adaptation.},
  archive      = {J_NG},
  author       = {Potente, Giacomo and Yue, Yuling and Szövényi, Péter},
  doi          = {10.1038/s41588-025-02240-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2355-2356},
  shortjournal = {Nat. Genet.},
  title        = {The dawn of bryophyte genomics is here},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dipping into the function of DNA methylation at centromeres. <em>NG</em>, <em>57</em>(10), 2352-2354. (<a href='https://doi.org/10.1038/s41588-025-02334-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human centromeres contain a small chromatin region with low levels of DNA cytosine methylation that resides with CENP-A. Salinas-Luypaert et al. find a role of DNA methylation in maintaining the size and function of centromeres by controlling the binding affinity of key centromere components.},
  archive      = {J_NG},
  author       = {Erhardt, Sylvia},
  doi          = {10.1038/s41588-025-02334-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2352-2354},
  shortjournal = {Nat. Genet.},
  title        = {Dipping into the function of DNA methylation at centromeres},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A high-power spatial, single-cell view of pericytes in esophageal cancer metastasis. <em>NG</em>, <em>57</em>(10), 2350-2351. (<a href='https://doi.org/10.1038/s41588-025-02318-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using integrated single-cell multi-omics and spatial analysis, this study identifies a PRRX1-driven GPR116⁺ pericyte subpopulation that promotes tumor metastasis and immune evasion in esophageal squamous cell carcinoma, offering a diagnostic biomarker and potential therapeutic strategies.},
  archive      = {J_NG},
  author       = {Sinha, Uttam K. and Park, Young Min and Lin, De-Chen},
  doi          = {10.1038/s41588-025-02318-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2350-2351},
  shortjournal = {Nat. Genet.},
  title        = {A high-power spatial, single-cell view of pericytes in esophageal cancer metastasis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A plasma proteomic signature for ALS. <em>NG</em>, <em>57</em>(10), 2349. (<a href='https://doi.org/10.1038/s41588-025-02384-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-025-02384-y},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2349},
  shortjournal = {Nat. Genet.},
  title        = {A plasma proteomic signature for ALS},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Recycling enhancers. <em>NG</em>, <em>57</em>(10), 2349. (<a href='https://doi.org/10.1038/s41588-025-02385-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02385-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2349},
  shortjournal = {Nat. Genet.},
  title        = {Recycling enhancers},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Detecting parent-of-origin genetic effects without parental genomes. <em>NG</em>, <em>57</em>(10), 2349. (<a href='https://doi.org/10.1038/s41588-025-02386-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Brandt, Margot},
  doi          = {10.1038/s41588-025-02386-w},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2349},
  shortjournal = {Nat. Genet.},
  title        = {Detecting parent-of-origin genetic effects without parental genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Basal cells drive small cell lung cancer plasticity. <em>NG</em>, <em>57</em>(10), 2349. (<a href='https://doi.org/10.1038/s41588-025-02387-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02387-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2349},
  shortjournal = {Nat. Genet.},
  title        = {Basal cells drive small cell lung cancer plasticity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Realizing the full potential of our future health through data linkage and trans-biobank efforts. <em>NG</em>, <em>57</em>(10), 2341-2348. (<a href='https://doi.org/10.1038/s41588-025-02327-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The scale and population coverage of Our Future Health, alongside other next-generation biobanks, offers unique opportunities to advance genomic medicine. Focusing on the UK context, we provide a researcher’s perspective of how this new resource could reach its full potential in a way that is impactful, user-friendly and informs related global efforts.},
  archive      = {J_NG},
  author       = {Straub, Vincent J. and Benonisdottir, Stefania and Kong, Augustine and Mills, Melinda C.},
  doi          = {10.1038/s41588-025-02327-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {2341-2348},
  shortjournal = {Nat. Genet.},
  title        = {Realizing the full potential of our future health through data linkage and trans-biobank efforts},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Alterations of the CIB2 calcium- and integrin-binding protein cause usher syndrome type 1J and nonsyndromic deafness DFNB48. <em>NG</em>, <em>57</em>(9), 2340-2342. (<a href='https://doi.org/10.1038/s41588-025-02332-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Riazuddin, Saima and Belyantseva, Inna A. and Giese, Arnaud P. J. and Lee, Kwanghyuk and Indzhykulian, Artur A. and Nandamuri, Sri Pratima and Yousaf, Rizwan and Sinha, Ghanshyam P. and Lee, Sue and Terrell, David and Hegde, Rashmi S. and Ali, Rana A. and Anwar, Saima and Andrade-Elizondo, Paula B. and Sirmaci, Asli and Parise, Leslie V. and Basit, Sulman and Wali, Abdul and Ayub, Muhammad and Ansar, Muhammad and Ahmad, Wasim and Khan, Shaheen N. and Akram, Javed and Tekin, Mustafa and Riazuddin, Sheikh and Cook, Tiffany and Buschbeck, Elke K. and Frolenkov, Gregory I. and Leal, Suzanne M. and Friedman, Thomas B. and Ahmed, Zubair M.},
  doi          = {10.1038/s41588-025-02332-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2340-2342},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Alterations of the CIB2 calcium- and integrin-binding protein cause usher syndrome type 1J and nonsyndromic deafness DFNB48},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Clonal evolution of hematopoietic stem cells after autologous stem cell transplantation. <em>NG</em>, <em>57</em>(9), 2339. (<a href='https://doi.org/10.1038/s41588-025-02328-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Uryu, Hidetaka and Saeki, Koichi and Haeno, Hiroshi and Kapadia, Chiraag Deepak and Furudate, Ken and Nangalia, Jyoti and Spencer Chapman, Michael and Zhang, Linda and Padilla, Jennifer and Zhao, Li and Hsu, Joanne I. and Zhao, Chong and Chen, Shujuan and Tanaka, Tomoyuki and Li, Zongrui and Ogata, Satoko and Hanache, Sarah and Yang, Hui and DiNardo, Courtney and Daver, Naval and Pemmaraju, Naveen and Jain, Nitin and Ravandi, Farhad and Zhang, Jianhua and Song, Xingzhi and Thompson, Erika and Tang, Hongli and Little, Latasha and Gumbs, Curtis and Orlowski, Robert Z. and Qazilbash, Muzaffar and Bhalla, Kapil and Colla, Simona and Kantarjian, Hagop and Kanagal-Shamanna, Rashmi and Bueso-Ramos, Carlos and Nakada, Daisuke and Al-Atrash, Gheath and Molldrem, Jeffery and Futreal, P. Andrew and Shpall, Elizabeth and Goodell, Margaret and Garcia-Manero, Guillermo and Takahashi, Koichi},
  doi          = {10.1038/s41588-025-02328-6},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2339},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Clonal evolution of hematopoietic stem cells after autologous stem cell transplantation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Clonal evolution during metastatic spread in high-risk neuroblastoma. <em>NG</em>, <em>57</em>(9), 2338. (<a href='https://doi.org/10.1038/s41588-025-02320-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gundem, Gunes and Levine, Max F. and Roberts, Stephen S. and Cheung, Irene Y. and Medina-Martínez, Juan S. and Feng, Yi and Arango-Ossa, Juan E. and Chadoutaud, Loic and Rita, Mathieu and Asimomitis, Georgios and Zhou, Joe and You, Daoqi and Bouvier, Nancy and Spitzer, Barbara and Solit, David B. and Dela Cruz, Filemon and LaQuaglia, Michael P. and Kushner, Brian H. and Modak, Shakeel and Shukla, Neerav and Iacobuzio-Donahue, Christine A. and Kung, Andrew L. and Cheung, Nai-Kong V. and Papaemmanuil, Elli},
  doi          = {10.1038/s41588-025-02320-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2338},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Clonal evolution during metastatic spread in high-risk neuroblastoma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states. <em>NG</em>, <em>57</em>(9), 2323-2334. (<a href='https://doi.org/10.1038/s41588-025-02284-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell atlas projects have revealed that common cell types often comprise distinct, recurrent transcriptional states, but the function and regulation of these states remain poorly understood. Here, we show that systematic activation of transcription factors can recreate such states in vitro, providing tractable models for mechanistic and functional studies. Using a scalable CRISPR activation (CRISPRa) Perturb-seq platform, we activated 1,836 transcription factors in two cell types. CRISPRa induced gene expression within physiological ranges, with chromatin features predicting responsiveness. Comparisons with atlas datasets showed that transcription factor perturbations recapitulated key fibroblast states and identified their regulators, including KLF2 and KLF4 for a universal state present in many tissues, and PLAGL1 for a disease-associated inflammatory state. Inducing the universal state suppressed the inflammatory state, suggesting therapeutic potential. These findings position CRISPRa as a nuanced tool for perturbing differentiated cells and establish a general strategy for studying clinically relevant transcriptional states ex vivo. CRISPR activation of 1,836 human transcription factors recapitulates fibroblast transcriptional states observed in vivo and identifies regulators that can revert inflammatory states.},
  archive      = {J_NG},
  author       = {Southard, Kaden M. and Ardy, Rico C. and Tang, Anran and O’Sullivan, Deirdre D. and Metzner, Eli and Guruvayurappan, Karthik and Norman, Thomas M.},
  doi          = {10.1038/s41588-025-02284-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2323-2334},
  shortjournal = {Nat. Genet.},
  title        = {Comprehensive transcription factor perturbations recapitulate fibroblast transcriptional states},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize. <em>NG</em>, <em>57</em>(9), 2313-2322. (<a href='https://doi.org/10.1038/s41588-025-02246-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comprehensive maps of functional variation at transcription factor (TF) binding sites (cis-elements) are crucial for elucidating how genotype shapes phenotype. Here, we report the construction of a pan-cistrome of the maize leaf under well-watered and drought conditions. We quantified haplotype-specific TF footprints across a pan-genome of 25 maize hybrids and mapped over 200,000 variants, genetic, epigenetic, or both (termed binding quantitative trait loci (bQTL)), linked to cis-element occupancy. Three lines of evidence support the functional significance of bQTL: (1) coincidence with causative loci that regulate traits, including vgt1, ZmTRE1 and the MITE transposon near ZmNAC111 under drought; (2) bQTL allelic bias is shared between inbred parents and matches chromatin immunoprecipitation sequencing results; and (3) partitioning genetic variation across genomic regions demonstrates that bQTL capture the majority of heritable trait variation across ~72% of 143 phenotypes. Our study provides an auspicious approach to make functional cis-variation accessible at scale for genetic studies and targeted engineering of complex traits. Pan-cistrome of the maize leaf under well-watered and drought conditions profiled by haplotype-specific MOA-seq highlights the relevance of transcription factor binding QTLs for understanding phenotypic diversity in maize.},
  archive      = {J_NG},
  author       = {Engelhorn, Julia and Snodgrass, Samantha J. and Kok, Amelie and Seetharam, Arun S. and Schneider, Michael and Kiwit, Tatjana and Singh, Ayush and Banf, Michael and Doan, Duong Thi Hai and Khaipho-Burch, Merritt and Runcie, Daniel E. and Sánchez-Camargo, Victor A. and Bader, Rechien and Vladimir Torres-Rodriguez, J. and Sun, Guangchao and Stam, Maike and Fiorani, Fabio and Beier, Sebastian and Schnable, James C. and Bass, Hank W. and Hufford, Matthew B. and Stich, Benjamin and Frommer, Wolf B. and Ross-Ibarra, Jeffrey and Hartwig, Thomas},
  doi          = {10.1038/s41588-025-02246-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2313-2322},
  shortjournal = {Nat. Genet.},
  title        = {Genetic variation at transcription factor binding sites largely explains phenotypic heritability in maize},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Inactivation of β-1,3-glucan synthase-like 5 confers broad-spectrum resistance to plasmodiophora brassicae pathotypes in cruciferous plants. <em>NG</em>, <em>57</em>(9), 2302-2312. (<a href='https://doi.org/10.1038/s41588-025-02306-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clubroot disease, caused by the obligate intracellular rhizarian protist Plasmodiophora brassicae, is devastating to cruciferous crops worldwide. Widespread field P. brassicae pathotypes frequently overcome the pathotype-specific resistance of modern varieties, posing a challenge for durable control of this disease. Here a genome-wide association study of 3 years of data comprising field clubroot phenotyping of 244 genome-resequenced Brassica napus accessions identified a strong association of β-1,3-glucan synthase-like 5 (GSL5) with clubroot susceptibility. GSL5 was evolutionarily conserved, and inactivation of GSL5 by genome editing in Arabidopsis, B. napus, Brassica rapa and Brassica oleracea conferred broad-spectrum, high-level resistance to P. brassicae pathotypes without yield penalties in B. napus. GSL5 inactivation derepressed the jasmonic acid-mediated immunity during P. brassicae secondary infection, and this immune repression was possibly reinforced through stabilization of GSL5 by a P. brassicae effector, facilitating clubroot susceptibility. Our study provides durable resistance resources for cruciferous clubroot disease control and insights into plant resistance against intracellular eukaryotic phytopathogens. This study implicates GSL5 inactivation in high, broad-spectrum resistance to the clubroot pathogen Plasmodiophora brassicae in Arabidopsis thaliana, Brassica napus, Brassica oleracea and Brassica rapa.},
  archive      = {J_NG},
  author       = {Wu, Yupo and Zhao, Chuanji and Zhang, Yi and Shen, Cuicui and Zhang, Yuanyuan and Zhang, Xiong and Gao, Lixia and Zeng, Lingyi and Ke, Qinglin and Qin, Li and Liu, Fan and Huang, Junyan and Ren, Li and Liu, Yueying and Cheng, Hongtao and Tong, Chaobo and Hu, Qiong and Cheng, Xiaohui and Wei, Yangdou and Liu, Shengyi and Liu, Lijiang},
  doi          = {10.1038/s41588-025-02306-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2302-2312},
  shortjournal = {Nat. Genet.},
  title        = {Inactivation of β-1,3-glucan synthase-like 5 confers broad-spectrum resistance to plasmodiophora brassicae pathotypes in cruciferous plants},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A comparison of 27 arabidopsis thaliana genomes and the path toward an unbiased characterization of genetic polymorphism. <em>NG</em>, <em>57</em>(9), 2289-2301. (<a href='https://doi.org/10.1038/s41588-025-02293-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Making sense of whole-genome polymorphism data is challenging, but it is essential for overcoming the biases in SNP data. Here we analyze 27 genomes of Arabidopsis thaliana to illustrate these issues. Genome size variation is mostly due to tandem repeat regions that are difficult to assemble. However, while the rest of the genome varies little in length, it is full of structural variants, mostly due to transposon insertions. Because of this, the pangenome coordinate system grows rapidly with sample size and ultimately becomes 70% larger than the size of any single genome, even for n = 27. Finally, we show how short-read data are biased by read mapping. SNP calling is biased by the choice of reference genome, and both transcriptome and methylome profiling results are affected by mapping reads to a reference genome rather than to the genome of the assayed individual. New concepts for comparing the genomes of 27 naturally inbred Arabidopsis thaliana accessions provide essential insights into obtaining a less biased view of whole-genome polymorphism.},
  archive      = {J_NG},
  author       = {Igolkina, Anna A. and Vorbrugg, Sebastian and Rabanal, Fernando A. and Liu, Hai-Jun and Ashkenazy, Haim and Kornienko, Aleksandra E. and Fitz, Joffrey and Collenberg, Max and Kubica, Christian and Mollá Morales, Almudena and Jaegle, Benjamin and Wrightsman, Travis and Voloshin, Vitaly and Bezlepsky, Alexander D. and Llaca, Victor and Nizhynska, Viktoria and Reichardt, Ilka and Bezrukov, Ilja and Lanz, Christa and Bemm, Felix and Flood, Pádraic J. and Nemomissa, Sileshi and Hancock, Angela and Guo, Ya-Long and Kersey, Paul and Weigel, Detlef and Nordborg, Magnus},
  doi          = {10.1038/s41588-025-02293-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2289-2301},
  shortjournal = {Nat. Genet.},
  title        = {A comparison of 27 arabidopsis thaliana genomes and the path toward an unbiased characterization of genetic polymorphism},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Super-pangenome analyses across 35 accessions of 23 avena species highlight their complex evolutionary history and extensive genomic diversity. <em>NG</em>, <em>57</em>(9), 2276-2288. (<a href='https://doi.org/10.1038/s41588-025-02294-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Common oat, belonging to the genus Avena with 30 recognized species, is a nutritionally important cereal crop and high-quality forage worldwide. Here, we construct a genus-level super-pangenome of Avena comprising 35 high-quality genomes from 14 cultivated oat accessions and 21 wild species. The fully resolved phylogenomic analysis unveils the origin and evolutionary scenario of Avena species, and the super-pangenome analysis identifies 26.62% and 59.93% specific genes and haplotypes in wild species. We delineate the landscape of structural variations (SVs) and the transcriptome profile based 1,401 RNA-sequencing (RNA-seq) samples from diverse abiotic stress treatments in oat. We highlight the crucial role of SVs in modulating gene expression and shaping adaptation to diverse stresses. Further combining SV-based genome-wide association studies (GWASs), we characterize 13 candidate genes associated with drought resistance such as AsARF7, validated by transgenic oat lines. Our study provides unprecedented genomic resources to facilitate genomic, evolution and molecular breeding research in oat. A genus-level super-pangenome of Avena comprising 35 high-quality genomes from 23 cultivated or wild oat species highlights the evolution of Avena and the landscape of structural variations related to abiotic stress resistance.},
  archive      = {J_NG},
  author       = {Zhang, Hongyu and Liu, Ningkun and Wang, Yaru and Zheng, Xinyuan and Li, Wei and Liu, Ze and Liu, Jianan and Wang, Yu and Xing, Longsheng and Li, Tao and Yun, Yange and Zhou, Qinghong and Wang, Meijia and Qin, Yujie and Yan, Jinjiang and Gong, Zhizhong and He, Qiang and Du, Huilong},
  doi          = {10.1038/s41588-025-02294-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2276-2288},
  shortjournal = {Nat. Genet.},
  title        = {Super-pangenome analyses across 35 accessions of 23 avena species highlight their complex evolutionary history and extensive genomic diversity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chromatin dynamics of a large-sized genome provides insights into polyphenism and x0 dosage compensation of locusts. <em>NG</em>, <em>57</em>(9), 2264-2275. (<a href='https://doi.org/10.1038/s41588-025-02330-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Locusts are characterized by a large genome size, polyphenism and an X0 sex determination system. Here we generated chromosome-level genomes for both desert and migratory locusts, as well as a comprehensive chromatin map for the latter. We found that genome enlargement is associated with an increased number of enhancers in expanded intronic regions. To explore the function of expanded enhancers, we identified a distal enhancer that contributes to behavioral differences between solitary and gregarious locusts. In the X0 sex system, H4K16ac enrichment and H4K20me1 depletion maintain balanced X-linked expression in male soma. Notably, the distance-dependent H4K16ac enrichment diminishes gradually in intergenic regions, revealing a special dosage compensation mechanism in large genomes. Furthermore, the distance-dependent H4K16ac results in a lag in the evolution of dosage compensation for the X-linked genes translocated recently from autosomes. Therefore, expanded intronic and intergenic regions shape a distinct chromatin regulation landscape in large genomes. Chromosome-level genome assemblies of migratory and desert locusts, coupled with epigenomic profiling of migratory locusts, reveal chromatin dynamics underlying polyphenism and X-linked dosage compensation following autosomal gene translocation.},
  archive      = {J_NG},
  author       = {Liu, Qing and Jiang, Feng and Li, Ran and Liu, Shanlin and Fang, Wenjing and Li, Xiao and Hu, Ting and Feng, Lianyun and Zhang, Jie and Liu, Zhikang and He, Jing and Guo, Wei and Wang, Xianhui and Sun, Zongyi and Li, Jingjing and Gao, Yunan and Yi, Jiacheng and Li, Qiye and Wang, Xiaoxiao and Wei, Liya and Kang, Le},
  doi          = {10.1038/s41588-025-02330-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2264-2275},
  shortjournal = {Nat. Genet.},
  title        = {Chromatin dynamics of a large-sized genome provides insights into polyphenism and x0 dosage compensation of locusts},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precise modulation of BRG1 levels reveals features of mSWI/SNF dosage sensitivity. <em>NG</em>, <em>57</em>(9), 2250-2263. (<a href='https://doi.org/10.1038/s41588-025-02305-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mammalian switch/sucrose nonfermentable (mSWI/SNF) complex regulates chromatin accessibility and frequently shows alterations due to mutation in cancer and neurological diseases. Inadequate expression of mSWI/SNF in heterozygous mice can lead to developmental defects, indicating dosage-sensitive effects of mSWI/SNF. However, how its dosage affects function has remained unclear. Using a targeted protein degradation system, we investigated its dosage-sensitive effects by precisely controlling protein levels of BRG1, the ATPase subunit of the mSWI/SNF complex. We found that binding of BRG1 to chromatin exhibited a linear response to the BRG1 protein level. Although chromatin accessibility at most promoters and insulators was largely unaffected by BRG1 depletion, 44% of enhancers, including 84% of defined superenhancers, showed reduced accessibility. Notably, half of the BRG1-regulated enhancers, particularly superenhancers, exhibited a buffered response to BRG1 loss. Consistently, transcription exhibited a predominantly buffered response to changes in BRG1 levels. Collectively, our findings demonstrate a genomic feature-specific response to BRG1 dosage, shedding light on the dosage-sensitive effects of mSWI/SNF complex defects in cancer and other diseases. This study investigates the dosage-sensitive effects of BRG1, a core SWI/SNF subunit, on chromatin binding, accessibility and transcriptional regulation by precisely controlling its protein level.},
  archive      = {J_NG},
  author       = {Hagihara, Yota and Zhang, Chao and Zhang, Yi},
  doi          = {10.1038/s41588-025-02305-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2250-2263},
  shortjournal = {Nat. Genet.},
  title        = {Precise modulation of BRG1 levels reveals features of mSWI/SNF dosage sensitivity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue. <em>NG</em>, <em>57</em>(9), 2238-2249. (<a href='https://doi.org/10.1038/s41588-025-02300-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cell-type-level epigenomic landscape of human subcutaneous adipose tissue (SAT) is not well characterized. Here, we elucidate the epigenomic landscape across SAT cell types using snm3C-seq. We find that SAT CG methylation (mCG) displays pronounced hypermethylation in myeloid cells and hypomethylation in adipocytes and adipose stem and progenitor cells, driving nearly half of the 705,063 differentially methylated regions (DMRs). Moreover, TET1 and DNMT3A are identified as plausible regulators of the cell-type-level mCG profiles. Both global mCG profiles and chromosomal compartmentalization reflect SAT cell-type lineage. Notably, adipocytes display more short-range chromosomal interactions, forming complex local 3D genomic structures that regulate transcriptional functions, including adipogenesis. Furthermore, adipocyte DMRs and A compartments are enriched for abdominal obesity genome-wide association study (GWAS) variants and polygenic risk, while myeloid A compartments are enriched for inflammation. Together, we characterize the SAT single-cell-level epigenomic landscape and link GWAS variants and partitioned polygenic risk of abdominal obesity and inflammation to the SAT epigenome. This multiomic study, including single-nucleus DNA methylation and chromatin conformation matched with single-nuclei RNA sequencing, provides insights into the epigenomic landscape of human subcutaneous adipose tissue.},
  archive      = {J_NG},
  author       = {Chen, Zeyuan Johnson and Das, Sankha Subhra and Kar, Asha and Lee, Seung Hyuk T. and Abuhanna, Kevin D. and Alvarez, Marcus and Sukhatme, Mihir G. and Wang, Zitian and Gelev, Kyla Z. and Heffel, Matthew G. and Zhang, Yi and Avram, Oren and Rahmani, Elior and Sankararaman, Sriram and Laakso, Markku and Heinonen, Sini and Peltoniemi, Hilkka and Halperin, Eran and Pietiläinen, Kirsi H. and Luo, Chongyuan and Pajukanta, Päivi},
  doi          = {10.1038/s41588-025-02300-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2238-2249},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell DNA methylome and 3D genome atlas of human subcutaneous adipose tissue},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution. <em>NG</em>, <em>57</em>(9), 2226-2237. (<a href='https://doi.org/10.1038/s41588-025-02307-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aberrant DNA methylation has been described in nearly all human cancers, yet its interplay with genomic alterations during tumor evolution is poorly understood. To explore this, we performed reduced representation bisulfite sequencing on 217 tumor and matched normal regions from 59 patients with non-small cell lung cancer from the TRACERx study to deconvolve tumor methylation. We developed two metrics for integrative evolutionary analysis with DNA and RNA sequencing data. Intratumoral methylation distance quantifies intratumor DNA methylation heterogeneity. MR/MN classifies genes based on the rate of hypermethylation at regulatory (MR) versus nonregulatory (MN) CpGs to identify driver genes exhibiting recurrent functional hypermethylation. We identified DNA methylation-linked dosage compensation of essential genes co-amplified with neighboring oncogenes. We propose two complementary mechanisms that converge for copy number alteration-affected chromatin to undergo the epigenetic equivalent of an allosteric activity transition. Hypermethylated driver genes under positive selection may open avenues for therapeutic stratification of patients. Integrated multi-omic analyses using samples from the TRACERx study highlight cross-talk between DNA hypermethylation and genomic lesions in non-small cell lung cancer.},
  archive      = {J_NG},
  author       = {Gimeno-Valiente, Francisco and Castignani, Carla and Larose Cadieux, Elizabeth and Mensah, Nana E. and Liu, Xiaohong and Chen, Kezhong and Chervova, Olga and Karasaki, Takahiro and Weeden, Clare E. and Richard, Corentin and Lai, Siqi and Martínez-Ruiz, Carlos and Lim, Emilia L. and Frankell, Alexander M. and Watkins, Thomas B. K. and Stavrou, Georgia and Usaite, Ieva and Lu, Wei-Ting and Marinelli, Daniele and Saghafinia, Sadegh and Wilson, Gareth A. and Dhami, Pawan and Vaikkinen, Heli and Steif, Jonathan and Veeriah, Selvaraju and Hynds, Robert E. and Hirst, Martin and Hiley, Crispin and Feber, Andrew and Deniz, Özgen and Jamal-Hanjani, Mariam and McGranahan, Nicholas and Beck, Stephan and Demeulemeester, Jonas and Tanić, Miljana and Swanton, Charles and Van Loo, Peter and Kanu, Nnennaya},
  doi          = {10.1038/s41588-025-02307-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2226-2237},
  shortjournal = {Nat. Genet.},
  title        = {DNA methylation cooperates with genomic alterations during non-small cell lung cancer evolution},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis. <em>NG</em>, <em>57</em>(9), 2215-2225. (<a href='https://doi.org/10.1038/s41588-025-02296-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanisms through which mutations in splicing factor genes drive clonal hematopoiesis (CH) and myeloid malignancies, and their close association with advanced age, remain poorly understood. Here we show that telomere maintenance plays an important role in this phenomenon. First, by studying 454,098 UK Biobank participants, we find that, unlike most CH subtypes, splicing-factor-mutant CH is more common in those with shorter genetically predicted telomeres, as is CH with mutations in PPM1D and the TERT gene promoter. We go on to show that telomere attrition becomes an instrument for clonal selection in advanced age, with splicing factor mutations ‘rescuing’ HSCs from critical telomere shortening. Our findings expose the lifelong influence of telomere maintenance on hematopoiesis and identify a potential shared mechanism through which different splicing factor mutations drive leukemogenesis. Understanding the mechanistic basis of these observations can open new therapeutic avenues against splicing-factor-mutant CH and hematological or other cancers. This study explores the relationship between telomere length and clonal hematopoiesis. Splicing factor and PPM1D gene mutations are more frequent in people with genetically predicted shorter telomere lengths, suggesting that these mutations protect against the consequences of telomere attrition.},
  archive      = {J_NG},
  author       = {McLoughlin, Matthew A. and Cheloor Kovilakam, Sruthi and Dunn, William G. and Gu, Muxin and Tobin, Jake and Pershad, Yash and Williams, Nicholas and Leongamornlert, Daniel and Dawson, Kevin and Bond, Laura and Marando, Ludovica and Wen, Sean and Wilson, Rachael and Valenzano, Giampiero and Symeonidou, Vasiliki and Rak, Justyna and Damaskou, Aristi and Gozdecka, Malgorzata and Liu, Xiaoxuan and Barcena, Clea and Nomdedeu, Josep and Costeas, Paul and Dimitriou, Ioannis D. and Fiorillo, Edoardo and Orrù, Valeria and de Almeida, Jose Guilherme and McKerrell, Thomas and Cullen, Matthew and Mohorianu, Irina and Foukaneli, Theodora and Warren, Alan J. and Wong, Chi and Follows, George and Godfrey, Anna L. and Gudgin, Emma and Cucca, Francesco and McKinney, Eoin and Baxter, E. Joanna and Gerstung, Moritz and Mitchell, Jonathan and Wiseman, Daniel and Bick, Alexander G. and Fabre, Margarete and Quiros, Pedro M. and Nangalia, Jyoti and Kar, Siddhartha and Vassiliou, George S.},
  doi          = {10.1038/s41588-025-02296-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2215-2225},
  shortjournal = {Nat. Genet.},
  title        = {Telomere attrition becomes an instrument for clonal selection in aging hematopoiesis and leukemogenesis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Temporal genomic dynamics shape clinical trajectory in multiple myeloma. <em>NG</em>, <em>57</em>(9), 2203-2214. (<a href='https://doi.org/10.1038/s41588-025-02292-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiple myeloma evolution is characterized by the accumulation of genomic drivers over time. To unravel this timeline and its impact on clinical outcomes, we analyzed 421 whole-genome sequences from 382 patients. Using clock-like mutational signatures, we estimated a time lag of two to four decades between the initiation of events and diagnosis. We demonstrate that odd-numbered chromosome trisomies in patients with hyperdiploidy can be acquired simultaneously with other chromosomal gains (for example, 1q gain). We show that hyperdiploidy is acquired after immunoglobulin heavy chain translocation when both events co-occur. Finally, patients with early 1q gain had adverse outcomes similar to those with 1q amplification (&gt;1 extra copy), but fared worse than those with late 1q gain. This finding underscores that the 1q gain prognostic impact depends more on the timing of acquisition than on the number of copies gained. Overall, this study contributes to a better understanding of the life history of myeloma and may have prognostic implications. This analysis of whole-genome sequencing data from 421 multiple myeloma samples elucidates the timing of key genomic events and shows associations between the timing of 1q gain and clinical outcome.},
  archive      = {J_NG},
  author       = {Maura, Francesco and Kaddoura, Marcella and Poos, Alexandra M. and Baughn, Linda B. and Ziccheddu, Bachisio and Bärtsch, Marc-Andrea and Cirrincione, Anthony and Maclachlan, Kylee and Chojnacka, Monika and Diamond, Benjamin and Papadimitriou, Marios and Blaney, Patrick and John, Lukas and Reichert, Philipp and Huhn, Stefanie and Gagler, Dylan and Zhang, Yanming and Dogan, Ahmet and Lesokhin, Alexander M. and Davies, Faith and Goldschmidt, Hartmut and Fenk, Roland and Weisel, Katja C. and Mai, Elias K. and Korde, Neha and Morgan, Gareth J. and Rajkumar, S. Vincent and Kumar, Shaji and Usmani, Saad and Landgren, Ola and Raab, Marc S. and Weinhold, Niels},
  doi          = {10.1038/s41588-025-02292-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2203-2214},
  shortjournal = {Nat. Genet.},
  title        = {Temporal genomic dynamics shape clinical trajectory in multiple myeloma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tamoxifen induces PI3K activation in uterine cancer. <em>NG</em>, <em>57</em>(9), 2192-2202. (<a href='https://doi.org/10.1038/s41588-025-02308-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutagenic processes and clonal selection contribute to the development of therapy-associated secondary neoplasms, a known complication of cancer treatment. The association between tamoxifen therapy and secondary uterine cancers is uncommon but well established; however, the genetic mechanisms underlying tamoxifen-driven tumorigenesis remain unclear. We find that oncogenic PIK3CA mutations, common in spontaneously arising estrogen-associated de novo uterine cancer, are significantly less frequent in tamoxifen-associated tumors. In vivo, tamoxifen-induced estrogen receptor stimulation activates phosphoinositide 3-kinase (PI3K) signaling in normal mouse uterine tissue, potentially eliminating the selective benefit of PI3K-activating mutations in tamoxifen-associated uterine cancer. Together, we present a unique pathway of therapy-associated carcinogenesis in which tamoxifen-induced activation of the PI3K pathway acts as a non-genetic driver event, contributing to the multistep model of uterine carcinogenesis. While this PI3K mechanism is specific to tamoxifen-associated uterine cancer, the concept of treatment-induced signaling events may have broader applicability to other routes of tumorigenesis. Sequencing analysis of tamoxifen-associated uterine cancers and further in vivo analyses suggest that the drug tamoxifen can activate the PI3K pathway in the absence of oncogenic mutations.},
  archive      = {J_NG},
  author       = {Kübler, Kirsten and Nardone, Agostina and Anand, Shankara and Gurevich, Daniel and Gao, Jianjiong and Droog, Marjolein and Hermida-Prado, Francisco and Akhshi, Tara and Feiglin, Ariel and Feit, Avery S. and Cohen Feit, Gabriella and Dackus, Gwen and Pun, Matthew and Kuang, Yanan and Cha, Justin and Miller, Mendy and Gregoricchio, Sebastian and Lanfermeijer, Mirthe and Cornelissen, Sten and Gibson, William J. and Paweletz, Cloud P. and Van Allen, Eliezer M. and van Leeuwen, Flora E. and Nederlof, Petra M. and Nguyen, Quang-Dé and Mourits, Marian J. E. and Radovich, Milan and Leshchiner, Ignaty and Stewart, Chip and Matulonis, Ursula A. and Zwart, Wilbert and Maruvka, Yosef E. and Getz, Gad and Jeselsohn, Rinath},
  doi          = {10.1038/s41588-025-02308-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2192-2202},
  shortjournal = {Nat. Genet.},
  title        = {Tamoxifen induces PI3K activation in uterine cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ERG-driven prostate cancer initiation is cell-context dependent and requires KMT2A and DOT1L. <em>NG</em>, <em>57</em>(9), 2177-2191. (<a href='https://doi.org/10.1038/s41588-025-02289-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the high prevalence of ERG transcription factor translocations in prostate cancer, the mechanism of tumorigenicity remains poorly understood. Using lineage tracing, we find the tumor-initiating activity of ERG resides in a subpopulation of murine basal cells that coexpress luminal genes (BasalLum) and not in the larger population of ERG+ luminal cells. Upon ERG activation, BasalLum cells give rise to highly proliferative intermediate (IM) cells with stem-like features that coexpress basal, luminal, hillock and club marker genes, before transitioning to Krt8+ luminal cells. Transcriptomic analysis of ERG+ human prostate cancers confirms the presence of rare ERG+ BasalLum cells, as well as IM cells whose presence is associated with a worse prognosis. Single-cell analysis revealed a chromatin state in ERG+ IM cells enriched for STAT3 transcription factor binding sites and elevated expression of the KMT2A/MLL1 and DOT1L, all three of which are essential for ERG-driven tumorigenicity in vivo. In addition to providing translational opportunities, this work illustrates how single-cell approaches combined with lineage tracing can identify cancer vulnerabilities not evident from bulk analysis. Lineage tracing in mice identifies a subpopulation of basal cells that express Tmprss2 and Nkx3 as the origin of ERG-driven prostate cancer. Upon expansion, these cells show an enrichment for STAT3 chromatin binding and elevated expression of KMT2A and DOT1L as dependencies for ERG oncogenicity.},
  archive      = {J_NG},
  author       = {Feng, Weiran and Ladewig, Erik and Lange, Matthew and Salsabeel, Nazifa and Zhao, Huiyong and Lee, Young Sun and Gopalan, Anuradha and Luo, Hanzhi and Kang, Wenfei and Fan, Ning and Rosiek, Eric and de Stanchina, Elisa and Chen, Yu and Carver, Brett S. and Leslie, Christina S. and Sawyers, Charles L.},
  doi          = {10.1038/s41588-025-02289-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2177-2191},
  shortjournal = {Nat. Genet.},
  title        = {ERG-driven prostate cancer initiation is cell-context dependent and requires KMT2A and DOT1L},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mutational landscape of triple-negative breast cancer in african american women. <em>NG</em>, <em>57</em>(9), 2166-2176. (<a href='https://doi.org/10.1038/s41588-025-02322-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {African American (AA) women have the highest incidence of triple-negative breast cancer (TNBC) among all racial groups, but are underrepresented in cancer genomic studies. In 462 AA women with TNBC, we characterized the tumor mutational landscape by whole-exome sequencing and RNA sequencing. We unveiled a high-resolution mutational portrait of TNBC in AA women reminiscent of that in Asian and non-Hispanic white women, with no evidence of associations of mutational features with African ancestry. We also made some distinctive discoveries, including an almost complete dominance of TP53 mutations, low frequency of PIK3CA mutations and mutational signature-based subtypes with etiologic and prognostic significance. These findings do not support major racial differences in TNBC biology at the level of somatic mutations. Our study contributes considerably to diversifying the knowledge base of breast cancer genomics and provides insights into the disease etiology, disparities and therapeutic vulnerability of TNBC in AA women. This study explores the genomic and transcriptomic landscapes of triple-negative breast cancer in African American women. The authors show that the mutational profile is broadly similar to that observed in European and East Asian ancestry women while highlighting some interesting differences.},
  archive      = {J_NG},
  author       = {Yao, Song and Wei, Lei and Hu, Qiang and Liu, Song and Manojlovic, Zarko and Fiorica, Peter N. and Long, Mark and Zirpoli, Gary R. and Cai, Qiuyin and Long, Jirong and Ping, Jie and Barnard, Mollie E. and Jin, Yuxin and Murakami, Mitsuko and Wang, Jianmin and Zhu, Qianqian and Davis, Warren and Chen, Jianhong and Ondracek, Rochelle P. and Khoury, Thaer and Gandhi, Shipra and Takabe, Kazuaki and Ko, Naomi and Sanderson, Maureen and Hong, Chi-Chen and Bandera, Elisa V. and Craig, David W. and Ambrosone, Christine B. and Palmer, Julie R. and Zheng, Wei and Carpten, John D.},
  doi          = {10.1038/s41588-025-02322-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2166-2176},
  shortjournal = {Nat. Genet.},
  title        = {Mutational landscape of triple-negative breast cancer in african american women},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiancestry brain pQTL fine-mapping and integration with genome-wide association studies of 21 neurologic and psychiatric conditions. <em>NG</em>, <em>57</em>(9), 2156-2165. (<a href='https://doi.org/10.1038/s41588-025-02291-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To understand shared and ancestry-specific genetic control of brain protein expression and its ramifications for disease, we mapped protein quantitative trait loci (pQTLs) in 1,362 brain proteomes from African American, Hispanic/Latin American and non-Hispanic white donors. Among the pQTLs that multiancestry fine-mapping MESuSiE confidently assigned as putative causal pQTLs in a specific population, most were shared across the three studied populations and are referred to as multiancestry causal pQTLs. These multiancestry causal pQTLs were enriched for exonic and promoter regions. To investigate their effects on disease, we modeled the 858 multiancestry causal pQTLs as instrumental variables using Mendelian randomization and genome-wide association study results for neurologic and psychiatric conditions (21 traits in participants with European ancestry, 10 in those with African ancestry and 4 in Hispanic participants). We identified 119 multiancestry pQTL–protein pairs consistent with a causal role in these conditions. Remarkably, 29% of the multiancestry pQTLs in these pairs were coding variants. These results lay an important foundation for the creation of new molecular models of neurologic and psychiatric conditions that are likely to be relevant to individuals across different genetic ancestries. Multiancestry fine-mapping of brain protein quantitative trait loci coupled with Mendelian randomization analyses identifies protein–trait pairs consistent with causal effects across neurological and psychiatric conditions.},
  archive      = {J_NG},
  author       = {Wingo, Aliza P. and Liu, Yue and Vattathil, Selina M. and Gerasimov, Ekaterina S. and Mei, Zhen and Ravindran, Suda Parimala and Liu, Jiaqi and Shantaraman, Ananth and Seifar, Fatemeh and Wang, Erming and Zhang, Bin and Reddy, Joseph and Allen, Mariet and Ertekin-Taner, Nilüfer and De Jager, Philip L. and Fox, Edward J. and Duong, Duc M. and Epstein, Michael P. and Cutler, David J. and Levey, Allan I. and Bennett, David A. and Seyfried, Nicholas T. and Wingo, Thomas S.},
  doi          = {10.1038/s41588-025-02291-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2156-2165},
  shortjournal = {Nat. Genet.},
  title        = {Multiancestry brain pQTL fine-mapping and integration with genome-wide association studies of 21 neurologic and psychiatric conditions},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Noncoding rare variant associations with blood traits in 166,740 UK biobank genomes. <em>NG</em>, <em>57</em>(9), 2146-2155. (<a href='https://doi.org/10.1038/s41588-025-02288-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large biobanks with whole-genome sequencing (WGS) now enable the association of noncoding rare variants with complex human traits. Given that &gt;98% of the genome is available for exploration, the selection of noncoding variants remains a critical yet unresolved challenge in these analyses. Here we leverage knowledge of blood gene regulation and deleteriousness scores to select noncoding variants pertinent for association with blood-related traits. Integrating WGS and 42 blood cell count and biomarker measurements for 166,740 UK Biobank samples, we perform variant collapsing tests, identifying hundreds of gene–trait associations involving noncoding variants. However, we demonstrate that most of these noncoding rare variant associations (1) reproduce associations known from previous studies and (2) are driven by linkage disequilibrium between nearby common and rare variants. This study underscores the prevailing challenges in rare variant analysis and the need for caution when interpreting noncoding rare variant association results. Noncoding rare variant analyses using whole-genome sequencing data from the UK Biobank identify gene–trait associations for 42 blood cell traits but find that most signals are driven by linkage disequilibrium between common and rare variants.},
  archive      = {J_NG},
  author       = {Ribeiro, Diogo M. and Hofmeister, Robin J. and Rubinacci, Simone and Delaneau, Olivier},
  doi          = {10.1038/s41588-025-02288-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2146-2155},
  shortjournal = {Nat. Genet.},
  title        = {Noncoding rare variant associations with blood traits in 166,740 UK biobank genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cross-biobank generalizability and accuracy of electronic health record-based predictors compared to polygenic scores. <em>NG</em>, <em>57</em>(9), 2136-2145. (<a href='https://doi.org/10.1038/s41588-025-02298-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Electronic health record (EHR)-based phenotype risk scores (PheRS) leverage individuals’ health trajectories to estimate disease risk, similar to how polygenic scores (PGS) use genetic information. While PGS generalizability has been studied, less is known about PheRS generalizability across healthcare systems and whether PheRS are complementary to PGS. We trained elastic-net-based PheRS to predict the onset of 13 common diseases for 845,929 individuals (age = 32–70 years) from three biobank-based studies in Finland (FinnGen), the UK (UKB) and Estonia (EstB). All PheRS were statistically significantly associated with the diseases of interest and most generalized well without retraining when applied to other studies. PheRS and PGS were only moderately correlated and models including both predictors improved onset prediction compared to PGS alone for 8 of 13 diseases. Our results indicate that EHR-based risk scores can transfer well between EHRs, capture largely independent information from PGS, and provide additive benefits for disease risk prediction. Comparison of electronic health record-based phenotype risk scores (PheRS) and polygenic scores (PGS) across 13 common diseases and three biobank-based studies indicates that PheRS and PGS may provide additive benefits for risk prediction.},
  archive      = {J_NG},
  author       = {Detrois, Kira E. and Hartonen, Tuomo and Teder-Laving, Maris and Jermy, Bradley and Läll, Kristi and Yang, Zhiyu and Mägi, Reedik and Ripatti, Samuli and Ganna, Andrea},
  doi          = {10.1038/s41588-025-02298-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2136-2145},
  shortjournal = {Nat. Genet.},
  title        = {Cross-biobank generalizability and accuracy of electronic health record-based predictors compared to polygenic scores},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Robust and accurate bayesian inference of genome-wide genealogies for hundreds of genomes. <em>NG</em>, <em>57</em>(9), 2124-2135. (<a href='https://doi.org/10.1038/s41588-025-02317-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Ancestral Recombination Graph (ARG), which describes the genealogical history of a sample of genomes, is a vital tool in population genomics and biomedical research. Recent advancements have substantially increased ARG reconstruction scalability, but they rely on approximations that can reduce accuracy, especially under model misspecification. Moreover, they reconstruct only a single ARG topology and cannot quantify the considerable uncertainty associated with ARG inferences. Here, to address these challenges, we introduce SINGER (sampling and inferring of genealogies with recombination), a method that accelerates ARG sampling from the posterior distribution by two orders of magnitude, enabling accurate inference and uncertainty quantification for hundreds of whole-genome sequences. Through extensive simulations, we demonstrate SINGER’s enhanced accuracy and robustness to model misspecification compared to existing methods. We demonstrate the utility of SINGER by applying it to individuals of British and African descent within the 1000 Genomes Project, identifying signals of population differentiation, archaic introgression and strong support for ancient polymorphism in the human leukocyte antigen region shared across primates. SINGER is a method for creating ancestral recombination graphs to understand the genealogical history of genomes. The method has increased speed, and thus scalability, without sacrificing accuracy.},
  archive      = {J_NG},
  author       = {Deng, Yun and Nielsen, Rasmus and Song, Yun S.},
  doi          = {10.1038/s41588-025-02317-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2124-2135},
  shortjournal = {Nat. Genet.},
  title        = {Robust and accurate bayesian inference of genome-wide genealogies for hundreds of genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes. <em>NG</em>, <em>57</em>(9), 2116-2123. (<a href='https://doi.org/10.1038/s41588-025-02286-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mixed-model association analysis (MMAA) is the preferred tool for performing genome-wide association studies. However, existing MMAA tools often have long runtimes and high memory requirements. Here we present LDAK-KVIK, an MMAA tool for analysis of quantitative and binary phenotypes. LDAK-KVIK is computationally efficient, requiring less than 10 CPU hours and 5 Gb memory to analyze genome-wide data for 350,000 individuals. Using simulated phenotypes, we show that LDAK-KVIK produces well-calibrated test statistics for both homogeneous and heterogeneous datasets. When applied to real phenotypes, LDAK-KVIK has the highest power among all tools considered. For example, across 40 quantitative UK Biobank phenotypes (average sample size 349,000), LDAK-KVIK finds 16% more independent, genome-wide significant loci than classical linear regression, whereas BOLT-LMM and REGENIE find 15% and 11% more, respectively. LDAK-KVIK can also be used to perform gene-based tests; across the 40 quantitative UK Biobank phenotypes, LDAK-KVIK finds 18% more significant genes than the leading existing tool. Last, LDAK-KVIK produces state-of-the-art polygenic scores. LDAK-KVIK is a mixed-model association method for genome-wide studies that optimizes computational performance and power. LDAK-KVIK can also perform gene-based tests and produces state-of-the-art polygenic scores.},
  archive      = {J_NG},
  author       = {Hof, Jasper P. and Speed, Doug},
  doi          = {10.1038/s41588-025-02286-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2116-2123},
  shortjournal = {Nat. Genet.},
  title        = {LDAK-KVIK performs fast and powerful mixed-model association analysis of quantitative and binary phenotypes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Toward improving multiomic research in understudied cereals. <em>NG</em>, <em>57</em>(9), 2106-2115. (<a href='https://doi.org/10.1038/s41588-025-02245-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the looming challenges of climate change and population pressure, understudied cereals hold notably untapped potential for future global food security and sustainable agriculture due to their unique nutritional profiles and notable stress resilience. Multiomics holds great promise in realizing the value of understudied cereals by understanding the genetic basis and molecular mechanisms regulating their unique traits; however, multiomic studies of understudied cereals still lag behind those of major cereals, hindering their genetic improvement and wide adoption in modern agriculture. Here, we discuss the limitations and challenges in multiomic studies of understudied cereals and how they can be overcome to enhance utilization of the traits unique to understudied cereals in improving both themselves and major cereals. We also offer a database specific to understudied cereals, including multiomics data from 12 understudied cereals and wild relatives for some of these, to serve the community. Understudied cereals offer untapped potential for food security, yet multiomic research substantially lags behind that of major cereals. This Perspective discusses these challenges and introduces a dedicated database to serve the community.},
  archive      = {J_NG},
  author       = {Yang, Fanjing and Gong, Xiaojiao and Zhao, Pan and Cheng, Lingtong and Zhang, Yujie and Luo, Yingting and Zhu, Qian-Hao and Ye, Chu-Yu},
  doi          = {10.1038/s41588-025-02245-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2106-2115},
  shortjournal = {Nat. Genet.},
  title        = {Toward improving multiomic research in understudied cereals},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The LISTEN principles for genetic sequence data governance and database engineering. <em>NG</em>, <em>57</em>(9), 2099-2105. (<a href='https://doi.org/10.1038/s41588-025-02270-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several international legal agreements include an ‘access and benefit-sharing’ (ABS) mechanism that attaches obligations to the use of genetic sequence data. These agreements are frequently subject to critique on the grounds that ABS is either fundamentally incompatible with the principles of open science, or technically challenging to implement in open scientific databases. Here, we argue that these critiques arise from a misinterpretation of the principles of open science and that both considerations can be addressed by a set of simple principles that link database engineering and governance. We introduce a checklist of six database design considerations, LISTEN: licensed, identified, supervised, transparent, enforced and non-exclusive, which can be readily adopted by both new and existing platforms participating in ABS systems. We also highlight how these principles can act in concert with familiar principles of open science, such as findable, accessible, interoperable and reusable (FAIR) data sharing. This Perspective proposes a checklist of six database design considerations, LISTEN: licensed, identified, supervised, transparent, enforced and non-exclusive, aimed at ensuring access and benefit-sharing principles in open science.},
  archive      = {J_NG},
  author       = {Carlson, Colin J. and Granados, Monica and Phelan, Alexandra and Ramakrishnan, Nithin and Poisot, Timothée},
  doi          = {10.1038/s41588-025-02270-7},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2099-2105},
  shortjournal = {Nat. Genet.},
  title        = {The LISTEN principles for genetic sequence data governance and database engineering},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Governance of cross-border genomic data sharing through a human rights approach. <em>NG</em>, <em>57</em>(9), 2090-2098. (<a href='https://doi.org/10.1038/s41588-025-02252-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cross-border genomic data sharing is crucial for advancing research progress, understanding of genetic diversity and international public health. However, this practice also leads to concerns regarding biosecurity, individual privacy and regulatory conflicts. Scientific communities often advocate data sovereignty approaches to address these challenges, yet related frameworks are often inadequate in their ability to address such challenges and have an overemphasis on national control and protectionism. They also may not fully consider the inherently transnational nature of genomic data and the need for a governance system that reflects the shared human heritage of the data and global interconnectivity. Instead, we propose here a human rights-based approach that surpasses territorial boundaries to promote shared responsibility and equitable research practices. A balanced governance framework that incorporates extraterritorial obligations and aligns data policy with human rights principles can safeguard individual privacy and biosecurity, encourage international collaboration and support ethically responsible genomic research. This Perspective proposes a human rights-based governance framework for cross-border genomic data sharing, addressing limitations of data sovereignty approaches while balancing privacy, security and global research collaboration.},
  archive      = {J_NG},
  author       = {Yu, Liang and Feng, Ruohan and Sun, Youhai and Peng, Yaojin},
  doi          = {10.1038/s41588-025-02252-9},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2090-2098},
  shortjournal = {Nat. Genet.},
  title        = {Governance of cross-border genomic data sharing through a human rights approach},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Recreating fibroblast diversity in vitro by activating transcription factors. <em>NG</em>, <em>57</em>(9), 2088-2089. (<a href='https://doi.org/10.1038/s41588-025-02283-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We combined CRISPR activation of all 1,836 known human transcription factors with high-throughput Perturb-seq to recreate the diverse transcriptional states occupied by fibroblasts in vivo. Our study revealed regulators of key states and showed that inducing normal states can, in some cases, suppress those linked to disease.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02283-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2088-2089},
  shortjournal = {Nat. Genet.},
  title        = {Recreating fibroblast diversity in vitro by activating transcription factors},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Improving broad-spectrum resistance to clubroot disease with one gene. <em>NG</em>, <em>57</em>(9), 2086-2087. (<a href='https://doi.org/10.1038/s41588-025-02304-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {GSL5, a glucan synthase, acts as a suppressor of jasmonic acid-mediated immunity in cruciferous plants. Inactivation of GSL5 by genome editing confers high-level and broad-spectrum resistance to pathogens that cause clubroot disease in four cruciferous species.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02304-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2086-2087},
  shortjournal = {Nat. Genet.},
  title        = {Improving broad-spectrum resistance to clubroot disease with one gene},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mystery of tamoxifen-associated uterine cancer uncovered. <em>NG</em>, <em>57</em>(9), 2084-2085. (<a href='https://doi.org/10.1038/s41588-025-02316-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tamoxifen is an essential drug in breast cancer therapy. Unlike prevailing models of therapy-related tumorigenesis, tamoxifen acts by directly activating the PI3K pathway, bypassing the need for mutations in one of the most common driver genes in sporadic uterine cancer. These findings open avenues for investigating similar mechanisms in other drugs.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02316-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2084-2085},
  shortjournal = {Nat. Genet.},
  title        = {Mystery of tamoxifen-associated uterine cancer uncovered},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A bivalent chromatin state regulates the function of composite transposons. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02337-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02337-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {A bivalent chromatin state regulates the function of composite transposons},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicting obesity from childhood. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02338-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Brandt, Margot},
  doi          = {10.1038/s41588-025-02338-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {Predicting obesity from childhood},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Building immune tolerance in the thymus. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02339-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02339-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {Building immune tolerance in the thymus},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI identifies promoter variants that alter gene expression. <em>NG</em>, <em>57</em>(9), 2083. (<a href='https://doi.org/10.1038/s41588-025-02340-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-025-02340-w},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2083},
  shortjournal = {Nat. Genet.},
  title        = {AI identifies promoter variants that alter gene expression},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). KidneyGenAfrica, a pan-african partnership to deliver research and training excellence in genomics of kidney disease. <em>NG</em>, <em>57</em>(9), 2079-2082. (<a href='https://doi.org/10.1038/s41588-025-02299-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite Africa’s vast genetic diversity, its populations are underrepresented in global genomic datasets. Here we describe the vision of the KidneyGenAfrica, a pan-African initiative launched to address this inequity, and call for more inclusive genomics research that recognizes Africa’s key role in genetic variation and potential to generate insights in chronic kidney disease.},
  archive      = {J_NG},
  author       = {Fatumo, Segun and Ojewunmi, Oyesola and Camenzuli, Rebecca and Kintu, Christopher and Kamiza, Abram and Brandenburg, Jean-Tristan and Kalungi, Allan and Kalyesubula, Robert and Salako, Babatunde and Nash, Oyekanmi and Crampin, Amelia and Adeyemo, Adebowale and Nyirenda, Moffat and Ramsay, Michele and Fabian, June},
  doi          = {10.1038/s41588-025-02299-8},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2079-2082},
  shortjournal = {Nat. Genet.},
  title        = {KidneyGenAfrica, a pan-african partnership to deliver research and training excellence in genomics of kidney disease},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The future of polygenic risk scores in direct-to-consumer genomics. <em>NG</em>, <em>57</em>(9), 2077-2078. (<a href='https://doi.org/10.1038/s41588-025-02313-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Sun, Yuhao and Vines, John and Tenesa, Albert},
  doi          = {10.1038/s41588-025-02313-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {2077-2078},
  shortjournal = {Nat. Genet.},
  title        = {The future of polygenic risk scores in direct-to-consumer genomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: The long-term effects of chemotherapy on normal blood cells. <em>NG</em>, <em>57</em>(8), 2075. (<a href='https://doi.org/10.1038/s41588-025-02315-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Mitchell, Emily and Pham, My H. and Clay, Anna and Sanghvi, Rashesh and Williams, Nicholas and Pietsch, Sandra and Hsu, Joanne I. and Øbro, Nina Friesgaard and Jung, Hyunchul and Vedi, Aditi and Moody, Sarah and Wang, Jingwei and Leonganmornlert, Daniel and Spencer Chapman, Michael and Dunstone, Ellie and Santarsieri, Anna and Cagan, Alex and Machado, Heather E. and Baxter, E. Joanna and Follows, George and Hodson, Daniel J. and McDermott, Ultan and Doherty, Gary J. and Martincorena, Inigo and Humphreys, Laura and Mahbubani, Krishnaa and Saeb Parsy, Kourosh and Takahashi, Koichi and Goodell, Margaret A. and Kent, David and Laurenti, Elisa and Campbell, Peter J. and Rahbari, Raheleh and Nangalia, Jyoti and Stratton, Michael R.},
  doi          = {10.1038/s41588-025-02315-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {2075},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: The long-term effects of chemotherapy on normal blood cells},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Incorporating genetic data improves target trial emulations and informs the use of polygenic scores in randomized controlled trial design. <em>NG</em>, <em>57</em>(8), 2074. (<a href='https://doi.org/10.1038/s41588-025-02275-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {German, Jakob and Yang, Zhiyu and Urbut, Sarah and Vartiainen, Pekka and Natarajan, Pradeep and Patorno, Elisabetta and Kutalik, Zoltan and Philippakis, Anthony and Ganna, Andrea},
  doi          = {10.1038/s41588-025-02275-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {2074},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Incorporating genetic data improves target trial emulations and informs the use of polygenic scores in randomized controlled trial design},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Massively parallel immunopeptidome by DNA sequencing provides insights into cancer antigen presentation. <em>NG</em>, <em>57</em>(8), 2062-2073. (<a href='https://doi.org/10.1038/s41588-025-02268-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human leukocyte antigens (HLAs) are encoded by the most polymorphic genes in the human genome. HLA class I alleles control antigen presentation for T cell recognition, which is pivotal for autoimmunity, infectious diseases and cancer. Current knowledge of HLA-bound peptides is limited, skewed and falls short of population-wide HLA binding profiles for high-value targets. Here we present ESCAPE-seq (enhanced single-chain antigen presentation sequencing), a massively parallel platform for comprehensive screening of class I HLA–peptide combinations for antigen presentation via deep DNA sequencing. ESCAPE-seq demonstrates programmability, high throughput, sensitivity and nominated viral and cancer epitopes. We simultaneously assessed over 75,000 peptide–HLA combinations, revealing broadly presented epitopes from oncogenic driver mutations and fusions across diverse HLA-A, HLA-B and HLA-C alleles that cover 90% of the human population. We further identified epitopes that are differentially presented, comparing oncogenic hotspot mutations versus wild type. ESCAPE-seq enables one-shot population-wide antigen presentation discovery, offering insights into HLA specificity and immune recognition of genomic mutations. ESCAPE-seq (enhanced single-chain antigen presentation sequencing) is a massively parallel platform for screening of class I HLA–peptide combinations for antigen presentation. The authors assess more than 75,000 peptide–HLA combinations, revealing presented epitopes from oncogenic driver mutations and fusions across diverse HLA-A, HLA-B and HLA-C alleles.},
  archive      = {J_NG},
  author       = {Shi, Quanming and Simon, Elana P. and Cimen Bozkus, Cansu and Kaminska, Anna and Velazquez, Leandra and Saxena, Mansi and Zhang, Zilin and Belk, Julia A. and Wang, Shuo and Yang, Nuoya and Zhang, Yaowen and Kwong, Ashley and Che, Yonglu and Stickels, Robert R. and Crain, Charles R. and Schmidt-Hong, Laura and Lichti, Cheryl F. and Gaiha, Gaurav D. and Roth, Theodore L. and Bhardwaj, Nina and Satpathy, Ansuman T. and Yu, Bingfei and Chang, Howard Y.},
  doi          = {10.1038/s41588-025-02268-1},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {2062-2073},
  shortjournal = {Nat. Genet.},
  title        = {Massively parallel immunopeptidome by DNA sequencing provides insights into cancer antigen presentation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pangenome analysis provides insights into legume evolution and breeding. <em>NG</em>, <em>57</em>(8), 2052-2061. (<a href='https://doi.org/10.1038/s41588-025-02280-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Grain legumes hold great promise for advancing sustainable agriculture. Although the evolutionary history of legume species has been investigated, the conserved mechanisms that drive adaptive evolution and govern agronomic improvement remain elusive. Here we present high-quality genome assemblies for nine widely consumed pulses, including common bean, chickpea, pea, lentil, faba bean, pigeon pea, cowpea, mung bean and hyacinth bean. Pangenome analysis reveals the expansion of distinct gene sets in cool-season and warm-season legumes, highlighting the role of gene birth and duplication in the autoregulation of nodulation. Notably, hundreds of genes undergo convergent selection during the evolution of legumes, affecting agronomic traits such as seed weight. In addition, we demonstrate that tandem amplification of transposable elements in gene-depleted regions has a crucial role in driving genome enlargement and the formation of regulatory elements in cool-season legumes. Our results provide insights into the molecular mechanisms underlying the diversification of legumes and represent a valuable resource for facilitating legume breeding. De novo long-read genome assembly for nine legume species and pangenome analysis identify genomic variations associated with environmental adaptation and domestication and highlight the role of transposable elements in genome evolution.},
  archive      = {J_NG},
  author       = {Wang, Longfei and Jiang, Xinyu and Jiao, Wu and Mao, Junrong and Ye, Wenxue and Cao, Yangrong and Chen, Qingshan and Song, Qingxin},
  doi          = {10.1038/s41588-025-02280-5},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {2052-2061},
  shortjournal = {Nat. Genet.},
  title        = {Pangenome analysis provides insights into legume evolution and breeding},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Haplotype-resolved, gap-free genome assemblies provide insights into the divergence between asian and european pears. <em>NG</em>, <em>57</em>(8), 2040-2051. (<a href='https://doi.org/10.1038/s41588-025-02273-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pears (Pyrus spp.) are self-incompatible crops with broad genetic diversity. High heterozygosity and technical limitations result in gaps within reference genomes. Our study presents the telomere-to-telomere, haplotype-resolved genomes of a representative Asian pear ‘Dangshansuli’ (Pyrus bretschneideri) and a European pear ‘Max Red Bartlett’ (Pyrus communis). Haplotype-specific genes exhibited notable differences from biallelic genes regarding transposable content, methylation patterns and expression levels. Allele-specific expression analysis suggested that the dominance effect is vital in the formation of fruit quality of pears. Population analysis of 362 accessions revealed that interspecific introgression increased pear diversity. We constructed a graph-based genome and identified structural variations associated with agronomic traits. A 286-bp insertion in the promoter region, along with differential expression of PyACS1, was identified between Asian and European pears, which exhibit distinct fruit-softening characteristics. Further experiments demonstrated the role of PyACS1 in fruit softening. Overall, this study provided insights into genetic variation and will facilitate pear improvement. Haplotype-resolved, gap-free genome assemblies for a representative Asian pear (Pyrus bretschneideri, ‘Dangshansuli’) and a European pear (Pyrus communis, ‘Max Red Bartlett’) provide insights into genome evolution and interspecies variation in Pyrus species.},
  archive      = {J_NG},
  author       = {Sun, Manyi and Cao, Beibei and Li, Kui and Li, Jiaming and Liu, Jun and Xue, Cheng and Gu, Kaidi and Xu, Shaozhuo and Li, Yuanjun and Li, Qingyu and Qu, Meina and Zhang, Mingyue and Wang, Runze and Liu, Yueyuan and Yao, Chenjie and He, Hang and Wu, Jun},
  doi          = {10.1038/s41588-025-02273-4},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {2040-2051},
  shortjournal = {Nat. Genet.},
  title        = {Haplotype-resolved, gap-free genome assemblies provide insights into the divergence between asian and european pears},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome assembly of two allotetraploid cotton germplasms reveals mechanisms of somatic embryogenesis and enables precise genome editing. <em>NG</em>, <em>57</em>(8), 2028-2039. (<a href='https://doi.org/10.1038/s41588-025-02258-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic embryogenesis is crucial for plant genetic engineering, yet the underlying mechanisms in cotton remain poorly understood. Here we present a telomere-to-telomere assembly of Jin668 and a high-quality assembly of YZ1, two highly regenerative allotetraploid cotton germplasms. The completion of the Jin668 genome enables characterization of ~30.1 Mb of centromeric regions invaded by centromeric retrotransposon of maize and Tekay retrotransposons, an ~8.1 Mb 5S rDNA array containing 25,190 copies and a ~75.1 Mb major 45S rDNA array with 8,131 copies. Comparative analyses of regenerative and recalcitrant genotypes reveal dynamic transcriptional patterns and chromatin accessibility during the initial regeneration process. A hierarchical gene regulatory network identifies AGL15 as a contributor to regeneration. Additionally, we demonstrate that genetic variation affects sgRNA target sites, while the Jin668 genome assembly reduces the risk of off-target effects in CRISPR-based genome editing. Together, the complete Jin668 genome reveals the complexity of genomic regions and cotton regeneration, and improves the precision of genome editing. Genome assemblies of two allotetraploid cotton germplasms, Jin668 and YZ1, reveal the regulatory mechanisms underlying somatic embryogenesis and plant regeneration, and provide potential for precise genome editing in cotton.},
  archive      = {J_NG},
  author       = {Xu, Zhongping and Wang, Guanying and Zhu, Xiangqian and Wang, Ruipeng and Zhu, Longfu and Tu, Lili and Liu, Yuling and Peng, Renhai and Lindsey, Keith and Wang, Maojun and Zhang, Xianlong and Jin, Shuangxia},
  doi          = {10.1038/s41588-025-02258-3},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {2028-2039},
  shortjournal = {Nat. Genet.},
  title        = {Genome assembly of two allotetraploid cotton germplasms reveals mechanisms of somatic embryogenesis and enables precise genome editing},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genomics-driven discovery of superior alleles and genes for yellow rust resistance in wheat. <em>NG</em>, <em>57</em>(8), 2017-2027. (<a href='https://doi.org/10.1038/s41588-025-02259-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Yellow rust (YR), caused by Puccinia striiformis f. sp. tritici, poses a significant threat to wheat production worldwide. Breeding resistant cultivars is crucial for managing this disease. However, our understanding of the genetic mechanisms underlying YR resistance remains fragmented. To address this, we conducted a comprehensive analysis with variome data from 2,191 wheat accessions worldwide and over 47,000 YR response records across several environments and pathogen races. Through genome-wide association studies, we established a landscape of 431 YR resistance loci, providing a rich resource for resistance gene deployment. Furthermore, we cloned genes corresponding to three resistance loci, namely Yr5x (effective against several P. striiformis f. sp. tritici races), Yr6/Pm5 (conferred resistance to two pathogen species) and YrKB (TaEDR2-B; conferring broad-spectrum rust resistance without yield penalty). These findings offer valuable insights into the genetic basis of YR resistance in wheat and lay the foundation for engineering wheat with durable disease resistance. Genomic analyses of 2,191 global common wheat accessions and over 47,000 yellow rust (YR) response data points across several environments and pathogen races provide a genome-wide landscape of YR resistance genes and effective alleles.},
  archive      = {J_NG},
  author       = {Wu, Jianhui and Ma, Shengwei and Niu, Jianqing and Sun, Weihang and Dong, Haitao and Zheng, Shusong and Zhao, Jiwen and Liu, Shengjie and Yu, Rui and Li, Ying and Han, Jinyu and Wang, Yimin and Chen, Tiantian and Zhang, Chuanliang and Zhang, Wenjing and Ding, Bingyou and Chang, Lu and Xue, Wenbo and Zheng, Weijun and Li, Chunlian and Han, Dejun and Kang, Zhensheng and Zeng, Qingdong and Ling, Hong-Qing},
  doi          = {10.1038/s41588-025-02259-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {2017-2027},
  shortjournal = {Nat. Genet.},
  title        = {Genomics-driven discovery of superior alleles and genes for yellow rust resistance in wheat},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Natural variation in SBRR1 shows high potential for sheath blight resistance breeding in rice. <em>NG</em>, <em>57</em>(8), 2004-2016. (<a href='https://doi.org/10.1038/s41588-025-02281-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sheath blight (ShB), caused by necrotrophic fungus Rhizoctonia solani, is one of the most serious rice diseases worldwide. To the best of our knowledge, no genes with high potential for rice ShB resistance breeding have been previously characterized. Here we identify a ShB resistance receptor-like kinase 1 (SBRR1) gene via a genome-wide association study. The SBRR1-R elite allele, containing a 256-bp insertion in its promoter, is preferentially present in indica varieties in geographical regions with highly favorable conditions for ShB development. Introduction of SBRR1-R into a commercial japonica rice variety significantly reduces yield loss under severe ShB disease pressure. Transcription factor bHLH57 specifically binds to the 256-bp sequence and accounts for highly induced expression and stronger resistance of SBRR1-R. Localization of SBRR1 on plasma membrane, aided by SBRR1-interaction-protein 1, and phosphorylation of SBRR1 are required for SBRR1 to rapidly upregulate downstream chitinase genes for resistance. These findings offer mechanistical insights into ShB resistance hidden in natural rice varieties. A genome-wide association study in rice identifies variation in SBRR1 associated with sheath blight resistance. Introducing the SBRR1-R allele into a commercial rice variety reduces yield loss under severe exposure to sheath blight disease.},
  archive      = {J_NG},
  author       = {Feng, Zhiming and Gao, Peng and Wang, Guangda and Kang, Houxiang and Zhao, Jianhua and Xie, Wenya and Chen, Rujia and Ju, Ran and Wang, Xuli and Wei, Zhaogen and Zhang, Huimin and Zhang, Jinqiao and Zhang, Yafang and Hu, Keming and Sun, Quanyi and Zhu, Yuntao and Xiong, Yehui and Liu, Xi and Chen, Xijun and Chen, Zongxiang and Yang, Chao and Liu, Jun and Wang, Guo-Liang and Zhang, Sheng-Wei and Yang, Zefeng and Zuo, Shimin},
  doi          = {10.1038/s41588-025-02281-4},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {2004-2016},
  shortjournal = {Nat. Genet.},
  title        = {Natural variation in SBRR1 shows high potential for sheath blight resistance breeding in rice},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Insights into yersinia pestis evolution through rearrangement analysis of 242 complete genomes. <em>NG</em>, <em>57</em>(8), 1994-2003. (<a href='https://doi.org/10.1038/s41588-025-02264-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Yersinia pestis, the bacterium that causes the plague, has a dynamic genome with highly conserved fragments prone to rearrangement, influencing gene function and evolution. However, understanding these patterns is limited by few complete genomes and analytical methods. We developed a dual-validation strategy to analyze 242 complete genomes of Y. pestis natural isolates from diverse phylogroups. We detected 459 rearrangements, which enhanced phylogenetic resolution and resolved the third pandemic’s polytomy. Rearrangements are primarily mediated by four common insertion sequences, with IS1661 and IS100 showing the highest activity. These rearrangements are under strong positive selection, evidenced by 43 hotspots and convergent evolution in the rpsO-pnp operon, whose disruptions and reconnections altered gene expressions and temperature stress responses. We also identified unique structural alterations in human avirulent phylogroups, inactivating three genes and reordering 17 intergenic regions, some affecting virulence-related genes. This study provides a fresh perspective on Y. pestis evolution, revealing experimental targets and establishing a methodology for microbes with frequent rearrangements. Analysis of synteny blocks and genomic rearrangement patterns of 178 newly sequenced and 64 publicly available complete genomes of Yersinia pestis highlights the impact of key variations on genes involved in adaptation and virulence.},
  archive      = {J_NG},
  author       = {Wu, Yarong and Yang, Chao and Mu, Kai and Guo, Yan and Song, Yajun and Yang, Ruifu and Cui, Yujun},
  doi          = {10.1038/s41588-025-02264-5},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1994-2003},
  shortjournal = {Nat. Genet.},
  title        = {Insights into yersinia pestis evolution through rearrangement analysis of 242 complete genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). G-quadruplexes are promoter elements controlling nucleosome exclusion and RNA polymerase II pausing. <em>NG</em>, <em>57</em>(8), 1981-1993. (<a href='https://doi.org/10.1038/s41588-025-02263-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite their central role in transcription, it has been difficult to define universal sequences associated with eukaryotic promoters. Within the chromatin context, recruitment of transcriptional machinery requires promoter opening, but how DNA elements contribute to this process is unclear. Here we show that G-quadruplex (G4) secondary DNA structures are highly enriched at mammalian promoters. G4s are located at the deepest point of nucleosome exclusion at promoters and correlate with maximum promoter activity. We found that experimental G4s exclude nucleosomes in vivo and in vitro while favouring strong positioning. At model promoters, impairing G4s affected both transcriptional activity and chromatin opening. G4 destabilization also resulted in an inactive promoter state and affected the transition to effective RNA production. Finally, G4 stabilization resulted in global reduction of proximal promoter pausing. Altogether, our data introduce G4s as bona fide promoter elements allowing nucleosome exclusion and facilitating pause–release by RNA polymerase II. This study examines the function of G-quadruplex DNA secondary structures in mammalian promoters, revealing that they represent core promoter elements contributing to gene regulation by nucleosome exclusion and by promoting RNA polymerase II pause–release.},
  archive      = {J_NG},
  author       = {Esnault, Cyril and Zine El Aabidine, Amal and Robert, Marie-Cécile and Cucchiarini, Anne and Magat, Talha and Pigeot, Alexia and Bouchouika, Soumya and Garcia-Oliver, Encar and Gawron, Kevin and Basyuk, Eugénia and Karpinska, Magdalena A. and Kozulic-Pirher, Alja and Luo, Yu and Verga, Daniela and Mourad, Raphael and Radulescu, Ovidiu and Mergny, Jean-Louis and Bertrand, Edouard and Andrau, Jean-Christophe},
  doi          = {10.1038/s41588-025-02263-6},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1981-1993},
  shortjournal = {Nat. Genet.},
  title        = {G-quadruplexes are promoter elements controlling nucleosome exclusion and RNA polymerase II pausing},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). NKX2-1 drives neuroendocrine transdifferentiation of prostate cancer via epigenetic and 3D chromatin remodeling. <em>NG</em>, <em>57</em>(8), 1966-1980. (<a href='https://doi.org/10.1038/s41588-025-02265-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A substantial amount of castration-resistant prostate cancer (CRPC) progresses into a neuroendocrine (NE) subtype, known as NEPC, which is associated with poor clinical outcomes. Here we report distinct three-dimensional chromatin architectures between NEPC and CRPC tumors, which were recapitulated by isogenic cell lines undergoing NE transformation (NET). Mechanistically, pioneer factors such as FOXA2 initiate binding at NE enhancers to mediate regional DNA demethylation and induce neural transcription factor (TF) NKX2-1 expression. NKX2-1 preferentially binds gene promoters and interacts with enhancer-bound FOXA2 through chromatin looping. NKX2-1 is highly expressed in NEPC and indispensable for NET of prostate cancer. NKX2-1/FOXA2 further recruits p300/CBP to activate NE enhancers, and pharmacological inhibition of p300/CBP effectively blunts NE gene expression and abolishes NEPC tumor growth. Taken together, our study reports a hierarchical network of TFs governed by NKX2-1 in critically regulating chromatin remodeling and driving luminal-to-NE transformation and suggests promising therapeutic approaches to mitigate NEPC. Single-cell RNA and assay for transposase-accessible chromatin sequencing in isogenic prostate adenocarcinoma cells undergoing neuroendocrine transition identify dynamic changes in chromatin and transcriptomic states coregulated by NKX2-1, FOXA2 and p300/CBP.},
  archive      = {J_NG},
  author       = {Lu, Xiaodong and Keo, Viriya and Cheng, Irina and Xie, Wanqing and Gritsina, Galina and Wang, Juan and Lu, Lina and Shiau, Cheng-Kai and He, Yueying and Jin, Qiushi and Jin, Peng and Sanda, Martin G. and Corces, Victor G. and Altemose, Nicolas and Gao, Ruli and Zhao, Jonathan C. and Yu, Jindan},
  doi          = {10.1038/s41588-025-02265-4},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1966-1980},
  shortjournal = {Nat. Genet.},
  title        = {NKX2-1 drives neuroendocrine transdifferentiation of prostate cancer via epigenetic and 3D chromatin remodeling},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell and spatial transcriptomics reveal mechanisms of radioresistance and immune escape in recurrent nasopharyngeal carcinoma. <em>NG</em>, <em>57</em>(8), 1950-1965. (<a href='https://doi.org/10.1038/s41588-025-02253-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Radiotherapy resistance and immune evasion are prominent features of recurrent nasopharyngeal carcinoma (rNPC). However, their mechanisms remain incompletely understood. Here, we conducted single-cell and spatial transcriptomics analysis of 39 tumors from 24 patients to reveal the microenvironmental differences between primary and rNPC. Specific MCAM+ cancer-associated fibroblasts are significantly enriched in rNPC, where they promote tumor radioresistance through the collagen IV–ITGA2–FAK–AKT axis. Furthermore, we found that collagen IV suppresses the infiltration of T cells, and we identified mechanisms of immune escape in rNPC. We uncovered the presence and function of CD8 ZNF683 cells in rNPC with lower cytotoxicity. The abundance of B cells and tertiary lymphoid structures significantly diminishes in rNPC. Finally, we confirmed that CD47–SIRPα commonly existed between myeloid and malignant cells in rNPC. This study provides an in-depth understanding of the mechanism of radioresistance and immune evasion in rNPC as well as highlighting critical preliminary targets for curing rNPC. Multi-omic analysis of 39 samples of treatment-naive and recurrent nasopharyngeal carcinomas highlights the importance of the tumor microenvironment in promoting radioresistance and immune escape.},
  archive      = {J_NG},
  author       = {You, Rui and Shen, Qunlun and Lin, Chao and Dong, Kangning and Liu, Xiao and Xu, Hanshi and Hu, Wanming and Xie, Yulong and Xie, Ruoqi and Song, Xiaoyi and Huang, Chunliu and Wu, Jinhui and Yu, Tao and Li, Huifeng and Wang, Zining and Chen, Keming and Zou, Xiong and Huang, Peiyu and Hua, Yijun and Liu, Youping and Xia, Tianliang and Zhang, Shihua and Chen, Mingyuan},
  doi          = {10.1038/s41588-025-02253-8},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1950-1965},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell and spatial transcriptomics reveal mechanisms of radioresistance and immune escape in recurrent nasopharyngeal carcinoma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multi-omic analysis of gallbladder cancer identifies distinct tumor microenvironments associated with disease progression. <em>NG</em>, <em>57</em>(8), 1935-1949. (<a href='https://doi.org/10.1038/s41588-025-02236-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gallbladder carcinoma (GBC) is the most aggressive biliary tract cancer and is associated with a high mortality rate. Treatment of GBC faces therapeutic challenges owing to the elusive nature of in situ drivers within the local tumor microenvironment that drive its progression. Here, we created a single-cell atlas of 1,117,245 cells and a mutational landscape from 102 patients, which unveiled spatial–temporal characterizations of cellular constitutions, spatial interplays and molecular functions, and generalized five local ecosystems stratifying clinical outcomes. An integrated epithelial program, AI-EPI, combined with spatial transcriptome analysis, revealed the concurrent localization of a highly malignant tumor subtype (GM16) and AREG+ T cell, B cell, dendritic cell and macrophage subtypes within the pro-metastatic niche of primary adenocarcinomas. In vitro and in vivo experiments suggest that in addition to promoting metastasis, AREG facilitates CXCL5 expression in tumor cells through EGFR–pERK–EGR1 signaling, leading to increased neutrophil infiltration and impeding the effectiveness of immunotherapy. Our study provides a spatial–temporal landscape of the GBC microenvironment and sheds light on potential strategies for preventing immunotherapy resistance. Single-cell RNA sequencing, whole-exome sequencing and spatial transcriptome sequencing of gallbladder adenocarcinomas from 75 patients stratifies the disease into groups with distinct clinical outcomes.},
  archive      = {J_NG},
  author       = {Zhou, Tao and Wu, Yanhong and Li, Shuai and Qiu, Xinyao and Liu, Erdong and Xie, Zhihua and Shi, Xuebing and Zhang, Yani and Ma, Guosheng and Guo, Wenbo and Wang, Xiang and Wang, Kaiting and Yao, Xiaomeng and Hu, Ji and Shen, Siyun and Yang, Shuai and Jiang, Xiaoqing and Fu, Jing and Wang, Hongyang and Gu, Jin and Chen, Lei},
  doi          = {10.1038/s41588-025-02236-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1935-1949},
  shortjournal = {Nat. Genet.},
  title        = {Multi-omic analysis of gallbladder cancer identifies distinct tumor microenvironments associated with disease progression},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Analysis of individual patient pathway coordination in a cross-species single-cell kidney atlas. <em>NG</em>, <em>57</em>(8), 1922-1934. (<a href='https://doi.org/10.1038/s41588-025-02285-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The use of single-cell RNA sequencing in clinical and translational research is limited by the challenge of identifying cell-type-specific, targetable molecular changes in individual patients and cross-species differences. Here we created an integrated single-cell kidney atlas including over 1 million cells from 140 samples, defining more than 70 conserved cell states in human and rodent models. We developed CellSpectra, a computational tool that quantifies changes in gene expression coordination across cellular functions, which we applied to kidney and lung cancer data. This tool powers our patient-level single-cell functional profiling report, which highlights cell-type-specific changes in the coordination of pathway gene expression in individuals. Our cross-species atlas facilitates the selection of a rodent model that closely reflects the cellular and pathway-level signatures observed in patient samples, advancing the application of single-cell methodologies in clinical precision medicine. Finally, using experimental models, we demonstrate how our informatics approach can be applied for the potential selection of suitable therapeutics. CellSpectra identifies functionally relevant changes in cellular pathways when analyzing single samples in comparison to a reference atlas, here applied to rodent models and patient samples of kidney disease.},
  archive      = {J_NG},
  author       = {Klötzer, Konstantin A. and Abedini, Amin and Li, Shen and Balzer, Michael S. and Liang, Xiujie and Levinsohn, Jonathan and Ha, Eunji and Dumoulin, Bernhard and Hogan, Jonathan J. and Quinn, Ghazal and Bloom, Roy D. and Schuller, Max and Eller, Kathrin and Halmos, Balazs and Zhang, Nancy R. and Susztak, Katalin},
  doi          = {10.1038/s41588-025-02285-0},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1922-1934},
  shortjournal = {Nat. Genet.},
  title        = {Analysis of individual patient pathway coordination in a cross-species single-cell kidney atlas},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deciphering state-dependent immune features from multi-layer omics data at single-cell resolution. <em>NG</em>, <em>57</em>(8), 1905-1921. (<a href='https://doi.org/10.1038/s41588-025-02266-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current molecular quantitative trait locus catalogs are mostly at bulk resolution and centered on Europeans. Here, we constructed an immune cell atlas with single-cell transcriptomics of &gt;1.5 million peripheral blood mononuclear cells, host genetics, plasma proteomics and gut metagenomics from 235 Japanese persons, including patients with coronavirus disease 2019 (COVID-19) and healthy individuals. We mapped germline genetic effects on gene expression within immune cell types and across cell states. We elucidated cell type- and context-specific human leukocyte antigen (HLA) and genome-wide associations with T and B cell receptor repertoires. Colocalization using dynamic genetic regulation provided better understanding of genome-wide association signals. Differential gene and protein expression analyses depicted cell type- and context-specific effects of polygenic risks. Various somatic mutations including mosaic chromosomal alterations, loss of Y chromosome and mitochondrial DNA (mtDNA) heteroplasmy were projected into single-cell resolution. We identified immune features specific to somatically mutated cells. Overall, immune cells are dynamically regulated in a cell state-dependent manner characterized with multiomic profiles. A single-cell multiomic immune cell atlas from 235 Japanese, including patients with COVID-19 and healthy individuals, linked with host genetics including germline and somatic mutation, plasma proteomics and metagenomics data reveals that immune cells are dynamically regulated in a cell state-dependent manner.},
  archive      = {J_NG},
  author       = {Edahiro, Ryuya and Sato, Go and Naito, Tatsuhiko and Shirai, Yuya and Saiki, Ryunosuke and Sonehara, Kyuto and Tomofuji, Yoshihiko and Yamamoto, Kenichi and Namba, Shinichi and Sasa, Noah and Nagao, Genta and Wang, Qingbo S. and Takahashi, Yugo and Hasegawa, Takanori and Kishikawa, Toshihiro and Suzuki, Ken and Liu, Yu-Chen and Motooka, Daisuke and Takuwa, Ayako and Tanaka, Hiromu and Azekawa, Shuhei and Namkoong, Ho and Koike, Ryuji and Kimura, Akinori and Imoto, Seiya and Miyano, Satoru and Kanai, Takanori and Fukunaga, Koichi and Uemura, Mamoru and Morita, Takayoshi and Kato, Yasuhiro and Hirata, Haruhiko and Takeda, Yoshito and Doki, Yuichiro and Eguchi, Hidetoshi and Okuzaki, Daisuke and Sakakibara, Shuhei and Ogawa, Seishi and Kumanogoh, Atsushi and Okada, Yukinori},
  doi          = {10.1038/s41588-025-02266-3},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1905-1921},
  shortjournal = {Nat. Genet.},
  title        = {Deciphering state-dependent immune features from multi-layer omics data at single-cell resolution},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-nucleus chromatin accessibility profiling identifies cell types and functional variants contributing to major depression. <em>NG</em>, <em>57</em>(8), 1890-1904. (<a href='https://doi.org/10.1038/s41588-025-02249-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic variants associated with major depressive disorder (MDD) are enriched in the regulatory genome. Here, we investigate gene-regulatory mechanisms underlying MDD compared to neurotypical controls by combining single-cell chromatin accessibility with gene expression in over 200,000 cells from the dorsolateral prefrontal cortex of 84 individuals. MDD-associated alterations in chromatin accessibility were prominent in deep-layer excitatory neurons characterized by transcription factor (TF) motif accessibility and binding of NR4A2, an activity-dependent TF reactive to stress. The same neurons were enriched for MDD-associated genetic variants, disrupting TF binding sites linked to genes that likely affect synaptic communication. Furthermore, a gray matter microglia cluster exhibited decreased accessibility in individuals with MDD at binding sites bound by TFs known to regulate immune homeostasis. Finally, we identified gene-regulatory effects of MDD-risk variants using sequence-based accessibility predictions, donor-specific genotypes and cell-based assays. These findings shed light on the cell types and regulatory mechanisms through which genetic variation may increase the risk of MDD. Integration of snATAC-seq and snRNA-seq data from brains of individuals with major depressive disorder identifies chromatin accessibility alterations and functional enrichment of risk variants in deep-layer excitatory neurons. Gray matter microglia in these individuals show decreased accessibility at sites bound by regulators of immune homeostasis.},
  archive      = {J_NG},
  author       = {Chawla, Anjali and Cakmakci, Doruk and Fiori, Laura M. and Zang, Wenmin and Maitra, Malosree and Yang, Jennie and Żurawek, Dariusz and Frosi, Gabriella and Rahimian, Reza and Mitsuhashi, Haruka and Davoli, Maria Antonietta and Denniston, Ryan and Chen, Gary Gang and Yerko, Volodymyr and Mash, Deborah and Girdhar, Kiran and Akbarian, Schahram and Mechawar, Naguib and Suderman, Matthew and Li, Yue and Nagy, Corina and Turecki, Gustavo},
  doi          = {10.1038/s41588-025-02249-4},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1890-1904},
  shortjournal = {Nat. Genet.},
  title        = {Single-nucleus chromatin accessibility profiling identifies cell types and functional variants contributing to major depression},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Improved multiancestry fine-mapping identifies cis-regulatory variants underlying molecular traits and disease risk. <em>NG</em>, <em>57</em>(8), 1881-1889. (<a href='https://doi.org/10.1038/s41588-025-02262-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiancestry statistical fine-mapping of cis-molecular quantitative trait loci (cis-molQTL) aims to improve the precision of distinguishing causal cis-molQTLs from tagging variants. Here we present the sum of shared single effects (SuShiE) model, which leverages linkage disequilibrium heterogeneity to improve fine-mapping precision, infer cross-ancestry effect size correlations and estimate ancestry-specific expression prediction weights. Through extensive simulations, we find that SuShiE consistently outperforms existing methods. We apply SuShiE to 36,907 molecular phenotypes including mRNA expression and protein levels from individuals of diverse ancestries in the TOPMed-MESA and GENOA studies. SuShiE fine-maps cis-molQTLs for 18.2% more genes compared with existing methods while prioritizing fewer variants and exhibiting greater functional enrichment. While SuShiE infers highly consistent cis-molQTL architectures across ancestries, it finds evidence of heterogeneity at genes with predicted loss-of-function intolerance. Lastly, using SuShiE-derived cis-molQTL effect sizes, we perform transcriptome- and proteome-wide association studies on six white blood cell-related traits in the All of Us biobank and identify 25.4% more genes compared with existing methods. Overall, SuShiE provides new insights into the cis-genetic architecture of molecular traits. SuShiE is a multiancestry fine-mapping method for molecular quantitative trait loci that leverages linkage disequilibrium heterogeneity to improve resolution, infer cross-ancestry effect size correlations and estimate ancestry-specific expression prediction weights.},
  archive      = {J_NG},
  author       = {Lu, Zeyun and Wang, Xinran and Carr, Matthew and Kim, Artem and Gazal, Steven and Mohammadi, Pejman and Wu, Lang and Pirruccello, James and Kachuri, Linda and Gusev, Alexander and Mancuso, Nicholas},
  doi          = {10.1038/s41588-025-02262-7},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1881-1889},
  shortjournal = {Nat. Genet.},
  title        = {Improved multiancestry fine-mapping identifies cis-regulatory variants underlying molecular traits and disease risk},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Germline genetic variation impacts clonal hematopoiesis landscape and progression to malignancy. <em>NG</em>, <em>57</em>(8), 1872-1880. (<a href='https://doi.org/10.1038/s41588-025-02250-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With age, clonal expansions occur pervasively across normal tissues yet only in rare instances lead to cancer, despite being driven by well-established cancer drivers. Characterization of the factors that influence clonal progression is needed to inform interventional approaches. Germline genetic variation influences cancer risk and shapes tumor mutational profile, but its influence on the mutational landscape of normal tissues is not well known. Here we studied the impact of germline genetic variation on clonal hematopoiesis (CH) in 731,835 individuals. We identified 22 new CH-predisposition genes, most of which predispose to CH driven by specific mutational events. CH-predisposition genes contribute to unique somatic landscapes, reflecting the influence of germline genetic backdrop on gene-specific CH fitness. Correspondingly, somatic–germline interactions influence the risk of CH progression to hematologic malignancies. These results demonstrate that germline genetic variation influences somatic evolution in the blood, findings that likely extend to other tissues. The relationship between pathogenic germline variation, clonal hematopoiesis (CH) and risk of hematologic malignancy is explored in 731,835 individuals across 6 cohorts. Carriers of variants in certain genes show distinct patterns of CH and increased risk of CH progression to malignancy.},
  archive      = {J_NG},
  author       = {Liu, Jie and Tran, Duc and Xue, Liying and Wiley, Brian J. and Vlasschaert, Caitlyn and Watson, Caroline J. and MacGregor, Hamish A. J. and Zong, Xiaoyu and Chan, Irenaeus C. C. and Das, Indraniel and Uddin, Md Mesbah and Niroula, Abhishek and Griffin, Gabriel and Ebert, Benjamin L. and Mack, Taralynn and Pershad, Yash and Sharber, Brian and Berger, Michael and Zehir, Ahmet and Ptashkin, Ryan and Levine, Ross L. and Papaemmanuil, Elli and Joseph, Vijai and Gao, Teng and Kemel, Yelena and Mandelker, Diana and Stopsack, Konrad H. and Pharoah, Paul D. P. and Mukherjee, Semanti and Ding, Li and Cao, Yin and Walter, Matthew J. and Blundell, Jamie R. and Chatterjee, Nilanjan and Offit, Kenneth and Godley, Lucy A. and Link, Daniel C. and Stadler, Zsofia K. and Bick, Alexander G. and Natarajan, Pradeep and Bolton, Kelly L.},
  doi          = {10.1038/s41588-025-02250-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1872-1880},
  shortjournal = {Nat. Genet.},
  title        = {Germline genetic variation impacts clonal hematopoiesis landscape and progression to malignancy},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A contextual genomic perspective on physical activity and its relationship to health, well being and illness. <em>NG</em>, <em>57</em>(8), 1860-1871. (<a href='https://doi.org/10.1038/s41588-025-02260-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Physical activity (PA) is one of the most fundamental traits in the animal kingdom, has pervasive health benefits, and is genetically influenced. Using data from the Million Veteran Program, we conducted genetic analyses of leisure, work and home-time PA. For leisure, we included 189,812 individuals of European ancestry (SNP-based heritability (h2) = 0.083 ± 0.005), 27,044 of African ancestry (h2 = 0.034 ± 0.017) and 10,263 of Latin American ancestry (h2 = 0.083 ± 0.036) in a cross-ancestry meta-analysis with UK Biobank data, identifying 70 lead variants. Leisure-time PA was genetically distinct from PA at home or work, with the latter two showing less health benefit with respect to health outcomes and lifespan. Mendelian randomization analyses showed a protective role of leisure-time PA against COVID-19 hospitalization (β = −0.067 ± 0.016; P = 2.8 × 10−5), and with other traits including cardiovascular and respiratory system diseases, metabolic traits and aging. These findings provide new insights into the biology of PA, showing specific causal health benefits of leisure-time PA. Genome-wide association meta-analyses of leisure, work and home-time physical activity (PA) from European, African and Latin American ancestries provide insights into the biology of PA and its relationship to health and diseases.},
  archive      = {J_NG},
  author       = {Galimberti, Marco and Levey, Daniel F. and Deak, Joseph D. and Adhikari, Keyrun and Overstreet, Cassie and Gupta, Priya and Nitin, Rachana and Zhou, Hang and Lake, Nicole J. and Harrington, Kelly M. and Djousse, Luc and Davis, Lea K. and Gaziano, J. Michael and Stein, Murray B. and Gelernter, Joel},
  doi          = {10.1038/s41588-025-02260-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1860-1871},
  shortjournal = {Nat. Genet.},
  title        = {A contextual genomic perspective on physical activity and its relationship to health, well being and illness},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Uncovering the multivariate genetic architecture of frailty with genomic structural equation modeling. <em>NG</em>, <em>57</em>(8), 1848-1859. (<a href='https://doi.org/10.1038/s41588-025-02269-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Frailty is a multifaceted clinical state associated with accelerated aging and adverse health outcomes. Informed etiological models of frailty hold promise for producing widespread health improvements across the aging population. Frailty is currently measured using aggregate scores, which obscure etiological pathways that are only relevant to subcomponents of frailty. Here we perform a multivariate genome-wide association study of the latent genetic architecture between 30 frailty deficits, which identifies 408 genomic risk loci. Our model includes a general factor of genetic overlap across all deficits, plus six new factors indexing a shared genetic signal across specific groups of deficits. We demonstrate the added clinical and etiological value of the six factors, including predicting frailty in external datasets, highlighting divergent genetic correlations with clinically relevant outcomes and uncovering unique underlying biology linked to aging. We show that nuanced models of frailty are key to understanding its causes and how it relates to worse health. Multivariate genome-wide association analyses of the latent genetic architecture of frailty identify one general factor of genetic overlap across all frailty deficits and six factors indexing a shared genetic signal across specific groups of deficits.},
  archive      = {J_NG},
  author       = {Foote, Isabelle F. and Flint, Jonny P. and Fürtjes, Anna E. and Lawrence, Jeremy M. and Mullin, Donncha S. and Fisk, John D. and Karakach, Tobias K. and Rutenberg, Andrew and Martin, Nicholas G. and Lupton, Michelle K. and Llewellyn, David J. and Ranson, Janice M. and Cox, Simon R. and Luciano, Michelle and Rockwood, Kenneth and Grotzinger, Andrew D.},
  doi          = {10.1038/s41588-025-02269-0},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1848-1859},
  shortjournal = {Nat. Genet.},
  title        = {Uncovering the multivariate genetic architecture of frailty with genomic structural equation modeling},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Large-scale genome-wide analyses of stuttering. <em>NG</em>, <em>57</em>(8), 1835-1847. (<a href='https://doi.org/10.1038/s41588-025-02267-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Developmental stuttering is a highly heritable, common speech condition characterized by prolongations, blocks and repetitions of speech. Although stuttering is highly heritable and enriched within families, the genetic architecture is largely understudied. We reasoned that there are both shared and distinct genetic variants impacting stuttering risk within sex and ancestry groups. To test this idea, we performed eight primary genome-wide association analyses of self-reported stuttering that were stratified by sex and ancestry, as well as secondary meta-analyses of more than one million individuals (99,776 cases and 1,023,243 controls), identifying 57 unique loci. We validated the genetic risk of self-reported stuttering in two independent datasets. We further show genetic similarity of stuttering with autism, depression and impaired musical rhythm across sexes, with follow-up analyses highlighting potentially causal relationships among these traits. Our findings provide well-powered insights into genetic factors underlying stuttering. Genome-wide analyses in over one million self-reported cases and controls identify genetic variants associated with stuttering and find genetic correlations with autism, depression and impaired musical rhythm, supporting a potential neurological basis for stuttering.},
  archive      = {J_NG},
  author       = {Polikowsky, Hannah G. and Scartozzi, Alyssa C. and Shaw, Douglas M. and Pruett, Dillon G. and Chen, Hung-Hsin and Petty, Lauren E. and Petty, Alexander S. and Lowther, Emily J. and Cho, Shu-Hsien and Yu, Yao and Mozaffari, Sahar and Avery, Christy L. and Harris, Kathleen Mullan and Gordon, Reyna L. and Beilby, Janet M. and Viljoen, Kathryn Z. and Jones, Robin M. and Huff, Chad D. and Highland, Heather M. and Kraft, Shelly Jo and Below, Jennifer E.},
  doi          = {10.1038/s41588-025-02267-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1835-1847},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale genome-wide analyses of stuttering},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How and when organisms edit their own genomes. <em>NG</em>, <em>57</em>(8), 1823-1834. (<a href='https://doi.org/10.1038/s41588-025-02230-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutations are often thought of as untargeted and non-adaptive, but in rare cases, organisms perform programmed, targeted and adaptive rearrangements of their own DNA sequences. Notable examples include the somatic diversification of immunoglobulin genes, which is the foundation of the vertebrate immune system, and natural CRISPR spacer arrays in bacteria, which recognize and cleave foreign DNA. These systems, along with a dozen known analogs scattered across the tree of life, often underlie critical biological functions, particularly in host–pathogen conflicts. In this Review, we compare the mechanisms by which organisms edit their own genomes. We show that superficially dissimilar editing systems often rely on surprisingly similar genetic mechanisms, regardless of function or taxon. Finally, we argue that the recurrence of editing in host–pathogen conflicts and the bias to a handful of well-studied organisms strongly suggest that new editing systems will be found in understudied pathogens and their hosts. This Review describes the changes that some organisms make to their own DNA sequences, linking many to common genetic mechanisms built around canonical DNA repair and to a set of functional circumstances that often involve host–pathogen conflicts.},
  archive      = {J_NG},
  author       = {Hanlon, Vincent C. T. and Cagan, Alex and Eves-van den Akker, Sebastian},
  doi          = {10.1038/s41588-025-02230-1},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1823-1834},
  shortjournal = {Nat. Genet.},
  title        = {How and when organisms edit their own genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Therapeutic strategies for fragile x syndrome and implications for other gene-silencing disorders. <em>NG</em>, <em>57</em>(8), 1812-1822. (<a href='https://doi.org/10.1038/s41588-025-02255-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene-silencing disorders, of which fragile X syndrome (FXS) is the most prevalent, are diseases caused by a blockade of gene transcription, usually due to DNA hypermethylation. FXS is a common form of inherited intellectual disability and autism. Unlike most hereditary diseases driven by mutations within the protein-coding region of a gene, FXS is caused by a trinucleotide expansion in the 5′-untranslated region of the FMR1 gene, leading to hypermethylation and transcriptional silencing. Modeling FXS with human pluripotent stem cells offers a clinically relevant platform to study disease mechanisms and explore potential therapies through reactivating FMR1 expression by genetic and epigenetic means or through drug screening. This Perspective reviews the various cellular models and therapeutic strategies proposed over the past decade, highlighting their potential to advance the treatment of FXS. We also discuss the benefits and challenges of gene activation therapies, drawing comparisons with other gene-silencing disorders, including imprinting diseases and X-linked disorders. This Perspective discusses therapeutic strategies to reactivate the silenced gene in fragile X syndrome by genetic and epigenetic means and drug screening and their relevance for other gene-silencing diseases, such as imprinting or X-linked disorders.},
  archive      = {J_NG},
  author       = {Gadban, Aseel and Gunapala, Keith M. and Taylor, Verdon and Benvenisty, Nissim},
  doi          = {10.1038/s41588-025-02255-6},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1812-1822},
  shortjournal = {Nat. Genet.},
  title        = {Therapeutic strategies for fragile x syndrome and implications for other gene-silencing disorders},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tumor aneuploidy as a prognostic and predictive biomarker in immune checkpoint blockade. <em>NG</em>, <em>57</em>(8), 1802-1811. (<a href='https://doi.org/10.1038/s41588-025-02226-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aneuploidy, a hallmark of cancer characterized by chromosome imbalances, drives tumorigenesis and facilitates cancer immune evasion. While high tumor aneuploidy is linked to immune checkpoint blockade (ICB) resistance and poor prognosis, evidence suggests that this resistance can be overcome through treatment intensification, for example, with the addition of ablative radiotherapy to ICB. In this Perspective, we argue that the predictive value of aneuploidy complements established biomarkers, such as tumor mutational burden (TMB) or programmed death ligand 1 (PD-L1) expression. We review contemporary methods for quantifying aneuploidy, explore novel approaches that target mitotic vulnerabilities in aneuploid tumors and highlight potential areas where aneuploidy-based stratification could be incorporated into ICB-based treatment paradigms across early-stage, locally advanced and metastatic cancers. Prospective trials incorporating aneuploidy-based stratification will be essential to validate its role in personalized cancer therapy. Aneuploidy, a defining feature of cancer, is linked to immune evasion and therapeutic resistance. This Perspective reviews approaches to quantify tumor aneuploidy and evaluates its potential as a biomarker for guiding immune checkpoint blockade.},
  archive      = {J_NG},
  author       = {Huang, Daniel and Spurr, Liam F. and Weichselbaum, Ralph R. and Pitroda, Sean P.},
  doi          = {10.1038/s41588-025-02226-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1802-1811},
  shortjournal = {Nat. Genet.},
  title        = {Tumor aneuploidy as a prognostic and predictive biomarker in immune checkpoint blockade},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SBRR1-R variant regulates sheath blight resistance in rice. <em>NG</em>, <em>57</em>(8), 1800-1801. (<a href='https://doi.org/10.1038/s41588-025-02282-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rice sheath blight is a major fungal disease caused by the necrotrophic pathogen Rhizoctonia solani. We identified a natural SBRR1-R allele that is activated by the bHLH57 transcription factor, leading to increased expression of chitinase antifungal protein genes that confer resistance to sheath blight.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02282-3},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1800-1801},
  shortjournal = {Nat. Genet.},
  title        = {SBRR1-R variant regulates sheath blight resistance in rice},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). G-quadruplexes define promoter identity in mammals. <em>NG</em>, <em>57</em>(8), 1798-1799. (<a href='https://doi.org/10.1038/s41588-025-02287-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We show that alternative DNA structures, called G-quadruplexes, behave as elements of eukaryotic promoters. Our data suggest that they function to support intrinsic nucleosome exclusion and to facilitate transcriptional pause release by RNA polymerase II.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02287-y},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1798-1799},
  shortjournal = {Nat. Genet.},
  title        = {G-quadruplexes define promoter identity in mammals},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial and temporal dynamics in progressive gallbladder cancer. <em>NG</em>, <em>57</em>(8), 1796-1797. (<a href='https://doi.org/10.1038/s41588-025-02232-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How primary gallbladder tumors evolve to acquire immunosuppressive and pro-metastatic capabilities remains unclear. A study using single-cell RNA sequencing coupled with spatial profiling and functional experiments has pinpointed mechanisms of immune-mediated tumor cell reprogramming, fostering disease progression.},
  archive      = {J_NG},
  author       = {O’Rourke, Colm J. and Andersen, Jesper B.},
  doi          = {10.1038/s41588-025-02232-z},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1796-1797},
  shortjournal = {Nat. Genet.},
  title        = {Spatial and temporal dynamics in progressive gallbladder cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Turning gene expression up and down. <em>NG</em>, <em>57</em>(8), 1795. (<a href='https://doi.org/10.1038/s41588-025-02309-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Brandt, Margot},
  doi          = {10.1038/s41588-025-02309-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1795},
  shortjournal = {Nat. Genet.},
  title        = {Turning gene expression up and down},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Whole-genome doubling in ovarian cancer. <em>NG</em>, <em>57</em>(8), 1795. (<a href='https://doi.org/10.1038/s41588-025-02310-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02310-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1795},
  shortjournal = {Nat. Genet.},
  title        = {Whole-genome doubling in ovarian cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Long-range enhancers rely on LHX motifs. <em>NG</em>, <em>57</em>(8), 1795. (<a href='https://doi.org/10.1038/s41588-025-02311-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-025-02311-1},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1795},
  shortjournal = {Nat. Genet.},
  title        = {Long-range enhancers rely on LHX motifs},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Somatic hypermutation at a multiway chromatin hub. <em>NG</em>, <em>57</em>(8), 1795. (<a href='https://doi.org/10.1038/s41588-025-02312-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gross, Petra},
  doi          = {10.1038/s41588-025-02312-0},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1795},
  shortjournal = {Nat. Genet.},
  title        = {Somatic hypermutation at a multiway chromatin hub},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Why genomic diversity should not be framed by census alone. <em>NG</em>, <em>57</em>(8), 1793-1794. (<a href='https://doi.org/10.1038/s41588-025-02272-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Corpas, Manuel and Guio, Heinner and Lopez-Correa, Catalina and Fatumo, Segun},
  doi          = {10.1038/s41588-025-02272-5},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1793-1794},
  shortjournal = {Nat. Genet.},
  title        = {Why genomic diversity should not be framed by census alone},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: Analysis of R-loop forming regions identifies RNU2-2 and RNU5B-1 as neurodevelopmental disorder genes. <em>NG</em>, <em>57</em>(7), 1792. (<a href='https://doi.org/10.1038/s41588-025-02274-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Jackson, Adam and Thaker, Nishi and Blakes, Alexander and Rice, Gillian and Griffiths-Jones, Sam and Balasubramanian, Meena and Campbell, Jennifer and Shannon, Nora and Choi, Jungmin and Hong, Juhyeon and Hunt, David and de Burca, Anna and Kim, Soo Yeon and Kim, Taekeun and Lee, Seungbok and Redman, Melody and Rius, Rocio and Simons, Cas and Tan, Tiong Yang and Ellingford, Jamie and O’Keefe, Raymond T. and Chae, Jong Hee and Banka, Siddharth},
  doi          = {10.1038/s41588-025-02274-3},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1792},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Analysis of R-loop forming regions identifies RNU2-2 and RNU5B-1 as neurodevelopmental disorder genes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: GWAS of multiple neuropathology endophenotypes identifies new risk loci and provides insights into the genetic risk of dementia. <em>NG</em>, <em>57</em>(7), 1791. (<a href='https://doi.org/10.1038/s41588-024-02046-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Shade, Lincoln M. P. and Katsumata, Yuriko and Abner, Erin L. and Aung, Khine Zin and Claas, Steven A. and Qiao, Qi and Heberle, Bernardo Aguzzoli and Brandon, J. Anthony and Page, Madeline L. and Hohman, Timothy J. and Mukherjee, Shubhabrata and Mayeux, Richard P. and Farrer, Lindsay A. and Schellenberg, Gerard D. and Haines, Jonathan L. and Kukull, Walter A. and Nho, Kwangsik and Saykin, Andrew J. and Bennett, David A. and Schneider, Julie A. and Ebbert, Mark T. W. and Nelson, Peter T. and Fardo, David W.},
  doi          = {10.1038/s41588-024-02046-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1791},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: GWAS of multiple neuropathology endophenotypes identifies new risk loci and provides insights into the genetic risk of dementia},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Next-generation phenotyping integrated in a national framework for patients with ultrarare disorders improves genetic diagnostics and yields new molecular findings. <em>NG</em>, <em>57</em>(7), 1790-1791. (<a href='https://doi.org/10.1038/s41588-025-02271-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Schmidt, Axel and Danyel, Magdalena and Grundmann, Kathrin and Brunet, Theresa and Klinkhammer, Hannah and Hsieh, Tzung-Chien and Engels, Hartmut and Peters, Sophia and Knaus, Alexej and Moosa, Shahida and Averdunk, Luisa and Boschann, Felix and Sczakiel, Henrike Lisa and Schwartzmann, Sarina and Mensah, Martin Atta and Pantel, Jean Tori and Holtgrewe, Manuel and Bösch, Annemarie and Weiß, Claudia and Weinhold, Natalie and Suter, Aude-Annick and Stoltenburg, Corinna and Neugebauer, Julia and Kallinich, Tillmann and Kaindl, Angela M. and Holzhauer, Susanne and Bührer, Christoph and Bufler, Philip and Kornak, Uwe and Ott, Claus-Eric and Schülke, Markus and Nguyen, Hoa Huu Phuc and Hoffjan, Sabine and Grasemann, Corinna and Rothoeft, Tobias and Brinkmann, Folke and Matar, Nora and Sivalingam, Sugirthan and Perne, Claudia and Mangold, Elisabeth and Kreiss, Martina and Cremer, Kirsten and Betz, Regina C. and Mücke, Martin and Grigull, Lorenz and Klockgether, Thomas and Spier, Isabel and Heimbach, André and Bender, Tim and Brand, Fabian and Stieber, Christiane and Morawiec, Alexandra Marzena and Karakostas, Pantelis and Schäfer, Valentin S. and Bernsen, Sarah and Weydt, Patrick and Castro-Gomez, Sergio and Aziz, Ahmad and Grobe-Einsler, Marcus and Kimmich, Okka and Kobeleva, Xenia and Önder, Demet and Lesmann, Hellen and Kumar, Sheetal and Tacik, Pawel and Bhasin, Meghna Ahuja and Incardona, Pietro and Lee-Kirsch, Min Ae and Berner, Reinhard and Schuetz, Catharina and Körholz, Julia and Kretschmer, Tanita and Di Donato, Nataliya and Schröck, Evelin and Heinen, André and Reuner, Ulrike and Hanßke, Amalia-Mihaela and Kaiser, Frank J. and Manka, Eva and Munteanu, Martin and Kuechler, Alma and Cordula, Kiewert and Hirtz, Raphael and Schlapakow, Elena and Schlein, Christian and Lisfeld, Jasmin and Kubisch, Christian and Herget, Theresia and Hempel, Maja and Weiler-Normann, Christina and Ullrich, Kurt and Schramm, Christoph and Rudolph, Cornelia and Rillig, Franziska and Groffmann, Maximilian and Muntau, Ania and Tibelius, Alexandra and Schwaibold, Eva M. C. and Schaaf, Christian P. and Zawada, Michal and Kaufmann, Lilian and Hinderhofer, Katrin and Okun, Pamela M. and Kotzaeridou, Urania and Hoffmann, Georg F. and Choukair, Daniela and Bettendorf, Markus and Spielmann, Malte and Ripke, Annekatrin and Pauly, Martje and Münchau, Alexander and Lohmann, Katja and Hüning, Irina and Hanker, Britta and Bäumer, Tobias and Herzog, Rebecca and Hellenbroich, Yorck and Westphal, Dominik S. and Strom, Tim and Kovacs, Reka and Riedhammer, Korbinian M. and Mayerhanser, Katharina and Graf, Elisabeth and Brugger, Melanie and Hoefele, Julia and Oexle, Konrad and Mirza-Schreiber, Nazanin and Berutti, Riccardo and Schatz, Ulrich and Krenn, Martin and Makowski, Christine and Weigand, Heike and Schröder, Sebastian and Rohlfs, Meino and Vill, Katharina and Hauck, Fabian and Borggraefe, Ingo and Müller-Felber, Wolfgang and Kurth, Ingo and Elbracht, Miriam and Knopp, Cordula and Begemann, Matthias and Kraft, Florian and Lemke, Johannes R. and Hentschel, Julia and Platzer, Konrad and Strehlow, Vincent and Abou Jamra, Rami and Kehrer, Martin and Demidov, German and Beck-Wödl, Stefanie and Graessner, Holm and Sturm, Marc and Zeltner, Lena and Schöls, Ludger J. and Magg, Janine and Bevot, Andrea and Kehrer, Christiane and Kaiser, Nadja and Turro, Ernest and Horn, Denise and Grüters-Kieslich, Annette and Klein, Christoph and Mundlos, Stefan and Nöthen, Markus and Riess, Olaf and Meitinger, Thomas and Krude, Heiko and Krawitz, Peter M. and Haack, Tobias and Ehmke, Nadja and Wagner, Matias},
  doi          = {10.1038/s41588-025-02271-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1790-1791},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Next-generation phenotyping integrated in a national framework for patients with ultrarare disorders improves genetic diagnostics and yields new molecular findings},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Comprehensive molecular profiling of multiple myeloma identifies refined copy number and expression subtypes. <em>NG</em>, <em>57</em>(7), 1789. (<a href='https://doi.org/10.1038/s41588-025-02251-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Skerget, Sheri and Penaherrera, Daniel and Chari, Ajai and Jagannath, Sundar and Siegel, David S. and Vij, Ravi and Orloff, Gregory and Jakubowiak, Andrzej and Niesvizky, Ruben and Liles, Darla and Berdeja, Jesus and Levy, Moshe and Wolf, Jeffrey and Usmani, Saad Z. and Christofferson, Austin W. and Nasser, Sara and Aldrich, Jessica L. and Legendre, Christophe and Benard, Brooks and Miller, Chase and Turner, Bryce and Kurdoglu, Ahmet and Washington, Megan and Yellapantula, Venkata and Adkins, Jonathan R. and Cuyugan, Lori and Boateng, Martin and Helland, Adrienne and Kyman, Shari and McDonald, Jackie and Reiman, Rebecca and Stephenson, Kristi and Tassone, Erica and Blanski, Alex and Livermore, Brianne and Kirchhoff, Meghan and Rohrer, Daniel C. and D’Agostino, Mattia and Gambella, Manuela and Collison, Kimberly and Stumph, Jennifer and Kidd, Pam and Donnelly, Andrea and Zaugg, Barbara and Toone, Maureen and McBride, Kyle and DeRome, Mary and Rogers, Jennifer and Craig, David and Liang, Winnie S. and Gutierrez, Norma C. and Jewell, Scott D. and Carpten, John and Anderson, Kenneth C. and Cho, Hearn Jay and Auclair, Daniel and Lonial, Sagar and Keats, Jonathan J.},
  doi          = {10.1038/s41588-025-02251-w},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1789},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Comprehensive molecular profiling of multiple myeloma identifies refined copy number and expression subtypes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: GWAS of multiple neuropathology endophenotypes identifies new risk loci and provides insights into the genetic risk of dementia. <em>NG</em>, <em>57</em>(7), 1788. (<a href='https://doi.org/10.1038/s41588-024-02045-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Shade, Lincoln M. P. and Katsumata, Yuriko and Abner, Erin L. and Aung, Khine Zin and Claas, Steven A. and Qiao, Qi and Heberle, Bernardo Aguzzoli and Brandon, J. Anthony and Page, Madeline L. and Hohman, Timothy J. and Mukherjee, Shubhabrata and Mayeux, Richard P. and Farrer, Lindsay A. and Schellenberg, Gerard D. and Haines, Jonathan L. and Kukull, Walter A. and Nho, Kwangsik and Saykin, Andrew J. and Bennett, David A. and Schneider, Julie A. and Ebbert, Mark T. W. and Nelson, Peter T. and Fardo, David W.},
  doi          = {10.1038/s41588-024-02045-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1788},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: GWAS of multiple neuropathology endophenotypes identifies new risk loci and provides insights into the genetic risk of dementia},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: The genomic landscape of gene-level structural variations in japanese and global soybean glycine max cultivars. <em>NG</em>, <em>57</em>(7), 1787. (<a href='https://doi.org/10.1038/s41588-025-02256-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Yano, Ryoichi and Li, Feng and Hiraga, Susumu and Takeshima, Ryoma and Kobayashi, Michie and Toda, Kyoko and Umehara, Yosuke and Kajiya-Kanegae, Hiromi and Iwata, Hiroyoshi and Kaga, Akito and Ishimoto, Masao},
  doi          = {10.1038/s41588-025-02256-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1787},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: The genomic landscape of gene-level structural variations in japanese and global soybean glycine max cultivars},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: A likelihood-based framework for demographic inference from genealogical trees. <em>NG</em>, <em>57</em>(7), 1787. (<a href='https://doi.org/10.1038/s41588-025-02257-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fan, Caoqi and Cahoon, Jordan L. and Dinh, Bryan L. and Ortega-Del Vecchyo, Diego and Huber, Christian D. and Edge, Michael D. and Mancuso, Nicholas and Chiang, Charleston W. K.},
  doi          = {10.1038/s41588-025-02257-4},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1787},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: A likelihood-based framework for demographic inference from genealogical trees},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell micro-C profiles 3D genome structures at high resolution and characterizes multi-enhancer hubs. <em>NG</em>, <em>57</em>(7), 1777-1786. (<a href='https://doi.org/10.1038/s41588-025-02247-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In animal genomes, regulatory DNA elements called enhancers govern precise spatiotemporal gene expression patterns in specific cell types. However, the spatial organization of enhancers within the nucleus to regulate target genes remains poorly understood. Here we report single-cell Micro-C (scMicro-C), a micrococcal nuclease-based three-dimensional (3D) genome mapping technique with an improved spatial resolution of 5 kb, and identified a specialized 3D enhancer structure termed ‘promoter–enhancer stripes (PESs)’, connecting a gene’s promoter to multiple enhancers. PES are formed by cohesin-mediated loop extrusion, which potentially brings multiple enhancers to the promoter. Further, we observed the prevalence of multi-enhancer hubs on genes with PES within single-cell 3D genome structures, wherein multiple enhancers form a spatial cluster in association with the gene promoter. Through its improved resolution, scMicro-C elucidates how enhancers are spatially coordinated to control genes. scMicro-C is a new method that provides high-resolution maps of the 3D genome. scMicro-C identifies structures called ‘promoter stripes’, which link a gene promoter to multiple downstream enhancers.},
  archive      = {J_NG},
  author       = {Wu, Honggui and Zhang, Jiankun and Tan, Longzhi and Xie, Xiaoliang Sunney},
  doi          = {10.1038/s41588-025-02247-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1777-1786},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell micro-C profiles 3D genome structures at high resolution and characterizes multi-enhancer hubs},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enhancer adoption by an LTR retrotransposon generates viral-like particles, causing developmental limb phenotypes. <em>NG</em>, <em>57</em>(7), 1766-1776. (<a href='https://doi.org/10.1038/s41588-025-02248-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transposable elements (TEs) are scattered across mammalian genomes. Silencing of TEs prevents harmful effects caused by either global activation leading to genome instability or insertional mutations disturbing gene transcription. However, whether the activation of a TE can cause disease without directly affecting gene expression is largely unknown. Here we show that a TE insertion can adopt nearby regulatory activity, resulting in the production of cell-type-specific viral-like particles (VLPs) that affect embryo formation. Failure to silence an LTR retrotransposon inserted upstream of the Fgf8 gene results in their co-expression during mouse development. VLP assembly in the Fgf8-expressing cells of the developing limb triggers apoptotic cell death, resulting in a limb malformation resembling human ectrodactyly. The phenotype can be rescued by mutating the retrotransposon coding sequence, thus preventing its full endogenous retroviral cycle. Our findings illustrate that TE insertions can be incorporated into the local genomic regulatory landscape and that VLP production in post-implantation embryos can cause developmental defects. Activation of an LTR retrotransposon inserted upstream of the Fgf8 gene produces viral-like particles in the mouse developing limb, triggering apoptosis and causing limb malformation. This phenotype can be rescued by mutations in the retrotransposon coding sequence.},
  archive      = {J_NG},
  author       = {Glaser, Juliane and Cova, Giulia and Fauler, Beatrix and Prada-Medina, Cesar A. and Stanislas, Virginie and Phan, Mai H. Q. and Schöpflin, Robert and Aktas, Yasmin and Franke, Martin and Andrey, Guillaume and Bartzoka, Natalia and Paliou, Christina and Laupert, Verena and Chan, Wing-Lee and Wittler, Lars and Mielke, Thorsten and Mundlos, Stefan},
  doi          = {10.1038/s41588-025-02248-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1766-1776},
  shortjournal = {Nat. Genet.},
  title        = {Enhancer adoption by an LTR retrotransposon generates viral-like particles, causing developmental limb phenotypes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Disruption of TAD hierarchy promotes LTR co-option in cancer. <em>NG</em>, <em>57</em>(7), 1754-1765. (<a href='https://doi.org/10.1038/s41588-025-02239-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transposable elements (TEs) are abundant in the human genome, and they provide the source for genetic and functional diversity. Previous studies have suggested that TEs are repressed by DNA methylation and chromatin modifications. Here through integrating transcriptome and 3D genome architecture studies, we showed that haploinsufficient loss of NIPBL selectively activates alternative promoters (altPs) at the long terminal repeats (LTRs) of the TE subclasses. This activation occurs through the reorganization of topologically associating domain (TAD) hierarchical structures and the recruitment of proximal enhancers. These observations indicate that TAD hierarchy restricts transcriptional activation of LTRs that already possess open chromatin features. Perturbation of hierarchical chromatin topology can lead to co-option of LTRs as functional altPs, driving aberrant transcriptional activation of oncogenes. These data uncovered a new layer of regulatory mechanisms of TE expression and posit TAD hierarchy dysregulation as a new mechanism for altP-mediated oncogene activation and transcriptional diversity in cancer. NIPBL perturbation activates long terminal repeat (LTR)-derived alternative promoters due to reorganization of chromatin’s hierarchical structure, leading to LTR co-option and oncogene activation in melanoma cell lines.},
  archive      = {J_NG},
  author       = {Wong, Elissa W. P. and Sahin, Merve and Yang, Rui and Lee, UkJin and Li, Dan and Zhan, Yingqian A. and Misra, Rohan and Tomas, Fanny and Alomran, Nawaf and Polyzos, Alexander and Lee, Cindy J. and Trieu, Tuan and Martinez-Fundichely, Alexander and Wiesner, Thomas and Rosowicz, Andrew and Cheng, Shuyuan and Liu, Christina and Lallo, Morgan and Shoushtari, Alexander N. and Merghoub, Taha and Hamard, Pierre-Jacques and Koche, Richard and Khurana, Ekta and Apostolou, Effie and Zheng, Deyou and Chen, Yu and Leslie, Christina S. and Chi, Ping},
  doi          = {10.1038/s41588-025-02239-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1754-1765},
  shortjournal = {Nat. Genet.},
  title        = {Disruption of TAD hierarchy promotes LTR co-option in cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell and spatial transcriptomics of stricturing crohn’s disease highlights a fibrosis-associated network. <em>NG</em>, <em>57</em>(7), 1742-1753. (<a href='https://doi.org/10.1038/s41588-025-02225-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fibrosis is a major complication of Crohn’s disease (CD) marked by excess deposition of extracellular matrix, leading to stricturing and functional impairment. As mechanistic characterization and therapeutic options are lacking, we paired single-cell and spatial transcriptomics in 61 samples from 21 patients with CD and 10 patients without inflammatory bowel disease (IBD). Intestinal strictures were characterized by increased immune cells, including IgG+ plasma cells, CCR7-hi CD4+ T cells and inflammatory fibroblasts. Spatial transcriptomics showed that key subsets colocalize within diseased tissues and identified additional populations such as interstitial cells of Cajal and enteric neurons. Furthermore, we mapped gene expression onto intestinal biogeography, finding that known genetic risk loci are enriched within discrete spatial modules, defined by the presence of inflammatory fibroblasts and lymphoid follicles. Altogether, our datasets chart the key transcriptomic and cellular networks in stricturing CD and highlight the spatial organization of multicellular genetic risk factors. Single-cell and spatial transcriptomics analysis of stricturing Crohn’s disease highlights cellular heterogeneity in fibrotic tissue and identifies neighborhoods of fibroblasts and immune cells contributing to this severe disease complication.},
  archive      = {J_NG},
  author       = {Kong, Lingjia and Subramanian, Sathish and Segerstolpe, Åsa and Tran, Vy and Shih, Angela R. and Carter, Grace T. and Kunitake, Hiroko and Twardus, Shaina W. and Li, Jasmine and Gandhi, Shivam and Kaper, Marco E. and Cauley, Christy and Chen, Eric J. and Porter, Caroline B. M. and Delorey, Toni M. and Bordeianou, Liliana and Ricciardi, Rocco and Ananthakrishnan, Ashwin N. and Lau, Helena and Graham, Daniel B. and Hodin, Richard and Deguine, Jacques and Smillie, Christopher S. and Xavier, Ramnik J.},
  doi          = {10.1038/s41588-025-02225-y},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1742-1753},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell and spatial transcriptomics of stricturing crohn’s disease highlights a fibrosis-associated network},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A scalable gut epithelial organoid model reveals the genome-wide colonization landscape of a human-adapted pathogen. <em>NG</em>, <em>57</em>(7), 1730-1741. (<a href='https://doi.org/10.1038/s41588-025-02218-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studying the pathogenesis of human-adapted microorganisms is challenging, since small animal models often fail to recapitulate human physiology. Hence, the comprehensive genetic and regulatory circuits driving the infection process of principal human pathogens such as Shigella flexneri remain to be defined. We combined large-scale Shigella infections of enteroids and colonoids with transposon-directed insertion sequencing and Bayesian statistical modeling to address infection bottlenecks, thereby establishing the comprehensive genome-wide map of Shigella genes required to infect human intestinal epithelium. This revealed the Shigella virulence effectors essential for epithelial cell colonization across geometries and intestinal segments, identified over 100 chromosomal genes involved in the process and uncovered a post-transcriptional mechanism whereby tRNA-modification enzymes and differential codon usage exert global control of a bacterial virulence program. Our findings provide a broadly applicable framework for combining advanced organotypic tissue culture with functional genomics and computational tools to map human–microorganism interactions at scale. A genome-wide screen using human gut epithelial organoids combined with transposon-directed insertion sequencing identifies over 100 Shigella flexneri genes required for epithelial colonization.},
  archive      = {J_NG},
  author       = {Di Martino, Maria Letizia and Jenniches, Laura and Bhetwal, Anjeela and Eriksson, Jens and Lopes, Ana C. C. and Ntokaki, Angelika and Pasqua, Martina and Sundbom, Magnus and Skogar, Martin and Graf, Wilhelm and Webb, Dominic-Luc and Hellström, Per M. and Mateus, André and Barquist, Lars and Sellin, Mikael E.},
  doi          = {10.1038/s41588-025-02218-x},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1730-1741},
  shortjournal = {Nat. Genet.},
  title        = {A scalable gut epithelial organoid model reveals the genome-wide colonization landscape of a human-adapted pathogen},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Detecting and quantifying clonal selection in somatic stem cells. <em>NG</em>, <em>57</em>(7), 1718-1729. (<a href='https://doi.org/10.1038/s41588-025-02217-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As DNA variants accumulate in somatic stem cells, become selected or evolve neutrally, they may ultimately alter tissue function. When, and how, selection occurs in homeostatic tissues is incompletely understood. Here, we introduce SCIFER, a scalable method that identifies selection in an individual tissue, without requiring knowledge of the driver event. SCIFER also infers self-renewal and mutation dynamics of the tissue’s stem cells, and the size and age of selected clones. Probing bulk whole-genome sequencing data of nonmalignant human bone marrow and brain, we detected pervasive selection in both tissues. Selected clones in hematopoiesis, with or without known drivers, were initiated uniformly across life. In the brain, we found pre-malignant clones with glioma-initiating mutations and clones without known drivers. In contrast to hematopoiesis, selected clones in the brain originated preferentially from childhood to young adulthood. SCIFER is broadly applicable to renewing somatic tissues to detect and quantify selection. SCIFER detects clonal selection in whole-genome sequencing data using a population genetics model. Applied to a range of somatic tissues, SCIFER quantifies stem cell dynamics and infers clonal ages and sizes without requiring knowledge of driver events.},
  archive      = {J_NG},
  author       = {Körber, Verena and Jakobsen, Niels Asger and Ansari-Pour, Naser and Moore, Rachel and Claudino, Nina and Metzner, Marlen and Thielecke, Eva and Esau, Franziska and Usukhbayar, Batchimeg and Salazar, Mirian Angulo and Newman, Simon and Kendrick, Benjamin J. L. and Taylor, Adrian H. and Afinowi-Luitz, Rasheed and Gundle, Roger and Watkins, Bridget and Wheway, Kim and Beazley, Debra and Dakin, Stephanie G. and Palmer, Antony and Carr, Andrew J. and Vyas, Paresh and Höfer, Thomas},
  doi          = {10.1038/s41588-025-02217-y},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1718-1729},
  shortjournal = {Nat. Genet.},
  title        = {Detecting and quantifying clonal selection in somatic stem cells},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicting resistance to chemotherapy using chromosomal instability signatures. <em>NG</em>, <em>57</em>(7), 1708-1717. (<a href='https://doi.org/10.1038/s41588-025-02233-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chemotherapies are often given without precision biomarkers, exposing patients to toxic side effects without guaranteed benefit. Here we present chromosomal instability signature biomarkers that identify resistance to platinum-, taxane- and anthracycline-based treatments using a single genomic test. In retrospectively emulated randomized-control biomarker clinical trials using real-world cohorts (n = 840), predicted resistant patients had elevated treatment failure risk for taxane (hazard ratio (HR) of 7.44) and anthracycline (HR of 1.88) in ovarian, taxane (HR of 3.98) and anthracycline (HR of 3.69) in metastatic breast and taxane (HR of 5.46) in metastatic prostate. Nonrandomized emulations showed predictive capacity for platinum resistance in ovarian (HR of 1.46) and anthracycline in sarcoma (HR of 3.59). We demonstrate feasibility using whole-genome sequencing, capture-panel sequencing and cell-free DNA. Our findings highlight the clinical value of chromosomal instability signatures in predicting resistance to chemotherapies across multiple cancer types, with the potential to transform the one-size-fits-all chemotherapy approach into precise, tailored treatment. Here the authors show that chromosomal instability signatures can predict resistance to anthracycline-, taxane- and platinum-based chemotherapeutics in breast, ovarian and prostate cancer and sarcoma. Validation is performed through emulation of phase 2 and 3 clinical trials using real-world data.},
  archive      = {J_NG},
  author       = {Thompson, Joe Sneath and Madrid, Laura and Hernando, Barbara and Sauer, Carolin M. and Vias, Maria and Escobar-Rey, Maria and Leung, Wing-Kit and Garcia-Lopez, Diego and Huckstep, Jamie and Sekowska, Magdalena and Hosking, Karen and Jimenez-Linan, Mercedes and Reinius, Marika A. V. and Roy, Abhipsa and Abdulle, Omar and Pangonyte, Justina and Dobson, Harry and Cullen, Amy E. and De Silva, Dilrini and Gómez-Sánchez, David and Torres, Marina and Fernández-Sanromán, Ángel and Sanders, Deborah and Martins, Filipe Correia and Funingana, Ionut-Gabriel and Codacci-Pisanelli, Giovanni and Quintela-Fandino, Miguel and Markowetz, Florian and Yip, Jason and Brenton, James D. and Piskorz, Anna M. and Macintyre, Geoff},
  doi          = {10.1038/s41588-025-02233-y},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1708-1717},
  shortjournal = {Nat. Genet.},
  title        = {Predicting resistance to chemotherapy using chromosomal instability signatures},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Clonal evolution of hematopoietic stem cells after autologous stem cell transplantation. <em>NG</em>, <em>57</em>(7), 1695-1707. (<a href='https://doi.org/10.1038/s41588-025-02235-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The impact of exogenous stressors, such as cancer chemotherapies, on the genomic integrity and clonal dynamics of normal hematopoiesis is not well defined. We conducted whole-genome sequencing on 1,276 single-cell-derived hematopoietic stem and progenitor cell (HSPC) colonies from ten patients with multiple myeloma treated with chemotherapies and six normal donors. Melphalan treatment significantly increased the mutational burden, producing a distinctive mutation signature, whereas other chemotherapeutic agents had minimal effects. Consequently, the clonal diversity and architecture of post-treatment HSPCs resemble those observed in normal elderly individuals, particularly through the progression of oligoclonal hematopoiesis, thereby suggesting that chemotherapy accelerates clonal aging. Integrated phylogenetic analysis of matched therapy-related myeloid neoplasm samples traced their clonal origin to a single-HSPC clone among multiple competing clones, supporting a model of oligoclonal to monoclonal transformation. These findings underscore the need for further systematic research on the long-term hematological consequences of cancer chemotherapy. Analysis of whole-genome sequencing data from hematopoietic stem and progenitor cells taken from patients with myeloma shows how treatment shapes clonal architecture and sheds light on the evolution of treatment-related myeloid neoplasms.},
  archive      = {J_NG},
  author       = {Uryu, Hidetaka and Saeki, Koichi and Haeno, Hiroshi and Kapadia, Chiraag Deepak and Furudate, Ken and Nangalia, Jyoti and Spencer Chapman, Michael and Zhao, Li and Hsu, Joanne I. and Zhao, Chong and Chen, Shujuan and Tanaka, Tomoyuki and Li, Zongrui and Ogata, Satoko and Hanache, Sarah and Yang, Hui and DiNardo, Courtney and Daver, Naval and Pemmaraju, Naveen and Jain, Nitin and Ravandi, Farhad and Zhang, Jianhua and Song, Xingzhi and Thompson, Erika and Tang, Hongli and Little, Latasha and Gumbs, Curtis and Orlowski, Robert Z. and Qazilbash, Muzaffar and Bhalla, Kapil and Colla, Simona and Kantarjian, Hagop and Kanagal-Shamanna, Rashmi and Bueso-Ramos, Carlos and Nakada, Daisuke and Al-Atrash, Gheath and Molldrem, Jeffery and Futreal, P. Andrew and Shpall, Elizabeth and Goodell, Margaret and Garcia-Manero, Guillermo and Takahashi, Koichi},
  doi          = {10.1038/s41588-025-02235-w},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1695-1707},
  shortjournal = {Nat. Genet.},
  title        = {Clonal evolution of hematopoietic stem cells after autologous stem cell transplantation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The long-term effects of chemotherapy on normal blood cells. <em>NG</em>, <em>57</em>(7), 1684-1694. (<a href='https://doi.org/10.1038/s41588-025-02234-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several chemotherapeutic agents act by increasing DNA damage in cancer cells, triggering cell death. However, there is limited understanding of the extent and long-term consequences of collateral DNA damage in normal tissues. To investigate the impact of chemotherapy on mutation burdens and the cell population structure of normal tissue, we sequenced blood cell genomes from 23 individuals aged 3–80 years who were treated with a range of chemotherapy regimens. Substantial additional somatic mutation loads with characteristic mutational signatures were imposed by some chemotherapeutic agents, but the effects were dependent on the drug and blood cell types. Chemotherapy induced premature changes in the cell population structure of normal blood, similar to those caused by normal aging. The results show the long-term biological consequences of cytotoxic agents to which a substantial fraction of the population is exposed as part of disease management, raising mechanistic questions and highlighting opportunities for the mitigation of adverse effects. Mutational signature analysis of blood cells isolated from 23 chemotherapy-exposed samples and 9 nonexposed controls characterizes the effects of various drugs on mutational burden, signature exposure and cell types.},
  archive      = {J_NG},
  author       = {Mitchell, Emily and Pham, My H. and Clay, Anna and Sanghvi, Rashesh and Williams, Nicholas and Pietsch, Sandra and Hsu, Joanne I. and Jung, Hyunchul and Vedi, Aditi and Moody, Sarah and Wang, Jingwei and Leonganmornlert, Daniel and Spencer Chapman, Michael and Dunstone, Ellie and Santarsieri, Anna and Cagan, Alex and Machado, Heather E. and Baxter, E. Joanna and Follows, George and Hodson, Daniel J. and McDermott, Ultan and Doherty, Gary J. and Martincorena, Inigo and Humphreys, Laura and Mahbubani, Krishnaa and Saeb Parsy, Kourosh and Takahashi, Koichi and Goodell, Margaret A. and Kent, David and Laurenti, Elisa and Campbell, Peter J. and Rahbari, Raheleh and Nangalia, Jyoti and Stratton, Michael R.},
  doi          = {10.1038/s41588-025-02234-x},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1684-1694},
  shortjournal = {Nat. Genet.},
  title        = {The long-term effects of chemotherapy on normal blood cells},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The synthetic lethal interaction between CDS1 and CDS2 is a vulnerability in uveal melanoma and across multiple tumor types. <em>NG</em>, <em>57</em>(7), 1672-1683. (<a href='https://doi.org/10.1038/s41588-025-02222-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metastatic uveal melanoma is an aggressive disease with limited effective therapeutic options. To comprehensively map monogenic and digenic dependencies, we performed CRISPR–Cas9 screening in ten extensively profiled human uveal melanoma cell line models. Analysis involved genome-wide single-gene and combinatorial paired-gene CRISPR libraries. Among our 76 uveal melanoma-specific essential genes and 105 synthetic lethal gene pairs, we identified and validated the CDP-diacylglycerol synthase 2 gene (CDS2) as a genetic dependency in the context of low CDP-diacylglycerol synthase 1 gene (CDS1) expression. We further demonstrate that CDS1/CDS2 forms a synthetic lethal interaction in vivo and reveal that CDS2 knockout results in the disruption of phosphoinositide synthesis and increased cellular apoptosis and that re-expression of CDS1 rescues this cell fitness defect. We extend our analysis using pan-cancer data, confirming increased CDS2 essentiality in diverse tumor types with low CDS1 expression. Thus, the CDS1/CDS2 axis is a therapeutic target across a range of cancers. This study employs a functional genomic approach to identify a synthetic lethal interaction between CDS1 and CDS2 in uveal melanoma and other cancers, which may represent a potential therapeutic target.},
  archive      = {J_NG},
  author       = {Chan, Pui Ying and Alexander, Diana and Mehta, Ishan and Matsuyama, Larissa Satiko Alcantara Sekimoto and Harle, Victoria and Olvera-León, Rebeca and Park, Jun Sung and Arriaga-González, Fernanda G. and van der Weyden, Louise and Cheema, Saamin and Iyer, Vivek and Offord, Victoria and Barneda, David and Hawkins, Phillip T. and Stephens, Len and Kozik, Zuza and Woods, Michael and Wong, Kim and Balmus, Gabriel and Vinceti, Alessandro and Thompson, Nicola A. and Del Castillo Velasco-Herrera, Martin and Wessels, Lodewyk and van de Haar, Joris and Gonçalves, Emanuel and Sinha, Sanju and Vázquez-Cruz, Martha Estefania and Bisceglia, Luisa and Raimondi, Francesco and Choudhary, Jyoti and Patiyal, Sumeet and Venkatesh, Anjan and Iorio, Francesco and Ryan, Colm J. and Adams, David J.},
  doi          = {10.1038/s41588-025-02222-1},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1672-1683},
  shortjournal = {Nat. Genet.},
  title        = {The synthetic lethal interaction between CDS1 and CDS2 is a vulnerability in uveal melanoma and across multiple tumor types},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cytidine diphosphate diacylglycerol synthase 2 is a synthetic lethal target in mesenchymal-like cancers. <em>NG</em>, <em>57</em>(7), 1659-1671. (<a href='https://doi.org/10.1038/s41588-025-02221-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Synthetic lethal interactions (SLIs) based on genomic alterations in cancer have been therapeutically explored. We investigated the SLI space as a function of differential RNA expression in cancer and normal tissue. Computational analyses of functional genomic and gene expression resources uncovered a cancer-specific SLI between the paralogs cytidine diphosphate diacylglycerol synthase 1 (CDS1) and CDS2. The essentiality of CDS2 for cell survival is observed for mesenchymal-like cancers, which have low or absent CDS1 expression and account for roughly half of all cancers. Mechanistically, the CDS1–2 SLI is accompanied by disruption of lipid homeostasis, including accumulation of cholesterol esters and triglycerides, and apoptosis. Genome-wide CRISPR–Cas9 knockout screens in CDS1-negative cancer cells identify no common escape mechanism of death caused by CDS2 ablation, indicating the robustness of the SLI. Synthetic lethality is driven by CDS2 dosage and depends on catalytic activity. Thus, CDS2 may serve as a pharmacologically tractable target in mesenchymal-like cancers. The paralogs cytidine diphosphate diacylglycerol synthase 1 and 2 form a potentially targetable synthetic lethal relationship in mesenchymal-like cancers that involves disruption of lipid metabolism.},
  archive      = {J_NG},
  author       = {Arnoldus, Tim and van Vliet, Alex and Bleijerveld, Onno B. and de Groot, Adriaan F. H. and Piao, Qinglin and Blomberg, Niek and Schatton, Désirée and Dong, Jing and van Hal-van Veen, Susan E. and Harkes, Rolf and Grootemaat, Anita E. and Proost, Natalie and Cabukusta, Birol and Frezza, Christian and van de Ven, Marieke and van der Wel, Nicole N. and Giera, Martin and Altelaar, Maarten and Peeper, Daniel S.},
  doi          = {10.1038/s41588-025-02221-2},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1659-1671},
  shortjournal = {Nat. Genet.},
  title        = {Cytidine diphosphate diacylglycerol synthase 2 is a synthetic lethal target in mesenchymal-like cancers},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Linking regulatory variants to target genes by integrating single-cell multiome methods and genomic distance. <em>NG</em>, <em>57</em>(7), 1649-1658. (<a href='https://doi.org/10.1038/s41588-025-02220-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods that analyze single-cell paired RNA sequencing (RNA-seq) and assay for transposase-accessible chromatin using sequencing (ATAC-seq) multiome data have shown promise in linking regulatory elements to genes. However, existing methods exhibit low concordance and do not capture the effects of genomic distance. We propose pgBoost, an integrative modeling framework that trains a non-linear combination of existing linking strategies (including genomic distance) on expression quantitative trait locus (eQTL) data to assign a probabilistic score to each candidate single-nucleotide polymorphism–gene link. pgBoost attained higher enrichment than existing methods for evaluation sets derived from eQTL, activity-by-contact, CRISPR and genome-wide association study (GWAS) data. We further determined that restricting pgBoost to features from a focal cell type improved power to identify links relevant to that cell type. We highlight several examples in which pgBoost linked fine-mapped GWAS variants to experimentally validated or biologically plausible target genes that were not implicated by other methods. In conclusion, a non-linear combination of linking strategies improves power to identify target genes underlying GWAS associations. This study introduces pgBoost, a gradient boosting framework that predicts regulatory single-nucleotide polymorphism–gene links by combining existing linking scores from single-cell multiomics datasets and genomic distance.},
  archive      = {J_NG},
  author       = {Dorans, Elizabeth and Jagadeesh, Karthik and Dey, Kushal and Price, Alkes L.},
  doi          = {10.1038/s41588-025-02220-3},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1649-1658},
  shortjournal = {Nat. Genet.},
  title        = {Linking regulatory variants to target genes by integrating single-cell multiome methods and genomic distance},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell eQTL analysis identifies genetic variation underlying metabolic dysfunction-associated steatohepatitis. <em>NG</em>, <em>57</em>(7), 1638-1648. (<a href='https://doi.org/10.1038/s41588-025-02237-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metabolic dysfunction-associated steatotic liver disease (MASLD) is increasingly recognized for its medical and socioeconomic impacts, driven by diverse genetic and environmental factors. Here, to address the urgent need for individually tailored therapies, we show results from single-cell expression quantitative trait locus (sc-eQTL) analysis on liver biopsies from 25 patients with MASLD and 23 controls. This approach identified over 3,500 sc-eQTLs across major liver cell types and cell-state-interacting eQTLs (ieQTLs) with significant enrichment for disease heritability (for MASLD trait, ieQTL enrichment odds ratio 10.27). We integrated transcription factors as upstream regulators of ieQTLs, revealing 601 functional units (‘quartets’) composed of transcription factors, cell states, SNP components of ieQTL (ieSNPs) and Gene component of ieQTL (ieGenes). From these results, we pinpoint the loss of an eQTL in EFHD1 during hepatocyte maladaptation associated with genotype-specific regulation by FOXO1, further contributing to the risk of MASLD. Our approach underscores the role of eQTL analysis in capturing crucial genetic variations that influence gene expression and clinical outcomes in complex diseases. Single-cell analyses of liver biopsies from patients with metabolic dysfunction-associated steatohepatitis and healthy controls identify cell-state-interacting expression quantitative trait loci with significant enrichment for disease heritability.},
  archive      = {J_NG},
  author       = {Hong, Sung Eun and Mun, Seon Ju and Lee, Young Joo and Yoo, Taekyeong and Suh, Kyung-Suk and Kang, Keon Wook and Son, Myung Jin and Kim, Won and Choi, Murim},
  doi          = {10.1038/s41588-025-02237-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1638-1648},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell eQTL analysis identifies genetic variation underlying metabolic dysfunction-associated steatohepatitis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A map of blood regulatory variation in south africans enables GWAS interpretation. <em>NG</em>, <em>57</em>(7), 1628-1637. (<a href='https://doi.org/10.1038/s41588-025-02223-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Functional genomics resources are critical for interpreting human genetic studies, but currently they are predominantly from European-ancestry individuals. Here we present the South African Blood Regulatory (SABR) resource, a map of blood regulatory variation that includes three South Eastern Bantu-speaking groups. Using paired whole-genome and blood transcriptome data from over 600 individuals, we map the genetic architecture of 40 blood cell traits derived from deconvolution analysis, as well as expression, splice and cell-type interaction quantitative trait loci. We comprehensively compare SABR to the Genotype Tissue Expression Project and characterize thousands of regulatory variants only observed in African-ancestry individuals. Finally, we demonstrate the increased utility of SABR for interpreting African-ancestry association studies by identifying putatively causal genes and molecular mechanisms through colocalization analysis of blood-relevant traits from the Pan-UK Biobank. Importantly, we make full SABR summary statistics publicly available to support the African genomics community. This study maps regulatory variation in blood using whole-genome and transcriptome data from 614 South Africans, providing a resource to aid the interpretation of genome-wide association studies in African populations.},
  archive      = {J_NG},
  author       = {Castel, Stephane E. and Tluway, Furahini D. and Emde, Anne-Katrin and Smyth, Natalie and Karim, Mohd and Sengupta, Dhriti and Gray, Olivia A. and Hendershott, Melissa and LeBaron von Baeyer, Sarah and Burke, Erin E. and Kaewert, Sarah and Nguyen, Khanh-Dung H. and Choma, Solomon S. R. and Mashaba, Reneilwe G. and Micklesfield, Lisa K. and Kabudula, Chodziwadziwa and Kahn, Kathleen and Xavier Gomez-Olive, F. and Tollman, Stephen and Choudhury, Ananyo and Mpangase, Phelelani T. and Hazelhurst, Scott and Wasik, Kaja A. and Yerges-Armstrong, Laura and Ramsay, Michèle},
  doi          = {10.1038/s41588-025-02223-0},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1628-1637},
  shortjournal = {Nat. Genet.},
  title        = {A map of blood regulatory variation in south africans enables GWAS interpretation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Incorporating genetic data improves target trial emulations and informs the use of polygenic scores in randomized controlled trial design. <em>NG</em>, <em>57</em>(7), 1620-1627. (<a href='https://doi.org/10.1038/s41588-025-02229-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Randomized controlled trials (RCTs) remain the gold standard for evaluating medical interventions, yet ethical, practical and financial constraints often necessitate reliance on observational data and trial emulations. This study explores how integrating genetic data can enhance both emulated and traditional trial designs. Using FinnGen (n = 425,483), we emulated four major cardiometabolic RCTs and showed how reduced differences in polygenic scores (PGS) between trial arms track improvement in study design. Simulation studies reveal that PGS alone cannot fully adjust for unmeasured confounding. Instead, Mendelian randomization analyses can be used to detect likely confounders. Finally, trial emulations provide a platform to assess and refine PGS implementation for genetic enrichment strategies. By comparing associations of PGS with trial outcomes in the general population and emulated trial cohorts, we highlight the need to validate prognostic enrichment approaches in trial-relevant populations. These results highlight the growing potential of incorporating genetic information to optimize clinical trial design. This study integrates polygenic risk scores with four emulated clinical trials using FinnGen data and shows the feasibility of this approach while highlighting potential pitfalls.},
  archive      = {J_NG},
  author       = {German, Jakob and Yang, Zhiyu and Urbut, Sarah and Vartiainen, Pekka and Natarajan, Pradeep and Patorno, Elisabetta and Kutalik, Zoltan and Philippakis, Anthony and Ganna, Andrea},
  doi          = {10.1038/s41588-025-02229-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1620-1627},
  shortjournal = {Nat. Genet.},
  title        = {Incorporating genetic data improves target trial emulations and informs the use of polygenic scores in randomized controlled trial design},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Decomposition of phenotypic heterogeneity in autism reveals underlying genetic programs. <em>NG</em>, <em>57</em>(7), 1611-1619. (<a href='https://doi.org/10.1038/s41588-025-02224-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unraveling the phenotypic and genetic complexity of autism is extremely challenging yet critical for understanding the biology, inheritance, trajectory and clinical manifestations of the many forms of the condition. Using a generative mixture modeling approach, we leverage broad phenotypic data from a large cohort with matched genetics to identify robust, clinically relevant classes of autism and their patterns of core, associated and co-occurring traits, which we further validate and replicate in an independent cohort. We demonstrate that phenotypic and clinical outcomes correspond to genetic and molecular programs of common, de novo and inherited variation and further characterize distinct pathways disrupted by the sets of mutations in each class. Remarkably, we discover that class-specific differences in the developmental timing of affected genes align with clinical outcome differences. These analyses demonstrate the phenotypic complexity of children with autism, identify genetic programs underlying their heterogeneity, and suggest specific biological dysregulation patterns and mechanistic hypotheses. Classes of autism are uncovered with a generative mixture modeling approach leveraging matched phenotypic and genetic data from a large cohort, revealing different genetic programs underlying their phenotypic and clinical traits.},
  archive      = {J_NG},
  author       = {Litman, Aviya and Sauerwald, Natalie and Green Snyder, LeeAnne and Foss-Feig, Jennifer and Park, Christopher Y. and Hao, Yun and Dinstein, Ilan and Theesfeld, Chandra L. and Troyanskaya, Olga G.},
  doi          = {10.1038/s41588-025-02224-z},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1611-1619},
  shortjournal = {Nat. Genet.},
  title        = {Decomposition of phenotypic heterogeneity in autism reveals underlying genetic programs},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Transferability of european-derived alzheimer’s disease polygenic risk scores across multiancestry populations. <em>NG</em>, <em>57</em>(7), 1598-1610. (<a href='https://doi.org/10.1038/s41588-025-02227-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A polygenic score (PGS) for Alzheimer’s disease (AD) was derived recently from data on genome-wide significant loci in European ancestry populations. We applied this PGS to populations in 17 European countries and observed a consistent association with the AD risk, age at onset and cerebrospinal fluid levels of AD biomarkers, independently of apolipoprotein E locus (APOE). This PGS was also associated with the AD risk in many other populations of diverse ancestries. A cross-ancestry polygenic risk score improved the association with the AD risk in most of the multiancestry populations tested when the APOE region was included. Finally, we found that the PGS/polygenic risk score captured AD-specific information because the association weakened as the diagnosis was broadened. In conclusion, a simple PGS captures the AD-specific genetic information that is common to populations of different ancestries, although studies of more diverse populations are still needed to better characterize the genetics of AD. Polygenic risk scores for Alzheimer’s disease derived from individuals of European ancestry can show improved performance in multiancestry settings after incorporating genome-wide association summary statistics from diverse populations.},
  archive      = {J_NG},
  author       = {Nicolas, Aude and Sherva, Richard and Grenier-Boley, Benjamin and Kim, Yoontae and Kikuchi, Masataka and Timsina, Jigyasha and de Rojas, Itziar and Dalmasso, María Carolina and Zhou, Xiaopu and Le Guen, Yann and Arboleda-Bustos, Carlos E. and Camargos Bicalho, Maria Aparecida and Guerchet, Maëlenn and van der Lee, Sven and Goss, Monica and Castillo, Atahualpa and Bellenguez, Céline and Küçükali, Fahri and Satizabal, Claudia L. and Fongang, Bernard and Yang, Qiong and Peters, Oliver and Schneider, Anja and Dichgans, Martin and Rujescu, Dan and Scherbaum, Norbert and Deckert, Jürgen and Riedel-Heller, Steffi and Hausner, Lucrezia and Molina-Porcel, Laura and Düzel, Emrah and Grimmer, Timo and Wiltfang, Jens and Heilmann-Heimbach, Stefanie and Moebus, Susanne and Tegos, Thomas and Scarmeas, Nikolaos and Dols-Icardo, Oriol and Moreno, Fermin and Pérez-Tur, Jordi and Bullido, María J. and Pastor, Pau and Sánchez-Valle, Raquel and Álvarez, Victoria and Cao, Han and Ip, Nancy Y. and Fu, Amy K. Y. and Ip, Fanny C. F. and Olivar, Natividad and Muchnik, Carolina and Cuesta, Carolina and Campanelli, Lorenzo and Solis, Patricia and Politis, Daniel Gustavo and Kochen, Silvia and Brusco, Luis Ignacio and Boada, Mercè and García-González, Pablo and Puerta, Raquel and Mir, Pablo and Real, Luis M. and Piñol-Ripoll, Gerard and García-Alberca, Jose María and Royo, Jose Luís and Rodriguez-Rodriguez, Eloy and Soininen, Hilkka and Heikkinen, Sami and de Mendonça, Alexandre and Mehrabian, Shima and Traykov, Latchezar and Hort, Jakub and Vyhnalek, Martin and Rasmussen, Katrine Laura and Thomassen, Jesper Qvist and Pijnenburg, Yolande A. L. and Holstege, Henne and van Swieten, John C. and Seelaar, Harro and Claassen, Jurgen A. H. R. and Jansen, Willemijn J. and Ramakers, Inez and Verhey, Frans and van der Lugt, Aad and Scheltens, Philip and Ortega-Rojas, Jenny and Concha Mera, Ana Gabriela and Mahecha, Maria F. and Pardo, Rodrigo and Arboleda, Gonzalo and Bahrami, Shahram and Fominykh, Vera and Selbæk, Geir and Graff, Caroline and Papenberg, Goran and Giedraitis, Vilmantas and Boland, Anne and Deleuze, Jean-François and de Marco, Luiz Armando and de Moraes, Edgar Nunes and de Mattos Viana, Bernardo and Túlio Gualberto Cintra, Marco and Juarez-Cedillo, Teresa and Griswold, Anthony J. and Forund, Tatiana and Haines, Jonathan and Farrer, Lindsay and DeStefano, Anita and Wijsman, Ellen and Mayeux, Richard and Pericak-Vance, Margaret and Kunkle, Brian and Goate, Alison and Schellenberg, Gerard D. and Vardarajan, Badri and Wang, Li-San and Leung, Yuk Yee and Dalgard, Clifton L. and Nicolas, Gael and Wallon, David and Dufouil, Carole and Pasquier, Florence and Hanon, Olivier and Debette, Stéphanie and Grünblatt, Edna and Popp, Julius and Angel, Bárbara and Gloger, Sergio and Chacon, Maria Victoria and Aranguiz, Rafael and Orellana, Paulina and Slachevsky, Andrea and Gonzalez-Billault, Christian and Albala, Cecilia and Fuentes, Patricio and Sachdev, Perminder and Mather, Karen A. and Hauger, Richard L. and Merritt, Victoria and Panizzon, Matthew and Zhang, Rui and Gaziano, J. Michael and Ghidoni, Roberta and Galimberti, Daniela and Arosio, Beatrice and Mecocci, Patrizia and Solfrizzi, Vincenzo and Parnetti, Lucilla and Squassina, Alessio and Tremolizzo, Lucio and Borroni, Barbara and Nacmias, Benedetta and Caffarra, Paolo and Seripa, Davide and Rainero, Innocenzo and Daniele, Antonio and Piras, Fabrizio and Leonard, Hampton L. and Yokoyama, Jenifer S. and Nalls, Mike A. and Miyashita, Akinori and Hara, Norikazu and Ozaki, Kouichi and Niida, Shumpei and Williams, Julie and Masullo, Carlo and Amouyel, Philippe and Preux, Pierre-Marie and Mbelesso, Pascal and Bandzouzi, Bébène and Saykin, Andy and Jessen, Frank and Kehoe, Patrick G. and Van Duijn, Cornelia and Ben Salem, Nesrine and Frikke-Schmidt, Ruth and Cherni, Lotfi and Greicius, Michael D. and Tsolaki, Magda and Sánchez-Juan, Pascual and Romano Silva, Marco Aurélio and Porter, Tenielle and Laws, Simon M. and Sleegers, Kristel and Ingelsson, Martin and Dartigues, Jean-François and Seshadri, Sudha and Rossi, Giacomina and Morelli, Laura and Hiltunen, Mikko and Sims, Rebecca and van der Flier, Wiesje and Andreassen, Ole A. and Arboleda, Humberto and Cruchaga, Carlos and Escott-Price, Valentina and Ruiz, Agustín and Lee, Kun Ho and Ikeuchi, Takeshi and Ramirez, Alfredo and Gim, Jungsoo and Logue, Mark and Lambert, Jean-Charles},
  doi          = {10.1038/s41588-025-02227-w},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1598-1610},
  shortjournal = {Nat. Genet.},
  title        = {Transferability of european-derived alzheimer’s disease polygenic risk scores across multiancestry populations},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Using large-scale population-based data to improve disease risk assessment of clinical variants. <em>NG</em>, <em>57</em>(7), 1588-1597. (<a href='https://doi.org/10.1038/s41588-025-02212-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the disease risk of genetic variants is fundamental to precision medicine. Estimates of penetrance—the probability of disease for individuals with a variant allele—rely on disease-specific cohorts, clinical testing and emerging electronic health record (EHR)-linked biobanks. These data sources, while valuable, each have limitations in quality, representativeness and analyzability. Here, we provide a historical account of the currently accepted pathogenicity classification system and data available in ClinVar, a public archive that aggregates variant interpretations but lacks detailed data for accurate penetrance assessment, highlighting its oversimplification of disease risk. We propose an integrative Bayesian framework that unifies pathogenicity and penetrance, leveraging both functional and real-world evidence to refine risk predictions. In addition, we advocate for enhancing ClinVar with the inclusion of high-priority phenotypes, age-stratified data and population-based cohorts linked to EHRs. We suggest developing a community repository of population-based penetrance estimates to support the clinical application of genetic data. This Perspective proposes a new framework that integrates large-scale population data with clinical evidence to improve disease risk assessment of genetic variants, enhancing the accuracy and value of genetic information in precision medicine.},
  archive      = {J_NG},
  author       = {Forrest, Iain S. and Huang, Kuan-Lin and Eggington, Julie M. and Chung, Wendy K. and Jordan, Daniel M. and Do, Ron},
  doi          = {10.1038/s41588-025-02212-3},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1588-1597},
  shortjournal = {Nat. Genet.},
  title        = {Using large-scale population-based data to improve disease risk assessment of clinical variants},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Realizing the promise of genome-wide association studies for effector gene prediction. <em>NG</em>, <em>57</em>(7), 1578-1587. (<a href='https://doi.org/10.1038/s41588-025-02210-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWAS) identify regions of the genome in which genetic variation is associated with the risk of complex diseases, such as diabetes, or the magnitude of traits, such as blood pressure. Determining which ‘effector genes’ mediate the effects of GWAS associations is essential to using GWAS to understand disease mechanisms and develop new therapies. In recent years, GWAS authors have increasingly included effector gene predictions as part of their study results. However, the research community has not yet converged on standards for generating or reporting these predictions. In this Perspective, we illustrate the diversity of the evidence types used to support effector gene predictions and argue for future initiatives to increase their accessibility and usefulness. This Perspective argues that predicting effector genes for complex diseases is a key outcome of genetic association studies where standards are urgently needed to maximize utility, improve interoperability and enable biological insight.},
  archive      = {J_NG},
  author       = {Costanzo, Maria C. and Harris, Laura W. and Ji, Yue and McMahon, Aoife and Burtt, Noël P. and Flannick, Jason},
  doi          = {10.1038/s41588-025-02210-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1578-1587},
  shortjournal = {Nat. Genet.},
  title        = {Realizing the promise of genome-wide association studies for effector gene prediction},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial organization of chromatin restricts activation of poised alternative promoters in LTRs. <em>NG</em>, <em>57</em>(7), 1576-1577. (<a href='https://doi.org/10.1038/s41588-025-02238-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transposable elements (TEs) are important in the evolution of genomic functions but the mechanisms of their precise role in cancer pathogenesis is unclear. Alternative promoters at the TE subclass long terminal repeats (LTRs) are activated when topologically associating domain (TAD) hierarchy maintained by NIPBL is lost, potentially leading to aberrant transcription of oncogenes.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02238-7},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1576-1577},
  shortjournal = {Nat. Genet.},
  title        = {Spatial organization of chromatin restricts activation of poised alternative promoters in LTRs},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Empowering african GWAS with a map of blood regulatory variation. <em>NG</em>, <em>57</em>(7), 1574-1575. (<a href='https://doi.org/10.1038/s41588-025-02228-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We created a population-scale functional genomic dataset that maps whole blood regulatory variation in three South African groups. This resource supports the interpretation of genome-wide association studies in African-ancestry individuals, enabling translation to actionable discoveries.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02228-9},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1574-1575},
  shortjournal = {Nat. Genet.},
  title        = {Empowering african GWAS with a map of blood regulatory variation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A retrotransposon caught red-handed in a curious case of missing digits. <em>NG</em>, <em>57</em>(7), 1571-1573. (<a href='https://doi.org/10.1038/s41588-025-02147-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A newly uncovered mechanism shows how a single transposable element of retroviral origin can adopt the expression pattern of a neighboring gene. This leads to the production of viral-like particles that disrupt organ formation when epigenetic silencing is compromised.},
  archive      = {J_NG},
  author       = {Guo, Youjia and Modzelewski, Andrew J.},
  doi          = {10.1038/s41588-025-02147-9},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1571-1573},
  shortjournal = {Nat. Genet.},
  title        = {A retrotransposon caught red-handed in a curious case of missing digits},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Organoid models advance dysentery control. <em>NG</em>, <em>57</em>(7), 1568-1570. (<a href='https://doi.org/10.1038/s41588-025-02219-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Shigella is a major human pathogen with a stark lack of pipelines linking genome content to clinical pathogenesis. An innovative study using large-scale organoid models, combined with genome-wide mutagenesis screening, reveals virulence factors required for Shigella colonization.},
  archive      = {J_NG},
  author       = {Miles, Sydney L. and Mostowy, Serge},
  doi          = {10.1038/s41588-025-02219-w},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1568-1570},
  shortjournal = {Nat. Genet.},
  title        = {Organoid models advance dysentery control},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How chemotherapy shapes hematopoietic stem cells. <em>NG</em>, <em>57</em>(7), 1566-1567. (<a href='https://doi.org/10.1038/s41588-025-02231-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two complementary studies used whole-genome sequencing of single cell-derived hematopoietic colonies to show that chemotherapy results in a marked decrease in hematopoietic stem and progenitor cell (HSPC) diversity, with the parallel evolution of multiple independent HSPCs harboring mutations in DNA damage response genes.},
  archive      = {J_NG},
  author       = {Wong, Terrence N. and Link, Daniel C.},
  doi          = {10.1038/s41588-025-02231-0},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1566-1567},
  shortjournal = {Nat. Genet.},
  title        = {How chemotherapy shapes hematopoietic stem cells},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Y chromosome loss in cancer. <em>NG</em>, <em>57</em>(7), 1565. (<a href='https://doi.org/10.1038/s41588-025-02276-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02276-1},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1565},
  shortjournal = {Nat. Genet.},
  title        = {Y chromosome loss in cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enhancer mutational screen in vivo. <em>NG</em>, <em>57</em>(7), 1565. (<a href='https://doi.org/10.1038/s41588-025-02277-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-025-02277-0},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1565},
  shortjournal = {Nat. Genet.},
  title        = {Enhancer mutational screen in vivo},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Expanding in situ genome sequencing. <em>NG</em>, <em>57</em>(7), 1565. (<a href='https://doi.org/10.1038/s41588-025-02278-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gross, Petra},
  doi          = {10.1038/s41588-025-02278-z},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1565},
  shortjournal = {Nat. Genet.},
  title        = {Expanding in situ genome sequencing},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Perturb-multimodal pooled screening in intact tissues. <em>NG</em>, <em>57</em>(7), 1565. (<a href='https://doi.org/10.1038/s41588-025-02279-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-025-02279-y},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1565},
  shortjournal = {Nat. Genet.},
  title        = {Perturb-multimodal pooled screening in intact tissues},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Distinct dynamics and functions of H2AK119ub1 and H3K27me3 in mouse preimplantation embryos. <em>NG</em>, <em>57</em>(6), 1564. (<a href='https://doi.org/10.1038/s41588-025-02211-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Chen, Zhiyuan and Djekidel, Mohamed Nadhir and Zhang, Yi},
  doi          = {10.1038/s41588-025-02211-4},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1564},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Distinct dynamics and functions of H2AK119ub1 and H3K27me3 in mouse preimplantation embryos},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Mapping genetic diversity with the GenomeIndia project. <em>NG</em>, <em>57</em>(6), 1563. (<a href='https://doi.org/10.1038/s41588-025-02206-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Bhattacharyya, Chandrika and Subramanian, Krithika and Uppili, Bharathram and Biswas, Nidhan K. and Ramdas, Shweta and Tallapaka, Karthik Bharadwaj and Arvind, Prathima and Rupanagudi, Khader Valli and Maitra, Arindam and Nagabandi, Tulasi and De, Tiyasha and Singh, Kuldeep and Sharma, Praveen and Sharma, Nanaocha and Raghav, Sunil K. and Prasad, Punit and Soniya, E. V. and Jaleel, Abdul and Nelson Sathi, Shijulal and Joshi, Madhvi and Joshi, Chaitanya and Lahiri, Mayurika and Dixit, Santosh and Shashidhara, L. S. and Senthil Kumar, Nachimuthu and Lalhruaitluanga, H. and Nundanga, Lal and Shivakumar, Venkataram and Venkatasubramanian, Ganesan and Rao, Naren P. and Ganie, Mohd Ashraf and Wani, Imtiyaz Ahmad and Jha, Ganganath and Dalal, Ashwin and Bashyam, Murali Dharan and Varadwaj, Pritish Kumar and BS, Sanjeev and Simmhan, Yogesh and Jain, Chirag and Sundar, Durai and Gupta, Ishaan and Yadav, Pankaj and Sinha, Himanshu and Narayanan, Manikandan and Raman, Karthik and Padinjat, Raghu and Sabarinathan, Radhakrishnan and Narahari, Yadati and Ravindranath, Vijayalakshmi and Thangaraj, Kumarasamy and Sowpati, Divya Tej and Faruq, Mohammed and Basu, Analabha and Kahali, Bratati},
  doi          = {10.1038/s41588-025-02206-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1563},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Mapping genetic diversity with the GenomeIndia project},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An atypical NLR pair TdCNL1/TdCNL5 from wild emmer confers powdery mildew resistance in wheat. <em>NG</em>, <em>57</em>(6), 1553-1562. (<a href='https://doi.org/10.1038/s41588-025-02208-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Resistance to wheat powdery mildew is commonly mediated by individual resistance proteins, most of which encode nucleotide-binding leucine-rich repeat (NLR) receptors. Here we report that the powdery mildew resistance gene MLIW170/PM26 in wild emmer and bread wheat derivatives is determined by a genetically linked atypical NLR pair TdCNL1/TdCNL5. Map-based cloning and PacBio HiFi long-read sequencing revealed that TdCNL1 encodes an atypical coiled-coil-domain-containing NLR protein (CNL) fused with a new potassium-dependent sodium-calcium exchanger integrated domain, whereas TdCNL5 encodes a canonical CNL protein. Mutagenesis and virus-induced gene silencing experiments indicated that both TdCNL1 and TdCNL5 are essential for powdery mildew resistance. Transgenic plants with TdCNL1 alone or TdCNL1/TdCNL5 together show resistance, whereas Fielder with TdCNL5 alone was susceptible. Geographically, MLIW170/PM26 occurs in a few Southern populations of wild emmer wheat. Our study highlights an atypical NLR pair coordinately regulating powdery mildew resistance and provides a diversified resistance gene resource for wheat improvement. The powdery mildew resistance of the MLIW170/PM26 locus, which originated from wild emmer wheat, is determined by a genetically linked atypical NLR gene pair, TdCNL1 and TdCNL5, with TdCNL1 encoding an atypical nucleotide-binding leucine-rich repeat protein integrating a potassium-dependent sodium-calcium exchanger domain.},
  archive      = {J_NG},
  author       = {Zhu, Keyu and Li, Miaomiao and Dong, Lingli and Zhang, Huaizhi and Zhang, Deyun and Lu, Ping and Wu, Qiuhong and Xie, Jingzhong and Chen, Yongxing and Guo, Guanghao and Zhang, Panpan and Li, Beibei and Li, Wenling and Dong, Lei and Hou, Yikun and Yang, Yijun and Qiu, Dan and Wang, Gaojie and Huang, Baoge and Cui, Xuejia and Fu, Hongkui and Yuan, Chengguo and Fahima, Tzion and Nevo, Eviatar and Li, Hongjie and Rong, Junkang and Hua, Wei and Liu, Zhiyong},
  doi          = {10.1038/s41588-025-02208-z},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1553-1562},
  shortjournal = {Nat. Genet.},
  title        = {An atypical NLR pair TdCNL1/TdCNL5 from wild emmer confers powdery mildew resistance in wheat},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A head-to-head NLR gene pair from wild emmer confers stripe rust resistance in wheat. <em>NG</em>, <em>57</em>(6), 1543-1552. (<a href='https://doi.org/10.1038/s41588-025-02207-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Wheat stripe rust poses a major threat to global food security. Discovery of disease resistance genes from wild relatives enables multigene stacking that could enhance durability. Here we use map-based cloning and long-read sequencing to isolate two adjacent nucleotide-binding and leucine-rich repeat (NLR) receptors from wild emmer wheat. Using mutagenesis, gene silencing and genetic transformation, we show that the genes TdNLR1 and TdNLR2 oriented head-to-head are both required for YrTD121-mediated stripe rust resistance. TdNLR1 encodes a canonical NLR (CC-NB-ARC-LRR) protein, whereas TdNLR2 encodes an atypical one (NB-ARC-LRR). Both genes lack an integrated domain previously associated with effector perception, representing an uncommon architecture for paired NLRs in plants. The coiled coil domain of TdNLR1 triggers cell death and self-associates in planta. YrTD121 was present in wild emmer but absent in all other Triticum species examined. Our work sheds light on the function of paired NLRs in conferring disease resistance and facilitates breeding for resistance. Two genetically linked TdNLR genes from wild emmer wheat are both required for YrTD121-mediated stripe rust resistance. The TdNLR1 and TdNLR2 pair lacks an integrated domain, representing an uncommon architecture for paired NLRs in plants.},
  archive      = {J_NG},
  author       = {Hu, Yanling and Li, Miaomiao and Li, Yuqin and Du, Lilin and Xie, Ruijie and Ni, Fei and Xia, Chongjing and Wang, Ke and Huang, Yanyan and Xu, Binyang and Li, Yinghui and Jiang, Yunfeng and Hao, Ming and Jiang, Bo and Ning, Shunzong and Yuan, Zhongwei and Feng, Lihua and Zhang, Lianquan and Chen, Shisheng and Wu, Bihua and Liu, Zhiyong and Fahima, Tzion and Liu, Dengcai and Huang, Lin},
  doi          = {10.1038/s41588-025-02207-0},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1543-1552},
  shortjournal = {Nat. Genet.},
  title        = {A head-to-head NLR gene pair from wild emmer confers stripe rust resistance in wheat},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Coordinated function of paired NLRs confers yr84-mediated stripe rust resistance in wheat. <em>NG</em>, <em>57</em>(6), 1535-1542. (<a href='https://doi.org/10.1038/s41588-025-02203-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cloning of resistance genes expands our understanding of their function and facilitates their deployment in breeding. Here we report the cloning of two genes from wild emmer wheat (Triticum turgidum ssp. dicoccoides) underlying Yr84-mediated stripe rust resistance using a combination of fine mapping, long-read sequencing and mutation-induced functional validation. In contrast to all previously cloned stripe rust genes, the incompletely dominant Yr84 phenotype is conferred through the coordinated function of paired nucleotide-binding leucine-rich repeat (NLR) genes CNL and NL. We reason that based on their genomic organization, annotation, expression profiles and predicted protein structure, CNL functions as a sensor NLR, responsible for effector recognition, and NL acts as a helper NLR, initiating downstream resistance cascades. Both the CNL and NL lack an integrated domain(s) previously implicated in effector recognition by paired NLRs; therefore, these findings contribute insights into the structure and molecular mechanisms of the function of plant paired NLRs. The study reports the cloning of paired NLR genes CNL and NL from wild emmer wheat and demonstrates their coordinated function in conferring Yr84-mediated wheat stripe rust resistance.},
  archive      = {J_NG},
  author       = {Klymiuk, Valentyna and Wiebe, Krystalee and Chawla, Harmeet Singh and Ens, Jennifer and Subramaniam, Rajagopal and Pozniak, Curtis J.},
  doi          = {10.1038/s41588-025-02203-4},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1535-1542},
  shortjournal = {Nat. Genet.},
  title        = {Coordinated function of paired NLRs confers yr84-mediated stripe rust resistance in wheat},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Conservation of regulatory elements with highly diverged sequences across large evolutionary distances. <em>NG</em>, <em>57</em>(6), 1524-1534. (<a href='https://doi.org/10.1038/s41588-025-02202-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Developmental gene expression is a remarkably conserved process, yet most cis-regulatory elements (CREs) lack sequence conservation, especially at larger evolutionary distances. Some evidence suggests that CREs at the same genomic position remain functionally conserved independent of sequence conservation. However, the extent of such positional conservation remains unclear. Here, we profiled the regulatory genome in mouse and chicken embryonic hearts at equivalent developmental stages and found that most CREs lack sequence conservation. To identify positionally conserved CREs, we introduced the synteny-based algorithm interspecies point projection, which identifies up to fivefold more orthologs than alignment-based approaches. We termed positionally conserved orthologs ‘indirectly conserved’ and showed that they exhibited chromatin signatures and sequence composition similar to sequence-conserved CREs but greater shuffling of transcription factor binding sites between orthologs. Finally, we validated indirectly conserved chicken enhancers using in vivo reporter assays in mouse. By overcoming alignment-based limitations, we revealed widespread functional conservation of sequence-divergent CREs. Combining functional genomic data from mouse and chicken with a synteny-based strategy identifies positionally conserved cis-regulatory elements in the absence of direct sequence conservation.},
  archive      = {J_NG},
  author       = {Phan, Mai H. Q. and Zehnder, Tobias and Puntieri, Fiona and Magg, Andreas and Majchrzycka, Blanka and Antonović, Milan and Wieler, Hannah and Lo, Bai-Wei and Baranasic, Damir and Lenhard, Boris and Müller, Ferenc and Vingron, Martin and Ibrahim, Daniel M.},
  doi          = {10.1038/s41588-025-02202-5},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1524-1534},
  shortjournal = {Nat. Genet.},
  title        = {Conservation of regulatory elements with highly diverged sequences across large evolutionary distances},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-definition spatial transcriptomic profiling of immune cell populations in colorectal cancer. <em>NG</em>, <em>57</em>(6), 1512-1523. (<a href='https://doi.org/10.1038/s41588-025-02193-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A comprehensive understanding of cellular behavior and response to the tumor microenvironment (TME) in colorectal cancer (CRC) remains elusive. Here, we introduce the high-definition Visium spatial transcriptomic technology (Visium HD) and investigate formalin-fixed paraffin-embedded human CRC samples (n = 5). We demonstrate the high sensitivity, single-cell-scale resolution and spatial accuracy of Visium HD, generating a highly refined whole-transcriptome spatial profile of CRC samples. We identify transcriptomically distinct macrophage subpopulations in different spatial niches with potential pro-tumor and anti-tumor functions via interactions with tumor and T cells. In situ gene expression analysis validates our findings and localizes a clonally expanded T cell population close to macrophages with anti-tumor features. Our study demonstrates the power of high-resolution spatial technologies to understand cellular interactions in the TME and paves the way for larger studies that will unravel mechanisms and biomarkers of CRC biology, improving diagnosis and disease management strategies. High-resolution, spatial transcriptomic analysis of colorectal cancers using matched sequencing- and imaging-based methods characterizes immune cells and their interactions with the tumor microenvironment.},
  archive      = {J_NG},
  author       = {Oliveira, Michelli Faria de and Romero, Juan Pablo and Chung, Meii and Williams, Stephen R. and Gottscho, Andrew D. and Gupta, Anushka and Pilipauskas, Susan E. and Mohabbat, Seayar and Raman, Nandhini and Sukovich, David J. and Patterson, David M. and Taylor, Sarah E. B.},
  doi          = {10.1038/s41588-025-02193-3},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1512-1523},
  shortjournal = {Nat. Genet.},
  title        = {High-definition spatial transcriptomic profiling of immune cell populations in colorectal cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Longitudinal and multisite sampling reveals mutational and copy number evolution in tumors during metastatic dissemination. <em>NG</em>, <em>57</em>(6), 1504-1511. (<a href='https://doi.org/10.1038/s41588-025-02204-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To understand genetic evolution in cancer during metastasis, we analyzed genomic profiles of 3,732 cancer patients in whom several tumor sites were longitudinally biopsied. During distant metastasis, tumors were observed to accumulate copy number alterations (CNAs) to a much greater degree than mutations. In particular, the development of whole genome duplication was a common event during metastasis, emerging de novo in 28% of patients. Loss of 9p (including CDKN2A) developed during metastasis in 11% of patients. To a lesser degree, mutations and allelic loss in human leukocyte antigen class I and other genes associated with antigen presentation also emerged. Increasing CNA, but not increasing mutational load, was associated with immune evasion in patients treated with immunotherapy. Taken together, these data suggest that CNA, rather than mutational accumulation, is enriched during cancer metastasis, perhaps due to a more favorable balance of enhanced cellular fitness versus immunogenicity. A pan-cancer analysis of genomic data from matched primary and metastatic tumors from 3,732 patients shows that increased genomic complexity and alterations that facilitate immune evasion are associated with disease progression.},
  archive      = {J_NG},
  author       = {Zhao, Karena and Vos, Joris and Lam, Stanley and Boe, Lillian A. and Muldoon, Daniel and Han, Catherine Y. and Valero, Cristina and Lee, Mark and Fitzgerald, Conall and Lee, Andrew S. and Prasad, Manu and Jain, Swati and Deng, Xinzhu and Chan, Timothy A. and Berger, Michael F. and Bandlamudi, Chaitanya and Zhou, Xi Kathy and Morris, Luc G. T.},
  doi          = {10.1038/s41588-025-02204-3},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1504-1511},
  shortjournal = {Nat. Genet.},
  title        = {Longitudinal and multisite sampling reveals mutational and copy number evolution in tumors during metastatic dissemination},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genomic landscape of multiple myeloma and its precursor conditions. <em>NG</em>, <em>57</em>(6), 1493-1503. (<a href='https://doi.org/10.1038/s41588-025-02196-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reliable strategies to capture patients at risk of progression from precursor stages of multiple myeloma (MM) to overt disease are still missing. We assembled a comprehensive collection of MM genomic data comprising 1,030 patients (218 with precursor conditions) that we used to identify recurrent coding and non-coding candidate drivers as well as significant hotspots of structural variation. We used those drivers to define and validate a simple ‘MM-like’ score, which we could use to place patients’ tumors on a gradual axis of progression toward active disease. Our MM precursor genomic map provides insights into the time of initiation and cell-of-origin of the disease, order of acquisition of genomic alterations and mutational processes found across the stages of transformation. Taken together, we highlight here the potential of genome sequencing to better inform risk assessment and monitoring of MM precursor conditions. The genomic features of precursor conditions of multiple myeloma provide multiple biological insights into disease origins and evolution, together with opportunities to identify those at highest risk of progression.},
  archive      = {J_NG},
  author       = {Alberge, Jean-Baptiste and Dutta, Ankit K. and Poletti, Andrea and Coorens, Tim H. H. and Lightbody, Elizabeth D. and Toenges, Rosa and Loinaz, Xavi and Wallin, Sofia and Dunford, Andrew and Priebe, Oliver and Dagan, Johnathan and Boehner, Cody J. and Horowitz, Erica and Su, Nang K. and Barr, Hadley and Hevenor, Laura and Towle, Katherine and Beesam, Rashmika and Beckwith, Jenna B. and Perry, Jacqueline and Cordas dos Santos, David M. and Bertamini, Luca and Greipp, Patricia T. and Kübler, Kirsten and Arndt, Peter F. and Terragna, Carolina and Zamagni, Elena and Boyle, Eileen M. and Yong, Kwee and Morgan, Gareth and Walker, Brian A. and Dimopoulos, Meletios Athanasios and Kastritis, Efstathios and Hess, Julian and Sklavenitis-Pistofidis, Romanos and Stewart, Chip and Getz, Gad and Ghobrial, Irene M.},
  doi          = {10.1038/s41588-025-02196-0},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1493-1503},
  shortjournal = {Nat. Genet.},
  title        = {Genomic landscape of multiple myeloma and its precursor conditions},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chromothripsis-associated chromosome 21 amplification orchestrates transformation to blast-phase MPN through targetable overexpression of DYRK1A. <em>NG</em>, <em>57</em>(6), 1478-1492. (<a href='https://doi.org/10.1038/s41588-025-02190-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromothripsis, the chaotic shattering and repair of chromosomes, is common in cancer. Whether chromothripsis generates actionable therapeutic targets remains an open question. In a cohort of 64 patients in blast phase of a myeloproliferative neoplasm (BP-MPN), we describe recurrent amplification of a region of chromosome 21q (‘chr. 21amp’) in 25%, driven by chromothripsis in a third of these cases. We report that chr. 21amp BP-MPN has a particularly aggressive and treatment-resistant phenotype. DYRK1A, a serine threonine kinase, is the only gene in the 2.7-megabase minimally amplified region that showed both increased expression and chromatin accessibility compared with non-chr. 21amp BP-MPN controls. DYRK1A is a central node at the nexus of multiple cellular functions critical for BP-MPN development and is essential for BP-MPN cell proliferation in vitro and in vivo, and represents a druggable axis. Collectively, these findings define chr. 21amp as a prognostic biomarker in BP-MPN, and link chromothripsis to a therapeutic target. Multiomic analysis of blast-phase myeloproliferative neoplasms identifies a chromosome 21 amplicon harboring DYRK1A as a clonal and therapeutically targetable event in around a quarter of cases.},
  archive      = {J_NG},
  author       = {Brierley, Charlotte K. and Yip, Bon Ham and Orlando, Giulia and Wen, Jeremy and Wen, Sean and Goyal, Harsh and Levine, Max and Jakobsdottir, G. Maria and Tapinos, Avraam and Cornish, Alex J. and Rodriguez-Romera, Antonio and Rodriguez-Meira, Alba and Bashton, Matthew and Hamblin, Angela and Clark, Sally Ann and Hamley, Joseph C. and Fox, Olivia and Giurgiu, Madalina and O’Sullivan, Jennifer and Murphy, Lauren and Adamo, Assunta and Olijnik, Aude Anais and Cotton, Anitria and Hendrix, Emily and Narina, Shilpa and Pruett-Miller, Shondra M. and Enshaei, Amir and Harrison, Claire and Drummond, Mark and Knapper, Steven and Tefferi, Ayalew and Antony-Debré, Iléana and Davies, James and Henssen, Anton G. and Thongjuea, Supat and Wedge, David C. and Constantinescu, Stefan N. and Papaemmanuil, Elli and Psaila, Bethan and Crispino, John D. and Mead, Adam J.},
  doi          = {10.1038/s41588-025-02190-6},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1478-1492},
  shortjournal = {Nat. Genet.},
  title        = {Chromothripsis-associated chromosome 21 amplification orchestrates transformation to blast-phase MPN through targetable overexpression of DYRK1A},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). KDM4C inhibition blocks tumor growth in basal breast cancer by promoting cathepsin L-mediated histone h3 cleavage. <em>NG</em>, <em>57</em>(6), 1463-1477. (<a href='https://doi.org/10.1038/s41588-025-02197-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Basal breast cancer is a subtype with a poor prognosis in need of more effective therapeutic approaches. Here we describe a unique role for the KDM4C histone lysine demethylase in KDM4C-amplified basal breast cancers, where KDM4C inhibition reshapes chromatin and transcriptomic landscapes without substantial alterations of its canonical substrates, trimethylated histone H3 lysine 9 (H3K9me3) and lysine 36 (H3K36me3). Rather, KDM4C loss causes proteolytic cleavage of histone H3 mediated by cathepsin L (CTSL), resulting in decreased glutamate–cysteine ligase expression and increased reactive oxygen species. CTSL is recruited to the chromatin by the grainyhead-like 2 (GRHL2) transcription factor that is methylated at lysine 453 following KDM4C inhibition, triggering CTSL histone clipping activity. Deletion of CTSL rescued KDM4-loss-mediated tumor suppression. Our study reveals a function for KDM4C that connects cellular redox regulation and chromatin remodeling. KDM4C regulates cathepsin L-mediated histone H3 N-terminal tail clipping through grainyhead-like 2 (GRHL2) methylation in breast cancer. This link between the cellular redox state and chromatin remodeling might represent a therapeutic target in KDM4C-amplified tumors.},
  archive      = {J_NG},
  author       = {Li, Zheqi and Peluffo, Guillermo and Stevens, Laura E. and Qiu, Xintao and Seehawer, Marco and Tawawalla, Amatullah and Huang, Xiao-Yun and Egri, Shawn B. and Raval, Shaunak and McFadden, Maeve and D’Santos, Clive S. and Papachristou, Eva and Kingston, Natalie L. and Nishida, Jun and Evans, Kyle E. and Seo, Ji-Heui and Clement, Kendell and Temko, Daniel and Ekram, Muhammad and Li, Rong and Rees, Matthew G. and Ronan, Melissa M. and Roth, Jennifer A. and Simeonov, Anton and Kales, Stephen C. and Rai, Ganesha and Lal-Nag, Madhu and Maloney, David J. and Jadhav, Ajit and Michor, Franziska and Meissner, Alex and Balko, Justin M. and Carroll, Jason S. and Freedman, Matthew L. and Jaffe, Jacob D. and Papanastasiou, Malvina and Long, Henry W. and Polyak, Kornelia},
  doi          = {10.1038/s41588-025-02197-z},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1463-1477},
  shortjournal = {Nat. Genet.},
  title        = {KDM4C inhibition blocks tumor growth in basal breast cancer by promoting cathepsin L-mediated histone h3 cleavage},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). APOBEC3 mutagenesis drives therapy resistance in breast cancer. <em>NG</em>, <em>57</em>(6), 1452-1462. (<a href='https://doi.org/10.1038/s41588-025-02187-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Acquired genetic alterations drive resistance to endocrine and targeted therapies in metastatic breast cancer; however, the underlying processes engendering these alterations are largely uncharacterized. To identify the underlying mutational processes, we utilized a clinically annotated cohort of 3,880 patient samples with tumor-normal sequencing. Mutational signatures associated with apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like 3 (APOBEC3) enzymes were prevalent and enriched in post-treatment hormone receptor-positive cancers. These signatures correlated with shorter progression-free survival on antiestrogen plus CDK4/6 inhibitor therapy in hormone receptor-positive metastatic breast cancer. Whole-genome sequencing of breast cancer models and paired primary-metastatic samples demonstrated that active APOBEC3 mutagenesis promoted therapy resistance through characteristic alterations such as RB1 loss. Evidence of APOBEC3 activity in pretreatment samples illustrated its pervasive role in breast cancer evolution. These studies reveal APOBEC3 mutagenesis to be a frequent mediator of therapy resistance in breast cancer and highlight its potential as a biomarker and target for overcoming resistance. Analysis of a large cohort of metastatic breast cancer samples shows that APOBEC mutational signatures are enriched in post-treatment samples. APOBEC activity was also associated with mutations known to drive drug resistance.},
  archive      = {J_NG},
  author       = {Gupta, Avantika and Gazzo, Andrea and Selenica, Pier and Safonov, Anton and Pareja, Fresia and da Silva, Edaise M. and Brown, David N. and Shao, Hong and Zhu, Yingjie and Patel, Juber and Blanco-Heredia, Juan and Stefanovska, Bojana and Carpenter, Michael A. and Chen, Yanjun and Vegas, Isabella and Pei, Xin and Frosina, Denise and Jungbluth, Achim A. and Ladanyi, Marc and Curigliano, Giuseppe and Weigelt, Britta and Riaz, Nadeem and Powell, Simon N. and Razavi, Pedram and Harris, Reuben S. and Reis-Filho, Jorge S. and Marra, Antonio and Chandarlapaty, Sarat},
  doi          = {10.1038/s41588-025-02187-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1452-1462},
  shortjournal = {Nat. Genet.},
  title        = {APOBEC3 mutagenesis drives therapy resistance in breast cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Base editing of trinucleotide repeats that cause huntington’s disease and friedreich’s ataxia reduces somatic repeat expansions in patient cells and in mice. <em>NG</em>, <em>57</em>(6), 1437-1451. (<a href='https://doi.org/10.1038/s41588-025-02172-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Trinucleotide repeat (TNR) diseases are neurological disorders caused by expanded genomic TNRs that become unstable in a length-dependent manner. The CAG•CTG sequence is found in approximately one-third of pathogenic TNR loci, including the HTT gene that causes Huntington’s disease. Friedreich’s ataxia, the most prevalent hereditary ataxia, results from GAA repeat expansion at the FXN gene. Here we used cytosine and adenine base editing to reduce the repetitiveness of TNRs in patient cells and in mice. Base editors introduced G•C&gt;A•T and A•T&gt;G•C interruptions at CAG and GAA repeats, mimicking stable, nonpathogenic alleles that naturally occur in people. AAV9 delivery of optimized base editors in Htt.Q111 Huntington’s disease and YG8s Friedreich’s ataxia mice resulted in efficient editing in transduced tissues, and significantly reduced repeat expansion in the central nervous system. These findings demonstrate that introducing interruptions in pathogenic TNRs can mitigate a key neurological feature of TNR diseases in vivo. Base editing of the pathogenic trinucleotide repeat expansions underlying Huntington’s disease and Friedreich’s ataxia introduces repeat interruptions that reduce somatic expansion in patient cells and mice.},
  archive      = {J_NG},
  author       = {Matuszek, Zaneta and Arbab, Mandana and Kesavan, Maheswaran and Hsu, Alvin and Roy, Jennie C. L. and Zhao, Jing and Yu, Tian and Weisburd, Ben and Newby, Gregory A. and Doherty, Neil J. and Wu, Muzhou and Shibata, Shota and Cristian, Ana and Tao, Y. Allen and Fearnley, Liam G. and Bahlo, Melanie and Rehm, Heidi L. and Xie, Jun and Gao, Guangping and Mouro Pinto, Ricardo and Liu, David R.},
  doi          = {10.1038/s41588-025-02172-8},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1437-1451},
  shortjournal = {Nat. Genet.},
  title        = {Base editing of trinucleotide repeats that cause huntington’s disease and friedreich’s ataxia reduces somatic repeat expansions in patient cells and in mice},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic modifiers of somatic expansion and clinical phenotypes in huntington’s disease highlight shared and tissue-specific effects. <em>NG</em>, <em>57</em>(6), 1426-1436. (<a href='https://doi.org/10.1038/s41588-025-02191-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An inherited, expanded CAG repeat in HTT undergoes further somatic expansion to cause Huntington’s disease (HD). To gain insights into this molecular mechanism, we compared genome-wide association studies of somatic expansion in blood and somatic expansion-driven HD clinical phenotypes. Here, we show that somatic expansion is driven by a mismatch repair-related process whose genetic modification and consequences show unexpected complexity, including cell-type specificity. The HD clinical trajectory is further modified by non-DNA repair genes that differentially influence measures of cognitive and motor dysfunction. In addition to shared (DNA repair genes MSH3, PMS2 and FAN1) and distinct trans-modifiers, a synonymous CAG-adjacent variant in HTT dramatically hastens motor onset without increasing somatic expansion, while a cis-acting 5′-untranslated region variant promotes blood repeat expansion without influencing clinical HD. Our findings are directly relevant to the therapeutic suppression of expansion in DNA repeat disorders and provide additional clues to HD pathogenic mechanisms beyond somatic expansion. Comparison of genome-wide association studies of HTT CAG repeat expansion in blood to expansion-driven clinical traits in Huntington’s disease identifies shared and distinct modifiers implicating DNA mismatch repair with tissue and cell-type specificity.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02191-5},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1426-1436},
  shortjournal = {Nat. Genet.},
  title        = {Genetic modifiers of somatic expansion and clinical phenotypes in huntington’s disease highlight shared and tissue-specific effects},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The contribution of gametic phase disequilibrium to the heritability of complex traits. <em>NG</em>, <em>57</em>(6), 1418-1425. (<a href='https://doi.org/10.1038/s41588-025-02192-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nonrandom mating induces genome-wide correlations between unlinked genetic variants, known as gametic phase disequilibrium (GPD), whose contribution to heritability remains uncharacterized. Here we introduce the disequilibrium genome-based restricted maximum likelihood (DGREML) method to simultaneously quantify the additive contribution of SNPs to heritability and that of their directional covariances. We applied DGREML to 26 phenotypes of 550,000 individuals from diverse biobanks and found that cross-autosome GPD contributes 10–27% of the SNP-based heritability of height, educational attainment, intelligence, income, self-rated health status and sedentary behaviors. We observed a differential contribution of GPD to the heritability of height between the UK, Chinese and Japanese populations. Finally, bivariate DGREML analyses of educational attainment and height show that cross-autosome GPD contributes at least 32% of their genetic correlation. Altogether, our versatile and powerful method reveals understudied features of the genetic architecture of complex traits and informs potential mechanisms generating these features. Disequilibrium genome-based restricted maximum likelihood (DGREML) simultaneously quantifies the contribution of SNPs and their directional covariances to trait heritability and shows that cross-autosomal correlations contribute substantially to SNP-based heritability for many complex traits.},
  archive      = {J_NG},
  author       = {Zhang, Yuanxiang and Sakaue, Saori and Morris, Sam and Graff, Mariaelisa and Yamamoto, Kenichi and Chen, Zhengming and Li, Liming and Keller, Matthew C. and Goddard, Michael E. and Walters, Robin G. and Okada, Yukinori and Visscher, Peter M. and Yengo, Loic},
  doi          = {10.1038/s41588-025-02192-4},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1418-1425},
  shortjournal = {Nat. Genet.},
  title        = {The contribution of gametic phase disequilibrium to the heritability of complex traits},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome-wide association study of long COVID. <em>NG</em>, <em>57</em>(6), 1402-1417. (<a href='https://doi.org/10.1038/s41588-025-02100-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Infections can lead to persistent symptoms and diseases such as shingles after varicella zoster or rheumatic fever after streptococcal infections. Similarly, severe acute respiratory syndrome coronavirus 2 (SARS‑CoV‑2) infection can result in long coronavirus disease (COVID), typically manifesting as fatigue, pulmonary symptoms and cognitive dysfunction. The biological mechanisms behind long COVID remain unclear. We performed a genome-wide association study for long COVID including up to 6,450 long COVID cases and 1,093,995 population controls from 24 studies across 16 countries. We discovered an association of FOXP4 with long COVID, independent of its previously identified association with severe COVID-19. The signal was replicated in 9,500 long COVID cases and 798,835 population controls. Given the transcription factor FOXP4’s role in lung physiology and pathology, our findings highlight the importance of lung function in the pathophysiology of long COVID. A genome-wide study by the Long COVID Host Genetics Initiative identifies an association between the FOXP4 locus and long COVID, implicating altered lung function in its pathophysiology.},
  archive      = {J_NG},
  author       = {Lammi, Vilma and Nakanishi, Tomoko and Jones, Samuel E. and Andrews, Shea J. and Karjalainen, Juha and Cortés, Beatriz and O’Brien, Heath E. and Ochoa-Guzman, Ana and Fulton-Howard, Brian E. and Broberg, Martin and Haapaniemi, Hele H. and Kanai, Masahiro and Pirinen, Matti and Schmidt, Axel and Mitchell, Ruth E. and Mousas, Abdou and Mangino, Massimo and Huerta-Chagoya, Alicia and Sinnott-Armstrong, Nasa and Cirulli, Elizabeth T. and Vaudel, Marc and Kwong, Alex S. F. and Maiti, Amit K. and Marttila, Minttu M. and Posner, Daniel C. and Rodriguez, Alexis A. and Batini, Chiara and Minnai, Francesca and Dearman, Anna R. and Warmerdam, C. A. Robert and Sequeros, Celia B. and Winkler, Thomas W. and Jordan, Daniel M. and Rešcenko, Raimonds and Miano, Lorenzo and Lane, Jacqueline M. and Chung, Ryan K. and Guillen-Guio, Beatriz and Leavy, Olivia C. and Carvajal-Silva, Laura and Aguilar-Valdés, Kevin and Frangione, Erika and Guare, Lindsay and Vergasova, Ekaterina and Marouli, Eirini and Striano, Pasquale and Zainulabid, Ummu Afeera and Kumar, Ashutosh and Ahmad, Hajar Fauzan and Edahiro, Ryuya and Azekawa, Shuhei and Luoh, Shiuh-Wen and Erikstrup, Christian and Pedersen, Ole B. V. and Lerner-Ellis, Jordan and Colombo, Alicia and Grzymski, Joseph J. and Ishii, Makoto and Okada, Yukinori and Beckmann, Noam D. and Kumari, Meena and Wagner, Ralf and Heid, Iris M. and John, Catherine and Short, Patrick J. and Magnus, Per and Ansone, Laura and Valenti, Luca V. C. and Lee, Sulggi A. and Wain, Louise V. and Verdugo, Ricardo A. and Banasik, Karina and Geller, Frank and Franke, Lude H. and Rakitko, Alexander and Duncan, Emma L. and Renieri, Alessandra and Tsilidis, Konstantinos K. and de Cid, Rafael and Niavarani, Ahmadreza and Abner, Erik and Tusié-Luna, Teresa and Verma, Shefali S. and Smith, George Davey and Timpson, Nicholas J. and Madduri, Ravi K. and Cho, Kelly and Daly, Mark J. and Ganna, Andrea and Schulte, Eva C. and Richards, J. Brent and Ludwig, Kerstin U. and Marks-Hultström, Michael and Zeberg, Hugo and Ollila, Hanna M.},
  doi          = {10.1038/s41588-025-02100-w},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1402-1417},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association study of long COVID},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome-wide analyses identify 30 loci associated with obsessive–compulsive disorder. <em>NG</em>, <em>57</em>(6), 1389-1401. (<a href='https://doi.org/10.1038/s41588-025-02189-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Obsessive–compulsive disorder (OCD) affects ~1% of children and adults and is partly caused by genetic factors. We conducted a genome-wide association study (GWAS) meta-analysis combining 53,660 OCD cases and 2,044,417 controls and identified 30 independent genome-wide significant loci. Gene-based approaches identified 249 potential effector genes for OCD, with 25 of these classified as the most likely causal candidates, including WDR6, DALRD3 and CTNND1 and multiple genes in the major histocompatibility complex (MHC) region. We estimated that ~11,500 genetic variants explained 90% of OCD genetic heritability. OCD genetic risk was associated with excitatory neurons in the hippocampus and the cortex, along with D1 and D2 type dopamine receptor-containing medium spiny neurons. OCD genetic risk was shared with 65 of 112 additional phenotypes, including all the psychiatric disorders we examined. In particular, OCD shared genetic risk with anxiety, depression, anorexia nervosa and Tourette syndrome and was negatively associated with inflammatory bowel diseases, educational attainment and body mass index. Genome-wide analyses identify 30 independent loci associated with obsessive–compulsive disorder, highlighting genetic overlap with other psychiatric disorders and implicating putative effector genes and cell types contributing to its etiology.},
  archive      = {J_NG},
  author       = {Strom, Nora I. and Gerring, Zachary F. and Galimberti, Marco and Yu, Dongmei and Halvorsen, Matthew W. and Abdellaoui, Abdel and Rodriguez-Fontenla, Cristina and Sealock, Julia M. and Bigdeli, Tim and Coleman, Jonathan R. and Mahjani, Behrang and Thorp, Jackson G. and Bey, Katharina and Burton, Christie L. and Luykx, Jurjen J. and Zai, Gwyneth and Alemany, Silvia and Andre, Christine and Askland, Kathleen D. and Bäckman, Julia and Banaj, Nerisa and Barlassina, Cristina and Nissen, Judith Becker and Bienvenu, O. Joseph and Black, Donald and Bloch, Michael H. and Børte, Sigrid and Bosch, Rosa and Breen, Michael and Brennan, Brian P. and Brentani, Helena and Buxbaum, Joseph D. and Bybjerg-Grauholm, Jonas and Byrne, Enda M. and Cabana-Dominguez, Judit and Camarena, Beatriz and Camarena, Adrian and Cappi, Carolina and Carracedo, Angel and Casas, Miguel and Cavallini, Maria Cristina and Ciullo, Valentina and Cook, Edwin H. and Crosby, Jesse and Cullen, Bernadette A. and De Schipper, Elles J. and Delorme, Richard and Djurovic, Srdjan and Elias, Jason A. and Estivill, Xavier and Falkenstein, Martha J. and Fundin, Bengt T. and Garner, Lauryn and Gironda, Christina and Goes, Fernando S. and Grados, Marco A. and Grove, Jakob and Guo, Wei and Haavik, Jan and Hagen, Kristen and Harrington, Kelly and Havdahl, Alexandra and Höffler, Kira D. and Hounie, Ana G. and Hucks, Donald and Hultman, Christina and Janecka, Magdalena and Jenike, Eric and Karlsson, Elinor K. and Kelley, Kara and Klawohn, Julia and Krasnow, Janice E. and Krebs, Kristi and Lange, Christoph and Lanzagorta, Nuria and Levey, Daniel and Lindblad-Toh, Kerstin and Macciardi, Fabio and Maher, Brion and Mathes, Brittany and McArthur, Evonne and McGregor, Nathaniel and McLaughlin, Nicole C. and Meier, Sandra and Miguel, Euripedes C. and Mulhern, Maureen and Nestadt, Paul S. and Nurmi, Erika L. and O’Connell, Kevin S. and Osiecki, Lisa and Ousdal, Olga Therese and Palviainen, Teemu and Pedersen, Nancy L. and Piras, Fabrizio and Piras, Federica and Potluri, Sriramya and Rabionet, Raquel and Ramirez, Alfredo and Rauch, Scott and Reichenberg, Abraham and Riddle, Mark A. and Ripke, Stephan and Rosário, Maria C. and Sampaio, Aline S. and Schiele, Miriam A. and Skogholt, Anne Heidi and Sloofman, Laura G. and Smit, Jan and Artigas, María Soler and Thomas, Laurent F. and Tifft, Eric and Vallada, Homero and van Kirk, Nathanial and Veenstra-VanderWeele, Jeremy and Vulink, Nienke N. and Walker, Christopher P. and Wang, Ying and Wendland, Jens R. and Winsvold, Bendik S. and Yao, Yin and Zhou, Hang and Agrawal, Arpana and Alonso, Pino and Berberich, Götz and Bucholz, Kathleen K. and Bulik, Cynthia M. and Cath, Danielle and Denys, Damiaan and Eapen, Valsamma and Edenberg, Howard and Falkai, Peter and Fernandez, Thomas V. and Fyer, Abby J. and Gaziano, J. M. and Geller, Dan A. and Grabe, Hans J. and Greenberg, Benjamin D. and Hanna, Gregory L. and Hickie, Ian B. and Hougaard, David M. and Kathmann, Norbert and Kennedy, James and Lai, Dongbing and Landén, Mikael and Hellard, Stéphanie Le and Leboyer, Marion and Lochner, Christine and McCracken, James T. and Medland, Sarah E. and Mortensen, Preben B. and Neale, Benjamin M. and Nicolini, Humberto and Nordentoft, Merete and Pato, Michele and Pato, Carlos and Pauls, David L. and Piacentini, John and Pittenger, Christopher and Posthuma, Danielle and Ramos-Quiroga, Josep Antoni and Rasmussen, Steven A. and Richter, Margaret A. and Rosenberg, David R. and Ruhrmann, Stephan and Samuels, Jack F. and Sandin, Sven and Sandor, Paul and Spalletta, Gianfranco and Stein, Dan J. and Stewart, S. Evelyn and Storch, Eric A. and Stranger, Barbara E. and Turiel, Maurizio and Werge, Thomas and Andreassen, Ole A. and Børglum, Anders D. and Walitza, Susanne and Hveem, Kristian and Hansen, Bjarne K. and Rück, Christian and Martin, Nicholas G. and Milani, Lili and Mors, Ole and Reichborn-Kjennerud, Ted and Ribasés, Marta and Kvale, Gerd and Mataix-Cols, David and Domschke, Katharina and Grünblatt, Edna and Wagner, Michael and Zwart, John-Anker and Breen, Gerome and Nestadt, Gerald and Kaprio, Jaakko and Arnold, Paul D. and Grice, Dorothy E. and Knowles, James A. and Ask, Helga and Verweij, Karin J. and Davis, Lea K. and Smit, Dirk J. and Crowley, James J. and Scharf, Jeremiah M. and Stein, Murray B. and Gelernter, Joel and Mathews, Carol A. and Derks, Eske M. and Mattheisen, Manuel},
  doi          = {10.1038/s41588-025-02189-z},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1389-1401},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide analyses identify 30 loci associated with obsessive–compulsive disorder},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dominant variants in major spliceosome u4 and u5 small nuclear RNA genes cause neurodevelopmental disorders through splicing disruption. <em>NG</em>, <em>57</em>(6), 1374-1388. (<a href='https://doi.org/10.1038/s41588-025-02184-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The major spliceosome contains five small nuclear RNAs (snRNAs; U1, U2, U4, U5 and U6) essential for splicing. Variants in RNU4-2, encoding U4, cause a neurodevelopmental disorder called ReNU syndrome. We investigated de novo variants in 50 snRNA-encoding genes in a French cohort of 23,649 individuals with rare disorders and gathered additional cases through international collaborations. Altogether, we identified 145 previously unreported probands with (likely) pathogenic variants in RNU4-2 and 21 individuals with de novo and/or recurrent variants in RNU5B-1 and RNU5A-1, encoding U5. Pathogenic variants typically arose de novo on the maternal allele and cluster in regions critical for splicing. RNU4-2 variants mainly localize to two structures, the stem III and T-loop/quasi-pseudoknot, which position the U6 ACAGAGA box for 5′ splice site recognition and associate with different phenotypic severity. RNU4-2 variants result in specific defects in alternative 5′ splice site usage and methylation patterns (episignatures) that correlate with variant location and clinical severity. This study establishes RNU5B-1 as a neurodevelopmental disorder gene, suggests RNU5A-1 as a strong candidate and highlights the role of de novo variants in snRNAs. Analysis of snRNA genes in individuals with rare disorders identifies de novo and recurrent variants in RNU5B-1 and RNU5A-1, in addition to previously unreported cases with pathogenic or likely pathogenic variants in RNU4-2.},
  archive      = {J_NG},
  author       = {Nava, Caroline and Cogne, Benjamin and Santini, Amandine and Leitão, Elsa and Lecoquierre, François and Chen, Yuyang and Stenton, Sarah L. and Besnard, Thomas and Heide, Solveig and Baer, Sarah and Jakhar, Abhilasha and Neuser, Sonja and Keren, Boris and Faudet, Anne and Forlani, Sylvie and Faoucher, Marie and Uguen, Kevin and Platzer, Konrad and Afenjar, Alexandra and Alessandri, Jean-Luc and Andres, Stephanie and Angelini, Chloé and Aral, Bernard and Arveiler, Benoit and Attie-Bitach, Tania and Aubert Mucca, Marion and Banneau, Guillaume and Barakat, Tahsin Stefan and Barcia, Giulia and Baulac, Stéphanie and Beneteau, Claire and Benkerdou, Fouzia and Bernard, Virginie and Bézieau, Stéphane and Bonneau, Dominique and Bonnet-Dupeyron, Marie-Noelle and Boussion, Simon and Boute, Odile and Brischoux-Boucher, Elise and Bryen, Samantha J. and Buratti, Julien and Busa, Tiffany and Caliebe, Almuth and Capri, Yline and Cassinari, Kévin and Caumes, Roseline and Cenni, Camille and Chambon, Pascal and Charles, Perrine and Christodoulou, John and Colson, Cindy and Conrad, Solène and Cospain, Auriane and Coursimault, Juliette and Courtin, Thomas and Couse, Madeline and Coutton, Charles and Creveaux, Isabelle and D’Gama, Alissa M. and Dauriat, Benjamin and de Sainte Agathe, Jean-Madeleine and Del Gobbo, Giulia and Delahaye-Duriez, Andrée and Delanne, Julian and Denommé-Pichon, Anne-Sophie and Dieux-Coeslier, Anne and Do Souto Ferreira, Laura and Doco-Fenzy, Martine and Drukewitz, Stephan and Duboc, Véronique and Dubourg, Christèle and Duffourd, Yannis and Dyment, David and El Chehadeh, Salima and Elmaleh, Monique and Faivre, Laurence and Fennelly, Samuel and Fischer, Hanna and Fradin, Mélanie and Galludec Vaillant, Camille and Ganne, Benjamin and Ghoumid, Jamal and Goel, Himanshu and Gokce-Samar, Zeynep and Goldenberg, Alice and Gonfreville Robert, Romain and Gorokhova, Svetlana and Goujon, Louise and Granier, Victoria and Gras, Mathilde and Greally, John M. and Greiten, Bianca and Gueguen, Paul and Guerrot, Anne-Marie and Guha, Saurav and Guimier, Anne and Haack, Tobias B. and Hadj Abdallah, Hamza and Halleb, Yosra and Harbuz, Radu and Harris, Madeleine and Hentschel, Julia and Héron, Bénédicte and Hitz, Marc-Phillip and Innes, A. Micheil and Jadas, Vincent and Januel, Louis and Jean-Marçais, Nolwenn and Jobanputra, Vaidehi and Jobic, Florence and Jornea, Ludmila and Jost, Céline and Julia, Sophie and Kaiser, Frank J. and Kaschta, Daniel and Kaya, Sabine and Ketteler, Petra and Khadija, Bochra and Kilpert, Fabian and Knopp, Cordula and Kraft, Florian and Krey, Ilona and Lackmy, Marilyn and Laffargue, Fanny and Lambert, Laetitia and Lamont, Ryan and Laugel, Vincent and Laurie, Steven and Lauzon, Julie L. and Lebreton, Louis and Lebrun, Marine and Legendre, Marine and Leguern, Eric and Lehalle, Daphné and Lejeune, Elodie and Lesca, Gaetan and Lesieur-Sebellin, Marion and Levy, Jonathan and Linglart, Agnès and Lyonnet, Stanislas and Lüthy, Kevin and Ma, Alan S. and Mach, Corinne and Mandel, Jean-Louis and Mansour-Hendili, Lamisse and Marcadier, Julien and Marin, Victor and Margot, Henri and Marquet, Valentine and May, Angèle and Mayr, Johannes A. and Meridda, Catherine and Michaud, Vincent and Michot, Caroline and Nadeau, Gwenael and Naudion, Sophie and Nguyen, Laetitia and Nizon, Mathilde and Nowak, Frédérique and Odent, Sylvie and Olin, Valerie and Osei-Owusu, Ikeoluwa A. and Osmond, Matthew and Õunap, Katrin and Pasquier, Laurent and Passemard, Sandrine and Pauly, Melissa and Patat, Olivier and Pensec, Marine and Perrin-Sabourin, Laurence and Petit, Florence and Philippe, Christophe and Planes, Marc and Poduri, Annapurna and Poirsier, Céline and Pouzet, Antoine and Prince, Bradley and Prouteau, Clément and Pujol, Aurora and Racine, Caroline and Rama, Mélanie and Ramond, Francis and Ranguin, Kara and Raway, Margaux and Reis, André and Renaud, Mathilde and Revencu, Nicole and Richard, Anne-Claire and Riera-Navarro, Lucile and Rius, Rocio and Rodriguez, Diana and Rodriguez-Palmero, Agustí and Rondeau, Sophie and Roser-Unruh, Annika and Rougeot Jung, Christelle and Safraou, Hana and Satre, Véronique and Saugier-Veber, Pascale and Sauvestre, Clément and Schaefer, Elise and Shao, Wanqing and Schanze, Ina and Schlump, Jan-Ulrich and Schlüter Martin, Agatha and Schluth-Bolard, Caroline and Schuhmann, Sarah and Schröder, Christopher and Sebastin, Monisha and Sigaudy, Sabine and Spielmann, Malte and Spodenkiewicz, Marta and St Clair, Laura and Steffann, Julie and Stoeva, Radka and Surowy, Harald and Tarnopolsky, Mark A. and Todosi, Calina and Toutain, Annick and Tran Mau-Them, Frédéric and Unterlauft, Astrid and Van-Gils, Julien and Vanlerberghe, Clémence and Vasileiou, Georgia and Vera, Gabriella and Verdel, André and Verloes, Alain and Vial, Yoann and Vignal, Cédric and Vincent, Marie and Vincent-Delorme, Catherine and Vincent-Devulder, Aline and Vitobello, Antonio and Weber, Sacha and Willems, Marjolaine and Zaafrane-Khachnaoui, Khaoula and Zacher, Pia and Zeltner, Lena and Ziegler, Alban and Galej, Wojciech P. and Dollfus, Hélène and Thauvin, Christel and Boycott, Kym M. and Marijon, Pierre and Lermine, Alban and Malan, Valérie and Rio, Marlène and Kuechler, Alma and Isidor, Bertrand and Drunat, Séverine and Smol, Thomas and Chatron, Nicolas and Piton, Amélie and Nicolas, Gael and Wagner, Matias and Abou Jamra, Rami and Héron, Delphine and Mignot, Cyril and Blanc, Pierre and O’Donnell-Luria, Anne and Whiffin, Nicola and Charbonnier, Camille and Charenton, Clément and Thevenon, Julien and Depienne, Christel},
  doi          = {10.1038/s41588-025-02184-4},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1374-1388},
  shortjournal = {Nat. Genet.},
  title        = {Dominant variants in major spliceosome u4 and u5 small nuclear RNA genes cause neurodevelopmental disorders through splicing disruption},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mutations in the small nuclear RNA gene RNU2-2 cause a severe neurodevelopmental disorder with prominent epilepsy. <em>NG</em>, <em>57</em>(6), 1367-1373. (<a href='https://doi.org/10.1038/s41588-025-02159-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The major spliceosome includes five small nuclear RNA (snRNAs), U1, U2, U4, U5 and U6, each of which is encoded by multiple genes. We recently showed that mutations in RNU4-2, the gene that encodes the U4-2 snRNA, cause one of the most prevalent monogenic neurodevelopmental disorders. Here, we report that recurrent germline mutations in RNU2-2 (previously known as pseudogene RNU2-2P), a 191-bp gene that encodes the U2-2 snRNA, are responsible for a related disorder. By genetic association, we identified recurrent de novo single-nucleotide mutations at nucleotide positions 4 and 35 of RNU2-2 in nine cases. We replicated this finding in 16 additional cases, bringing the total to 25. We estimate that RNU2-2 syndrome has a prevalence of ~20% that of RNU4-2 syndrome. The disorder is characterized by intellectual disability, autistic behavior, microcephaly, hypotonia, epilepsy and hyperventilation. All cases display a severe and complex seizure phenotype. We found that U2-2 and canonical U2-1 were similarly expressed in blood. Despite mutant U2-2 being expressed in patient blood samples, we found no evidence of missplicing. Our findings cement the role of major spliceosomal snRNAs in the etiologies of neurodevelopmental disorders. Recurrent de novo mutations at nucleotide positions 4 and 35 of RNU2-2 cause a neurodevelopmental disorder whose prominent features include intellectual disability, developmental delay and a complex seizure phenotype.},
  archive      = {J_NG},
  author       = {Greene, Daniel and De Wispelaere, Koenraad and Lees, Jon and Codina-Solà, Marta and Jensson, Brynjar O. and Hales, Emma and Katrinecz, Andrea and Nieto Molina, Esther and Pascoal, Sonia and Pfundt, Rolph and Schot, Rachel and Sevilla Porras, Marta and Sleutels, Frank and Valenzuela, Irene and Wijngaard, Robin and Arroyo Carrera, Ignacio and Atton, Giles and Casas-Alba, Didac and Donnelly, Deirdre and Duat Rodríguez, Anna and Fernández Garoz, Bárbara and Foulds, Nicola and García-Navas Núñez, Deyanira and González Alguacil, Elena and Jarvis, Joanna and Kant, Sarina G. and Madrigal Bajo, Irene and Martinez-Monseny, Antonio F. and McKee, Shane and Ortiz Cabrera, Nelmar Valentina and Rodríguez-Revenga Bodi, Laia and Sariego Jamardo, Andrea and Stefansson, Kari and Sulem, Patrick and Suri, Mohnish and Van Karnebeek, Clara and Vasudevan, Pradeep and Vega Pajares, Ana Isabel and Carracedo, Ángel and Engelen, Marc and Lapunzina, Pablo and Morgan, Natasha P. and Morte, Beatriz and Rump, Patrick and Stirrups, Kathy and Tizzano, Eduardo F. and Barakat, Tahsin Stefan and O’Donoghue, Michael and Pérez-Jurado, Luis Alberto and Freson, Kathleen and Mumford, Andrew D. and Turro, Ernest},
  doi          = {10.1038/s41588-025-02159-5},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1367-1373},
  shortjournal = {Nat. Genet.},
  title        = {Mutations in the small nuclear RNA gene RNU2-2 cause a severe neurodevelopmental disorder with prominent epilepsy},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Analysis of R-loop forming regions identifies RNU2-2 and RNU5B-1 as neurodevelopmental disorder genes. <em>NG</em>, <em>57</em>(6), 1362-1366. (<a href='https://doi.org/10.1038/s41588-025-02209-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {R-loops are DNA–RNA hybrid structures that may promote mutagenesis. However, their contribution to human Mendelian disorders is unexplored. Here we show excess de novo variants in genomic regions that form R-loops (henceforth, ‘R-loop regions’) and demonstrate enrichment of R-loop region variants (RRVs) in ribozyme, snoRNA and snRNA genes, specifically in rare disease cohorts. Using this insight, we report neurodevelopmental disorders (NDDs) caused by rare variants in two major spliceosomal RNA encoding genes, RNU2-2 and RNU5B-1. These, along with the recently described RNU4-2-related ReNU syndrome, provide a genetic explanation for a substantial proportion of individuals with NDDs. Genomic analyses focused on regions that form R-loops identify rare mutations in RNU2-2 and RNU5B-1 in individuals with neurodevelopmental disorders.},
  archive      = {J_NG},
  author       = {Jackson, Adam and Thaker, Nishi and Blakes, Alexander and Rice, Gillian and Griffiths-Jones, Sam and Balasubramanian, Meena and Campbell, Jennifer and Shannon, Nora and Choi, Jungmin and Hong, Juhyeon and Hunt, David and de Burca, Anna and Kim, Soo Yeon and Kim, Taekeun and Lee, Seungbok and Redman, Melody and Rius, Rocio and Simons, Cas and Tan, Tiong Yang and Ellingford, Jamie and O’Keefe, Raymond T. and Chae, Jong Hee and Banka, Siddharth},
  doi          = {10.1038/s41588-025-02209-y},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1362-1366},
  shortjournal = {Nat. Genet.},
  title        = {Analysis of R-loop forming regions identifies RNU2-2 and RNU5B-1 as neurodevelopmental disorder genes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Phylogenetic inference reveals clonal heterogeneity in circulating tumor cell clusters. <em>NG</em>, <em>57</em>(6), 1357-1361. (<a href='https://doi.org/10.1038/s41588-025-02205-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Circulating tumor cell (CTC) clusters are highly efficient metastatic seeds in various cancers. Yet, their genetic heterogeneity and clonal architecture is poorly characterized. Using whole-exome sequencing coupled with phylogenetic inference from CTC clusters of patients with breast and prostate cancer, as well as mouse cancer models alongside barcode-mediated clonal tracking in vivo, we demonstrate oligoclonal composition of individual CTC clusters. These results improve our understanding of metastasis-relevant clonal dynamics. Phylogenetic analyses in patients with cancer and xenograft models highlight heterogeneity within individual CTC clusters, providing insights into clonal dynamics during metastasis.},
  archive      = {J_NG},
  author       = {Gremmelspacher, David and Gawron, Johannes and Szczerba, Barbara M. and Jahn, Katharina and Castro-Giner, Francesc and Kuipers, Jack and Singer, Jochen and Marass, Francesco and Gvozdenovic, Ana and Budinjas, Selina and Pueschel, Heike and Rentsch, Cyrill A. and Zippelius, Alfred and Heinzelmann-Schwarz, Viola and Kurzeder, Christian and Weber, Walter Paul and Rochlitz, Christoph and Vetter, Marcus and Beerenwinkel, Niko and Aceto, Nicola},
  doi          = {10.1038/s41588-025-02205-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1357-1361},
  shortjournal = {Nat. Genet.},
  title        = {Phylogenetic inference reveals clonal heterogeneity in circulating tumor cell clusters},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering next-generation microfluidic technologies for single-cell phenomics. <em>NG</em>, <em>57</em>(6), 1344-1356. (<a href='https://doi.org/10.1038/s41588-025-02198-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The completion of the Human Genome Project catalyzed the development of ‘omics’ technologies, enabling the detailed exploration of biological systems at an unprecedented molecular scale. Microfluidics has transformed the omics toolbox by facilitating large-scale, high-throughput and highly accurate measurements of DNA and RNA, driving the transition from bulk to single-cell analyses. This transition has ushered in a new era, moving beyond a gene- and protein-centric perspective toward a holistic understanding of cellular phenotypes. This emerging ‘single-cell phenomics era’ integrates diverse omics datasets with spatial, morphological and temporal phenotypes to provide a comprehensive perspective on cellular function. This Review highlights how microfluidics addressed key challenges in the transition to single-cell omics and explores how lessons learned from these efforts will propel the single-cell phenomics revolution. Furthermore, we discuss emerging opportunities in which integrative single-cell phenomics could serve as a foundation for transformative discoveries in biology. Research is moving from a gene- and protein-centric view toward a holistic understanding of cellular phenotypes. This Review discusses the technological microfluidics challenges that must be addressed to realize integrative single-cell phenomics.},
  archive      = {J_NG},
  author       = {Lambert, Camille L. G. and van Mierlo, Guido and Bues, Johannes J. and Guillaume-Gentil, Orane J. and Deplancke, Bart},
  doi          = {10.1038/s41588-025-02198-y},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1344-1356},
  shortjournal = {Nat. Genet.},
  title        = {Engineering next-generation microfluidic technologies for single-cell phenomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Toward clinical long-read genome sequencing for rare diseases. <em>NG</em>, <em>57</em>(6), 1334-1343. (<a href='https://doi.org/10.1038/s41588-025-02160-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic diagnostics is driven by technological advances, forming a tight interface between research, clinic and industry, which enables rapid implementation of new technologies. Short-read genome and exome sequencing, the current state of the art in clinical genetics, can detect a broad spectrum of genetic variants across the genome. However, despite these advancements, more than half of individuals with rare diseases remain undiagnosed after genomic investigations. Long-read whole-genome sequencing (LR-WGS) is a promising technology that identifies previously difficult-to-detect variants while also enabling phasing and methylation analysis and has the potential of generating complete personal assemblies. To pave the way for clinical use of LR-WGS, the clinical genomic community must establish standardized protocols and quality parameters while also developing innovative tools for data analysis and interpretation. In this Perspective, we explore the key challenges and benefits in integrating LR-WGS into routine clinical diagnostics. Long-read genome sequencing has the potential to dramatically impact genetic diagnostics of rare diseases. This Perspective discusses the key challenges and benefits of the technology that will enable its more streamlined clinical implementation.},
  archive      = {J_NG},
  author       = {Eisfeldt, Jesper and Ek, Marlene and Nordenskjöld, Magnus and Lindstrand, Anna},
  doi          = {10.1038/s41588-025-02160-y},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1334-1343},
  shortjournal = {Nat. Genet.},
  title        = {Toward clinical long-read genome sequencing for rare diseases},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A simple genomic score to predict progression of multiple myeloma. <em>NG</em>, <em>57</em>(6), 1332-1333. (<a href='https://doi.org/10.1038/s41588-025-02199-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ‘multiple myeloma-like’ genomic score identifies somatic mutations that are more commonly found in active forms of multiple myeloma. In patients with a precursor stage of multiple myeloma, this score helps in predicting who is at a higher risk of progressing to an active form of cancer and may benefit from early therapeutic interventions.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02199-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1332-1333},
  shortjournal = {Nat. Genet.},
  title        = {A simple genomic score to predict progression of multiple myeloma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). KDM4C shields breast cancer from the histone scissor cathepsin l. <em>NG</em>, <em>57</em>(6), 1330-1331. (<a href='https://doi.org/10.1038/s41588-025-02200-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We uncovered a mechanism by which KDM4C promotes tumorigenesis in basal breast cancer with KDM4C amplification. Using knockdown models, we showed that KDM4C prevents activation of cathepsin L by demethylating its recruiter GRHL2, thereby reducing clipping of histone H3 and oxidative stress and, ultimately, promoting tumor growth.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02200-7},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1330-1331},
  shortjournal = {Nat. Genet.},
  title        = {KDM4C shields breast cancer from the histone scissor cathepsin l},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome synteny reveals hidden enhancer conservation. <em>NG</em>, <em>57</em>(6), 1328-1329. (<a href='https://doi.org/10.1038/s41588-025-02194-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Enhancer sequences evolve rapidly, which has led to the prevailing view that most are not functionally conserved across species. A study now challenges this assumption by leveraging interspecies point projection — a method that uses genome synteny to uncover hidden enhancer conservation.},
  archive      = {J_NG},
  author       = {de Mendoza, Alex},
  doi          = {10.1038/s41588-025-02194-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1328-1329},
  shortjournal = {Nat. Genet.},
  title        = {Genome synteny reveals hidden enhancer conservation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Base editing as a therapeutic strategy for somatic repeat expansion diseases. <em>NG</em>, <em>57</em>(6), 1326-1327. (<a href='https://doi.org/10.1038/s41588-025-02201-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR–Cas base editing of trinucleotide repeats shows promise in reducing somatic repeat expansions in Huntington’s disease and Friedreich’s ataxia, offering a potential new therapeutic strategy.},
  archive      = {J_NG},
  author       = {Saha, Krishanu},
  doi          = {10.1038/s41588-025-02201-6},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1326-1327},
  shortjournal = {Nat. Genet.},
  title        = {Base editing as a therapeutic strategy for somatic repeat expansion diseases},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cell-state-specific enhancer design in hematopoiesis. <em>NG</em>, <em>57</em>(6), 1325. (<a href='https://doi.org/10.1038/s41588-025-02241-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gross, Petra},
  doi          = {10.1038/s41588-025-02241-y},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1325},
  shortjournal = {Nat. Genet.},
  title        = {Cell-state-specific enhancer design in hematopoiesis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mendel’s pea traits pinned down. <em>NG</em>, <em>57</em>(6), 1325. (<a href='https://doi.org/10.1038/s41588-025-02242-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-025-02242-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1325},
  shortjournal = {Nat. Genet.},
  title        = {Mendel’s pea traits pinned down},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). In vivo epigenome editing with NovaIscB. <em>NG</em>, <em>57</em>(6), 1325. (<a href='https://doi.org/10.1038/s41588-025-02243-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-025-02243-w},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1325},
  shortjournal = {Nat. Genet.},
  title        = {In vivo epigenome editing with NovaIscB},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The maize cis-regulatory landscape. <em>NG</em>, <em>57</em>(6), 1325. (<a href='https://doi.org/10.1038/s41588-025-02244-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-025-02244-9},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1325},
  shortjournal = {Nat. Genet.},
  title        = {The maize cis-regulatory landscape},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Towards trustworthiness of precision medicine research for people with disabilities. <em>NG</em>, <em>57</em>(6), 1321-1324. (<a href='https://doi.org/10.1038/s41588-025-02195-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {People with disabilities are under-represented in general (non-disability-specific) precision medicine research (PMR), limiting access to its benefits. We examine key reasons for this, focusing on the role of (dis)trust, and identify areas for further inquiry to guide researchers and enhance PMR’s trustworthiness for people with disabilities.},
  archive      = {J_NG},
  author       = {Aguirre, Alejandra and Lee, Sandra Soo-Jin and Callier, Shawneequa and Spicer, Paul and Sabatello, Maya},
  doi          = {10.1038/s41588-025-02195-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {1321-1324},
  shortjournal = {Nat. Genet.},
  title        = {Towards trustworthiness of precision medicine research for people with disabilities},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Identification of genetic elements that autonomously determine DNA methylation states. <em>NG</em>, <em>57</em>(5), 1320. (<a href='https://doi.org/10.1038/s41588-025-02185-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Lienert, Florian and Wirbelauer, Christiane and Som, Indrani and Dean, Ann and Mohn, Fabio and Schübeler, Dirk},
  doi          = {10.1038/s41588-025-02185-3},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1320},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Identification of genetic elements that autonomously determine DNA methylation states},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). CHOIR improves significance-based detection of cell types and states from single-cell data. <em>NG</em>, <em>57</em>(5), 1309-1319. (<a href='https://doi.org/10.1038/s41588-025-02148-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clustering is a critical step in the analysis of single-cell data, enabling the discovery and characterization of cell types and states. However, most popular clustering tools do not subject results to statistical inference testing, leading to risks of overclustering or underclustering data and often resulting in ineffective identification of cell types with widely differing prevalence. To address these challenges, we present CHOIR (cluster hierarchy optimization by iterative random forests), which applies a framework of random forest classifiers and permutation tests across a hierarchical clustering tree to statistically determine clusters representing distinct populations. We demonstrate the performance of CHOIR through extensive benchmarking against 15 existing clustering methods across 230 simulated and five real single-cell RNA sequencing, assay for transposase-accessible chromatin sequencing, spatial transcriptomic and multi-omic datasets. CHOIR can be applied to any single-cell data type and provides a flexible, scalable and robust solution to the challenge of identifying biologically relevant cell groupings within heterogeneous single-cell data. Cluster hierarchy optimization by iterative random forests (CHOIR) offers a robust and accurate method to identify cell clusters across a variety of single-cell resolution data with statistical support.},
  archive      = {J_NG},
  author       = {Sant, Cathrine and Mucke, Lennart and Corces, M. Ryan},
  doi          = {10.1038/s41588-025-02148-8},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1309-1319},
  shortjournal = {Nat. Genet.},
  title        = {CHOIR improves significance-based detection of cell types and states from single-cell data},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic regulation of gene expression across multiple tissues in chickens. <em>NG</em>, <em>57</em>(5), 1298-1308. (<a href='https://doi.org/10.1038/s41588-025-02155-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The chicken is a valuable model for understanding fundamental biology and vertebrate evolution and is a major global source of nutrient-dense and lean protein. Despite being the first non-mammalian amniote to have its genome sequenced, a systematic characterization of functional variation on the chicken genome remains lacking. Here, we integrated bulk RNA sequencing (RNA-seq) data from 7,015 samples, single-cell RNA-seq data from 127,598 cells and 2,869 whole-genome sequences to present a pilot atlas of regulatory variants across 28 chicken tissues. This atlas reveals millions of regulatory effects on primary expression (protein-coding genes, long non-coding RNA and exons) and post-transcriptional modifications (alternative splicing and 3′-untranslated region alternative polyadenylation). We highlighted distinct molecular mechanisms underlying these regulatory variants, their context-dependent behavior and their utility in interpreting genome-wide associations for 39 chicken complex traits. Finally, our comparative analyses of gene regulation between chickens and mammals demonstrate how this resource can facilitate cross-species gene mapping of complex traits. This study introduces a ChickenGTEx atlas of regulatory variants across 28 tissues and illustrates its utility in deciphering association signals for 39 complex traits in chickens.},
  archive      = {J_NG},
  author       = {Guan, Dailu and Bai, Zhonghao and Zhu, Xiaoning and Zhong, Conghao and Hou, Yali and Zhu, Di and Li, Houcheng and Lan, Fangren and Diao, Shuqi and Yao, Yuelin and Zhao, Bingru and Li, Xiaochang and Pan, Zhangyuan and Gao, Yahui and Wang, Yuzhe and Zou, Dong and Wang, Ruizhen and Xu, Tianyi and Sun, Congjiao and Yin, Hongwei and Teng, Jinyan and Xu, Zhiting and Lin, Qing and Shi, Shourong and Shao, Dan and Degalez, Fabien and Lagarrigue, Sandrine and Wang, Ying and Wang, Mingshan and Peng, Minsheng and Rocha, Dominique and Charles, Mathieu and Smith, Jacqueline and Watson, Kellie and Buitenhuis, Albert Johannes and Sahana, Goutam and Lund, Mogens Sandø and Warren, Wesley and Frantz, Laurent and Larson, Greger and Lamont, Susan J. and Si, Wei and Zhao, Xin and Li, Bingjie and Zhang, Haihan and Luo, Chenglong and Shu, Dingming and Qu, Hao and Luo, Wei and Li, Zhenhui and Nie, Qinghua and Zhang, Xiquan and Xiang, Ruidong and Liu, Shuli and Zhang, Zhe and Zhang, Zhang and Liu, George E. and Cheng, Hans and Yang, Ning and Hu, Xiaoxiang and Zhou, Huaijun and Fang, Lingzhao},
  doi          = {10.1038/s41588-025-02155-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1298-1308},
  shortjournal = {Nat. Genet.},
  title        = {Genetic regulation of gene expression across multiple tissues in chickens},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Oryza genome evolution through a tetraploid lens. <em>NG</em>, <em>57</em>(5), 1287-1297. (<a href='https://doi.org/10.1038/s41588-025-02183-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Oryza is a remarkable genus comprising 27 species and 11 genome types, with ~3.4-fold genome size variation, that possesses a virtually untapped reservoir of genes that can be used for crop improvement and neodomestication. Here we present 11 chromosome-level assemblies (nine tetraploid, two diploid) in the context of ~15 million years of evolution and show that the core Oryza (sub)genome is only ~200 Mb and largely syntenic, whereas the remaining nuclear fractions (~80–600 Mb) are intermingled, plastic and rapidly evolving. For the halophyte Oryza coarctata, we found that despite detection of gene fractionation in the subgenomes, homoeologous genes were expressed at higher levels in one subgenome over the other in a mosaic form, demonstrating subgenome equivalence. The integration of these 11 new reference genomes with previously published genome datasets provides a nearly complete view of the consequences of evolution for genome diversification across the genus. Chromosome-level genome assemblies of nine tetraploid and two diploid wild Oryza species provide insights into genome evolution within the genus Oryza and the potential for crop improvement and neodomestication.},
  archive      = {J_NG},
  author       = {Fornasiero, Alice and Feng, Tao and Al-Bader, Noor and Alsantely, Aseel and Mussurova, Saule and Hoang, Nam V. and Misra, Gopal and Zhou, Yong and Fabbian, Leonardo and Mohammed, Nahed and Rivera Serna, Luis and Thimma, Manjula and Llaca, Victor and Parakkal, Praveena and Kudrna, David and Copetti, Dario and Rajasekar, Shanmugam and Lee, Seunghee and Talag, Jayson and Sobel-Sorenson, Chandler and Carpentier, Marie-Christine and Panaud, Olivier and McNally, Kenneth L. and Zhang, Jianwei and Zuccolo, Andrea and Schranz, M. Eric and Wing, Rod A.},
  doi          = {10.1038/s41588-025-02183-5},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1287-1297},
  shortjournal = {Nat. Genet.},
  title        = {Oryza genome evolution through a tetraploid lens},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pan-genome analysis reveals the evolution and diversity of malus. <em>NG</em>, <em>57</em>(5), 1274-1286. (<a href='https://doi.org/10.1038/s41588-025-02166-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Malus Mill., a genus of temperate perennial trees with great agricultural and ecological value, has diversified through hybridization, polyploidy and environmental adaptation. Limited genomic resources for wild Malus species have hindered the understanding of their evolutionary history and genetic diversity. We sequenced and assembled 30 high-quality Malus genomes, representing 20 diploids and 10 polyploids across major evolutionary lineages and geographical regions. Phylogenomic analyses revealed ancient gene duplications and conversions, while six newly defined genome types, including an ancestral type shared by polyploid species, facilitated the detection of strong signals for extensive introgressions. The graph-based pan-genome captured shared and species-specific structural variations, facilitating the development of a molecular marker for apple scab resistance. Our pipeline for analyzing selective sweep identified a mutation in MdMYB5 having reduced cold and disease resistance during domestication. This study advances Malus genomics, uncovering genetic diversity and evolutionary insights while enhancing breeding for desirable traits. A graph-based pan-genome constructed from 30 genome assemblies covering Malus domestica ‘Golden Delicious’ and 29 wild diploid or polyploid Malus species highlights structural variation and genome evolution in the Malus genus.},
  archive      = {J_NG},
  author       = {Li, Wei and Chu, Chong and Zhang, Taikui and Sun, Haochen and Wang, Shiyao and Liu, Zeyuan and Wang, Zijun and Li, Hui and Li, Yuqi and Zhang, Xingtan and Geng, Zhiqiang and Wang, Youqing and Li, Yi and Zhang, Hengtao and Fan, Weishu and Wang, Yi and Xu, Xuefeng and Cheng, Lailiang and Zhang, Dehui and Xiong, Yao and Li, Huixia and Zhou, Bowen and Guan, Qingmei and Deng, Cecilia H. and Han, Yongming and Ma, Hong and Han, Zhenhai},
  doi          = {10.1038/s41588-025-02166-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1274-1286},
  shortjournal = {Nat. Genet.},
  title        = {Pan-genome analysis reveals the evolution and diversity of malus},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pan-genomic analysis highlights genes associated with agronomic traits and enhances genomics-assisted breeding in alfalfa. <em>NG</em>, <em>57</em>(5), 1262-1273. (<a href='https://doi.org/10.1038/s41588-025-02164-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Alfalfa (Medicago sativa L.), a globally important forage crop, is valued for its high nutritional quality and nitrogen-fixing capacity. Here, we present a high-quality pan-genome constructed from 24 diverse alfalfa accessions, encompassing a wide range of genetic backgrounds. This comprehensive analysis identified 433,765 structural variations and characterized 54,002 pan-gene families, highlighting the pivotal role of genomic diversity in alfalfa domestication and adaptation. Key structural variations associated with salt tolerance and quality traits were discovered, with functional analysis implicating genes such as MsMAP65 and MsGA3ox1. Notably, overexpression of MsGA3ox1 led to a reduced stem–leaf ratio and enhanced forage quality. The integration of genomic selection and marker-assisted breeding strategies improved genomic estimated breeding values across multiple traits, offering valuable genomic resources for advancing alfalfa breeding. These findings provide insights into the genetic basis of important agronomic traits and establish a solid foundation for future crop improvement. Pan-genome analysis of 24 alfalfa accessions identifies key structural variations associated with salt tolerance and quality traits and provides resources for genomics-assisted breeding in alfalfa.},
  archive      = {J_NG},
  author       = {He, Fei and Chen, Shuai and Zhang, Yangyang and Chai, Kun and Zhang, Qing and Kong, Weilong and Qu, Shenyang and Chen, Lin and Zhang, Fan and Li, Mingna and Wang, Xue and Lv, Huigang and Zhang, Tiejun and He, Xiaofan and Li, Xiao and Li, Yajing and Li, Xianyang and Jiang, Xueqian and Xu, Ming and Sod, Bilig and Kang, Junmei and Zhang, Xingtan and Long, Ruicai and Yang, Qingchuan},
  doi          = {10.1038/s41588-025-02164-8},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1262-1273},
  shortjournal = {Nat. Genet.},
  title        = {Pan-genomic analysis highlights genes associated with agronomic traits and enhances genomics-assisted breeding in alfalfa},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pangenome analysis reveals structural variation associated with seed size and weight traits in peanut. <em>NG</em>, <em>57</em>(5), 1250-1261. (<a href='https://doi.org/10.1038/s41588-025-02170-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Peanut (Arachis hypogaea L.) is an important oilseed and food legume crop, with seed size and weight being critical traits for domestication and breeding. However, genomic rearrangements like structural variations (SVs) underlying seed size and weight remain unclear. Here we present a comprehensive pangenome analysis utilizing eight high-quality genomes (two diploid wild, two tetraploid wild and four tetraploid cultivated peanuts) and resequencing data of 269 accessions with diverse seed sizes. We identified 22,222 core or soft-core, 22,232 distributed and 5,643 private gene families. The frequency of SVs in subgenome A is higher than in subgenome B. We identified 1,335 domestication-related SVs and 190 SVs associated with seed size or weight. Notably, a 275-bp deletion in gene AhARF2-2 results in loss of interaction with AhIAA13 and TOPLESS, reducing the inhibitory effect on AhGRF5 and promoting seed expansion. This high-quality pangenome serves as a fundamental resource for the genetic enhancement of peanuts and other legume crops. A graph-based peanut pangenome constructed using 6 newly assembled and 2 previously published genomes and 269 resequenced accessions highlights the contribution of structural variants to pod size.},
  archive      = {J_NG},
  author       = {Zhao, Kunkun and Xue, Hongzhang and Li, Guowei and Chitikineni, Annapurna and Fan, Yi and Cao, Zenghui and Dong, Xiaorui and Lu, Huimin and Zhao, Kai and Zhang, Lin and Qiu, Ding and Ren, Rui and Gong, Fangping and Li, Zhongfeng and Ma, Xingli and Wan, Shubo and Varshney, Rajeev K. and Wei, Chaochun and Yin, Dongmei},
  doi          = {10.1038/s41588-025-02170-w},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1250-1261},
  shortjournal = {Nat. Genet.},
  title        = {Pangenome analysis reveals structural variation associated with seed size and weight traits in peanut},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Wheat tandem kinase RWT4 directly binds a fungal effector to activate defense. <em>NG</em>, <em>57</em>(5), 1238-1249. (<a href='https://doi.org/10.1038/s41588-025-02162-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plants have intricate innate immune receptors that detect pathogens. Research has intensely focused on two receptor classes recognizing external and internal threats. Recent research has identified a class of disease-resistance proteins called tandem kinase proteins (TKPs). We investigated RWT4, a wheat TKP that confers resistance to the devastating fungal pathogen Magnaporthe oryzae. We established a rice protoplast system, revealing RWT4 specifically recognizes the AvrPWT4 effector, leading to the transcription of defense genes and inducing cell death. RWT4 possesses both kinase and pseudokinase domains, with its kinase activity essential for defense. RWT4 directly interacts with and transphosphorylates AvrPWT4. Biolayer interferometry revealed both RWT4 kinase and pseudokinase regions bind the effector. Sequence similarity and structural modeling revealed a partial kinase duplication in RWT4’s kinase region as critical for effector interaction and defense activation. Collectively, these findings demonstrate that TKPs can directly bind a recognized effector, leading to downstream defense activation. This study shows that RWT4, a wheat tandem kinase, confers resistance to the fungal pathogen Magnaporthe oryzae by directly binding to the pathogen effector AvrPWT4 and activating host defense.},
  archive      = {J_NG},
  author       = {Sung, Yi-Chang and Li, Yinghui and Bernasconi, Zoe and Baik, Suji and Asuke, Soichiro and Keller, Beat and Fahima, Tzion and Coaker, Gitta},
  doi          = {10.1038/s41588-025-02162-w},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1238-1249},
  shortjournal = {Nat. Genet.},
  title        = {Wheat tandem kinase RWT4 directly binds a fungal effector to activate defense},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Transcriptome-wide analysis of differential expression in perturbation atlases. <em>NG</em>, <em>57</em>(5), 1228-1237. (<a href='https://doi.org/10.1038/s41588-025-02169-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell CRISPR screens such as Perturb-seq enable transcriptomic profiling of genetic perturbations at scale. However, the data produced by these screens are noisy, and many effects may go undetected. Here we introduce transcriptome-wide analysis of differential expression (TRADE)—a statistical model for the distribution of true differential expression effects that accounts for estimation error appropriately. TRADE estimates the ‘transcriptome-wide impact’, which quantifies the total effect of a perturbation across the transcriptome. Analyzing several large Perturb-seq datasets, we show that many transcriptional effects remain undetected in standard analyses but emerge in aggregate using TRADE. A typical gene perturbation affects an estimated 45 genes, whereas a typical essential gene affects over 500. We find moderate consistency of perturbation effects across cell types, identify perturbations where transcriptional responses vary qualitatively across dosage levels and clarify the relationship between genetic and transcriptomic correlations across neuropsychiatric disorders. Transcriptome-wide analysis of differential expression (TRADE) is a broadly applicable tool for characterizing patterns of differential expression across the genome.},
  archive      = {J_NG},
  author       = {Nadig, Ajay and Replogle, Joseph M. and Pogson, Angela N. and Murthy, Mukundh and McCarroll, Steven A. and Weissman, Jonathan S. and Robinson, Elise B. and O’Connor, Luke J.},
  doi          = {10.1038/s41588-025-02169-3},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1228-1237},
  shortjournal = {Nat. Genet.},
  title        = {Transcriptome-wide analysis of differential expression in perturbation atlases},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multi-omic and spatial analysis of mouse kidneys highlights sex-specific differences in gene regulation across the lifespan. <em>NG</em>, <em>57</em>(5), 1213-1227. (<a href='https://doi.org/10.1038/s41588-025-02161-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There is a sex bias in the incidence and progression of many kidney diseases. To better understand such sexual dimorphism, we integrated data from six platforms, characterizing 76 kidney samples from 68 mice at six developmental and adult time points, creating a molecular atlas of the mouse kidney across the lifespan for both sexes. We show that proximal tubules have the most sex-biased differentially expressed genes emerging after 3 weeks of age and are associated with hormonal regulations. We reveal potential mechanisms involving both direct and indirect regulation by androgens and estrogens. Spatial profiling identifies distinct sex-biased spatial patterns in the cortex and outer stripe of the outer medulla. Additionally, older mice exhibit more aging-related gene alterations in loops of Henle, proximal tubules and collecting ducts in a sex-dependent manner. Our results enhance the understanding of spatially resolved gene expression and hormone regulation underlying kidney sexual dimorphism across the lifespan. Multi-omic analysis of the developing mouse kidney shows strong sex-dependent differences in gene expression during development and aging. These differences are likely mediated by sex hormones.},
  archive      = {J_NG},
  author       = {Chen, Siqi and Liu, Ruiyang and Mo, Chia-Kuei and Wendl, Michael C. and Houston, Andrew and Lal, Preet and Zhao, Yanyan and Caravan, Wagma and Shinkle, Andrew T. and Abedin-Do, Atieh and Naser Al Deen, Nataly and Sato, Kazuhito and Li, Xiang and Targino da Costa, André Luiz N. and Li, Yize and Karpova, Alla and Herndon, John M. and Artyomov, Maxim N. and Rubin, Joshua B. and Jain, Sanjay and Li, Xue and Stewart, Sheila A. and Ding, Li and Chen, Feng},
  doi          = {10.1038/s41588-025-02161-x},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1213-1227},
  shortjournal = {Nat. Genet.},
  title        = {Multi-omic and spatial analysis of mouse kidneys highlights sex-specific differences in gene regulation across the lifespan},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An integrated transcriptomic cell atlas of human endoderm-derived organoids. <em>NG</em>, <em>57</em>(5), 1201-1212. (<a href='https://doi.org/10.1038/s41588-025-02182-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human pluripotent stem cells and tissue-resident fetal and adult stem cells can generate epithelial tissues of endodermal origin in vitro that recapitulate aspects of developing and adult human physiology. Here, we integrate single-cell transcriptomes from 218 samples covering organoids and other models of diverse endoderm-derived tissues to establish an initial version of a human endoderm-derived organoid cell atlas. The integration includes nearly one million cells across diverse conditions, data sources and protocols. We compare cell types and states between organoid models and harmonize cell annotations through mapping to primary tissue counterparts. Focusing on the intestine and lung, we provide examples of mapping data from new protocols and show how the atlas can be used as a diverse cohort to assess perturbations and disease models. The human endoderm-derived organoid cell atlas makes diverse datasets centrally available and will be valuable to assess fidelity, characterize perturbed and diseased states, and streamline protocol development. The human endoderm-derived organoid cell atlas (HEOCA) presents an integrative analysis of single-cell transcriptomes across different conditions, sources and protocols. It compares cell types and states between models, and harmonizes cell annotations through mapping to primary tissues.},
  archive      = {J_NG},
  author       = {Xu, Quan and Halle, Lennard and Hediyeh-zadeh, Soroor and Kuijs, Merel and Riedweg, Rya and Kilik, Umut and Recaldin, Timothy and Yu, Qianhui and Rall, Isabell and Frum, Tristan and Adam, Lukas and Parikh, Shrey and Kfuri-Rubens, Raphael and Gander, Manuel and Klein, Dominik and Curion, Fabiola and He, Zhisong and Fleck, Jonas Simon and Oost, Koen and Kahnwald, Maurice and Barbiero, Silvia and Mitrofanova, Olga and Maciag, Grzegorz Jerzy and Jensen, Kim B. and Lutolf, Matthias and Liberali, Prisca and Spence, Jason R. and Gjorevski, Nikolche and Beumer, Joep and Treutlein, Barbara and Theis, Fabian J. and Camp, J. Gray},
  doi          = {10.1038/s41588-025-02182-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1201-1212},
  shortjournal = {Nat. Genet.},
  title        = {An integrated transcriptomic cell atlas of human endoderm-derived organoids},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Three-dimensional genome landscape of primary human cancers. <em>NG</em>, <em>57</em>(5), 1189-1200. (<a href='https://doi.org/10.1038/s41588-025-02188-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome conformation underlies transcriptional regulation by distal enhancers, and genomic rearrangements in cancer can alter critical regulatory interactions. Here we profiled the three-dimensional genome architecture and enhancer connectome of 69 tumor samples spanning 15 primary human cancer types from The Cancer Genome Atlas. We discovered the following three archetypes of enhancer usage for over 100 oncogenes across human cancers: static, selective gain or dynamic rewiring. Integrative analyses revealed the enhancer landscape of noncancer cells in the tumor microenvironment for genes related to immune escape. Deep whole-genome sequencing and enhancer connectome mapping provided accurate detection and validation of diverse structural variants across cancer genomes and revealed distinct enhancer rewiring consequences from noncoding point mutations, genomic inversions, translocations and focal amplifications. Extrachromosomal DNA promoted more extensive enhancer rewiring among several types of focal amplification mechanisms. These results suggest a systematic approach to understanding genome topology in cancer etiology and therapy. This study characterizes the three-dimensional (3D) genome architecture of 15 primary human cancer types from The Cancer Genome Atlas. The analyses identify different archetypes of enhancer usage and enhancer rewiring events due to different classes of mutations and structural variants.},
  archive      = {J_NG},
  author       = {Yost, Kathryn E. and Zhao, Yanding and Hung, King L. and Zhu, Kaiyuan and Xu, Duo and Corces, M. Ryan and Shams, Shadi and Louie, Bryan H. and Sarmashghi, Shahab and Sundaram, Laksshman and Luebeck, Jens and Clarke, Stanley and Doane, Ashley S. and Granja, Jeffrey M. and Choudhry, Hani and Imieliński, Marcin and Cherniack, Andrew D. and Khurana, Ekta and Bafna, Vineet and Felau, Ina and Zenklusen, Jean C. and Laird, Peter W. and Curtis, Christina and Greenleaf, William J. and Chang, Howard Y.},
  doi          = {10.1038/s41588-025-02188-0},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1189-1200},
  shortjournal = {Nat. Genet.},
  title        = {Three-dimensional genome landscape of primary human cancers},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Desmosome mutations impact the tumor microenvironment to promote melanoma proliferation. <em>NG</em>, <em>57</em>(5), 1179-1188. (<a href='https://doi.org/10.1038/s41588-025-02163-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Desmosomes are transmembrane protein complexes that contribute to cell–cell adhesion in epithelia and other tissues. Here, we report the discovery of frequent genetic alterations in the desmosome in human cancers, with the strongest signal seen in cutaneous melanoma, where desmosomes are mutated in more than 70% of cases. In primary but not metastatic melanoma biopsies, the burden of coding mutations in desmosome genes is associated with a strong reduction in desmosome gene expression. Analysis by spatial transcriptomics and protein immunofluorescence suggests that these decreases in expression occur in keratinocytes in the microenvironment rather than in primary melanoma cells. In further support of a microenvironmental origin, we find that desmosome gene knockdown in keratinocytes yields markedly increased proliferation of adjacent melanoma cells in keratinocyte and melanoma cocultures. Similar increases in melanoma proliferation are observed in media preconditioned with desmosome-deficient keratinocytes. Thus, gradual accumulation of desmosome mutations in neighboring cells may prime melanoma cells for neoplastic transformation. In cutaneous melanoma, desmosome genes are frequently affected by coding mutations that decrease their expression. This decrease in expression occurs predominantly in keratinocytes in the tumor microenvironment and can induce hyperproliferation of adjacent melanoma cells in coculture.},
  archive      = {J_NG},
  author       = {Baron, Maayan and Tagore, Mohita and Wall, Patrick and Zheng, Fan and Barkley, Dalia and Yanai, Itai and Yang, Jing and Kiuru, Maija and White, Richard M. and Ideker, Trey},
  doi          = {10.1038/s41588-025-02163-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1179-1188},
  shortjournal = {Nat. Genet.},
  title        = {Desmosome mutations impact the tumor microenvironment to promote melanoma proliferation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deciphering the longitudinal trajectories of glioblastoma ecosystems by integrative single-cell genomics. <em>NG</em>, <em>57</em>(5), 1168-1178. (<a href='https://doi.org/10.1038/s41588-025-02168-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The evolution of isocitrate dehydrogenase (IDH)-wildtype glioblastoma (GBM) after standard-of-care therapy remains poorly understood. Here we analyzed matched primary and recurrent GBMs from 59 patients using single-nucleus RNA sequencing and bulk DNA sequencing, assessing the longitudinal evolution of the GBM ecosystem across layers of cellular and molecular heterogeneity. The most consistent change was a lower malignant cell fraction at recurrence and a reciprocal increase in glial and neuronal cell types in the tumor microenvironment (TME). The predominant malignant cell state differed between most matched pairs, but no states were exclusive or highly enriched in either time point, nor was there a consistent longitudinal trajectory across the cohort. Nevertheless, specific trajectories were enriched in subsets of patients. Changes in malignant state abundances mirrored changes in TME composition and baseline profiles, reflecting the co-evolution of the GBM ecosystem. Our study provides a blueprint of GBM’s diverse longitudinal trajectories and highlights the treatment and TME modifiers that shape them. Comparison of paired primary and recurrent glioblastomas at the single-cell transcriptomic level describes molecular and cellular trajectories associated with tumor recurrence, highlighting extensive heterogeneity and microenvironmental co-evolution.},
  archive      = {J_NG},
  author       = {Spitzer, Avishay and Johnson, Kevin C. and Nomura, Masashi and Garofano, Luciano and Nehar-belaid, Djamel and Darnell, Noam Galili and Greenwald, Alissa C. and Bussema, Lillian and Oh, Young Taek and Varn, Frederick S. and D’Angelo, Fulvio and Gritsch, Simon and Anderson, Kevin J. and Migliozzi, Simona and Gonzalez Castro, L. Nicolas and Chowdhury, Tamrin and Robine, Nicolas and Reeves, Catherine and Park, Jong Bae and Lipsa, Anuja and Hertel, Frank and Golebiewska, Anna and Niclou, Simone P. and Nusrat, Labeeba and Kellet, Sorcha and Das, Sunit and Moon, Hyo-Eun and Paek, Sun Ha and Bielle, Franck and Laurenge, Alice and Di Stefano, Anna Luisa and Mathon, Bertrand and Picca, Alberto and Sanson, Marc and Tanaka, Shota and Saito, Nobuhito and Ashley, David M. and Keir, Stephen T. and Ligon, Keith L. and Huse, Jason T. and Yung, W. K. Alfred and Lasorella, Anna and Iavarone, Antonio and Verhaak, Roel G. W. and Tirosh, Itay and Suvà, Mario L.},
  doi          = {10.1038/s41588-025-02168-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1168-1178},
  shortjournal = {Nat. Genet.},
  title        = {Deciphering the longitudinal trajectories of glioblastoma ecosystems by integrative single-cell genomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The multilayered transcriptional architecture of glioblastoma ecosystems. <em>NG</em>, <em>57</em>(5), 1155-1167. (<a href='https://doi.org/10.1038/s41588-025-02167-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In isocitrate dehydrogenase wildtype glioblastoma (GBM), cellular heterogeneity across and within tumors may drive therapeutic resistance. Here we analyzed 121 primary and recurrent GBM samples from 59 patients using single-nucleus RNA sequencing and bulk tumor DNA sequencing to characterize GBM transcriptional heterogeneity. First, GBMs can be classified by their broad cellular composition, encompassing malignant and nonmalignant cell types. Second, in each cell type we describe the diversity of cellular states and their pathway activation, particularly an expanded set of malignant cell states, including glial progenitor cell-like, neuronal-like and cilia-like. Third, the remaining variation between GBMs highlights three baseline gene expression programs. These three layers of heterogeneity are interrelated and partially associated with specific genetic aberrations, thereby defining three stereotypic GBM ecosystems. This work provides an unparalleled view of the multilayered transcriptional architecture of GBM. How this architecture evolves during disease progression is addressed in the companion manuscript by Spitzer et al. Integrated single-cell transcriptomic and genetic characterization of 121 adult glioblastomas identifies heterogeneity at cell type, cell state and baseline expression program levels associated with specific mutations that form three stereotypical ecosystems.},
  archive      = {J_NG},
  author       = {Nomura, Masashi and Spitzer, Avishay and Johnson, Kevin C. and Garofano, Luciano and Nehar-belaid, Djamel and Galili Darnell, Noam and Greenwald, Alissa C. and Bussema, Lillian and Oh, Young Taek and Varn, Frederick S. and D’Angelo, Fulvio and Gritsch, Simon and Anderson, Kevin J. and Migliozzi, Simona and Gonzalez Castro, L. Nicolas and ChowdhFury, Tamrin and Robine, Nicolas and Reeves, Catherine and Park, Jong Bae and Lipsa, Anuja and Hertel, Frank and Golebiewska, Anna and Niclou, Simone P. and Nusrat, Labeeba and Kellet, Sorcha and Das, Sunit and Moon, Hyo Eun and Paek, Sun Ha and Bielle, Franck and Laurenge, Alice and Di Stefano, Anna Luisa and Mathon, Bertrand and Picca, Alberto and Sanson, Marc and Tanaka, Shota and Saito, Nobuhito and Ashley, David M. and Keir, Stephen T. and Ligon, Keith L. and Huse, Jason T. and Yung, W. K. Alfred and Lasorella, Anna and Verhaak, Roel G. W. and Iavarone, Antonio and Suvà, Mario L. and Tirosh, Itay},
  doi          = {10.1038/s41588-025-02167-5},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1155-1167},
  shortjournal = {Nat. Genet.},
  title        = {The multilayered transcriptional architecture of glioblastoma ecosystems},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Longitudinal single-cell multiomic atlas of high-risk neuroblastoma reveals chemotherapy-induced tumor microenvironment rewiring. <em>NG</em>, <em>57</em>(5), 1142-1154. (<a href='https://doi.org/10.1038/s41588-025-02158-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-risk neuroblastoma, a leading cause of pediatric cancer mortality, exhibits substantial intratumoral heterogeneity, contributing to therapeutic resistance. To understand tumor microenvironment evolution during therapy, we longitudinally profiled 22 patients with high-risk neuroblastoma before and after induction chemotherapy using single-nucleus RNA and ATAC sequencing and whole-genome sequencing. This revealed profound shifts in tumor and immune cell subpopulations after therapy and identified enhancer-driven transcriptional regulators of neuroblastoma neoplastic states. Poor outcome correlated with proliferative and metabolically active neoplastic states, whereas more differentiated neuronal-like states predicted better prognosis. Proportions of mesenchymal neoplastic cells increased after therapy and a high proportion correlated with a poorer chemotherapy response. Macrophages significantly expanded towards pro-angiogenic, immunosuppressive and metabolic phenotypes. We identified paracrine signaling networks and validated the HB-EGF–ERBB4 axis between macrophage and neoplastic subsets, which promoted tumor growth through the induction of ERK signaling. These findings collectively reveal intrinsic and extrinsic regulators of therapy response in high-risk neuroblastoma. Analysis of pre- and post-chemotherapy samples collected from 22 patients with high-risk neuroblastoma using single-nucleus RNA and ATAC sequencing and spatial omics characterizes therapy-related changes in cell composition and the tumor microenvironment.},
  archive      = {J_NG},
  author       = {Yu, Wenbao and Biyik-Sit, Rumeysa and Uzun, Yasin and Chen, Chia-Hui and Thadi, Anusha and Sussman, Jonathan H. and Pang, Minxing and Wu, Chi-Yun and Grossmann, Liron D. and Gao, Peng and Wu, David W. and Yousey, Aliza and Zhang, Mei and Turn, Christina S. and Zhang, Zhan and Bandyopadhyay, Shovik and Huang, Jeffrey and Patel, Tasleema and Chen, Changya and Martinez, Daniel and Surrey, Lea F. and Hogarty, Michael D. and Bernt, Kathrin and Zhang, Nancy R. and Maris, John M. and Tan, Kai},
  doi          = {10.1038/s41588-025-02158-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1142-1154},
  shortjournal = {Nat. Genet.},
  title        = {Longitudinal single-cell multiomic atlas of high-risk neuroblastoma reveals chemotherapy-induced tumor microenvironment rewiring},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A redefined InDel taxonomy provides insights into mutational signatures. <em>NG</em>, <em>57</em>(5), 1132-1141. (<a href='https://doi.org/10.1038/s41588-025-02152-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite their deleterious effects, small insertions and deletions (InDels) have received far less attention than substitutions. Here we generated isogenic CRISPR-edited human cellular models of postreplicative repair dysfunction (PRRd), including individual and combined gene edits of DNA mismatch repair (MMR) and replicative polymerases (Pol ε and Pol δ). Unique, diverse InDel mutational footprints were revealed. However, the prevailing InDel classification framework was unable to discriminate these InDel signatures from background mutagenesis and from each other. To address this, we developed an alternative InDel classification system that considers flanking sequences and informative motifs (for example, longer homopolymers), enabling unambiguous InDel classification into 89 subtypes. Through focused characterization of seven tumor types from the 100,000 Genomes Project, we uncovered 37 InDel signatures; 27 were new. In addition to unveiling previously hidden biological insights, we also developed PRRDetect—a highly specific classifier of PRRd status in tumors, with potential implications for immunotherapies. This paper describes a framework for classifying small insertions and deletions from genomic data and applies it to a large dataset comprising seven tumor types. The analysis highlights new insertion and deletion signatures and a classifier of postreplicative repair dysfunction.},
  archive      = {J_NG},
  author       = {Koh, Gene Ching Chiek and Nanda, Arjun Scott and Rinaldi, Giuseppe and Boushaki, Soraya and Degasperi, Andrea and Badja, Cherif and Pregnall, Andrew Marcel and Zhao, Salome Jingchen and Chmelova, Lucia and Black, Daniella and Heskin, Laura and Dias, João and Young, Jamie and Memari, Yasin and Shooter, Scott and Czarnecki, Jan and Brown, Matthew Arthur and Davies, Helen Ruth and Zou, Xueqing and Nik-Zainal, Serena},
  doi          = {10.1038/s41588-025-02152-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1132-1141},
  shortjournal = {Nat. Genet.},
  title        = {A redefined InDel taxonomy provides insights into mutational signatures},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Near-complete middle eastern genomes refine autozygosity and enhance disease-causing and population-specific variant discovery. <em>NG</em>, <em>57</em>(5), 1119-1131. (<a href='https://doi.org/10.1038/s41588-025-02173-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in long-read sequencing have enabled routine complete assembly of human genomes, but much remains to be done to represent broader populations and show impact on disease-gene discovery. Here, we report highly accurate, near-complete and phased genomes from six Middle Eastern (ME) family trios (n = 18) with neurodevelopmental conditions, representing ancestries from Sudan, Jordan, Syria, Qatar and Afghanistan. These genomes revealed 42.2 Mb of new sequence (13.8% impacting known genes), 75 new HLA/KIR alleles and strong signals of inbreeding, with ROH covering up to one-third of chromosomes 6 and 12 in one individual. Using assembly-based variant calling, we identified 23 de novo and recessive variants as strong candidates for causing previously unresolved symptoms in the probands. The ME genomes revealed unique variation relative to existing references, showing enhanced mappability and variant calling. These results underscore the value of de novo assembly for disease variant discovery and the need for sampled ME-specific references to better characterize population-relevant variation. Generation and analysis of high-quality, genome assemblies from Middle Eastern trios demonstrate the utility of ancestry-matched data and assembly-based variation analysis.},
  archive      = {J_NG},
  author       = {Ghorbani, Mohammadmersad and Moosa, Shabir and Siddig, Zenab and Farhad, Radi and Naeem, Haroon and Harvey, William T. and Mastrorosa, Francesco Kumara and Munson, Katherine M. and Mohamad Razali, Rozaimi and Aliyev, Elbay and Diboun, Ilhame and Abouelhassan, Rawan and Tauro, Melissa and Hassan, Sondoss and Mathew, Rebecca and Al Hashmi, Muna and Mathew, Lisa S. and Wang, Kun and Salhab, Abdul Rahman and Vempalli, Fazulur Rehaman and El Khouly, Ahmed and Alazwani, Iman and Tomei, Sara and Fakhro, Khalid A. and Satti, Alia and Benini, Ruba and Rhie, Arang and Eichler, Evan E. and Mokrab, Younes},
  doi          = {10.1038/s41588-025-02173-7},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1119-1131},
  shortjournal = {Nat. Genet.},
  title        = {Near-complete middle eastern genomes refine autozygosity and enhance disease-causing and population-specific variant discovery},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome-wide analyses identify 25 infertility loci and relationships with reproductive traits across the allele frequency spectrum. <em>NG</em>, <em>57</em>(5), 1107-1118. (<a href='https://doi.org/10.1038/s41588-025-02156-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWASs) may help inform the etiology of infertility. Here, we perform GWAS meta-analyses across seven cohorts in up to 42,629 cases and 740,619 controls and identify 25 genetic risk loci for male and female infertility. We additionally identify up to 269 genetic loci associated with follicle-stimulating hormone, luteinizing hormone, estradiol and testosterone through sex-specific GWAS meta-analyses (n = 6,095–246,862). Exome sequencing analyses reveal that women carrying testosterone-lowering rare variants in some genes are at risk of infertility. However, we find no local or genome-wide genetic correlation between female infertility and reproductive hormones. While infertility is genetically correlated with endometriosis and polycystic ovary syndrome, we find limited genetic overlap between infertility and obesity. Finally, we show that the evolutionary persistence of infertility-risk alleles may be explained by directional selection. Taken together, we provide a comprehensive view of the genetic determinants of infertility across multiple diagnostic criteria. Genome-wide analyses identify variants associated with infertility and reproductive hormone levels but find limited polygenic overlap between reproductive hormone levels and infertility at the population level.},
  archive      = {J_NG},
  author       = {Venkatesh, Samvida S. and Wittemans, Laura B. L. and Palmer, Duncan S. and Baya, Nikolas A. and Ferreira, Teresa and Hill, Barney and Lassen, Frederik Heymann and Parker, Melody J. and Reibe, Saskia and Elhakeem, Ahmed and Banasik, Karina and Bruun, Mie T. and Erikstrup, Christian and Aagard Jensen, Bitten and Juul, Anders and Mikkelsen, Christina and Nielsen, Henriette S. and Ostrowski, Sisse R. and Pedersen, Ole B. and Rohde, Palle Duun and Sørensen, Erik and Ullum, Henrik and Westergaard, David and Haraldsson, Asgeir and Holm, Hilma and Jonsdottir, Ingileif and Olafsson, Isleifur and Steingrimsdottir, Thora and Steinthorsdottir, Valgerdur and Thorleifsson, Gudmar and Figueredo, Jessica and Karjalainen, Minna K. and Pasanen, Anu and Jacobs, Benjamin M. and Kalantzis, Georgios and Hubers, Nikki and Lippincott, Margaret and Fraser, Abigail and Lawlor, Deborah A. and Timpson, Nicholas J. and Nyegaard, Mette and Stefansson, Kari and Magi, Reedik and Laivuori, Hannele and van Heel, David A. and Boomsma, Dorret I. and Balasubramanian, Ravikumar and Seminara, Stephanie B. and Chan, Yee-Ming and Laisk, Triin and Lindgren, Cecilia M.},
  doi          = {10.1038/s41588-025-02156-8},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1107-1118},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide analyses identify 25 infertility loci and relationships with reproductive traits across the allele frequency spectrum},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reply to: An alternative model for maternal mtDNA inheritance. <em>NG</em>, <em>57</em>(5), 1105-1106. (<a href='https://doi.org/10.1038/s41588-025-02150-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Lee, William and Zamudio-Ochoa, Angelica and Buchel, Gina and Podlesniy, Petar and Marti Gutierrez, Nuria and Mikhalchenko, Aleksei and Trullas, Ramon and Mitalipov, Shoukhrat and Temiakov, Dmitry},
  doi          = {10.1038/s41588-025-02150-0},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1105-1106},
  shortjournal = {Nat. Genet.},
  title        = {Reply to: An alternative model for maternal mtDNA inheritance},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An alternative model for maternal mtDNA inheritance. <em>NG</em>, <em>57</em>(5), 1103-1104. (<a href='https://doi.org/10.1038/s41588-025-02149-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Alexeyev, Mikhail and Bai, Yidong},
  doi          = {10.1038/s41588-025-02149-7},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1103-1104},
  shortjournal = {Nat. Genet.},
  title        = {An alternative model for maternal mtDNA inheritance},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Circular RNA discovery with emerging sequencing and deep learning technologies. <em>NG</em>, <em>57</em>(5), 1089-1102. (<a href='https://doi.org/10.1038/s41588-025-02157-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Circular RNA (circRNA) represents a type of RNA molecule characterized by a closed-loop structure that is distinct from linear RNA counterparts. Recent studies have revealed the emerging role of these circular transcripts in gene regulation and disease pathogenesis. However, their low expression levels and high sequence similarity to linear RNAs present substantial challenges for circRNA detection and characterization. Recent advances in long-read and single-cell RNA sequencing technologies, coupled with sophisticated deep learning-based algorithms, have revolutionized the investigation of circRNAs at unprecedented resolution and scale. This Review summarizes recent breakthroughs in circRNA discovery, characterization and functional analysis algorithms. We also discuss the challenges associated with integrating large-scale circRNA sequencing data and explore the potential future development of artificial intelligence (AI)-driven algorithms to unlock the full potential of circRNA research in biomedical applications. This Review discusses recent advances in circular RNA discovery afforded by breakthroughs in sequencing technologies and deep learning-based algorithms and highlights outstanding challenges in integrating large-scale circular RNA datasets.},
  archive      = {J_NG},
  author       = {Zhang, Jinyang and Zhao, Fangqing},
  doi          = {10.1038/s41588-025-02157-7},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1089-1102},
  shortjournal = {Nat. Genet.},
  title        = {Circular RNA discovery with emerging sequencing and deep learning technologies},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Psychiatric genetics in the diverse landscape of latin american populations. <em>NG</em>, <em>57</em>(5), 1074-1088. (<a href='https://doi.org/10.1038/s41588-025-02127-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Psychiatric disorders are highly heritable and polygenic, influenced by environmental factors and often comorbid. Large-scale genome-wide association studies (GWASs) through consortium efforts have identified genetic risk loci and revealed the underlying biology of psychiatric disorders and traits. However, over 85% of psychiatric GWAS participants are of European ancestry, limiting the applicability of these findings to non-European populations. Latin America and the Caribbean, regions marked by diverse genetic admixture, distinct environments and healthcare disparities, remain critically understudied in psychiatric genomics. This threatens access to precision psychiatry, where diversity is crucial for innovation and equity. This Review evaluates the current state and advancements in psychiatric genomics within Latin America and the Caribbean, discusses the prevalence and burden of psychiatric disorders, explores contributions to psychiatric GWASs from these regions and highlights methods that account for genetic diversity. We also identify existing gaps and challenges and propose recommendations to promote equity in psychiatric genomics. Latin America and the Caribbean remain largely underrepresented in psychiatric genetics research. This Review highlights the need for more research in these populations to advance genetic insights and ensure equitable precision medicine access.},
  archive      = {J_NG},
  author       = {Bruxel, Estela M. and Rovaris, Diego L. and Belangero, Sintia I. and Chavarría-Soley, Gabriela and Cuellar-Barboza, Alfredo B. and Martínez-Magaña, José J. and Nagamatsu, Sheila T. and Nievergelt, Caroline M. and Núñez-Ríos, Diana L. and Ota, Vanessa K. and Peterson, Roseann E. and Sloofman, Laura G. and Adams, Amy M. and Albino, Elinette and Alvarado, Angel T. and Andrade-Brito, Diego and Arguello-Pascualli, Paola Y. and Bandeira, Cibele E. and Bau, Claiton H. D. and Bulik, Cynthia M. and Buxbaum, Joseph D. and Cappi, Carolina and Corral-Frias, Nadia S. and Corrales, Alejo and Corsi-Zuelli, Fabiana and Crowley, James J. and Cupertino, Renata B. and da Silva, Bruna S. and De Almeida, Suzannah S. and De la Hoz, Juan F. and Forero, Diego A. and Fries, Gabriel R. and Gelernter, Joel and González-Giraldo, Yeimy and Grevet, Eugenio H. and Grice, Dorothy E. and Hernández-Garayua, Adriana and Hettema, John M. and Ibáñez, Agustín and Ionita-Laza, Iuliana and Lattig, Maria Claudia and Lima, Yago C. and Lin, Yi-Sian and López-León, Sandra and Loureiro, Camila M. and Martínez-Cerdeño, Verónica and Martínez-Levy, Gabriela A. and Melin, Kyle and Moreno-De-Luca, Daniel and Muniz Carvalho, Carolina and Olivares, Ana Maria and Oliveira, Victor F. and Ormond, Rafaella and Palmer, Abraham A. and Panzenhagen, Alana C. and Passos-Bueno, Maria Rita and Peng, Qian and Pérez-Palma, Eduardo and Prieto, Miguel L. and Roussos, Panos and Sanchez-Roige, Sandra and Santamaría-García, Hernando and Shansis, Flávio M. and Sharp, Rachel R. and Storch, Eric A. and Tavares, Maria Eduarda A. and Tietz, Grace E. and Torres-Hernández, Bianca A. and Tovo-Rodrigues, Luciana and Trelles, Pilar and Trujillo-ChiVacuan, Eva M. and Velásquez, Maria M. and Vera-Urbina, Fernando and Voloudakis, Georgios and Wegman-Ostrosky, Talia and Zhen-Duan, Jenny and Zhou, Hang and Santoro, Marcos L. and Nicolini, Humberto and Atkinson, Elizabeth G. and Giusti-Rodríguez, Paola and Montalvo-Ortiz, Janitza L.},
  doi          = {10.1038/s41588-025-02127-z},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1074-1088},
  shortjournal = {Nat. Genet.},
  title        = {Psychiatric genetics in the diverse landscape of latin american populations},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Insights into the evolution and genetic diversity of the malus genus from pan-genome analysis. <em>NG</em>, <em>57</em>(5), 1072-1073. (<a href='https://doi.org/10.1038/s41588-025-02171-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We provide an in-depth pan-genome analysis of 30 Malus species, revealing potential mechanisms of ancient gene duplications, introgressions and polyploidization that drive genetic diversity and adaptability. Furthermore, we shed light on apple domestication, identify markers for disease resistance and present new strategies for uncovering genes with breeding potential.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02171-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1072-1073},
  shortjournal = {Nat. Genet.},
  title        = {Insights into the evolution and genetic diversity of the malus genus from pan-genome analysis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A plant dual-kinase protein traps a pathogen effector to trigger immunity. <em>NG</em>, <em>57</em>(5), 1070-1071. (<a href='https://doi.org/10.1038/s41588-025-02176-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tandem kinase proteins (TKPs) are key regulators of plant immunity. We found that RWT4, a wheat TKP, confers resistance to the fungal pathogen Magnaporthe oryzae by directly recognizing the effector AvrPWT4. Recognition of AvrPWT4 depends on the RWT4 N-terminal partial kinase duplication region, which is essential for pathogen perception and defense activation.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02176-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1070-1071},
  shortjournal = {Nat. Genet.},
  title        = {A plant dual-kinase protein traps a pathogen effector to trigger immunity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Middle eastern genomes: New references for disease and population diversity studies. <em>NG</em>, <em>57</em>(5), 1068-1069. (<a href='https://doi.org/10.1038/s41588-025-02174-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-quality, near-complete and phased genomes were assembled from diverse Middle Eastern families, enabling refined autozygosity mapping and enhanced discovery of rare disease-causing variants. The genomes serve as valuable references for detecting population-specific genetic variation, paving the way for improved genetic diagnosis and a deeper understanding of human population diversity.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02174-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1068-1069},
  shortjournal = {Nat. Genet.},
  title        = {Middle eastern genomes: New references for disease and population diversity studies},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantifying reproducible multivariate differential expression in genetic perturbation screens. <em>NG</em>, <em>57</em>(5), 1066-1067. (<a href='https://doi.org/10.1038/s41588-025-02177-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multivariate estimators of how much a perturbation affects the overall gene expression phenotype in Perturb-seq data can be derived using univariate statistics from differential gene expression tools.},
  archive      = {J_NG},
  author       = {Peidli, Stefan},
  doi          = {10.1038/s41588-025-02177-3},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1066-1067},
  shortjournal = {Nat. Genet.},
  title        = {Quantifying reproducible multivariate differential expression in genetic perturbation screens},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Desmosome mutations in keratinocytes fuel melanoma development. <em>NG</em>, <em>57</em>(5), 1064-1065. (<a href='https://doi.org/10.1038/s41588-025-02133-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this study, somatic mutations in desmosome genes of keratinocytes were found to support melanoma growth. This work has fundamental implications for our understanding of the somatic landscape of cancer.},
  archive      = {J_NG},
  author       = {Possik, Patricia A. and Marie, Kerrie L. and Adams, David J.},
  doi          = {10.1038/s41588-025-02133-1},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1064-1065},
  shortjournal = {Nat. Genet.},
  title        = {Desmosome mutations in keratinocytes fuel melanoma development},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetic insights into infertility from large-scale analyses. <em>NG</em>, <em>57</em>(5), 1062-1063. (<a href='https://doi.org/10.1038/s41588-025-02186-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A large genome-wide meta-analysis across the allele frequency spectrum has identified genetic associations with infertility. This study reveals wide-ranging genetic insights underlying diagnostic heterogeneity in infertility.},
  archive      = {J_NG},
  author       = {Ruth, Katherine S.},
  doi          = {10.1038/s41588-025-02186-2},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1062-1063},
  shortjournal = {Nat. Genet.},
  title        = {Genetic insights into infertility from large-scale analyses},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nanoscale-folding of mitotic chromosomes. <em>NG</em>, <em>57</em>(5), 1061. (<a href='https://doi.org/10.1038/s41588-025-02213-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gross, Petra},
  doi          = {10.1038/s41588-025-02213-2},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1061},
  shortjournal = {Nat. Genet.},
  title        = {Nanoscale-folding of mitotic chromosomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Complete genomes of six ape species. <em>NG</em>, <em>57</em>(5), 1061. (<a href='https://doi.org/10.1038/s41588-025-02214-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-025-02214-1},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1061},
  shortjournal = {Nat. Genet.},
  title        = {Complete genomes of six ape species},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Preventing polyglutamine assembly. <em>NG</em>, <em>57</em>(5), 1061. (<a href='https://doi.org/10.1038/s41588-025-02215-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-025-02215-0},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1061},
  shortjournal = {Nat. Genet.},
  title        = {Preventing polyglutamine assembly},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A systematic assessment of transcription factor function. <em>NG</em>, <em>57</em>(5), 1061. (<a href='https://doi.org/10.1038/s41588-025-02216-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02216-z},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {1061},
  shortjournal = {Nat. Genet.},
  title        = {A systematic assessment of transcription factor function},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Rare variant analyses in 51,256 type 2 diabetes cases and 370,487 controls reveal the pathogenicity spectrum of monogenic diabetes genes. <em>NG</em>, <em>57</em>(4), 1060. (<a href='https://doi.org/10.1038/s41588-025-02175-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Huerta-Chagoya, Alicia and Schroeder, Philip and Mandla, Ravi and Li, Jiang and Morris, Lowri and Vora, Maheak and Alkanaq, Ahmed and Nagy, Dorka and Szczerbinski, Lukasz and Madsen, Jesper G. S. and Bonàs-Guarch, Silvia and Mollandin, Fanny and Cole, Joanne B. and Porneala, Bianca and Westerman, Kenneth and Li, Josephine H. and Pollin, Toni I. and Florez, Jose C. and Gloyn, Anna L. and Carey, David J. and Cebola, Inês and Mirshahi, Uyenlinh L. and Manning, Alisa K. and Leong, Aaron and Udler, Miriam and Mercader, Josep M.},
  doi          = {10.1038/s41588-025-02175-5},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {1060},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Rare variant analyses in 51,256 type 2 diabetes cases and 370,487 controls reveal the pathogenicity spectrum of monogenic diabetes genes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Origin and de novo domestication of sweet orange. <em>NG</em>, <em>57</em>(4), 1059. (<a href='https://doi.org/10.1038/s41588-025-02165-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Liu, Shengjun and Xu, Yuantao and Yang, Kun and Huang, Yue and Lu, Zhihao and Chen, Shulin and Gao, Xiang and Xiao, Gongao and Chen, Peng and Zeng, Xiuli and Wang, Lun and Zheng, Weikang and Liu, Zishuang and Liao, Guanglian and He, Fa and Liu, Junjie and Wan, Pengfei and Ding, Fang and Ye, Junli and Jiao, Wenbiao and Chai, Lijun and Pan, Zhiyong and Zhang, Fei and Lin, Zongcheng and Zan, Yanjun and Guo, Wenwu and Larkin, Robert M. and Xie, Zongzhou and Wang, Xia and Deng, Xiuxin and Xu, Qiang},
  doi          = {10.1038/s41588-025-02165-7},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {1059},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Origin and de novo domestication of sweet orange},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An enhanced framework for local genetic correlation analysis. <em>NG</em>, <em>57</em>(4), 1053-1058. (<a href='https://doi.org/10.1038/s41588-025-02123-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic correlation is a key parameter in the joint genetic model of complex traits, but it is usually estimated on a global genomic scale. Understanding local genetic correlations provides more detailed insight into the shared genetic architecture of complex traits. However, a state-of-the-art tool for local genetic correlation analysis, LAVA, is prone to false inference. Here we extend the high-definition likelihood (HDL) method to a local version, HDL-L, which performs genetic correlation analysis in small, approximately independent linkage disequilibrium blocks. HDL-L allows a more granular estimation of genetic variances and covariances. Simulations show that HDL-L offers more consistent heritability estimates and more efficient genetic correlation estimates compared with LAVA. HDL-L demonstrated robust performance across a wide range of simulations conducted under varying parameter settings. In the analysis of 30 phenotypes from the UK Biobank, HDL-L identified 109 significant local genetic correlations and showed a notable computational advantage. HDL-L proves to be a powerful tool for uncovering the detailed genetic landscape that underlies complex human traits, offering both accuracy and computational efficiency. HDL-L is an extension of the high-definition likelihood method that enables local heritability and genetic correlation analysis with higher accuracy and computational efficiency than LAVA.},
  archive      = {J_NG},
  author       = {Li, Yuying and Pawitan, Yudi and Shen, Xia},
  doi          = {10.1038/s41588-025-02123-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {1053-1058},
  shortjournal = {Nat. Genet.},
  title        = {An enhanced framework for local genetic correlation analysis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Family-based genome-wide association study designs for increased power and robustness. <em>NG</em>, <em>57</em>(4), 1044-1052. (<a href='https://doi.org/10.1038/s41588-025-02118-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Family-based genome-wide association studies (FGWASs) use random, within-family genetic variation to remove confounding from estimates of direct genetic effects (DGEs). Here we introduce a ‘unified estimator’ that includes individuals without genotyped relatives, unifying standard and FGWAS while increasing power for DGE estimation. We also introduce a ‘robust estimator’ that is not biased in structured and/or admixed populations. In an analysis of 19 phenotypes in the UK Biobank, the unified estimator in the White British subsample and the robust estimator (applied without ancestry restrictions) increased the effective sample size for DGEs by 46.9% to 106.5% and 10.3% to 21.0%, respectively, compared to using genetic differences between siblings. Polygenic predictors derived from the unified estimator demonstrated superior out-of-sample prediction ability compared to other family-based methods. We implemented the methods in the software package snipar in an efficient linear mixed model that accounts for sample relatedness and sibling shared environment. This study develops family-based genome-wide association study methods that maximize power in homogeneous samples through inclusion of singletons and in diverse samples by using all available parental genotypes.},
  archive      = {J_NG},
  author       = {Guan, Junming and Tan, Tammy and Nehzati, Seyed Moeen and Bennett, Michael and Turley, Patrick and Benjamin, Daniel J. and Young, Alexander Strudwick},
  doi          = {10.1038/s41588-025-02118-0},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {1044-1052},
  shortjournal = {Nat. Genet.},
  title        = {Family-based genome-wide association study designs for increased power and robustness},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A telomere-to-telomere genome assembly of cotton provides insights into centromere evolution and short-season adaptation. <em>NG</em>, <em>57</em>(4), 1031-1043. (<a href='https://doi.org/10.1038/s41588-025-02130-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cotton (Gossypium hirsutum L.) is a key allopolyploid crop with global economic importance. Here we present a telomere-to-telomere assembly of the elite variety Zhongmian 113. Leveraging technologies including PacBio HiFi, Oxford Nanopore Technology (ONT) ultralong-read sequencing and Hi-C, our assembly surpasses previous genomes in contiguity and completeness, resolving 26 centromeric and 52 telomeric regions, 5S rDNA clusters and nucleolar organizer regions. A phylogenetically recent centromere repositioning on chromosome D08 was discovered specific to G. hirsutum, involving deactivation of an ancestral centromere and the formation of a unique, satellite repeat-based centromere. Genomic analyses evaluated favorable allele aggregation for key agronomic traits and uncovered an early-maturing haplotype derived from an 11 Mb pericentric inversion that evolved early during G. hirsutum domestication. Our study sheds light on the genomic origins of short-season adaptation, potentially involving introgression of an inversion from primitively domesticated forms, followed by subsequent haplotype differentiation in modern breeding programs. A telomere-to-telomere genome assembly of the elite Gossypium hirsutum variety ZM113 provides insights into upland cotton short-season adaptation and identifies a repositioned, satellite repeat-rich centromere.},
  archive      = {J_NG},
  author       = {Hu, Guanjing and Wang, Zhenyu and Tian, Zunzhe and Wang, Kai and Ji, Gaoxiang and Wang, Xingxing and Zhang, Xianliang and Yang, Zhaoen and Liu, Xuan and Niu, Ruoyu and Zhu, De and Zhang, Yuzhi and Duan, Lian and Ma, Xueyuan and Xiong, Xianpeng and Kong, Jiali and Zhao, Xianjia and Zhang, Ya and Zhao, Junjie and He, Shoupu and Grover, Corrinne E. and Su, Junji and Feng, Keyun and Yu, Guangrun and Han, Jinlei and Zang, Xinshan and Wu, Zhiqiang and Pan, Weihua and Wendel, Jonathan F. and Ma, Xiongfeng},
  doi          = {10.1038/s41588-025-02130-4},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {1031-1043},
  shortjournal = {Nat. Genet.},
  title        = {A telomere-to-telomere genome assembly of cotton provides insights into centromere evolution and short-season adaptation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Post-polyploidization centromere evolution in cotton. <em>NG</em>, <em>57</em>(4), 1021-1030. (<a href='https://doi.org/10.1038/s41588-025-02115-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Upland cotton (Gossypium hirsutum) accounts for more than 90% of the world’s cotton production and, as an allotetraploid, is a model plant for polyploid crop domestication. In the present study, we reported a complete telomere-to-telomere (T2T) genome assembly of Upland cotton accession Texas Marker-1 (T2T-TM-1), which has a total size of 2,299.6 Mb, and annotated 79,642 genes. Based on T2T-TM-1, interspecific centromere divergence was detected between the A- and D-subgenomes and their corresponding diploid progenitors. Centromere-associated repetitive sequences (CRCs) were found to be enriched for Gypsy-like retroelements. Centromere size expansion, repositioning and structure variations occurred post-polyploidization. It is interesting that CRC homologs were transferred from the diploid D-genome progenitor to the D-subgenome, invaded the A-subgenome and then underwent post-tetraploidization proliferation. This suggests an evolutionary advantage for the CRCs of the D-genome progenitor, presents a D-genome-adopted inheritance of centromere repeats after polyploidization and shapes the dynamic centromeric landscape during polyploidization in polyploid species. A telomere-to-telomere genome assembly of Upland cotton (Gossypium hirsutum) accession TM-1 including a set of complete centromeres provides insights into centromere evolution following polyploidization.},
  archive      = {J_NG},
  author       = {Yan, Hu and Han, Jinlei and Jin, Shangkun and Han, Zegang and Si, Zhanfeng and Yan, Sunyi and Xuan, Lisha and Yu, Guangrun and Guan, Xueying and Fang, Lei and Wang, Kai and Zhang, Tianzhen},
  doi          = {10.1038/s41588-025-02115-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {1021-1030},
  shortjournal = {Nat. Genet.},
  title        = {Post-polyploidization centromere evolution in cotton},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A telomere-to-telomere genome assembly coupled with multi-omic data provides insights into the evolution of hexaploid bread wheat. <em>NG</em>, <em>57</em>(4), 1008-1020. (<a href='https://doi.org/10.1038/s41588-025-02137-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The complete assembly of vast and complex plant genomes, like the hexaploid wheat genome, remains challenging. Here we present CS-IAAS, a comprehensive telomere-to-telomere (T2T) gap-free Triticum aestivum L. genome, encompassing 14.51 billion base pairs and featuring all 21 centromeres and 42 telomeres. Annotation revealed 90.8 Mb additional centromeric satellite arrays and 5,611 rDNA units. Genome-wide rearrangements, centromeric elements, transposable element expansion and segmental duplications were deciphered during tetraploidization and hexaploidization, providing a comprehensive understanding of wheat subgenome evolution. Among them, transposable element insertions during hexaploidization greatly influenced gene expression balances, thus increasing the genome plasticity of transcriptional levels. Additionally, we generated 163,329 full-length cDNA sequences and proteomic data that helped annotate 141,035 high-confidence protein-coding genes. The complete T2T reference genome (CS-IAAS), along with its transcriptome and proteome, represents a significant step in our understanding of wheat genome complexity and provides insights for future wheat research and breeding. A telomere-to-telomere genome assembly of allohexaploid bread wheat cv. Chinese Spring (CS-IAAS), along with transcriptomic and proteomic data, highlights wheat genome complexity and its structural and functional features.},
  archive      = {J_NG},
  author       = {Liu, Shoucheng and Li, Kui and Dai, Xiuru and Qin, Guochen and Lu, Dongdong and Gao, Zhaoxu and Li, Xiaopeng and Song, Bolong and Bian, Jianxin and Ren, Da and Liu, Yongqi and Chen, Xiaofeng and Xu, Yunbi and Liu, Weimin and Yang, Chen and Liu, Xiaoqin and Chen, Shisheng and Li, Jian and Li, Bosheng and He, Hang and Deng, Xing Wang},
  doi          = {10.1038/s41588-025-02137-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {1008-1020},
  shortjournal = {Nat. Genet.},
  title        = {A telomere-to-telomere genome assembly coupled with multi-omic data provides insights into the evolution of hexaploid bread wheat},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genomic analysis of 1,325 camellia accessions sheds light on agronomic and metabolic traits for tea plant improvement. <em>NG</em>, <em>57</em>(4), 997-1007. (<a href='https://doi.org/10.1038/s41588-025-02135-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The tea plant stands as a globally cherished nonalcoholic beverage crop, but the genetic underpinnings of important agronomic and metabolomic traits remain largely unexplored. Here we de novo deep resequenced 802 tea plants and their relative accessions globally. By integrating public Camellia accessions, we constructed a comprehensive genome-wide genetic variation map and annotated deleterious mutations for 1,325 accessions. Population genetic analyses provided insights into genetic divergence from its relatives, different evolutionary bottlenecks, interspecific introgression and conservation of wild relatives. Our findings suggest the pivotal role of southwest China as the origin of tea plants, revealing the genetic diversity and domestication status of ancient tea plants. Genome-wide association studies herein identified thousands of substantial associations with leaf shape and metabolite traits, pinpointing candidate genes for crucial agronomic and flavor traits. This study illuminates the tea plant’s evolution and provides references for tea plant design breeding. This study provides a de novo resequencing of 802 Camellia accessions and a genome-wide variation map. Genome-wide association study (GWAS) and metabolite GWAS identify genes responsible for crucial agronomic and flavor traits.},
  archive      = {J_NG},
  author       = {Kong, Weilong and Kong, Xiangrui and Xia, Zhongqiang and Li, Xiaofeng and Wang, Fang and Shan, Ruiyang and Chen, Zhihui and You, Xiaomei and Zhao, Yuanyan and Hu, Yanping and Zheng, Shiqin and Zhong, Sitong and Zhang, Shengcheng and Zhang, Yanbing and Fang, Kaixing and Wang, Yinghao and Liu, Hui and Zhang, Yazhen and Li, Xinlei and Wu, Hualing and Chen, Guo-Bo and Zhang, Xingtan and Chen, Changsong},
  doi          = {10.1038/s41588-025-02135-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {997-1007},
  shortjournal = {Nat. Genet.},
  title        = {Genomic analysis of 1,325 camellia accessions sheds light on agronomic and metabolic traits for tea plant improvement},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The genome and GeneBank genomics of allotetraploid nicotiana tabacum provide insights into genome evolution and complex trait regulation. <em>NG</em>, <em>57</em>(4), 986-996. (<a href='https://doi.org/10.1038/s41588-025-02126-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nicotiana tabacum is an allotetraploid hybrid of Nicotiana sylvestris and Nicotiana tomentosiformis and a model organism in genetics. However, features of subgenome evolution, expression coordination, genetic diversity and complex traits regulation of N. tabacum remain unresolved. Here we present chromosome-scale assemblies for all three species, and genotype and phenotypic data for 5,196 N. tabacum germplasms. Chromosome rearrangements and epigenetic modifications are associated with genome evolution and expression coordination following polyploidization. Two subgenomes and genes biased toward one subgenome contributed unevenly to complex trait variation. Using 178 marker–trait associations, a reference genotype-to-phenotype map was built for 39 morphological, developmental and disease resistance traits, and a novel gene regulating leaf width was validated. Signatures of positive and polygenic selection during the process of selective breeding were detected. Our study provides insights into genome evolution, complex traits regulation in allotetraploid N. tabacum and the use of GeneBank-scale resources for advancing genetic and genomic research. Chromosome-level genome assemblies of allotetraploid Nicotiana tabacum and its ancestors, along with their transcriptomes, epigenomes and genotype and phenotype data for 5,196 N. tabacum germplasms, provide insights into genome evolution and complex trait regulation.},
  archive      = {J_NG},
  author       = {Zan, Yanjun and Chen, Shuai and Ren, Min and Liu, Guoxiang and Liu, Yutong and Han, Yu and Dong, Yang and Zhang, Yao and Si, Huan and Liu, Zhengwen and Liu, Dan and Zhang, Xingwei and Tong, Ying and Li, Yuan and Jiang, Caihong and Wen, Liuying and Xiao, Zhiliang and Sun, Yangyang and Geng, Ruimei and Ji, Yan and Feng, Quanfu and Wang, Yuanying and Ye, Guoyou and Fang, Lingzhao and Chen, Yong and Cheng, Lirui and Yang, Aiguo},
  doi          = {10.1038/s41588-025-02126-0},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {986-996},
  shortjournal = {Nat. Genet.},
  title        = {The genome and GeneBank genomics of allotetraploid nicotiana tabacum provide insights into genome evolution and complex trait regulation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The genomic landscape of gene-level structural variations in japanese and global soybean glycine max cultivars. <em>NG</em>, <em>57</em>(4), 973-985. (<a href='https://doi.org/10.1038/s41588-025-02113-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Japanese soybeans are traditionally bred to produce soy foods such as tofu, miso and boiled soybeans. Here, to investigate their distinctive genomic features, including genomic structural variations (SVs), we constructed 11 nanopore-based genome references for Japanese and other soybean lines. Our assembly-based comparative method, designated ‘Asm2sv’, identified gene-level SVs comprehensively, enabling pangenome analysis of 462 worldwide cultivars and varieties. Based on these, we identified selective sweeps between Japanese and US soybeans, one of which was the pod-shattering resistance gene PDH1. Genome-wide association studies further identified several quantitative trait loci that accounted for large-seed phenotypes of Japanese soybean lines, some of which were also close to regions of the selective sweeps, including PDH1. Notably, specific combinations of alleles, including SVs, were found to increase the seed size of some Japanese landraces. In addition to the differences in cultivation environments, distinct food processing usages might result in changes in Japanese soybean genomes. Long-read genome assemblies for seven Japanese, three North American and one primitive Glycine max cultivars highlight gene-level structural variation underlying distinct seed morphology phenotypes.},
  archive      = {J_NG},
  author       = {Yano, Ryoichi and Li, Feng and Hiraga, Susumu and Takeshima, Ryoma and Kobayashi, Michie and Toda, Kyoko and Umehara, Yosuke and Kajiya-Kanegae, Hiromi and Iwata, Hiroyoshi and Kaga, Akito and Ishimoto, Masao},
  doi          = {10.1038/s41588-025-02113-5},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {973-985},
  shortjournal = {Nat. Genet.},
  title        = {The genomic landscape of gene-level structural variations in japanese and global soybean glycine max cultivars},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An eRNA transcription checkpoint for diverse signal-dependent enhancer activation programs. <em>NG</em>, <em>57</em>(4), 962-972. (<a href='https://doi.org/10.1038/s41588-025-02138-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The evidence that signal- and ligand-dependent pathways function by activating regulatory enhancer programs suggests that a ‘checkpoint’ strategy may underline activation of many diversely regulated enhancers. Here we report a molecular mechanism common to several acute signal- and ligand-dependent enhancer activation programs based on release of a shared enhancer RNA (eRNA) transcription checkpoint. It requires recruitment of a DNA-dependent protein kinase catalytic subunit (DNA-PKcs)-phosphorylated RING finger repressor (Krüppel-associated box)-associated protein 1 (KAP1) as a modulator, inhibiting its association with 7SK and E3 small ubiquitin-like modifier (SUMO) ligase activity on the CDK9 subunit of positive transcription elongation factor b (P-TEFb). This facilitates formation of an activated P-TEFb complex, licensing eRNA elongation. Overcoming this checkpoint for signal-dependent enhancer activation occurs in diverse pathways, including estrogen receptor-α, NF-κB-regulated proinflammatory stimulation, androgen receptor and neuronal depolarization. Therefore, a specific strategy required to convert a basal state enhancer P-TEFb complex to an active state to release a conserved checkpoint is apparently employed by several functionally important signal-regulated regulatory enhancers to implement the instructions of the endocrine and paracrine system. Phosphorylation of KAP1 by DNA-PKcs at enhancers regulated by diverse stimuli prevents association with 7SK small nuclear RNPs and CDK9 SUMOylation, thereby activating the P-TEFb complex and promoting enhancer RNA transcription.},
  archive      = {J_NG},
  author       = {Wang, Lishuan and Yuan, Wei and Gamliel, Amir and Ma, Wubin and Lee, Seowon and Tan, Yuliang and Chen, Zeyu and Taylor, Havilah and Ohgi, Kenneth and Oh, Soohwan and Aggarwal, Aneel K. and Rosenfeld, Michael G.},
  doi          = {10.1038/s41588-025-02138-w},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {962-972},
  shortjournal = {Nat. Genet.},
  title        = {An eRNA transcription checkpoint for diverse signal-dependent enhancer activation programs},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation. <em>NG</em>, <em>57</em>(4), 949-961. (<a href='https://doi.org/10.1038/s41588-024-02053-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequence-based machine-learning models trained on genomics data improve genetic variant interpretation by providing functional predictions describing their impact on the cis-regulatory code. However, current tools do not predict RNA-seq expression profiles because of modeling challenges. Here, we introduce Borzoi, a model that learns to predict cell-type-specific and tissue-specific RNA-seq coverage from DNA sequence. Using statistics derived from Borzoi’s predicted coverage, we isolate and accurately score DNA variant effects across multiple layers of regulation, including transcription, splicing and polyadenylation. Evaluated on quantitative trait loci, Borzoi is competitive with and often outperforms state-of-the-art models trained on individual regulatory functions. By applying attribution methods to the derived statistics, we extract cis-regulatory motifs driving RNA expression and post-transcriptional regulation in normal tissues. The wide availability of RNA-seq data across species, conditions and assays profiling specific aspects of regulation emphasizes the potential of this approach to decipher the mapping from DNA sequence to regulatory function. Borzoi adapts the Enformer sequence-to-expression model to directly predict RNA-seq coverage, enabling the in-silico analysis of variant effects across multiple layers of gene regulation.},
  archive      = {J_NG},
  author       = {Linder, Johannes and Srivastava, Divyanshi and Yuan, Han and Agarwal, Vikram and Kelley, David R.},
  doi          = {10.1038/s41588-024-02053-6},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {949-961},
  shortjournal = {Nat. Genet.},
  title        = {Predicting RNA-seq coverage from DNA sequence as a unifying model of gene regulation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The landscape of n6-methyladenosine in localized primary prostate cancer. <em>NG</em>, <em>57</em>(4), 934-948. (<a href='https://doi.org/10.1038/s41588-025-02128-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {N6-methyladenosine (m6A), the most abundant internal RNA modification in humans, regulates most aspects of RNA processing. Prostate cancer is characterized by widespread transcriptomic dysregulation; therefore, we characterized the m6A landscape of 162 localized prostate tumors with matched DNA, RNA and protein profiling. m6A abundance varied dramatically across tumors, with global patterns emerging via complex germline–somatic cooperative regulation. Individual germline polymorphisms regulated m6A abundance, cooperating with somatic mutation of cancer driver genes and m6A regulators. The resulting complex patterns were associated with prognostic clinical features and established the biomarker potential of global and locus-specific m6A patterns. Tumor hypoxia dysregulates m6A profiles, bridging prior genomic and proteomic observations. Specific m6A sites, such as those in VCAN, drive disease aggression, associating with poor outcomes, tumor growth and metastasis. m6A dysregulation is thus associated with key events in the natural history of prostate cancer: germline risk, microenvironmental dysregulation, somatic mutation and metastasis. Transcriptome-wide m6A RNA methylation profile in 162 primary prostate tumors identifies m6A association with prognostic clinical features and disease aggression.},
  archive      = {J_NG},
  author       = {Xu, Xin and Zhu, Helen and Hugh-White, Rupert and Livingstone, Julie and Eng, Stefan and Zeltser, Nicole and Wang, Yujuan and Pajdzik, Kinga and Chen, Sujun and Houlahan, Kathleen E. and Luo, Wenqin and Liu, Shun and Xu, Xi and Sheng, Minzhi and Guo, Wang Yuan and Arbet, Jaron and Song, Yuxi and Wang, Miranda and Zeng, Yong and Wang, Shiyan and Zhu, Guanghui and Gao, Tingxiao and Chen, Wei and Ci, Xinpei and Xu, Wenjie and Xu, Kexin and Orain, Michele and Picard, Valerie and Hovington, Helene and Bergeron, Alain and Lacombe, Louis and Têtu, Bernard and Fradet, Yves and Lupien, Mathieu and Wei, Gong-Hong and Koritzinsky, Marianne and Bristow, Robert G. and Fleshner, Neil E. and Wu, Xue and Shao, Yang and He, Chuan and Berlin, Alejandro and van der Kwast, Theodorus and Leong, Hon and Boutros, Paul C. and He, Housheng Hansen},
  doi          = {10.1038/s41588-025-02128-y},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {934-948},
  shortjournal = {Nat. Genet.},
  title        = {The landscape of n6-methyladenosine in localized primary prostate cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatially resolved transcriptomic analysis of the adult human prostate. <em>NG</em>, <em>57</em>(4), 922-933. (<a href='https://doi.org/10.1038/s41588-025-02139-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The prostate is an organ characterized by significant spatial heterogeneity. To better understand its intricate structure and cellular composition, we constructed a comprehensive single-cell atlas of the adult human prostate. Our high-resolution mapping effort identified 253,381 single cells and 34,876 nuclei sampled from 11 patients who underwent radical resection of bladder cancer, which were categorized into 126 unique subpopulations. This work revealed various new cell types in the human prostate and their specific spatial localization. Notably, we discovered four distinct acini, two of which were tightly associated with E-twenty-six transcription factor family (ETS)-fusion-negative prostate cancer. Through the integration of spatial, single-cell and bulk-seq analyses, we propose that two specific luminal cell types could serve as the common origins of prostate cancer. Additionally, our findings suggest that zone-specific fibroblasts may contribute to the observed heterogeneity among luminal cells. This atlas will serve as a valuable reference for studying prostate biology and diseases such as prostate cancer. Multi-omic analysis of the adult human prostate identifies spatially resolved cell populations with potential links to prostate carcinogenesis.},
  archive      = {J_NG},
  author       = {Hu, Junyi and Liu, Fei and Zhang, Jing and Yin, Lei and Cao, Wanli and Xu, Weidong and Chang, Yifan and Wang, Ye and Wang, Jian and Hou, Yaxin and Liu, Lilong and Chen, Sujun and Zhu, Guanghui and Jiang, Junhui and Wang, Zixian and Wei, Gong-Hong and He, Housheng Hansen and Gu, Di and Chen, Ke and Ren, Shancheng},
  doi          = {10.1038/s41588-025-02139-9},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {922-933},
  shortjournal = {Nat. Genet.},
  title        = {Spatially resolved transcriptomic analysis of the adult human prostate},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantitative characterization of tissue states using multiomics and ecological spatial analysis. <em>NG</em>, <em>57</em>(4), 910-921. (<a href='https://doi.org/10.1038/s41588-025-02119-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The spatial organization of cells in tissues underlies biological function, and recent advances in spatial profiling technologies have enhanced our ability to analyze such arrangements to study biological processes and disease progression. We propose MESA (multiomics and ecological spatial analysis), a framework drawing inspiration from ecological concepts to delineate functional and spatial shifts across tissue states. MESA introduces metrics to systematically quantify spatial diversity and identify hot spots, linking spatial patterns to phenotypic outcomes, including disease progression. Furthermore, MESA integrates spatial and single-cell multiomics data to facilitate an in-depth, molecular understanding of cellular neighborhoods and their spatial interactions within tissue microenvironments. Applying MESA to diverse datasets demonstrates additional insights it brings over prior methods, including newly identified spatial structures and key cell populations linked to disease states. Available as a Python package, MESA offers a versatile framework for quantitative decoding of tissue architectures in spatial omics across health and disease. Multiomics and ecological spatial analysis (MESA) calculates ecodiversity-inspired metrics in spatially resolved omics integrated with single-cell data, enabling the quantitative comparison of tissue states across a range of conditions.},
  archive      = {J_NG},
  author       = {Ding, Daisy Yi and Tang, Zeyu and Zhu, Bokai and Ren, Hongyu and Shalek, Alex K. and Tibshirani, Robert and Nolan, Garry P.},
  doi          = {10.1038/s41588-025-02119-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {910-921},
  shortjournal = {Nat. Genet.},
  title        = {Quantitative characterization of tissue states using multiomics and ecological spatial analysis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantitative characterization of cell niches in spatially resolved omics data. <em>NG</em>, <em>57</em>(4), 897-909. (<a href='https://doi.org/10.1038/s41588-025-02120-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial omics enable the characterization of colocalized cell communities that coordinate specific functions within tissues. These communities, or niches, are shaped by interactions between neighboring cells, yet existing computational methods rarely leverage such interactions for their identification and characterization. To address this gap, here we introduce NicheCompass, a graph deep-learning method that models cellular communication to learn interpretable cell embeddings that encode signaling events, enabling the identification of niches and their underlying processes. Unlike existing methods, NicheCompass quantitatively characterizes niches based on communication pathways and consistently outperforms alternatives. We show its versatility by mapping tissue architecture during mouse embryonic development and delineating tumor niches in human cancers, including a spatial reference mapping application. Finally, we extend its capabilities to spatial multi-omics, demonstrate cross-technology integration with datasets from different sequencing platforms and construct a whole mouse brain spatial atlas comprising 8.4 million cells, highlighting NicheCompass’ scalability. Overall, NicheCompass provides a scalable framework for identifying and analyzing niches through signaling events. NicheCompass is a graph variational autoencoder approach for identifying cellular niches defined by cell–cell communication and other interactions, applicable to a broad variety of spatial multi-omic data.},
  archive      = {J_NG},
  author       = {Birk, Sebastian and Bonafonte-Pardàs, Irene and Feriz, Adib Miraki and Boxall, Adam and Agirre, Eneritz and Memi, Fani and Maguza, Anna and Yadav, Anamika and Armingol, Erick and Fan, Rong and Castelo-Branco, Gonçalo and Theis, Fabian J. and Bayraktar, Omer Ali and Talavera-López, Carlos and Lotfollahi, Mohammad},
  doi          = {10.1038/s41588-025-02120-6},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {897-909},
  shortjournal = {Nat. Genet.},
  title        = {Quantitative characterization of cell niches in spatially resolved omics data},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The complexity of tobacco smoke-induced mutagenesis in head and neck cancer. <em>NG</em>, <em>57</em>(4), 884-896. (<a href='https://doi.org/10.1038/s41588-025-02134-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tobacco smoke, alone or combined with alcohol, is the predominant cause of head and neck cancer (HNC). We explore how tobacco exposure contributes to cancer development by mutational signature analysis of 265 whole-genome sequenced HNC samples from eight countries. Six tobacco-associated mutational signatures were detected, including some not previously reported. Differences in HNC incidence between countries corresponded with differences in mutation burdens of tobacco-associated signatures, consistent with the dominant role of tobacco in HNC causation. Differences were found in the burden of tobacco-associated signatures between anatomical subsites, suggesting that tissue-specific factors modulate mutagenesis. We identified an association between tobacco smoking and alcohol-related signatures, indicating a combined effect of these exposures. Tobacco smoking was associated with differences in the mutational spectra, repertoire of driver mutations in cancer genes and patterns of copy number change. Our results demonstrate the multiple pathways by which tobacco smoke can influence the evolution of cancer cell clones. Whole-genome sequencing and mutational signature analysis of 265 head and neck cancer samples collected from eight different countries provide insight into the vital contribution of tobacco smoke in disease etiology.},
  archive      = {J_NG},
  author       = {Torrens, Laura and Moody, Sarah and de Carvalho, Ana Carolina and Kazachkova, Mariya and Abedi-Ardekani, Behnoush and Cheema, Saamin and Senkin, Sergey and Cattiaux, Thomas and Cortez Cardoso Penha, Ricardo and Atkins, Joshua R. and Gaborieau, Valérie and Chopard, Priscilia and Carreira, Christine and Abbasi, Ammal and Bergstrom, Erik N. and Vangara, Raviteja and Wang, Jingwei and Fitzgerald, Stephen and Latimer, Calli and Diaz-Gay, Marcos and Jones, David and Teague, Jon and Ribeiro Pinto, Felipe and Kowalski, Luiz Paulo and Polesel, Jerry and Giudici, Fabiola and de Oliveira, José Carlos and Lagiou, Pagona and Lagiou, Areti and Vilensky, Marta and Mates, Dana and Mates, Ioan N. and Arantes, Lidia M. and Reis, Rui and Podesta, Jose Roberto V. and von Zeidler, Sandra V. and Holcatova, Ivana and Curado, Maria Paula and Canova, Cristina and Fabianova, Elenora and Rodríguez-Urrego, Paula A. and Humphreys, Laura and Alexandrov, Ludmil B. and Brennan, Paul and Stratton, Michael R. and Perdomo, Sandra},
  doi          = {10.1038/s41588-025-02134-0},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {884-896},
  shortjournal = {Nat. Genet.},
  title        = {The complexity of tobacco smoke-induced mutagenesis in head and neck cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Selection for somatic escape variants in SERPINA1 in the liver of patients with alpha-1 antitrypsin deficiency. <em>NG</em>, <em>57</em>(4), 875-883. (<a href='https://doi.org/10.1038/s41588-025-02125-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic variants accumulate in non-malignant tissues with age. Functional variants, leading to clonal advantage of hepatocytes, accumulate in the liver of patients with acquired chronic liver disease (CLD). Whether somatic variants are common to CLD from differing etiologies is unknown. We analyzed liver somatic variants in patients with genetic CLD from alpha-1 antitrypsin (A1AT) deficiency or hemochromatosis. We show that somatic variants in SERPINA1, the gene encoding A1AT, are strongly selected for in A1AT deficiency, with evidence of convergent evolution. Acquired SERPINA1 variants are clustered at the carboxyl terminus of A1AT, leading to truncation. In vitro and in vivo, C-terminal truncation variants reduce disease-associated Z-A1AT polymer accumulation and disruption of the endoplasmic reticulum, supporting the C-terminal domain swap mechanism. Therefore, somatic escape variants from a deleterious germline variant are selected for in A1AT deficiency, suggesting that functional somatic variants are disease-specific in CLD and point to disease-associated mechanisms. The expansion of clones with distinct SERPINA1 somatic mutants in the livers of alpha-1 antitrypsin deficiency (A1AT) patients is consistent with convergent evolution. These variants interfere with the auto-polymerization and intra-ER accumulation of the Z-A1AT protein, thus highlighting potentially targetable domains.},
  archive      = {J_NG},
  author       = {Brzozowska, Natalia and Wu, Lily Y. D. and Khodzhaeva, Vera and Griffiths, William J. and Duckworth, Adam and Jung, Hyunchul and Coorens, Tim H. H. and Hooks, Yvette and Chambers, Joseph E. and Campbell, Peter J. and Marciniak, Stefan J. and Hoare, Matthew},
  doi          = {10.1038/s41588-025-02125-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {875-883},
  shortjournal = {Nat. Genet.},
  title        = {Selection for somatic escape variants in SERPINA1 in the liver of patients with alpha-1 antitrypsin deficiency},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A likelihood-based framework for demographic inference from genealogical trees. <em>NG</em>, <em>57</em>(4), 865-874. (<a href='https://doi.org/10.1038/s41588-025-02129-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The demographic history of a population underlies patterns of genetic variation and is encoded in the gene-genealogical trees of the sampled haplotypes. Here we propose a demographic inference framework called the genealogical likelihood (gLike). Our method uses a graph-based structure to summarize the relationships among all lineages in a gene-genealogical tree with all possible trajectories of population memberships through time and derives the full likelihood across trees under a parameterized demographic model. We show through simulations and empirical applications that for populations that have experienced multiple admixtures, gLike can accurately estimate dozens of demographic parameters, including ancestral population sizes, admixture timing and admixture proportions, and it outperforms conventional demographic inference methods using the site frequency spectrum. Taken together, our proposed gLike framework harnesses underused genealogical information to offer high sensitivity and accuracy in inferring complex demographies for humans and other species. gLike infers population demographic histories with a variety of complex admixture events by analysis of graphs of states, which conceptualize the relationships of all lineages found in trees encoded in the ancestral recombination graph.},
  archive      = {J_NG},
  author       = {Fan, Caoqi and Cahoon, Jordan L. and Dinh, Bryan L. and Ortega-Del Vecchyo, Diego and Huber, Christian D. and Edge, Michael D. and Mancuso, Nicholas and Chiang, Charleston W. K.},
  doi          = {10.1038/s41588-025-02129-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {865-874},
  shortjournal = {Nat. Genet.},
  title        = {A likelihood-based framework for demographic inference from genealogical trees},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A structured coalescent model reveals deep ancestral structure shared by all modern humans. <em>NG</em>, <em>57</em>(4), 856-864. (<a href='https://doi.org/10.1038/s41588-025-02117-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the history of admixture events and population size changes leading to modern humans is central to human evolutionary genetics. Here we introduce a coalescence-based hidden Markov model, cobraa, that explicitly represents an ancestral population split and rejoin, and demonstrate its application on simulated and real data across multiple species. Using cobraa, we present evidence for an extended period of structure in the history of all modern humans, in which two ancestral populations that diverged ~1.5 million years ago came together in an admixture event ~300 thousand years ago, in a ratio of ~80:20%. Immediately after their divergence, we detect a strong bottleneck in the major ancestral population. We inferred regions of the present-day genome derived from each ancestral population, finding that material from the minority correlates strongly with distance to coding sequence, suggesting it was deleterious against the majority background. Moreover, we found a strong correlation between regions of majority ancestry and human–Neanderthal or human–Denisovan divergence, suggesting the majority population was also ancestral to those archaic humans. The cobraa model extends the pairwise sequentially Markovian coalescent to identify structured population history by examination of the model transition matrix. Applied to human polymorphism data, cobraa identifies an ancient admixture event ancestral to all modern humans.},
  archive      = {J_NG},
  author       = {Cousins, Trevor and Scally, Aylwyn and Durbin, Richard},
  doi          = {10.1038/s41588-025-02117-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {856-864},
  shortjournal = {Nat. Genet.},
  title        = {A structured coalescent model reveals deep ancestral structure shared by all modern humans},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rare loss-of-function variants in HECTD2 and AKAP11 confer risk of bipolar disorder. <em>NG</em>, <em>57</em>(4), 851-855. (<a href='https://doi.org/10.1038/s41588-025-02141-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bipolar disorder is a highly heritable psychiatric disorder; genome-wide association studies of bipolar disorder have yielded over 60 risk loci harboring common variants. To harness the information contained in rare loss-of-function (LOF) variants, holding promise for informing on the underlying biology, we performed a variant burden analysis for bipolar disorder using gene-based aggregation of LOF variants in whole-genome sequencing data from Iceland (4,197 cases, more than 200,000 controls) and the UK Biobank (1,881 cases, 426,622 controls). We found that HECTD2 was associated with bipolar disorder and confirmed it using the Bipolar Exome dataset. Meta-analysis with Bipolar Exome also revealed that LOF variants in AKAP11 were associated with bipolar disorder. Both associations with bipolar disorder are new, but AKAP11 has previously been associated with psychosis and schizophrenia. The products of AKAP11 and HECTD2 interact with GSK3β, a protein inhibited by lithium, the most effective mood stabilizer available to treat bipolar disorder. Analysis of whole-genome sequencing data from Iceland and the UK Biobank identifies an excess burden of rare loss-of-function variants in HECTD2 and AKAP11 in individuals diagnosed with bipolar disorder.},
  archive      = {J_NG},
  author       = {Thorgeirsson, Thorgeir E. and Tragante, Vinicius and Sveinbjornsson, Gardar and Jonsdottir, Gudrun A. and Walters, G. Bragi and Ivarsdottir, Erna V. and Arnadottir, Gudny A. and Sturluson, Arni and Jensson, Brynjar O. and Fridriksdottir, Run and Skuladottir, Astros Th. and Einarsson, Gudmundur and Bjornsdottir, Gyda and Gunnarsson, Arni F. and Gisladottir, Rosa S. and Sigurdsson, Asgeir and Oddsson, Asmundur and Jonsson, Hakon and Magnusson, Olafur Th. and Helgason, Hannes and Norddahl, Gudmundur and Thorleifsson, Gudmar and Haraldsson, Magnus and Sigurdsson, Engilbert and Holm, Hilma and Masson, Gisli and Gudbjartsson, Daniel F. and Stefansson, Hreinn and Sulem, Patrick and Stefansson, Kari},
  doi          = {10.1038/s41588-025-02141-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {851-855},
  shortjournal = {Nat. Genet.},
  title        = {Rare loss-of-function variants in HECTD2 and AKAP11 confer risk of bipolar disorder},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cross-ancestry and sex-stratified genome-wide association analyses of amygdala and subnucleus volumes. <em>NG</em>, <em>57</em>(4), 839-850. (<a href='https://doi.org/10.1038/s41588-025-02136-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The amygdala is a small but critical multi-nucleus structure for emotion, cognition and neuropsychiatric disorders. Although genetic associations with amygdala volumetric traits have been investigated in sex-combined European populations, cross-ancestry and sex-stratified analyses are lacking. Here we conducted cross-ancestry and sex-stratified genome-wide association analyses for 21 amygdala volumetric traits in 6,923 Chinese and 48,634 European individuals. We identified 191 variant–trait associations (P &lt; 2.38 × 10−9), including 47 new associations (12 new loci) in sex-combined univariate analyses and seven additional new loci in sex-combined and sex-stratified multivariate analyses. We identified 12 ancestry-specific and two sex-specific associations. The identified genetic variants include 16 fine-mapped causal variants and regulate amygdala and fetal brain gene expression. The variants were enriched for brain development and colocalized with mood, cognition and neuropsychiatric disorders. These results indicate that cross-ancestry and sex-stratified genetic association analyses may provide insight into the genetic architectures of amygdala and subnucleus volumes. Cross-ancestry and sex-stratified genome-wide association analyses for 21 amygdala volumetric traits in East Asian and European individuals provide insight into the genetic architectures of amygdala and subnucleus volumes.},
  archive      = {J_NG},
  author       = {Ji, Yuan and Liu, Nana and Yang, Yunjun and Wang, Meiyun and Cheng, Jingliang and Zhu, Wenzhen and Qiu, Shijun and Geng, Zuojun and Cui, Guangbin and Yu, Yongqiang and Liao, Weihua and Zhang, Hui and Gao, Bo and Xu, Xiaojun and Han, Tong and Yao, Zhenwei and Zhang, Quan and Qin, Wen and Liu, Feng and Liang, Meng and Wang, Sijia and Xu, Qiang and Xu, Jiayuan and Fu, Jilian and Zhang, Peng and Li, Wei and Shi, Dapeng and Wang, Caihong and Lui, Su and Yan, Zhihan and Chen, Feng and Zhang, Jing and Shen, Wen and Miao, Yanwei and Wang, Dawei and Gao, Jia-Hong and Zhang, Xiaochu and Xu, Kai and Zuo, Xi-Nian and Zhang, Longjiang and Ye, Zhaoxiang and Li, Mulin Jun and Xian, Junfang and Zhang, Bing and Yu, Chunshui},
  doi          = {10.1038/s41588-025-02136-y},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {839-850},
  shortjournal = {Nat. Genet.},
  title        = {Cross-ancestry and sex-stratified genome-wide association analyses of amygdala and subnucleus volumes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Common-variant and rare-variant genetic architecture of heart failure across the allele-frequency spectrum. <em>NG</em>, <em>57</em>(4), 829-838. (<a href='https://doi.org/10.1038/s41588-025-02140-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Heart failure is a complex trait, influenced by environmental and genetic factors, affecting over 30 million individuals worldwide. Here we report common-variant and rare-variant association studies of all-cause heart failure and examine how different classes of genetic variation impact its heritability. We identify 176 common-variant risk loci at genome-wide significance in 2,358,556 individuals and cluster these signals into five broad modules based on pleiotropic associations with anthropomorphic traits/obesity, blood pressure/renal function, atherosclerosis/lipids, immune activity and arrhythmias. In parallel, we uncover exome-wide significant associations for heart failure and rare predicted loss-of-function variants in TTN, MYBPC3, FLNC and BAG3 using exome sequencing of 376,334 individuals. We find that total burden heritability of rare coding variants is highly concentrated in a small set of Mendelian cardiomyopathy genes, while common-variant heritability is diffusely spread throughout the genome. Finally, we show that common-variant background modifies heart failure risk among carriers of rare pathogenic truncating variants in TTN. Together, these findings discern genetic links between dysregulated metabolism and heart failure and highlight a polygenic component to heart failure not captured by current clinical genetic testing. Common-variant and rare-variant association analyses combining datasets from multiple populations yield insights into the genetic architecture of all-cause heart failure across the allele-frequency spectrum.},
  archive      = {J_NG},
  author       = {Lee, David S. M. and Cardone, Kathleen M. and Zhang, David Y. and Tsao, Noah L. and Abramowitz, Sarah and Sharma, Pranav and DePaolo, John S. and Conery, Mitchell and Aragam, Krishna G. and Biddinger, Kiran and Dilitikas, Ozan and Hoffman-Andrews, Lily and Judy, Renae L. and Khan, Atlas and Kullo, Iftikhar J. and Puckelwartz, Megan J. and Reza, Nosheen and Satterfield, Benjamin A. and Singhal, Pankhuri and Arany, Zoltan and Cappola, Thomas P. and Carruth, Eric D. and Day, Sharlene M. and Do, Ron and Haggerty, Christopher M. and Joseph, Jacob and McNally, Elizabeth M. and Nadkarni, Girish and Owens, Anjali T. and Rader, Daniel J. and Ritchie, Marylyn D. and Sun, Yan V. and Voight, Benjamin F. and Levin, Michael G. and Damrauer, Scott M.},
  doi          = {10.1038/s41588-025-02140-2},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {829-838},
  shortjournal = {Nat. Genet.},
  title        = {Common-variant and rare-variant genetic architecture of heart failure across the allele-frequency spectrum},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome-wide association study meta-analysis provides insights into the etiology of heart failure and its subtypes. <em>NG</em>, <em>57</em>(4), 815-828. (<a href='https://doi.org/10.1038/s41588-024-02064-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Heart failure (HF) is a major contributor to global morbidity and mortality. While distinct clinical subtypes, defined by etiology and left ventricular ejection fraction, are well recognized, their genetic determinants remain inadequately understood. In this study, we report a genome-wide association study of HF and its subtypes in a sample of 1.9 million individuals. A total of 153,174 individuals had HF, of whom 44,012 had a nonischemic etiology (ni-HF). A subset of patients with ni-HF were stratified based on left ventricular systolic function, where data were available, identifying 5,406 individuals with reduced ejection fraction and 3,841 with preserved ejection fraction. We identify 66 genetic loci associated with HF and its subtypes, 37 of which have not previously been reported. Using functionally informed gene prioritization methods, we predict effector genes for each identified locus, and map these to etiologic disease clusters through phenome-wide association analysis, network analysis and colocalization. Through heritability enrichment analysis, we highlight the role of extracardiac tissues in disease etiology. We then examine the differential associations of upstream risk factors with HF subtypes using Mendelian randomization. These findings extend our understanding of the mechanisms underlying HF etiology and may inform future approaches to prevention and treatment. Genome-wide association analyses identify new risk loci for heart failure and its subtypes, extending understanding of the underlying biological pathways and disease mechanisms.},
  archive      = {J_NG},
  author       = {Henry, Albert and Mo, Xiaodong and Finan, Chris and Chaffin, Mark D. and Speed, Doug and Issa, Hanane and Denaxas, Spiros and Ware, James S. and Zheng, Sean L. and Malarstig, Anders and Gratton, Jasmine and Bond, Isabelle and Roselli, Carolina and Miller, David and Chopade, Sandesh and Schmidt, A. Floriaan and Abner, Erik and Adams, Lance and Andersson, Charlotte and Aragam, Krishna G. and Ärnlöv, Johan and Asselin, Geraldine and Raja, Anna Axelsson and Backman, Joshua D. and Bartz, Traci M. and Biddinger, Kiran J. and Biggs, Mary L. and Bloom, Heather L. and Boersma, Eric and Brandimarto, Jeffrey and Brown, Michael R. and Brunak, Søren and Bruun, Mie Topholm and Buckbinder, Leonard and Bundgaard, Henning and Carey, David J. and Chasman, Daniel I. and Chen, Xing and Cook, James P. and Czuba, Tomasz and de Denus, Simon and Dehghan, Abbas and Delgado, Graciela E. and Doney, Alexander S. and Dörr, Marcus and Dowsett, Joseph and Dudley, Samuel C. and Engström, Gunnar and Erikstrup, Christian and Esko, Tõnu and Farber-Eger, Eric H. and Felix, Stephan B. and Finer, Sarah and Ford, Ian and Ghanbari, Mohsen and Ghasemi, Sahar and Ghouse, Jonas and Giedraitis, Vilmantas and Giulianini, Franco and Gottdiener, John S. and Gross, Stefan and Guðbjartsson, Daníel F. and Gui, Hongsheng and Gutmann, Rebecca and Hägg, Sara and Haggerty, Christopher M. and Hedman, Åsa K. and Helgadottir, Anna and Hemingway, Harry and Hillege, Hans and Hyde, Craig L. and Aagaard Jensen, Bitten and Jukema, J. Wouter and Kardys, Isabella and Karra, Ravi and Kavousi, Maryam and Kizer, Jorge R. and Kleber, Marcus E. and Køber, Lars and Koekemoer, Andrea and Kuchenbaecker, Karoline and Lai, Yi-Pin and Lanfear, David and Langenberg, Claudia and Lin, Honghuang and Lind, Lars and Lindgren, Cecilia M. and Liu, Peter P. and London, Barry and Lowery, Brandon D. and Luan, Jian’an and Lubitz, Steven A. and Magnusson, Patrik and Margulies, Kenneth B. and Marston, Nicholas A. and Martin, Hilary and März, Winfried and Melander, Olle and Mordi, Ify R. and Morley, Michael P. and Morris, Andrew P. and Morrison, Alanna C. and Morton, Lori and Nagle, Michael W. and Nelson, Christopher P. and Niessner, Alexander and Niiranen, Teemu and Noordam, Raymond and Nowak, Christoph and O’Donoghue, Michelle L. and Ostrowski, Sisse Rye and Owens, Anjali T. and Palmer, Colin N. A. and Paré, Guillaume and Pedersen, Ole Birger and Perola, Markus and Pigeyre, Marie and Psaty, Bruce M. and Rice, Kenneth M. and Ridker, Paul M. and Romaine, Simon P. R. and Rotter, Jerome I. and Ruff, Christian T. and Sabatine, Marc S. and Sallah, Neneh and Salomaa, Veikko and Sattar, Naveed and Shalaby, Alaa A. and Shekhar, Akshay and Smelser, Diane T. and Smith, Nicholas L. and Sørensen, Erik and Srinivasan, Sundararajan and Stefansson, Kari and Sveinbjörnsson, Garðar and Svensson, Per and Tammesoo, Mari-Liis and Tardif, Jean-Claude and Teder-Laving, Maris and Teumer, Alexander and Thorgeirsson, Guðmundur and Thorsteinsdottir, Unnur and Torp-Pedersen, Christian and Tragante, Vinicius and Trompet, Stella and Uitterlinden, Andre G. and Ullum, Henrik and van der Harst, Pim and van Heel, David and van Setten, Jessica and van Vugt, Marion and Veluchamy, Abirami and Verschuuren, Monique and Verweij, Niek and Vissing, Christoffer Rasmus and Völker, Uwe and Voors, Adriaan A. and Wallentin, Lars and Wang, Yunzhang and Weeke, Peter E. and Wiggins, Kerri L. and Williams, L. Keoki and Yang, Yifan and Yu, Bing and Zannad, Faiez and Zheng, Chaoqun and Asselbergs, Folkert W. and Cappola, Thomas P. and Dubé, Marie-Pierre and Dunn, Michael E. and Lang, Chim C. and Samani, Nilesh J. and Shah, Svati and Vasan, Ramachandran S. and Smith, J. Gustav and Holm, Hilma and Shah, Sonia and Ellinor, Patrick T. and Hingorani, Aroon D. and Wells, Quinn and Lumbers, R. Thomas},
  doi          = {10.1038/s41588-024-02064-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {815-828},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association study meta-analysis provides insights into the etiology of heart failure and its subtypes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Secure and federated genome-wide association studies for biobank-scale datasets. <em>NG</em>, <em>57</em>(4), 809-814. (<a href='https://doi.org/10.1038/s41588-025-02109-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sharing data across institutions for genome-wide association studies (GWAS) would enhance the discovery of genetic variation linked to health and disease1,2. However, existing data-sharing regulations limit the scope of such collaborations3. Although cryptographic tools for secure computation promise to enable collaborative analysis with formal privacy guarantees, existing approaches either are computationally impractical or do not implement current state-of-the-art methods4–6. We introduce secure federated genome-wide association studies (SF-GWAS), a combination of secure computation frameworks and distributed algorithms that empowers efficient and accurate GWAS on private data held by multiple entities while ensuring data confidentiality. SF-GWAS supports widely used GWAS pipelines based on principal-component analysis or linear mixed models. We demonstrate the accuracy and practical runtimes of SF-GWAS on five datasets, including a UK Biobank cohort of 410,000 individuals, showcasing an order-of-magnitude improvement in runtime compared to previous methods. Our work enables secure collaborative genomic studies at unprecedented scale. SF-GWAS is a workflow for secure, federated genome-wide association studies, implementing accurate, privacy-preserving principal-component analysis, linear/logistic regression and linear mixed model methods for biobank-scale multisite analyses.},
  archive      = {J_NG},
  author       = {Cho, Hyunghoon and Froelicher, David and Chen, Jeffrey and Edupalli, Manaswitha and Pyrgelis, Apostolos and Troncoso-Pastoriza, Juan R. and Hubaux, Jean-Pierre and Berger, Bonnie},
  doi          = {10.1038/s41588-025-02109-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {809-814},
  shortjournal = {Nat. Genet.},
  title        = {Secure and federated genome-wide association studies for biobank-scale datasets},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Causal machine learning for single-cell genomics. <em>NG</em>, <em>57</em>(4), 797-808. (<a href='https://doi.org/10.1038/s41588-025-02124-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in single-cell '-omics' allow unprecedented insights into the transcriptional profiles of individual cells and, when combined with large-scale perturbation screens, enable measuring of the effect of targeted perturbations on the whole transcriptome. These advances provide an opportunity to better understand the causative role of genes in complex biological processes. In this Perspective, we delineate the application of causal machine learning to single-cell genomics and its associated challenges. We first present the causal model that is most commonly applied to single-cell biology and then identify and discuss potential approaches to three open problems: the lack of generalization of models to novel experimental conditions, the complexity of interpreting learned models, and the difficulty of learning cell dynamics. This Perspective explores causal machine learning in single-cell genomics, addressing challenges such as generalization, interpretability and cell dynamics, while highlighting advances and the potential to uncover new insights into cellular mechanisms.},
  archive      = {J_NG},
  author       = {Tejada-Lapuerta, Alejandro and Bertin, Paul and Bauer, Stefan and Aliee, Hananeh and Bengio, Yoshua and Theis, Fabian J.},
  doi          = {10.1038/s41588-025-02124-2},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {797-808},
  shortjournal = {Nat. Genet.},
  title        = {Causal machine learning for single-cell genomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The farm animal Genotype–Tissue expression (FarmGTEx) project. <em>NG</em>, <em>57</em>(4), 786-796. (<a href='https://doi.org/10.1038/s41588-025-02121-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic mutation and drift, coupled with natural and human-mediated selection and migration, have produced a wide variety of genotypes and phenotypes in farmed animals. We here introduce the Farm Animal Genotype–Tissue Expression (FarmGTEx) Project, which aims to elucidate the genetic determinants of gene expression across 16 terrestrial and aquatic domestic species under diverse biological and environmental contexts. For each species, we aim to collect multiomics data, particularly genomics and transcriptomics, from 50 tissues of 1,000 healthy adults and 200 additional animals representing a specific context. This Perspective provides an overview of the priorities of FarmGTEx and advocates for coordinated strategies of data analysis and resource-sharing initiatives. FarmGTEx aims to serve as a platform for investigating context-specific regulatory effects, which will deepen our understanding of molecular mechanisms underlying complex phenotypes. The knowledge and insights provided by FarmGTEx will contribute to improving sustainable agriculture-based food systems, comparative biology and eventual human biomedicine. The FarmGTEx Project aims to understand genetic control of gene activity under diverse biological and environmental contexts in domestic animals, providing a foundation for improving animal precision breeding, adaptation and human health.},
  archive      = {J_NG},
  author       = {Fang, Lingzhao and Teng, Jinyan and Lin, Qing and Bai, Zhonghao and Liu, Shuli and Guan, Dailu and Li, Bingjie and Gao, Yahui and Hou, Yali and Gong, Mian and Pan, Zhangyuan and Yu, Ying and Clark, Emily L. and Smith, Jacqueline and Rawlik, Konrad and Xiang, Ruidong and Chamberlain, Amanda J. and Goddard, Michael E. and Littlejohn, Mathew and Larson, Greger and MacHugh, David E. and O’Grady, John F. and Sørensen, Peter and Sahana, Goutam and Lund, Mogens Sandø and Jiang, Zhihua and Pan, Xiangchun and Gong, Wentao and Zhang, Haihan and He, Xi and Zhang, Yuebo and Gao, Ning and He, Jun and Yi, Guoqiang and Liu, Yuwen and Tang, Zhonglin and Zhao, Pengju and Zhou, Yang and Fu, Liangliang and Wang, Xiao and Hao, Dan and Liu, Lei and Chen, Siqian and Young, Robert S. and Shen, Xia and Xia, Charley and Cheng, Hao and Ma, Li and Cole, John B. and Baldwin, Ransom L. and Li, Cong-jun and Van Tassell, Curtis P. and Rosen, Benjamin D. and Bhowmik, Nayan and Lunney, Joan and Liu, Wansheng and Guan, Leluo and Zhao, Xin and Ibeagha-Awemu, Eveline M. and Luo, Yonglun and Lin, Lin and Canela-Xandri, Oriol and Derks, Martijn F. L. and Crooijmans, Richard P. M. A. and Gòdia, Marta and Madsen, Ole and Groenen, Martien A. M. and Koltes, James E. and Tuggle, Christopher K. and McCarthy, Fiona M. and Rocha, Dominique and Giuffra, Elisabetta and Amills, Marcel and Clop, Alex and Ballester, Maria and Tosser-Klopp, Gwenola and Li, Jing and Fang, Chao and Fang, Ming and Wang, Qishan and Hou, Zhuocheng and Wang, Qin and Zhao, Fuping and Jiang, Lin and Zhao, Guiping and Zhou, Zhengkui and Zhou, Rong and Liu, Hehe and Deng, Juan and Jin, Long and Li, Mingzhou and Mo, Delin and Liu, Xiaohong and Chen, Yaosheng and Yuan, Xiaolong and Li, Jiaqi and Zhao, Shuhong and Zhang, Yi and Ding, Xiangdong and Sun, Dongxiao and Sun, Hui-Zeng and Li, Cong and Wang, Yu and Jiang, Yu and Wu, Dongdong and Wang, Wenwen and Fan, Xinzhong and Zhang, Qin and Li, Kui and Zhang, Hao and Yang, Ning and Hu, Xiaoxiang and Huang, Wen and Song, Jiuzhou and Wu, Yang and Yang, Jian and Wu, Weiwei and Kasper, Claudia and Liu, Xinfeng and Yu, Xiaofei and Cui, Leilei and Zhou, Xiang and Kim, Seyoung and Li, Wei and Im, Hae Kyung and Buckler, Edward S. and Ren, Bing and Schatz, Michael C. and Li, Jingyi Jessica and Palmer, Abraham A. and Frantz, Laurent and Zhou, Huaijun and Zhang, Zhe and Liu, George E.},
  doi          = {10.1038/s41588-025-02121-5},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {786-796},
  shortjournal = {Nat. Genet.},
  title        = {The farm animal Genotype–Tissue expression (FarmGTEx) project},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Revealing the distinct genomic features of japanese soybeans. <em>NG</em>, <em>57</em>(4), 784-785. (<a href='https://doi.org/10.1038/s41588-025-02114-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Japanese soybeans have been bred for centuries for traditional soy-based food production. A pangenome study using genome data from 462 soybeans worldwide revealed distinct genomic features in Japanese soybeans, as well as several quantitative trait loci alleles that are important for breeding cultivars suited to traditional soy-based foods.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02114-4},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {784-785},
  shortjournal = {Nat. Genet.},
  title        = {Revealing the distinct genomic features of japanese soybeans},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A conserved checkpoint for ligand-dependent enhancer activation. <em>NG</em>, <em>57</em>(4), 782-783. (<a href='https://doi.org/10.1038/s41588-025-02146-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We identify that signal-dependent and ligand-dependent enhancer activation programs require release of a shared enhancer RNA (eRNA) transcription checkpoint. Further mechanistic evidence reveals signal-dependent activation of P-TEFb on enhancers, which licenses the enhancer transcriptional program by overcoming pausing of transcription and promoting eRNA elongation.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02146-w},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {782-783},
  shortjournal = {Nat. Genet.},
  title        = {A conserved checkpoint for ligand-dependent enhancer activation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Collaborative genome-wide association analysis with cryptography. <em>NG</em>, <em>57</em>(4), 780-781. (<a href='https://doi.org/10.1038/s41588-025-02110-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern genome-wide association study (GWAS) datasets are biobank scale and are distributed across isolated repositories, owing to privacy concerns and regulations. Secure federated GWAS leverages cryptographic tools to enable joint analysis of these siloed datasets, thereby enhancing a range of common analyses while providing formal guarantees of data privacy.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02110-8},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {780-781},
  shortjournal = {Nat. Genet.},
  title        = {Collaborative genome-wide association analysis with cryptography},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Borzoi decodes the complex DNA signals governing gene regulation. <em>NG</em>, <em>57</em>(4), 777-779. (<a href='https://doi.org/10.1038/s41588-025-02154-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Borzoi is a deep learning model that predicts RNA sequencing coverage across each exon of every human gene, across different cells and tissues, based on DNA sequence alone.},
  archive      = {J_NG},
  author       = {Cooper, Sandra T.},
  doi          = {10.1038/s41588-025-02154-w},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {777-779},
  shortjournal = {Nat. Genet.},
  title        = {Borzoi decodes the complex DNA signals governing gene regulation},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The interplay between germline and somatic variants in alpha-1 anti-trypsin deficiency liver disease. <em>NG</em>, <em>57</em>(4), 775-776. (<a href='https://doi.org/10.1038/s41588-025-02151-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The biological significance of somatic mutations in rare genetic liver diseases has remained elusive. A study using genomic sequencing paired with experimental cell-based assays has shed light on the selective advantage and accumulation of somatic variants in SERPINA1 found in liver explants from patients with alpha-1 anti-trypsin deficiency.},
  archive      = {J_NG},
  author       = {Vilarinho, Sílvia},
  doi          = {10.1038/s41588-025-02151-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {775-776},
  shortjournal = {Nat. Genet.},
  title        = {The interplay between germline and somatic variants in alpha-1 anti-trypsin deficiency liver disease},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Solanum pan-genome highlights paralog diversification. <em>NG</em>, <em>57</em>(4), 774. (<a href='https://doi.org/10.1038/s41588-025-02178-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-025-02178-2},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {774},
  shortjournal = {Nat. Genet.},
  title        = {Solanum pan-genome highlights paralog diversification},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetics of coronary artery dominance. <em>NG</em>, <em>57</em>(4), 774. (<a href='https://doi.org/10.1038/s41588-025-02179-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-025-02179-1},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {774},
  shortjournal = {Nat. Genet.},
  title        = {Genetics of coronary artery dominance},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The PCOS-endometrial cell atlas. <em>NG</em>, <em>57</em>(4), 774. (<a href='https://doi.org/10.1038/s41588-025-02180-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02180-8},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {774},
  shortjournal = {Nat. Genet.},
  title        = {The PCOS-endometrial cell atlas},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Somatic mutations in the stomach. <em>NG</em>, <em>57</em>(4), 774. (<a href='https://doi.org/10.1038/s41588-025-02181-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02181-7},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {774},
  shortjournal = {Nat. Genet.},
  title        = {Somatic mutations in the stomach},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mapping genetic diversity with the GenomeIndia project. <em>NG</em>, <em>57</em>(4), 767-773. (<a href='https://doi.org/10.1038/s41588-025-02153-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rich ethnolinguistic and sociocultural differences that exist in India offers a unique opportunity to study human diversity. With the whole genomes of 10,000 healthy and unrelated Indians from 83 populations, the GenomeIndia project captures the genetic diversity of one of the highly underrepresented populations in the global genomics landscape.},
  archive      = {J_NG},
  author       = {Bhattacharyya, Chandrika and Subramanian, Krithika and Uppili, Bharathram and Biswas, Nidhan K. and Ramdas, Shweta and Tallapaka, Karthik Bharadwaj and Arvind, Prathima and Rupanagudi, Khader Valli and Maitra, Arindam and Nagabandi, Tulasi and De, Tiyasha and Singh, Kuldeep and Sharma, Praveen and Sharma, Nanaocha and Raghav, Sunil K. and Prasad, Punit and Soniya, E. V. and Jaleel, Abdul and Nelson Sathi, Shijulal and Joshi, Madhvi and Joshi, Chaitanya and Lahiri, Mayurika and Dixit, Santosh and Shashidhara, L. S. and Senthil Kumar, Nachimuthu and Lalhruaitluanga, H. and Nundanga, Lal and Shivakumar, Venkataram and Venkatasubramanian, Ganesan and Rao, Naren P. and Ganie, Mohd Ashraf and Wani, Imtiyaz Ahmad and Jha, Ganganath and Dalal, Ashwin and Bashyam, Murali Dharan and Varadwaj, Pritish Kumar and BS, Sanjeev and Simmhan, Yogesh and Jain, Chirag and Sundar, Durai and Gupta, Ishaan and Yadav, Pankaj and Sinha, Himanshu and Narayanan, Manikandan and Raman, Karthik and Padinjat, Raghu and Sabarinathan, Radhakrishnan and Narahari, Yadati and Ravindranath, Vijayalakshmi and Kumarasamy, Thangaraj and Sowpati, Divya Tej and Faruq, Mohammed and Basu, Analabha and Kahali, Bratati},
  doi          = {10.1038/s41588-025-02153-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {767-773},
  shortjournal = {Nat. Genet.},
  title        = {Mapping genetic diversity with the GenomeIndia project},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Equity, ethics and the promise of partnership in genomic medicine. <em>NG</em>, <em>57</em>(4), 765-766. (<a href='https://doi.org/10.1038/s41588-025-02131-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Hawley, Nicola L. and Viali, Satupaitea},
  doi          = {10.1038/s41588-025-02131-3},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {765-766},
  shortjournal = {Nat. Genet.},
  title        = {Equity, ethics and the promise of partnership in genomic medicine},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: Functional innovation through new genes as a general evolutionary process. <em>NG</em>, <em>57</em>(3), 764. (<a href='https://doi.org/10.1038/s41588-025-02116-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Xia, Shengqian and Chen, Jianhai and Arsala, Deanna and Emerson, J. J. and Long, Manyuan},
  doi          = {10.1038/s41588-025-02116-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {764},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Functional innovation through new genes as a general evolutionary process},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: The NAT1–bHLH110–CER1/CER1L module regulates heat stress tolerance in rice. <em>NG</em>, <em>57</em>(3), 763. (<a href='https://doi.org/10.1038/s41588-025-02106-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Lu, Hai-Ping and Liu, Xue-Huan and Wang, Mei-Jing and Zhu, Qiao-Yun and Lyu, Yu-Shu and Xu, Jian-Hang and Liu, Jian-Xiang},
  doi          = {10.1038/s41588-025-02106-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {763},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: The NAT1–bHLH110–CER1/CER1L module regulates heat stress tolerance in rice},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Pangenome graphs and their applications in biodiversity genomics. <em>NG</em>, <em>57</em>(3), 763. (<a href='https://doi.org/10.1038/s41588-025-02132-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Secomandi, Simona and Gallo, Guido Roberto and Rossi, Riccardo and Rodríguez Fernandes, Carlos and Jarvis, Erich D. and Bonisoli-Alquati, Andrea and Gianfranceschi, Luca and Formenti, Giulio},
  doi          = {10.1038/s41588-025-02132-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {763},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Pangenome graphs and their applications in biodiversity genomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Origin and de novo domestication of sweet orange. <em>NG</em>, <em>57</em>(3), 754-762. (<a href='https://doi.org/10.1038/s41588-025-02122-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sweet orange is cultivated worldwide but suffers from various devastating diseases because of its monogenetic background. The elucidation of the origin of a crop facilitates the domestication of new crops that may better cope with new challenges. Here we collected and sequenced 226 citrus accessions and assembled telomere-to-telomere phased diploid genomes of sweet orange and sour orange. On the basis of a high-resolution haplotype-resolved genome analysis, we inferred that sweet orange originated from a sour orange × mandarin cross and confirmed this model using artificial hybridization experiments. We identified defense-related metabolites that potently inhibited the growth of multiple industrially important pathogenic bacteria. We introduced diversity to sweet orange, which showed wide segregation in fruit flavor and disease resistance and produced canker-resistant sweet orange by selecting defense-related metabolites. Our findings elucidate the origin of sweet orange and de novo domesticated disease-resistant sweet oranges, illuminating a strategy for the rapid domestication of perennial crops. Genomic analyses of Citrus species including haplotype-resolved genomes of Citrus sinensis and Citrus aurantium highlight the origin of sweet orange and provide a strategy for de novo domestication of perennial crops.},
  archive      = {J_NG},
  author       = {Liu, Shengjun and Xu, Yuantao and Yang, Kun and Huang, Yue and Lu, Zhihao and Chen, Shulin and Gao, Xiang and Xiao, Gongao and Chen, Peng and Zeng, Xiuli and Wang, Lun and Zheng, Weikang and Liu, Zishuang and Liao, Guanglian and He, Fa and Liu, Junjie and Wan, Pengfei and Ding, Fang and Ye, Junli and Jiao, Wenbiao and Chai, Lijun and Pan, Zhiyong and Zhang, Fei and Lin, Zongcheng and Zan, Yanjun and Guo, Wenwu and Larkin, Robert M. and Xie, Zongzhou and Wang, Xia and Deng, Xiuxin and Xu, Qiang},
  doi          = {10.1038/s41588-025-02122-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {754-762},
  shortjournal = {Nat. Genet.},
  title        = {Origin and de novo domestication of sweet orange},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Super pangenome of vitis empowers identification of downy mildew resistance genes for grapevine improvement. <em>NG</em>, <em>57</em>(3), 741-753. (<a href='https://doi.org/10.1038/s41588-025-02111-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Grapevine (Vitis) is one of the oldest domesticated fruit crops with great cultural and economic importance. Here we assembled and annotated haplotype-resolved genomes of 72 global Vitis accessions including 25 wild and 47 cultivated grapevines, among which genomes for 60 grapevines are newly released. Haplotype-aware phylogenomics disentangled the mysterious hybridization history of grapevines, revealing the enormous genetic diversity of the Vitis genus. Pangenomic analysis reveals that European cultivars, more susceptible to the destructive disease downy mildew (DM), have a smaller repertoire of resistance genes in the NLR family encoding the TIR-NBARC-LRR domain. Through extensive structural variation (SV) characterization, phenotyping, DM-infection transcriptome profiling of 113 Vitis accessions, and SV–expression quantitative trait loci analysis, we have identified over 63 SVs and their relevant genes significantly associated with DM resistance, exemplified by a lysine histidine transporter, VvLHT8. This haplotype-resolved super pangenome of the Vitis genus will accelerate breeding and enrich our understanding of the evolution and biology of grapevines. Super pangenome of the Vitis genus assembled from haplotype-resolved genomes of 72 global Vitis accessions characterizes the landscape of structural variants (SVs) and identifies SV–expression quantitative trait loci for downy mildew resistance.},
  archive      = {J_NG},
  author       = {Guo, Li and Wang, Xiangfeng and Ayhan, Dilay Hazal and Rhaman, Mohammad Saidur and Yan, Ming and Jiang, Jianfu and Wang, Dongyue and Zheng, Wei and Mei, Junjie and Ji, Wei and Jiao, Jian and Chen, Shaoying and Sun, Jie and Yi, Shu and Meng, Dian and Wang, Jing and Bhuiyan, Mohammad Nasim and Qin, Guochen and Guo, Linling and Yang, Qingxian and Zhang, Xuenan and Sun, Haisheng and Liu, Chonghuai and Deng, Xing Wang and Ye, Wenxiu},
  doi          = {10.1038/s41588-025-02111-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {741-753},
  shortjournal = {Nat. Genet.},
  title        = {Super pangenome of vitis empowers identification of downy mildew resistance genes for grapevine improvement},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The marchantia polymorpha pangenome reveals ancient mechanisms of plant adaptation to the environment. <em>NG</em>, <em>57</em>(3), 729-740. (<a href='https://doi.org/10.1038/s41588-024-02071-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plant adaptation to terrestrial life started 450 million years ago and has played a major role in the evolution of life on Earth. The genetic mechanisms allowing this adaptation to a diversity of terrestrial constraints have been mostly studied by focusing on flowering plants. Here, we gathered a collection of 133 accessions of the model bryophyte Marchantia polymorpha and studied its intraspecific diversity using selection signature analyses, a genome–environment association study and a pangenome. We identified adaptive features, such as peroxidases or nucleotide-binding and leucine-rich repeats (NLRs), also observed in flowering plants, likely inherited from the first land plants. The M. polymorpha pangenome also harbors lineage-specific accessory genes absent from seed plants. We conclude that different land plant lineages still share many elements from the genetic toolkit evolved by their most recent common ancestor to adapt to the terrestrial habitat, refined by lineage-specific polymorphisms and gene family evolution. Pangenome analyses of 133 wild accessions of the model bryophyte Marchantia polymorpha identify adaptive features and provide insights into the mechanisms of plant adaptation to the terrestrial environment.},
  archive      = {J_NG},
  author       = {Beaulieu, Chloé and Libourel, Cyril and Mbadinga Zamar, Duchesse Lacourt and El Mahboubi, Karima and Hoey, David J. and Greiff, George R. L. and Keller, Jean and Girou, Camille and San Clemente, Helene and Diop, Issa and Amblard, Emilie and Castel, Baptiste and Théron, Anthony and Cauet, Stéphane and Rodde, Nathalie and Zachgo, Sabine and Halpape, Wiebke and Meierhenrich, Anja and Laker, Bianca and Bräutigam, Andrea and Szovenyi, Peter and Cheng, Shifeng and Tanizawa, Yasuhiro and Aziz, Simon and Leebens-Mack, James H. and Schmutz, Jeremy and Webber, Jenell and Grimwood, Jane and Jacquet, Christophe and Dunand, Christophe and Nelson, Jessica M. and Roux, Fabrice and Philippe, Hervé and Schornack, Sebastian and Bonhomme, Maxime and Delaux, Pierre-Marc},
  doi          = {10.1038/s41588-024-02071-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {729-740},
  shortjournal = {Nat. Genet.},
  title        = {The marchantia polymorpha pangenome reveals ancient mechanisms of plant adaptation to the environment},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Functional analysis of cancer-associated germline risk variants. <em>NG</em>, <em>57</em>(3), 718-728. (<a href='https://doi.org/10.1038/s41588-024-02070-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-nucleotide variants (SNVs) in regulatory DNA are linked to inherited cancer risk. Massively parallel reporter assays of 4,041 SNVs linked to 13 neoplasms comprising &gt;90% of human malignancies were performed in pertinent primary human cell types and then integrated with matching chromatin accessibility, DNA looping and expression quantitative trait loci data to nominate 380 potentially regulatory SNVs and their putative target genes. The latter highlighted specific protein networks in lifetime cancer risk, including mitochondrial translation, DNA damage repair and Rho GTPase activity. A CRISPR knockout screen demonstrated that a subset of germline putative risk genes also enables the growth of established cancers. Editing one SNV, rs10411210 , showed that its risk allele increases rhophilin RHPN2 expression and stimulus-responsive RhoA activation, indicating that individual SNVs may upregulate cancer-linked pathways. These functional data are a resource for variant prioritization efforts and further interrogation of the mechanisms underlying inherited risk for cancer. Analysis of 4,041 single-nucleotide variants (SNVs) linked to 13 cancers performed in primary human cell types identifies 380 potentially regulatory SNVs and their putative target genes. Editing one SNV, rs10411210 , revealed that the risk allele increases RHPN2 expression and stimulus-responsive RhoA activation.},
  archive      = {J_NG},
  author       = {Kellman, Laura N. and Neela, Poornima H. and Srinivasan, Suhas and Siprashvili, Zurab and Shanderson, Ronald L. and Hong, Audrey W. and Rao, Deepti and Porter, Douglas F. and Reynolds, David L. and Meyers, Robin M. and Guo, Margaret G. and Yang, Xue and Zhao, Yang and Wozniak, Glenn G. and Donohue, Laura K. H. and Shenoy, Rajani and Ko, Lisa A. and Nguyen, Duy T. and Mondal, Smarajit and Garcia, Omar S. and Elcavage, Lara E. and Elfaki, Ibtihal and Abell, Nathan S. and Tao, Shiying and Lopez, Christopher M. and Montgomery, Stephen B. and Khavari, Paul A.},
  doi          = {10.1038/s41588-024-02070-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {718-728},
  shortjournal = {Nat. Genet.},
  title        = {Functional analysis of cancer-associated germline risk variants},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quantifying cell divisions along evolutionary lineages in cancer. <em>NG</em>, <em>57</em>(3), 706-717. (<a href='https://doi.org/10.1038/s41588-025-02078-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell division drives somatic evolution but is challenging to quantify. We developed a framework to count cell divisions with DNA replication-related mutations in polyguanine homopolymers. Analyzing 505 samples from 37 patients, we studied the milestones of colorectal cancer evolution. Primary tumors diversify at ~250 divisions from the founder cell, while distant metastasis divergence occurs significantly later, at ~500 divisions. Notably, distant but not lymph node metastases originate from primary tumor regions that have undergone surplus divisions, tying subclonal expansion to metastatic capacity. Then, we analyzed a cohort of 73 multifocal lung cancers and showed that the cell division burden of the tumors’ common ancestor distinguishes independent primary tumors from intrapulmonary metastases and correlates with patient survival. In lung cancer too, metastatic capacity is tied to more extensive proliferation. The cell division history of human cancers is easily accessible using our simple framework and contains valuable biological and clinical information. This work presents a framework for estimating cell division numbers using DNA replication-associated polyguanine tract mutations, with applications for understanding tumor natural histories and origins.},
  archive      = {J_NG},
  author       = {Blohmer, Martin and Cheek, David M. and Hung, Wei-Ting and Kessler, Maria and Chatzidimitriou, Foivos and Wang, Jiahe and Hung, William and Lee, I-Hsiu and Gorelick, Alexander N. and Wassenaar, Emma CE and Yang, Ching-Yeuh and Yeh, Yi-Chen and Ho, Hsiang-Ling and Speiser, Dorothee and Karsten, Maria M. and Lanuti, Michael and Pai, Sara I. and Kranenburg, Onno and Lennerz, Jochen K. and Chou, Teh-Ying and Kloor, Matthias and Naxerova, Kamila},
  doi          = {10.1038/s41588-025-02078-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {706-717},
  shortjournal = {Nat. Genet.},
  title        = {Quantifying cell divisions along evolutionary lineages in cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ImmuneLENS characterizes systemic immune dysregulation in aging and cancer. <em>NG</em>, <em>57</em>(3), 694-705. (<a href='https://doi.org/10.1038/s41588-025-02086-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recognition and elimination of pathogens and cancer cells depend on the adaptive immune system. Thus, accurate quantification of immune subsets is vital for precision medicine. We present immune lymphocyte estimation from nucleotide sequencing (ImmuneLENS), which estimates T cell and B cell fractions, class switching and clonotype diversity from whole-genome sequencing data at depths as low as 5× coverage. By applying ImmuneLENS to the 100,000 Genomes Project, we identify genes enriched with somatic mutations in T cell-rich tumors, significant sex-based differences in circulating T cell fraction and demonstrated that the circulating T cell fraction in patients with cancer is significantly lower than in healthy individuals. Low circulating B cell fraction was linked to increased cancer incidence. Finally, circulating T cell abundance was more prognostic of 5-year cancer survival than infiltrating T cells. Immune lymphocyte estimation from nucleotide sequencing (ImmuneLENS) infers B cell and T cell fractions from whole-genome sequencing data. Applied to the 100,000 Genomes Project datasets, circulating T cell fraction provides sex-dependent and prognostic insights in patients.},
  archive      = {J_NG},
  author       = {Bentham, Robert and Jones, Thomas P. and Black, James R. M. and Martinez-Ruiz, Carlos and Dietzen, Michelle and Litovchenko, Maria and Thol, Kerstin and Watkins, Thomas B. K. and Bailey, Chris and Pich, Oriol and Zhang, Zhihui and Van Loo, Peter and Swanton, Charles and McGranahan, Nicholas},
  doi          = {10.1038/s41588-025-02086-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {694-705},
  shortjournal = {Nat. Genet.},
  title        = {ImmuneLENS characterizes systemic immune dysregulation in aging and cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Alterations in PD-l1 succinylation shape anti-tumor immune responses in melanoma. <em>NG</em>, <em>57</em>(3), 680-693. (<a href='https://doi.org/10.1038/s41588-025-02077-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumors undergo metabolic reprogramming to meet the energetic, synthetic and redox demands essential for malignancy, often characterized by increased glycolysis and lactate production. However, the role of mitochondrial metabolism in tumor immunity remains unclear. The present study integrates spatial transcriptomics, bulk transcriptomics and proteomics, revealing a strong link between the metabolite succinyl-CoA and tumor immunity as well as the efficacy of anti-programmed cell death protein-1 (PD-1) therapy in patients with melanoma. Elevated succinyl-CoA levels, through α-ketoglutarate or succinate supplementation, enhanced T cell-mediated tumor elimination, both in vitro and in vivo. Mechanistically, succinylation of the ligand of PD-1 (PD-L1) at lysine 129 led to its degradation. Increased carnitine palmitoyltransferase 1A (CPT1A), identified as a succinyltransferase for PD-L1, boosted anti-tumor activity. Preclinically, bezafibrate, a hyperlipidemia drug, upregulated CPT1A and synergized with CTLA-4 monoclonal antibody to inhibit tumor growth. Clinically, higher PD-L1 and lower CPT1A levels in tumors correlated with better anti-PD-1 therapy responses, suggesting potential biomarkers for prediction of treatment efficacy. Succinylation of PD-L1 by carnitine palmitoyltransferase 1A (CPT1A) in melanoma leads to its degradation and enhanced T cell-dependent killing in vitro. Increasing CPT1A levels synergizes with anti-CTLA-4 treatment to suppress tumor growth in a mouse melanoma model.},
  archive      = {J_NG},
  author       = {Liang, Long and Kuang, Xinwei and He, Yi and Zhu, Lin and Lau, Poyee and Li, Xin and Luo, Dingan and Gong, Lan and Zhou, Wenbin and Zhang, Fanglin and Liang, Xiaowei and Li, Zhuofeng and Hu, Bin and Liu, Dandan and Ding, Tao and Li, Hui and Zhao, Shuang and Su, Juan and Hung, Mien-Chie and Liu, Jing and Liu, Hong and Chen, Xiang},
  doi          = {10.1038/s41588-025-02077-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {680-693},
  shortjournal = {Nat. Genet.},
  title        = {Alterations in PD-l1 succinylation shape anti-tumor immune responses in melanoma},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Active repression of cell fate plasticity by PROX1 safeguards hepatocyte identity and prevents liver tumorigenesis. <em>NG</em>, <em>57</em>(3), 668-679. (<a href='https://doi.org/10.1038/s41588-025-02081-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell fate plasticity enables development, yet unlocked plasticity is a cancer hallmark. While transcription master regulators induce lineage-specific genes to restrict plasticity, it remains unclear whether plasticity is actively suppressed by lineage-specific repressors. Here we computationally predict so-called safeguard repressors for 18 cell types that block phenotypic plasticity lifelong. We validated hepatocyte-specific candidates using reprogramming, revealing that prospero homeobox protein 1 (PROX1) enhanced hepatocyte identity by direct repression of alternative fate master regulators. In mice, Prox1 was required for efficient hepatocyte regeneration after injury and was sufficient to prevent liver tumorigenesis. In line with patient data, Prox1 depletion caused hepatocyte fate loss in vivo and enabled the transition of hepatocellular carcinoma to cholangiocarcinoma. Conversely, overexpression promoted cholangiocarcinoma to hepatocellular carcinoma transdifferentiation. Our findings provide evidence for PROX1 as a hepatocyte-specific safeguard and support a model where cell-type-specific repressors actively suppress plasticity throughout life to safeguard lineage identity and thus prevent disease. This study identifies candidate safeguard repressor transcription factors that repress alternate lineages in mature cell types and provides functional evidence that Prox1 performs such a function in hepatocytes during reprogramming, regeneration and in cancer.},
  archive      = {J_NG},
  author       = {Lim, Bryce and Kamal, Aryan and Gomez Ramos, Borja and Adrian Segarra, Juan M. and Ibarra, Ignacio L. and Dignas, Lennart and Kindinger, Tim and Volz, Kai and Rahbari, Mohammad and Rahbari, Nuh and Poisel, Eric and Kafetzopoulou, Kanela and Böse, Lio and Breinig, Marco and Heide, Danijela and Gallage, Suchira and Barragan Avila, Jose E. and Wiethoff, Hendrik and Berest, Ivan and Schnabellehner, Sarah and Schneider, Martin and Becker, Jonas and Helm, Dominic and Grimm, Dirk and Mäkinen, Taija and Tschaharganeh, Darjus F. and Heikenwalder, Mathias and Zaugg, Judith B. and Mall, Moritz},
  doi          = {10.1038/s41588-025-02081-w},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {668-679},
  shortjournal = {Nat. Genet.},
  title        = {Active repression of cell fate plasticity by PROX1 safeguards hepatocyte identity and prevents liver tumorigenesis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Analysis of multi-condition single-cell data with latent embedding multivariate regression. <em>NG</em>, <em>57</em>(3), 659-667. (<a href='https://doi.org/10.1038/s41588-024-01996-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying gene expression differences in heterogeneous tissues across conditions is a fundamental biological task, enabled by multi-condition single-cell RNA sequencing (RNA-seq). Current data analysis approaches divide the constituent cells into clusters meant to represent cell types, but such discrete categorization tends to be an unsatisfactory model of the underlying biology. Here, we introduce latent embedding multivariate regression (LEMUR), a model that operates without, or before, commitment to discrete categorization. LEMUR (1) integrates data from different conditions, (2) predicts each cell’s gene expression changes as a function of the conditions and its position in latent space and (3) for each gene, identifies a compact neighborhood of cells with consistent differential expression. We apply LEMUR to cancer, zebrafish development and spatial gradients in Alzheimer’s disease, demonstrating its broad applicability. Latent embedding multivariate regression models multi-condition single-cell RNA-seq using a continuous latent space, enabling data integration, per-cell gene expression prediction and clustering-free differential expression analysis.},
  archive      = {J_NG},
  author       = {Ahlmann-Eltze, Constantin and Huber, Wolfgang},
  doi          = {10.1038/s41588-024-01996-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {659-667},
  shortjournal = {Nat. Genet.},
  title        = {Analysis of multi-condition single-cell data with latent embedding multivariate regression},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis. <em>NG</em>, <em>57</em>(3), 647-658. (<a href='https://doi.org/10.1038/s41588-025-02080-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale changes in the structure and cellular makeup of the distal lung are a hallmark of pulmonary fibrosis (PF), but the spatial contexts that contribute to disease pathogenesis have remained uncertain. Using image-based spatial transcriptomics, we analyzed the gene expression of 1.6 million cells from 35 unique lungs. Through complementary cell-based and innovative cell-agnostic analyses, we characterized the localization of PF-emergent cell types, established the cellular and molecular basis of classical PF histopathologic features and identified a diversity of distinct molecularly defined spatial niches in control and PF lungs. Using machine learning and trajectory analysis to segment and rank airspaces on a gradient of remodeling severity, we identified compositional and molecular changes associated with progressive distal lung pathology, beginning with alveolar epithelial dysregulation and culminating with changes in macrophage polarization. Together, these results provide a unique, spatially resolved view of PF and establish methods that could be applied to other spatial transcriptomic studies. Xenium spatial transcriptomic profiling of pulmonary fibrosis characterizes cell composition dynamics and histopathological features associated with the disease.},
  archive      = {J_NG},
  author       = {Vannan, Annika and Lyu, Ruqian and Williams, Arianna L. and Negretti, Nicholas M. and Mee, Evan D. and Hirsh, Joseph and Hirsh, Samuel and Hadad, Niran and Nichols, David S. and Calvi, Carla L. and Taylor, Chase J. and Polosukhin, Vasiliy. V. and Serezani, Ana P. M. and McCall, A. Scott and Gokey, Jason J. and Shim, Heejung and Ware, Lorraine B. and Bacchetta, Matthew J. and Shaver, Ciara M. and Blackwell, Timothy S. and Walia, Rajat and Sucre, Jennifer M. S. and Kropski, Jonathan A. and McCarthy, Davis J. and Banovich, Nicholas E.},
  doi          = {10.1038/s41588-025-02080-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {647-658},
  shortjournal = {Nat. Genet.},
  title        = {Spatial transcriptomics identifies molecular niche dysregulation associated with distal lung remodeling in pulmonary fibrosis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Plasma proteome variation and its genetic determinants in children and adolescents. <em>NG</em>, <em>57</em>(3), 635-646. (<a href='https://doi.org/10.1038/s41588-025-02089-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our current understanding of the determinants of plasma proteome variation during pediatric development remains incomplete. Here, we show that genetic variants, age, sex and body mass index significantly influence this variation. Using a streamlined and highly quantitative mass spectrometry-based proteomics workflow, we analyzed plasma from 2,147 children and adolescents, identifying 1,216 proteins after quality control. Notably, the levels of 70% of these were associated with at least one of the aforementioned factors, with protein levels also being predictive. Quantitative trait loci (QTLs) regulated at least one-third of the proteins; between a few percent and up to 30-fold. Together with excellent replication in an additional 1,000 children and 558 adults, this reveals substantial genetic effects on plasma protein levels, persisting from childhood into adulthood. Through Mendelian randomization and colocalization analyses, we identified 41 causal genes for 33 cardiometabolic traits, emphasizing the value of protein QTLs in drug target identification and disease understanding. This mass spectrometry-based proteomic study profiles the plasma proteome in 2,147 children and adolescents and reveals its association with age, sex, puberty, body mass index and genetics.},
  archive      = {J_NG},
  author       = {Niu, Lili and Stinson, Sara Elizabeth and Holm, Louise Aas and Lund, Morten Asp Vonsild and Fonvig, Cilius Esmann and Cobuccio, Leonardo and Meisner, Jonas and Juel, Helene Bæk and Fadista, Joao and Thiele, Maja and Krag, Aleksander and Holm, Jens-Christian and Rasmussen, Simon and Hansen, Torben and Mann, Matthias},
  doi          = {10.1038/s41588-025-02089-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {635-646},
  shortjournal = {Nat. Genet.},
  title        = {Plasma proteome variation and its genetic determinants in children and adolescents},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Whole-genome sequencing analysis identifies rare, large-effect noncoding variants and regulatory regions associated with circulating protein levels. <em>NG</em>, <em>57</em>(3), 626-634. (<a href='https://doi.org/10.1038/s41588-025-02095-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The contribution of rare noncoding genetic variation to common phenotypes is largely unknown, as a result of a historical lack of population-scale whole-genome sequencing data and the difficulty of categorizing noncoding variants into functionally similar groups. To begin addressing these challenges, we performed a cis association analysis using whole-genome sequencing data, consisting of 1.1 billion variants, 123 million noncoding aggregate-based tests and 2,907 circulating protein levels in ~50,000 UK Biobank participants. We identified 604 independent rare noncoding single-variant associations with circulating protein levels. Unlike protein-coding variation, rare noncoding genetic variation was almost as likely to increase or decrease protein levels. Rare noncoding aggregate testing identified 357 conditionally independent associated regions. Of these, 74 (21%) were not detectable by single-variant testing alone. Our findings have important implications for the identification, and role, of rare noncoding genetic variation associated with common human phenotypes, including the importance of testing aggregates of noncoding variants. Rare variant association analysis of plasma proteins using whole-genome sequencing data in 54,306 individuals in the UK Biobank demonstrates that combining both single-variant and aggregate testing will be needed for noncoding variant analysis.},
  archive      = {J_NG},
  author       = {Hawkes, Gareth and Chundru, Kartik and Jackson, Leigh and Patel, Kashyap A. and Murray, Anna and Wood, Andrew R. and Wright, Caroline F. and Weedon, Michael N. and Frayling, Timothy M. and Beaumont, Robin N.},
  doi          = {10.1038/s41588-025-02095-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {626-634},
  shortjournal = {Nat. Genet.},
  title        = {Whole-genome sequencing analysis identifies rare, large-effect noncoding variants and regulatory regions associated with circulating protein levels},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The contribution of genetic determinants of blood gene expression and splicing to molecular phenotypes and health outcomes. <em>NG</em>, <em>57</em>(3), 616-625. (<a href='https://doi.org/10.1038/s41588-025-02096-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The biological mechanisms through which most nonprotein-coding genetic variants affect disease risk are unknown. To investigate gene-regulatory mechanisms, we mapped blood gene expression and splicing quantitative trait loci (QTLs) through bulk RNA sequencing in 4,732 participants and integrated protein, metabolite and lipid data from the same individuals. We identified cis-QTLs for the expression of 17,233 genes and 29,514 splicing events (in 6,853 genes). Colocalization analyses revealed 3,430 proteomic and metabolomic traits with a shared association signal with either gene expression or splicing. We quantified the relative contribution of the genetic effects at loci with shared etiology, observing 222 molecular phenotypes significantly mediated by gene expression or splicing. We uncovered gene-regulatory mechanisms at disease loci with therapeutic implications, such as WARS1 in hypertension, IL7R in dermatitis and IFNAR2 in COVID-19. Our study provides an open-access resource on the shared genetic etiology across transcriptional phenotypes, molecular traits and health outcomes in humans ( https://IntervalRNA.org.uk ). Integrative analyses of whole-blood gene expression and splicing QTLs with protein, metabolite and lipid QTLs from the same individuals shed light on the shared genetic architecture of molecular traits and health outcomes.},
  archive      = {J_NG},
  author       = {Tokolyi, Alex and Persyn, Elodie and Nath, Artika P. and Burnham, Katie L. and Marten, Jonathan and Vanderstichele, Thomas and Tardaguila, Manuel and Stacey, David and Farr, Ben and Iyer, Vivek and Jiang, Xilin and Lambert, Samuel A. and Noell, Guillaume and Quail, Michael A. and Rajan, Diana and Ritchie, Scott C. and Sun, Benjamin B. and Thurston, Scott A. J. and Xu, Yu and Whelan, Christopher D. and Runz, Heiko and Petrovski, Slavé and Gaffney, Daniel J. and Roberts, David J. and Di Angelantonio, Emanuele and Peters, James E. and Soranzo, Nicole and Danesh, John and Butterworth, Adam S. and Inouye, Michael and Davenport, Emma E. and Paul, Dirk S.},
  doi          = {10.1038/s41588-025-02096-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {616-625},
  shortjournal = {Nat. Genet.},
  title        = {The contribution of genetic determinants of blood gene expression and splicing to molecular phenotypes and health outcomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Long-read RNA sequencing atlas of human microglia isoforms elucidates disease-associated genetic regulation of splicing. <em>NG</em>, <em>57</em>(3), 604-615. (<a href='https://doi.org/10.1038/s41588-025-02099-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microglia, the innate immune cells of the central nervous system, have been genetically implicated in multiple neurodegenerative diseases. Mapping the genetics of gene expression in human microglia has identified several loci associated with disease-associated genetic variants in microglia-specific regulatory elements. However, identifying genetic effects on splicing is challenging because of the use of short sequencing reads. Here, we present the isoform-centric microglia genomic atlas (isoMiGA), which leverages long-read RNA sequencing to identify 35,879 novel microglia isoforms. We show that these isoforms are involved in stimulation response and brain region specificity. We then quantified the expression of both known and novel isoforms in a multi-ancestry meta-analysis of 555 human microglia short-read RNA sequencing samples from 391 donors, and found associations with genetic risk loci in Alzheimer’s and Parkinson’s disease. We nominate several loci that may act through complex changes in isoform and splice-site usage. This isoform-centric microglia genomic atlas includes 35,879 novel human microglia isoforms identified by long-read RNA sequencing. A multi-ancestry quantitative trait locus meta-analysis of known and novel isoforms in 555 samples from 391 donors finds associations with genetic risk loci in Alzheimer’s and Parkinson’s diseases.},
  archive      = {J_NG},
  author       = {Humphrey, Jack and Brophy, Erica and Kosoy, Roman and Zeng, Biao and Coccia, Elena and Mattei, Daniele and Ravi, Ashvin and Naito, Tatsuhiko and Efthymiou, Anastasia G. and Navarro, Elisa and De Sanctis, Claudia and Flores-Almazan, Victoria and Muller, Benjamin Z. and Snijders, Gijsje J. L. J. and Allan, Amanda and Münch, Alexandra and Kitata, Reta Birhanu and Kleopoulos, Steven P. and Argyriou, Stathis and Malakates, Periklis and Psychogyiou, Konstantina and Shao, Zhiping and Francoeur, Nancy and Tsai, Chia-Feng and Gritsenko, Marina A. and Monroe, Matthew E. and Paurus, Vanessa L. and Weitz, Karl K. and Shi, Tujin and Sebra, Robert and Liu, Tao and de Witte, Lot D. and Goate, Alison M. and Bennett, David A. and Haroutunian, Vahram and Hoffman, Gabriel E. and Fullard, John F. and Roussos, Panos and Raj, Towfique},
  doi          = {10.1038/s41588-025-02099-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {604-615},
  shortjournal = {Nat. Genet.},
  title        = {Long-read RNA sequencing atlas of human microglia isoforms elucidates disease-associated genetic regulation of splicing},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiomic single-cell profiling identifies critical regulators of postnatal brain. <em>NG</em>, <em>57</em>(3), 591-603. (<a href='https://doi.org/10.1038/s41588-025-02083-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human brain development spans from embryogenesis to adulthood, with dynamic gene expression controlled by cell-type-specific cis-regulatory element activity and three-dimensional genome organization. To advance our understanding of postnatal brain development, we simultaneously profiled gene expression and chromatin accessibility in 101,924 single nuclei from four brain regions across ten donors, covering five key postnatal stages from infancy to late adulthood. Using this dataset and chromosome conformation capture data, we constructed enhancer-based gene regulatory networks to identify cell-type-specific regulators of brain development and interpret genome-wide association study loci for ten main brain disorders. Our analysis connected 2,318 cell-specific loci to 1,149 unique genes, representing 41% of loci linked to the investigated traits, and highlighted 55 genes influencing several disease phenotypes. Pseudotime analysis revealed distinct stages of postnatal oligodendrogenesis and their regulatory programs. These findings provide a comprehensive dataset of cell-type-specific gene regulation at critical timepoints in postnatal brain development. Simultaneous profiling of RNA and chromatin accessibility in single nuclei isolated from human postnatal brain regions from infancy to late adulthood identifies key cellular regulators and nominates target genes and mechanisms for brain-related diseases and disorders.},
  archive      = {J_NG},
  author       = {Clarence, Tereza and Bendl, Jaroslav and Cao, Xuan and Wang, Xinyi and Zheng, Shiwei and Hoffman, Gabriel E. and Kozlenkov, Alexey and Hong, Aram and Iskhakova, Marina and Jaiswal, Manoj K. and Murphy, Sarah and Yu, Alexander and Haroutunian, Vahram and Dracheva, Stella and Akbarian, Schahram and Fullard, John F. and Yuan, Guo-Cheng and Lee, Donghoon and Roussos, Panos},
  doi          = {10.1038/s41588-025-02083-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {591-603},
  shortjournal = {Nat. Genet.},
  title        = {Multiomic single-cell profiling identifies critical regulators of postnatal brain},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comparison of the multivariate genetic architecture of eight major psychiatric disorders across sex. <em>NG</em>, <em>57</em>(3), 583-590. (<a href='https://doi.org/10.1038/s41588-025-02093-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Differences in the patterning of genetic sharing between groups of individuals may arise from biological pathways, social mechanisms, phenotyping and ascertainment. We expand genomic structural equation modeling to allow for testing genomic structural invariance (GSI), that is, the formal comparison of multivariate genetic architecture across groups. We apply GSI to compare the autosomal multivariate genetic architecture of eight psychiatric disorders spanning three factors (psychotic, neurodevelopmental and internalizing) between cisgender males and females. We find that the genetic factor structure is largely similar across sex, permitting meaningful comparisons of associations at the level of the factors. However, in females, problematic alcohol use and posttraumatic stress disorder loaded more strongly on the internalizing factor, while the neurodevelopmental disorder factor exhibited weaker genetic correlations with the other factors. Four phenotypes (educational attainment, insomnia, smoking and deprivation) showed significant, albeit small, sex-differentiated associations with the psychotic factor. As genome-wide association study samples grow and diversify, GSI will become increasingly valuable for comparing multivariate genetic architecture across groups. Genomic structural invariance, an extension of genomic structural equation modeling, identifies similarities and differences between males and females in the genetic architectures of eight psychiatric disorders spanning three broad factors.},
  archive      = {J_NG},
  author       = {Schwaba, Ted and Mallard, Travis T. and Maihofer, Adam X. and Rhemtulla, Mijke and Lee, Phil H. and Smoller, Jordan W. and Davis, Lea K. and Nivard, Michel G. and Grotzinger, Andrew D. and Tucker-Drob, Elliot M.},
  doi          = {10.1038/s41588-025-02093-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {583-590},
  shortjournal = {Nat. Genet.},
  title        = {Comparison of the multivariate genetic architecture of eight major psychiatric disorders across sex},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comparative analysis of the mexico city prospective study and the UK biobank identifies ancestry-specific effects on clonal hematopoiesis. <em>NG</em>, <em>57</em>(3), 572-582. (<a href='https://doi.org/10.1038/s41588-025-02085-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The impact of genetic ancestry on the development of clonal hematopoiesis (CH) remains largely unexplored. Here, we compared CH in 136,401 participants from the Mexico City Prospective Study (MCPS) to 416,118 individuals from the UK Biobank (UKB) and observed CH to be significantly less common in MCPS compared to UKB (adjusted odds ratio = 0.59, 95% confidence interval (CI) = [0.57, 0.61], P = 7.31 × 10−185). Among MCPS participants, CH frequency was positively correlated with the percentage of European ancestry (adjusted beta = 0.84, 95% CI = [0.66, 1.03], P = 7.35 × 10−19). Genome-wide and exome-wide association analyses in MCPS identified ancestry-specific variants in the TCL1B locus with opposing effects on DNMT3A-CH versus non-DNMT3A-CH. Meta-analysis of MCPS and UKB identified five novel loci associated with CH, including polymorphisms at PARP11/CCND2, MEIS1 and MYCN. Our CH study, the largest in a non-European population to date, demonstrates the power of cross-ancestry comparisons to derive novel insights into CH pathogenesis. This paper compares the frequency and genetic basis of clonal hematopoiesis in 136,401 admixed American participants from the Mexico City Prospective Cohort with 416,118 largely European ancestry individuals from the UK Biobank cohort.},
  archive      = {J_NG},
  author       = {Wen, Sean and Kuri-Morales, Pablo and Hu, Fengyuan and Nag, Abhishek and Tachmazidou, Ioanna and Deevi, Sri V. V. and Taiy, Haeyam and Smith, Katherine R. and Loesch, Douglas P. and Burren, Oliver S. and Dhindsa, Ryan S. and Wasilewski, Sebastian and Alegre-Díaz, Jesus and Berumen, Jaime and Emberson, Jonathan and Torres, Jason M. and Collins, Rory and Carss, Keren and Wang, Quanli and Petrovski, Slavé and Tapia-Conyer, Roberto and Fabre, Margarete A. and Harper, Andrew R. and Vassiliou, George S. and Mitchell, Jonathan},
  doi          = {10.1038/s41588-025-02085-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {572-582},
  shortjournal = {Nat. Genet.},
  title        = {Comparative analysis of the mexico city prospective study and the UK biobank identifies ancestry-specific effects on clonal hematopoiesis},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evaluation of polygenic scores for hypertrophic cardiomyopathy in the general population and across clinical settings. <em>NG</em>, <em>57</em>(3), 563-571. (<a href='https://doi.org/10.1038/s41588-025-02094-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hypertrophic cardiomyopathy (HCM) is an important cause of morbidity and mortality, with pathogenic variants found in about a third of cases. Large-scale genome-wide association studies (GWAS) demonstrate that common genetic variation contributes to HCM risk. Here we derive polygenic scores (PGS) from HCM GWAS and genetically correlated traits and test their performance in the UK Biobank, 100,000 Genomes Project, and clinical cohorts. We show that higher PGS significantly increases the risk of HCM in the general population, particularly among pathogenic variant carriers, where HCM penetrance differs 10-fold between those in the highest and lowest PGS quintiles. Among relatives of HCM probands, PGS stratifies risks of developing HCM and adverse outcomes. Finally, among HCM cases, PGS strongly predicts the risk of adverse outcomes and death. These findings support the broad utility of PGS across clinical settings, enabling tailored screening and surveillance and stratification of risk of adverse outcomes. Polygenic scores derived from genome-wide association analyses of hypertrophic cardiomyopathy and genetically correlated left ventricular traits effectively stratify risk of disease and adverse outcomes in the general population and across clinical cohorts.},
  archive      = {J_NG},
  author       = {Zheng, Sean L. and Jurgens, Sean J. and McGurk, Kathryn A. and Xu, Xiao and Grace, Chris and Theotokis, Pantazis I. and Buchan, Rachel J. and Francis, Catherine and de Marvao, Antonio and Curran, Lara and Bai, Wenjia and Pua, Chee Jian and Tang, Hak Chiaw and Jorda, Paloma and van Slegtenhorst, Marjon A. and Verhagen, Judith M. A. and Harper, Andrew R. and Ormondroyd, Elizabeth and Chin, Calvin W. L. and Pantazis, Antonis and Baksi, John and Halliday, Brian P. and Matthews, Paul and Pinto, Yigal M. and Walsh, Roddy and Amin, Ahmad S. and Wilde, Arthur A. M. and Cook, Stuart A. and Prasad, Sanjay K. and Barton, Paul J. R. and O’Regan, Declan P. and Lumbers, R. T. and Goel, Anuj and Tadros, Rafik and Michels, Michelle and Watkins, Hugh and Bezzina, Connie R. and Ware, James S.},
  doi          = {10.1038/s41588-025-02094-5},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {563-571},
  shortjournal = {Nat. Genet.},
  title        = {Evaluation of polygenic scores for hypertrophic cardiomyopathy in the general population and across clinical settings},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sequencing in over 50,000 cases identifies coding and structural variation underlying atrial fibrillation risk. <em>NG</em>, <em>57</em>(3), 548-562. (<a href='https://doi.org/10.1038/s41588-025-02074-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Atrial fibrillation (AF) is a prevalent and morbid abnormality of the heart rhythm with a strong genetic component. Here, we meta-analyzed genome and exome sequencing data from 36 studies that included 52,416 AF cases and 277,762 controls. In burden tests of rare coding variation, we identified novel associations between AF and the genes MYBPC3, LMNA, PKP2, FAM189A2 and KDM5B. We further identified associations between AF and rare structural variants owing to deletions in CTNNA3 and duplications of GATA4. We broadly replicated our findings in independent samples from MyCode, deCODE and UK Biobank. Finally, we found that CRISPR knockout of KDM5B in stem-cell-derived atrial cardiomyocytes led to a shortening of the action potential duration and widespread transcriptomic dysregulation of genes relevant to atrial homeostasis and conduction. Our results highlight the contribution of rare coding and structural variants to AF, including genetic links between AF and cardiomyopathies, and expand our understanding of the rare variant architecture for this common arrhythmia. Analyses of genome and exome sequencing data identify rare coding and structural variants associated with atrial fibrillation and highlight genetic links between atrial fibrillation and cardiomyopathies.},
  archive      = {J_NG},
  author       = {Choi, Seung Hoan and Jurgens, Sean J. and Xiao, Ling and Hill, Matthew C. and Haggerty, Christopher M. and Sveinbjörnsson, Garðar and Morrill, Valerie N. and Marston, Nicholas A. and Weng, Lu-Chen and Pirruccello, James P. and Arnar, David O. and Gudbjartsson, Daniel Fannar and Mantineo, Helene and von Falkenhausen, Aenne S. and Natale, Andrea and Tveit, Arnljot and Geelhoed, Bastiaan and Roselli, Carolina and Van Wagoner, David R. and Darbar, Dawood and Haase, Doreen and Soliman, Elsayed Z. and Davogustto, Giovanni E. and Jun, Goo and Calkins, Hugh and Anderson, Jeffrey L. and Brody, Jennifer A. and Halford, Jennifer L. and Barnard, John and Hokanson, John E. and Smith, Jonathan D. and Bis, Joshua C. and Young, Kendra and Johnson, Linda S. B. and Risch, Lorenz and Gula, Lorne J. and Kwee, Lydia Coulter and Chaffin, Mark D. and Kühne, Michael and Preuss, Michael and Gupta, Namrata and Nafissi, Navid A. and Smith, Nicholas L. and Nilsson, Peter M. and van der Harst, Pim and Wells, Quinn S. and Judy, Renae L. and Schnabel, Renate B. and Johnson, Renee and Smit, Roelof A. J. and Gabriel, Stacey and Knight, Stacey and Furukawa, Tetsushi and Blackwell, Thomas W. and Nauffal, Victor and Wang, Xin and Min, Yuan-I and Yoneda, Zachary T. and Laksman, Zachary W. M. and Bezzina, Connie R. and Alonso, Alvaro and Psaty, Bruce M. and Albert, Christine M. and Arking, Dan E. and Roden, Dan M. and Chasman, Daniel I. and Rader, Daniel J. and Conen, David and McManus, David D. and Fatkin, Diane and Benjamin, Emelia J. and Boerwinkle, Eric and Marcus, Gregory M. and Christophersen, Ingrid E. and Smith, J. Gustav and Roberts, Jason D. and Raffield, Laura M. and Shoemaker, M. Benjamin and Cho, Michael H. and Cutler, Michael J. and Rienstra, Michiel and Chung, Mina K. and S. Olesen, Morten and Sinner, Moritz F. and Sotoodehnia, Nona and Kirchhof, Paulus and Loos, Ruth J. F. and Nazarian, Saman and Mohanty, Sanghamitra and Damrauer, Scott M. and Kaab, Stefan and Heckbert, Susan R. and Redline, Susan and Shah, Svati H. and Tanaka, Toshihiro and Ebana, Yusuke and Holm, Hilma and Stefansson, Kari and Ruff, Christian T. and Sabatine, Marc S. and Lunetta, Kathryn L. and Lubitz, Steven A. and Ellinor, Patrick T.},
  doi          = {10.1038/s41588-025-02074-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {548-562},
  shortjournal = {Nat. Genet.},
  title        = {Sequencing in over 50,000 cases identifies coding and structural variation underlying atrial fibrillation risk},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Meta-analysis of genome-wide associations and polygenic risk prediction for atrial fibrillation in more than 180,000 cases. <em>NG</em>, <em>57</em>(3), 539-547. (<a href='https://doi.org/10.1038/s41588-024-02072-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Atrial fibrillation (AF) is the most common heart rhythm abnormality and is a leading cause of heart failure and stroke. This large-scale meta-analysis of genome-wide association studies increased the power to detect single-nucleotide variant associations and found more than 350 AF-associated genetic loci. We identified candidate genes related to muscle contractility, cardiac muscle development and cell–cell communication at 139 loci. Furthermore, we assayed chromatin accessibility using assay for transposase-accessible chromatin with sequencing and histone H3 lysine 4 trimethylation in stem cell-derived atrial cardiomyocytes. We observed a marked increase in chromatin accessibility for our sentinel variants and prioritized genes in atrial cardiomyocytes. Finally, a polygenic risk score (PRS) based on our updated effect estimates improved AF risk prediction compared to the CHARGE-AF clinical risk score and a previously reported PRS for AF. The doubling of known risk loci will facilitate a greater understanding of the pathways underlying AF. Genome-wide association meta-analyses identify more than 350 loci associated with atrial fibrillation. A polygenic score derived from these results improves atrial fibrillation risk prediction compared to published clinical and polygenic scores.},
  archive      = {J_NG},
  author       = {Roselli, Carolina and Surakka, Ida and Olesen, Morten S. and Sveinbjornsson, Gardar and Marston, Nicholas A. and Choi, Seung Hoan and Holm, Hilma and Chaffin, Mark and Gudbjartsson, Daniel and Hill, Matthew C. and Aegisdottir, Hildur and Albert, Christine M. and Alonso, Alvaro and Anderson, Christopher D. and Arking, Dan E. and Arnar, David O. and Barnard, John and Benjamin, Emelia J. and Braunwald, Eugene and Brumpton, Ben and Campbell, Archie and Chami, Nathalie and Chasman, Daniel I. and Cho, Kelly and Choi, Eue-Keun and Christophersen, Ingrid E. and Chung, Mina K. and Conen, David and Crijns, Harry J. and Cutler, Michael J. and Czuba, Tomasz and Damrauer, Scott M. and Dichgans, Martin and Dörr, Marcus and Dudink, Elton and Duong, ThuyVy and Erikstrup, Christian and Esko, Tõnu and Fatkin, Diane and Faul, Jessica D. and Ferreira, Manuel and Freitag, Daniel F. and Ganesh, Santhi K. and Gaziano, J. Michael and Geelhoed, Bastiaan and Ghouse, Jonas and Gieger, Christian and Giulianini, Franco and Graham, Sarah E. and Gudnason, Vilmundur and Guo, Xiuqing and Haggerty, Christopher and Hayward, Caroline and Heckbert, Susan R. and Hveem, Kristian and Ito, Kaoru and Johnson, Renee and Jukema, J. Wouter and Jurgens, Sean J. and Kääb, Stefan and Kane, John P. and Kany, Shinwan and Kardia, Sharon L. R. and Kavousi, Maryam and Khurshid, Shaan and Kamanu, Frederick K. and Kirchhof, Paulus and Kleber, Marcus E. and Knight, Stacey and Komuro, Issei and Krieger, Jose E. and Launer, Lenore J. and Li, Dadong and Lin, Honghuang and Lin, Henry J. and Loos, Ruth J. F. and Lotta, Luca and Lubitz, Steven A. and Lunetta, Kathryn L. and Macfarlane, Peter W. and Magnusson, Patrik K. E. and Malik, Rainer and Mantineo, Helene and Marcus, Gregory M. and März, Winfried and McManus, David D. and Melander, Olle and Melloni, Giorgio E. M. and Meyre, Pascal B. and Miyazawa, Kazuo and Mohanty, Sanghamitra and Monfort, Laia M. and Müller-Nurasyid, Martina and Nafissi, Navid A. and Natale, Andrea and Nazarian, Saman and Ostrowski, Sisse R. and Pak, Hui-Nam and Pang, Shichao and Pedersen, Ole B. and Pedersen, Nancy L. and Pereira, Alexandre C. and Pirruccello, James P. and Preuss, Michael and Psaty, Bruce M. and Pullinger, Clive R. and Rader, Daniel J. and Rämö, Joel T. and Ridker, Paul M. and Rienstra, Michiel and Risch, Lorenz and Roden, Dan M. and Rotter, Jerome I. and Sabatine, Marc S. and Schunkert, Heribert and Shah, Svati H. and Shim, Jaemin and Shoemaker, M. Benjamin and Simonson, Bridget and Sinner, Moritz F. and Smit, Roelof A. J. and Smith, Jennifer A. and Smith, Nicholas L. and Smith, J. Gustav and Soliman, Elsayed Z. and Sørensen, Erik and Sotoodehnia, Nona and Strbian, Daniel and Stricker, Bruno H. and Teder-Laving, Maris and Sun, Yan V. and Thériault, Sébastien and Thorolfsdottir, Rosa B. and Thorsteinsdottir, Unnur and Tveit, Arnljot and van der Harst, Pim and van Meurs, Joyce and Wang, Biqi and Weiss, Stefan and Wells, Quinn S. and Weng, Lu-Chen and Wilson, Peter W. and Xiao, Ling and Yang, Pil-Sung and Yao, Jie and Yoneda, Zachary T. and Zeller, Tanja and Zeng, Lingyao and Zhao, Wei and Zhou, Xiang and Zöllner, Sebastian and Ruff, Christian T. and Bundgaard, Henning and Willer, Cristen and Stefansson, Kari and Ellinor, Patrick T.},
  doi          = {10.1038/s41588-024-02072-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {539-547},
  shortjournal = {Nat. Genet.},
  title        = {Meta-analysis of genome-wide associations and polygenic risk prediction for atrial fibrillation in more than 180,000 cases},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Large-scale genome-wide association analyses identify novel genetic loci and mechanisms in hypertrophic cardiomyopathy. <em>NG</em>, <em>57</em>(3), 530-538. (<a href='https://doi.org/10.1038/s41588-025-02087-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hypertrophic cardiomyopathy (HCM) is an important cause of morbidity and mortality with both monogenic and polygenic components. Here, we report results from a large genome-wide association study and multitrait analysis including 5,900 HCM cases, 68,359 controls and 36,083 UK Biobank participants with cardiac magnetic resonance imaging. We identified 70 loci (50 novel) associated with HCM and 62 loci (20 novel) associated with relevant left ventricular traits. Among the prioritized genes in the HCM loci, we identify a novel HCM disease gene, SVIL, which encodes the actin-binding protein supervillin, showing that rare truncating SVIL variants confer a roughly tenfold increased risk of HCM. Mendelian randomization analyses support a causal role of increased left ventricular contractility in both obstructive and nonobstructive forms of HCM, suggesting common disease mechanisms and anticipating shared response to therapy. Taken together, these findings increase our understanding of the genetic basis of HCM, with potential implications for disease management. Genome-wide and multitrait analyses identify novel loci associated with hypertrophic cardiomyopathy and relevant left ventricular traits. Gene-level burden analyses show that rare truncating SVIL variants are associated with high risk of hypertrophic cardiomyopathy.},
  archive      = {J_NG},
  author       = {Tadros, Rafik and Zheng, Sean L. and Grace, Christopher and Jordà, Paloma and Francis, Catherine and West, Dominique M. and Jurgens, Sean J. and Thomson, Kate L. and Harper, Andrew R. and Ormondroyd, Elizabeth and Xu, Xiao and Theotokis, Pantazis I. and Buchan, Rachel J. and McGurk, Kathryn A. and Mazzarotto, Francesco and Boschi, Beatrice and Pelo, Elisabetta and Lee, Michael and Noseda, Michela and Varnava, Amanda and Vermeer, Alexa M. C. and Walsh, Roddy and Amin, Ahmad S. and van Slegtenhorst, Marjon A. and Roslin, Nicole M. and Strug, Lisa J. and Salvi, Erika and Lanzani, Chiara and de Marvao, Antonio and Roberts, Jason D. and Tremblay-Gravel, Maxime and Giraldeau, Genevieve and Cadrin-Tourigny, Julia and L’Allier, Philippe L. and Garceau, Patrick and Talajic, Mario and Gagliano Taliun, Sarah A. and Pinto, Yigal M. and Rakowski, Harry and Pantazis, Antonis and Bai, Wenjia and Baksi, John and Halliday, Brian P. and Prasad, Sanjay K. and Barton, Paul J. R. and O’Regan, Declan P. and Cook, Stuart A. and de Boer, Rudolf A. and Christiaans, Imke and Michels, Michelle and Kramer, Christopher M. and Ho, Carolyn Y. and Neubauer, Stefan and Matthews, Paul M. and Wilde, Arthur A. M. and Tardif, Jean-Claude and Olivotto, Iacopo and Adler, Arnon and Goel, Anuj and Ware, James S. and Bezzina, Connie R. and Watkins, Hugh},
  doi          = {10.1038/s41588-025-02087-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {530-538},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale genome-wide association analyses identify novel genetic loci and mechanisms in hypertrophic cardiomyopathy},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The transcriptomic architecture of common cancers reflects synthetic lethal interactions. <em>NG</em>, <em>57</em>(3), 522-529. (<a href='https://doi.org/10.1038/s41588-025-02108-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To maintain cell fitness, deleterious genetic alterations are buffered by compensatory changes in additional genes. In cancer, buffering processes could be targeted by synthetic lethality. However, despite the large-scale identification of synthetic lethal effects in preclinical models, evidence that these operate clinically is limited. This impedes the application of synthetic lethal approaches. By integrating molecular profiling data from &gt;9,000 cancers with synthetic lethal screens, we show that transcriptomic buffering of tumor suppressor gene (TSG) loss by hyperexpression of synthetic lethal partners is a common phenomenon, extending to multiple TSGs and histotypes. Transcriptomic buffering is also notable in cancers that phenocopy TSG loss, such as BRCAness cancers, where expression of BRCA1/2 synthetic lethal genes correlates with clinical outcome. Synthetic lethal genes that exhibit transcriptomic buffering also represent more robust synthetic lethal effects. These observations have implications for understanding how tumor cells tolerate TSG loss, in part explain transcriptomic architectures in cancer and provide insight into target selection. Tumor cells upregulate compensatory buffering genes following tumor suppressor loss. These genes may represent new synthetic lethal partners that could be harnessed therapeutically.},
  archive      = {J_NG},
  author       = {Haider, Syed and Brough, Rachel and Madera, Santiago and Iacovacci, Jacopo and Gulati, Aditi and Wicks, Andrew and Alexander, John and Pettitt, Stephen J. and Tutt, Andrew N. J. and Lord, Christopher J.},
  doi          = {10.1038/s41588-025-02108-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {522-529},
  shortjournal = {Nat. Genet.},
  title        = {The transcriptomic architecture of common cancers reflects synthetic lethal interactions},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cancer-independent somatic mutation of the wild-type NF1 allele in normal tissues in neurofibromatosis type 1. <em>NG</em>, <em>57</em>(3), 515-521. (<a href='https://doi.org/10.1038/s41588-025-02097-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer predisposition syndromes mediated by recessive cancer genes generate tumors via somatic variants (second hits) in the unaffected allele. Second hits may or may not be sufficient for neoplastic transformation. Here we performed whole-genome and whole-exome sequencing on 479 tissue biopsies from a child with neurofibromatosis type 1, a multisystem cancer-predisposing syndrome mediated by constitutive monoallelic NF1 inactivation. We identified multiple independent NF1 driver variants in histologically normal tissues, but not in 610 biopsies from two nonpredisposed children. We corroborated this finding using targeted duplex sequencing, including a further nine adults with the same syndrome. Overall, truncating NF1 mutations were under positive selection in normal tissues from individuals with neurofibromatosis type 1. We demonstrate that normal tissues in neurofibromatosis type 1 commonly harbor second hits in NF1, the extent and pattern of which may underpin the syndrome’s cancer phenotype. Histologically normal tissues of individuals with a germline NF1 mutation exhibited multiple second NF1 hits, unrelated to their tumors, that conferred a selective advantage.},
  archive      = {J_NG},
  author       = {Oliver, Thomas R. W. and Lawson, Andrew R. J. and Lee-Six, Henry and Tollit, Anna and Jung, Hyunchul and Hooks, Yvette and Sanghvi, Rashesh and Young, Matthew D. and Butler, Timothy M. and Nicola, Pantelis A. and Treger, Taryn D. and Lensing, Stefanie V. and Burke, G. A. Amos and Aquilina, Kristian and Löbel, Ulrike and Cortes-Ciriano, Isidro and Hargrave, Darren and Jorgensen, Mette and Jessop, Flora A. and Coorens, Tim H. H. and Flanagan, Adrienne M. and Allinson, Kieren and Martincorena, Inigo and Jacques, Thomas S. and Behjati, Sam},
  doi          = {10.1038/s41588-025-02097-2},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {515-521},
  shortjournal = {Nat. Genet.},
  title        = {Cancer-independent somatic mutation of the wild-type NF1 allele in normal tissues in neurofibromatosis type 1},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genomics of rare diseases in the greater middle east. <em>NG</em>, <em>57</em>(3), 505-514. (<a href='https://doi.org/10.1038/s41588-025-02075-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Greater Middle East (GME) represents a concentrated region of unparalleled genetic diversity, characterized by an abundance of distinct alleles, founder mutations and extensive autozygosity driven by high consanguinity rates. These genetic hallmarks present a unique, yet vastly untapped resource for genomic research on Mendelian diseases. Despite this immense potential, the GME continues to face substantial challenges in comprehensive data collection and analysis. This Perspective highlights the region’s unique position as a natural laboratory for genetic discovery and explores the challenges that have stifled progress thus far. Importantly, we propose strategic solutions, advocating for an all-inclusive research approach. With targeted investment and focused efforts, the latent genetic wealth in the GME can be transformed into a global hub for genomic research that will redefine and advance our understanding of the human genome. The Greater Middle East is home to a rich genetic diversity that remains largely untapped in genomic research. This Perspective explores its potential for rare disease research, obstacles limiting progress and strategies to drive genomic discoveries.},
  archive      = {J_NG},
  author       = {Chekroun, Ikram and Shenbagam, Shruti and Almarri, Mohamed A. and Mokrab, Younes and Uddin, Mohammed and Alkhnbashi, Omer S. and Zaki, Maha S. and Najmabadi, Hossein and Kahrizi, Kimia and Fakhro, Khalid A. and Almontashiri, Naif A. M. and Ali, Fahad R. and Özbek, Uğur and Reversade, Bruno and Alkuraya, Fowzan S. and Alsheikh-Ali, Alawi and Abou Tayoun, Ahmad N.},
  doi          = {10.1038/s41588-025-02075-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {505-514},
  shortjournal = {Nat. Genet.},
  title        = {Genomics of rare diseases in the greater middle east},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The african animal breeding network as a pathway towards genetic improvement of livestock. <em>NG</em>, <em>57</em>(3), 498-504. (<a href='https://doi.org/10.1038/s41588-025-02079-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transforming Africa’s agricultural production and food systems is an imperative to achieve the United Nations Sustainable Development Goals and deliver on the Africa Union’s 2063 vision, ‘the Africa we want’. Transforming livestock systems through genetic improvement will sustainably increase productivity and proffer an inclusive socioeconomic development of farming communities. The African Animal Breeding Network (AABNet) is a platform of highly knowledgeable geneticists, animal breeders and professionals willing to provide information, training, advice and support across the continent. It will leverage available human resources among its members, facilitate partnerships and investment and develop infrastructure for innovative livestock genetic improvement in Africa. This Perspective introduces the African Animal Breeding Network as a platform to develop infrastructure for innovative livestock genetic improvement in Africa and discusses the challenges in implementing successful animal breeding programs.},
  archive      = {J_NG},
  author       = {Djikeng, Appolinaire and Olori, Victor E. and Houaga, Isidore and Aggrey, Samuel E. and Mwai, Okeyo and Ibeagha-Awemu, Eveline M. and Mrode, Raphael and Chagunda, Mizeck G. G. and Tiambo, Christian K. and Rekaya, Romdhane and Nash, Oyenkanmi and Nziku, Zabron and Opoola, Oluyinka and Ntanganedzeni, Mapholi and Ekine-Dzivenu, Chinyere and Kahi, Alexander and Okeno, Tobias and Hickey, John M. and Enyew, Negussie and Rege, Edward J. O.},
  doi          = {10.1038/s41588-025-02079-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {498-504},
  shortjournal = {Nat. Genet.},
  title        = {The african animal breeding network as a pathway towards genetic improvement of livestock},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unraveling the role of succinyl-CoA in tumor immunity. <em>NG</em>, <em>57</em>(3), 496-497. (<a href='https://doi.org/10.1038/s41588-025-02076-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The study highlights the crucial role of succinyl-CoA in tumor immunity by inducing the succinylation of PD-L1. We identified the succinyltransferase CPT1A as a potential therapeutic target. CPT1A could be a valuable biomarker for evaluating the efficacy of anti-PD-1 therapy and suggests a promising treatment strategy.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02076-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {496-497},
  shortjournal = {Nat. Genet.},
  title        = {Unraveling the role of succinyl-CoA in tumor immunity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Safeguard repressor locks hepatocyte identity and blocks liver cancer. <em>NG</em>, <em>57</em>(3), 494-495. (<a href='https://doi.org/10.1038/s41588-025-02082-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {PROX1 acts as a molecular ‘cell fate guardian’ in hepatocytes, preserving liver cell identity throughout life by actively repressing unwanted plasticity. PROX1 loss impairs liver regeneration and fuels tumor formation, whereas PROX1 activation halts cancer formation and progression, revealing unexpected therapeutic potential.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02082-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {494-495},
  shortjournal = {Nat. Genet.},
  title        = {Safeguard repressor locks hepatocyte identity and blocks liver cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Novel mRNA isoforms in human microglia refine genetic associations with neurodegeneration. <em>NG</em>, <em>57</em>(3), 492-493. (<a href='https://doi.org/10.1038/s41588-025-02112-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read RNA sequencing of human microglia expands the catalog of mRNA isoforms in this important cell type and assists with the interpretation of genetic associations with Alzheimer’s disease and Parkinson’s disease.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02112-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {492-493},
  shortjournal = {Nat. Genet.},
  title        = {Novel mRNA isoforms in human microglia refine genetic associations with neurodegeneration},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Polyguanine microsatellites are robust replication clocks in cancer. <em>NG</em>, <em>57</em>(3), 489-491. (<a href='https://doi.org/10.1038/s41588-025-02098-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer phylogenetic trees describe the evolutionary relationship between primary tumors and metastatic sites. A study now shows that mutations in guanine homopolymer microsatellites represent accurate molecular clocks, revealing the number of cell divisions that have occurred during cancer development and progression.},
  archive      = {J_NG},
  author       = {Gejman, Ron S. and Izar, Benjamin},
  doi          = {10.1038/s41588-025-02098-1},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {489-491},
  shortjournal = {Nat. Genet.},
  title        = {Polyguanine microsatellites are robust replication clocks in cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mass spectrometry-based mapping of plasma protein QTLs in children and adolescents. <em>NG</em>, <em>57</em>(3), 487-488. (<a href='https://doi.org/10.1038/s41588-025-02088-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mass spectrometry-based proteomics demonstrated its increasing analytical power and unique strengths in measuring genome-wide plasma proteomes and identifying protein quantitative trait loci (pQTLs), as reported in a study of over 3,000 Danish children and adolescents.},
  archive      = {J_NG},
  author       = {Liu, Yansheng},
  doi          = {10.1038/s41588-025-02088-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {487-488},
  shortjournal = {Nat. Genet.},
  title        = {Mass spectrometry-based mapping of plasma protein QTLs in children and adolescents},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell mapping of HTT somatic repeat expansions. <em>NG</em>, <em>57</em>(3), 486. (<a href='https://doi.org/10.1038/s41588-025-02142-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-025-02142-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell mapping of HTT somatic repeat expansions},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-throughput screening identifies modulators of gene bursting. <em>NG</em>, <em>57</em>(3), 486. (<a href='https://doi.org/10.1038/s41588-025-02143-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02143-z},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Genet.},
  title        = {High-throughput screening identifies modulators of gene bursting},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Understanding immune escape in lung cancer. <em>NG</em>, <em>57</em>(3), 486. (<a href='https://doi.org/10.1038/s41588-025-02144-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02144-y},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Genet.},
  title        = {Understanding immune escape in lung cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Hyperactive histone genes fuel cancer. <em>NG</em>, <em>57</em>(3), 486. (<a href='https://doi.org/10.1038/s41588-025-02145-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-025-02145-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {486},
  shortjournal = {Nat. Genet.},
  title        = {Hyperactive histone genes fuel cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The federated european Genome–Phenome archive as a global network for sharing human genomics data. <em>NG</em>, <em>57</em>(3), 481-485. (<a href='https://doi.org/10.1038/s41588-025-02101-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this era of rapidly expanding human genomics in research and healthcare, efficient data reuse is essential to maximize benefits for society. In response, the Federated European Genome–Phenome Archive (FEGA) was launched in 2022, and as of 2024, the FEGA network was composed of seven national nodes. Here we describe the complexities, challenges and achievements of FEGA, unravelling the dynamic interplay of regulatory frameworks, technical challenges and the shared vision of advancing genomic research.},
  archive      = {J_NG},
  author       = {D’Altri, Teresa and Freeberg, Mallory Ann and Curwin, Amy J. and Alonso, Ana and Freitas, Ana T. and Capella-Gutierrez, Salvador and Gadelha, Luiz and Hagwall, Anna and Hovig, Eivind and Kerry, Giselle and Kirli, Koray and Kochel, Krzysztof and Kohlbacher, Oliver and Korbel, Jan O. and Leinonen, Jaakko and Marciniak, Blazej and Oliveira, Jorge S. and Petersen, Kjell and Silva, Mário J. and Stegle, Oliver and Valencia, Alfonso and Viklund, Johan and Guigo, Roderic and Parkinson, Helen and Navarro, Arcadi and Rambla, Jordi and Keane, Thomas M.},
  doi          = {10.1038/s41588-025-02101-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {481-485},
  shortjournal = {Nat. Genet.},
  title        = {The federated european Genome–Phenome archive as a global network for sharing human genomics data},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: A multilineage screen identifies actionable synthetic lethal interactions in human cancers. <em>NG</em>, <em>57</em>(2), 480. (<a href='https://doi.org/10.1038/s41588-025-02090-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fong, Samson H. and Kuenzi, Brent M. and Mattson, Nicole M. and Lee, John and Sanchez, Kyle and Bojorquez-Gomez, Ana and Ford, Kyle and Munson, Brenton P. and Licon, Katherine and Bergendahl, Sarah and Shen, John Paul and Kreisberg, Jason F. and Mali, Prashant and Hager, Jeffrey H. and White, Michael A. and Ideker, Trey},
  doi          = {10.1038/s41588-025-02090-9},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {480},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: A multilineage screen identifies actionable synthetic lethal interactions in human cancers},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: The ZmCPK39–ZmDi19–ZmPR10 immune module regulates quantitative resistance to multiple foliar diseases in maize. <em>NG</em>, <em>57</em>(2), 480. (<a href='https://doi.org/10.1038/s41588-025-02091-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Zhu, Mang and Zhong, Tao and Xu, Ling and Guo, Chenyu and Zhang, Xiaohui and Liu, Yulin and Zhang, Yan and Li, Yancong and Xie, Zhijian and Liu, Tingting and Jiang, Fuyan and Fan, Xingming and Balint-Kurti, Peter and Xu, Mingliang},
  doi          = {10.1038/s41588-025-02091-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {480},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: The ZmCPK39–ZmDi19–ZmPR10 immune module regulates quantitative resistance to multiple foliar diseases in maize},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Synchronized long-read genome, methylome, epigenome and transcriptome profiling resolve a mendelian condition. <em>NG</em>, <em>57</em>(2), 469-479. (<a href='https://doi.org/10.1038/s41588-024-02067-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Resolving the molecular basis of a Mendelian condition remains challenging owing to the diverse mechanisms by which genetic variants cause disease. To address this, we developed a synchronized long-read genome, methylome, epigenome and transcriptome sequencing approach, which enables accurate single-nucleotide, insertion–deletion and structural variant calling and diploid de novo genome assembly. This permits the simultaneous elucidation of haplotype-resolved CpG methylation, chromatin accessibility and full-length transcript information in a single long-read sequencing run. Application of this approach to an Undiagnosed Diseases Network participant with a chromosome X;13-balanced translocation of uncertain significance revealed that this translocation disrupted the functioning of four separate genes (NBEA, PDK3, MAB21L1 and RB1) previously associated with single-gene Mendelian conditions. Notably, the function of each gene was disrupted via a distinct mechanism that required integration of the four ‘omes’ to resolve. These included fusion transcript formation, enhancer adoption, transcriptional readthrough silencing and inappropriate X-chromosome inactivation of autosomal genes. Overall, this highlights the utility of synchronized long-read multi-omic profiling for mechanistically resolving complex phenotypes. Simultaneous profiling of the genome, methylome, epigenome and transcriptome using single-molecule chromatin fiber sequencing and multiplexed arrays isoform sequencing identifies the genetic and molecular basis of an undiagnosed Mendelian disease case with an X;13-balanced translocation.},
  archive      = {J_NG},
  author       = {Vollger, Mitchell R. and Korlach, Jonas and Eldred, Kiara C. and Swanson, Elliott and Underwood, Jason G. and Bohaczuk, Stephanie C. and Mao, Yizi and Cheng, Yong-Han H. and Ranchalis, Jane and Blue, Elizabeth E. and Schwarze, Ulrike and Munson, Katherine M. and Saunders, Christopher T. and Wenger, Aaron M. and Allworth, Aimee and Chanprasert, Sirisak and Duerden, Brittney L. and Glass, Ian and Horike-Pyne, Martha and Kim, Michelle and Leppig, Kathleen A. and McLaughlin, Ian J. and Ogawa, Jessica and Rosenthal, Elisabeth A. and Sheppeard, Sam and Sherman, Stephanie M. and Strohbehn, Samuel and Yuen, Amy L. and Stacey, Andrew W. and Reh, Thomas A. and Byers, Peter H. and Bamshad, Michael J. and Hisama, Fuki M. and Jarvik, Gail P. and Sancak, Yasemin and Dipple, Katrina M. and Stergachis, Andrew B.},
  doi          = {10.1038/s41588-024-02067-0},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {469-479},
  shortjournal = {Nat. Genet.},
  title        = {Synchronized long-read genome, methylome, epigenome and transcriptome profiling resolve a mendelian condition},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A scalable variational inference approach for increased mixed-model association power. <em>NG</em>, <em>57</em>(2), 461-468. (<a href='https://doi.org/10.1038/s41588-024-02044-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rapid growth of modern biobanks is creating new opportunities for large-scale genome-wide association studies (GWASs) and the analysis of complex traits. However, performing GWASs on millions of samples often leads to trade-offs between computational efficiency and statistical power, reducing the benefits of large-scale data collection efforts. We developed Quickdraws, a method that increases association power in quantitative and binary traits without sacrificing computational efficiency, leveraging a spike-and-slab prior on variant effects, stochastic variational inference and graphics processing unit acceleration. We applied Quickdraws to 79 quantitative and 50 binary traits in 405,088 UK Biobank samples, identifying 4.97% and 3.25% more associations than REGENIE and 22.71% and 7.07% more than FastGWA. Quickdraws had costs comparable to REGENIE, FastGWA and SAIGE on the UK Biobank Research Analysis Platform service, while being substantially faster than BOLT-LMM. These results highlight the promise of leveraging machine learning techniques for scalable GWASs without sacrificing power or robustness. Quickdraws is a mixed-model association tool with a noninfinitesimal prior for analyzing binary and quantitative traits, using a scalable variational inference that allows analysis of biobank-scale cohorts.},
  archive      = {J_NG},
  author       = {Loya, Hrushikesh and Kalantzis, Georgios and Cooper, Fergus and Palamara, Pier Francesco},
  doi          = {10.1038/s41588-024-02044-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {461-468},
  shortjournal = {Nat. Genet.},
  title        = {A scalable variational inference approach for increased mixed-model association power},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Transcript-specific enrichment enables profiling of rare cell states via single-cell RNA sequencing. <em>NG</em>, <em>57</em>(2), 451-460. (<a href='https://doi.org/10.1038/s41588-024-02036-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell genomics technologies have accelerated our understanding of cell-state heterogeneity in diverse contexts. Although single-cell RNA sequencing identifies rare populations that express specific marker transcript combinations, traditional flow sorting requires cell surface markers with high-fidelity antibodies, limiting our ability to interrogate these populations. In addition, many single-cell studies require the isolation of nuclei from tissue, eliminating the ability to enrich learned rare cell states based on extranuclear protein markers. In the present report, we addressed these limitations by developing Programmable Enrichment via RNA FlowFISH by sequencing (PERFF-seq), a scalable assay that enables scRNA-seq profiling of subpopulations defined by the abundance of specific RNA transcripts. Across immune populations (n = 184,126 cells) and fresh-frozen and formalin-fixed, paraffin-embedded brain tissue (n = 33,145 nuclei), we demonstrated that programmable sorting logic via RNA-based cytometry can isolate rare cell populations and uncover phenotypic heterogeneity via downstream, high-throughput, single-cell genomics analyses. Programmable Enrichment via RNA FlowFISH by sequencing (PERFF-seq) isolates rare cells based on RNA marker transcripts for single-cell RNA sequencing profiling of complex tissues, with applicability to a broad variety of samples and cell types.},
  archive      = {J_NG},
  author       = {Abay, Tsion and Stickels, Robert R. and Takizawa, Meril T. and Nalbant, Benan N. and Hsieh, Yu-Hsin and Hwang, Sidney and Snopkowski, Catherine and Yu, Kenny Kwok Hei and Abou-Mrad, Zaki and Tabar, Viviane and Howitt, Brooke E. and Ludwig, Leif S. and Chaligné, Ronan and Satpathy, Ansuman T. and Lareau, Caleb A.},
  doi          = {10.1038/s41588-024-02036-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {451-460},
  shortjournal = {Nat. Genet.},
  title        = {Transcript-specific enrichment enables profiling of rare cell states via single-cell RNA sequencing},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A barley pan-transcriptome reveals layers of genotype-dependent transcriptional complexity. <em>NG</em>, <em>57</em>(2), 441-450. (<a href='https://doi.org/10.1038/s41588-024-02069-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A pan-transcriptome describes the transcriptional and post-transcriptional consequences of genome diversity from multiple individuals within a species. We developed a barley pan-transcriptome using 20 inbred genotypes representing domesticated barley diversity by generating and analyzing short- and long-read RNA-sequencing datasets from multiple tissues. To overcome single reference bias in transcript quantification, we constructed genotype-specific reference transcript datasets (RTDs) and integrated these into a linear pan-genome framework to create a pan-RTD, allowing transcript categorization as core, shell or cloud. Focusing on the core (expressed in all genotypes), we observed significant transcript abundance variation among tissues and between genotypes driven partly by RNA processing, gene copy number, structural rearrangements and conservation of promotor motifs. Network analyses revealed conserved co-expression module::tissue correlations and frequent functional diversification. To complement the pan-transcriptome, we constructed a comprehensive cultivar (cv.) Morex gene-expression atlas and illustrate how these combined datasets can be used to guide biological inquiry. A long- and short-read barley pan-transcriptome assembled from 20 diverse barley genotypes offers insights into genotype- and tissue-dependent gene expression and function.},
  archive      = {J_NG},
  author       = {Guo, Wenbin and Schreiber, Miriam and Marosi, Vanda B. and Bagnaresi, Paolo and Jørgensen, Morten Egevang and Braune, Katarzyna B. and Chalmers, Ken and Chapman, Brett and Dang, Viet and Dockter, Christoph and Fiebig, Anne and Fincher, Geoffrey B. and Fricano, Agostino and Fuller, John and Haaning, Allison and Haberer, Georg and Himmelbach, Axel and Jayakodi, Murukarthick and Jia, Yong and Kamal, Nadia and Langridge, Peter and Li, Chengdao and Lu, Qiongxian and Lux, Thomas and Mascher, Martin and Mayer, Klaus F. X. and McCallum, Nicola and Milne, Linda and Muehlbauer, Gary J. and Nielsen, Martin T. S. and Padmarasu, Sudharsan and Pedas, Pai Rosager and Pillen, Klaus and Pozniak, Curtis and Rasmussen, Magnus W. and Sato, Kazuhiro and Schmutzer, Thomas and Scholz, Uwe and Schüler, Danuta and Šimková, Hana and Skadhauge, Birgitte and Stein, Nils and Thomsen, Nina W. and Voss, Cynthia and Wang, Penghao and Wonneberger, Ronja and Zhang, Xiao-Qi and Zhang, Guoping and Cattivelli, Luigi and Spannagl, Manuel and Bayer, Micha and Simpson, Craig and Zhang, Runxuan and Waugh, Robbie},
  doi          = {10.1038/s41588-024-02069-y},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {441-450},
  shortjournal = {Nat. Genet.},
  title        = {A barley pan-transcriptome reveals layers of genotype-dependent transcriptional complexity},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The NAT1–bHLH110–CER1/CER1L module regulates heat stress tolerance in rice. <em>NG</em>, <em>57</em>(2), 427-440. (<a href='https://doi.org/10.1038/s41588-024-02065-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rice production is facing substantial threats from global warming associated with extreme temperatures. Here we report that modifying a heat stress-induced negative regulator, a negative regulator of thermotolerance 1 (NAT1), increases wax deposition and enhances thermotolerance in rice. We demonstrated that the C2H2 family transcription factor NAT1 directly inhibits bHLH110 expression, and bHLH110 directly promotes the expression of wax biosynthetic genes CER1/CER1L under heat stress conditions. In situ hybridization revealed that both NAT1 and bHLH110 are predominantly expressed in epidermal layers. By using gene-editing technology, we successfully mutated NAT1 to eliminate its inhibitory effects on wax biosynthesis and improved thermotolerance without yield penalty under normal temperature conditions. Field trials further confirmed the potential of NAT1-edited rice to increase seed-setting rate and grain yield. Therefore, our findings shed light on the regulatory mechanisms governing wax biosynthesis under heat stress conditions in rice and provide a strategy to enhance heat resilience through the modification of NAT1. Negative regulator of thermotolerance 1 (NAT1) is identified as a negative regulator of thermotolerance in rice through the NAT1–bHLH110–CER1/CER1L module. Modifying NAT1 by targeted gene editing increases wax deposition and enhances thermotolerance in rice.},
  archive      = {J_NG},
  author       = {Lu, Hai-Ping and Liu, Xue-Huan and Wang, Mei-Jing and Zhu, Qiao-Yun and Lyu, Yu-Shu and Xu, Jian-Hang and Liu, Jian-Xiang},
  doi          = {10.1038/s41588-024-02065-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {427-440},
  shortjournal = {Nat. Genet.},
  title        = {The NAT1–bHLH110–CER1/CER1L module regulates heat stress tolerance in rice},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Human subcutaneous and visceral adipocyte atlases uncover classical and nonclassical adipocytes and depot-specific patterns. <em>NG</em>, <em>57</em>(2), 413-426. (<a href='https://doi.org/10.1038/s41588-024-02048-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human adipose depots are functionally distinct. Yet, recent single-nucleus RNA sequencing (snRNA-seq) analyses largely uncovered overlapping or similar cell-type landscapes. We hypothesized that adipocyte subtypes, differentiation trajectories and/or intercellular communication patterns could illuminate this depot similarity–difference gap. For this, we performed snRNA-seq of human subcutaneous or visceral adipose tissues (five or ten samples, respectively). Of 27,665 adipocyte nuclei in both depots, most were ‘classical’, namely enriched in lipid metabolism pathways. However, we also observed ‘nonclassical’ adipocyte subtypes, enriched in immune-related, extracellular matrix deposition (fibrosis), vascularization or angiogenesis or ribosomal and mitochondrial processes. Pseudo-temporal analysis showed a developmental trajectory from adipose progenitor cells to classical adipocytes via nonclassical adipocytes, suggesting that the classical state stems from loss, rather than gain, of specialized functions. Last, intercellular communication routes were consistent with the different inflammatory tone of the two depots. Jointly, these findings provide a high-resolution view into the contribution of cellular composition, differentiation and intercellular communication patterns to human fat depot differences. Single-nucleus RNA sequencing of human visceral and subcutaneous adipose tissues is used to identify adipocyte subpopulations and explore their developmental trajectories and interactions.},
  archive      = {J_NG},
  author       = {Lazarescu, Or and Ziv-Agam, Maya and Haim, Yulia and Hekselman, Idan and Jubran, Juman and Shneyour, Ariel and Muallem, Habib and Zemer, Alon and Rosengarten-Levin, Marina and Kitsberg, Daniel and Levin, Liron and Liberty, Idit F. and Yoel, Uri and Dukhno, Oleg and Adam, Miriam and Braune, Julia and Müller, Claudia and Raulien, Nora and Gericke, Martin and Körner, Antje and Murphy, Rinki and Blüher, Matthias and Habib, Naomi and Rudich, Assaf and Yeger-Lotem, Esti},
  doi          = {10.1038/s41588-024-02048-3},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {413-426},
  shortjournal = {Nat. Genet.},
  title        = {Human subcutaneous and visceral adipocyte atlases uncover classical and nonclassical adipocytes and depot-specific patterns},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Oncofetal reprogramming drives phenotypic plasticity in WNT-dependent colorectal cancer. <em>NG</em>, <em>57</em>(2), 402-412. (<a href='https://doi.org/10.1038/s41588-024-02058-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeting cancer stem cells (CSCs) is crucial for effective cancer treatment, yet resistance mechanisms to LGR5+ CSC depletion in WNT-driven colorectal cancer (CRC) remain elusive. In the present study, we revealed that mutant intestinal stem cells (SCs) depart from their canonical identity, traversing a dynamic phenotypic spectrum. This enhanced plasticity is initiated by oncofetal (OnF) reprogramming, driven by YAP and AP-1, with subsequent AP-1 hyperactivation promoting lineage infidelity. The retinoid X receptor serves as a gatekeeper of OnF reprogramming and its deregulation after adenomatous polyposis coli (APC) loss of function establishes an OnF ‘memory’ sustained by YAP and AP-1. Notably, the clinical significance of OnF and LGR5+ states in isolation is constrained by their functional redundancy. Although the canonical LGR5+ state is sensitive to the FOLFIRI regimen, an active OnF program correlates with resistance, supporting its role in driving drug-tolerant states. Targeting this program in combination with the current standard of care is pivotal for achieving effective and durable CRC treatment. Oncofetal (OnF) reprogramming, driven by YAP and AP-1, induces phenotypic plasticity and therapy resistance in WNT-dependent colorectal cancer (CRC). Targeting the OnF state in combination with chemotherapy substantially attenuates tumor growth in mouse models and patient-derived CRC tumoroids.},
  archive      = {J_NG},
  author       = {Mzoughi, Slim and Schwarz, Megan and Wang, Xuedi and Demircioglu, Deniz and Ulukaya, Gulay and Mohammed, Kevin and Zorgati, Habiba and Torre, Denis and Tomalin, Lewis E. and Di Tullio, Federico and Company, Carlos and Dramaretska, Yuliia and Leushacke, Marc and Giotti, Bruno and Lannagan, Tamsin RM and Lozano-Ojalvo, Daniel and Karras, Panagiotis and Vermeulen, Peter B. and Hasson, Dan and Sebra, Robert and Tsankov, Alexander M. and Sansom, Owen J. and Marine, Jean-Christophe and Barker, Nick and Gargiulo, Gaetano and Guccione, Ernesto},
  doi          = {10.1038/s41588-024-02058-1},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {402-412},
  shortjournal = {Nat. Genet.},
  title        = {Oncofetal reprogramming drives phenotypic plasticity in WNT-dependent colorectal cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Structural polymorphism and diversity of human segmental duplications. <em>NG</em>, <em>57</em>(2), 390-401. (<a href='https://doi.org/10.1038/s41588-024-02051-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Segmental duplications (SDs) contribute significantly to human disease, evolution and diversity but have been difficult to resolve at the sequence level. We present a population genetics survey of SDs by analyzing 170 human genome assemblies (from 85 samples representing 38 Africans and 47 non-Africans) in which the majority of autosomal SDs are fully resolved using long-read sequence assembly. Excluding the acrocentric short arms and sex chromosomes, we identify 173.2 Mb of duplicated sequence (47.4 Mb not present in the telomere-to-telomere reference) distinguishing fixed from structurally polymorphic events. We find that intrachromosomal SDs are among the most variable, with rare events mapping near their progenitor sequences. African genomes harbor significantly more intrachromosomal SDs and are more likely to have recently duplicated gene families with higher copy numbers than non-African samples. Comparison to a resource of 563 million full-length isoform sequencing reads identifies 201 novel, potentially protein-coding genes corresponding to these copy number polymorphic SDs. Analysis of 170 human genomes assembled using long-read sequencing provides a map of structural variation within regions of segmental duplication and identifies novel candidate protein-coding genes supported by full-length Iso-Seq reads.},
  archive      = {J_NG},
  author       = {Jeong, Hyeonsoo and Dishuck, Philip C. and Yoo, DongAhn and Harvey, William T. and Munson, Katherine M. and Lewis, Alexandra P. and Kordosky, Jennifer and Garcia, Gage H. and Yilmaz, Feyza and Hallast, Pille and Lee, Charles and Pastinen, Tomi and Eichler, Evan E.},
  doi          = {10.1038/s41588-024-02051-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {390-401},
  shortjournal = {Nat. Genet.},
  title        = {Structural polymorphism and diversity of human segmental duplications},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fine-scale population structure and widespread conservation of genetic effect sizes between human groups across traits. <em>NG</em>, <em>57</em>(2), 379-389. (<a href='https://doi.org/10.1038/s41588-024-02035-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding genetic differences between populations is essential for avoiding confounding in genome-wide association studies and improving polygenic score (PGS) portability. We developed a statistical pipeline to infer fine-scale Ancestry Components and applied it to UK Biobank data. Ancestry Components identify population structure not captured by widely used principal components, improving stratification correction for geographically correlated traits. To estimate the similarity of genetic effect sizes between groups, we developed ANCHOR, which estimates changes in the predictive power of an existing PGS in distinct local ancestry segments. ANCHOR infers highly similar (estimated correlation 0.98 ± 0.07) effect sizes between UK Biobank participants of African and European ancestry for 47 of 53 quantitative phenotypes, suggesting that gene–environment and gene–gene interactions do not play major roles in poor cross-ancestry PGS transferability for these traits in the United Kingdom, and providing optimism that shared causal mutations operate similarly in different populations. This study introduces the concept of Ancestry Components and shows that they can offer improved population stratification correction for geographically correlated traits. By using ancestry-aware polygenic score construction in admixed individuals, the authors find that effect sizes are conserved across ancestry groups.},
  archive      = {J_NG},
  author       = {Hu, Sile and Ferreira, Lino A. F. and Shi, Sinan and Hellenthal, Garrett and Marchini, Jonathan and Lawson, Daniel J. and Myers, Simon R.},
  doi          = {10.1038/s41588-024-02035-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {379-389},
  shortjournal = {Nat. Genet.},
  title        = {Fine-scale population structure and widespread conservation of genetic effect sizes between human groups across traits},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multi-omic quantitative trait loci link tandem repeat size variation to gene regulation in human brain. <em>NG</em>, <em>57</em>(2), 369-378. (<a href='https://doi.org/10.1038/s41588-024-02057-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tandem repeat (TR) size variation is implicated in ~50 neurological disorders, yet its impact on gene regulation in the human brain remains largely unknown. In the present study, we quantified the impact of TR size variation on brain gene regulation across distinct molecular phenotypes, based on 4,412 multi-omics samples from 1,597 donors, including 1,586 newly sequenced ones. We identified ~2.2 million TR molecular quantitative trait loci (TR-xQTLs), linking ~139,000 unique TRs to nearby molecular phenotypes, including many known disease-risk TRs, such as the G2C4 expansion in C9orf72 associated with amyotrophic lateral sclerosis. Fine-mapping revealed ~18,700 TRs as potential causal variants. Our in vitro experiments further confirmed the causal and independent regulatory effects of three TRs. Additional colocalization analysis indicated the potential causal role of TR variation in brain-related phenotypes, highlighted by a 3ʹ-UTR TR in NUDT14 linked to cortical surface area and a TG repeat in PLEKHA1, associated with Alzheimer’s disease. Mapping of multi-omic molecular quantitative trait loci associated with tandem repeat size variation in up to 4,412 human brain samples from 1,597 donors offers insights into how these variants affect gene regulation and mediate disease risk.},
  archive      = {J_NG},
  author       = {Cui, Ya and Arnold, Frederick J. and Li, Jason Sheng and Wu, Jie and Wang, Dan and Philippe, Julien and Colwin, Michael R. and Michels, Sebastian and Chen, Chaorong and Sallam, Tamer and Thompson, Leslie M. and La Spada, Albert R. and Li, Wei},
  doi          = {10.1038/s41588-024-02057-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {369-378},
  shortjournal = {Nat. Genet.},
  title        = {Multi-omic quantitative trait loci link tandem repeat size variation to gene regulation in human brain},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cell state-dependent allelic effects and contextual mendelian randomization analysis for human brain phenotypes. <em>NG</em>, <em>57</em>(2), 358-368. (<a href='https://doi.org/10.1038/s41588-024-02050-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression quantitative trait loci are widely used to infer relationships between genes and central nervous system (CNS) phenotypes; however, the effect of brain disease on these inferences is unclear. Using 2,348,438 single-nuclei profiles from 391 disease-case and control brains, we report 13,939 genes whose expression correlated with genetic variation, of which 16.7–40.8% (depending on cell type) showed disease-dependent allelic effects. Across 501 colocalizations for 30 CNS traits, 23.6% had a disease dependency, even after adjusting for disease status. To estimate the unconfounded effect of genes on outcomes, we repeated the analysis using nondiseased brains (n = 183) and reported an additional 91 colocalizations not present in the larger mixed disease and control dataset, demonstrating enhanced interpretation of disease-associated variants. Principled implementation of single-cell Mendelian randomization in control-only brains identified 140 putatively causal gene–trait associations, of which 11 were replicated in the UK Biobank, prioritizing candidate peripheral biomarkers predictive of CNS outcomes. Analysis of single nuclei RNA-sequencing data across eight major brain cell types identifies putative causal associations between cell-type-specific expression and brain phenotypes.},
  archive      = {J_NG},
  author       = {Haglund, Alexander and Zuber, Verena and Abouzeid, Maya and Yang, Yifei and Ko, Jeong Hun and Wiemann, Liv and Otero-Jimenez, Maria and Muhammed, Louwai and Feleke, Rahel and Nott, Alexi and Mills, James D. and Laaniste, Liisi and Gveric, Djordje O. and Clode, Daniel and Babtie, Ann C. and Pagni, Susanna and Bellampalli, Ravishankara and Somani, Alyma and McDade, Karina and Anink, Jasper J. and Mesarosova, Lucia and Fancy, Nurun and Willumsen, Nanet and Smith, Amy and Jackson, Johanna and Alegre-Abarrategui, Javier and Aronica, Eleonora and Matthews, Paul M. and Thom, Maria and Sisodiya, Sanjay M. and Srivastava, Prashant K. and Malhotra, Dheeraj and Bryois, Julien and Bottolo, Leonardo and Johnson, Michael R.},
  doi          = {10.1038/s41588-024-02050-9},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {358-368},
  shortjournal = {Nat. Genet.},
  title        = {Cell state-dependent allelic effects and contextual mendelian randomization analysis for human brain phenotypes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Integrative proteogenomic analysis identifies COL6A3-derived endotrophin as a mediator of the effect of obesity on coronary artery disease. <em>NG</em>, <em>57</em>(2), 345-357. (<a href='https://doi.org/10.1038/s41588-024-02052-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Obesity strongly increases the risk of cardiometabolic diseases, yet the underlying mediators of this relationship are not fully understood. Given that obesity strongly influences circulating protein levels, we investigated proteins mediating the effects of obesity on coronary artery disease, stroke and type 2 diabetes. By integrating two-step proteome-wide Mendelian randomization, colocalization, epigenomics and single-cell RNA sequencing, we identified five mediators and prioritized collagen type VI α3 (COL6A3). COL6A3 levels were strongly increased by body mass index and increased coronary artery disease risk. Notably, the carboxyl terminus product of COL6A3, endotrophin, drove this effect. COL6A3 was highly expressed in disease-relevant cell types and tissues. Finally, we found that body fat reduction could reduce plasma levels of COL6A3-derived endotrophin, indicating a tractable way to modify endotrophin levels. In summary, we provide actionable insights into how circulating proteins mediate the effects of obesity on cardiometabolic diseases and prioritize endotrophin as a potential therapeutic target. Two-step Mendelian randomization, combined with multiple layers of omics evidence, implicates COL6A3-derived endotrophin as a mediator of coronary artery disease risk in the context of obesity.},
  archive      = {J_NG},
  author       = {Yoshiji, Satoshi and Lu, Tianyuan and Butler-Laporte, Guillaume and Carrasco-Zanini-Sanchez, Julia and Su, Chen-Yang and Chen, Yiheng and Liang, Kevin and Willett, Julian Daniel Sunday and Wang, Shidong and Adra, Darin and Ilboudo, Yann and Sasako, Takayoshi and Koyama, Satoshi and Nakao, Tetsushi and Forgetta, Vincenzo and Farjoun, Yossi and Zeberg, Hugo and Zhou, Sirui and Marks-Hultström, Michael and Machiela, Mitchell J. and Kaalia, Rama and Dashti, Hesam and Claussnitzer, Melina and Flannick, Jason and Wareham, Nicholas J. and Mooser, Vincent and Timpson, Nicholas J. and Langenberg, Claudia and Richards, J. Brent},
  doi          = {10.1038/s41588-024-02052-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {345-357},
  shortjournal = {Nat. Genet.},
  title        = {Integrative proteogenomic analysis identifies COL6A3-derived endotrophin as a mediator of the effect of obesity on coronary artery disease},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome-wide association study of prostate-specific antigen levels in 392,522 men identifies new loci and improves prediction across ancestry groups. <em>NG</em>, <em>57</em>(2), 334-344. (<a href='https://doi.org/10.1038/s41588-024-02068-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We conducted a multiancestry genome-wide association study of prostate-specific antigen (PSA) levels in 296,754 men (211,342 European ancestry, 58,236 African ancestry, 23,546 Hispanic/Latino and 3,630 Asian ancestry; 96.5% of participants were from the Million Veteran Program). We identified 318 independent genome-wide significant (P ≤ 5 × 10−8) variants, 184 of which were novel. Most demonstrated evidence of replication in an independent cohort (n = 95,768). Meta-analyzing discovery and replication (n = 392,522) identified 447 variants, of which a further 111 were novel. Out-of-sample variance in PSA explained by our genome-wide polygenic risk scores ranged from 11.6% to 16.6% for European ancestry, 5.5% to 9.5% for African ancestry, 13.5% to 18.2% for Hispanic/Latino and 8.6% to 15.3% for Asian ancestry and decreased with increasing age. Midlife genetically adjusted PSA levels were more strongly associated with overall and aggressive prostate cancer than unadjusted PSA levels. Our study highlights how including proportionally more participants from underrepresented populations improves genetic prediction of PSA levels, offering potential to personalize prostate cancer screening. This study shows how including different ancestry groups in a genome-wide association study for prostate-specific antigen levels can improve prostate cancer risk prediction, with implications for population screening.},
  archive      = {J_NG},
  author       = {Hoffmann, Thomas J. and Graff, Rebecca E. and Madduri, Ravi K. and Rodriguez, Alex A. and Cario, Clinton L. and Feng, Karen and Jiang, Yu and Wang, Anqi and Klein, Robert J. and Pierce, Brandon L. and Eggener, Scott and Tong, Lin and Blot, William and Long, Jirong and Goss, Louisa B. and Darst, Burcu F. and Rebbeck, Timothy and Lachance, Joseph and Andrews, Caroline and Adebiyi, Akindele O. and Adusei, Ben and Aisuodionoe-Shadrach, Oseremen I. and Fernandez, Pedro W. and Jalloh, Mohamed and Janivara, Rohini and Chen, Wenlong C. and Mensah, James E. and Agalliu, Ilir and Berndt, Sonja I. and Shelley, John P. and Schaffer, Kerry and Machiela, Mitchell J. and Freedman, Neal D. and Huang, Wen-Yi and Li, Shengchao A. and Goodman, Phyllis J. and Till, Cathee and Thompson, Ian and Lilja, Hans and Ranatunga, Dilrini K. and Presti, Joseph and Van Den Eeden, Stephen K. and Chanock, Stephen J. and Mosley, Jonathan D. and Conti, David V. and Haiman, Christopher A. and Justice, Amy C. and Kachuri, Linda and Witte, John S.},
  doi          = {10.1038/s41588-024-02068-z},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {334-344},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association study of prostate-specific antigen levels in 392,522 men identifies new loci and improves prediction across ancestry groups},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Prioritizing effector genes at trait-associated loci using multimodal evidence. <em>NG</em>, <em>57</em>(2), 323-333. (<a href='https://doi.org/10.1038/s41588-025-02084-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWAS) yield large numbers of genetic loci associated with traits and diseases. Predicting the effector genes that mediate these locus-trait associations remains challenging. Here we present the FLAMES (fine-mapped locus assessment model of effector genes) framework, which predicts the most likely effector gene in a locus. FLAMES creates machine learning predictions from biological data linking single-nucleotide polymorphisms to genes, and then evaluates these scores together with gene-centric evidence of convergence of the GWAS signal in functional networks. We benchmark FLAMES on gene-locus pairs derived by expert curation, rare variant implication and domain knowledge of molecular traits. We demonstrate that combining single-nucleotide-polymorphism-based and convergence-based modalities outperforms prioritization strategies using a single line of evidence. Applying FLAMES, we resolve the FSHB locus in the GWAS for dizygotic twinning and further leverage this framework to find schizophrenia risk genes that converge with rare coding evidence and are relevant in different stages of life. FLAMES is a machine learning approach combining variant fine-mapping, SNP-to-gene annotations and convergence-based gene prioritization scores to identify candidate effector genes at genome-wide associated loci with high accuracy.},
  archive      = {J_NG},
  author       = {Schipper, Marijn and de Leeuw, Christiaan A. and Maciel, Bernardo A. P. C. and Wightman, Douglas P. and Hubers, Nikki and Boomsma, Dorret I. and O’Donovan, Michael C. and Posthuma, Danielle},
  doi          = {10.1038/s41588-025-02084-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {323-333},
  shortjournal = {Nat. Genet.},
  title        = {Prioritizing effector genes at trait-associated loci using multimodal evidence},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). In vivo CRISPR–Cas9 genome editing in mice identifies genetic modifiers of somatic CAG repeat instability in huntington’s disease. <em>NG</em>, <em>57</em>(2), 314-322. (<a href='https://doi.org/10.1038/s41588-024-02054-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Huntington’s disease, one of more than 50 inherited repeat expansion disorders1, is a dominantly inherited neurodegenerative disease caused by a CAG expansion in HTT2. Inherited CAG repeat length is the primary determinant of age of onset, with human genetic studies underscoring that the disease is driven by the CAG length-dependent propensity of the repeat to further expand in the brain3–9. Routes to slowing somatic CAG expansion, therefore, hold promise for disease-modifying therapies. Several DNA repair genes, notably in the mismatch repair pathway, modify somatic expansion in Huntington’s disease mouse models10. To identify novel modifiers of somatic expansion, we used CRISPR–Cas9 editing in Huntington’s disease knock-in mice to enable in vivo screening of expansion-modifier candidates at scale. This included testing of Huntington’s disease onset modifier genes emerging from human genome-wide association studies as well as interactions between modifier genes, providing insight into pathways underlying CAG expansion and potential therapeutic targets. A novel in vivo screening strategy identifies new modifiers of somatic CAG repeat expansion that contribute to age of onset in Huntington’s disease.},
  archive      = {J_NG},
  author       = {Mouro Pinto, Ricardo and Murtha, Ryan and Azevedo, António and Douglas, Cameron and Kovalenko, Marina and Ulloa, Jessica and Crescenti, Steven and Burch, Zoe and Oliver, Esaria and Kesavan, Maheswaran and Shibata, Shota and Vitalo, Antonia and Mota-Silva, Eduarda and Riggs, Marion J. and Correia, Kevin and Elezi, Emanuela and Demelo, Brigitte and Carroll, Jeffrey B. and Gillis, Tammy and Gusella, James F. and MacDonald, Marcy E. and Wheeler, Vanessa C.},
  doi          = {10.1038/s41588-024-02054-5},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {314-322},
  shortjournal = {Nat. Genet.},
  title        = {In vivo CRISPR–Cas9 genome editing in mice identifies genetic modifiers of somatic CAG repeat instability in huntington’s disease},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mutagenesis and analysis of contrasting wheat lines do not support a role for PFT in fusarium head blight resistance. <em>NG</em>, <em>57</em>(2), 310-313. (<a href='https://doi.org/10.1038/s41588-024-02056-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ambiguity about whether the histidine-rich calcium-binding protein-coding gene (HisR) or the pore-forming toxin-like gene (PFT) or both are responsible for Fusarium head blight (FHB) resistance conferred by the Fhb1 quantitative trait locus hinders progress toward elucidating Fhb1 resistance mechanisms. Here, with a series of developed lines with or without PFT but all possessing HisR and five HisR-carrying PFT mutant lines created via gene editing, we show that PFT does not confer FHB resistance and that the HisR resistance effect does not require PFT in the tested conditions. We also show that PFT mutations are not associated with morphological and phenological characteristics that often affect FHB severity. Analysis of wheat lines generated through introgression or gene editing provides evidence that Fhb1-mediated resistance to Fusarium head blight is independent of the pore-forming toxin gene PFT.},
  archive      = {J_NG},
  author       = {Shi, Jinxing and Kong, Zhongxin and Zhong, Jinkun and Zhang, Xiaoqiu and Luo, Xianmin and Wang, Heyu and Xu, Boyang and Wang, Xin and Cheng, Rui and Yuan, Yang and Li, Na and Xie, Quan and Li, Guoqiang and Jia, Haiyan and Ma, Zhengqiang},
  doi          = {10.1038/s41588-024-02056-3},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {310-313},
  shortjournal = {Nat. Genet.},
  title        = {Mutagenesis and analysis of contrasting wheat lines do not support a role for PFT in fusarium head blight resistance},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Functional innovation through new genes as a general evolutionary process. <em>NG</em>, <em>57</em>(2), 295-309. (<a href='https://doi.org/10.1038/s41588-024-02059-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the past decade, our understanding of how new genes originate in diverse organisms has advanced substantially, and more than a dozen molecular mechanisms for generating initial gene structures were identified, in addition to gene duplication. These new genes have been found to integrate into and modify pre-existing gene networks primarily through mutation and selection, revealing new patterns and rules with stable origination rates across various organisms. This progress has challenged the prevailing belief that new proteins evolve from pre-existing genes, as new genes may arise de novo from noncoding DNA sequences in many organisms, with high rates observed in flowering plants. New genes have important roles in phenotypic and functional evolution across diverse biological processes and structures, with detectable fitness effects of sexual conflict genes that can shape species divergence. Such knowledge of new genes can be of translational value in agriculture and medicine. This Review discusses the various molecular mechanisms underlying the generation of new genes and highlights their important functions and phenotypes with an emphasis on the evolutionary forces underlying natural selection and sexual conflict.},
  archive      = {J_NG},
  author       = {Xia, Shengqian and Chen, Jianhai and Arsala, Deanna and Emerson, J. J. and Long, Manyuan},
  doi          = {10.1038/s41588-024-02059-0},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {295-309},
  shortjournal = {Nat. Genet.},
  title        = {Functional innovation through new genes as a general evolutionary process},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Overcoming challenges associated with broad sharing of human genomic data. <em>NG</em>, <em>57</em>(2), 287-294. (<a href='https://doi.org/10.1038/s41588-024-02049-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Since the Human Genome Project, the consensus position in genomics has been that data should be shared widely to achieve the greatest societal benefit. This position relies on imprecise definitions of the concept of ‘broad data sharing’. Accordingly, the implementation of data sharing varies among landmark genomic studies. In this Perspective, we identify definitions of broad that have been used interchangeably, despite their distinct implications. We further offer a framework with clarified concepts for genomic data sharing and probe six examples in genomics that produced public data. Finally, we articulate three challenges. First, we explore the need to reinterpret the limits of general research use data. Second, we consider the governance of public data deposition from extant samples. Third, we ask whether, in light of changing concepts of broad, participants should be encouraged to share their status as participants publicly or not. Each of these challenges is followed with recommendations. This Perspective discusses the definitions of ‘broad’ that have been used in the context of sharing of human genomic data and proposes a clarified and reformed terminology for describing genomic data sharing more precisely.},
  archive      = {J_NG},
  author       = {LoTempio, Jonathan E. and Moreno, Jonathan D.},
  doi          = {10.1038/s41588-024-02049-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {287-294},
  shortjournal = {Nat. Genet.},
  title        = {Overcoming challenges associated with broad sharing of human genomic data},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Oncofetal reprogramming induces phenotypic heterogeneity in colorectal cancer. <em>NG</em>, <em>57</em>(2), 285-286. (<a href='https://doi.org/10.1038/s41588-025-02092-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phenotypic plasticity of cancer cells is increasingly recognized as a mechanism of tumor escape from targeted therapies. Yet, the phenotypic heterogeneity of colorectal cancer remains poorly explored. Our study identifies oncofetal reprogramming of neoplastic stem cells, driven by the transcription factors YAP and AP-1, as a critical driver of phenotypic heterogeneity, lineage plasticity and therapy resistance. Targeting the oncofetal program enhances the efficacy and durability of current treatments.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-025-02092-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {285-286},
  shortjournal = {Nat. Genet.},
  title        = {Oncofetal reprogramming induces phenotypic heterogeneity in colorectal cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unveiling heterogeneity in rare cells by combining RNA-based sorting and scRNA-seq. <em>NG</em>, <em>57</em>(2), 283-284. (<a href='https://doi.org/10.1038/s41588-024-02073-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A study describes the development of PERFF-seq, a method to ‘FISH’ out rare cells using RNA markers, helping to solve the challenge in single-cell biology of studying cells that make up less than 1% of a sample.},
  archive      = {J_NG},
  author       = {Wahlsten, Madison and Shaffer, Sydney M.},
  doi          = {10.1038/s41588-024-02073-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {283-284},
  shortjournal = {Nat. Genet.},
  title        = {Unveiling heterogeneity in rare cells by combining RNA-based sorting and scRNA-seq},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Defining genes and pathways that modify huntingtin CAG repeat somatic instability in vivo. <em>NG</em>, <em>57</em>(2), 281-282. (<a href='https://doi.org/10.1038/s41588-024-02055-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A novel in vivo CRISPR screening platform identifies genetic modifiers of huntingtin CAG repeat somatic instability. These modifiers include known and novel genes that are promising therapeutic targets for Huntington’s disease.},
  archive      = {J_NG},
  author       = {Sathitloetsakun, Suphinya and Heiman, Myriam},
  doi          = {10.1038/s41588-024-02055-4},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {281-282},
  shortjournal = {Nat. Genet.},
  title        = {Defining genes and pathways that modify huntingtin CAG repeat somatic instability in vivo},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). DNA loop extrusion is asymmetric but can switch direction. <em>NG</em>, <em>57</em>(2), 280. (<a href='https://doi.org/10.1038/s41588-025-02102-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Anania, Chiara},
  doi          = {10.1038/s41588-025-02102-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {280},
  shortjournal = {Nat. Genet.},
  title        = {DNA loop extrusion is asymmetric but can switch direction},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering extrachromosomal DNA. <em>NG</em>, <em>57</em>(2), 280. (<a href='https://doi.org/10.1038/s41588-025-02103-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-025-02103-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {280},
  shortjournal = {Nat. Genet.},
  title        = {Engineering extrachromosomal DNA},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Germline mutations increasing pediatric cancer risk. <em>NG</em>, <em>57</em>(2), 280. (<a href='https://doi.org/10.1038/s41588-025-02104-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-025-02104-6},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {280},
  shortjournal = {Nat. Genet.},
  title        = {Germline mutations increasing pediatric cancer risk},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A model of heritable genome editing for complex traits. <em>NG</em>, <em>57</em>(2), 280. (<a href='https://doi.org/10.1038/s41588-025-02105-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-025-02105-5},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {280},
  shortjournal = {Nat. Genet.},
  title        = {A model of heritable genome editing for complex traits},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Introducing the global alliance for spatial technologies (GESTALT). <em>NG</em>, <em>57</em>(2), 275-279. (<a href='https://doi.org/10.1038/s41588-024-02066-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This Comment outlines the creation of the Global Alliance for Spatial Technologies (GESTALT), a collaborative initiative aimed at fostering the growth and standardization of spatial tissue profiling technologies. It explores the need for GESTALT, its community-driven structure and its goals, spanning from the immediate to the long term.},
  archive      = {J_NG},
  author       = {Plummer, Jasmine T. and Vlachos, Ioannis S. and Martelotto, Luciano G.},
  doi          = {10.1038/s41588-024-02066-1},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {275-279},
  shortjournal = {Nat. Genet.},
  title        = {Introducing the global alliance for spatial technologies (GESTALT)},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dissecting tumor transcriptional heterogeneity from single-cell RNA-seq data by generalized binary covariance decomposition. <em>NG</em>, <em>57</em>(1), 263-273. (<a href='https://doi.org/10.1038/s41588-024-01997-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Profiling tumors with single-cell RNA sequencing has the potential to identify recurrent patterns of transcription variation related to cancer progression, and to produce therapeutically relevant insights. However, strong intertumor heterogeneity can obscure more subtle patterns that are shared across tumors. Here we introduce a statistical method, generalized binary covariance decomposition (GBCD), to address this problem. We show that GBCD can decompose transcriptional heterogeneity into interpretable components—including patient-specific, dataset-specific and shared components relevant to disease subtypes—and that, in the presence of strong intertumor heterogeneity, it can produce more interpretable results than existing methods. Applied to data on pancreatic ductal adenocarcinoma, GBCD produced a refined characterization of existing tumor subtypes, and identified a gene expression program prognostic of poor survival independent of tumor stage and subtype. The gene expression program is enriched for genes involved in stress responses, and suggests a role for the integrated stress response in pancreatic ductal adenocarcinoma. Generalized binary covariance decomposition (GBCD) applies empirical Bayes matrix factorization to identify shared and sample-specific gene expression signatures in single-cell RNA sequencing data, and can more accurately capture inter- and intrasample heterogeneity than existing methods.},
  archive      = {J_NG},
  author       = {Liu, Yusha and Carbonetto, Peter and Willwerscheid, Jason and Oakes, Scott A. and Macleod, Kay F. and Stephens, Matthew},
  doi          = {10.1038/s41588-024-01997-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {263-273},
  shortjournal = {Nat. Genet.},
  title        = {Dissecting tumor transcriptional heterogeneity from single-cell RNA-seq data by generalized binary covariance decomposition},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tandem kinase proteins across the plant kingdom. <em>NG</em>, <em>57</em>(1), 254-262. (<a href='https://doi.org/10.1038/s41588-024-02032-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plant pathogens pose a continuous threat to global food production. Recent discoveries in plant immunity research unveiled a unique protein family characterized by an unusual resistance protein structure that combines two kinase domains. This study demonstrates the widespread occurrence of tandem kinase proteins (TKPs) across the plant kingdom. An examination of 104 plant species’ genomes uncovered 2,682 TKPs. The majority (95.6%) of these kinase domains are part of the receptor-like kinase–Pelle family, which is crucial for cell surface responses in plant immunity. Notably, 90% of TKPs comprise dual kinase domains, with over 50% being pseudokinases. Over 56% of these proteins harbor 127 different integrated domains, and over 47% include a transmembrane domain. TKP pseudokinases and/or integrated domains probably serve as decoys, engaging with pathogen effectors to trigger plant immunity. The TKP Atlas we created sheds light on the mechanisms of TKP convergent molecular evolution and potential function. This genomic analysis of tandem kinase proteins across 104 plant species highlights their mechanisms of convergent molecular evolution and potential roles in plant immunity.},
  archive      = {J_NG},
  author       = {Reveguk, Tamara and Fatiukha, Andrii and Potapenko, Evgenii and Reveguk, Ivan and Sela, Hanan and Klymiuk, Valentyna and Li, Yinghui and Pozniak, Curtis and Wicker, Thomas and Coaker, Gitta and Fahima, Tzion},
  doi          = {10.1038/s41588-024-02032-x},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {254-262},
  shortjournal = {Nat. Genet.},
  title        = {Tandem kinase proteins across the plant kingdom},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The highly allo-autopolyploid modern sugarcane genome and very recent allopolyploidization in saccharum. <em>NG</em>, <em>57</em>(1), 242-253. (<a href='https://doi.org/10.1038/s41588-024-02033-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern sugarcane, a highly allo-autopolyploid organism, has a very complex genome. In the present study, the karyotype and genome architecture of modern sugarcane were investigated, resulting in a genome assembly of 97 chromosomes (8.84 Gb). The allopolyploid genome was divided into subgenomes from Saccharum officinarum (Soh) and S. spontaneum (Ssh), with Soh dominance in the Saccharum hybrid (S. hybrid). Genome shock affected transcriptome dynamics during allopolyploidization. Analysis of an inbreeding population with 192 individuals revealed the underlying genetic basis of transgressive segregation. Population genomics of 310 Saccharum accessions clarified the breeding history of modern sugarcane. Using the haplotype-resolved S. hybrid genome as a reference, genome-wide association studies identified a potential candidate gene for sugar content from S. spontaneum. These findings illuminate the complex genome evolution of allopolyploids, offering opportunities for genomic enhancements and innovative breeding strategies for sugarcane. A haplotype-resolved genome of hybrid sugarcane cultivar XTT22 and population analyses of Saccharum accessions highlight the genome evolution of allopolyploids and provide opportunities for sugarcane breeding.},
  archive      = {J_NG},
  author       = {Zhang, Jisen and Qi, Yiying and Hua, Xiuting and Wang, Yongjun and Wang, Baiyu and Qi, Yongwen and Huang, Yumin and Yu, Zehuai and Gao, Ruiting and Zhang, Yixing and Wang, Tianyou and Wang, Yuhao and Mei, Jing and Zhang, Qing and Wang, Gang and Pan, Haoran and Li, Zhen and Li, Shuangyu and Liu, Jia and Qi, Nameng and Feng, Xiaoxi and Wu, Mingxing and Chen, Shuqi and Du, Cuicui and Li, Yihan and Xu, Yi and Fang, Yaxue and Ma, Panpan and Li, Qingyun and Sun, Yuanchang and Feng, Xiaomin and Yao, Wei and Zhang, Muqing and Chen, Baoshan and Liu, Xinlong and Ming, Ray and Wang, Jianping and Deng, Zuhu and Tang, Haibao},
  doi          = {10.1038/s41588-024-02033-w},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {242-253},
  shortjournal = {Nat. Genet.},
  title        = {The highly allo-autopolyploid modern sugarcane genome and very recent allopolyploidization in saccharum},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Repairing a deleterious domestication variant in a floral regulator gene of tomato by base editing. <em>NG</em>, <em>57</em>(1), 231-241. (<a href='https://doi.org/10.1038/s41588-024-02026-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Crop genomes accumulate deleterious mutations—a phenomenon known as the cost of domestication. Precision genome editing has been proposed to eliminate such potentially harmful mutations; however, experimental demonstration is lacking. Here we identified a deleterious mutation in the tomato transcription factor SUPPRESSOR OF SP2 (SSP2), which became prevalent in the domesticated germplasm and diminished DNA binding to genome-wide targets. We found that the action of SSP2 is partially redundant with that of its paralog SSP in regulating shoot and inflorescence architecture. However, redundancy was compromised during tomato domestication and lost completely in the closely related species Physalis grisea, in which a single ortholog regulates shoot branching. We applied base editing to directly repair the deleterious mutation in cultivated tomato and obtained plants with compact growth that provide an early fruit yield. Our work shows how deleterious variants have sensitized modern genotypes for phenotypic tuning and illustrates how repairing deleterious mutations with genome editing may allow predictable crop improvement. A deleterious mutation in the tomato transcription factor SSP2 was enriched during domestication. Repairing the deleterious mutation in cultivated tomato by base editing leads to compact growth and early fruit yield.},
  archive      = {J_NG},
  author       = {Glaus, Anna N. and Brechet, Marion and Swinnen, Gwen and Lebeigle, Ludivine and Iwaszkiewicz, Justyna and Ambrosini, Giovanna and Julca, Irene and Zhang, Jing and Roberts, Robyn and Iseli, Christian and Guex, Nicolas and Jiménez-Gómez, José and Glover, Natasha and Martin, Gregory B. and Strickler, Susan and Soyk, Sebastian},
  doi          = {10.1038/s41588-024-02026-9},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {231-241},
  shortjournal = {Nat. Genet.},
  title        = {Repairing a deleterious domestication variant in a floral regulator gene of tomato by base editing},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Telomere-to-telomere sheep genome assembly identifies variants associated with wool fineness. <em>NG</em>, <em>57</em>(1), 218-230. (<a href='https://doi.org/10.1038/s41588-024-02037-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ongoing efforts to improve sheep reference genome assemblies still leave many gaps and incomplete regions, resulting in a few common failures and errors in genomic studies. Here, we report a 2.85-Gb gap-free telomere-to-telomere genome of a ram (T2T-sheep1.0), including all autosomes and the X and Y chromosomes. This genome adds 220.05 Mb of previously unresolved regions and 754 new genes to the most updated reference assembly ARS-UI_Ramb_v3.0; it contains four types of repeat units (SatI, SatII, SatIII and CenY) in centromeric regions. T2T-sheep1.0 has a base accuracy of more than 99.999%, corrects several structural errors in previous reference assemblies and improves structural variant detection in repetitive sequences. Alignment of whole-genome short-read sequences of global domestic and wild sheep against T2T-sheep1.0 identifies 2,664,979 new single-nucleotide polymorphisms in previously unresolved regions, which improves the population genetic analyses and detection of selective signals for domestication (for example, ABCC4) and wool fineness (for example, FOXQ1). A gap-free telomere-to-telomere genome assembly for a ram of Hu sheep uncovers genes and variants associated with domestication and selection for wool fineness.},
  archive      = {J_NG},
  author       = {Luo, Ling-Yun and Wu, Hui and Zhao, Li-Ming and Zhang, Ya-Hui and Huang, Jia-Hui and Liu, Qiu-Yue and Wang, Hai-Tao and Mo, Dong-Xin and EEr, He-Hua and Zhang, Lian-Quan and Chen, Hai-Liang and Jia, Shan-Gang and Wang, Wei-Min and Li, Meng-Hua},
  doi          = {10.1038/s41588-024-02037-6},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {218-230},
  shortjournal = {Nat. Genet.},
  title        = {Telomere-to-telomere sheep genome assembly identifies variants associated with wool fineness},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gene regulation by convergent promoters. <em>NG</em>, <em>57</em>(1), 206-217. (<a href='https://doi.org/10.1038/s41588-024-02025-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Convergent transcription, that is, the collision of sense and antisense transcription, is ubiquitous in mammalian genomes and believed to diminish RNA expression. Recently, antisense transcription downstream of promoters was found to be surprisingly prevalent. However, functional characteristics of affected promoters are poorly investigated. Here we show that convergent transcription marks an unexpected positively co-regulated promoter constellation. By assessing transcriptional dynamic systems, we identified co-regulated constituent promoters connected through a distinct chromatin structure. Within these cis-regulatory domains, transcription factors can regulate both constituting promoters by binding to only one of them. Convergent promoters comprise about a quarter of all active transcript start sites and initiate 5′-overlapping antisense RNAs—an RNA class believed previously to be rare. Visualization of nascent RNA molecules reveals convergent cotranscription at these loci. Together, our results demonstrate that co-regulated convergent promoters substantially expand the cis-regulatory repertoire, reveal limitations of the transcription interference model and call for adjusting the promoter concept. Genome-wide analysis and genetic manipulation at loci regulated by p53, E2F4 and RFX7 show that convergent promoters with similar epigenetic features can be co-regulated and simultaneously expressed in the same direction.},
  archive      = {J_NG},
  author       = {Wiechens, Elina and Vigliotti, Flavia and Siniuk, Kanstantsin and Schwarz, Robert and Schwab, Katjana and Riege, Konstantin and van Bömmel, Alena and Görlich, Ivonne and Bens, Martin and Sahm, Arne and Groth, Marco and Sammons, Morgan A. and Loewer, Alexander and Hoffmann, Steve and Fischer, Martin},
  doi          = {10.1038/s41588-024-02025-w},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {206-217},
  shortjournal = {Nat. Genet.},
  title        = {Gene regulation by convergent promoters},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Coupling metabolomics and exome sequencing reveals graded effects of rare damaging heterozygous variants on gene function and human traits. <em>NG</em>, <em>57</em>(1), 193-205. (<a href='https://doi.org/10.1038/s41588-024-01965-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic studies of the metabolome can uncover enzymatic and transport processes shaping human metabolism. Using rare variant aggregation testing based on whole-exome sequencing data to detect genes associated with levels of 1,294 plasma and 1,396 urine metabolites, we discovered 235 gene–metabolite associations, many previously unreported. Complementary approaches (genetic, computational (in silico gene knockouts in whole-body models of human metabolism) and one experimental proof of principle) provided orthogonal evidence that studies of rare, damaging variants in the heterozygous state permit inferences concordant with those from inborn errors of metabolism. Allelic series of functional variants in transporters responsible for transcellular sulfate reabsorption (SLC13A1, SLC26A1) exhibited graded effects on plasma sulfate and human height and pinpointed alleles associated with increased odds of diverse musculoskeletal traits and diseases in the population. This integrative approach can identify new players in incompletely characterized human metabolic reactions and reveal metabolic readouts informative of human traits and diseases. Gene-based rare variant aggregation study with the levels of 1,294 plasma and 1,396 urine metabolites from paired specimens of 4,737 participants reveals graded effects of rare, putatively damaging variants on gene function and human traits.},
  archive      = {J_NG},
  author       = {Scherer, Nora and Fässler, Daniel and Borisov, Oleg and Cheng, Yurong and Schlosser, Pascal and Wuttke, Matthias and Haug, Stefan and Li, Yong and Telkämper, Fabian and Patil, Suraj and Meiselbach, Heike and Wong, Casper and Berger, Urs and Sekula, Peggy and Hoppmann, Anselm and Schultheiss, Ulla T. and Mozaffari, Sahar and Xi, Yannan and Graham, Robert and Schmidts, Miriam and Köttgen, Michael and Oefner, Peter J. and Knauf, Felix and Eckardt, Kai-Uwe and Grünert, Sarah C. and Estrada, Karol and Thiele, Ines and Hertel, Johannes and Köttgen, Anna},
  doi          = {10.1038/s41588-024-01965-7},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {193-205},
  shortjournal = {Nat. Genet.},
  title        = {Coupling metabolomics and exome sequencing reveals graded effects of rare damaging heterozygous variants on gene function and human traits},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Adipose tissue eQTL meta-analysis highlights the contribution of allelic heterogeneity to gene expression regulation and cardiometabolic traits. <em>NG</em>, <em>57</em>(1), 180-192. (<a href='https://doi.org/10.1038/s41588-024-01982-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Complete characterization of the genetic effects on gene expression is needed to elucidate tissue biology and the etiology of complex traits. In the present study, we analyzed 2,344 subcutaneous adipose tissue samples and identified 34,774 conditionally distinct expression quantitative trait locus (eQTL) signals at 18,476 genes. Over half of eQTL genes exhibited at least two eQTL signals. Compared with primary eQTL signals, nonprimary eQTL signals had lower effect sizes, lower minor allele frequencies and less promoter enrichment; they corresponded to genes with higher heritability and higher tolerance for loss of function. Colocalization of eQTLs with genome-wide association study (GWAS) signals for 28 cardiometabolic traits identified 1,835 genes. Inclusion of nonprimary eQTL signals increased discovery of colocalized GWAS–eQTL signals by 46%. Furthermore, 21 genes with ≥2 colocalized GWAS–eQTL signals showed a mediating gene dosage effect on the GWAS trait. Thus, expanded eQTL identification reveals more mechanisms underlying complex traits and improves understanding of the complexity of gene expression regulation. This study introduces AdipoExpress, an eQTL meta-analysis of 2,344 subcutaneous adipose tissue samples, which triples the size of previous studies and expands the discovery of eQTLs colocalized with GWAS signals for cardiometabolic traits.},
  archive      = {J_NG},
  author       = {Brotman, Sarah M. and El-Sayed Moustafa, Julia S. and Guan, Li and Broadaway, K. Alaine and Wang, Dongmeng and Jackson, Anne U. and Welch, Ryan and Currin, Kevin W. and Tomlinson, Max and Vadlamudi, Swarooparani and Stringham, Heather M. and Roberts, Amy L. and Lakka, Timo A. and Oravilahti, Anniina and Fernandes Silva, Lilian and Narisu, Narisu and Erdos, Michael R. and Yan, Tingfen and Bonnycastle, Lori L. and Raulerson, Chelsea K. and Raza, Yasrab and Yan, Xinyu and Parker, Stephen C. J. and Kuusisto, Johanna and Pajukanta, Päivi and Tuomilehto, Jaakko and Collins, Francis S. and Boehnke, Michael and Love, Michael I. and Koistinen, Heikki A. and Laakso, Markku and Mohlke, Karen L. and Small, Kerrin S. and Scott, Laura J.},
  doi          = {10.1038/s41588-024-01982-6},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {180-192},
  shortjournal = {Nat. Genet.},
  title        = {Adipose tissue eQTL meta-analysis highlights the contribution of allelic heterogeneity to gene expression regulation and cardiometabolic traits},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Loss of kmt2c or kmt2d primes urothelium for tumorigenesis and redistributes KMT2A–menin to bivalent promoters. <em>NG</em>, <em>57</em>(1), 165-179. (<a href='https://doi.org/10.1038/s41588-024-02015-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Members of the KMT2C/D–KDM6A complex are recurrently mutated in urothelial carcinoma and in histologically normal urothelium. Here, using genetically engineered mouse models, we demonstrate that Kmt2c/d knockout in the urothelium led to impaired differentiation, augmented responses to growth and inflammatory stimuli and sensitization to oncogenic transformation by carcinogen and oncogenes. Mechanistically, KMT2D localized to active enhancers and CpG-poor promoters that preferentially regulate the urothelial lineage program and Kmt2c/d knockout led to diminished H3K4me1, H3K27ac and nascent RNA transcription at these sites, which leads to impaired differentiation. Kmt2c/d knockout further led to KMT2A–menin redistribution from KMT2D localized enhancers to CpG-high and bivalent promoters, resulting in derepression of signal-induced immediate early genes. Therapeutically, Kmt2c/d knockout upregulated epidermal growth factor receptor signaling and conferred vulnerability to epidermal growth factor receptor inhibitors. Together, our data posit that functional loss of Kmt2c/d licenses a molecular ‘field effect’ priming histologically normal urothelium for oncogenic transformation and presents therapeutic vulnerabilities. Loss of Kmt2c or Kmt2d in mice drives the redeployment of KMT2A–menin to bivalent promoters, leading to changes in gene expression. This primes cells for transformation and elicits sensitivity to EGFR inhibitors.},
  archive      = {J_NG},
  author       = {Wang, Naitao and Pachai, Mohini R. and Li, Dan and Lee, Cindy J. and Warda, Sarah and Khudoynazarova, Makhzuna N. and Cho, Woo Hyun and Xie, Guojia and Shah, Sagar R. and Yao, Li and Qian, Cheng and Wong, Elissa W. P. and Yan, Juan and Tomas, Fanny V. and Hu, Wenhuo and Kuo, Fengshen and Gao, Sizhi P. and Luo, Jiaqian and Smith, Alison E. and Han, Ming and Gao, Dong and Ge, Kai and Yu, Haiyuan and Chandarlapaty, Sarat and Iyer, Gopakumar V. and Rosenberg, Jonathan E. and Solit, David B. and Al-Ahmadie, Hikmat A. and Chi, Ping and Chen, Yu},
  doi          = {10.1038/s41588-024-02015-y},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {165-179},
  shortjournal = {Nat. Genet.},
  title        = {Loss of kmt2c or kmt2d primes urothelium for tumorigenesis and redistributes KMT2A–menin to bivalent promoters},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A multilineage screen identifies actionable synthetic lethal interactions in human cancers. <em>NG</em>, <em>57</em>(1), 154-164. (<a href='https://doi.org/10.1038/s41588-024-01971-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancers are driven by alterations in diverse genes, creating dependencies that can be therapeutically targeted. However, many genetic dependencies have proven inconsistent across tumors. Here we describe SCHEMATIC, a strategy to identify a core network of highly penetrant, actionable genetic interactions. First, fundamental cellular processes are perturbed by systematic combinatorial knockouts across tumor lineages, identifying 1,805 synthetic lethal interactions (95% unreported). Interactions are then analyzed by hierarchical pooling, revealing that half segregate reliably by tissue type or biomarker status (51%) and a substantial minority are penetrant across lineages (34%). Interactions converge on 49 multigene systems, including MAPK signaling and BAF transcriptional regulatory complexes, which become essential on disruption of polymerases. Some 266 interactions translate to robust biomarkers of drug sensitivity, including frequent genetic alterations in the KDM5C/6A histone demethylases, which sensitize to inhibition of TIPARP (PARP7). SCHEMATIC offers a context-aware, data-driven approach to match genetic alterations to targeted therapies. A framework combining combinatorial CRISPR knockout in breast, lung and oropharyngeal cancer in vitro models with public data identifies synthetic lethal interactions, such as perturbed KDM5C sensitizing cells to PARP7 inhibition.},
  archive      = {J_NG},
  author       = {Fong, Samson H. and Kuenzi, Brent M. and Mattson, Nicole M. and Lee, John and Sanchez, Kyle and Bojorquez-Gomez, Ana and Ford, Kyle and Munson, Brenton P. and Licon, Katherine and Bergendahl, Sarah and Shen, John Paul and Kreisberg, Jason F. and Mali, Prashant and Hager, Jeffrey H. and White, Michael A. and Ideker, Trey},
  doi          = {10.1038/s41588-024-01971-9},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {154-164},
  shortjournal = {Nat. Genet.},
  title        = {A multilineage screen identifies actionable synthetic lethal interactions in human cancers},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Deep CRISPR mutagenesis characterizes the functional diversity of TP53 mutations. <em>NG</em>, <em>57</em>(1), 140-153. (<a href='https://doi.org/10.1038/s41588-024-02039-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mutational landscape of TP53, a tumor suppressor mutated in about half of all cancers, includes over 2,000 known missense mutations. To fully leverage TP53 mutation status for personalized medicine, a thorough understanding of the functional diversity of these mutations is essential. We conducted a deep mutational scan using saturation genome editing with CRISPR-mediated homology-directed repair to engineer 9,225 TP53 variants in cancer cells. This high-resolution approach, covering 94.5% of all cancer-associated TP53 missense mutations, precisely mapped the impact of individual mutations on tumor cell fitness, surpassing previous deep mutational scan studies in distinguishing benign from pathogenic variants. Our results revealed even subtle loss-of-function phenotypes and identified promising mutants for pharmacological reactivation. Moreover, we uncovered the roles of splicing alterations and nonsense-mediated messenger RNA decay in mutation-driven TP53 dysfunction. These findings underscore the power of saturation genome editing in advancing clinical TP53 variant interpretation for genetic counseling and personalized cancer therapy. A large-scale CRISPR-mediated deep mutational scanning approach is used to interrogate the function of mutations in the endogenous locus of TP53 mapping to the DNA-binding domain.},
  archive      = {J_NG},
  author       = {Funk, Julianne S. and Klimovich, Maria and Drangenstein, Daniel and Pielhoop, Ole and Hunold, Pascal and Borowek, Anna and Noeparast, Maxim and Pavlakis, Evangelos and Neumann, Michelle and Balourdas, Dimitrios-Ilias and Kochhan, Katharina and Merle, Nastasja and Bullwinkel, Imke and Wanzel, Michael and Elmshäuser, Sabrina and Teply-Szymanski, Julia and Nist, Andrea and Procida, Tara and Bartkuhn, Marek and Humpert, Katharina and Mernberger, Marco and Savai, Rajkumar and Soussi, Thierry and Joerger, Andreas C. and Stiewe, Thorsten},
  doi          = {10.1038/s41588-024-02039-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {140-153},
  shortjournal = {Nat. Genet.},
  title        = {Deep CRISPR mutagenesis characterizes the functional diversity of TP53 mutations},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multi-omic profiling highlights factors associated with resistance to immuno-chemotherapy in non-small-cell lung cancer. <em>NG</em>, <em>57</em>(1), 126-139. (<a href='https://doi.org/10.1038/s41588-024-01998-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although immune checkpoint blockade (ICB) therapies have shifted the treatment paradigm for non-small-cell lung cancer (NSCLC), many patients remain resistant. Here we characterize the tumor cell states and spatial cellular compositions of the NSCLC tumor microenvironment (TME) by analyzing single-cell transcriptomes of 232,080 cells and spatially resolved transcriptomes of tumors from 19 patients before and after ICB–chemotherapy. We find that tumor cells and secreted phosphoprotein 1-positive macrophages interact with collagen type XI alpha 1 chain-positive cancer-associated fibroblasts to stimulate the deposition and entanglement of collagen fibers at tumor boundaries, obstructing T cell infiltration and leading to poor prognosis. We also reveal distinct states of tertiary lymphoid structures (TLSs) in the TME. Activated TLSs are associated with improved prognosis, whereas a hypoxic microenvironment appears to suppress TLS development and is associated with poor prognosis. Our study provides novel insights into different cellular and molecular components corresponding to NSCLC ICB–chemotherapeutic responsiveness, which will benefit future individualized immuno-chemotherapy. Samples from 19 patients with non-small-cell lung cancer treated with a combination of chemotherapy and immune checkpoint blockade are profiled with single-cell RNA sequencing and spatial transcriptomics to identify factors associated with treatment resistance.},
  archive      = {J_NG},
  author       = {Yan, Yilv and Sun, Dongqing and Hu, Junjie and Chen, Yue and Sun, Liangdong and Yu, Huansha and Xiong, Yicheng and Huang, Zhida and Xia, Haoran and Zhu, Xinsheng and Bian, Dongliang and Sun, Fenghuan and Hou, Likun and Wu, Chunyan and Fan, Orion R. and Hu, Haiyang and Zeng, An and Zhang, Lele and Sun, Yi Eve and Wang, Chenfei and Zhang, Peng},
  doi          = {10.1038/s41588-024-01998-y},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {126-139},
  shortjournal = {Nat. Genet.},
  title        = {Multi-omic profiling highlights factors associated with resistance to immuno-chemotherapy in non-small-cell lung cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comprehensive genomic characterization of early-stage bladder cancer. <em>NG</em>, <em>57</em>(1), 115-125. (<a href='https://doi.org/10.1038/s41588-024-02030-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the molecular landscape of nonmuscle-invasive bladder cancer (NMIBC) is essential to improve risk assessment and treatment regimens. We performed a comprehensive genomic analysis of patients with NMIBC using whole-exome sequencing (n = 438), shallow whole-genome sequencing (n = 362) and total RNA sequencing (n = 414). A large genomic variation within NMIBC was observed and correlated with different molecular subtypes. Frequent loss of heterozygosity in FGFR3 and 17p (affecting TP53) was found in tumors with mutations in FGFR3 and TP53, respectively. Whole-genome doubling (WGD) was observed in 15% of the tumors and was associated with worse outcomes. Tumors with WGD were genomically unstable, with alterations in cell-cycle-related genes and an altered immune composition. Finally, integrative clustering of multi-omics data highlighted the important role of genomic instability and immune cell exhaustion in disease aggressiveness. These findings advance our understanding of genomic differences associated with disease aggressiveness in NMIBC and may ultimately improve patient stratification. A genomic and transcriptomic analysis of nonmuscle-invasive bladder cancer identifies four molecular subtypes, and associates whole-genome duplication and immune exhaustion with tumor progression.},
  archive      = {J_NG},
  author       = {Prip, Frederik and Lamy, Philippe and Lindskrog, Sia Viborg and Strandgaard, Trine and Nordentoft, Iver and Birkenkamp-Demtröder, Karin and Birkbak, Nicolai Juul and Kristjánsdóttir, Nanna and Kjær, Asbjørn and Andreasen, Tine G. and Ahrenfeldt, Johanne and Pedersen, Jakob Skou and Rasmussen, Asta Mannstaedt and Hermann, Gregers G. and Mogensen, Karin and Petersen, Astrid C. and Hartmann, Arndt and Grimm, Marc-Oliver and Horstmann, Marcus and Nawroth, Roman and Segersten, Ulrika and Sikic, Danijel and van Kessel, Kim E. M. and Zwarthoff, Ellen C. and Maurer, Tobias and Simic, Tatjana and Malmström, Per-Uno and Malats, Núria and Jensen, Jørgen Bjerggaard and Real, Francisco X. and Dyrskjøt, Lars},
  doi          = {10.1038/s41588-024-02030-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {115-125},
  shortjournal = {Nat. Genet.},
  title        = {Comprehensive genomic characterization of early-stage bladder cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER. <em>NG</em>, <em>57</em>(1), 103-114. (<a href='https://doi.org/10.1038/s41588-024-01989-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proliferation is a key hallmark of cancer, but whether it differs between evolutionarily distinct clones co-existing within a tumor is unknown. We introduce the Single-cell Proliferation Rate Inference in Non-homogeneous Tumors through Evolutionary Routes (SPRINTER) algorithm that uses single-cell whole-genome DNA sequencing data to enable accurate identification and clone assignment of S- and G2-phase cells, as assessed by generating accurate ground truth data. Applied to a newly generated longitudinal, primary-metastasis-matched dataset of 14,994 non-small cell lung cancer cells, SPRINTER revealed widespread clone proliferation heterogeneity, orthogonally supported by Ki-67 staining, nuclei imaging and clinical imaging. We further demonstrated that high-proliferation clones have increased metastatic seeding potential, increased circulating tumor DNA shedding and clone-specific altered replication timing in proliferation- or metastasis-related genes associated with expression changes. Applied to previously generated datasets of 61,914 breast and ovarian cancer cells, SPRINTER revealed increased single-cell rates of different genomic variants and enrichment of proliferation-related gene amplifications in high-proliferation clones. Single-cell Proliferation Rate Inference in Non-homogeneous Tumors through Evolutionary Routes (SPRINTER) allows users to infer proliferation rates of individual clones within a tumor from single-cell DNA sequencing data. Applying SPRINTER to human tumor datasets highlighted a link between proliferation and metastatic potential.},
  archive      = {J_NG},
  author       = {Lucas, Olivia and Ward, Sophia and Zaidi, Rija and Bunkum, Abigail and Frankell, Alexander M. and Moore, David A. and Hill, Mark S. and Liu, Wing Kin and Marinelli, Daniele and Lim, Emilia L. and Hessey, Sonya and Naceur-Lombardelli, Cristina and Rowan, Andrew and Purewal-Mann, Sukhveer Kaur and Zhai, Haoran and Dietzen, Michelle and Ding, Boyue and Royle, Gary and Aparicio, Samuel and McGranahan, Nicholas and Jamal-Hanjani, Mariam and Kanu, Nnennaya and Swanton, Charles and Zaccaria, Simone},
  doi          = {10.1038/s41588-024-01989-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {103-114},
  shortjournal = {Nat. Genet.},
  title        = {Characterizing the evolutionary dynamics of cancer proliferation in single-cell clones with SPRINTER},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ZIC1 is a context-dependent medulloblastoma driver in the rhombic lip. <em>NG</em>, <em>57</em>(1), 88-102. (<a href='https://doi.org/10.1038/s41588-024-02014-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcription factors are frequent cancer driver genes, exhibiting noted specificity based on the precise cell of origin. We demonstrate that ZIC1 exhibits loss-of-function (LOF) somatic events in group 4 (G4) medulloblastoma through recurrent point mutations, subchromosomal deletions and mono-allelic epigenetic repression (60% of G4 medulloblastoma). In contrast, highly similar SHH medulloblastoma exhibits distinct and diametrically opposed gain-of-function mutations and copy number gains (20% of SHH medulloblastoma). Overexpression of ZIC1 suppresses the growth of group 3 medulloblastoma models, whereas it promotes the proliferation of SHH medulloblastoma precursor cells. SHH medulloblastoma ZIC1 mutants show increased activity versus wild-type ZIC1, whereas G4 medulloblastoma ZIC1 mutants exhibit LOF phenotypes. Distinct ZIC1 mutations affect cells of the rhombic lip in diametrically opposed ways, suggesting that ZIC1 is a critical developmental transcriptional regulator in both the normal and transformed rhombic lip and identifying ZIC1 as an exquisitely context-dependent driver gene in medulloblastoma. Analysis of medulloblastomas in humans and mice shows that the functional consequences of ZIC1 mutations are exquisitely dependent on the cells of origin that give rise to different subgroups of medulloblastoma.},
  archive      = {J_NG},
  author       = {Lee, John J. Y. and Tao, Ran and You, Zhen and Haldipur, Parthiv and Erickson, Anders W. and Farooq, Hamza and Hendriske, Liam D. and Abeysundara, Namal and Richman, Cory M. and Wang, Evan Y. and Das Gupta, Neha and Hadley, Jennifer and Batts, Melissa and Mount, Christopher W. and Wu, Xiaochong and Rasnitsyn, Alex and Bailey, Swneke and Cavalli, Florence M. G. and Morrissy, Sorana and Garzia, Livia and Michealraj, Kulandaimanuvel Antony and Visvanathan, Abhi and Fong, Vernon and Palotta, Jonelle and Suarez, Raul and Livingston, Bryn G. and Liu, Miao and Luu, Betty and Daniels, Craig and Loukides, James and Bendel, Anne and French, Pim J. and Kros, Johan M. and Korshunov, Andrey and Kool, Marcel and Chico Ponce de León, Fernando and Perezpeña-Diazconti, Mario and Lach, Boleslaw and Singh, Sheila K. and Leary, Sarah E. S. and Cho, Byung-Kyu and Kim, Seung-Ki and Wang, Kyu-Chang and Lee, Ji-Yeoun and Tominaga, Teiji and Weiss, William A. and Phillips, Joanna J. and Dai, Shizhong and Zadeh, Gelareh and Saad, Ali G. and Bognár, László and Klekner, Almos and Pollack, Ian F. and Hamilton, Ronald L. and Ra, Young-shin and Grajkowska, Wieslawa A. and Perek-Polnik, Marta and Thompson, Reid C. and Kenney, Anna M. and Cooper, Michael K. and Mack, Stephen C. and Jabado, Nada and Lupien, Mathieu and Gallo, Marco and Ramaswamy, Vijay and Suva, Mario L. and Suzuki, Hiromichi and Millen, Kathleen J. and Huang, L. Frank and Northcott, Paul A. and Taylor, Michael D.},
  doi          = {10.1038/s41588-024-02014-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {88-102},
  shortjournal = {Nat. Genet.},
  title        = {ZIC1 is a context-dependent medulloblastoma driver in the rhombic lip},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Refining breast cancer genetic risk and biology through multi-ancestry fine-mapping analyses of 192 risk regions. <em>NG</em>, <em>57</em>(1), 80-87. (<a href='https://doi.org/10.1038/s41588-024-02031-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies have identified approximately 200 genetic risk loci for breast cancer, but the causal variants and target genes are mostly unknown. We sought to fine-map all known breast cancer risk loci using genome-wide association study data from 172,737 female breast cancer cases and 242,009 controls of African, Asian and European ancestry. We identified 332 independent association signals for breast cancer risk, including 131 signals not reported previously, and for 50 of them, we narrowed the credible causal variants down to a single variant. Analyses integrating functional genomics data identified 195 putative susceptibility genes, enriched in PI3K/AKT, TNF/NF-κB, p53 and Wnt/β-catenin pathways. Single-cell RNA sequencing or in vitro experiment data provided additional functional evidence for 105 genes. Our study uncovered large numbers of association signals and candidate susceptibility genes for breast cancer, uncovered breast cancer genetics and biology, and supported the value of including multi-ancestry data in fine-mapping analyses. Multi-ancestry fine-mapping of breast cancer susceptibility regions identifies candidate causal variants and prioritizes likely effector genes supported by functional genomic evidence.},
  archive      = {J_NG},
  author       = {Jia, Guochong and Chen, Zhishan and Ping, Jie and Cai, Qiuyin and Tao, Ran and Li, Chao and Bauer, Joshua A. and Xie, Yuhan and Ambs, Stefan and Barnard, Mollie E. and Chen, Yu and Choi, Ji-Yeob and Gao, Yu-Tang and Garcia-Closas, Montserrat and Gu, Jian and Hu, Jennifer J. and Iwasaki, Motoki and John, Esther M. and Kweon, Sun-Seog and Li, Christopher I. and Matsuda, Koichi and Matsuo, Keitaro and Nathanson, Katherine L. and Nemesure, Barbara and Olopade, Olufunmilayo I. and Pal, Tuya and Park, Sue K. and Park, Boyoung and Press, Michael F. and Sanderson, Maureen and Sandler, Dale P. and Shen, Chen-Yang and Troester, Melissa A. and Yao, Song and Zheng, Ying and Ahearn, Thomas and Brewster, Abenaa M. and Falusi, Adeyinka and Hennis, Anselm J. M. and Ito, Hidemi and Kubo, Michiaki and Lee, Eun-Sook and Makumbi, Timothy and Ndom, Paul and Noh, Dong-Young and O’Brien, Katie M. and Ojengbede, Oladosu and Olshan, Andrew F. and Park, Min-Ho and Reid, Sonya and Yamaji, Taiki and Zirpoli, Gary and Butler, Ebonee N. and Huang, Maosheng and Low, Siew-Kee and Obafunwa, John and Weinberg, Clarice R. and Zhang, Haoyu and Zhao, Hongyu and Cote, Michelle L. and Ambrosone, Christine B. and Huo, Dezheng and Li, Bingshan and Kang, Daehee and Palmer, Julie R. and Shu, Xiao-Ou and Haiman, Christopher A. and Guo, Xingyi and Long, Jirong and Zheng, Wei},
  doi          = {10.1038/s41588-024-02031-y},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {80-87},
  shortjournal = {Nat. Genet.},
  title        = {Refining breast cancer genetic risk and biology through multi-ancestry fine-mapping analyses of 192 risk regions},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Blood DNA virome associates with autoimmune diseases and COVID-19. <em>NG</em>, <em>57</em>(1), 65-79. (<a href='https://doi.org/10.1038/s41588-024-02022-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aberrant immune responses to viral pathogens contribute to pathogenesis, but our understanding of pathological immune responses caused by viruses within the human virome, especially at a population scale, remains limited. We analyzed whole-genome sequencing datasets of 6,321 Japanese individuals, including patients with autoimmune diseases (psoriasis vulgaris, rheumatoid arthritis (RA), systemic lupus erythematosus (SLE), pulmonary alveolar proteinosis (PAP) or multiple sclerosis) and coronavirus disease 2019 (COVID-19), or healthy controls. We systematically quantified two constituents of the blood DNA virome, endogenous HHV-6 (eHHV-6) and anellovirus. Participants with eHHV-6B had higher risks of SLE and PAP; the former was validated in All of Us. eHHV-6B-positivity and high SLE disease activity index scores had strong correlations. Genome-wide association study and long-read sequencing mapped the integration of the HHV-6B genome to a locus on chromosome 22q. Epitope mapping and single-cell RNA sequencing revealed distinctive immune induction by eHHV-6B in patients with SLE. In addition, high anellovirus load correlated strongly with SLE, RA and COVID-19 status. Our analyses unveil relationships between the human virome and autoimmune and infectious diseases. Analysis of the blood DNA virome in patients with COVID-19 and autoimmune disease associates endogenous HHV-6 (eHHV-6) and high anellovirus load with increased disease risk, most notably for systemic lupus erythematosus. eHHV-6 carriers show a distinct immune response.},
  archive      = {J_NG},
  author       = {Sasa, Noah and Kojima, Shohei and Koide, Rie and Hasegawa, Takanori and Namkoong, Ho and Hirota, Tomomitsu and Watanabe, Rei and Nakamura, Yuumi and Oguro-Igashira, Eri and Ogawa, Kotaro and Yata, Tomohiro and Sonehara, Kyuto and Yamamoto, Kenichi and Kishikawa, Toshihiro and Sakaue, Saori and Edahiro, Ryuya and Shirai, Yuya and Maeda, Yuichi and Nii, Takuro and Chubachi, Shotaro and Tanaka, Hiromu and Yabukami, Haruka and Suzuki, Akari and Nakajima, Kimiko and Arase, Noriko and Okamoto, Takashi and Nishikawa, Rika and Namba, Shinichi and Naito, Tatsuhiko and Miyagawa, Ippei and Tanaka, Hiroaki and Ueno, Masanobu and Ishitsuka, Yosuke and Furuta, Junichi and Kunimoto, Kayo and Kajihara, Ikko and Fukushima, Satoshi and Miyachi, Hideaki and Matsue, Hiroyuki and Kamata, Masahiro and Momose, Mami and Bito, Toshinori and Nagai, Hiroshi and Ikeda, Tetsuya and Horikawa, Tatsuya and Adachi, Atsuko and Matsubara, Tsukasa and Ikumi, Kyoko and Nishida, Emi and Nakagawa, Ikuma and Yagita-Sakamaki, Mayu and Yoshimura, Maiko and Ohshima, Shiro and Kinoshita, Makoto and Ito, Satoru and Arai, Toru and Hirose, Masaki and Tanino, Yoshinori and Nikaido, Takefumi and Ichiwata, Toshio and Ohkouchi, Shinya and Hirano, Taizou and Takada, Toshinori and Tazawa, Ryushi and Morimoto, Konosuke and Takaki, Masahiro and Konno, Satoshi and Suzuki, Masaru and Tomii, Keisuke and Nakagawa, Atsushi and Handa, Tomohiro and Tanizawa, Kiminobu and Ishii, Haruyuki and Ishida, Manabu and Kato, Toshiyuki and Takeda, Naoya and Yokomura, Koshi and Matsui, Takashi and Uchida, Akifumi and Inoue, Hiromasa and Imaizumi, Kazuyoshi and Goto, Yasuhiro and Kida, Hiroshi and Fujisawa, Tomoyuki and Suda, Takafumi and Yamada, Takashi and Satake, Yasuomi and Ibata, Hidenori and Saigusa, Mika and Shirai, Toshihiro and Hizawa, Nobuyuki and Nakata, Koh and Imafuku, Shinichi and Tada, Yayoi and Asano, Yoshihide and Sato, Shinichi and Nishigori, Chikako and Jinnin, Masatoshi and Ihn, Hironobu and Asahina, Akihiko and Saeki, Hidehisa and Kawamura, Tatsuyoshi and Shimada, Shinji and Katayama, Ichiro and Poisner, Hannah M. and Mack, Taralynn M. and Bick, Alexander G. and Higasa, Koichiro and Okuno, Tatsusada and Mochizuki, Hideki and Ishii, Makoto and Koike, Ryuji and Kimura, Akinori and Noguchi, Emiko and Sano, Shigetoshi and Inohara, Hidenori and Fujimoto, Manabu and Inoue, Yoshikazu and Yamaguchi, Etsuro and Ogawa, Seishi and Kanai, Takanori and Morita, Akimichi and Matsuda, Fumihiko and Tamari, Mayumi and Kumanogoh, Atsushi and Tanaka, Yoshiya and Ohmura, Koichiro and Fukunaga, Koichi and Imoto, Seiya and Miyano, Satoru and Parrish, Nicholas F. and Okada, Yukinori},
  doi          = {10.1038/s41588-024-02022-z},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {65-79},
  shortjournal = {Nat. Genet.},
  title        = {Blood DNA virome associates with autoimmune diseases and COVID-19},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The impact of common and rare genetic variants on bradyarrhythmia development. <em>NG</em>, <em>57</em>(1), 53-64. (<a href='https://doi.org/10.1038/s41588-024-01978-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To broaden our understanding of bradyarrhythmias and conduction disease, we performed common variant genome-wide association analyses in up to 1.3 million individuals and rare variant burden testing in 460,000 individuals for sinus node dysfunction (SND), distal conduction disease (DCD) and pacemaker (PM) implantation. We identified 13, 31 and 21 common variant loci for SND, DCD and PM, respectively. Four well-known loci (SCN5A/SCN10A, CCDC141, TBX20 and CAMK2D) were shared for SND and DCD, while others were more specific for SND or DCD. SND and DCD showed a moderate genetic correlation (rg = 0.63). Cardiomyocyte-expressed genes were enriched for contributions to DCD heritability. Rare-variant analyses implicated LMNA for all bradyarrhythmia phenotypes, SMAD6 and SCN5A for DCD and TTN, MYBPC3 and SCN5A for PM. These results show that variation in multiple genetic pathways (for example, ion channel function, cardiac developmental programs, sarcomeric structure and cellular homeostasis) appear critical to the development of bradyarrhythmias. Genome-wide analyses identify variants associated with sinus node dysfunction, distal conduction disease and pacemaker implantation, implicating ion channel function, cardiac developmental programs and sarcomeric structure in bradyarrhythmia susceptibility.},
  archive      = {J_NG},
  author       = {Weng, Lu-Chen and Rämö, Joel T. and Jurgens, Sean J. and Khurshid, Shaan and Chaffin, Mark and Hall, Amelia Weber and Morrill, Valerie N. and Wang, Xin and Nauffal, Victor and Sun, Yan V. and Beer, Dominik and Lee, Simon and Nadkarni, Girish N. and Duong, ThuyVy and Wang, Biqi and Czuba, Tomasz and Austin, Thomas R. and Yoneda, Zachary T. and Friedman, Daniel J. and Clayton, Anne and Hyman, Matthew C. and Judy, Renae L. and Skanes, Allan C. and Orland, Kate M. and Treu, Timothy M. and Oetjens, Matthew T. and Alonso, Alvaro and Soliman, Elsayed Z. and Lin, Honghuang and Lunetta, Kathryn L. and van der Pals, Jesper and Issa, Tariq Z. and Nafissi, Navid A. and May, Heidi T. and Leong-Sit, Peter and Roselli, Carolina and Choi, Seung Hoan and Khan, Habib R. and Knight, Stacey and Karlsson Linnér, Richard and Bezzina, Connie R. and Ripatti, Samuli and Heckbert, Susan R. and Gaziano, J. Michael and Loos, Ruth J. F. and Psaty, Bruce M. and Smith, J. Gustav and Benjamin, Emelia J. and Arking, Dan E. and Rader, Daniel J. and Shah, Svati H. and Roden, Dan M. and Damrauer, Scott M. and Eckhardt, Lee L. and Roberts, Jason D. and Cutler, Michael J. and Shoemaker, M. Benjamin and Haggerty, Christopher M. and Cho, Kelly and Palotie, Aarno and Wilson, Peter W. F. and Ellinor, Patrick T. and Lubitz, Steven A.},
  doi          = {10.1038/s41588-024-01978-2},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {53-64},
  shortjournal = {Nat. Genet.},
  title        = {The impact of common and rare genetic variants on bradyarrhythmia development},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fine-mapping causal tissues and genes at disease-associated loci. <em>NG</em>, <em>57</em>(1), 42-52. (<a href='https://doi.org/10.1038/s41588-024-01994-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Complex diseases often have distinct mechanisms spanning multiple tissues. We propose tissue–gene fine-mapping (TGFM), which infers the posterior inclusion probability (PIP) for each gene–tissue pair to mediate a disease locus by analyzing summary statistics and expression quantitative trait loci (eQTL) data; TGFM also assigns PIPs to non-mediated variants. TGFM accounts for co-regulation across genes and tissues and models uncertainty in cis-predicted expression models, enabling correct calibration. We applied TGFM to 45 UK Biobank diseases or traits using eQTL data from 38 Genotype–Tissue Expression (GTEx) tissues. TGFM identified an average of 147 PIP &gt; 0.5 causal genetic elements per disease or trait, of which 11% were gene–tissue pairs. Causal gene–tissue pairs identified by TGFM reflected both known biology (for example, TPO–thyroid for hypothyroidism) and biologically plausible findings (for example, SLC20A2–artery aorta for diastolic blood pressure). Application of TGFM to single-cell eQTL data from nine cell types in peripheral blood mononuclear cells (PBMCs), analyzed jointly with GTEx tissues, identified 30 additional causal gene–PBMC cell type pairs. Tissue–gene fine-mapping (TGFM) generalizes the SuSiE method to fine-map causal tissues and genes at disease loci using external eQTL data, offering improved calibration owing to modeling of cis-predicted expression uncertainty.},
  archive      = {J_NG},
  author       = {Strober, Benjamin J. and Zhang, Martin Jinye and Amariuta, Tiffany and Rossen, Jordan and Price, Alkes L.},
  doi          = {10.1038/s41588-024-01994-2},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {42-52},
  shortjournal = {Nat. Genet.},
  title        = {Fine-mapping causal tissues and genes at disease-associated loci},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Profiling the epigenome using long-read sequencing. <em>NG</em>, <em>57</em>(1), 27-41. (<a href='https://doi.org/10.1038/s41588-024-02038-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The advent of single-molecule, long-read sequencing (LRS) technologies by Oxford Nanopore Technologies and Pacific Biosciences has revolutionized genomics, transcriptomics and, more recently, epigenomics research. These technologies offer distinct advantages, including the direct detection of methylated DNA and simultaneous assessment of DNA sequences spanning multiple kilobases along with their modifications at the single-molecule level. This has enabled the development of new assays for analyzing chromatin states and made it possible to integrate data for DNA methylation, chromatin accessibility, transcription factor binding and histone modifications, thereby facilitating comprehensive epigenomic profiling. Owing to recent advancements, alternative, nascent and translating transcripts can be detected using LRS approaches. This Review discusses LRS-based experimental and computational strategies for characterizing chromatin states and highlights their advantages over short-read sequencing methods. Furthermore, we demonstrate how various long-read methods can be integrated to design multi-omics studies to investigate the relationship between chromatin states and transcriptional dynamics. Long-read sequencing technologies have revolutionized genomics and transcriptomics, and more recently enabled comprehensive epigenomic profiling. These advances now also allow investigation of the relationship between chromatin states and transcriptional dynamics.},
  archive      = {J_NG},
  author       = {Liu, Tianyuan and Conesa, Ana},
  doi          = {10.1038/s41588-024-02038-5},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {27-41},
  shortjournal = {Nat. Genet.},
  title        = {Profiling the epigenome using long-read sequencing},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pangenome graphs and their applications in biodiversity genomics. <em>NG</em>, <em>57</em>(1), 13-26. (<a href='https://doi.org/10.1038/s41588-024-02029-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Complete datasets of genetic variants are key to biodiversity genomic studies. Long-read sequencing technologies allow the routine assembly of highly contiguous, haplotype-resolved reference genomes. However, even when complete, reference genomes from a single individual may bias downstream analyses and fail to adequately represent genetic diversity within a population or species. Pangenome graphs assembled from aligned collections of high-quality genomes can overcome representation bias by integrating sequence information from multiple genomes from the same population, species or genus into a single reference. Here, we review the available tools and data structures to build, visualize and manipulate pangenome graphs while providing practical examples and discussing their applications in biodiversity and conservation genomics across the tree of life. Pangenomes integrate multiple genomes to mitigate reference bias. This Review presents tools to build, visualize and manipulate pangenome graphs and also highlights pangenome applications in biodiversity and conservation genomics.},
  archive      = {J_NG},
  author       = {Secomandi, Simona and Gallo, Guido Roberto and Rossi, Riccardo and Rodríguez Fernandes, Carlos and Jarvis, Erich D. and Bonisoli-Alquati, Andrea and Gianfranceschi, Luca and Formenti, Giulio},
  doi          = {10.1038/s41588-024-02029-6},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {13-26},
  shortjournal = {Nat. Genet.},
  title        = {Pangenome graphs and their applications in biodiversity genomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Purging deleterious mutations through precision genome editing for crop improvement. <em>NG</em>, <em>57</em>(1), 11-12. (<a href='https://doi.org/10.1038/s41588-024-02027-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Crop domestication led to the accumulation of mutations that are deleterious for gene function. We predicted the extent of deleterious mutations during the domestication of tomato and applied precision genome editing to correct a floral regulator, resulting in desirable crop attributes, with implications for crop improvement and breeding.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-02027-8},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {11-12},
  shortjournal = {Nat. Genet.},
  title        = {Purging deleterious mutations through precision genome editing for crop improvement},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A blueprint to discovering synthetic lethal gene interactions for precision oncology. <em>NG</em>, <em>57</em>(1), 9-10. (<a href='https://doi.org/10.1038/s41588-024-02024-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The SCHEMATIC resource combines CRISPR pairwise gene knockout experiments across tumor cell types with large-scale drug sensitivity assays to identify a core network of highly penetrant, synthetic lethal genetic interactions that can help to match individuals with cancer to targeted therapies.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-024-02024-x},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {9-10},
  shortjournal = {Nat. Genet.},
  title        = {A blueprint to discovering synthetic lethal gene interactions for precision oncology},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spatial biology captures the effects of neoadjuvant chemo-immunotherapy in lung cancer. <em>NG</em>, <em>57</em>(1), 6-8. (<a href='https://doi.org/10.1038/s41588-024-01992-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {PD-1 pathway blockade in combination with chemotherapy has emerged as a treatment paradigm for patients with resectable lung cancer, but insights into predictive biomarkers and mechanisms of immune responses are lacking. A study uses spatial transcriptomic methods to identify patterns within the tumor microenvironment associated with response.},
  archive      = {J_NG},
  author       = {Chen, Jonathan H. and Gainor, Justin F.},
  doi          = {10.1038/s41588-024-01992-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {6-8},
  shortjournal = {Nat. Genet.},
  title        = {Spatial biology captures the effects of neoadjuvant chemo-immunotherapy in lung cancer},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Single-cell genomics breaks new ground in cell cycle detection. <em>NG</em>, <em>57</em>(1), 3-5. (<a href='https://doi.org/10.1038/s41588-024-01987-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A novel method for analyzing single-cell genomics enables direct inference of cell cycle and proliferation status, highlighting the diversity of proliferation rates in clonal cancer. This approach opens a new avenue for high-resolution exploration of the role of proliferation in cancer evolution at the single-cell level.},
  archive      = {J_NG},
  author       = {Du, Qian},
  doi          = {10.1038/s41588-024-01987-1},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {3-5},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell genomics breaks new ground in cell cycle detection},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mutations in healthy breast tissue. <em>NG</em>, <em>57</em>(1), 2. (<a href='https://doi.org/10.1038/s41588-024-02060-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-024-02060-7},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {2},
  shortjournal = {Nat. Genet.},
  title        = {Mutations in healthy breast tissue},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Brain metastasis prediction. <em>NG</em>, <em>57</em>(1), 2. (<a href='https://doi.org/10.1038/s41588-024-02061-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-024-02061-6},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {2},
  shortjournal = {Nat. Genet.},
  title        = {Brain metastasis prediction},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nucleotide-resolution DNA foundation models of prokaryotic genomes. <em>NG</em>, <em>57</em>(1), 2. (<a href='https://doi.org/10.1038/s41588-024-02062-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-024-02062-5},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {2},
  shortjournal = {Nat. Genet.},
  title        = {Nucleotide-resolution DNA foundation models of prokaryotic genomes},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Behavioral insights from single-nucleus neuronal transcriptomics. <em>NG</em>, <em>57</em>(1), 2. (<a href='https://doi.org/10.1038/s41588-024-02063-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gross, Petra},
  doi          = {10.1038/s41588-024-02063-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {2},
  shortjournal = {Nat. Genet.},
  title        = {Behavioral insights from single-nucleus neuronal transcriptomics},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Emil skamene (1941–2024). <em>NG</em>, <em>57</em>(1), 1. (<a href='https://doi.org/10.1038/s41588-024-02028-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gros, Philippe and Schurr, Erwin},
  doi          = {10.1038/s41588-024-02028-7},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Genet.},
  title        = {Emil skamene (1941–2024)},
  volume       = {57},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
