<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NBT</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nbt">NBT - 318</h2>
<ul>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(9), 1580. (<a href='https://doi.org/10.1038/s41587-025-02804-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02804-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1580},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chronic silencing is a critical barrier to breaking the cycle of bullying in academia and industry. <em>NBT</em>, <em>43</em>(9), 1577-1579. (<a href='https://doi.org/10.1038/s41587-025-02803-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Breaking the cycle of institutional betrayal and the culture of silence requires decisive actions to promote transparency, accountability and supportive environments in both academia and industry.},
  archive      = {J_NBT},
  author       = {Vogelaar, Alison E. and Mahmoudi, Morteza},
  doi          = {10.1038/s41587-025-02803-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1577-1579},
  shortjournal = {Nature Biotech.},
  title        = {Chronic silencing is a critical barrier to breaking the cycle of bullying in academia and industry},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Global profiling of protein complex dynamics with an experimental library of protein interaction markers. <em>NBT</em>, <em>43</em>(9), 1562-1576. (<a href='https://doi.org/10.1038/s41587-024-02432-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to systematically monitor protein complex dynamics are needed. We introduce serial ultrafiltration combined with limited proteolysis-coupled mass spectrometry (FLiP–MS), a structural proteomics workflow that generates a library of peptide markers specific to changes in PPIs by probing differences in protease susceptibility between complex-bound and monomeric forms of proteins. The library includes markers mapping to protein-binding interfaces and markers reporting on structural changes that accompany PPI changes. Integrating the marker library with LiP–MS data allows for global profiling of protein–protein interactions (PPIs) from unfractionated lysates. We apply FLiP–MS to Saccharomyces cerevisiae and probe changes in protein complex dynamics after DNA replication stress, identifying links between Spt-Ada-Gcn5 acetyltransferase activity and the assembly state of several complexes. FLiP–MS enables protein complex dynamics to be probed on any perturbation, proteome-wide, at high throughput, with peptide-level structural resolution and informing on occupancy of binding interfaces, thus providing both global and molecular views of a system under study. FLiP–MS uses a library of protein–protein interaction markers to understand protein complex dynamics.},
  archive      = {J_NBT},
  author       = {Dörig, Christian and Marulli, Cathy and Peskett, Thomas and Volkmar, Norbert and Pantolini, Lorenzo and Studer, Gabriel and Paleari, Camilla and Frommelt, Fabian and Schwede, Torsten and de Souza, Natalie and Barral, Yves and Picotti, Paola},
  doi          = {10.1038/s41587-024-02432-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1562-1576},
  shortjournal = {Nature Biotech.},
  title        = {Global profiling of protein complex dynamics with an experimental library of protein interaction markers},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A circularly permuted CasRx platform for efficient, site-specific RNA editing. <em>NBT</em>, <em>43</em>(9), 1548-1561. (<a href='https://doi.org/10.1038/s41587-024-02430-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inactive Cas13 orthologs have been fused to a mutant human ADAR2 deaminase domain at the C terminus to enable programmable adenosine-to-inosine (A-to-I) RNA editing in selected transcripts. Although promising, existing RNA-editing tools generally suffer from a trade-off between efficacy and specificity, and off-target editing remains an unsolved problem. Here we describe the development of an optimized RNA-editing platform by rational protein engineering, CasRx-based Programmable Editing of RNA Technology (xPERT). We demonstrate that the topological rearrangement of a CasRx K940L mutant by circular permutation results in a robust scaffold for the tethering of a deaminase domain. We benchmark our tool against the REPAIR system and show that xPERT exhibits strong on-target activity like REPAIRv1 but low off-target editing like REPAIRv2. Our xPERT platform can be used to alter RNA sequence information without risking genome damage, effect temporary cellular changes and customize protein function. Site-specific and efficient RNA editing is achieved with a CasRx editor.},
  archive      = {J_NBT},
  author       = {Wang, Yuanming and Liu, Kaiwen Ivy and Liu, Mengying Mandy and Ooi, Kean Hean and Nguyen, Tram Anh and Chee, Jiunn En and Teo, Shun Xiang Danny and He, Shan and Tay, Jie Wen Douglas and Teo, Seok Yee and Liew, Kai Shin and Ge, Xiao Yu and Ng, Zhi Jian and Avagyan, Hasmik and Liu, Hao and Yi, Zirong and Chang, Keziah and Kok, Eng Piew Louis and Chen, Runjia and Yau, Chun En and Koh, Jun Wei and Wan, Yue and Tan, Meng How},
  doi          = {10.1038/s41587-024-02430-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1548-1561},
  shortjournal = {Nature Biotech.},
  title        = {A circularly permuted CasRx platform for efficient, site-specific RNA editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). One-step nanoscale expansion microscopy reveals individual protein shapes. <em>NBT</em>, <em>43</em>(9), 1539-1547. (<a href='https://doi.org/10.1038/s41587-024-02431-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The attainable resolution of fluorescence microscopy has reached the subnanometer range, but this technique still fails to image the morphology of single proteins or small molecular complexes. Here, we expand the specimens at least tenfold, label them with conventional fluorophores and image them with conventional light microscopes, acquiring videos in which we analyze fluorescence fluctuations. One-step nanoscale expansion (ONE) microscopy enables the visualization of the shapes of individual membrane and soluble proteins, achieving around 1-nm resolution. We show that conformational changes are readily observable, such as those undergone by the ~17-kDa protein calmodulin upon Ca2+ binding. ONE is also applied to clinical samples, analyzing the morphology of protein aggregates in cerebrospinal fluid from persons with Parkinson disease, potentially aiding disease diagnosis. This technology bridges the gap between high-resolution structural biology techniques and light microscopy, providing new avenues for discoveries in biology and medicine. Combining expansion microscopy with super-resolution radial fluctuations captures the morphology of single proteins.},
  archive      = {J_NBT},
  author       = {Shaib, Ali H. and Chouaib, Abed Alrahman and Chowdhury, Rajdeep and Altendorf, Jonas and Mihaylov, Daniel and Zhang, Chi and Krah, Donatus and Imani, Vanessa and Spencer, Russell K. W. and Georgiev, Svilen Veselinov and Mougios, Nikolaos and Monga, Mehar and Reshetniak, Sofiia and Mimoso, Tiago and Chen, Han and Fatehbasharzad, Parisa and Crzan, Dagmar and Saal, Kim-Ann and Alawieh, Mohamad Mahdi and Alawar, Nadia and Eilts, Janna and Kang, Jinyoung and Soleimani, Alireza and Müller, Marcus and Pape, Constantin and Alvarez, Luis and Trenkwalder, Claudia and Mollenhauer, Brit and Outeiro, Tiago F. and Köster, Sarah and Preobraschenski, Julia and Becherer, Ute and Moser, Tobias and Boyden, Edward S. and Aricescu, A. Radu and Sauer, Markus and Opazo, Felipe and Rizzoli, Silvio O.},
  doi          = {10.1038/s41587-024-02431-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1539-1547},
  shortjournal = {Nature Biotech.},
  title        = {One-step nanoscale expansion microscopy reveals individual protein shapes},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Development of compact transcriptional effectors using high-throughput measurements in diverse contexts. <em>NBT</em>, <em>43</em>(9), 1525-1538. (<a href='https://doi.org/10.1038/s41587-024-02442-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcriptional effectors are protein domains known to activate or repress gene expression; however, a systematic understanding of which effector domains regulate transcription across genomic, cell type and DNA-binding domain (DBD) contexts is lacking. Here we develop dCas9-mediated high-throughput recruitment (HT-recruit), a pooled screening method for quantifying effector function at endogenous target genes and test effector function for a library containing 5,092 nuclear protein Pfam domains across varied contexts. We also map context dependencies of effectors drawn from unannotated protein regions using a larger library tiling chromatin regulators and transcription factors. We find that many effectors depend on target and DBD contexts, such as HLH domains that can act as either activators or repressors. To enable efficient perturbations, we select context-robust domains, including ZNF705 KRAB, that improve CRISPRi tools to silence promoters and enhancers. We engineer a compact human activator called NFZ, by combining NCOA3, FOXO3 and ZNF473 domains, which enables efficient CRISPRa with better viral delivery and inducible control of chimeric antigen receptor T cells. Improved effectors for CRISPRi/CRISPRa are developed following high-throughput screening of transcriptional domains.},
  archive      = {J_NBT},
  author       = {Tycko, Josh and Van, Mike V. and Aradhana and DelRosso, Nicole and Ye, Hanrong and Yao, David and Valbuena, Raeline and Vaughan-Jackson, Alun and Xu, Xiaoshu and Ludwig, Connor and Spees, Kaitlyn and Liu, Katherine and Gu, Mingxin and Khare, Venya and Mukund, Adi Xiyal and Suzuki, Peter H. and Arana, Sophia and Zhang, Catherine and Du, Peter P. and Ornstein, Thea S. and Hess, Gaelen T. and Kamber, Roarke A. and Qi, Lei S. and Khalil, Ahmad S. and Bintu, Lacramioara and Bassik, Michael C.},
  doi          = {10.1038/s41587-024-02442-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1525-1538},
  shortjournal = {Nature Biotech.},
  title        = {Development of compact transcriptional effectors using high-throughput measurements in diverse contexts},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A structurally informed human protein–protein interactome reveals proteome-wide perturbations caused by disease mutations. <em>NBT</em>, <em>43</em>(9), 1510-1524. (<a href='https://doi.org/10.1038/s41587-024-02428-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To assist the translation of genetic findings to disease pathobiology and therapeutics discovery, we present an ensemble deep learning framework, termed PIONEER (Protein–protein InteractiOn iNtErfacE pRediction), that predicts protein-binding partner-specific interfaces for all known protein interactions in humans and seven other common model organisms to generate comprehensive structurally informed protein interactomes. We demonstrate that PIONEER outperforms existing state-of-the-art methods and experimentally validate its predictions. We show that disease-associated mutations are enriched in PIONEER-predicted protein–protein interfaces and explore their impact on disease prognosis and drug responses. We identify 586 significant protein–protein interactions (PPIs) enriched with PIONEER-predicted interface somatic mutations (termed oncoPPIs) from analysis of approximately 11,000 whole exomes across 33 cancer types and show significant associations of oncoPPIs with patient survival and drug responses. PIONEER, implemented as both a web server platform and a software package, identifies functional consequences of disease-associated alleles and offers a deep learning tool for precision medicine at multiscale interactome network levels. Protein–protein interactomes incorporating structural data predict the functional consequences of disease mutations.},
  archive      = {J_NBT},
  author       = {Xiong, Dapeng and Qiu, Yunguang and Zhao, Junfei and Zhou, Yadi and Lee, Dongjin and Gupta, Shobhita and Torres, Mateo and Lu, Weiqiang and Liang, Siqi and Kang, Jin Joo and Eng, Charis and Loscalzo, Joseph and Cheng, Feixiong and Yu, Haiyuan},
  doi          = {10.1038/s41587-024-02428-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1510-1524},
  shortjournal = {Nature Biotech.},
  title        = {A structurally informed human protein–protein interactome reveals proteome-wide perturbations caused by disease mutations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gold-siRNA supraclusters enhance the anti-tumor immune response of stereotactic ablative radiotherapy at primary and metastatic tumors. <em>NBT</em>, <em>43</em>(9), 1496-1509. (<a href='https://doi.org/10.1038/s41587-024-02448-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Strategies to enhance the anti-tumor immune response of stereotactic ablative radiotherapy (SABR) at primary tumors and abscopal sites are under intensive investigation. Here we report a metabolizable binary supracluster (BSCgal) that combines gold nanoclusters as radiosensitizing adjuvants with small interfering RNA (siRNA) targeting the immunosuppressive mediator galectin-1 (Gal-1). BSCgal comprises reversibly crosslinked cationic gold nanoclusters and siRNA complexes in a polymer matrix that biodegrades over weeks, facilitating clearance (90.3% in vivo clearance at 4 weeks) to reduce toxicity. The particle size well above the renal filtration threshold facilitates passive delivery to tumors. Using mouse models of head and neck cancer, we show that BSCgal augments the radiodynamic and immunotherapeutic effects of SABR at the primary and metastatic tumors by promoting tumor-inhibitory leukocytes, upregulating cytotoxic granzyme B and reducing immunosuppressive cell populations. It outperforms SABR plus Gal-1 antagonists, chemoradiation drug cisplatin or PD-1 inhibitor. This work presents a translatable strategy to converge focal radiosensitization with targeted immune checkpoint silencing for personalized radioimmunotherapy. Gold-siRNA clusters boost the immune response of radiotherapy against primary and distant tumors.},
  archive      = {J_NBT},
  author       = {Jiang, Yuyan and Cao, Hongbin and Deng, Huaping and Guan, Li and Langthasa, Jimpi and Colburg, Deana Rae Crystal and Melemenidis, Stavros and Cotton, Renee M. and Aleman, John and Wang, Xiao-Jing and Graves, Edward E. and Kalbasi, Anusha and Pu, Kanyi and Rao, Jianghong and Le, Quynh-Thu},
  doi          = {10.1038/s41587-024-02448-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1496-1509},
  shortjournal = {Nature Biotech.},
  title        = {Gold-siRNA supraclusters enhance the anti-tumor immune response of stereotactic ablative radiotherapy at primary and metastatic tumors},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Amplifying mutational profiling of extracellular vesicle mRNA with SCOPE. <em>NBT</em>, <em>43</em>(9), 1485-1495. (<a href='https://doi.org/10.1038/s41587-024-02426-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequencing of messenger RNA (mRNA) found in extracellular vesicles (EVs) in liquid biopsies can provide clinical information such as somatic mutations, resistance profiles and tumor recurrence. Despite this, EV mRNA remains underused due to its low abundance in liquid biopsies, and large sample volumes or specialized techniques for analysis are required. Here we introduce Self-amplified and CRISPR-aided Operation to Profile EVs (SCOPE), a platform for EV mRNA detection. SCOPE leverages CRISPR-mediated recognition of target RNA using Cas13 to initiate replication and signal amplification, achieving a sub-attomolar detection limit while maintaining single-nucleotide resolution. As a proof of concept, we designed probes for key mutations in KRAS, BRAF, EGFR and IDH1 genes, optimized protocols for single-pot assays and implemented an automated device for multi-sample detection. We validated SCOPE’s ability to detect early-stage lung cancer in animal models, monitored tumor mutational burden in patients with colorectal cancer and stratified patients with glioblastoma. SCOPE can expedite readouts, augmenting the clinical use of EVs in precision oncology. SCOPE detects low-abundance extracellular vesicle mRNA with high sensitivity for clinical applications.},
  archive      = {J_NBT},
  author       = {Song, Jayeon and Cho, Mi Hyeon and Cho, Hayoung and Song, Younseong and Lee, Sung Woon and Nam, Ho Chul and Yoon, Tae Ho and Shin, Jong Cheol and Hong, Jae-Sang and Kim, Yejin and Ekanayake, Emil and Jeon, Jueun and You, Dong Gil and Im, Sung Gap and Choi, Gyu-Seog and Park, Jun Seok and Carter, Bob C. and Balaj, Leonora and Seo, An Na and Miller, Miles A. and Park, Soo Yeun and Kang, Taejoon and Castro, Cesar M. and Lee, Hakho},
  doi          = {10.1038/s41587-024-02426-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1485-1495},
  shortjournal = {Nature Biotech.},
  title        = {Amplifying mutational profiling of extracellular vesicle mRNA with SCOPE},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Saturation profiling of drug-resistant genetic variants using prime editing. <em>NBT</em>, <em>43</em>(9), 1471-1484. (<a href='https://doi.org/10.1038/s41587-024-02465-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods to characterize the functional effects of genetic variants of uncertain significance (VUSs) have been limited by incomplete coverage of the mutational space. In clinical oncology, drug resistance arising from VUSs can prevent optimal treatment. Here we introduce PEER-seq, a high-throughput method based on prime editing that can evaluate the functional effects of single-nucleotide variants (SNVs). PEER-seq introduces both intended SNVs and synonymous marker mutations using prime editing and deep sequences the endogenous target regions to identify the introduced SNVs. We generate and functionally evaluate 2,476 SNVs in the epidermal growth factor receptor gene (EGFR), including 99% of all possible variants in the canonical tyrosine kinase domain. We determined resistance profiles of 95% of all possible EGFR protein variants encoded in the whole tyrosine kinase domain against the common tyrosine kinase inhibitors afatinib, osimertinib and osimertinib in the presence of the co-occurring substitution T790M, in PC-9 cells. Our study has the potential to substantially improve the precision of therapeutic choices in clinical settings. PEER-seq characterizes the drug resistance profiles of thousands of genetic variants.},
  archive      = {J_NBT},
  author       = {Kim, Younggwang and Oh, Hyeong-Cheol and Lee, Seungho and Kim, Hyongbum Henry},
  doi          = {10.1038/s41587-024-02465-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1471-1484},
  shortjournal = {Nature Biotech.},
  title        = {Saturation profiling of drug-resistant genetic variants using prime editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multimodal scanning of genetic variants with base and prime editing. <em>NBT</em>, <em>43</em>(9), 1458-1470. (<a href='https://doi.org/10.1038/s41587-024-02439-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutational scanning connects genetic variants to phenotype, enabling the interrogation of protein functions, interactions and variant pathogenicity. However, current methodologies cannot efficiently engineer customizable sets of diverse genetic variants in endogenous loci across cellular contexts in high throughput. Here, we combine cytosine and adenine base editors and a prime editor to assess the pathogenicity of a broad spectrum of variants in the epithelial growth factor receptor gene (EGFR). Using pooled base editing and prime editing guide RNA libraries, we install tens of thousands of variants spanning the full coding sequence of EGFR in multiple cell lines and assess the role of these variants in tumorigenesis and resistance to tyrosine kinase inhibitors. Our EGFR variant scan identifies important hits, supporting the robustness of the approach and revealing underappreciated routes to EGFR activation and drug response. We anticipate that multimodal precision mutational scanning can be applied broadly to characterize genetic variation in any genetic element of interest at high and single-nucleotide resolution. Thousands of rare oncogene variants are evaluated using multimodal gene editing screens.},
  archive      = {J_NBT},
  author       = {Belli, Olivier and Karava, Kyriaki and Farouni, Rick and Platt, Randall J.},
  doi          = {10.1038/s41587-024-02439-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1458-1470},
  shortjournal = {Nature Biotech.},
  title        = {Multimodal scanning of genetic variants with base and prime editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR–Cas9 ribonucleoprotein. <em>NBT</em>, <em>43</em>(9), 1445-1457. (<a href='https://doi.org/10.1038/s41587-024-02437-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lipid nanoparticle (LNP) delivery of clustered regularly interspaced short palindromic repeat (CRISPR) ribonucleoproteins (RNPs) could enable high-efficiency, low-toxicity and scalable in vivo genome editing if efficacious RNP–LNP complexes can be reliably produced. Here we engineer a thermostable Cas9 from Geobacillus stearothermophilus (GeoCas9) to generate iGeoCas9 variants capable of &gt;100× more genome editing of cells and organs compared with the native GeoCas9 enzyme. Furthermore, iGeoCas9 RNP–LNP complexes edit a variety of cell types and induce homology-directed repair in cells receiving codelivered single-stranded DNA templates. Using tissue-selective LNP formulations, we observe genome-editing levels of 16‒37% in the liver and lungs of reporter mice that receive single intravenous injections of iGeoCas9 RNP–LNPs. In addition, iGeoCas9 RNPs complexed to biodegradable LNPs edit the disease-causing SFTPC gene in lung tissue with 19% average efficiency, representing a major improvement over genome-editing levels observed previously using viral or nonviral delivery strategies. These results show that thermostable Cas9 RNP–LNP complexes can expand the therapeutic potential of genome editing. An engineered clustered regularly interspaced short palindromic repeat ribonucleoprotein delivered in lipid nanoparticles efficiently edits cells in vivo.},
  archive      = {J_NBT},
  author       = {Chen, Kai and Han, Hesong and Zhao, Sheng and Xu, Bryant and Yin, Boyan and Lawanprasert, Atip and Trinidad, Marena and Burgstone, Benjamin W. and Murthy, Niren and Doudna, Jennifer A.},
  doi          = {10.1038/s41587-024-02437-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1445-1457},
  shortjournal = {Nature Biotech.},
  title        = {Lung and liver editing by lipid nanoparticle delivery of a stable CRISPR–Cas9 ribonucleoprotein},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering b cells to treat and study human disease. <em>NBT</em>, <em>43</em>(9), 1431-1444. (<a href='https://doi.org/10.1038/s41587-025-02757-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern cell therapies are designed to harness natural biology to treat a range of complex diseases. The field of immunology has shown that B cells exhibit multiple unique features, including a natural propensity to interact with and regulate other immune cells, a high capacity to produce proteins, and a long cellular lifespan, which are being creatively applied in engineered B cell (eB cell) therapies. In recent years, advances in genome editing technologies and animal modeling have facilitated rapid progress in our ability to study eB cells and execute proof-of-concept studies, thus enabling the first clinical trials of eB cell therapies. In this review, we provide an overview of recent developments in eB cell therapies, including early clinical studies. We discuss challenges to clinical implementation, and promising directions for leveraging B cell biology in future applications for cancer and chronic disease. Recent developments in B cell engineering expand our ability to study human disease and drive progress toward new B cell therapies, balancing challenges in clinical implementation with emerging opportunities.},
  archive      = {J_NBT},
  author       = {Trivedi, Nikita and Pitner, Ragan A. and Rawlings, David J. and James, Richard G.},
  doi          = {10.1038/s41587-025-02757-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1431-1444},
  shortjournal = {Nature Biotech.},
  title        = {Engineering b cells to treat and study human disease},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A CRISPR-based method for detecting mRNA in extracellular vesicles. <em>NBT</em>, <em>43</em>(9), 1428-1429. (<a href='https://doi.org/10.1038/s41587-024-02440-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A CRISPR-based assay both recognizes and amplifies target mRNA, achieving sub-attomolar sensitivity with single-nucleotide resolution. This method enables the detection of low-abundance mRNA in extracellular vesicles, providing clinically relevant information for precision oncology.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02440-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1428-1429},
  shortjournal = {Nature Biotech.},
  title        = {A CRISPR-based method for detecting mRNA in extracellular vesicles},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Bao and rogers. <em>NBT</em>, <em>43</em>(9), 1427. (<a href='https://doi.org/10.1038/s41587-025-02784-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02784-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1427},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Bao and rogers},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Ntziachristos and yao. <em>NBT</em>, <em>43</em>(9), 1427. (<a href='https://doi.org/10.1038/s41587-025-02785-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02785-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1427},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Ntziachristos and yao},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI designs de novo antibiotics. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02822-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02822-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {AI designs de novo antibiotics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Boosted mitochondria reverse cognitive impairment in mice. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02823-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02823-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {Boosted mitochondria reverse cognitive impairment in mice},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mining archaea for antibiotics. <em>NBT</em>, <em>43</em>(9), 1425. (<a href='https://doi.org/10.1038/s41587-025-02824-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02824-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1425},
  shortjournal = {Nature Biotech.},
  title        = {Mining archaea for antibiotics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MicroRNAs. <em>NBT</em>, <em>43</em>(9), 1424. (<a href='https://doi.org/10.1038/s41587-025-02800-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to pharmaceutical compositions and methods of treatment involving microRNAs.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02800-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1424},
  shortjournal = {Nature Biotech.},
  title        = {MicroRNAs},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Intellectual property rights in the cultivated meat industry: Opportunities and challenges. <em>NBT</em>, <em>43</em>(9), 1419-1423. (<a href='https://doi.org/10.1038/s41587-025-02798-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cultivated meat faces escalating intellectual property (IP) issues as the industry expands, including conventional patent rights, the private IP rights of animal breeders and the Convention on Biological Diversity.},
  archive      = {J_NBT},
  author       = {Baek, Garam and Chung, Sukhyun and Yu, Peter and Park, Yooheon},
  doi          = {10.1038/s41587-025-02798-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1419-1423},
  shortjournal = {Nature Biotech.},
  title        = {Intellectual property rights in the cultivated meat industry: Opportunities and challenges},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Redefining druggable targets with artificial intelligence. <em>NBT</em>, <em>43</em>(9), 1416-1418. (<a href='https://doi.org/10.1038/s41587-025-02770-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A vast landscape of ‘undruggable’ cancer targets remains beyond the reach of conventional therapeutic agents. Recent advances in artificial intelligence (AI), however, are challenging this paradigm. Synthesizing insights from a Cancer Moonshot workshop, we argue that systemically addressing the undruggable target space with AI requires a new conceptual framework. We highlight the failure of current target taxonomies and the need for benchmarking datasets, and re-evaluate clinical validation for novel AI-driven modalities.},
  archive      = {J_NBT},
  author       = {Akinsanya, Karen and AlQuraishi, Mohammed and Boija, Ann and Chodera, John and Cichońska, Anna and Ghassemi, Marzyeh and Head, Martha and Jin, Wengong and Kibbe, Warren A. and Krogan, Nevan and LeVine, Michael V. and Moult, John and Stuart, Lynda and Tourassi, Georgia and Zou, James and Barzilay, Regina and Elemento, Olivier},
  doi          = {10.1038/s41587-025-02770-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1416-1418},
  shortjournal = {Nature Biotech.},
  title        = {Redefining druggable targets with artificial intelligence},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Senolytics under scrutiny in the quest to slow aging. <em>NBT</em>, <em>43</em>(9), 1413-1415. (<a href='https://doi.org/10.1038/s41587-025-02740-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {de Magalhães, João Pedro},
  doi          = {10.1038/s41587-025-02740-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1413-1415},
  shortjournal = {Nature Biotech.},
  title        = {Senolytics under scrutiny in the quest to slow aging},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Freya biosciences. <em>NBT</em>, <em>43</em>(9), 1411-1412. (<a href='https://doi.org/10.1038/s41587-025-02783-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that received sizeable early-stage funding in the previous year. Freya Biosciences is restoring balance in the vaginal microbiome to improve pregnancy outcomes.},
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02783-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1411-1412},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Freya biosciences},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Vilya therapeutics. <em>NBT</em>, <em>43</em>(9), 1409-1410. (<a href='https://doi.org/10.1038/s41587-025-02782-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that have received sizeable early-stage funding in the previous year. Vilya Therapeutics is designing macrocycle drug candidates.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-025-02782-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1409-1410},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Vilya therapeutics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Editor’s pick: Seaport therapeutics. <em>NBT</em>, <em>43</em>(9), 1407-1408. (<a href='https://doi.org/10.1038/s41587-025-02792-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Each year, Nature Biotechnology highlights companies that received sizeable early-stage funding in the previous year. Seaport Therapeutics is modifying central nervous system drugs for delivery through the lymphatic system, bypassing first-pass liver metabolism.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-025-02792-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1407-1408},
  shortjournal = {Nature Biotech.},
  title        = {Editor’s pick: Seaport therapeutics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(9), 1406. (<a href='https://doi.org/10.1038/s41587-025-02817-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02817-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1406},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Blooms supersized. <em>NBT</em>, <em>43</em>(9), 1405. (<a href='https://doi.org/10.1038/s41587-025-02825-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-025-02825-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1405},
  shortjournal = {Nature Biotech.},
  title        = {Blooms supersized},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Wegovy approved for MASH. <em>NBT</em>, <em>43</em>(9), 1404. (<a href='https://doi.org/10.1038/s41587-025-02828-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02828-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1404},
  shortjournal = {Nature Biotech.},
  title        = {Wegovy approved for MASH},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). FDA go-ahead for HPV immunotherapy. <em>NBT</em>, <em>43</em>(9), 1402. (<a href='https://doi.org/10.1038/s41587-025-02827-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02827-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1402},
  shortjournal = {Nature Biotech.},
  title        = {FDA go-ahead for HPV immunotherapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Regenerative cell therapy marches into corneal blindness. <em>NBT</em>, <em>43</em>(9), 1401-1404. (<a href='https://doi.org/10.1038/s41587-025-02808-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With a newly marketed product in Japan and 10 years of clinical data, regenerative cell therapies promise new options for patients facing a choice between invasive surgery or blindness.},
  archive      = {J_NBT},
  author       = {Marshall, Andrew},
  doi          = {10.1038/s41587-025-02808-4},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1401-1404},
  shortjournal = {Nature Biotech.},
  title        = {Regenerative cell therapy marches into corneal blindness},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). US government slams mRNA vaccine research. <em>NBT</em>, <em>43</em>(9), 1400. (<a href='https://doi.org/10.1038/s41587-025-02826-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02826-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1400},
  shortjournal = {Nature Biotech.},
  title        = {US government slams mRNA vaccine research},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mitochondrial transplants make healthy babies. <em>NBT</em>, <em>43</em>(9), 1399-1400. (<a href='https://doi.org/10.1038/s41587-025-02799-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After decades of research and consultation with regulators, mitochondrial donation is taking off in the UK. Will other countries follow?},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-025-02799-2},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1399-1400},
  shortjournal = {Nature Biotech.},
  title        = {Mitochondrial transplants make healthy babies},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing antivirals. <em>NBT</em>, <em>43</em>(9), 1397-1398. (<a href='https://doi.org/10.1038/s41587-025-02829-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Funding and public support for vaccines in the United States is at an all-time low. Promising research is resulting in effective antivirals, but they should not take the place of vaccines.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02829-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1397-1398},
  shortjournal = {Nature Biotech.},
  title        = {Advancing antivirals},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(8), 1396. (<a href='https://doi.org/10.1038/s41587-025-02768-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02768-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1396},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Five questions with kevin zhao. <em>NBT</em>, <em>43</em>(8), 1393. (<a href='https://doi.org/10.1038/s41587-025-02766-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An innovator of new genome editing technologies on the similarities between science and cooking, and the challenge of turning a new technology into a viable business.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-025-02766-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1393},
  shortjournal = {Nature Biotech.},
  title        = {Five questions with kevin zhao},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gender disparities persist within french biotech startup companies despite progress. <em>NBT</em>, <em>43</em>(8), 1389-1392. (<a href='https://doi.org/10.1038/s41587-025-02763-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Women are still underrepresented in executive roles in French biotech startups, while outperforming men in merger and acquisition activities.},
  archive      = {J_NBT},
  author       = {Khalef, Nawel and Catherine, David and Revelen, Anthony and Bakri, Aziz},
  doi          = {10.1038/s41587-025-02763-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1389-1392},
  shortjournal = {Nature Biotech.},
  title        = {Gender disparities persist within french biotech startup companies despite progress},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Addendum: Carbon-negative production of acetone and isopropanol by gas fermentation at industrial pilot scale. <em>NBT</em>, <em>43</em>(8), 1385-1387. (<a href='https://doi.org/10.1038/s41587-025-02767-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Liew, Fungmin Eric and Nogle, Robert and Abdalla, Tanus and Rasor, Blake J. and Canter, Christina and Jensen, Rasmus O. and Wang, Lan and Strutz, Jonathan and Chirania, Payal and De Tissera, Sashini and Mueller, Alexander P. and Ruan, Zhenhua and Gao, Allan and Tran, Loan and Engle, Nancy L. and Bromley, Jason C. and Daniell, James and Conrado, Robert and Tschaplinski, Timothy J. and Giannone, Richard J. and Hettich, Robert L. and Karim, Ashty S. and Simpson, Séan D. and Brown, Steven D. and Leang, Ching and Jewett, Michael C. and Köpke, Michael},
  doi          = {10.1038/s41587-025-02767-w},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1385-1387},
  shortjournal = {Nature Biotech.},
  title        = {Addendum: Carbon-negative production of acetone and isopropanol by gas fermentation at industrial pilot scale},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: Multistate and functional protein design using RoseTTAFold sequence space diffusion. <em>NBT</em>, <em>43</em>(8), 1384. (<a href='https://doi.org/10.1038/s41587-024-02456-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Lisanza, Sidney Lyayuga and Gershon, Jacob Merle and Tipps, Samuel W. K. and Sims, Jeremiah Nelson and Arnoldt, Lucas and Hendel, Samuel J. and Simma, Miriam K. and Liu, Ge and Yase, Muna and Wu, Hongwei and Tharp, Claire D. and Li, Xinting and Kang, Alex and Brackenbrough, Evans and Bera, Asim K. and Gerben, Stacey and Wittmann, Bruce J. and McShan, Andrew C. and Baker, David},
  doi          = {10.1038/s41587-024-02456-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1384},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Multistate and functional protein design using RoseTTAFold sequence space diffusion},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A community effort to optimize sequence-based deep learning models of gene regulation. <em>NBT</em>, <em>43</em>(8), 1373-1383. (<a href='https://doi.org/10.1038/s41587-024-02414-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A systematic evaluation of how model architectures and training strategies impact genomics model performance is needed. To address this gap, we held a DREAM Challenge where competitors trained models on a dataset of millions of random promoter DNA sequences and corresponding expression levels, experimentally determined in yeast. For a robust evaluation of the models, we designed a comprehensive suite of benchmarks encompassing various sequence types. All top-performing models used neural networks but diverged in architectures and training strategies. To dissect how architectural and training choices impact performance, we developed the Prix Fixe framework to divide models into modular building blocks. We tested all possible combinations for the top three models, further improving their performance. The DREAM Challenge models not only achieved state-of-the-art results on our comprehensive yeast dataset but also consistently surpassed existing benchmarks on Drosophila and human genomic datasets, demonstrating the progress that can be driven by gold-standard genomics datasets. A benchmarking competition improves tools that predict how regulatory regions control gene expression.},
  archive      = {J_NBT},
  author       = {Rafi, Abdul Muntakim and Nogina, Daria and Penzar, Dmitry and Lee, Dohoon and Lee, Danyeong and Kim, Nayeon and Kim, Sangyeup and Kim, Dohyeon and Shin, Yeojin and Kwak, Il-Youp and Meshcheryakov, Georgy and Lando, Andrey and Zinkevich, Arsenii and Kim, Byeong-Chan and Lee, Juhyun and Kang, Taein and Vaishnav, Eeshit Dhaval and Yadollahpour, Payman and Kim, Sun and Albrecht, Jake and Regev, Aviv and Gong, Wuming and Kulakovskiy, Ivan V. and Meyer, Pablo and de Boer, Carl G.},
  doi          = {10.1038/s41587-024-02414-w},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1373-1383},
  shortjournal = {Nature Biotech.},
  title        = {A community effort to optimize sequence-based deep learning models of gene regulation},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A comprehensive proteogenomic pipeline for neoantigen discovery to advance personalized cancer immunotherapy. <em>NBT</em>, <em>43</em>(8), 1360-1372. (<a href='https://doi.org/10.1038/s41587-024-02420-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The accurate identification and prioritization of antigenic peptides is crucial for the development of personalized cancer immunotherapies. Publicly available pipelines to predict clinical neoantigens do not allow direct integration of mass spectrometry immunopeptidomics data, which can uncover antigenic peptides derived from various canonical and noncanonical sources. To address this, we present an end-to-end clinical proteogenomic pipeline, called NeoDisc, that combines state-of-the-art publicly available and in-house software for immunopeptidomics, genomics and transcriptomics with in silico tools for the identification, prediction and prioritization of tumor-specific and immunogenic antigens from multiple sources, including neoantigens, viral antigens, high-confidence tumor-specific antigens and tumor-specific noncanonical antigens. We demonstrate the superiority of NeoDisc in accurately prioritizing immunogenic neoantigens over recent prioritization pipelines. We showcase the various features offered by NeoDisc that enable both rule-based and machine-learning approaches for personalized antigen discovery and neoantigen cancer vaccine design. Additionally, we demonstrate how NeoDisc’s multiomics integration identifies defects in the cellular antigen presentation machinery, which influence the heterogeneous tumor antigenic landscape. The neoantigen discovery pipeline NeoDisc incorporates mass spectrometry immunopeptidomics data.},
  archive      = {J_NBT},
  author       = {Huber, Florian and Arnaud, Marion and Stevenson, Brian J. and Michaux, Justine and Benedetti, Fabrizio and Thevenet, Jonathan and Bobisse, Sara and Chiffelle, Johanna and Gehert, Talita and Müller, Markus and Pak, HuiSong and Krämer, Anne I. and Altimiras, Emma Ricart and Racle, Julien and Taillandier-Coindard, Marie and Muehlethaler, Katja and Auger, Aymeric and Saugy, Damien and Murgues, Baptiste and Benyagoub, Abdelkader and Gfeller, David and Laniti, Denarda Dangaj and Kandalaft, Lana and Rodrigo, Blanca Navarro and Bouchaab, Hasna and Tissot, Stephanie and Coukos, George and Harari, Alexandre and Bassani-Sternberg, Michal},
  doi          = {10.1038/s41587-024-02420-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1360-1372},
  shortjournal = {Nature Biotech.},
  title        = {A comprehensive proteogenomic pipeline for neoantigen discovery to advance personalized cancer immunotherapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rapid species-level metagenome profiling and containment estimation with sylph. <em>NBT</em>, <em>43</em>(8), 1348-1359. (<a href='https://doi.org/10.1038/s41587-024-02412-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Profiling metagenomes against databases allows for the detection and quantification of microorganisms, even at low abundances where assembly is not possible. We introduce sylph, a species-level metagenome profiler that estimates genome-to-metagenome containment average nucleotide identity (ANI) through zero-inflated Poisson k-mer statistics, enabling ANI-based taxa detection. On the Critical Assessment of Metagenome Interpretation II (CAMI2) Marine dataset, sylph was the most accurate profiling method of seven tested. For multisample profiling, sylph took &gt;10-fold less central processing unit time compared to Kraken2 and used 30-fold less memory. Sylph’s ANI estimates provided an orthogonal signal to abundance, allowing for an ANI-based metagenome-wide association study for Parkinson disease (PD) against 289,232 genomes while confirming known butyrate–PD associations at the strain level. Sylph took &lt;1 min and 16 GB of random-access memory to profile metagenomes against 85,205 prokaryotic and 2,917,516 viral genomes, detecting 30-fold more viral sequences in the human gut compared to RefSeq. Sylph offers precise, efficient profiling with accurate containment ANI estimation even for low-coverage genomes. The microbial composition of metagenomes is identified in seconds by profiling against large databases.},
  archive      = {J_NBT},
  author       = {Shaw, Jim and Yu, Yun William},
  doi          = {10.1038/s41587-024-02412-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1348-1359},
  shortjournal = {Nature Biotech.},
  title        = {Rapid species-level metagenome profiling and containment estimation with sylph},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). NIS-seq enables cell-type-agnostic optical perturbation screening. <em>NBT</em>, <em>43</em>(8), 1337-1347. (<a href='https://doi.org/10.1038/s41587-024-02516-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optical pooled screening offers a broader-scale alternative to enrichment-based perturbation screening, using fluorescence microscopy to correlate phenotypes and perturbations across single cells. Previous methods work well in large, transcriptionally active cell lines, because they rely on cytosolic detection of endogenously expressed barcoded transcripts; however, they are limited by reliable cell segmentation, cytosol size, transcriptional activity and cell density. Nuclear In-Situ Sequencing (NIS-Seq) expands this technology by creating bright sequencing signals directly from nuclear genomic DNA to screen nucleated cells at high density and high library complexity. By inserting an inverted phage promoter downstream of the single guide RNA (sgRNA), many RNA copies of the sgRNA can be generated and sequenced independently of cellular transcription. In this study, we benchmarked NIS-Seq across eight cell types from two species and performed four genome-scale optical perturbation screens, identifying key players of inflammation-related cellular pathways. Finally, we performed a small-scale pooled optical screen in primary human macrophages from blood of healthy donors and demonstrated barcode identification in lentivirally transduced human skin tissue. An optical pooled screening method can correlate phenotypes and perturbations in any nucleated cell type.},
  archive      = {J_NBT},
  author       = {Fandrey, Caroline I. and Jentzsch, Marius and Konopka, Peter and Hoch, Alexander and Blumenstock, Katja and Zackria, Afraa and Maasewerd, Salie and Lovotti, Marta and Lapp, Dorothee J. and Gohr, Florian N. and Suwara, Piotr and Świeżewski, Jędrzej and Rossnagel, Lukas and Gobs, Fabienne and Cristodaro, Maia and Muhandes, Lina and Behrendt, Rayk and Lam, Martin C. and Walgenbach, Klaus J. and Bald, Tobias and Schmidt, Florian I. and Latz, Eicke and Schmid-Burgk, Jonathan L.},
  doi          = {10.1038/s41587-024-02516-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1337-1347},
  shortjournal = {Nature Biotech.},
  title        = {NIS-seq enables cell-type-agnostic optical perturbation screening},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scalable, compressed phenotypic screening using pooled perturbations. <em>NBT</em>, <em>43</em>(8), 1324-1336. (<a href='https://doi.org/10.1038/s41587-024-02403-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-throughput phenotypic screens using biochemical perturbations and high-content readouts are constrained by limitations of scale. To address this, we establish a method of pooling exogenous perturbations followed by computational deconvolution to reduce required sample size, labor and cost. We demonstrate the increased efficiency of compressed experimental designs compared to conventional approaches through benchmarking with a bioactive small-molecule library and a high-content imaging readout. We then apply compressed screening in two biological discovery campaigns. In the first, we use early-passage pancreatic cancer organoids to map transcriptional responses to a library of recombinant tumor microenvironment protein ligands, uncovering reproducible phenotypic shifts induced by specific ligands distinct from canonical reference signatures and correlated with clinical outcome. In the second, we identify the pleotropic modulatory effects of a chemical compound library with known mechanisms of action on primary human peripheral blood mononuclear cell immune responses. In sum, our approach empowers phenotypic screens with information-rich readouts to advance drug discovery efforts and basic biological inquiry. Phenotypic screens in organoids and primary cells are scaled up using perturbation pooling.},
  archive      = {J_NBT},
  author       = {Liu, Nuo and Kattan, Walaa E. and Mead, Benjamin E. and Kummerlowe, Conner and Cheng, Thomas and Ingabire, Sarah and Cheah, Jaime H. and Soule, Christian K. and Vrcic, Anita and McIninch, Jane K. and Triana, Sergio and Guzman, Manuel and Dao, Tyler T. and Peters, Joshua M. and Lowder, Kristen E. and Crawford, Lorin and Amini, Ava P. and Blainey, Paul C. and Hahn, William C. and Cleary, Brian and Bryson, Bryan and Winter, Peter S. and Raghavan, Srivatsan and Shalek, Alex K.},
  doi          = {10.1038/s41587-024-02403-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1324-1336},
  shortjournal = {Nature Biotech.},
  title        = {Scalable, compressed phenotypic screening using pooled perturbations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Non-pathogenic e. coli displaying decoy-resistant IL18 mutein boosts anti-tumor and CAR NK cell responses. <em>NBT</em>, <em>43</em>(8), 1311-1323. (<a href='https://doi.org/10.1038/s41587-024-02418-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The tumor microenvironment can inhibit the efficacy of cancer therapies through mechanisms such as poor trafficking and exhaustion of immune cells. Here, to address this challenge, we exploited the safety, tumor tropism and ease of genetic manipulation of non-pathogenic Escherichia coli (E. coli) to deliver key immune-activating cytokines to tumors via surface display on the outer membrane of E. coli K-12 DH5α. Non-pathogenic E. coli expressing murine decoy-resistant IL18 mutein (DR18) induced robust CD8+ T and natural killer (NK) cell-dependent immune responses and suppressed tumor progression in immune-competent colorectal carcinoma and melanoma mouse models. E. coli K-12 DH5α engineered to display human DR18 potently activated mesothelin-targeting chimeric antigen receptor (CAR) NK cells and enhance their trafficking into tumors, which extended survival in an NK cell treatment-resistant mesothelioma xenograft model by enhancing TNF signaling and upregulating NK activation markers. Our live bacteria-based immunotherapeutic system safely and effectively induces potent anti-tumor responses in treatment-resistant solid tumors, motivating further evaluation of this approach in the clinic. Bacteria expressing an anti-tumor cytokine act as a potent cancer immunotherapy.},
  archive      = {J_NBT},
  author       = {Yang, Shaobo and Sheffer, Michal and Kaplan, Isabel E. and Wang, Zongqi and Tarannum, Mubin and Dinh, Khanhlinh and Abdulhamid, Yasmin and Bobilev, Eden and Shapiro, Roman and Porter, Rebecca and Soiffer, Robert and Ritz, Jerome and Koreth, John and Wei, Yun and Chen, Peiru and Zhang, Ke and Márquez-Pellegrin, Valeria and Bonanno, Shanna and Joshi, Neel and Guan, Ming and Yang, Mengdi and Li, Deng and Bellini, Chiara and Liu, Fuguo and Chen, Jianzhu and Wu, Catherine J. and Barbie, David and Li, Jiahe and Romee, Rizwan},
  doi          = {10.1038/s41587-024-02418-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1311-1323},
  shortjournal = {Nature Biotech.},
  title        = {Non-pathogenic e. coli displaying decoy-resistant IL18 mutein boosts anti-tumor and CAR NK cell responses},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Continuous multiplexed phage genome editing using recombitrons. <em>NBT</em>, <em>43</em>(8), 1299-1310. (<a href='https://doi.org/10.1038/s41587-024-02370-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bacteriophage genome editing can enhance the efficacy of phages to eliminate pathogenic bacteria in patients and in the environment. However, current methods for editing phage genomes require laborious screening, counterselection or in vitro construction of modified genomes. Here, we present a scalable approach that uses modified bacterial retrons called recombitrons to generate recombineering donor DNA paired with single-stranded binding and annealing proteins for integration into phage genomes. This system can efficiently create genome modifications in multiple phages without the need for counterselection. The approach also supports larger insertions and deletions, which can be combined with simultaneous counterselection for &gt;99% efficiency. Moreover, we show that the process is continuous, with more edits accumulating the longer the phage is cultured with the host, and multiplexable. We install up to five distinct mutations on a single lambda phage genome without counterselection in only a few hours of hands-on time and identify a residue-level epistatic interaction in the T7 gp17 tail fiber. Phage genomes are readily engineered using retron recombineering.},
  archive      = {J_NBT},
  author       = {Fishman, Chloe B. and Crawford, Kate D. and Bhattarai-Kline, Santi and Poola, Darshini and Zhang, Karen and González-Delgado, Alejandro and Rojas-Montero, Matías and Shipman, Seth L.},
  doi          = {10.1038/s41587-024-02370-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1299-1310},
  shortjournal = {Nature Biotech.},
  title        = {Continuous multiplexed phage genome editing using recombitrons},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multistate and functional protein design using RoseTTAFold sequence space diffusion. <em>NBT</em>, <em>43</em>(8), 1288-1298. (<a href='https://doi.org/10.1038/s41587-024-02395-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein denoising diffusion probabilistic models are used for the de novo generation of protein backbones but are limited in their ability to guide generation of proteins with sequence-specific attributes and functional properties. To overcome this limitation, we developed ProteinGenerator (PG), a sequence space diffusion model based on RoseTTAFold that simultaneously generates protein sequences and structures. Beginning from a noised sequence representation, PG generates sequence and structure pairs by iterative denoising, guided by desired sequence and structural protein attributes. We designed thermostable proteins with varying amino acid compositions and internal sequence repeats and cage bioactive peptides, such as melittin. By averaging sequence logits between diffusion trajectories with distinct structural constraints, we designed multistate parent–child protein triples in which the same sequence folds to different supersecondary structures when intact in the parent versus split into two child domains. PG design trajectories can be guided by experimental sequence–activity data, providing a general approach for integrated computational and experimental optimization of protein function. ProteinGenerator simultaneously generates protein sequences and structures using sequence space diffusion.},
  archive      = {J_NBT},
  author       = {Lisanza, Sidney Lyayuga and Gershon, Jacob Merle and Tipps, Samuel W. K. and Sims, Jeremiah Nelson and Arnoldt, Lucas and Hendel, Samuel J. and Simma, Miriam K. and Liu, Ge and Yase, Muna and Wu, Hongwei and Tharp, Claire D. and Li, Xinting and Kang, Alex and Brackenbrough, Evans and Bera, Asim K. and Gerben, Stacey and Wittmann, Bruce J. and McShan, Andrew C. and Baker, David},
  doi          = {10.1038/s41587-024-02395-w},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1288-1298},
  shortjournal = {Nature Biotech.},
  title        = {Multistate and functional protein design using RoseTTAFold sequence space diffusion},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Long-term engrafting multilineage hematopoietic cells differentiated from human induced pluripotent stem cells. <em>NBT</em>, <em>43</em>(8), 1274-1287. (<a href='https://doi.org/10.1038/s41587-024-02360-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hematopoietic stem cells (HSCs) derived from human induced pluripotent stem cells (iPS cells) have important biomedical applications. We identified differentiation conditions that generate HSCs defined by robust long-term multilineage engraftment in immune-deficient NOD,B6.Prkdcscid Il2rgtm1Wjl/SzJ KitW41/W41 mice. We guided differentiating iPS cells, as embryoid bodies in a defined culture medium supplemented with retinyl acetate, through HOXA-patterned mesoderm to hemogenic endothelium specified by bone morphogenetic protein 4 and vascular endothelial growth factor (VEGF). Removal of VEGF facilitated an efficient endothelial-to-hematopoietic transition, evidenced by release into the culture medium of CD34+ blood cells, which were cryopreserved. Intravenous transplantation of two million thawed CD34+ cells differentiated from four independent iPS cell lines produced multilineage bone marrow engraftment in 25–50% of immune-deficient recipient mice. These functionally defined, multipotent CD34+ hematopoietic cells, designated iPS cell-derived HSCs (iHSCs), produced levels of engraftment similar to those achieved following umbilical cord blood transplantation. Our study provides a step toward the goal of generating HSCs for clinical translation. Human hematopoietic cells derived from pluripotent cells repopulate multiple blood lineages in mice.},
  archive      = {J_NBT},
  author       = {Ng, Elizabeth S. and Sarila, Gulcan and Li, Jacky Y. and Edirisinghe, Hasindu S. and Saxena, Ritika and Sun, Shicheng and Bruveris, Freya F. and Labonne, Tanya and Sleebs, Nerida and Maytum, Alexander and Yow, Raymond Y. and Inguanti, Chantelle and Motazedian, Ali and Calvanese, Vincenzo and Capellera-Garcia, Sandra and Ma, Feiyang and Nim, Hieu T. and Ramialison, Mirana and Bonifer, Constanze and Mikkola, Hanna K. A. and Stanley, Edouard G. and Elefanty, Andrew G.},
  doi          = {10.1038/s41587-024-02360-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1274-1287},
  shortjournal = {Nature Biotech.},
  title        = {Long-term engrafting multilineage hematopoietic cells differentiated from human induced pluripotent stem cells},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Model-directed generation of artificial CRISPR–Cas13a guide RNA sequences improves nucleic acid detection. <em>NBT</em>, <em>43</em>(8), 1266-1273. (<a href='https://doi.org/10.1038/s41587-024-02422-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR guide RNA sequences deriving exactly from natural sequences may not perform optimally in every application. Here we implement and evaluate algorithms for designing maximally fit, artificial CRISPR–Cas13a guides with multiple mismatches to natural sequences that are tailored for diagnostic applications. These guides offer more sensitive detection of diverse pathogens and discrimination of pathogen variants compared with guides derived directly from natural sequences and illuminate design principles that broaden Cas13a targeting. Model-directed generative design is applied to CRISPR–Cas13a guide RNAs, outperforming natural sequences.},
  archive      = {J_NBT},
  author       = {Mantena, Sreekar and Pillai, Priya P. and Petros, Brittany A. and Welch, Nicole L. and Myhrvold, Cameron and Sabeti, Pardis C. and Metsky, Hayden C.},
  doi          = {10.1038/s41587-024-02422-w},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1266-1273},
  shortjournal = {Nature Biotech.},
  title        = {Model-directed generation of artificial CRISPR–Cas13a guide RNA sequences improves nucleic acid detection},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Unlocking gene regulatory networks for crop resilience and sustainable agriculture. <em>NBT</em>, <em>43</em>(8), 1254-1265. (<a href='https://doi.org/10.1038/s41587-025-02727-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the complex mechanisms of gene regulatory networks (GRNs) has emerged as a transformative approach in agricultural research. By deciphering the regulatory mechanisms underlying key traits, GRN studies offer opportunities to enhance crop resilience to environmental challenges, improve yield and ensure sustainable food production. In this Review, we highlight the importance of GRN research in agriculture and explore how cutting-edge biotechnology, interdisciplinary approaches and computational modeling techniques are addressing the challenges in the field. We discuss how integrating diverse datasets at different resolutions empowers us to unravel the complex genetic networks governing crop responses to climate change, pests and diseases. By harnessing the power of GRNs, we have the potential to transform crop improvement strategies, develop stress-tolerant varieties and ensure global food security. We provide insights into the current opportunities and challenges of GRN research in agriculture, bridging the gap between scientific advancements and the pressing need for sustainable agricultural practices. Gene regulatory networks are crucial for understanding complex gene regulation in plants and could advance crop improvement for more sustainable and climate-resilient agricultural practices.},
  archive      = {J_NBT},
  author       = {Leong, Richalynn and He, Xin and Beijen, Bob Sebastiaan and Sakai, Toshiyuki and Goncalves, Jorge and Ding, Pingtao},
  doi          = {10.1038/s41587-025-02727-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1254-1265},
  shortjournal = {Nature Biotech.},
  title        = {Unlocking gene regulatory networks for crop resilience and sustainable agriculture},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An antigen discovery pipeline integrates multi-omics data and informs immunotherapy. <em>NBT</em>, <em>43</em>(8), 1252-1253. (<a href='https://doi.org/10.1038/s41587-024-02427-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed NeoDisc, a computational antigen discovery pipeline that integrates multi-omics data, including genomics, transcriptomics and mass spectrometry-based immunopeptidomics. NeoDisc accurately identifies and prioritizes tumor-specific antigens and designs personalized cancer vaccines. The pipeline reveals tumor heterogeneity and emphasizes defects in antigen presentation that might affect the success of cancer immunotherapy.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02427-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1252-1253},
  shortjournal = {Nature Biotech.},
  title        = {An antigen discovery pipeline integrates multi-omics data and informs immunotherapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A framework for expanding discovery efforts with compressed phenotypic screens. <em>NBT</em>, <em>43</em>(8), 1250-1251. (<a href='https://doi.org/10.1038/s41587-024-02404-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sample, financial and labor requirements are key barriers to scaling up high-content phenotypic discovery efforts. A broadly applicable method overcomes these challenges through experimental compression (by pooling various perturbations) and computational deconvolution (of their individual effects), empowering the use of phenotypic screening to advance therapeutic discovery.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02404-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1250-1251},
  shortjournal = {Nature Biotech.},
  title        = {A framework for expanding discovery efforts with compressed phenotypic screens},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Hie and lopez. <em>NBT</em>, <em>43</em>(8), 1249. (<a href='https://doi.org/10.1038/s41587-025-02775-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02775-w},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1249},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Hie and lopez},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Forum: Maresca and shapiro. <em>NBT</em>, <em>43</em>(8), 1249. (<a href='https://doi.org/10.1038/s41587-025-02776-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02776-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1249},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Maresca and shapiro},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bacteria displaying cytokines heat up the tumor microenvironment. <em>NBT</em>, <em>43</em>(8), 1246-1248. (<a href='https://doi.org/10.1038/s41587-024-02429-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Escherichia coli engineered to display cytokines destroy hard-to-treat tumors by boosting the activity of local native and adoptive immune effector cells.},
  archive      = {J_NBT},
  author       = {Fidelle, Marine and Zitvogel, Laurence},
  doi          = {10.1038/s41587-024-02429-3},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1246-1248},
  shortjournal = {Nature Biotech.},
  title        = {Bacteria displaying cytokines heat up the tumor microenvironment},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Phage genome engineering with retrons. <em>NBT</em>, <em>43</em>(8), 1244-1245. (<a href='https://doi.org/10.1038/s41587-024-02392-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A method using retron recombineering for phage genome modification may facilitate access to therapeutic phages.},
  archive      = {J_NBT},
  author       = {Osterman, Ilya and Sorek, Rotem},
  doi          = {10.1038/s41587-024-02392-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1244-1245},
  shortjournal = {Nature Biotech.},
  title        = {Phage genome engineering with retrons},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A developmental route to hematopoietic stem cells. <em>NBT</em>, <em>43</em>(8), 1242-1243. (<a href='https://doi.org/10.1038/s41587-024-02398-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A differentiation method informed by developmental biology converts human pluripotent stem cells to engraftable hematopoietic stem and progenitor cells without the use of transgenes.},
  archive      = {J_NBT},
  author       = {Wilkinson, Adam C. and de Bruijn, Marella F. T. R.},
  doi          = {10.1038/s41587-024-02398-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1242-1243},
  shortjournal = {Nature Biotech.},
  title        = {A developmental route to hematopoietic stem cells},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineered immune receptors protect plants from multiple viruses. <em>NBT</em>, <em>43</em>(8), 1241. (<a href='https://doi.org/10.1038/s41587-025-02787-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02787-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1241},
  shortjournal = {Nature Biotech.},
  title        = {Engineered immune receptors protect plants from multiple viruses},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Microglia replacement halts rare brain disease progression. <em>NBT</em>, <em>43</em>(8), 1241. (<a href='https://doi.org/10.1038/s41587-025-02788-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02788-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1241},
  shortjournal = {Nature Biotech.},
  title        = {Microglia replacement halts rare brain disease progression},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reversible engraftment of an engineered microbe in the human gut. <em>NBT</em>, <em>43</em>(8), 1241. (<a href='https://doi.org/10.1038/s41587-025-02790-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02790-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1241},
  shortjournal = {Nature Biotech.},
  title        = {Reversible engraftment of an engineered microbe in the human gut},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Antivirals. <em>NBT</em>, <em>43</em>(8), 1240. (<a href='https://doi.org/10.1038/s41587-025-02760-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to antiviral compositions and methods of treatment.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02760-3},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1240},
  shortjournal = {Nature Biotech.},
  title        = {Antivirals},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The patent landscape of chagas disease vaccines indicates major underinvestment in an emerging global health threat. <em>NBT</em>, <em>43</em>(8), 1231-1239. (<a href='https://doi.org/10.1038/s41587-025-02759-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Considering the enormous human and economic toll Chagas disease inflicts, investment in R&amp;D of medical innovations such as diagnostics, pharmaceuticals and vaccines remains disproportionately low.},
  archive      = {J_NBT},
  author       = {Widdup, Lily and Johnson, Bradley and Kowalski, Stanley P.},
  doi          = {10.1038/s41587-025-02759-w},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1231-1239},
  shortjournal = {Nature Biotech.},
  title        = {The patent landscape of chagas disease vaccines indicates major underinvestment in an emerging global health threat},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Jefferson et al. reply. <em>NBT</em>, <em>43</em>(8), 1229-1230. (<a href='https://doi.org/10.1038/s41587-025-02765-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Jefferson, Osmat and Price, W. Nicholson and Tu, S. Sean and Vishnubhakat, Saurabh and Rai, Arti K.},
  doi          = {10.1038/s41587-025-02765-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1229-1230},
  shortjournal = {Nature Biotech.},
  title        = {Jefferson et al. reply},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Response to ‘The puzzle of biologics manufacturing platform patents’. <em>NBT</em>, <em>43</em>(8), 1227-1228. (<a href='https://doi.org/10.1038/s41587-025-02764-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Watt, Stuart and Johnson, Emily and Doss, Raymond},
  doi          = {10.1038/s41587-025-02764-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1227-1228},
  shortjournal = {Nature Biotech.},
  title        = {Response to ‘The puzzle of biologics manufacturing platform patents’},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Non-profits wade in where for-profit life science investment fears to tread. <em>NBT</em>, <em>43</em>(8), 1221-1226. (<a href='https://doi.org/10.1038/s41587-025-02752-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For decades, and in the exceptional instance of the Institut Pasteur for over a century, non-profit organizations have augmented R&amp;D to advance scientific understanding and develop affordable and accessible pipelines of vaccines, medicines and diagnostics. Small and medium enterprises and former biopharmaceutical industry executives make a sizeable contribution to global health goals by pursuing opportunities in this corner of the global biopharmaceutical ecosystem.},
  archive      = {J_NBT},
  author       = {Nasto, Barbara},
  doi          = {10.1038/s41587-025-02752-3},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1221-1226},
  shortjournal = {Nature Biotech.},
  title        = {Non-profits wade in where for-profit life science investment fears to tread},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Probing the proteome. <em>NBT</em>, <em>43</em>(8), 1216-1220. (<a href='https://doi.org/10.1038/s41587-025-02737-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chemical proteomics has brought rigor to covalent drug discovery and drugs to the clinic. Can it deliver a new generation of drug targets?},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-025-02737-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1216-1220},
  shortjournal = {Nature Biotech.},
  title        = {Probing the proteome},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(8), 1215. (<a href='https://doi.org/10.1038/s41587-025-02786-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02786-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1215},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Children with rare genetic diseases get CRISPR cures center. <em>NBT</em>, <em>43</em>(8), 1214. (<a href='https://doi.org/10.1038/s41587-025-02795-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02795-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1214},
  shortjournal = {Nature Biotech.},
  title        = {Children with rare genetic diseases get CRISPR cures center},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). FDA crackdown on cells across borders. <em>NBT</em>, <em>43</em>(8), 1213. (<a href='https://doi.org/10.1038/s41587-025-02789-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02789-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1213},
  shortjournal = {Nature Biotech.},
  title        = {FDA crackdown on cells across borders},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Natural molecular glues spur next-gen small molecules for ‘undruggable’ targets. <em>NBT</em>, <em>43</em>(8), 1212-1214. (<a href='https://doi.org/10.1038/s41587-025-02779-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inspired by natural molecular glues, drug hunters are chasing protein–protein interactions with next-gen small molecules tuned for enhanced tissue specificity and reduced toxicity.},
  archive      = {J_NBT},
  author       = {Cully, Megan},
  doi          = {10.1038/s41587-025-02779-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1212-1214},
  shortjournal = {Nature Biotech.},
  title        = {Natural molecular glues spur next-gen small molecules for ‘undruggable’ targets},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AbbVie doubles down on trispecific engagers. <em>NBT</em>, <em>43</em>(8), 1211. (<a href='https://doi.org/10.1038/s41587-025-02796-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02796-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1211},
  shortjournal = {Nature Biotech.},
  title        = {AbbVie doubles down on trispecific engagers},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Can amylin-based drugs upstage GLP-1 agonists?. <em>NBT</em>, <em>43</em>(8), 1209-1211. (<a href='https://doi.org/10.1038/s41587-025-02778-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analogs of the pancreatic peptide hormone amylin are making strides in obesity as developers zoom in on oral drugs that can emulate or even improve on injectable GLP-1 agonists while avoiding muscle loss and gastrointestinal troubles.},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/s41587-025-02778-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1209-1211},
  shortjournal = {Nature Biotech.},
  title        = {Can amylin-based drugs upstage GLP-1 agonists?},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(7), 1208. (<a href='https://doi.org/10.1038/s41587-025-02732-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02732-7},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1208},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Innovation to investment: How to build your first life sciences pitch deck. <em>NBT</em>, <em>43</em>(7), 1204-1207. (<a href='https://doi.org/10.1038/s41587-025-02730-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A strong pitch deck can help make a good first impression, establish credibility and build the confidence to secure the support and financial resources needed for eventual commercial success.},
  archive      = {J_NBT},
  author       = {Reuther, Katherine E. and Cooke, Darren and Kessel, Mark},
  doi          = {10.1038/s41587-025-02730-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1204-1207},
  shortjournal = {Nature Biotech.},
  title        = {Innovation to investment: How to build your first life sciences pitch deck},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Efficient genetic code expansion without host genome modifications. <em>NBT</em>, <em>43</em>(7), 1203. (<a href='https://doi.org/10.1038/s41587-024-02454-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Costello, Alan and Peterson, Alexander A. and Lanster, David L. and Li, Zhiyi and Carver, Gavriela D. and Badran, Ahmed H.},
  doi          = {10.1038/s41587-024-02454-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1203},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Efficient genetic code expansion without host genome modifications},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Mapping multimodal phenotypes to perturbations in cells and tissue with CRISPRmap. <em>NBT</em>, <em>43</em>(7), 1203. (<a href='https://doi.org/10.1038/s41587-025-02594-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Gu, Jiacheng and Iyer, Abhishek and Wesley, Ben and Taglialatela, Angelo and Leuzzi, Giuseppe and Hangai, Sho and Decker, Aubrianna and Gu, Ruoyu and Klickstein, Naomi and Shuai, Yuanlong and Jankovic, Kristina and Parker-Burns, Lucy and Jin, Yinuo and Zhang, Jia Yi and Hong, Justin and Niu, Xiang and Costa, Jonathon A. and Pezet, Mikael G. and Chou, Jacqueline and Chen, Cao ‘Claire’ and Paiva, Margaret and Snoeck, Hans-Willem and Landau, Dan A. and Azizi, Elham and Chan, Edmond M. and Ciccia, Alberto and Gaublomme, Jellert T.},
  doi          = {10.1038/s41587-025-02594-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1203},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Mapping multimodal phenotypes to perturbations in cells and tissue with CRISPRmap},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Multiplexed, image-based pooled screens in primary cells and tissues with PerturbView. <em>NBT</em>, <em>43</em>(7), 1202. (<a href='https://doi.org/10.1038/s41587-025-02602-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kudo, Takamasa and Meireles, Ana M. and Moncada, Reuben and Chen, Yushu and Wu, Ping and Gould, Joshua and Hu, Xiaoyu and Kornfeld, Opher and Jesudason, Rajiv and Foo, Conrad and Höckendorf, Burkhard and Corrada Bravo, Hector and Town, Jason P. and Wei, Runmin and Rios, Antonio and Chandrasekar, Vineethkrishna and Heinlein, Melanie and Chuong, Amy S. and Cai, Shuangyi and Lu, Cherry Sakura and Coelho, Paula and Mis, Monika and Celen, Cemre and Kljavin, Noelyn and Jiang, Jian and Richmond, David and Thakore, Pratiksha and Benito-Gutiérrez, Elia and Geiger-Schuller, Kathryn and Hleap, Jose Sergio and Kayagaki, Nobuhiko and de Sousa e Melo, Felipe and McGinnis, Lisa and Li, Bo and Singh, Avtar and Garraway, Levi and Rozenblatt-Rosen, Orit and Regev, Aviv and Lubeck, Eric},
  doi          = {10.1038/s41587-025-02602-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1202},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Multiplexed, image-based pooled screens in primary cells and tissues with PerturbView},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Coordinated, multicellular patterns of transcriptional variation that stratify patient cohorts are revealed by tensor decomposition. <em>NBT</em>, <em>43</em>(7), 1192-1201. (<a href='https://doi.org/10.1038/s41587-024-02411-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tissue-level and organism-level biological processes often involve the coordinated action of multiple distinct cell types. The recent application of single-cell assays to many individuals should enable the study of how donor-level variation in one cell type is linked to that in other cell types. Here we introduce a computational approach called single-cell interpretable tensor decomposition (scITD) to identify common axes of interindividual variation by considering joint expression variation across multiple cell types. scITD combines expression matrices from each cell type into a higher-order matrix and factorizes the result using the Tucker tensor decomposition. Applying scITD to single-cell RNA-sequencing data on 115 persons with lupus and 83 persons with coronavirus disease 2019, we identify patterns of coordinated cellular activity linked to disease severity and specific phenotypes, such as lupus nephritis. scITD results also implicate specific signaling pathways likely mediating coordination between cell types. Overall, scITD offers a tool for understanding the covariation of cell states across individuals, which can yield insights into the complex processes that define and stratify disease. Unsupervised analysis of single-cell RNA-sequencing data across cell types and individuals predicts disease severity.},
  archive      = {J_NBT},
  author       = {Mitchel, Jonathan and Gordon, M. Grace and Perez, Richard K. and Biederstedt, Evan and Bueno, Raymund and Ye, Chun Jimmie and Kharchenko, Peter V.},
  doi          = {10.1038/s41587-024-02411-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1192-1201},
  shortjournal = {Nature Biotech.},
  title        = {Coordinated, multicellular patterns of transcriptional variation that stratify patient cohorts are revealed by tensor decomposition},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Comprehensive genome analysis and variant detection at scale using DRAGEN. <em>NBT</em>, <em>43</em>(7), 1177-1191. (<a href='https://doi.org/10.1038/s41587-024-02382-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Research and medical genomics require comprehensive, scalable methods for the discovery of novel disease targets, evolutionary drivers and genetic markers with clinical significance. This necessitates a framework to identify all types of variants independent of their size or location. Here we present DRAGEN, which uses multigenome mapping with pangenome references, hardware acceleration and machine learning-based variant detection to provide insights into individual genomes, with ~30 min of computation time from raw reads to variant detection. DRAGEN outperforms current state-of-the-art methods in speed and accuracy across all variant types (single-nucleotide variations, insertions or deletions, short tandem repeats, structural variations and copy number variations) and incorporates specialized methods for analysis of medically relevant genes. We demonstrate the performance of DRAGEN across 3,202 whole-genome sequencing datasets by generating fully genotyped multisample variant call format files and demonstrate its scalability, accuracy and innovation to further advance the integration of comprehensive genomics. Overall, DRAGEN marks a major milestone in sequencing data analysis and will provide insights across various diseases, including Mendelian and rare diseases, with a highly comprehensive and scalable platform. DRAGEN rapidly identifies diverse types of genetic variants.},
  archive      = {J_NBT},
  author       = {Behera, Sairam and Catreux, Severine and Rossi, Massimiliano and Truong, Sean and Huang, Zhuoyi and Ruehle, Michael and Visvanath, Arun and Parnaby, Gavin and Roddey, Cooper and Onuchic, Vitor and Finocchio, Andrea and Cameron, Daniel L. and English, Adam and Mehtalia, Shyamal and Han, James and Mehio, Rami and Sedlazeck, Fritz J.},
  doi          = {10.1038/s41587-024-02382-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1177-1191},
  shortjournal = {Nature Biotech.},
  title        = {Comprehensive genome analysis and variant detection at scale using DRAGEN},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Removal of TREX1 activity enhances CRISPR–Cas9-mediated homologous recombination. <em>NBT</em>, <em>43</em>(7), 1168-1176. (<a href='https://doi.org/10.1038/s41587-024-02356-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR–Cas9-mediated homology-directed repair (HDR) can introduce desired mutations at targeted genomic sites, but achieving high efficiencies is a major hurdle in many cell types, including cells deficient in DNA repair activity. In this study, we used genome-wide screening in Fanconi anemia patient lymphoblastic cell lines to uncover suppressors of CRISPR–Cas9-mediated HDR. We found that a single exonuclease, TREX1, reduces HDR efficiency when the repair template is a single-stranded or linearized double-stranded DNA. TREX1 expression serves as a biomarker for CRISPR–Cas9-mediated HDR in that the high TREX1 expression present in many different cell types (such as U2OS, Jurkat, MDA-MB-231 and primary T cells as well as hematopoietic stem and progenitor cells) predicts poor HDR. Here we demonstrate rescue of HDR efficiency (ranging from two-fold to eight-fold improvement) either by TREX1 knockout or by the use of single-stranded DNA templates chemically protected from TREX1 activity. Our data explain why some cell types are easier to edit than others and indicate routes for increasing CRISPR–Cas9-mediated HDR in TREX1-expressing contexts. Homologous recombination in CRISPR–Cas9 genome editing is increased by blocking an exonuclease activity.},
  archive      = {J_NBT},
  author       = {Karasu, Mehmet E. and Toufektchan, Eléonore and Chen, Yanyang and Albertelli, Alessandra and Cullot, Grégoire and Maciejowski, John and Corn, Jacob E.},
  doi          = {10.1038/s41587-024-02356-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1168-1176},
  shortjournal = {Nature Biotech.},
  title        = {Removal of TREX1 activity enhances CRISPR–Cas9-mediated homologous recombination},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rapid generation of long, chemically modified pegRNAs for prime editing. <em>NBT</em>, <em>43</em>(7), 1156-1167. (<a href='https://doi.org/10.1038/s41587-024-02394-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The editing efficiencies of prime editing (PE) using ribonucleoprotein (RNP) and RNA delivery are not optimal due to the challenges in solid-phase synthesis of long PE guide RNA (pegRNA) (&gt;125 nt). Here, we develop an efficient, rapid and cost-effective method for generating chemically modified pegRNA (125–145 nt) and engineered pegRNA (epegRNA) (170–190 nt). We use an optimized splint ligation approach and achieve approximately 90% production efficiency for these RNAs, referred to as L-pegRNA and L-epegRNA. L-epegRNA demonstrates enhanced editing efficiencies across various cell lines and human primary cells with improvements of up to more than tenfold when using RNP delivery and several hundredfold with RNA delivery of PE, compared to epegRNA produced by in vitro transcription. L-epegRNA-mediated RNP delivery also outperforms plasmid-encoded PE in most comparisons. Our study provides a solution to obtaining high-quality pegRNA and epegRNA with desired chemical modifications, paving the way for the use of PE in therapeutics and various other fields. Long, chemically modified pegRNAs are rapidly synthesized using an optimized splint ligation approach.},
  archive      = {J_NBT},
  author       = {Lei, Xinlin and Huang, Anhui and Chen, Didi and Wang, Xuebin and Ji, Ruijin and Wang, Jinlin and Zhang, Yizhou and Zhang, Yuming and Lu, Shuhan and Zhang, Kun and Chen, Qiubing and Zhang, Ying and Yin, Hao},
  doi          = {10.1038/s41587-024-02394-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1156-1167},
  shortjournal = {Nature Biotech.},
  title        = {Rapid generation of long, chemically modified pegRNAs for prime editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bacteriophage λ exonuclease and a 5′-phosphorylated DNA guide allow PAM-independent targeting of double-stranded nucleic acids. <em>NBT</em>, <em>43</em>(7), 1144-1155. (<a href='https://doi.org/10.1038/s41587-024-02388-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequence-specific recognition of double-stranded nucleic acids is essential for molecular diagnostics and in situ imaging. Clustered regularly interspaced short palindromic repeats and Cas systems rely on protospacer-adjacent motif (PAM)-dependent double-stranded DNA (dsDNA) recognition, limiting the range of targetable sequences and leading to undesired off-target effects. Using single-molecule fluorescence resonance energy transfer analysis, we discover the enzymatic activity of bacteriophage λ exonuclease (λExo). We show binding of 5′-phosphorylated single-stranded DNA (pDNA) to complementary regions on dsDNA and DNA–RNA duplexes, without the need for a PAM-like motif. Upon binding, the λExo–pDNA system catalytically digests the pDNA into nucleotides in the presence of Mg2+. This process is sensitive to mismatches within a wide range of the pDNA-binding region, resulting in exceptional sequence specificity and reduced off-target effects in various applications. The absence of a requirement for a specific motif such as a PAM sequence greatly broadens the range of targets. We demonstrate that the λExo–pDNA system is a versatile tool for molecular diagnostics, DNA computing and gene imaging applications. A λ exonuclease system exhibits protospacer-adjacent motif-independent DNA-binding specificity, enabling diagnostic and imaging applications.},
  archive      = {J_NBT},
  author       = {Fu, Shengnan and Li, Junjie and Chen, Jing and Zhang, Linghao and Liu, Jiajia and Liu, Huiyu and Su, Xin},
  doi          = {10.1038/s41587-024-02388-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1144-1155},
  shortjournal = {Nature Biotech.},
  title        = {Bacteriophage λ exonuclease and a 5′-phosphorylated DNA guide allow PAM-independent targeting of double-stranded nucleic acids},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Chemical and topological design of multicapped mRNA and capped circular RNA to augment translation. <em>NBT</em>, <em>43</em>(7), 1128-1143. (<a href='https://doi.org/10.1038/s41587-024-02393-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein and vaccine therapies based on mRNA would benefit from an increase in translation capacity. Here, we report a method to augment translation named ligation-enabled mRNA–oligonucleotide assembly (LEGO). We systematically screen different chemotopological motifs and find that a branched mRNA cap effectively initiates translation on linear or circular mRNAs without internal ribosome entry sites. Two types of chemical modification, locked nucleic acid (LNA) N7-methylguanosine modifications on the cap and LNA + 5 × 2′ O-methyl on the 5′ untranslated region, enhance RNA–eukaryotic translation initiation factor (eIF4E–eIF4G) binding and RNA stability against decapping in vitro. Through multidimensional chemotopological engineering of dual-capped mRNA and capped circular RNA, we enhanced mRNA protein production by up to tenfold in vivo, resulting in 17-fold and 3.7-fold higher antibody production after prime and boost doses in a severe acute respiratory syndrome coronavirus 2 vaccine setting, respectively. The LEGO platform opens possibilities to design unnatural RNA structures and topologies beyond canonical linear and circular RNAs for both basic research and therapeutic applications. mRNA cap designs boost translation &gt;10-fold, with potential benefits for RNA therapies.},
  archive      = {J_NBT},
  author       = {Chen, Hongyu and Liu, Dangliang and Aditham, Abhishek and Guo, Jianting and Huang, Jiahao and Kostas, Franklin and Maher, Kamal and Friedrich, Mirco J. and Xavier, Ramnik J. and Zhang, Feng and Wang, Xiao},
  doi          = {10.1038/s41587-024-02393-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1128-1143},
  shortjournal = {Nature Biotech.},
  title        = {Chemical and topological design of multicapped mRNA and capped circular RNA to augment translation},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficient genetic code expansion without host genome modifications. <em>NBT</em>, <em>43</em>(7), 1116-1127. (<a href='https://doi.org/10.1038/s41587-024-02385-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Supplementing translation with noncanonical amino acids (ncAAs) can yield protein sequences with new-to-nature functions but existing ncAA incorporation strategies suffer from low efficiency and context dependence. We uncover codon usage as a previously unrecognized contributor to efficient genetic code expansion using non-native codons. Relying only on conventional Escherichia coli strains with native ribosomes, we develop a plasmid-based codon compression strategy that minimizes context dependence and improves ncAA incorporation at quadruplet codons. We confirm that this strategy is compatible with all known genetic code expansion resources, which allowed us to identify 12 mutually orthogonal transfer RNA (tRNA)–synthetase pairs. Enabled by these findings, we evolved and optimized five tRNA–synthetase pairs to incorporate a broad repertoire of ncAAs at orthogonal quadruplet codons. Lastly, we extend these resources to an in vivo biosynthesis platform that can readily create &gt;100 new-to-nature peptide macrocycles bearing up to three unique ncAAs. Our approach will accelerate innovations in multiplexed genetic code expansion and the discovery of chemically diverse biomolecules. Noncanonical amino acids are efficiently incorporated into proteins by optimizing mRNA codon usage.},
  archive      = {J_NBT},
  author       = {Costello, Alan and Peterson, Alexander A. and Lanster, David L. and Li, Zhiyi and Carver, Gavriela D. and Badran, Ahmed H.},
  doi          = {10.1038/s41587-024-02385-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1116-1127},
  shortjournal = {Nature Biotech.},
  title        = {Efficient genetic code expansion without host genome modifications},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mapping multimodal phenotypes to perturbations in cells and tissue with CRISPRmap. <em>NBT</em>, <em>43</em>(7), 1101-1115. (<a href='https://doi.org/10.1038/s41587-024-02386-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unlike sequencing-based methods, which require cell lysis, optical pooled genetic screens enable investigation of spatial phenotypes, including cell morphology, protein subcellular localization, cell–cell interactions and tissue organization, in response to targeted CRISPR perturbations. Here we report a multimodal optical pooled CRISPR screening method, which we call CRISPRmap. CRISPRmap combines in situ CRISPR guide-identifying barcode readout with multiplexed immunofluorescence and RNA detection. Barcodes are detected and read out through combinatorial hybridization of DNA oligos, enhancing barcode detection efficiency. CRISPRmap enables in situ barcode readout in cell types and contexts that were elusive to conventional optical pooled screening, including cultured primary cells, embryonic stem cells, induced pluripotent stem cells, derived neurons and in vivo cells in a tissue context. We conducted a screen in a breast cancer cell line of the effects of DNA damage repair gene variants on cellular responses to commonly used cancer therapies, and we show that optical phenotyping pinpoints likely pathogenic patient-derived mutations that were previously classified as variants of unknown clinical significance. Optical pooled CRISPR screening is improved by combinatorial oligo hybridization for barcode detection.},
  archive      = {J_NBT},
  author       = {Gu, Jiacheng and Iyer, Abhishek and Wesley, Ben and Taglialatela, Angelo and Leuzzi, Giuseppe and Hangai, Sho and Decker, Aubrianna and Gu, Ruoyu and Klickstein, Naomi and Shuai, Yuanlong and Jankovic, Kristina and Parker-Burns, Lucy and Jin, Yinuo and Zhang, Jia Yi and Hong, Justin and Niu, Xiang and Costa, Jonathon A. and Pezet, Mikael G. and Chou, Jacqueline and Chen, Cao ‘Claire’ and Paiva, Margaret and Snoeck, Hans-Willem and Landau, Dan A. and Azizi, Elham and Chan, Edmond M. and Ciccia, Alberto and Gaublomme, Jellert T.},
  doi          = {10.1038/s41587-024-02386-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1101-1115},
  shortjournal = {Nature Biotech.},
  title        = {Mapping multimodal phenotypes to perturbations in cells and tissue with CRISPRmap},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiplexed, image-based pooled screens in primary cells and tissues with PerturbView. <em>NBT</em>, <em>43</em>(7), 1091-1100. (<a href='https://doi.org/10.1038/s41587-024-02391-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optical pooled screening (OPS) is a scalable method for linking image-based phenotypes with cellular perturbations. However, it has thus far been restricted to relatively low-plex phenotypic readouts in cancer cell lines in culture due to limitations associated with in situ sequencing of perturbation barcodes. Here, we develop PerturbView, an OPS technology that leverages in vitro transcription to amplify barcodes before in situ sequencing, enabling screens with highly multiplexed phenotypic readouts across diverse systems, including primary cells and tissues. We demonstrate PerturbView in induced pluripotent stem cell-derived neurons, primary immune cells and tumor tissue sections from animal models. In a screen of immune signaling pathways in primary bone marrow-derived macrophages, PerturbView uncovered both known and novel regulators of NF-κB signaling. Furthermore, we combine PerturbView with spatial transcriptomics in tissue sections from a mouse xenograft model, paving the way to in situ screens with rich optical and transcriptomic phenotypes. PerturbView broadens the scope of OPS to a wide range of models and applications. Optical pooled screening is enhanced by amplification of barcodes before in situ sequencing.},
  archive      = {J_NBT},
  author       = {Kudo, Takamasa and Meireles, Ana M. and Moncada, Reuben and Chen, Yushu and Wu, Ping and Gould, Joshua and Hu, Xiaoyu and Kornfeld, Opher and Jesudason, Rajiv and Foo, Conrad and Höckendorf, Burkhard and Corrada Bravo, Hector and Town, Jason P. and Wei, Runmin and Rios, Antonio and Chandrasekar, Vineethkrishna and Heinlein, Melanie and Chuong, Amy S. and Cai, Shuangyi and Lu, Cherry Sakura and Coelho, Paula and Mis, Monika and Celen, Cemre and Kljavin, Noelyn and Jiang, Jian and Richmond, David and Thakore, Pratiksha and Benito-Gutiérrez, Elia and Geiger-Schuller, Kathryn and Hleap, Jose Sergio and Kayagaki, Nobuhiko and de Sousa e Melo, Felipe and McGinnis, Lisa and Li, Bo and Singh, Avtar and Garraway, Levi and Rozenblatt-Rosen, Orit and Regev, Aviv and Lubeck, Eric},
  doi          = {10.1038/s41587-024-02391-0},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1091-1100},
  shortjournal = {Nature Biotech.},
  title        = {Multiplexed, image-based pooled screens in primary cells and tissues with PerturbView},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Scalable and unsupervised discovery from raw sequencing reads using SPLASH2. <em>NBT</em>, <em>43</em>(7), 1084-1090. (<a href='https://doi.org/10.1038/s41587-024-02381-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce SPLASH2, a fast, scalable implementation of SPLASH based on an efficient k-mer counting approach for regulated sequence variation detection in massive datasets from a wide range of sequencing technologies and biological contexts. We demonstrate biological discovery by SPLASH2 in single-cell RNA sequencing (RNA-seq) data and in bulk RNA-seq data from the Cancer Cell Line Encyclopedia, including unannotated alternative splicing in cancer transcriptomes and sensitive detection of circular RNA. SPLASH2 speeds up analysis of sequence variation in massive datasets.},
  archive      = {J_NBT},
  author       = {Kokot, Marek and Dehghannasiri, Roozbeh and Baharav, Tavor and Salzman, Julia and Deorowicz, Sebastian},
  doi          = {10.1038/s41587-024-02381-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1084-1090},
  shortjournal = {Nature Biotech.},
  title        = {Scalable and unsupervised discovery from raw sequencing reads using SPLASH2},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Improving engineered biological systems with electronics and microfluidics. <em>NBT</em>, <em>43</em>(7), 1067-1083. (<a href='https://doi.org/10.1038/s41587-025-02709-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hybrid engineered biological systems integrate electronics and microfluidics with engineered biological components, such as microbes or cell-free DNA-based systems, to effectively sense, act upon and report on biological environments. As these engineered biological systems become essential in addressing challenges in healthcare, environmental monitoring, remediation and agriculture, this Review offers an in-depth discussion of their applications, critical design choices and challenges. Alongside an overview of the state of the field, we present a classification framework aimed at helping researchers make informed and optimized design decisions tailored to the intended biological application. Furthermore, we outline how the development of cyber-secure biological systems could enhance the security and functionality of engineered biological platforms. Last, we introduce a ‘Living Roadmap’ ( https://www.programmingbiology.org/csbs ) to dynamically reflect the field’s progress and support continuous monitoring of future advancements. A classification of hybrid engineered biological systems is presented to support researchers in making design choices that expand their use.},
  archive      = {J_NBT},
  author       = {Yazicigil, Rabia Tugce and Bali, Akshaya and Caygara, Dilara and Densmore, Douglas},
  doi          = {10.1038/s41587-025-02709-6},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1067-1083},
  shortjournal = {Nature Biotech.},
  title        = {Improving engineered biological systems with electronics and microfluidics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Targeting double-stranded nucleic acids using the λExo–pDNA system. <em>NBT</em>, <em>43</em>(7), 1064-1065. (<a href='https://doi.org/10.1038/s41587-024-02409-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {λ exonuclease (λExo) binds 5′-phosphorylated single-stranded DNA (pDNA) at complementary regions on double-stranded DNA and DNA–RNA duplexes under ambient conditions without a PAM-like motif. In the presence of Mg2+, λExo then digests the pDNA into nucleotides.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02409-7},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1064-1065},
  shortjournal = {Nature Biotech.},
  title        = {Targeting double-stranded nucleic acids using the λExo–pDNA system},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Recoded gene circuits for multiplexed genetic code expansion. <em>NBT</em>, <em>43</em>(7), 1062-1063. (<a href='https://doi.org/10.1038/s41587-024-02387-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quadruplet codons allow multiplexing of non-canonical amino acids within single polypeptides in living cells. We show that including high-usage triplet codons after quadruplet codons can improve their decoding efficiency in genetic circuits, which allowed us to develop a system for the programmable biosynthesis of exotic macrocyclic peptides in cells.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02387-w},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1062-1063},
  shortjournal = {Nature Biotech.},
  title        = {Recoded gene circuits for multiplexed genetic code expansion},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Building better mRNA for therapeutics. <em>NBT</em>, <em>43</em>(7), 1058-1060. (<a href='https://doi.org/10.1038/s41587-024-02424-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Redesigning mRNA with chemo-topological strategies improves its stability and translation efficiency, paving the way for more effective mRNA therapeutics.},
  archive      = {J_NBT},
  author       = {Liu, Bei and Pan, Tao},
  doi          = {10.1038/s41587-024-02424-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1058-1060},
  shortjournal = {Nature Biotech.},
  title        = {Building better mRNA for therapeutics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advances in optical pooled screening to map spatial complexity. <em>NBT</em>, <em>43</em>(7), 1055-1057. (<a href='https://doi.org/10.1038/s41587-024-02434-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two major advances in optical pooled screening improve substantially on sensitivity and robustness, expanding its applicability to a broader range of biological contexts.},
  archive      = {J_NBT},
  author       = {Kahnwald, Maurice and Mählen, Marius and Oost, Koen C. and Liberali, Prisca},
  doi          = {10.1038/s41587-024-02434-6},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1055-1057},
  shortjournal = {Nature Biotech.},
  title        = {Advances in optical pooled screening to map spatial complexity},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vascular organoids recover blood flow in vivo. <em>NBT</em>, <em>43</em>(7), 1054. (<a href='https://doi.org/10.1038/s41587-025-02747-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02747-0},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1054},
  shortjournal = {Nature Biotech.},
  title        = {Vascular organoids recover blood flow in vivo},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Profiling translated regions in viral genomes at scale. <em>NBT</em>, <em>43</em>(7), 1054. (<a href='https://doi.org/10.1038/s41587-025-02748-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02748-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1054},
  shortjournal = {Nature Biotech.},
  title        = {Profiling translated regions in viral genomes at scale},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Computationally designed enzymes show potent catalytic activity. <em>NBT</em>, <em>43</em>(7), 1054. (<a href='https://doi.org/10.1038/s41587-025-02751-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02751-4},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1054},
  shortjournal = {Nature Biotech.},
  title        = {Computationally designed enzymes show potent catalytic activity},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Radiopharmaceuticals. <em>NBT</em>, <em>43</em>(7), 1053. (<a href='https://doi.org/10.1038/s41587-025-02728-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to radiopharmaceutical compositions, preparation methods and delivery systems.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02728-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1053},
  shortjournal = {Nature Biotech.},
  title        = {Radiopharmaceuticals},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The strength and importance of government-funded patents for approved drugs. <em>NBT</em>, <em>43</em>(7), 1050-1052. (<a href='https://doi.org/10.1038/s41587-025-02724-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the strength and importance of US government-sponsored patents can help to limit high drug prices as well as constitute a justification to enact reasonable pricing clauses and exercise march-in rights.},
  archive      = {J_NBT},
  author       = {Gabriele, Sarah M. E. and Martin, Matthew J. and Kesselheim, Aaron S. and Tu, S. Sean},
  doi          = {10.1038/s41587-025-02724-7},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1050-1052},
  shortjournal = {Nature Biotech.},
  title        = {The strength and importance of government-funded patents for approved drugs},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Platform solutions for commercial challenges to expanding patient access and making gene editing sustainable. <em>NBT</em>, <em>43</em>(7), 1047-1049. (<a href='https://doi.org/10.1038/s41587-025-02744-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Platform-based approaches for gene-editing therapies could markedly improve development efficiency, reduce costs and increase access for patients with rare diseases. Although gene editing has shown remarkable clinical success for a small number of Mendelian disease indications, broader adoption faces substantial hurdles. We propose strategies to overcome these challenges through modular platforms for nonclinical and chemistry, manufacturing and controls (CMC) data reuse, risk-based manufacturing quality, and streamlined umbrella clinical trials for regulatory efficiency and accelerated approval.},
  archive      = {J_NBT},
  author       = {Kassim, Sadik H. and Urnov, Fyodor and Musunuru, Kiran and Lee, Ann and Barrera, Luis and Stetkiewicz, Pamela and Bruno, Julianne and Hewitt, Matthew and Lister, Troy and Malech, Harry and Gasch, Lindsay and Diver, Matt and Gertler, Nicholas and Grignon, Felix and Le, Audrey and Lehmicke, Michael and Horwitz, Edwin M. and Liu, David R. and Lembong, Josephine and Almendro-Navarro, Vanessa},
  doi          = {10.1038/s41587-025-02744-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1047-1049},
  shortjournal = {Nature Biotech.},
  title        = {Platform solutions for commercial challenges to expanding patient access and making gene editing sustainable},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A new impact framework for investors in life sciences bridges financial returns and patient outcomes. <em>NBT</em>, <em>43</em>(7), 1041-1046. (<a href='https://doi.org/10.1038/s41587-025-02726-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new model for quantifying the impact of capital investments in life science interventions could help investors, policymakers and healthcare stakeholders to assess, monitor and optimize the societal impact of their investments.},
  archive      = {J_NBT},
  author       = {Brainin, Philip},
  doi          = {10.1038/s41587-025-02726-5},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1041-1046},
  shortjournal = {Nature Biotech.},
  title        = {A new impact framework for investors in life sciences bridges financial returns and patient outcomes},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Defining and benchmarking open problems in single-cell analysis. <em>NBT</em>, <em>43</em>(7), 1035-1040. (<a href='https://doi.org/10.1038/s41587-025-02694-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Luecken, Malte D. and Gigante, Scott and Burkhardt, Daniel B. and Cannoodt, Robrecht and Strobl, Daniel C. and Markov, Nikolay S. and Zappia, Luke and Palla, Giovanni and Lewis, Wesley and Dimitrov, Daniel and Vinyard, Michael E. and Magruder, D. S. and Mueller, Michaela F. and Andersson, Alma and Dann, Emma and Qin, Qian and Otto, Dominik J. and Klein, Michal and Botvinnik, Olga Borisovna and Deconinck, Louise and Waldrant, Kai and Yasa, Sai Nirmayi and Szałata, Artur and Benz, Andrew and Li, Zhijian and Bloom, Jonathan M. and Pisco, Angela Oliveira and Saez-Rodriguez, Julio and Wulsin, Drausin and Pinello, Luca and Saeys, Yvan and Theis, Fabian J. and Krishnaswamy, Smita},
  doi          = {10.1038/s41587-025-02694-w},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1035-1040},
  shortjournal = {Nature Biotech.},
  title        = {Defining and benchmarking open problems in single-cell analysis},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech financing: Divide and reset. <em>NBT</em>, <em>43</em>(7), 1028-1034. (<a href='https://doi.org/10.1038/s41587-025-02723-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Today’s uniquely challenging biotech climate may result in a more resilient sector.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-025-02723-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1028-1034},
  shortjournal = {Nature Biotech.},
  title        = {Biotech financing: Divide and reset},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ocean-grown bioplastics. <em>NBT</em>, <em>43</em>(7), 1027. (<a href='https://doi.org/10.1038/s41587-025-02746-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-025-02746-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1027},
  shortjournal = {Nature Biotech.},
  title        = {Ocean-grown bioplastics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(7), 1026. (<a href='https://doi.org/10.1038/s41587-025-02749-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02749-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1026},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Eli lilly in $650M pact to boost muscle. <em>NBT</em>, <em>43</em>(7), 1025. (<a href='https://doi.org/10.1038/s41587-025-02753-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02753-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1025},
  shortjournal = {Nature Biotech.},
  title        = {Eli lilly in $650M pact to boost muscle},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). FDA approves first-in-class HIV prevention shot. <em>NBT</em>, <em>43</em>(7), 1024. (<a href='https://doi.org/10.1038/s41587-025-02756-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02756-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1024},
  shortjournal = {Nature Biotech.},
  title        = {FDA approves first-in-class HIV prevention shot},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). De-extinction: Debate continues around tech to ‘resurrect’ lost species. <em>NBT</em>, <em>43</em>(7), 1023-1026. (<a href='https://doi.org/10.1038/s41587-025-02745-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Experts disagree about the extent to which hybrids based on dire wolves reflect the original extinct species, and the broader role of de-extinction technologies in ecosystem restoration.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-025-02745-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1023-1026},
  shortjournal = {Nature Biotech.},
  title        = {De-extinction: Debate continues around tech to ‘resurrect’ lost species},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mast cells drive blueprint mega-deal. <em>NBT</em>, <em>43</em>(7), 1021. (<a href='https://doi.org/10.1038/s41587-025-02750-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02750-5},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1021},
  shortjournal = {Nature Biotech.},
  title        = {Mast cells drive blueprint mega-deal},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precision gene editing medicine makes history, and it’s just getting started. <em>NBT</em>, <em>43</em>(7), 1019-1022. (<a href='https://doi.org/10.1038/s41587-025-02741-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite their astounding success, custom-made base editors and prime editors will need time to broaden their clinical impact.},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-025-02741-6},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1019-1022},
  shortjournal = {Nature Biotech.},
  title        = {Precision gene editing medicine makes history, and it’s just getting started},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Clinical trials gain intelligence. <em>NBT</em>, <em>43</em>(7), 1017-1018. (<a href='https://doi.org/10.1038/s41587-025-02754-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As governments around the world begin to embrace artificial intelligence in healthcare, clinical trials are entering a new phase — faster, smarter and more inclusive. With regulatory momentum building to adapt new technologies, artificial intelligence is no longer just a promise for the future but a powerful tool reshaping how trials are designed, conducted and analyzed today.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02754-1},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {1017-1018},
  shortjournal = {Nature Biotech.},
  title        = {Clinical trials gain intelligence},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(6), 1016. (<a href='https://doi.org/10.1038/s41587-025-02703-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02703-y},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {1016},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Thinking about a move from academia to industry? look before you leap. <em>NBT</em>, <em>43</em>(6), 1013-1015. (<a href='https://doi.org/10.1038/s41587-025-02700-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To those considering a change in careers from academia to the pharmaceutical industry, here are the keys to a successful transition.},
  archive      = {J_NBT},
  author       = {Hait, William N.},
  doi          = {10.1038/s41587-025-02700-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {1013-1015},
  shortjournal = {Nature Biotech.},
  title        = {Thinking about a move from academia to industry? look before you leap},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Precise, predictable multi-nucleotide deletions in rice and wheat using APOBEC–Cas9. <em>NBT</em>, <em>43</em>(6), 1012. (<a href='https://doi.org/10.1038/s41587-025-02674-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Wang, Shengxing and Zong, Yuan and Lin, Qiupeng and Zhang, Huawei and Chai, Zhuangzhuang and Zhang, Dandan and Chen, Kunling and Qiu, Jin-Long and Gao, Caixia},
  doi          = {10.1038/s41587-025-02674-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {1012},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Precise, predictable multi-nucleotide deletions in rice and wheat using APOBEC–Cas9},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Precise base editing in rice, wheat and maize with a cas9-cytidine deaminase fusion. <em>NBT</em>, <em>43</em>(6), 1011. (<a href='https://doi.org/10.1038/s41587-025-02671-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Zong, Yuan and Wang, Yanpeng and Li, Chao and Zhang, Rui and Chen, Kunling and Ran, Yidong and Qiu, Jin-Long and Wang, Daowen and Gao, Caixia},
  doi          = {10.1038/s41587-025-02671-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {1011},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Precise base editing in rice, wheat and maize with a cas9-cytidine deaminase fusion},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Efficient C-to-T base editing in plants using a fusion of nCas9 and human APOBEC3A. <em>NBT</em>, <em>43</em>(6), 1011. (<a href='https://doi.org/10.1038/s41587-025-02676-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Zong, Yuan and Song, Qianna and Li, Chao and Jin, Shuai and Zhang, Dingbo and Wang, Yanpeng and Qiu, Jin-Long and Gao, Caixia},
  doi          = {10.1038/s41587-025-02676-y},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {1011},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Efficient C-to-T base editing in plants using a fusion of nCas9 and human APOBEC3A},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Large-scale discovery of chromatin dysregulation induced by oncofusions and other protein-coding variants. <em>NBT</em>, <em>43</em>(6), 996-1010. (<a href='https://doi.org/10.1038/s41587-024-02347-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Population-scale databases have expanded to millions of protein-coding variants, yet insight into their mechanistic consequences has lagged. Here we present PROD-ATAC, a high-throughput method for discovering the effects of protein-coding variants on chromatin regulation. A pooled variant library is expressed in a disease-agnostic cell line, and single-cell assay for transposase-accessible chromatin resolves each variant’s effect on the chromatin landscape. Using PROD-ATAC, we characterized the effects of more than 100 oncofusions (cancer-causing chimeric proteins) and controls and revealed that chromatin remodeling is common to fusions spanning an enormous range of fusion frequencies. Furthermore, fusion-induced dysregulation can be context agnostic, as observed mechanisms often overlapped with cancer and cell-type-specific prior knowledge. We also showed that gain-of-function activity is common among oncofusions. This work begins to outline a global map of fusion-induced chromatin alterations. We suggest that there might be convergent mechanisms among disparate oncofusions and shared modes of dysregulation among fusions present in tumors at different frequencies. PROD-ATAC is generalizable to any set of protein-coding variants. The effects of oncofusions and other protein variants on chromatin remodeling are identified by PROD-ATAC.},
  archive      = {J_NBT},
  author       = {Frenkel, Max and Corban, James E. and Hujoel, Margaux L. A. and Morris, Zachary and Raman, Srivatsan},
  doi          = {10.1038/s41587-024-02347-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {996-1010},
  shortjournal = {Nature Biotech.},
  title        = {Large-scale discovery of chromatin dysregulation induced by oncofusions and other protein-coding variants},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fast, sensitive detection of protein homologs using deep dense retrieval. <em>NBT</em>, <em>43</em>(6), 983-995. (<a href='https://doi.org/10.1038/s41587-024-02353-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The identification of protein homologs in large databases using conventional methods, such as protein sequence comparison, often misses remote homologs. Here, we offer an ultrafast, highly sensitive method, dense homolog retriever (DHR), for detecting homologs on the basis of a protein language model and dense retrieval techniques. Its dual-encoder architecture generates different embeddings for the same protein sequence and easily locates homologs by comparing these representations. Its alignment-free nature improves speed and the protein language model incorporates rich evolutionary and structural information within DHR embeddings. DHR achieves a &gt;10% increase in sensitivity compared to previous methods and a &gt;56% increase in sensitivity at the superfamily level for samples that are challenging to identify using alignment-based approaches. It is up to 22 times faster than traditional methods such as PSI-BLAST and DIAMOND and up to 28,700 times faster than HMMER. The new remote homologs exclusively found by DHR are useful for revealing connections between well-characterized proteins and improving our knowledge of protein evolution, structure and function. Protein homologs in large databases are rapidly identified with an alignment-free pipeline.},
  archive      = {J_NBT},
  author       = {Hong, Liang and Hu, Zhihang and Sun, Siqi and Tang, Xiangru and Wang, Jiuming and Tan, Qingxiong and Zheng, Liangzhen and Wang, Sheng and Xu, Sheng and King, Irwin and Gerstein, Mark and Li, Yu},
  doi          = {10.1038/s41587-024-02353-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {983-995},
  shortjournal = {Nature Biotech.},
  title        = {Fast, sensitive detection of protein homologs using deep dense retrieval},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Systematic identification of minor histocompatibility antigens predicts outcomes of allogeneic hematopoietic cell transplantation. <em>NBT</em>, <em>43</em>(6), 971-982. (<a href='https://doi.org/10.1038/s41587-024-02348-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cell alloreactivity against minor histocompatibility antigens (mHAgs)—polymorphic peptides resulting from donor–recipient (D–R) disparity at sites of genetic polymorphisms—is at the core of the therapeutic effect of allogeneic hematopoietic cell transplantation (allo-HCT). Despite the crucial role of mHAgs in graft-versus-leukemia (GvL) and graft-versus-host disease (GvHD) reactions, it remains challenging to consistently link patient-specific mHAg repertoires to clinical outcomes. Here we devise an analytic framework to systematically identify mHAgs, including their detection on HLA class I ligandomes and functional verification of their immunogenicity. The method relies on the integration of polymorphism detection by whole-exome sequencing of germline DNA from D–R pairs with organ-specific transcriptional- and proteome-level expression. Application of this pipeline to 220 HLA-matched allo-HCT D–R pairs demonstrated that total and organ-specific mHAg load could independently predict the occurrence of acute GvHD and chronic pulmonary GvHD, respectively, and defined promising GvL targets, confirmed in a validation cohort of 58 D–R pairs, for the prevention or treatment of post-transplant disease recurrence. The success of hematopoietic cell transplants is predicted by profiling minor histocompatibility antigens.},
  archive      = {J_NBT},
  author       = {Cieri, Nicoletta and Hookeri, Nidhi and Stromhaug, Kari and Li, Liang and Keating, Julia and Díaz-Fernández, Paula and Gómez-García de Soria, Valle and Stevens, Jonathan and Kfuri-Rubens, Raphael and Shao, Yiren and Kooshesh, Kameron A. and Powell, Kaila and Ji, Helen and Hernandez, Gabrielle M. and Abelin, Jennifer and Klaeger, Susan and Forman, Cleo and Clauser, Karl R. and Sarkizova, Siranush and Braun, David A. and Penter, Livius and Kim, Haesook T. and Lane, William J. and Oliveira, Giacomo and Kean, Leslie S. and Li, Shuqiang and Livak, Kenneth J. and Carr, Steven A. and Keskin, Derin B. and Muñoz-Calleja, Cecilia and Ho, Vincent T. and Ritz, Jerome and Soiffer, Robert J. and Neuberg, Donna and Stewart, Chip and Getz, Gad and Wu, Catherine J.},
  doi          = {10.1038/s41587-024-02348-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {971-982},
  shortjournal = {Nature Biotech.},
  title        = {Systematic identification of minor histocompatibility antigens predicts outcomes of allogeneic hematopoietic cell transplantation},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Rapid discovery of monoclonal antibodies by microfluidics-enabled FACS of single pathogen-specific antibody-secreting cells. <em>NBT</em>, <em>43</em>(6), 960-970. (<a href='https://doi.org/10.1038/s41587-024-02346-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Monoclonal antibodies are increasingly used to prevent and treat viral infections and are pivotal in pandemic response efforts. Antibody-secreting cells (ASCs; plasma cells and plasmablasts) are an excellent source of high-affinity antibodies with therapeutic potential. Current methods to study antigen-specific ASCs either have low throughput, require expensive and labor-intensive screening or are technically demanding and therefore not widely accessible. Here we present a straightforward technology for the rapid discovery of monoclonal antibodies from ASCs. Our approach combines microfluidic encapsulation of single cells into an antibody capture hydrogel with antigen bait sorting by conventional flow cytometry. With our technology, we screened millions of mouse and human ASCs and obtained monoclonal antibodies against severe acute respiratory syndrome coronavirus 2 with high affinity (&lt;1 pM) and neutralizing capacity (&lt;100 ng ml−1) in 2 weeks with a high hit rate (&gt;85% of characterized antibodies bound the target). By facilitating access to the underexplored ASC compartment, the approach enables efficient antibody discovery and immunological studies into the generation of protective antibodies. Antigen-specific antibodies secreted by plasma cells are rapidly identified using microfluidics and fluorescence-activated cell sorting (FACS).},
  archive      = {J_NBT},
  author       = {Fischer, Katrin and Lulla, Aleksei and So, Tsz Y. and Pereyra-Gerber, Pehuén and Raybould, Matthew I. J. and Kohler, Timo N. and Yam-Puc, Juan Carlos and Kaminski, Tomasz S. and Hughes, Robert and Pyeatt, Gwendolyn L. and Leiss-Maier, Florian and Brear, Paul and Matheson, Nicholas J. and Deane, Charlotte M. and Hyvönen, Marko and Thaventhiran, James E. D. and Hollfelder, Florian},
  doi          = {10.1038/s41587-024-02346-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {960-970},
  shortjournal = {Nature Biotech.},
  title        = {Rapid discovery of monoclonal antibodies by microfluidics-enabled FACS of single pathogen-specific antibody-secreting cells},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A CAR enhancer increases the activity and persistence of CAR t cells. <em>NBT</em>, <em>43</em>(6), 948-959. (<a href='https://doi.org/10.1038/s41587-024-02339-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although chimeric antigen receptor (CAR) T cell therapies have demonstrated promising clinical outcomes, durable remissions remain limited. To extend the efficacy of CAR T cells, we develop a CAR enhancer (CAR-E), comprising a CAR T cell antigen fused to an immunomodulatory molecule. Here we demonstrate this strategy using B cell maturation antigen (BCMA) CAR T cells for the treatment of multiple myeloma, with a CAR-E consisting of the BCMA fused to a low-affinity interleukin 2 (IL-2). This selectively induces IL-2 signaling in CAR T cells upon antigen–CAR binding, enhancing T cell activation and antitumor activity while reducing IL-2-associated toxicities. We show that the BCMA CAR-E selectively binds CAR T cells and increases CAR T cell proliferation, clearance of tumor cells and development of memory CAR T cells. The memory cells retain the ability to re-expand upon restimulation, effectively controlling tumor growth upon rechallenge. Mechanistic studies reveal the involvement of both CAR and IL-2 receptor endodomains in the CAR-E mechanism of action. The CAR-E approach avoids the need for specific engineering and enables CAR T cell therapy with lower cell doses. Chimeric antigen receptor T cell efficacy is enhanced with an interleukin 2-recruiting molecule.},
  archive      = {J_NBT},
  author       = {Rakhshandehroo, Taha and Mantri, Shreya R. and Moravej, Heydar and Louis, Benjamin B. V. and Salehi Farid, Ali and Munaretto, Leila and Regan, Kathryn and Khan, Radia M. M. and Wolff, Alexandra and Farkash, Zoe and Cong, Min and Kuhnast, Adrien and Nili, Ali and Lee, Uk-Jae and Allen, Harris H. and Berland, Lea and Simkova, Ester and Uslu, Safak C. and Tavakolpour, Soheil and Rowley, Jennifer E. and Codet, Elisabeth and Shahbazian, Haneyeh and Baral, Jessika and Pyrdol, Jason and Jacobson, Caron A. and Nadeem, Omar and Nia, Hadi T. and Wucherpfennig, Kai W. and Rashidian, Mohammad},
  doi          = {10.1038/s41587-024-02339-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {948-959},
  shortjournal = {Nature Biotech.},
  title        = {A CAR enhancer increases the activity and persistence of CAR t cells},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Self-stacked small molecules for ultrasensitive, substrate-free raman imaging in vivo. <em>NBT</em>, <em>43</em>(6), 936-947. (<a href='https://doi.org/10.1038/s41587-024-02342-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Raman spectroscopy using surface-enhanced Raman scattering (SERS) nanoprobes represents an ultrasensitive and high-precision technique for in vivo imaging. Clinical translation of SERS nanoprobes has been hampered by biosafety concerns about the metal substrates used to enhance Raman signals. We report a set of small molecules with bis-thienyl-substituted benzobisthiadiazole structures that enhance Raman signal through self-stacking rather than external substrates. In our technique, called stacking-induced charge transfer-enhanced Raman scattering (SICTERS), the self-stacked small molecules form an ordered spatial arrangement that enables three-dimensional charge transfer between neighboring molecules. The Raman scattering cross-section of SICTERS nanoprobes is 1350 times higher than that of conventional SERS gold nanoprobes of similar particle size. SICTERS outperforms SERS in terms of in vivo imaging sensitivity, resolution and depth. SICTERS is capable of noninvasive Raman imaging of blood and lymphatic vasculatures, which has not been achieved by SERS. SICTERS represents an alternative technique to enhance Raman scattering for guiding the design of ultrasensitive substrate-free Raman imaging probes. The sensitivity of in vivo Raman imaging is improved with self-stacked small molecules.},
  archive      = {J_NBT},
  author       = {Gao, Shuai and Zhang, Yongming and Cui, Kai and Zhang, Sihang and Qiu, Yuanyuan and Liao, Yunhui and Wang, Haoze and Yu, Sheng and Ma, Liyang and Chen, Hongzhuan and Ji, Minbiao and Fang, Xiaohong and Lu, Wei and Xiao, Zeyu},
  doi          = {10.1038/s41587-024-02342-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {936-947},
  shortjournal = {Nature Biotech.},
  title        = {Self-stacked small molecules for ultrasensitive, substrate-free raman imaging in vivo},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases. <em>NBT</em>, <em>43</em>(6), 923-935. (<a href='https://doi.org/10.1038/s41587-024-02324-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome editing technologies based on DNA-dependent polymerases (DDPs) could offer several benefits compared with other types of editors to install diverse edits. Here, we develop click editing, a genome writing platform that couples the advantageous properties of DDPs with RNA-programmable nickases to permit the installation of a range of edits, including substitutions, insertions and deletions. Click editors (CEs) leverage the ‘click’-like bioconjugation ability of HUH endonucleases with single-stranded DNA substrates to covalently tether ‘click DNA’ (clkDNA) templates encoding user-specifiable edits at targeted genomic loci. Through iterative optimization of the modular components of CEs and their clkDNAs, we demonstrate the ability to install precise genome edits with minimal indels in diverse immortalized human cell types and primary fibroblasts with precise editing efficiencies of up to ~30%. Editing efficiency can be improved by rapidly screening clkDNA oligonucleotides with various modifications, including repair-evading substitutions. Click editing is a precise and versatile genome editing approach for diverse biological applications. Click editing uses DNA polymerases, HUH endonucleases and oligonucleotide templates for genome modification.},
  archive      = {J_NBT},
  author       = {Ferreira da Silva, Joana and Tou, Connor J. and King, Emily M. and Eller, Madeline L. and Rufino-Ramos, David and Ma, Linyuan and Cromwell, Christopher R. and Metovic, Jasna and Benning, Friederike M. C. and Chao, Luke H. and Eichler, Florian S. and Kleinstiver, Benjamin P.},
  doi          = {10.1038/s41587-024-02324-x},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {923-935},
  shortjournal = {Nature Biotech.},
  title        = {Click editing enables programmable genome writing using DNA polymerases and HUH endonucleases},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An experimental census of retrons for DNA production and genome editing. <em>NBT</em>, <em>43</em>(6), 914-922. (<a href='https://doi.org/10.1038/s41587-024-02384-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Retrons are bacterial immune systems that use reverse-transcribed DNA (RT-DNA) to detect phage infection. They are also deployed for genome editing, where they are modified so that the RT-DNA encodes an editing donor. Retrons are common in bacterial genomes, and thousands of unique retrons have been predicted bioinformatically. However, few have been characterized experimentally. We add to the corpus of experimentally studied retrons, finding 62 empirically determined, natural RT-DNAs that are not predictable from the retron sequence alone. We synthesize &gt;100 previously untested retrons to identify the natural sequence of RT-DNA they produce, quantify their RT-DNA production and test the relative efficacy of editing using retron-derived donors to edit bacterial, phage and human genomes. We observe large diversity in RT-DNA production and editing rates across retrons, finding that top-performing editors are drawn from a subset of the retron phylogeny and outperform those used in previous studies, reaching precise editing rates of up to 40% in human cells. Genome editing tools are developed through large-scale characterization of bacterial retrons.},
  archive      = {J_NBT},
  author       = {Khan, Asim G. and Rojas-Montero, Matías and González-Delgado, Alejandro and Lopez, Santiago C. and Fang, Rebecca F. and Crawford, Kate D. and Shipman, Seth L.},
  doi          = {10.1038/s41587-024-02384-z},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {914-922},
  shortjournal = {Nature Biotech.},
  title        = {An experimental census of retrons for DNA production and genome editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enhancing siRNA efficacy in vivo with extended nucleic acid backbones. <em>NBT</em>, <em>43</em>(6), 904-913. (<a href='https://doi.org/10.1038/s41587-024-02336-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Therapeutic small interfering RNA (siRNA) requires sugar and backbone modifications to inhibit nuclease degradation. However, metabolic stabilization by phosphorothioate (PS), the only backbone chemistry used clinically, may be insufficient for targeting extrahepatic tissues. To improve oligonucleotide stabilization, we report the discovery, synthesis and characterization of extended nucleic acid (exNA) consisting of a methylene insertion between the 5′-C and 5′-OH of a nucleoside. exNA incorporation is compatible with common oligonucleotide synthetic protocols and the PS backbone, provides stabilization against 3′ and 5′ exonucleases and is tolerated at multiple oligonucleotide positions. A combined exNA–PS backbone enhances resistance to 3′ exonuclease by ~32-fold over the conventional PS backbone and by &gt;1,000-fold over the natural phosphodiester backbone, improving tissue exposure, tissue accumulation and efficacy in mice, both systemically and in the brain. The improved efficacy and durability imparted by exNA may enable therapeutic interventions in extrahepatic tissues, both with siRNA and with other oligonucleotides such as CRISPR guide RNA, antisense oligonucleotides, mRNA and tRNA. siRNA stability and efficacy are improved by a backbone modification.},
  archive      = {J_NBT},
  author       = {Yamada, Ken and Hariharan, Vignesh N. and Caiazzi, Jillian and Miller, Rachael and Ferguson, Chantal M. and Sapp, Ellen and Fakih, Hassan H. and Tang, Qi and Yamada, Nozomi and Furgal, Raymond C. and Paquette, Joseph D. and Biscans, Annabelle and Bramato, Brianna M. and McHugh, Nicholas and Summers, Ashley and Lochmann, Clemens and Godinho, Bruno M. D. C. and Hildebrand, Samuel and Jackson, Samuel O. and Echeverria, Dimas and Hassler, Matthew R. and Alterman, Julia F. and DiFiglia, Marian and Aronin, Neil and Khvorova, Anastasia},
  doi          = {10.1038/s41587-024-02336-7},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {904-913},
  shortjournal = {Nature Biotech.},
  title        = {Enhancing siRNA efficacy in vivo with extended nucleic acid backbones},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Organoid culture promotes dedifferentiation of mouse myoblasts into stem cells capable of complete muscle regeneration. <em>NBT</em>, <em>43</em>(6), 889-903. (<a href='https://doi.org/10.1038/s41587-024-02344-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Experimental cell therapies for skeletal muscle conditions have shown little success, primarily because they use committed myogenic progenitors rather than true muscle stem cells, known as satellite cells. Here we present a method to generate in vitro-derived satellite cells (idSCs) from skeletal muscle tissue. When transplanted in small numbers into mouse muscle, mouse idSCs fuse into myofibers, repopulate the satellite cell niche, self-renew, support multiple rounds of muscle regeneration and improve force production on par with freshly isolated satellite cells in damaged skeletal muscle. We compared the epigenomic and transcriptional signatures between idSCs, myoblasts and satellite cells and used these signatures to identify core signaling pathways and genes that confer idSC functionality. Finally, from human muscle biopsies, we successfully generated satellite cell-like cells in vitro. After further development, idSCs may provide a scalable source of cells for the treatment of genetic muscle disorders, trauma-induced muscle damage and age-related muscle weakness. Mouse stem cells that regenerate muscle are derived by organoid dedifferentiation of myoblasts.},
  archive      = {J_NBT},
  author       = {Price, Feodor D. and Matyas, Mark N. and Gehrke, Andrew R. and Chen, William and Wolin, Erica A. and Holton, Kristina M. and Gibbs, Rebecca M. and Lee, Alice and Singu, Pooja S. and Sakakeeny, Jeffrey S. and Poteracki, James M. and Goune, Kelsey and Pfeiffer, Isabella T. and Boswell, Sarah A. and Sorger, Peter K. and Srivastava, Mansi and Pfaff, Kathleen Lindahl and Gussoni, Emanuela and Buchanan, Sean M. and Rubin, Lee L.},
  doi          = {10.1038/s41587-024-02344-7},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {889-903},
  shortjournal = {Nature Biotech.},
  title        = {Organoid culture promotes dedifferentiation of mouse myoblasts into stem cells capable of complete muscle regeneration},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enabling next-generation anaerobic cultivation through biotechnology to advance functional microbiome research. <em>NBT</em>, <em>43</em>(6), 878-888. (<a href='https://doi.org/10.1038/s41587-025-02660-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microbiomes are complex communities of microorganisms that are essential for biochemical processes on Earth and for the health of humans, animals and plants. Many environmental and host-associated microbiomes are dominated by anaerobic microbes, some of which cannot tolerate oxygen. Anaerobic microbial communities have been extensively studied over the last 20 years using molecular techniques, especially next-generation sequencing. However, there is a renewed interest in microbial cultivation because isolates provide the basis for understanding the taxonomic and functional units of biodiversity, elucidating novel biochemical pathways and the mechanisms underlying microbe–microbe and microbe–host interactions and opening new avenues for biotechnological and clinical applications. In this Perspective, we present areas of research and applications that will benefit from advancement in anaerobic microbial cultivation. We highlight key technical and infrastructural hurdles associated with the development and deployment of sophisticated cultivation workflows. Improving the performance of cultivation techniques will set new trends in functional microbiome research in the coming years. Overcoming key hurdles for the development of next-generation anaerobic cultivation methods could have major impacts on microbial community research and applications.},
  archive      = {J_NBT},
  author       = {Clavel, Thomas and Faber, Franziska and Groussin, Mathieu and Haller, Dirk and Overmann, Jörg and Pauvert, Charlie and Poyet, Mathilde and Selkrig, Joel and Stecher, Bärbel and Typas, Athanasios and Vehreschild, Maria J. G. T. and Westermann, Alexander J. and Wylensek, David and Maier, Lisa},
  doi          = {10.1038/s41587-025-02660-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {878-888},
  shortjournal = {Nature Biotech.},
  title        = {Enabling next-generation anaerobic cultivation through biotechnology to advance functional microbiome research},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Protein language model enables fast and sensitive remote homolog detection. <em>NBT</em>, <em>43</em>(6), 876-877. (<a href='https://doi.org/10.1038/s41587-024-02359-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To handle increasingly large protein databases, a new ultrafast, highly sensitive method — Dense Homolog Retriever (DHR) — detects remote homologs using dense retrieval and protein language models. Its alignment-free nature makes it much faster than traditional approaches, and the newly found remote homologs benefit our understanding of protein evolution, structure and function.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02359-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {876-877},
  shortjournal = {Nature Biotech.},
  title        = {Protein language model enables fast and sensitive remote homolog detection},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Framework for predicting alloreactivity in hematopoietic cell transplants. <em>NBT</em>, <em>43</em>(6), 874-875. (<a href='https://doi.org/10.1038/s41587-024-02354-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The therapeutic principle of allogeneic hematopoietic cell transplantation, one the most common forms of cancer immunotherapy, is alloreactivity, yet its molecular determinants remain largely unknown. An analytical framework now enables personalized assessment of alloreactivity from whole-exome sequencing of donor–recipient pairs, to help with prognostication of disease relapse and immune-mediated complications.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02354-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {874-875},
  shortjournal = {Nature Biotech.},
  title        = {Framework for predicting alloreactivity in hematopoietic cell transplants},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Small molecules self-organized in an orderly manner to enhance raman signals. <em>NBT</em>, <em>43</em>(6), 872-873. (<a href='https://doi.org/10.1038/s41587-024-02350-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We have discovered an effect, termed stacking-induced intermolecular charge transfer-enhanced Raman scattering (SICTERS), that enhances the Raman signal intensities of small molecules by relying on their self-stacking rather than external substrates. This effect enables the design of substrate-free small-molecule probes for high-resolution, non-invasive transdermal Raman imaging of lymphatic drainage and microvessels.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02350-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {872-873},
  shortjournal = {Nature Biotech.},
  title        = {Small molecules self-organized in an orderly manner to enhance raman signals},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precise and versatile genome editing with click editors. <em>NBT</em>, <em>43</em>(6), 870-871. (<a href='https://doi.org/10.1038/s41587-024-02340-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed click editors, comprising HUH endonucleases, DNA-dependent DNA polymerases and CRISPR–Cas9 nickases, which together enable programmable precision genome engineering from simple DNA templates.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02340-x},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {870-871},
  shortjournal = {Nature Biotech.},
  title        = {Precise and versatile genome editing with click editors},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Dedifferentiating myoblasts back into a satellite cell state in vitro. <em>NBT</em>, <em>43</em>(6), 868-869. (<a href='https://doi.org/10.1038/s41587-024-02345-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Satellite cells, the stem cells of skeletal muscle, are responsible for muscle development and regeneration. Although low in abundance, satellite cells can be isolated from muscle but cannot be propagated successfully in culture in numbers needed for therapeutic use. We developed a method to generate cells with satellite cell characteristics from skeletal muscle organoid cultures.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02345-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {868-869},
  shortjournal = {Nature Biotech.},
  title        = {Dedifferentiating myoblasts back into a satellite cell state in vitro},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome editing with DNA-dependent polymerases. <em>NBT</em>, <em>43</em>(6), 865-867. (<a href='https://doi.org/10.1038/s41587-024-02372-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two CRISPR tools harness DNA-dependent polymerases for genome editing — and may overcome some limitations of prime editors.},
  archive      = {J_NBT},
  author       = {Cowan, Quinn T. and Komor, Alexis C.},
  doi          = {10.1038/s41587-024-02372-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {865-867},
  shortjournal = {Nature Biotech.},
  title        = {Genome editing with DNA-dependent polymerases},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Infant receives the first customized CRISPR therapy. <em>NBT</em>, <em>43</em>(6), 864. (<a href='https://doi.org/10.1038/s41587-025-02719-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02719-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {864},
  shortjournal = {Nature Biotech.},
  title        = {Infant receives the first customized CRISPR therapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A laboratory-evolved CRISPR-associated transposase adapts to human cells. <em>NBT</em>, <em>43</em>(6), 864. (<a href='https://doi.org/10.1038/s41587-025-02720-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02720-x},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {864},
  shortjournal = {Nature Biotech.},
  title        = {A laboratory-evolved CRISPR-associated transposase adapts to human cells},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). CRISPR-TO directs RNA to defined intracellular locations. <em>NBT</em>, <em>43</em>(6), 864. (<a href='https://doi.org/10.1038/s41587-025-02721-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02721-w},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {864},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR-TO directs RNA to defined intracellular locations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Organoids. <em>NBT</em>, <em>43</em>(6), 863. (<a href='https://doi.org/10.1038/s41587-025-02698-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to the production, culture and use of organoids.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02698-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {863},
  shortjournal = {Nature Biotech.},
  title        = {Organoids},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The new clinical investigation exclusivity: A substantial source of monopoly time for brand drugs. <em>NBT</em>, <em>43</em>(6), 857-862. (<a href='https://doi.org/10.1038/s41587-025-02696-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the United States, some non-patent exclusivities can have an outsized impact on the ability of generic competitors to enter the market and lower drug prices.},
  archive      = {J_NBT},
  author       = {Feldman, Robin and Schor, Gideon and Alsaffar, Ramy},
  doi          = {10.1038/s41587-025-02696-8},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {857-862},
  shortjournal = {Nature Biotech.},
  title        = {The new clinical investigation exclusivity: A substantial source of monopoly time for brand drugs},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Anatomy of the 23andMe fall and implications for consumer genomics. <em>NBT</em>, <em>43</em>(6), 854-856. (<a href='https://doi.org/10.1038/s41587-025-02683-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {23andMe’s bankruptcy serves as a moment of reflection for the direct-to-consumer (DTC) genomics industry. We analyzed 23andMe financial data and business practices to reveal the factors behind the fall of the company, once valued at US $6 billion and now being considered for acquisition by Regeneron for merely $250 million. Key challenges faced by 23andMe in monetizing its genomic data reveal that this information, at least in a typical DTC setting, is simply not worth that much.},
  archive      = {J_NBT},
  author       = {Erlich, Yaniv and Zielinski, Dina},
  doi          = {10.1038/s41587-025-02683-z},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {854-856},
  shortjournal = {Nature Biotech.},
  title        = {Anatomy of the 23andMe fall and implications for consumer genomics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Electro-microbial production techno-economic viability and environmental implications. <em>NBT</em>, <em>43</em>(6), 848-853. (<a href='https://doi.org/10.1038/s41587-025-02632-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Producing goods, such as foods and fuels, with minimal environmental impacts is urgently needed. Although advances in bioproduction are promising, there is often a noticeable gap in our understanding of whether and where new processes can compete with existing methods on an economic and environmental basis. Transparent lower bound calculations from basic principles highlight potential benefits of producing foods, but not fuels, from electro-microbial production of biomass.},
  archive      = {J_NBT},
  author       = {Lovat, Samuel J. and Ben-Nissan, Roee and Milshtein, Eliya and Tzachor, Asaf and Flamholz, Avi and Leger, Dorian and Noor, Elad and Milo, Ron},
  doi          = {10.1038/s41587-025-02632-w},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {848-853},
  shortjournal = {Nature Biotech.},
  title        = {Electro-microbial production techno-economic viability and environmental implications},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A call for built-in biosecurity safeguards for generative AI tools. <em>NBT</em>, <em>43</em>(6), 845-847. (<a href='https://doi.org/10.1038/s41587-025-02650-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Wang, Mengdi and Zhang, Zaixi and Bedi, Amrit Singh and Velasquez, Alvaro and Guerra, Stephanie and Lin-Gibson, Sheng and Cong, Le and Qu, Yuanhao and Chakraborty, Souradip and Blewett, Megan and Ma, Jian and Xing, Eric and Church, George},
  doi          = {10.1038/s41587-025-02650-8},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {845-847},
  shortjournal = {Nature Biotech.},
  title        = {A call for built-in biosecurity safeguards for generative AI tools},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Beyond cell atlases: Spatial biology reveals mechanisms behind disease. <em>NBT</em>, <em>43</em>(6), 841-844. (<a href='https://doi.org/10.1038/s41587-025-02699-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As the next generation of spatial transcriptomics tools hits the market, researchers are uncovering previously unknown interactions that could transform clinical research.},
  archive      = {J_NBT},
  author       = {Seydel, Caroline},
  doi          = {10.1038/s41587-025-02699-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {841-844},
  shortjournal = {Nature Biotech.},
  title        = {Beyond cell atlases: Spatial biology reveals mechanisms behind disease},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). FDA says GM pigs safe to eat. <em>NBT</em>, <em>43</em>(6), 839. (<a href='https://doi.org/10.1038/s41587-025-02716-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02716-7},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {839},
  shortjournal = {Nature Biotech.},
  title        = {FDA says GM pigs safe to eat},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Merck’s anti-RSV antibody expands protection for infants. <em>NBT</em>, <em>43</em>(6), 837-840. (<a href='https://doi.org/10.1038/s41587-025-02713-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Merck’s new antibody clesrovimab and other long-lasting shots promise to both expand and simplify care of newborns across the world to combat respiratory syncytial virus.},
  archive      = {J_NBT},
  author       = {Marshall, Andrew and Nordling, Linda},
  doi          = {10.1038/s41587-025-02713-w},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {837-840},
  shortjournal = {Nature Biotech.},
  title        = {Merck’s anti-RSV antibody expands protection for infants},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). England poised to green-light precision breeding. <em>NBT</em>, <em>43</em>(6), 837. (<a href='https://doi.org/10.1038/s41587-025-02715-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02715-8},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {837},
  shortjournal = {Nature Biotech.},
  title        = {England poised to green-light precision breeding},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(6), 836. (<a href='https://doi.org/10.1038/s41587-025-02714-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02714-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {836},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Court reignites CRISPR patent dispute. <em>NBT</em>, <em>43</em>(6), 835. (<a href='https://doi.org/10.1038/s41587-025-02717-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02717-6},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {835},
  shortjournal = {Nature Biotech.},
  title        = {Court reignites CRISPR patent dispute},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Individualized mRNA cancer vaccines make strides. <em>NBT</em>, <em>43</em>(6), 833-836. (<a href='https://doi.org/10.1038/s41587-025-02708-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first cancer vaccines matched to a person’s unique tumor neoantigens are forging ahead, with expectations running high as the field awaits results from the first pivotal trial.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-025-02708-7},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {833-836},
  shortjournal = {Nature Biotech.},
  title        = {Individualized mRNA cancer vaccines make strides},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Powering new therapeutics with precision mitochondrial editing. <em>NBT</em>, <em>43</em>(6), 831-832. (<a href='https://doi.org/10.1038/s41587-025-02693-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {More precise base editors make mitochondrial DNA editing efficient enough to model disease and correct pathogenic mutations in rodents, but they are slow to move into clinical trials.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02693-x},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {831-832},
  shortjournal = {Nature Biotech.},
  title        = {Powering new therapeutics with precision mitochondrial editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Go fund a brain gain. <em>NBT</em>, <em>43</em>(6), 829-830. (<a href='https://doi.org/10.1038/s41587-025-02722-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The gutting of US research funding represents a once-in-a-generation opportunity for Europe to boost world-class research deprioritized by the Trump administration.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02722-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {829-830},
  shortjournal = {Nature Biotech.},
  title        = {Go fund a brain gain},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(5), 828. (<a href='https://doi.org/10.1038/s41587-025-02670-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02670-4},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {828},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The biotech explorers pathway equips students for STEM futures. <em>NBT</em>, <em>43</em>(5), 823-827. (<a href='https://doi.org/10.1038/s41587-025-02666-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Building undergraduate courses around biotechnology ventures integrates real-world science, business, career preparation and creative thinking for STEM student preparation.},
  archive      = {J_NBT},
  author       = {Robinson, Anne E. and Cascella, Barrie and Wegorzewska, Marta M. and Berkovich, Daniel A. and Jez, Joseph M.},
  doi          = {10.1038/s41587-025-02666-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {823-827},
  shortjournal = {Nature Biotech.},
  title        = {The biotech explorers pathway equips students for STEM futures},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: Machine learning prediction of prime editing efficiency across diverse chromatin contexts. <em>NBT</em>, <em>43</em>(5), 822. (<a href='https://doi.org/10.1038/s41587-024-02383-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Mathis, Nicolas and Allam, Ahmed and Tálas, András and Kissling, Lucas and Benvenuto, Elena and Schmidheini, Lukas and Schep, Ruben and Damodharan, Tanav and Balázs, Zsolt and Janjuha, Sharan and Ioannidi, Eleonora I. and Böck, Desirée and van Steensel, Bas and Krauthammer, Michael and Schwank, Gerald},
  doi          = {10.1038/s41587-024-02383-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {822},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Machine learning prediction of prime editing efficiency across diverse chromatin contexts},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Signal amplification by cyclic extension enables high-sensitivity single-cell mass cytometry. <em>NBT</em>, <em>43</em>(5), 811-821. (<a href='https://doi.org/10.1038/s41587-024-02316-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mass cytometry uses metal-isotope-tagged antibodies to label targets of interest, which enables simultaneous measurements of ~50 proteins or protein modifications in millions of single cells, but its sensitivity is limited. Here, we present a signal amplification technology, termed Amplification by Cyclic Extension (ACE), implementing thermal-cycling-based DNA in situ concatenation in combination with 3-cyanovinylcarbazole phosphoramidite-based DNA crosslinking to enable signal amplification simultaneously on &gt;30 protein epitopes. We demonstrate the utility of ACE in low-abundance protein quantification with suspension mass cytometry to characterize molecular reprogramming during the epithelial-to-mesenchymal transition as well as the mesenchymal-to-epithelial transition. We show the capability of ACE to quantify the dynamics of signaling network responses in human T lymphocytes. We further present the application of ACE in imaging mass cytometry-based multiparametric tissue imaging to identify tissue compartments and profile spatial aspects related to pathological states in polycystic kidney tissues. Mass cytometry with signal amplification enables measurement of low-abundance proteins.},
  archive      = {J_NBT},
  author       = {Lun, Xiao-Kang and Sheng, Kuanwei and Yu, Xueyang and Lam, Ching Yeung and Gowri, Gokul and Serrata, Matthew and Zhai, Yunhao and Su, Hanquan and Luan, Jingyi and Kim, Youngeun and Ingber, Donald E. and Jackson, Hartland W. and Yaffe, Michael B. and Yin, Peng},
  doi          = {10.1038/s41587-024-02316-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {811-821},
  shortjournal = {Nature Biotech.},
  title        = {Signal amplification by cyclic extension enables high-sensitivity single-cell mass cytometry},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tracking-seq reveals the heterogeneity of off-target effects in CRISPR–Cas9-mediated genome editing. <em>NBT</em>, <em>43</em>(5), 799-810. (<a href='https://doi.org/10.1038/s41587-024-02307-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The continued development of novel genome editors calls for a universal method to analyze their off-target effects. Here we describe a versatile method, called Tracking-seq, for in situ identification of off-target effects that is broadly applicable to common genome-editing tools, including Cas9, base editors and prime editors. Through tracking replication protein A (RPA)-bound single-stranded DNA followed by strand-specific library construction, Tracking-seq requires a low cell input and is suitable for in vitro, ex vivo and in vivo genome editing, providing a sensitive and practical genome-wide approach for off-target detection in various scenarios. We show, using the same guide RNA, that Tracking-seq detects heterogeneity in off-target effects between different editor modalities and between different cell types, underscoring the necessity of direct measurement in the original system. The off-target effects of any CRISPR-based genome editor are identified with Tracking-seq.},
  archive      = {J_NBT},
  author       = {Zhu, Ming and Xu, Runda and Yuan, Junsong and Wang, Jiacheng and Ren, Xiaoyu and Cong, Tingting and You, Yaxian and Ju, Anji and Xu, Longchen and Wang, Huimin and Zheng, Peiyuan and Tao, Huiying and Lin, Chunhua and Yu, Honghao and Du, Juanjuan and Lin, Xin and Xie, Wei and Li, Yinqing and Lan, Xun},
  doi          = {10.1038/s41587-024-02307-y},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {799-810},
  shortjournal = {Nature Biotech.},
  title        = {Tracking-seq reveals the heterogeneity of off-target effects in CRISPR–Cas9-mediated genome editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tracking single-cell evolution using clock-like chromatin accessibility loci. <em>NBT</em>, <em>43</em>(5), 784-798. (<a href='https://doi.org/10.1038/s41587-024-02241-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell chromatin accessibility sequencing (scATAC-seq) reconstructs developmental trajectory by phenotypic similarity. However, inferring the exact developmental trajectory is challenging. Previous studies showed age-associated DNA methylation (DNAm) changes in specific genomic regions, termed clock-like differential methylation loci (ClockDML). Age-associated DNAm could either result from or result in chromatin accessibility changes at ClockDML. As cells undergo mitosis, the heterogeneity of chromatin accessibility on clock-like loci is reduced, providing a measure of mitotic age. In this study, we developed a method, called EpiTrace, that counts the fraction of opened clock-like loci from scATAC-seq data to determine cell age and perform lineage tracing in various cell lineages and animal species. It shows concordance with known developmental hierarchies, correlates well with DNAm-based clocks and is complementary with mutation-based lineage tracing, RNA velocity and stemness predictions. Applying EpiTrace to scATAC-seq data reveals biological insights with clinically relevant implications, ranging from hematopoiesis, organ development, tumor biology and immunity to cortical gyrification. EpiTrace infers cell mitotic age and evolution from single-cell ATAC-seq data.},
  archive      = {J_NBT},
  author       = {Xiao, Yu and Jin, Wan and Ju, Lingao and Fu, Jie and Wang, Gang and Yu, Mengxue and Chen, Fangjin and Qian, Kaiyu and Wang, Xinghuan and Zhang, Yi},
  doi          = {10.1038/s41587-024-02241-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {784-798},
  shortjournal = {Nature Biotech.},
  title        = {Tracking single-cell evolution using clock-like chromatin accessibility loci},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Strain tracking in complex microbiomes using synteny analysis reveals per-species modes of evolution. <em>NBT</em>, <em>43</em>(5), 773-783. (<a href='https://doi.org/10.1038/s41587-024-02276-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microbial species diversify into strains through single-nucleotide mutations and structural changes, such as recombination, insertions and deletions. Most strain-comparison methods quantify differences in single-nucleotide polymorphisms (SNPs) and are insensitive to structural changes. However, recombination is an important driver of phenotypic diversification in many species, including human pathogens. We introduce SynTracker, a tool that compares microbial strains using genome synteny—the order of sequence blocks in homologous genomic regions—in pairs of metagenomic assemblies or genomes. Genome synteny is a rich source of genomic information untapped by current strain-comparison tools. SynTracker has low sensitivity to SNPs, has no database requirement and is robust to sequencing errors. It outperforms existing tools when tracking strains in metagenomic data and is particularly suited for phages, plasmids and other low-data contexts. Applied to single-species datasets and human gut metagenomes, SynTracker, combined with an SNP-based tool, detects strains enriched in either point mutations or structural changes, providing insights into microbial evolution in situ. The evolution of microbial strains is revealed by analyzing structural variation.},
  archive      = {J_NBT},
  author       = {Enav, Hagay and Paz, Inbal and Ley, Ruth E.},
  doi          = {10.1038/s41587-024-02276-2},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {773-783},
  shortjournal = {Nature Biotech.},
  title        = {Strain tracking in complex microbiomes using synteny analysis reveals per-species modes of evolution},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Template-independent enzymatic synthesis of RNA oligonucleotides. <em>NBT</em>, <em>43</em>(5), 762-772. (<a href='https://doi.org/10.1038/s41587-024-02244-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA oligonucleotides have emerged as a powerful therapeutic modality to treat disease, yet current manufacturing methods may not be able to deliver on anticipated future demand. Here, we report the development and optimization of an aqueous-based, template-independent enzymatic RNA oligonucleotide synthesis platform as an alternative to traditional chemical methods. The enzymatic synthesis of RNA oligonucleotides is made possible by controlled incorporation of reversible terminator nucleotides with a common 3′-O-allyl ether blocking group using new CID1 poly(U) polymerase mutant variants. We achieved an average coupling efficiency of 95% and demonstrated ten full cycles of liquid phase synthesis to produce natural and therapeutically relevant modified sequences. We then qualitatively assessed the platform on a solid phase, performing enzymatic synthesis of several N + 5 oligonucleotides on a controlled-pore glass support. Adoption of an aqueous-based process will offer key advantages including the reduction of solvent use and sustainable therapeutic oligonucleotide manufacturing. Single-stranded RNA is enzymatically synthesized without use of a template strand.},
  archive      = {J_NBT},
  author       = {Wiegand, Daniel J. and Rittichier, Jonathan and Meyer, Ella and Lee, Howon and Conway, Nicholas J. and Ahlstedt, Daniel and Yurtsever, Zeynep and Rainone, Dominic and Kuru, Erkin and Church, George M.},
  doi          = {10.1038/s41587-024-02244-w},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {762-772},
  shortjournal = {Nature Biotech.},
  title        = {Template-independent enzymatic synthesis of RNA oligonucleotides},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). In vivo AAV–SB-CRISPR screens of tumor-infiltrating primary NK cells identify genetic checkpoints of CAR-NK therapy. <em>NBT</em>, <em>43</em>(5), 752-761. (<a href='https://doi.org/10.1038/s41587-024-02282-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Natural killer (NK) cells have clinical potential against cancer; however, multiple limitations hinder the success of NK cell therapy. Here, we performed unbiased functional mapping of tumor-infiltrating NK (TINK) cells using in vivo adeno-associated virus (AAV)–SB (Sleeping Beauty)-CRISPR (clustered regularly interspaced short palindromic repeats) screens in four solid tumor mouse models. In parallel, we characterized single-cell transcriptomic landscapes of TINK cells, which identified previously unexplored subpopulations of NK cells and differentially expressed TINK genes. As a convergent hit, CALHM2-knockout (KO) NK cells showed enhanced cytotoxicity and tumor infiltration in mouse primary NK cells and human chimeric antigen receptor (CAR)-NK cells. CALHM2 mRNA reversed the CALHM2-KO phenotype. CALHM2 KO in human primary NK cells enhanced their cytotoxicity, degranulation and cytokine production. Transcriptomics profiling revealed CALHM2-KO-altered genes and pathways in both baseline and stimulated conditions. In a solid tumor model resistant to unmodified CAR-NK cells, CALHM2-KO CAR-NK cells showed potent in vivo antitumor efficacy. These data identify endogenous genetic checkpoints that naturally limit NK cell function and demonstrate the use of CALHM2 KO for engineering enhanced NK cell-based immunotherapies. The antitumor efficacy of chimeric antigen receptor natural killer cells is enhanced by genetic engineering.},
  archive      = {J_NBT},
  author       = {Peng, Lei and Renauer, Paul A. and Sferruzza, Giacomo and Yang, Luojia and Zou, Yongji and Fang, Zhenghao and Park, Jonathan J. and Chow, Ryan D. and Zhang, Yueqi and Lin, Qianqian and Bai, Meizhu and Sanchez, Angelica and Zhang, Yongzhan and Lam, Stanley Z. and Ye, Lupeng and Chen, Sidi},
  doi          = {10.1038/s41587-024-02282-4},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {752-761},
  shortjournal = {Nature Biotech.},
  title        = {In vivo AAV–SB-CRISPR screens of tumor-infiltrating primary NK cells identify genetic checkpoints of CAR-NK therapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Programming tumor evolution with selection gene drives to proactively combat drug resistance. <em>NBT</em>, <em>43</em>(5), 737-751. (<a href='https://doi.org/10.1038/s41587-024-02271-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most targeted anticancer therapies fail due to drug resistance evolution. Here we show that tumor evolution can be reproducibly redirected to engineer therapeutic opportunity, regardless of the exact ensemble of pre-existing genetic heterogeneity. We develop a selection gene drive system that is stably introduced into cancer cells and is composed of two genes, or switches, that couple an inducible fitness advantage with a shared fitness cost. Using stochastic models of evolutionary dynamics, we identify the design criteria for selection gene drives. We then build prototypes that harness the selective pressure of multiple approved tyrosine kinase inhibitors and employ therapeutic mechanisms as diverse as prodrug catalysis and immune activity induction. We show that selection gene drives can eradicate diverse forms of genetic resistance in vitro. Finally, we demonstrate that model-informed switch engagement effectively targets pre-existing resistance in mouse models of solid tumors. These results establish selection gene drives as a powerful framework for evolution-guided anticancer therapy. Selection gene drives re-engineer tumors to suppress drug resistance.},
  archive      = {J_NBT},
  author       = {Leighow, Scott M. and Reynolds, Joshua A. and Sokirniy, Ivan and Yao, Shun and Yang, Zeyu and Inam, Haider and Wodarz, Dominik and Archetti, Marco and Pritchard, Justin R.},
  doi          = {10.1038/s41587-024-02271-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {737-751},
  shortjournal = {Nature Biotech.},
  title        = {Programming tumor evolution with selection gene drives to proactively combat drug resistance},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Patient-derived mini-colons enable long-term modeling of tumor–microenvironment complexity. <em>NBT</em>, <em>43</em>(5), 727-736. (<a href='https://doi.org/10.1038/s41587-024-02301-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing organoid models fall short of fully capturing the complexity of cancer because they lack sufficient multicellular diversity, tissue-level organization, biological durability and experimental flexibility. Thus, many multifactorial cancer processes, especially those involving the tumor microenvironment, are difficult to study ex vivo. To overcome these limitations, we herein implemented tissue-engineering and microfabrication technologies to develop topobiologically complex, patient-specific cancer avatars. Focusing on colorectal cancer, we generated miniature tissues consisting of long-lived gut-shaped human colon epithelia (‘mini-colons’) that stably integrate cancer cells and their native tumor microenvironment in a format optimized for real-time, high-resolution evaluation of cellular dynamics. We demonstrate the potential of this system through several applications: a comprehensive evaluation of drug effectivity, toxicity and resistance in anticancer therapies; the discovery of a mechanism triggered by cancer-associated fibroblasts that drives cancer invasion; and the identification of immunomodulatory interactions among different components of the tumor microenvironment. Similar approaches should be feasible for diverse tumor types. Organoid avatars of colorectal cancer and its microenvironment model immune interactions and drug efficacy.},
  archive      = {J_NBT},
  author       = {Lorenzo-Martín, L. Francisco and Broguiere, Nicolas and Langer, Jakob and Tillard, Lucie and Nikolaev, Mikhail and Coukos, George and Homicsko, Krisztian and Lutolf, Matthias P.},
  doi          = {10.1038/s41587-024-02301-4},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {727-736},
  shortjournal = {Nature Biotech.},
  title        = {Patient-derived mini-colons enable long-term modeling of tumor–microenvironment complexity},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Complete substitution with modified nucleotides in self-amplifying RNA suppresses the interferon response and increases potency. <em>NBT</em>, <em>43</em>(5), 720-726. (<a href='https://doi.org/10.1038/s41587-024-02306-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The use of modified nucleotides to suppress the interferon response and maintain translation of self-amplifying RNA (saRNA), which has been achieved for mRNA, has not yet succeeded. We identify modified nucleotides that, when substituted at 100% in saRNA, confer innate immune evasion and robust long-term protein expression, and when formulated as a vaccine, protect against lethal SARS-CoV-2 challenge in mice. This discovery advances saRNA therapeutics by enabling prolonged protein expression at low doses. Complete 5-methylcytidine substitution enhances the efficacy of self-amplifying RNA.},
  archive      = {J_NBT},
  author       = {McGee, Joshua E. and Kirsch, Jack R. and Kenney, Devin and Cerbo, Faith and Chavez, Elizabeth C. and Shih, Ting-Yu and Douam, Florian and Wong, Wilson W. and Grinstaff, Mark W.},
  doi          = {10.1038/s41587-024-02306-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {720-726},
  shortjournal = {Nature Biotech.},
  title        = {Complete substitution with modified nucleotides in self-amplifying RNA suppresses the interferon response and increases potency},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Machine learning prediction of prime editing efficiency across diverse chromatin contexts. <em>NBT</em>, <em>43</em>(5), 712-719. (<a href='https://doi.org/10.1038/s41587-024-02268-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The success of prime editing depends on the prime editing guide RNA (pegRNA) design and target locus. Here, we developed machine learning models that reliably predict prime editing efficiency. PRIDICT2.0 assesses the performance of pegRNAs for all edit types up to 15 bp in length in mismatch repair-deficient and mismatch repair-proficient cell lines and in vivo in primary cells. With ePRIDICT, we further developed a model that quantifies how local chromatin environments impact prime editing rates. A machine learning model for prime editing efficiency prediction takes into account chromatin context.},
  archive      = {J_NBT},
  author       = {Mathis, Nicolas and Allam, Ahmed and Tálas, András and Kissling, Lucas and Benvenuto, Elena and Schmidheini, Lukas and Schep, Ruben and Damodharan, Tanav and Balázs, Zsolt and Janjuha, Sharan and Ioannidi, Eleonora I. and Böck, Desirée and van Steensel, Bas and Krauthammer, Michael and Schwank, Gerald},
  doi          = {10.1038/s41587-024-02268-2},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {712-719},
  shortjournal = {Nature Biotech.},
  title        = {Machine learning prediction of prime editing efficiency across diverse chromatin contexts},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Induced proximity at the cell surface. <em>NBT</em>, <em>43</em>(5), 702-711. (<a href='https://doi.org/10.1038/s41587-025-02592-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular proximity is a governing principle of biology that is essential to normal and disease-related biochemical pathways. At the cell surface, protein–protein proximity regulates receptor activation, inhibition and protein recycling and degradation. Induced proximity is a molecular engineering principle in which bifunctional molecules are designed to bring two protein targets into close contact, inducing a desired biological outcome. Researchers use this engineering principle for therapeutic purposes and to interrogate fundamental biological mechanisms. This Review focuses on the use of induced proximity at the cell surface for diverse applications, such as targeted protein degradation, receptor inhibition and activating intracellular signaling cascades. We see a rich future for proximity-based modulation of cell surface protein activity both in basic and translational science. Technologies to modulate induced proximity at the cell surface advance the manipulation of diverse biological responses.},
  archive      = {J_NBT},
  author       = {Till, Nicholas A. and Ramanathan, Muthukumar and Bertozzi, Carolyn R.},
  doi          = {10.1038/s41587-025-02592-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {702-711},
  shortjournal = {Nature Biotech.},
  title        = {Induced proximity at the cell surface},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Decoding cell replicational age from single-cell ATAC-seq data. <em>NBT</em>, <em>43</em>(5), 700-701. (<a href='https://doi.org/10.1038/s41587-024-02256-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The replicational age of single cells provides a temporal reference for tracking cell fate transition trajectories. The computational framework EpiTrace measures cell age using single-cell ATAC-seq data, specifically by considering chromatin accessibility at clock-like genomic loci, enabling the reconstruction of the history of developmental and pathological processes.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02256-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {700-701},
  shortjournal = {Nature Biotech.},
  title        = {Decoding cell replicational age from single-cell ATAC-seq data},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tracking gene order provides a new perspective on intraspecific evolution in microbiotas. <em>NBT</em>, <em>43</em>(5), 698-699. (<a href='https://doi.org/10.1038/s41587-024-02277-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed SynTracker, a tool to track conspecific microbial strains using genome synteny. SynTracker is sensitive to genomic structural variation but not to single-nucleotide polymorphisms (SNPs). Combining SynTracker with methods for tracking strains using SNP profiles, we were able to detect species evolving by accumulating predominantly SNPs or predominantly structural variants.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02277-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {698-699},
  shortjournal = {Nature Biotech.},
  title        = {Tracking gene order provides a new perspective on intraspecific evolution in microbiotas},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). CRISPR screens of tumor-infiltrating NK cells identify genetic checkpoints for CAR-NK therapy. <em>NBT</em>, <em>43</em>(5), 696-697. (<a href='https://doi.org/10.1038/s41587-024-02319-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We use a CRISPR screening platform based on adeno-associated virus and the Sleeping Beauty transposon (AAV-SB-CRISPR) to perform in vivo CRISPR screens in primary natural killer (NK) cells across four different tumor models, and identify calcium homeostasis modulator family member 2 (CALHM2) as an NK cellular checkpoint protein.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02319-8},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {696-697},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR screens of tumor-infiltrating NK cells identify genetic checkpoints for CAR-NK therapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Next-generation patient models for colorectal cancer research. <em>NBT</em>, <em>43</em>(5), 694-695. (<a href='https://doi.org/10.1038/s41587-024-02302-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lab-grown ‘mini-colons’ enable the simulation of colorectal tumor dynamics and the high-resolution study of interactions with cells from the tumor’s native microenvironment. These next-generation organoids offer a wealth of new experimental opportunities, propelling the frontiers of cancer modeling in multiple directions.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02302-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {694-695},
  shortjournal = {Nature Biotech.},
  title        = {Next-generation patient models for colorectal cancer research},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enzymatic synthesis of RNA oligonucleotides. <em>NBT</em>, <em>43</em>(5), 691-693. (<a href='https://doi.org/10.1038/s41587-024-02322-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Research on enzymatic RNA synthesis has long been eclipsed by work on DNA—but a new method provides a leap forward for RNA.},
  archive      = {J_NBT},
  author       = {Hollenstein, Marcel},
  doi          = {10.1038/s41587-024-02322-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {691-693},
  shortjournal = {Nature Biotech.},
  title        = {Enzymatic synthesis of RNA oligonucleotides},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Evolutionary gambit to defeat drug resistance in cancer. <em>NBT</em>, <em>43</em>(5), 689-690. (<a href='https://doi.org/10.1038/s41587-024-02284-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A therapeutic strategy harnesses evolutionary dynamics to overcome tumor resistance to targeted therapies.},
  archive      = {J_NBT},
  author       = {Marusyk, Andriy},
  doi          = {10.1038/s41587-024-02284-2},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {689-690},
  shortjournal = {Nature Biotech.},
  title        = {Evolutionary gambit to defeat drug resistance in cancer},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A mini-colon models colon cancer and its microenvironment. <em>NBT</em>, <em>43</em>(5), 686-688. (<a href='https://doi.org/10.1038/s41587-024-02343-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A mini-colon device mimics colorectal tumors and their interactions with neighboring healthy tissue, stromal cells and immune cells.},
  archive      = {J_NBT},
  author       = {Hutton, Colin and Li, Vivian S. W.},
  doi          = {10.1038/s41587-024-02343-8},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {686-688},
  shortjournal = {Nature Biotech.},
  title        = {A mini-colon models colon cancer and its microenvironment},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Retargeted retrotransposons insert multi-kilobase cargo at new sites. <em>NBT</em>, <em>43</em>(5), 685. (<a href='https://doi.org/10.1038/s41587-025-02682-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02682-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {685},
  shortjournal = {Nature Biotech.},
  title        = {Retargeted retrotransposons insert multi-kilobase cargo at new sites},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reprogramming endogenous regulatory DNA to fine-tune gene expression. <em>NBT</em>, <em>43</em>(5), 685. (<a href='https://doi.org/10.1038/s41587-025-02686-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02686-w},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {685},
  shortjournal = {Nature Biotech.},
  title        = {Reprogramming endogenous regulatory DNA to fine-tune gene expression},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A gut-derived peptide protects citrus trees from huanglongbing. <em>NBT</em>, <em>43</em>(5), 685. (<a href='https://doi.org/10.1038/s41587-025-02687-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02687-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {685},
  shortjournal = {Nature Biotech.},
  title        = {A gut-derived peptide protects citrus trees from huanglongbing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bioprinting. <em>NBT</em>, <em>43</em>(5), 684. (<a href='https://doi.org/10.1038/s41587-025-02662-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to systems, methods and compositions for bioprinting.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02662-4},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {684},
  shortjournal = {Nature Biotech.},
  title        = {Bioprinting},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). 3D bioprinting innovation and the patentability hurdle. <em>NBT</em>, <em>43</em>(5), 677-683. (<a href='https://doi.org/10.1038/s41587-025-02661-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How are patent examiners in the United States, Europe and Australia interpreting and applying the patentability criteria to specific subject matter related to 3D bioprinting technologies?},
  archive      = {J_NBT},
  author       = {Devarapalli, Pratap and Nicol, Dianne and Nielsen, Jane},
  doi          = {10.1038/s41587-025-02661-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {677-683},
  shortjournal = {Nature Biotech.},
  title        = {3D bioprinting innovation and the patentability hurdle},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The emerging landscape of engineered bacteria cancer therapies. <em>NBT</em>, <em>43</em>(5), 672-676. (<a href='https://doi.org/10.1038/s41587-025-02623-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our ability to use engineered bacteria for cancer therapy is rapidly expanding. A survey of preclinical, clinical and commercial efforts provides an overview of the state of the field, revealing trends that could inform future directions.},
  archive      = {J_NBT},
  author       = {Ballister, Edward R. and Michels, Alexander and Vincent, Rosa L. and Kreindler, Lior and Chowdhury, Sreyan and Upadhaya, Samik and Saez-Ibañez, Ana Rosa and Tu, Tao and Gottweis, Juraj and Danino, Tal},
  doi          = {10.1038/s41587-025-02623-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {672-676},
  shortjournal = {Nature Biotech.},
  title        = {The emerging landscape of engineered bacteria cancer therapies},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The protein design archive (PDA): Insights from 40 years of protein design. <em>NBT</em>, <em>43</em>(5), 669-671. (<a href='https://doi.org/10.1038/s41587-025-02607-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Chronowska, Marta and Stam, Michael J. and Woolfson, Derek N. and Di Costanzo, Luigi F. and Wood, Christopher W.},
  doi          = {10.1038/s41587-025-02607-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {669-671},
  shortjournal = {Nature Biotech.},
  title        = {The protein design archive (PDA): Insights from 40 years of protein design},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Openness guides discovery. <em>NBT</em>, <em>43</em>(5), 667-668. (<a href='https://doi.org/10.1038/s41587-025-02635-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Yanai, Itai and Lercher, Martin J.},
  doi          = {10.1038/s41587-025-02635-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {667-668},
  shortjournal = {Nature Biotech.},
  title        = {Openness guides discovery},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Lengthy delays in H5N1 genome submissions to GISAID. <em>NBT</em>, <em>43</em>(5), 665-666. (<a href='https://doi.org/10.1038/s41587-025-02636-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Otto, Sarah P. and Edgerton, Sean Vidal},
  doi          = {10.1038/s41587-025-02636-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {665-666},
  shortjournal = {Nature Biotech.},
  title        = {Lengthy delays in H5N1 genome submissions to GISAID},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precision neuroscience. <em>NBT</em>, <em>43</em>(5), 659-664. (<a href='https://doi.org/10.1038/s41587-025-02673-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Developers are recombining or sharpening established drugs, molecules and mechanisms to create a new generation of neurology therapies.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-025-02673-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {659-664},
  shortjournal = {Nature Biotech.},
  title        = {Precision neuroscience},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(5), 658. (<a href='https://doi.org/10.1038/s41587-025-02680-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02680-2},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {658},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fruit waste for catwalk fashion. <em>NBT</em>, <em>43</em>(5), 657. (<a href='https://doi.org/10.1038/s41587-025-02679-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-025-02679-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {657},
  shortjournal = {Nature Biotech.},
  title        = {Fruit waste for catwalk fashion},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Brazil’s low-cost CAR-ts take on global south. <em>NBT</em>, <em>43</em>(5), 657. (<a href='https://doi.org/10.1038/s41587-025-02691-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02691-z},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {657},
  shortjournal = {Nature Biotech.},
  title        = {Brazil’s low-cost CAR-ts take on global south},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). FDA pushes to replace animal testing. <em>NBT</em>, <em>43</em>(5), 655. (<a href='https://doi.org/10.1038/s41587-025-02690-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02690-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {655},
  shortjournal = {Nature Biotech.},
  title        = {FDA pushes to replace animal testing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Consternation follows marks’ FDA departure. <em>NBT</em>, <em>43</em>(5), 654. (<a href='https://doi.org/10.1038/s41587-025-02689-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02689-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {654},
  shortjournal = {Nature Biotech.},
  title        = {Consternation follows marks’ FDA departure},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biopharma dives into tumor-seeking radioactive drugs. <em>NBT</em>, <em>43</em>(5), 653-656. (<a href='https://doi.org/10.1038/s41587-025-02681-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Big pharma and investors are piling into the precision radiation space as the next generation of increasingly potent targeted α-particle therapies promises to destroy cancerous cells with minimum damage to healthy tissues.},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/s41587-025-02681-1},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {653-656},
  shortjournal = {Nature Biotech.},
  title        = {Biopharma dives into tumor-seeking radioactive drugs},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(4), 652. (<a href='https://doi.org/10.1038/s41587-025-02637-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02637-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {652},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Five questions with jie sun. <em>NBT</em>, <em>43</em>(4), 651. (<a href='https://doi.org/10.1038/s41587-025-02631-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This developer of cancer immunotherapies discusses her early fascination with cellular behavior, perseverance and deep desire to directly impact patient care.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-025-02631-x},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {651},
  shortjournal = {Nature Biotech.},
  title        = {Five questions with jie sun},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SPARK taiwan: A decade of insights in adapting US translational medicine and commercialization methods. <em>NBT</em>, <em>43</em>(4), 647-650. (<a href='https://doi.org/10.1038/s41587-025-02630-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SPARK Taiwan has strengthened the country’s biotech sector, encouraged scientists to start businesses and fostered cross-disciplinary collaboration.},
  archive      = {J_NBT},
  author       = {Lo, Ting-Fang and Huang, Yi-Chih and Cho, Nai-Yu and Tseng, Yufeng Jane},
  doi          = {10.1038/s41587-025-02630-y},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {647-650},
  shortjournal = {Nature Biotech.},
  title        = {SPARK taiwan: A decade of insights in adapting US translational medicine and commercialization methods},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mapping medically relevant RNA isoform diversity in the aged human frontal cortex with deep long-read RNA-seq. <em>NBT</em>, <em>43</em>(4), 635-646. (<a href='https://doi.org/10.1038/s41587-024-02245-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Determining whether the RNA isoforms from medically relevant genes have distinct functions could facilitate direct targeting of RNA isoforms for disease treatment. Here, as a step toward this goal for neurological diseases, we sequenced 12 postmortem, aged human frontal cortices (6 Alzheimer disease cases and 6 controls; 50% female) using one Oxford Nanopore PromethION flow cell per sample. We identified 1,917 medically relevant genes expressing multiple isoforms in the frontal cortex where 1,018 had multiple isoforms with different protein-coding sequences. Of these 1,018 genes, 57 are implicated in brain-related diseases including major depression, schizophrenia, Parkinson’s disease and Alzheimer disease. Our study also uncovered 53 new RNA isoforms in medically relevant genes, including several where the new isoform was one of the most highly expressed for that gene. We also reported on five mitochondrially encoded, spliced RNA isoforms. We found 99 differentially expressed RNA isoforms between cases with Alzheimer disease and controls. The landscape of RNA isoforms in human cortex is revealed by deep long-read RNA sequencing.},
  archive      = {J_NBT},
  author       = {Aguzzoli Heberle, Bernardo and Brandon, J. Anthony and Page, Madeline L. and Nations, Kayla A. and Dikobe, Ketsile I. and White, Brendan J. and Gordon, Lacey A. and Fox, Grant A. and Wadsworth, Mark E. and Doyle, Patricia H. and Williams, Brittney A. and Fox, Edward J. and Shantaraman, Anantharaman and Ryten, Mina and Goodwin, Sara and Ghiban, Elena and Wappel, Robert and Mavruk-Eskipehlivan, Senem and Miller, Justin B. and Seyfried, Nicholas T. and Nelson, Peter T. and Fryer, John D. and Ebbert, Mark T. W.},
  doi          = {10.1038/s41587-024-02245-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {635-646},
  shortjournal = {Nature Biotech.},
  title        = {Mapping medically relevant RNA isoform diversity in the aged human frontal cortex with deep long-read RNA-seq},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-throughput discovery of MHC class I- and II-restricted t cell epitopes using synthetic cellular circuits. <em>NBT</em>, <em>43</em>(4), 623-634. (<a href='https://doi.org/10.1038/s41587-024-02248-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antigen discovery technologies have largely focused on major histocompatibility complex (MHC) class I-restricted human T cell receptors (TCRs), leaving methods for MHC class II-restricted and mouse TCR reactivities relatively undeveloped. Here we present TCR mapping of antigenic peptides (TCR-MAP), an antigen discovery method that uses a synthetic TCR-stimulated circuit in immortalized T cells to activate sortase-mediated tagging of engineered antigen-presenting cells (APCs) expressing processed peptides on MHCs. Live, tagged APCs can be directly purified for deconvolution by sequencing, enabling TCRs with unknown specificity to be queried against barcoded peptide libraries in a pooled screening context. TCR-MAP accurately captures self-reactivities or viral reactivities with high throughput and sensitivity for both MHC class I-restricted and class II-restricted TCRs. We elucidate problematic cross-reactivities of clinical TCRs targeting the cancer/testis melanoma-associated antigen A3 and discover targets of myocarditis-inciting autoreactive T cells in mice. TCR-MAP has the potential to accelerate T cell antigen discovery efforts in the context of cancer, infectious disease and autoimmunity. T cell-specific antigens are discovered at high throughput without killing target cells.},
  archive      = {J_NBT},
  author       = {Kohlgruber, Ayano C. and Dezfulian, Mohammad H. and Sie, Brandon M. and Wang, Charlotte I. and Kula, Tomasz and Laserson, Uri and Larman, H. Benjamin and Elledge, Stephen J.},
  doi          = {10.1038/s41587-024-02248-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {623-634},
  shortjournal = {Nature Biotech.},
  title        = {High-throughput discovery of MHC class I- and II-restricted t cell epitopes using synthetic cellular circuits},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Linking CRISPR–Cas9 double-strand break profiles to gene editing precision with BreakTag. <em>NBT</em>, <em>43</em>(4), 608-622. (<a href='https://doi.org/10.1038/s41587-024-02238-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cas9 can cleave DNA in both blunt and staggered configurations, resulting in distinct editing outcomes, but what dictates the type of Cas9 incisions is largely unknown. In this study, we developed BreakTag, a versatile method for profiling Cas9-induced DNA double-strand breaks (DSBs) and identifying the determinants of Cas9 incisions. Overall, we assessed cleavage by SpCas9 at more than 150,000 endogenous on-target and off-target sites targeted by approximately 3,500 single guide RNAs. We found that approximately 35% of SpCas9 DSBs are staggered, and the type of incision is influenced by DNA:gRNA complementarity and the use of engineered Cas9 variants. A machine learning model shows that Cas9 incision is dependent on the protospacer sequence and that human genetic variation impacts the configuration of Cas9 cuts and the DSB repair outcome. Matched datasets of Cas9 and engineered variant incisions with repair outcomes show that Cas9-mediated staggered breaks are linked with precise, templated and predictable single-nucleotide insertions, demonstrating that a scission-based gRNA design can be used to correct clinically relevant pathogenic single-nucleotide deletions. The genome-wide landscape of Cas-induced double-strand breaks and end structures is profiled at nucleotide resolution.},
  archive      = {J_NBT},
  author       = {Longo, Gabriel M. C. and Sayols, Sergi and Kotini, Andriana G. and Heinen, Sabine and Möckel, Martin M. and Beli, Petra and Roukos, Vassilis},
  doi          = {10.1038/s41587-024-02238-8},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {608-622},
  shortjournal = {Nature Biotech.},
  title        = {Linking CRISPR–Cas9 double-strand break profiles to gene editing precision with BreakTag},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Development of multiplexed orthogonal base editor (MOBE) systems. <em>NBT</em>, <em>43</em>(4), 593-607. (<a href='https://doi.org/10.1038/s41587-024-02240-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Base editors (BEs) enable efficient, programmable installation of point mutations while avoiding the use of double-strand breaks. Simultaneous application of two or more different BEs, such as an adenine BE (which converts A·T base pairs to G·C) and a cytosine BE (which converts C·G base pairs to T·A), is not feasible because guide RNA crosstalk results in non-orthogonal editing, with all BEs modifying all target loci. Here we engineer both adenine BEs and cytosine BEs that can be orthogonally multiplexed by using RNA aptamer–coat protein systems to recruit the DNA-modifying enzymes directly to the guide RNAs. We generate four multiplexed orthogonal BE systems that enable rates of precise co-occurring edits of up to 7.1% in the same DNA strand without enrichment or selection strategies. The addition of a fluorescent enrichment strategy increases co-occurring edit rates up to 24.8% in human cells. These systems are compatible with expanded protospacer adjacent motif and high-fidelity Cas9 variants, function well in multiple cell types, have equivalent or reduced off-target propensities compared with their parental systems and can model disease-relevant point mutation combinations. Multiplexed orthogonal base editing systems introduce different point mutations at two loci simultaneously.},
  archive      = {J_NBT},
  author       = {Cowan, Quinn T. and Gu, Sifeng and Gu, Wanjun and Ranzau, Brodie L. and Simonson, Tatum S. and Komor, Alexis C.},
  doi          = {10.1038/s41587-024-02240-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {593-607},
  shortjournal = {Nature Biotech.},
  title        = {Development of multiplexed orthogonal base editor (MOBE) systems},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Crowd-sourced benchmarking of single-sample tumor subclonal reconstruction. <em>NBT</em>, <em>43</em>(4), 581-592. (<a href='https://doi.org/10.1038/s41587-024-02250-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Subclonal reconstruction algorithms use bulk DNA sequencing data to quantify parameters of tumor evolution, allowing an assessment of how cancers initiate, progress and respond to selective pressures. We launched the ICGC–TCGA (International Cancer Genome Consortium–The Cancer Genome Atlas) DREAM Somatic Mutation Calling Tumor Heterogeneity and Evolution Challenge to benchmark existing subclonal reconstruction algorithms. This 7-year community effort used cloud computing to benchmark 31 subclonal reconstruction algorithms on 51 simulated tumors. Algorithms were scored on seven independent tasks, leading to 12,061 total runs. Algorithm choice influenced performance substantially more than tumor features but purity-adjusted read depth, copy-number state and read mappability were associated with the performance of most algorithms on most tasks. No single algorithm was a top performer for all seven tasks and existing ensemble strategies were unable to outperform the best individual methods, highlighting a key research need. All containerized methods, evaluation code and datasets are available to support further assessment of the determinants of subclonal reconstruction accuracy and development of improved methods to understand tumor evolution. Benchmarking of tumor subclonal reconstruction algorithms finds that interalgorithm variability overwhelms intertumor variability and identifies key quality metrics.},
  archive      = {J_NBT},
  author       = {Salcedo, Adriana and Tarabichi, Maxime and Buchanan, Alex and Espiritu, Shadrielle M. G. and Zhang, Hongjiu and Zhu, Kaiyi and Ou Yang, Tai-Hsien and Leshchiner, Ignaty and Anastassiou, Dimitris and Guan, Yuanfang and Jang, Gun Ho and Mootor, Mohammed F. E. and Haase, Kerstin and Deshwar, Amit G. and Zou, William and Umar, Imaad and Dentro, Stefan and Wintersinger, Jeff A. and Chiotti, Kami and Demeulemeester, Jonas and Jolly, Clemency and Sycza, Lesia and Ko, Minjeong and Wedge, David C. and Morris, Quaid D. and Ellrott, Kyle and Van Loo, Peter and Boutros, Paul C.},
  doi          = {10.1038/s41587-024-02250-y},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {581-592},
  shortjournal = {Nature Biotech.},
  title        = {Crowd-sourced benchmarking of single-sample tumor subclonal reconstruction},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Long-term intravital subcellular imaging with confocal scanning light-field microscopy. <em>NBT</em>, <em>43</em>(4), 569-580. (<a href='https://doi.org/10.1038/s41587-024-02249-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-term observation of subcellular dynamics in living organisms is limited by background fluorescence originating from tissue scattering or dense labeling. Existing confocal approaches face an inevitable tradeoff among parallelization, resolution and phototoxicity. Here we present confocal scanning light-field microscopy (csLFM), which integrates axially elongated line-confocal illumination with the rolling shutter in scanning light-field microscopy (sLFM). csLFM enables high-fidelity, high-speed, three-dimensional (3D) imaging at near-diffraction-limit resolution with both optical sectioning and low phototoxicity. By simultaneous 3D excitation and detection, the excitation intensity can be reduced below 1 mW mm−2, with 15-fold higher signal-to-background ratio over sLFM. We imaged subcellular dynamics over 25,000 timeframes in optically challenging environments in different species, such as migrasome delivery in mouse spleen, retractosome generation in mouse liver and 3D voltage imaging in Drosophila. Moreover, csLFM facilitates high-fidelity, large-scale neural recording with reduced crosstalk, leading to high orientation selectivity to visual stimuli, similar to two-photon microscopy, which aids understanding of neural coding mechanisms. A confocal microscopy technique enables high-resolution intravital imaging over multiple hours.},
  archive      = {J_NBT},
  author       = {Lu, Zhi and Zuo, Siqing and Shi, Minghui and Fan, Jiaqi and Xie, Jingyu and Xiao, Guihua and Yu, Li and Wu, Jiamin and Dai, Qionghai},
  doi          = {10.1038/s41587-024-02249-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {569-580},
  shortjournal = {Nature Biotech.},
  title        = {Long-term intravital subcellular imaging with confocal scanning light-field microscopy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Trans-nuclease activity of cas9 activated by DNA or RNA target binding. <em>NBT</em>, <em>43</em>(4), 558-568. (<a href='https://doi.org/10.1038/s41587-024-02255-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Type V and type VI CRISPR–Cas systems have been shown to cleave nonspecific single-stranded DNA (ssDNA) or single-stranded RNA (ssRNA) in trans, but this has not been observed in type II CRISPR–Cas systems using single guide RNA. We show here that the type II CRISPR–Cas9 systems directed by CRISPR RNA and trans-activating CRISPR RNA dual RNAs show RuvC domain-dependent trans-cleavage activity for both ssDNA and ssRNA substrates. Cas9 possesses sequence preferences for trans-cleavage substrates, preferring to cleave T- or C-rich ssDNA substrates. We find that the trans-cleavage activity of Cas9 can be activated by target ssDNA, double-stranded DNA and ssRNA. The crystal structure of Cas9 in complex with guide RNA and target RNA provides a structural basis for the binding of target RNA to activate Cas9. Based on the trans-cleavage activity of Cas9 and nucleic acid amplification technology, we develop the nucleic acid detection platforms DNA-activated Cas9 detection and RNA-activated Cas9 detection, which are capable of detecting DNA and RNA samples with high sensitivity and specificity. Cas9 shows trans-cleavage activity and can be used for DNA and RNA detection.},
  archive      = {J_NBT},
  author       = {Chen, Jiyun and Chen, Ying and Huang, Linglong and Lin, Xiaofeng and Chen, Hong and Xiang, Wenwen and Liu, Liang},
  doi          = {10.1038/s41587-024-02255-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {558-568},
  shortjournal = {Nature Biotech.},
  title        = {Trans-nuclease activity of cas9 activated by DNA or RNA target binding},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Precise in vivo RNA base editing with a wobble-enhanced circular CLUSTER guide RNA. <em>NBT</em>, <em>43</em>(4), 545-557. (<a href='https://doi.org/10.1038/s41587-024-02313-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recruiting the endogenous editing enzyme adenosine deaminase acting on RNA (ADAR) with tailored guide RNAs for adenosine-to-inosine (A-to-I) RNA base editing is promising for safely manipulating genetic information at the RNA level. However, the precision and efficiency of editing are often compromised by bystander off-target editing. Here, we find that in 5′-UAN triplets, which dominate bystander editing, G•U wobble base pairs effectively mitigate off-target events while maintaining high on-target efficiency. This strategy is universally applicable to existing A-to-I RNA base-editing systems and complements other suppression methods such as G•A mismatches and uridine (U) depletion. Combining wobble base pairing with a circularized format of the CLUSTER approach achieves highly precise and efficient editing (up to 87%) of a disease-relevant mutation in the Mecp2 transcript in cell culture. Virus-mediated delivery of the guide RNA alone realizes functional MeCP2 protein restoration in the central nervous system of a murine Rett syndrome model with editing yields of up to 19% and excellent bystander control in vivo. Using G•U wobble base pairs at specific loci increases RNA base-editing precision and efficiency.},
  archive      = {J_NBT},
  author       = {Reautschnig, Philipp and Fruhner, Carolin and Wahn, Nicolai and Wiegand, Charlotte P. and Kragness, Sabrina and Yung, John F. and Hofacker, Daniel T. and Fisk, Jenna and Eidelman, Michelle and Waffenschmidt, Nils and Feige, Maximilian and Pfeiffer, Laura S. and Schulz, Annika E. and Füll, Yvonne and Levanon, Erez Y. and Mandel, Gail and Stafforst, Thorsten},
  doi          = {10.1038/s41587-024-02313-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {545-557},
  shortjournal = {Nature Biotech.},
  title        = {Precise in vivo RNA base editing with a wobble-enhanced circular CLUSTER guide RNA},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Increasing intracellular dNTP levels improves prime editing efficiency. <em>NBT</em>, <em>43</em>(4), 539-544. (<a href='https://doi.org/10.1038/s41587-024-02405-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In primary cell types, intracellular deoxynucleotide triphosphate (dNTP) levels are tightly regulated in a cell cycle-dependent manner. We report that prime editing efficiency is increased by mutations that improve the enzymatic properties of Moloney murine leukemia virus reverse transcriptase and treatments that increase intracellular dNTP levels. In combination, these modifications produce substantial increases in precise editing rates. Prime editing efficiency is enhanced using a mutated reverse transcriptase and addition of deoxynucleosides.},
  archive      = {J_NBT},
  author       = {Liu, Pengpeng and Ponnienselvan, Karthikeyan and Nyalile, Thomas and Oikemus, Sarah and Joynt, Anya T. and Iyer, Sukanya and Kelly, Karen and Guo, Dongsheng and Kyawe, Pyae P. and Vanderleeden, Emma and Redick, Sambra D. and Huang, Lei and Chen, Zexiang and Lee, Jeong Min and Schiffer, Celia A. and Harlan, David M. and Wang, Jennifer P. and Emerson, Charles P. and Lawson, Nathan D. and Watts, Jonathan K. and Sontheimer, Erik J. and Luban, Jeremy and Wolfe, Scot A.},
  doi          = {10.1038/s41587-024-02405-x},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {539-544},
  shortjournal = {Nature Biotech.},
  title        = {Increasing intracellular dNTP levels improves prime editing efficiency},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enhancing prime editing in hematopoietic stem and progenitor cells by modulating nucleotide metabolism. <em>NBT</em>, <em>43</em>(4), 534-538. (<a href='https://doi.org/10.1038/s41587-024-02266-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Therapeutic prime editing of hematopoietic stem and progenitor cells (HSPCs) holds great potential to remedy blood disorders. Quiescent cells have low nucleotide levels and resist retroviral infection, and it is possible that nucleotide metabolism could limit reverse transcription-mediated prime editing in HSPCs. We demonstrate that deoxynucleoside supplementation and Vpx-mediated degradation of SAMHD1 improve prime editing efficiency in HSPCs, especially when coupled with editing approaches that evade mismatch repair. Prime editing in hematopoietic cells is improved by engineering nucleotide metabolism.},
  archive      = {J_NBT},
  author       = {Levesque, Sébastien and Cosentino, Andrea and Verma, Archana and Genovese, Pietro and Bauer, Daniel E.},
  doi          = {10.1038/s41587-024-02266-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {534-538},
  shortjournal = {Nature Biotech.},
  title        = {Enhancing prime editing in hematopoietic stem and progenitor cells by modulating nucleotide metabolism},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering innate immune cells for cancer immunotherapy. <em>NBT</em>, <em>43</em>(4), 516-533. (<a href='https://doi.org/10.1038/s41587-025-02629-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Innate immune cells, including natural killer cells, macrophages and γδ T cells, are gaining prominence as promising candidates for cancer immunotherapy. Unlike conventional T cells, these cells possess attributes such as inherent antitumor activity, rapid immune responses, favorable safety profiles and the ability to target diverse malignancies without requiring prior antigen sensitization. In this Review, we examine the engineering strategies used to enhance their anticancer potential. We discuss challenges associated with each cell type and summarize insights from preclinical and clinical work. We propose strategies to address existing barriers, providing a perspective on the advancement of innate immune engineering as a powerful modality in anticancer treatment. The unique attributes of innate immune cells position them as attractive candidates for cancer immunotherapy, with engineering strategies to enhance their antitumor potential, offering notable therapeutic promise.},
  archive      = {J_NBT},
  author       = {Tarannum, Mubin and Ding, Xizhong and Barisa, Marta and Hu, Sabrina and Anderson, John and Romee, Rizwan and Zhang, Jin},
  doi          = {10.1038/s41587-025-02629-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {516-533},
  shortjournal = {Nature Biotech.},
  title        = {Engineering innate immune cells for cancer immunotherapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Beyond the blood: Expanding CAR t cell therapy to solid tumors. <em>NBT</em>, <em>43</em>(4), 506-515. (<a href='https://doi.org/10.1038/s41587-024-02446-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chimeric antigen receptor (CAR) T cell therapy stands as a transformative advancement in immunotherapy, triumphing against hematological malignancies and, increasingly, autoimmune disorders. After a decade of relatively modest results for solid tumors, recent clinical trials and patient reports have also started to yield promising outcomes in glioblastoma and other challenging solid tumor entities. This Perspective seeks to explore the reasons behind these latest achievements and discusses how they can be sustained and expanded through different strategies involving CAR engineering and synthetic biology. Furthermore, we critically analyze how these breakthroughs can be leveraged to maintain momentum and broaden the therapeutic impact of CAR T cells across a variety of solid tumor landscapes. Recent clinical trials testing CAR T cells on solid tumors have yielded promising outcomes that can be enhanced with further CAR engineering.},
  archive      = {J_NBT},
  author       = {Uslu, Ugur and June, Carl H.},
  doi          = {10.1038/s41587-024-02446-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {506-515},
  shortjournal = {Nature Biotech.},
  title        = {Beyond the blood: Expanding CAR t cell therapy to solid tumors},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). High-fidelity and fast 3D imaging of subcellular dynamics in native states. <em>NBT</em>, <em>43</em>(4), 504-505. (<a href='https://doi.org/10.1038/s41587-024-02293-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By integrating confocal illumination along a moving axis with scanning light-field microscopy, confocal scanning light-field microscopy captures images with high spatiotemporal resolution and optical sectioning simultaneously, with low phototoxicity. This compact technique enabled us to investigate a wide variety of rapid subcellular 3D dynamics in vivo with high fidelity over extended periods.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02293-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {504-505},
  shortjournal = {Nature Biotech.},
  title        = {High-fidelity and fast 3D imaging of subcellular dynamics in native states},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). G•U wobble base pairs improve site-directed RNA base editing. <em>NBT</em>, <em>43</em>(4), 502-503. (<a href='https://doi.org/10.1038/s41587-024-02323-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Improving the precision of gene correction through RNA base editing typically involves a trade-off with efficiency. We find using G•U wobble base pairs is a universally applicable strategy to improve editing precision and efficiency, and is effective in a mouse model of Rett syndrome in vivo.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02323-y},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {502-503},
  shortjournal = {Nature Biotech.},
  title        = {G•U wobble base pairs improve site-directed RNA base editing},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI tool adjusts for ancestral bias in genetic data. <em>NBT</em>, <em>43</em>(4), 501. (<a href='https://doi.org/10.1038/s41587-025-02651-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02651-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {501},
  shortjournal = {Nature Biotech.},
  title        = {AI tool adjusts for ancestral bias in genetic data},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineering mitochondrial DNA deletions in human cells improves disease modeling. <em>NBT</em>, <em>43</em>(4), 501. (<a href='https://doi.org/10.1038/s41587-025-02652-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02652-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {501},
  shortjournal = {Nature Biotech.},
  title        = {Engineering mitochondrial DNA deletions in human cells improves disease modeling},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineered e. coli creates biodegradable plastics. <em>NBT</em>, <em>43</em>(4), 501. (<a href='https://doi.org/10.1038/s41587-025-02653-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02653-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {501},
  shortjournal = {Nature Biotech.},
  title        = {Engineered e. coli creates biodegradable plastics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mapping the patent landscape of TROP2-targeted biologics through deep learning. <em>NBT</em>, <em>43</em>(4), 491-500. (<a href='https://doi.org/10.1038/s41587-025-02626-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There has been a rapid growth in patents related to TROP2, a transmembrane glycoprotein involved in calcium signal transduction and an attractive target for cancer therapy.},
  archive      = {J_NBT},
  author       = {Xiang, Yu and Jin, Yuyang and Liu, Kunmeng and Wei, Benzheng},
  doi          = {10.1038/s41587-025-02626-8},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {491-500},
  shortjournal = {Nature Biotech.},
  title        = {Mapping the patent landscape of TROP2-targeted biologics through deep learning},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A shifting market environment and strategies fueling innovation in prostate cancer treatment. <em>NBT</em>, <em>43</em>(4), 487-490. (<a href='https://doi.org/10.1038/s41587-025-02617-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Blockbuster drug patent expiration, advances in enabling technology, and the emergence of personalized medicine for prostate cancer treatment create a landscape worth examining.},
  archive      = {J_NBT},
  author       = {Casasanta, Michael and Frame, Nicholas and Ben-Joseph, Oded},
  doi          = {10.1038/s41587-025-02617-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {487-490},
  shortjournal = {Nature Biotech.},
  title        = {A shifting market environment and strategies fueling innovation in prostate cancer treatment},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Making cell therapy accessible: Challenges and opportunities. <em>NBT</em>, <em>43</em>(4), 482-486. (<a href='https://doi.org/10.1038/s41587-025-02625-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite major advances in bringing cancer cell therapies to the clinic, their widespread accessibility remains a substantial challenge. High manufacturing costs, lack of funding and varying regulatory standards have restricted patient access. As the field advances, researchers, clinicians, regulatory bodies, funding agencies and industry will need to work together to overcome these barriers. A group of experts share their insights into the most pressing challenges and promising strategies to make cell therapies more broadly accessible.},
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02625-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {482-486},
  shortjournal = {Nature Biotech.},
  title        = {Making cell therapy accessible: Challenges and opportunities},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cancer vaccines: The experts speak. <em>NBT</em>, <em>43</em>(4), 477-481. (<a href='https://doi.org/10.1038/s41587-025-02606-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of therapeutic cancer vaccines has long been a promising yet challenging field of research. While early efforts were impeded by limited efficacy and immune escape, recent advances in immunotherapy, mRNA technology and tumor antigen discovery have sparked renewed interest in cancer vaccine development. What is the current state of the field, and how should we address the remaining challenges to bring therapeutic cancer vaccines to the clinic? Which modalities hold potential, and what might the future of therapeutic cancer vaccines look like?},
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02606-y},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {477-481},
  shortjournal = {Nature Biotech.},
  title        = {Cancer vaccines: The experts speak},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Putting cancer immunotherapy into spatial context in the clinic. <em>NBT</em>, <em>43</em>(4), 471-476. (<a href='https://doi.org/10.1038/s41587-025-02596-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial omics technologies offer insights into the organization of cellular and molecular components and their interactions within the tumor ecosystem. Overcoming the key challenges to integrating these advances into routine clinical practice will help unlock new treatment options for patients receiving cancer immunotherapy.},
  archive      = {J_NBT},
  author       = {Engblom, Camilla and Lundeberg, Joakim},
  doi          = {10.1038/s41587-025-02596-x},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {471-476},
  shortjournal = {Nature Biotech.},
  title        = {Putting cancer immunotherapy into spatial context in the clinic},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Turn cold tumors hot by reprogramming the tumor microenvironment. <em>NBT</em>, <em>43</em>(4), 466-470. (<a href='https://doi.org/10.1038/s41587-025-02597-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The identification of novel targets together with technological advances is transforming our ability to effectively modulate the tumor microenvironment.},
  archive      = {J_NBT},
  author       = {Yuan, Xiangliang and Xiao, Yi and Yu, Dihua},
  doi          = {10.1038/s41587-025-02597-w},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {466-470},
  shortjournal = {Nature Biotech.},
  title        = {Turn cold tumors hot by reprogramming the tumor microenvironment},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Virtual cells for predictive immunotherapy. <em>NBT</em>, <em>43</em>(4), 464-465. (<a href='https://doi.org/10.1038/s41587-025-02583-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Synergizing advances in artificial intelligence with mathematical modeling will improve our ability to computationally predict immunotherapy outcomes.},
  archive      = {J_NBT},
  author       = {Bergman, Daniel R. and Fertig, Elana J.},
  doi          = {10.1038/s41587-025-02583-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {464-465},
  shortjournal = {Nature Biotech.},
  title        = {Virtual cells for predictive immunotherapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(4), 463. (<a href='https://doi.org/10.1038/s41587-025-02643-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02643-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {463},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Wildflowers soak up nickel from toxic soil. <em>NBT</em>, <em>43</em>(4), 462. (<a href='https://doi.org/10.1038/s41587-025-02647-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-025-02647-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {462},
  shortjournal = {Nature Biotech.},
  title        = {Wildflowers soak up nickel from toxic soil},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). USDA conditionally approves H5N1 poultry vaccine. <em>NBT</em>, <em>43</em>(4), 461. (<a href='https://doi.org/10.1038/s41587-025-02658-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02658-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {461},
  shortjournal = {Nature Biotech.},
  title        = {USDA conditionally approves H5N1 poultry vaccine},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Roche strikes $5.3 billion amylin deal. <em>NBT</em>, <em>43</em>(4), 460. (<a href='https://doi.org/10.1038/s41587-025-02657-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02657-1},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {460},
  shortjournal = {Nature Biotech.},
  title        = {Roche strikes $5.3 billion amylin deal},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mismatch repair inhibitors lift off in huntington’s. <em>NBT</em>, <em>43</em>(4), 458-461. (<a href='https://doi.org/10.1038/s41587-025-02646-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Startups targeting DNA repair seek to build on recent findings that neurodegeneration in Huntington’s disease is triggered only after CAG triplet repeats reach a critical threshold.},
  archive      = {J_NBT},
  author       = {Marshall, Andrew},
  doi          = {10.1038/s41587-025-02646-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {458-461},
  shortjournal = {Nature Biotech.},
  title        = {Mismatch repair inhibitors lift off in huntington’s},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). FDA approves alnylam’s anti-amyloid RNAi drug for cardiomyopathy. <em>NBT</em>, <em>43</em>(4), 457. (<a href='https://doi.org/10.1038/s41587-025-02656-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02656-2},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {457},
  shortjournal = {Nature Biotech.},
  title        = {FDA approves alnylam’s anti-amyloid RNAi drug for cardiomyopathy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Not just obesity: GLP-1 receptor agonists advance on addiction. <em>NBT</em>, <em>43</em>(4), 455-457. (<a href='https://doi.org/10.1038/s41587-025-02644-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {GLP-1 agonists may lessen alcohol, smoking and possibly drug addictions. Evidence from clinical trials remains sparse, but that is about to change.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-025-02644-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {455-457},
  shortjournal = {Nature Biotech.},
  title        = {Not just obesity: GLP-1 receptor agonists advance on addiction},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Spotlight on cancer immunotherapies. <em>NBT</em>, <em>43</em>(4), 453-454. (<a href='https://doi.org/10.1038/s41587-025-02645-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in cancer immunotherapy have shown promising results for treatment of tumors, yet challenges related to therapeutic efficacy and manufacturing processes need to be overcome.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02645-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {453-454},
  shortjournal = {Nature Biotech.},
  title        = {Spotlight on cancer immunotherapies},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(3), 452. (<a href='https://doi.org/10.1038/s41587-025-02588-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02588-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {452},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). 2H24 biotech job picture. <em>NBT</em>, <em>43</em>(3), 451. (<a href='https://doi.org/10.1038/s41587-025-02586-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A semiannual snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-025-02586-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {451},
  shortjournal = {Nature Biotech.},
  title        = {2H24 biotech job picture},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An open letter to graduate students and other procrastinators: It’s time to write. <em>NBT</em>, <em>43</em>(3), 447-450. (<a href='https://doi.org/10.1038/s41587-025-02584-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hazelett, Dennis J.},
  doi          = {10.1038/s41587-025-02584-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {447-450},
  shortjournal = {Nature Biotech.},
  title        = {An open letter to graduate students and other procrastinators: It’s time to write},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Identification of clinically relevant t cell receptors for personalized t cell therapy using combinatorial algorithms. <em>NBT</em>, <em>43</em>(3), 445. (<a href='https://doi.org/10.1038/s41587-024-02318-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Pétremand, Rémy and Chiffelle, Johanna and Bobisse, Sara and Perez, Marta A. S. and Schmidt, Julien and Arnaud, Marion and Barras, David and Lozano-Rabella, Maria and Genolet, Raphael and Sauvage, Christophe and Saugy, Damien and Michel, Alexandra and Huguenin-Bergenat, Anne-Laure and Capt, Charlotte and Moore, Jonathan S. and De Vito, Claudio and Labidi-Galy, S. Intidhar and Kandalaft, Lana E. and Dangaj Laniti, Denarda and Bassani-Sternberg, Michal and Oliveira, Giacomo and Wu, Catherine J. and Coukos, George and Zoete, Vincent and Harari, Alexandre},
  doi          = {10.1038/s41587-024-02318-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {445},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Identification of clinically relevant t cell receptors for personalized t cell therapy using combinatorial algorithms},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Computational scoring and experimental evaluation of enzymes generated by neural networks. <em>NBT</em>, <em>43</em>(3), 444. (<a href='https://doi.org/10.1038/s41587-024-02275-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Johnson, Sean R. and Fu, Xiaozhi and Viknander, Sandra and Goldin, Clara and Monaco, Sarah and Zelezniak, Aleksej and Yang, Kevin K.},
  doi          = {10.1038/s41587-024-02275-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {444},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Computational scoring and experimental evaluation of enzymes generated by neural networks},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Bioinstructive implantable scaffolds for rapid in vivo manufacture and release of CAR-T cells. <em>NBT</em>, <em>43</em>(3), 443. (<a href='https://doi.org/10.1038/s41587-025-02572-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Agarwalla, Pritha and Ogunnaike, Edikan A. and Ahn, Sarah and Froehlich, Kristen A. and Jansson, Anton and Ligler, Frances S. and Dotti, Gianpietro and Brudno, Yevgeny},
  doi          = {10.1038/s41587-025-02572-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {443},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Bioinstructive implantable scaffolds for rapid in vivo manufacture and release of CAR-T cells},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Analysis and benchmarking of small and large genomic variants across tandem repeats. <em>NBT</em>, <em>43</em>(3), 431-442. (<a href='https://doi.org/10.1038/s41587-024-02225-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tandem repeats (TRs) are highly polymorphic in the human genome, have thousands of associated molecular traits and are linked to over 60 disease phenotypes. However, they are often excluded from at-scale studies because of challenges with variant calling and representation, as well as a lack of a genome-wide standard. Here, to promote the development of TR methods, we created a catalog of TR regions and explored TR properties across 86 haplotype-resolved long-read human assemblies. We curated variants from the Genome in a Bottle (GIAB) HG002 individual to create a TR dataset to benchmark existing and future TR analysis methods. We also present an improved variant comparison method that handles variants greater than 4 bp in length and varying allelic representation. The 8.1% of the genome covered by the TR catalog holds ~24.9% of variants per individual, including 124,728 small and 17,988 large variants for the GIAB HG002 ‘truth-set’ TR benchmark. We demonstrate the utility of this pipeline across short-read and long-read technologies. A truth-set catalog of human tandem repeats allows benchmarking of variant analysis tools.},
  archive      = {J_NBT},
  author       = {English, Adam C. and Dolzhenko, Egor and Ziaei Jam, Helyaneh and McKenzie, Sean K. and Olson, Nathan D. and De Coster, Wouter and Park, Jonghun and Gu, Bida and Wagner, Justin and Eberle, Michael A. and Gymrek, Melissa and Chaisson, Mark J. P. and Zook, Justin M. and Sedlazeck, Fritz J.},
  doi          = {10.1038/s41587-024-02225-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {431-442},
  shortjournal = {Nature Biotech.},
  title        = {Analysis and benchmarking of small and large genomic variants across tandem repeats},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). CRISPRi-based circuits to control gene expression in plants. <em>NBT</em>, <em>43</em>(3), 416-430. (<a href='https://doi.org/10.1038/s41587-024-02236-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The construction of synthetic gene circuits in plants has been limited by a lack of orthogonal and modular parts. Here, we implement a CRISPR (clustered regularly interspaced short palindromic repeats) interference (CRISPRi)-based reversible gene circuit platform in plants. We create a toolkit of engineered repressible promoters of different strengths and construct NOT and NOR gates in Arabidopsis thaliana protoplasts. We determine the optimal processing system to express single guide RNAs from RNA Pol II promoters to introduce NOR gate programmability for interfacing with host regulatory sequences. The performance of a NOR gate in stably transformed Arabidopsis plants demonstrates the system’s programmability and reversibility in a complex multicellular organism. Furthermore, cross-species activity of CRISPRi-based logic gates is shown in Physcomitrium patens, Triticum aestivum and Brassica napus protoplasts. Layering multiple NOR gates together creates OR, NIMPLY and AND logic functions, highlighting the modularity of our system. Our CRISPRi circuits are orthogonal, compact, reversible, programmable and modular and provide a platform for sophisticated spatiotemporal control of gene expression in plants. Programmable and reversible CRISPRi-based genetic circuits function in a variety of plants.},
  archive      = {J_NBT},
  author       = {Khan, Muhammad Adil and Herring, Gabrielle and Zhu, Jia Yuan and Oliva, Marina and Fourie, Elliott and Johnston, Benjamin and Zhang, Zhining and Potter, Jarred and Pineda, Luke and Pflueger, Jahnvi and Swain, Tessa and Pflueger, Christian and Lloyd, James P. B. and Secco, David and Small, Ian and Kidd, Brendan N. and Lister, Ryan},
  doi          = {10.1038/s41587-024-02236-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {416-430},
  shortjournal = {Nature Biotech.},
  title        = {CRISPRi-based circuits to control gene expression in plants},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Decrypting the molecular basis of cellular drug phenotypes by dose-resolved expression proteomics. <em>NBT</em>, <em>43</em>(3), 406-415. (<a href='https://doi.org/10.1038/s41587-024-02218-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proteomics is making important contributions to drug discovery, from target deconvolution to mechanism of action (MoA) elucidation and the identification of biomarkers of drug response. Here we introduce decryptE, a proteome-wide approach that measures the full dose–response characteristics of drug-induced protein expression changes that informs cellular drug MoA. Assaying 144 clinical drugs and research compounds against 8,000 proteins resulted in more than 1 million dose–response curves that can be interactively explored online in ProteomicsDB and a custom-built Shiny App. Analysis of the collective data provided molecular explanations for known phenotypic drug effects and uncovered new aspects of the MoA of human medicines. We found that histone deacetylase inhibitors potently and strongly down-regulated the T cell receptor complex resulting in impaired human T cell activation in vitro and ex vivo. This offers a rational explanation for the efficacy of histone deacetylase inhibitors in certain lymphomas and autoimmune diseases and explains their poor performance in treating solid tumors. Proteome-wide measurements of dose–response curves for 144 drugs reveal drug mechanisms of action.},
  archive      = {J_NBT},
  author       = {Eckert, Stephan and Berner, Nicola and Kramer, Karl and Schneider, Annika and Müller, Julian and Lechner, Severin and Brajkovic, Sarah and Sakhteman, Amirhossein and Graetz, Christian and Fackler, Jonas and Dudek, Michael and Pfaffl, Michael W. and Knolle, Percy and Wilhelm, Stephanie and Kuster, Bernhard},
  doi          = {10.1038/s41587-024-02218-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {406-415},
  shortjournal = {Nature Biotech.},
  title        = {Decrypting the molecular basis of cellular drug phenotypes by dose-resolved expression proteomics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Computational scoring and experimental evaluation of enzymes generated by neural networks. <em>NBT</em>, <em>43</em>(3), 396-405. (<a href='https://doi.org/10.1038/s41587-024-02214-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In recent years, generative protein sequence models have been developed to sample novel sequences. However, predicting whether generated proteins will fold and function remains challenging. We evaluate a set of 20 diverse computational metrics to assess the quality of enzyme sequences produced by three contrasting generative models: ancestral sequence reconstruction, a generative adversarial network and a protein language model. Focusing on two enzyme families, we expressed and purified over 500 natural and generated sequences with 70–90% identity to the most similar natural sequences to benchmark computational metrics for predicting in vitro enzyme activity. Over three rounds of experiments, we developed a computational filter that improved the rate of experimental success by 50–150%. The proposed metrics and models will drive protein engineering research by serving as a benchmark for generative protein sequence models and helping to select active variants for experimental testing. Metrics that predict the success of folding and the activity of designed protein sequences are developed and experimentally validated.},
  archive      = {J_NBT},
  author       = {Johnson, Sean R. and Fu, Xiaozhi and Viknander, Sandra and Goldin, Clara and Monaco, Sarah and Zelezniak, Aleksej and Yang, Kevin K.},
  doi          = {10.1038/s41587-024-02214-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {396-405},
  shortjournal = {Nature Biotech.},
  title        = {Computational scoring and experimental evaluation of enzymes generated by neural networks},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mapping variant effects on anti-tumor hallmarks of primary human t cells with base-editing screens. <em>NBT</em>, <em>43</em>(3), 384-395. (<a href='https://doi.org/10.1038/s41587-024-02235-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-nucleotide variants (SNVs) in key T cell genes can drive clinical pathologies and could be repurposed to improve cellular cancer immunotherapies. Here, we perform massively parallel base-editing screens to generate thousands of variants at gene loci annotated with known or potential clinical relevance. We discover a broad landscape of putative gain-of-function (GOF) and loss-of-function (LOF) mutations, including in PIK3CD and the gene encoding its regulatory subunit, PIK3R1, LCK, SOS1, AKT1 and RHOA. Base editing of PIK3CD and PIK3R1 variants in T cells with an engineered T cell receptor specific to a melanoma epitope or in different generations of CD19 chimeric antigen receptor (CAR) T cells demonstrates that discovered GOF variants, but not LOF or silent mutation controls, enhanced signaling, cytokine production and lysis of cognate melanoma and leukemia cell models, respectively. Additionally, we show that generations of CD19 CAR T cells engineered with PIK3CD GOF mutations demonstrate enhanced antigen-specific signaling, cytokine production and leukemia cell killing, including when benchmarked against other recent strategies. Base-editing screens of primary T cells reveal genetic variants beneficial to immunotherapies.},
  archive      = {J_NBT},
  author       = {Walsh, Zachary H. and Shah, Parin and Kothapalli, Neeharika and Shah, Shivem B. and Nikolenyi, Gergo and Brodtman, D. Zack and Leuzzi, Giuseppe and Rogava, Meri and Mu, Michael and Ho, Patricia and Abuzaid, Sinan and Vasan, Neil and AlQuraishi, Mohammed and Milner, Joshua D. and Ciccia, Alberto and Melms, Johannes C. and Izar, Benjamin},
  doi          = {10.1038/s41587-024-02235-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {384-395},
  shortjournal = {Nature Biotech.},
  title        = {Mapping variant effects on anti-tumor hallmarks of primary human t cells with base-editing screens},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Engineered CRISPR-cas12a for higher-order combinatorial chromatin perturbations. <em>NBT</em>, <em>43</em>(3), 369-383. (<a href='https://doi.org/10.1038/s41587-024-02224-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiplexed genetic perturbations are critical for testing functional interactions among coding or non-coding genetic elements. Compared to double-stranded DNA cutting, repressive chromatin formation using CRISPR interference (CRISPRi) avoids genotoxicity and is more effective for perturbing non-coding regulatory elements in pooled assays. However, current CRISPRi pooled screening approaches are limited to targeting one to three genomic sites per cell. We engineer an Acidaminococcus Cas12a (AsCas12a) variant, multiplexed transcriptional interference AsCas12a (multiAsCas12a), that incorporates R1226A, a mutation that stabilizes the ribonucleoprotein–DNA complex via DNA nicking. The multiAsCas12a-KRAB fusion improves CRISPRi activity over DNase-dead AsCas12a-KRAB fusions, often rescuing the activities of lentivirally delivered CRISPR RNAs (crRNA) that are inactive when used with the latter. multiAsCas12a-KRAB supports CRISPRi using 6-plex crRNA arrays in high-throughput pooled screens. Using multiAsCas12a-KRAB, we discover enhancer elements and dissect the combinatorial function of cis-regulatory elements in human cells. These results instantiate a group testing framework for efficiently surveying numerous combinations of chromatin perturbations for biological discovery and engineering. An engineered Cas12a enables higher-order combinatorial functional genomic screens using CRISPR interference.},
  archive      = {J_NBT},
  author       = {Hsiung, C. C.-S. and Wilson, C. M. and Sambold, N. A. and Dai, R. and Chen, Q. and Teyssier, N. and Misiukiewicz, S. and Arab, A. and O’Loughlin, T. and Cofsky, J. C. and Shi, J. and Gilbert, L. A.},
  doi          = {10.1038/s41587-024-02224-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {369-383},
  shortjournal = {Nature Biotech.},
  title        = {Engineered CRISPR-cas12a for higher-order combinatorial chromatin perturbations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bidirectional epigenetic editing reveals hierarchies in gene regulation. <em>NBT</em>, <em>43</em>(3), 355-368. (<a href='https://doi.org/10.1038/s41587-024-02213-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR perturbation methods are limited in their ability to study non-coding elements and genetic interactions. In this study, we developed a system for bidirectional epigenetic editing, called CRISPRai, in which we apply activating (CRISPRa) and repressive (CRISPRi) perturbations to two loci simultaneously in the same cell. We developed CRISPRai Perturb-seq by coupling dual perturbation gRNA detection with single-cell RNA sequencing, enabling study of pooled perturbations in a mixed single-cell population. We applied this platform to study the genetic interaction between two hematopoietic lineage transcription factors, SPI1 and GATA1, and discovered novel characteristics of their co-regulation on downstream target genes, including differences in SPI1 and GATA1 occupancy at genes that are regulated through different modes. We also studied the regulatory landscape of IL2 (interleukin-2) in Jurkat T cells, primary T cells and chimeric antigen receptor (CAR) T cells and elucidated mechanisms of enhancer-mediated IL2 gene regulation. CRISPRai facilitates investigation of context-specific genetic interactions, provides new insights into gene regulation and will enable exploration of non-coding disease-associated variants. CRISPRai simultaneously activates and represses two genes in single cells.},
  archive      = {J_NBT},
  author       = {Pacalin, Naomi M. and Steinhart, Zachary and Shi, Quanming and Belk, Julia A. and Dorovskyi, Dmytro and Kraft, Katerina and Parker, Kevin R. and Shy, Brian R. and Marson, Alexander and Chang, Howard Y.},
  doi          = {10.1038/s41587-024-02213-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {355-368},
  shortjournal = {Nature Biotech.},
  title        = {Bidirectional epigenetic editing reveals hierarchies in gene regulation},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Self-pigmenting textiles grown from cellulose-producing bacteria with engineered tyrosinase expression. <em>NBT</em>, <em>43</em>(3), 345-354. (<a href='https://doi.org/10.1038/s41587-024-02194-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Environmental concerns are driving interest in postpetroleum synthetic textiles produced from microbial and fungal sources. Bacterial cellulose (BC) is a promising sustainable leather alternative, on account of its material properties, low infrastructure needs and biodegradability. However, for alternative textiles like BC to be fully sustainable, alternative ways to dye textiles need to be developed alongside alternative production methods. To address this, we genetically engineer Komagataeibacter rhaeticus to create a bacterial strain that grows self-pigmenting BC. Melanin biosynthesis in the bacteria from recombinant tyrosinase expression achieves dark black coloration robust to material use. Melanated BC production can be scaled up for the construction of prototype fashion products, and we illustrate the potential of combining engineered self-pigmentation with tools from synthetic biology, through the optogenetic patterning of gene expression in cellulose-producing bacteria. With this study, we demonstrate that combining genetic engineering with current and future methods of textile biofabrication has the potential to create a new class of textiles. Genetic engineering of material-producing bacteria creates pigmented products.},
  archive      = {J_NBT},
  author       = {Walker, Kenneth T. and Li, Ivy S. and Keane, Jennifer and Goosens, Vivianne J. and Song, Wenzhe and Lee, Koon-Yang and Ellis, Tom},
  doi          = {10.1038/s41587-024-02194-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {345-354},
  shortjournal = {Nature Biotech.},
  title        = {Self-pigmenting textiles grown from cellulose-producing bacteria with engineered tyrosinase expression},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Generation of allogeneic CAR-NKT cells from hematopoietic stem and progenitor cells using a clinically guided culture method. <em>NBT</em>, <em>43</em>(3), 329-344. (<a href='https://doi.org/10.1038/s41587-024-02226-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer immunotherapy with autologous chimeric antigen receptor (CAR) T cells faces challenges in manufacturing and patient selection that could be avoided by using ‘off-the-shelf’ products, such as allogeneic CAR natural killer T (AlloCAR-NKT) cells. Previously, we reported a system for differentiating human hematopoietic stem and progenitor cells into AlloCAR-NKT cells, but the use of three-dimensional culture and xenogeneic feeders precluded its clinical application. Here we describe a clinically guided method to differentiate and expand IL-15-enhanced AlloCAR-NKT cells with high yield and purity. We generated AlloCAR-NKT cells targeting seven cancers and, in a multiple myeloma model, demonstrated their antitumor efficacy, expansion and persistence. The cells also selectively depleted immunosuppressive cells in the tumor microenviroment and antagonized tumor immune evasion via triple targeting of CAR, TCR and NK receptors. They exhibited a stable hypoimmunogenic phenotype associated with epigenetic and signaling regulation and did not induce detectable graft versus host disease or cytokine release syndrome. These properties of AlloCAR-NKT cells support their potential for clinical translation. Off-the-shelf CAR-NKT cells for cancer immunotherapy are advanced toward clinical translation.},
  archive      = {J_NBT},
  author       = {Li, Yan-Ruide and Zhou, Yang and Yu, Jiaji and Kim, Yu Jeong and Li, Miao and Lee, Derek and Zhou, Kuangyi and Chen, Yuning and Zhu, Yichen and Wang, Yu-Chen and Li, Zhe and Yu, Yanqi and Dunn, Zachary Spencer and Guo, Wenbin and Cen, Xinjian and Husman, Tiffany and Bajpai, Aarushi and Kramer, Adam and Wilson, Matthew and Fang, Ying and Huang, Jie and Li, Shuo and Zhou, Yonggang and Zhang, Yuchong and Hahn, Zoe and Zhu, Enbo and Ma, Feiyang and Pan, Calvin and Lusis, Aldons J. and Zhou, Jin J. and Seet, Christopher S. and Kohn, Donald B. and Wang, Pin and Zhou, Xianghong Jasmine and Pellegrini, Matteo and Puliafito, Benjamin R. and Larson, Sarah M. and Yang, Lili},
  doi          = {10.1038/s41587-024-02226-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {329-344},
  shortjournal = {Nature Biotech.},
  title        = {Generation of allogeneic CAR-NKT cells from hematopoietic stem and progenitor cells using a clinically guided culture method},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Identification of clinically relevant t cell receptors for personalized t cell therapy using combinatorial algorithms. <em>NBT</em>, <em>43</em>(3), 323-328. (<a href='https://doi.org/10.1038/s41587-024-02232-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A central challenge in developing personalized cancer cell immunotherapy is the identification of tumor-reactive T cell receptors (TCRs). By exploiting the distinct transcriptomic profile of tumor-reactive T cells relative to bystander cells, we build and benchmark TRTpred, an antigen-agnostic in silico predictor of tumor-reactive TCRs. We integrate TRTpred with an avidity predictor to derive a combinatorial algorithm of clinically relevant TCRs for personalized T cell therapy and benchmark it in patient-derived xenografts. Tumor-reactive T cell receptors are predicted based on single-cell RNA sequencing and single-cell T cell receptor sequencing.},
  archive      = {J_NBT},
  author       = {Pétremand, Rémy and Chiffelle, Johanna and Bobisse, Sara and Perez, Marta A. S. and Schmidt, Julien and Arnaud, Marion and Barras, David and Lozano-Rabella, Maria and Genolet, Raphael and Sauvage, Christophe and Saugy, Damien and Michel, Alexandra and Huguenin-Bergenat, Anne-Laure and Capt, Charlotte and Moore, Jonathan S. and De Vito, Claudio and Labidi-Galy, S. Intidhar and Kandalaft, Lana E. and Dangaj Laniti, Denarda and Bassani-Sternberg, Michal and Oliveira, Giacomo and Wu, Catherine J. and Coukos, George and Zoete, Vincent and Harari, Alexandre},
  doi          = {10.1038/s41587-024-02232-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {323-328},
  shortjournal = {Nature Biotech.},
  title        = {Identification of clinically relevant t cell receptors for personalized t cell therapy using combinatorial algorithms},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Toward single-molecule protein sequencing using nanopores. <em>NBT</em>, <em>43</em>(3), 312-322. (<a href='https://doi.org/10.1038/s41587-025-02587-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the past three decades, biological nanopore sequencing has grown from a research curiosity to a mature technology to sequence nucleic acids at the single-molecule level. Now, recent achievements suggest that nanopores might be able to sequence proteins soon. In this Perspective, we analyze the different approaches that have been proposed to measure proteins and peptides using nanopores. We predict that, more likely than not, nanopores will be capable of identifying full-length proteins at the single-molecule level and with single-amino acid resolution, paving the way to single-molecule protein sequencing. This would allow several applications in proteomics that are at present challenging, including measuring the heterogeneity of post-translational modifications, quantifying low-abundance proteins and characterizing protein splicing. Maglia and colleagues discuss advances in nanopore technology en route to single-molecule protein sequencing},
  archive      = {J_NBT},
  author       = {Lu, Chunzhe and Bonini, Andrea and Viel, Jakob H. and Maglia, Giovanni},
  doi          = {10.1038/s41587-025-02587-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {312-322},
  shortjournal = {Nature Biotech.},
  title        = {Toward single-molecule protein sequencing using nanopores},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Advancing programmable gene expression in plants using CRISPRi-based boolean gates. <em>NBT</em>, <em>43</em>(3), 310-311. (<a href='https://doi.org/10.1038/s41587-024-02273-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To advance the toolset for controlling plant gene expression, we developed a CRISPR interference-based platform for the construction of synthetic Boolean logic gates that is functional in multiple plant species. These genetic circuits are programmable and reversible in nature, which will enable spatiotemporal control of plant responses to dynamic cues.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02273-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {310-311},
  shortjournal = {Nature Biotech.},
  title        = {Advancing programmable gene expression in plants using CRISPRi-based boolean gates},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Assessing the laboratory performance of AI-generated enzymes. <em>NBT</em>, <em>43</em>(3), 308-309. (<a href='https://doi.org/10.1038/s41587-024-02239-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A set of 20 computational metrics was evaluated to determine whether they could predict the functionality of synthetic enzyme sequences produced by generative protein models, resulting in the development of a computational filter, COMPSS, that increased experimental success rates by 50–150%, tested in over 500 natural and AI-generated enzymes.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02239-7},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {308-309},
  shortjournal = {Nature Biotech.},
  title        = {Assessing the laboratory performance of AI-generated enzymes},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genetically encoding colors and images into bioengineered microbial materials. <em>NBT</em>, <em>43</em>(3), 306-307. (<a href='https://doi.org/10.1038/s41587-024-02200-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using synthetic biology, we engineered a cellulose-producing bacterium that can produce eumelanin and respond to light, so that it is possible to grow a microbial leather material that is colored black or contains projected black patterns.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02200-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {306-307},
  shortjournal = {Nature Biotech.},
  title        = {Genetically encoding colors and images into bioengineered microbial materials},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Decoding gene regulation with CRISPR perturbations. <em>NBT</em>, <em>43</em>(3), 304-305. (<a href='https://doi.org/10.1038/s41587-024-02222-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two CRISPR tools for combinatorial genetic perturbations reveal gene regulatory networks.},
  archive      = {J_NBT},
  author       = {Oberlin, Stefan and McManus, Michael T.},
  doi          = {10.1038/s41587-024-02222-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {304-305},
  shortjournal = {Nature Biotech.},
  title        = {Decoding gene regulation with CRISPR perturbations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A milestone method to make natural killer t cells. <em>NBT</em>, <em>43</em>(3), 302-303. (<a href='https://doi.org/10.1038/s41587-024-02243-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A differentiation protocol to produce off-the-shelf natural killer T cells may enable clinical application.},
  archive      = {J_NBT},
  author       = {Metelitsa, Leonid S.},
  doi          = {10.1038/s41587-024-02243-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {302-303},
  shortjournal = {Nature Biotech.},
  title        = {A milestone method to make natural killer t cells},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Intranasal bacteria transport obesity drugs to the brain. <em>NBT</em>, <em>43</em>(3), 301. (<a href='https://doi.org/10.1038/s41587-025-02598-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02598-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {301},
  shortjournal = {Nature Biotech.},
  title        = {Intranasal bacteria transport obesity drugs to the brain},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fluorescence imaging of RNA and protein in deep tissue. <em>NBT</em>, <em>43</em>(3), 301. (<a href='https://doi.org/10.1038/s41587-025-02604-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02604-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {301},
  shortjournal = {Nature Biotech.},
  title        = {Fluorescence imaging of RNA and protein in deep tissue},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Microbial metalloenzyme boosts cellulose breakdown. <em>NBT</em>, <em>43</em>(3), 301. (<a href='https://doi.org/10.1038/s41587-025-02615-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-025-02615-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {301},
  shortjournal = {Nature Biotech.},
  title        = {Microbial metalloenzyme boosts cellulose breakdown},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Epigenetics. <em>NBT</em>, <em>43</em>(3), 300. (<a href='https://doi.org/10.1038/s41587-025-02582-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to epigenetic modification and analysis.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02582-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {300},
  shortjournal = {Nature Biotech.},
  title        = {Epigenetics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The puzzle of biologics manufacturing platform patents. <em>NBT</em>, <em>43</em>(3), 295-299. (<a href='https://doi.org/10.1038/s41587-025-02579-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The predominance of manufacturing process patents asserted in litigation by originator biologics companies against would-be biosimilar entrants has resulted in a number of Congressional and administrative agency proposals that could increase scrutiny and limit enforceability.},
  archive      = {J_NBT},
  author       = {Jefferson, Osmat Azzam and Price, W. Nicholson and Tu, S. Sean and Vishnubhakat, Saurabh and Rai, Arti K.},
  doi          = {10.1038/s41587-025-02579-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {295-299},
  shortjournal = {Nature Biotech.},
  title        = {The puzzle of biologics manufacturing platform patents},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(3), 294. (<a href='https://doi.org/10.1038/s41587-025-02599-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02599-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {294},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Trapped in a lake, founder spots a bioplastic opportunity. <em>NBT</em>, <em>43</em>(3), 293. (<a href='https://doi.org/10.1038/s41587-025-02605-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-025-02605-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {293},
  shortjournal = {Nature Biotech.},
  title        = {Trapped in a lake, founder spots a bioplastic opportunity},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Tough times for CRISPR startups. <em>NBT</em>, <em>43</em>(3), 292. (<a href='https://doi.org/10.1038/s41587-025-02609-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02609-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {292},
  shortjournal = {Nature Biotech.},
  title        = {Tough times for CRISPR startups},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Baby genomics: Newborn sequencing starts to fulfill its promise. <em>NBT</em>, <em>43</em>(3), 290-292. (<a href='https://doi.org/10.1038/s41587-025-02595-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome sequencing in newborns is delivering insights into severe genetic diseases not afforded by traditional tests, but integrating genomics into newborn screening remains a logistics and cost challenge.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-025-02595-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {290-292},
  shortjournal = {Nature Biotech.},
  title        = {Baby genomics: Newborn sequencing starts to fulfill its promise},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Philanthropic giants join forces over antimicrobial resistance. <em>NBT</em>, <em>43</em>(3), 289. (<a href='https://doi.org/10.1038/s41587-025-02620-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02620-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {289},
  shortjournal = {Nature Biotech.},
  title        = {Philanthropic giants join forces over antimicrobial resistance},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). First inhaled lentiviral gene therapy enters cystic fibrosis trial. <em>NBT</em>, <em>43</em>(3), 288. (<a href='https://doi.org/10.1038/s41587-025-02616-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02616-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {288},
  shortjournal = {Nature Biotech.},
  title        = {First inhaled lentiviral gene therapy enters cystic fibrosis trial},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Bite-sized solutions. <em>NBT</em>, <em>43</em>(3), 285-286. (<a href='https://doi.org/10.1038/s41587-025-02603-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study found high amounts of microplastics in the human brain. This could spur funding and technological advances for plastic degradation.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02603-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {285-286},
  shortjournal = {Nature Biotech.},
  title        = {Bite-sized solutions},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vertex’s opioid-free drug for acute pain wins FDA approval. <em>NBT</em>, <em>43</em>(3), 287-289. (<a href='https://doi.org/10.1038/s41587-025-02590-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Society, patients and clinicians welcome a much-needed non-opioid pain medication. Vertex’s first-in-class analgesic Journavx could soon be followed by a new generation of addiction-free pain drugs acting at NaV1.8 sodium channels.},
  archive      = {J_NBT},
  author       = {Harrison, Charlotte},
  doi          = {10.1038/s41587-025-02590-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {3},
  pages        = {287-289},
  shortjournal = {Nature Biotech.},
  title        = {Vertex’s opioid-free drug for acute pain wins FDA approval},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(2), 284. (<a href='https://doi.org/10.1038/s41587-025-02557-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02557-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {284},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Undergraduate education as catalyst for public–private partnership in biotechnology. <em>NBT</em>, <em>43</em>(2), 282-283. (<a href='https://doi.org/10.1038/s41587-025-02552-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An innovative challenge-based and research-driven educational model immerses groups of undergraduate students in the discovery stages of biotech industry research.},
  archive      = {J_NBT},
  author       = {ten Broeke, Toine and Koorman, Thijs and Crnko, Sandra and Voets, Erik and de Goeje, Pauline and Schakelaar, Michael Y. and Rebel, Heggert G. and Pijnappel, Emma W. and Sanders, Jorine G. F. and Meeldijk, Jan and Terveer-Couperus, Rian and Geven, Esther L. and Dilaver, Gönül and van Tilborg, Erik and Beekman, Lucas M. P. and Guelen, Lars and Bovenschen, Niels},
  doi          = {10.1038/s41587-025-02552-9},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {282-283},
  shortjournal = {Nature Biotech.},
  title        = {Undergraduate education as catalyst for public–private partnership in biotechnology},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: The global patent landscape of functional food innovation. <em>NBT</em>, <em>43</em>(2), 281. (<a href='https://doi.org/10.1038/s41587-024-02544-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Matin, Maima and Hrg, Dalibor and Litvinova, Olena and Łysek-Gładysinska, Małgorzata and Wierzbicka, Agnieszka and Horbańczuk, Jarosław Olav and Jóźwik, Artur and Atanasov, Atanas G.},
  doi          = {10.1038/s41587-024-02544-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {281},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: The global patent landscape of functional food innovation},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The covariance environment defines cellular niches for spatial inference. <em>NBT</em>, <em>43</em>(2), 269-280. (<a href='https://doi.org/10.1038/s41587-024-02193-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A key challenge of analyzing data from high-resolution spatial profiling technologies is to suitably represent the features of cellular neighborhoods or niches. Here we introduce the covariance environment (COVET), a representation that leverages the gene–gene covariate structure across cells in the niche to capture the multivariate nature of cellular interactions within it. We define a principled optimal transport-based distance metric between COVET niches that scales to millions of cells. Using COVET to encode spatial context, we developed environmental variational inference (ENVI), a conditional variational autoencoder that jointly embeds spatial and single-cell RNA sequencing data into a latent space. ENVI includes two decoders: one to impute gene expression across the spatial modality and a second to project spatial information onto single-cell data. ENVI can confer spatial context to genomics data from single dissociated cells and outperforms alternatives for imputing gene expression on diverse spatial datasets. Using gene–gene covariance structure to represent cellular neighborhoods improves the integration of single-cell RNA sequencing and spatial transcriptomics data.},
  archive      = {J_NBT},
  author       = {Haviv, Doron and Remšík, Ján and Gatie, Mohamed and Snopkowski, Catherine and Takizawa, Meril and Pereira, Nathan and Bashkin, John and Jovanovich, Stevan and Nawy, Tal and Chaligne, Ronan and Boire, Adrienne and Hadjantonakis, Anna-Katerina and Pe’er, Dana},
  doi          = {10.1038/s41587-024-02193-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {269-280},
  shortjournal = {Nature Biotech.},
  title        = {The covariance environment defines cellular niches for spatial inference},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gene trajectory inference for single-cell data by optimal transport metrics. <em>NBT</em>, <em>43</em>(2), 258-268. (<a href='https://doi.org/10.1038/s41587-024-02186-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing has been widely used to investigate cell state transitions and gene dynamics of biological processes. Current strategies to infer the sequential dynamics of genes in a process typically rely on constructing cell pseudotime through cell trajectory inference. However, the presence of concurrent gene processes in the same group of cells and technical noise can obscure the true progression of the processes studied. To address this challenge, we present GeneTrajectory, an approach that identifies trajectories of genes rather than trajectories of cells. Specifically, optimal transport distances are calculated between gene distributions across the cell–cell graph to extract gene programs and define their gene pseudotemporal order. Here we demonstrate that GeneTrajectory accurately extracts progressive gene dynamics in myeloid lineage maturation. Moreover, we show that GeneTrajectory deconvolves key gene programs underlying mouse skin hair follicle dermal condensate differentiation that could not be resolved by cell trajectory approaches. GeneTrajectory facilitates the discovery of gene programs that control the changes and activities of biological processes. Gene dynamics of concurrent biological processes are unraveled by considering gene trajectories instead of cell trajectories.},
  archive      = {J_NBT},
  author       = {Qu, Rihao and Cheng, Xiuyuan and Sefik, Esen and Stanley III, Jay S. and Landa, Boris and Strino, Francesco and Platt, Sarah and Garritano, James and Odell, Ian D. and Coifman, Ronald and Flavell, Richard A. and Myung, Peggy and Kluger, Yuval},
  doi          = {10.1038/s41587-024-02186-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {258-268},
  shortjournal = {Nature Biotech.},
  title        = {Gene trajectory inference for single-cell data by optimal transport metrics},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Inferring gene regulatory networks from single-cell multiome data using atlas-scale external data. <em>NBT</em>, <em>43</em>(2), 247-257. (<a href='https://doi.org/10.1038/s41587-024-02182-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing methods for gene regulatory network (GRN) inference rely on gene expression data alone or on lower resolution bulk data. Despite the recent integration of chromatin accessibility and RNA sequencing data, learning complex mechanisms from limited independent data points still presents a daunting challenge. Here we present LINGER (Lifelong neural network for gene regulation), a machine-learning method to infer GRNs from single-cell paired gene expression and chromatin accessibility data. LINGER incorporates atlas-scale external bulk data across diverse cellular contexts and prior knowledge of transcription factor motifs as a manifold regularization. LINGER achieves a fourfold to sevenfold relative increase in accuracy over existing methods and reveals a complex regulatory landscape of genome-wide association studies, enabling enhanced interpretation of disease-associated variants and genes. Following the GRN inference from reference single-cell multiome data, LINGER enables the estimation of transcription factor activity solely from bulk or single-cell gene expression data, leveraging the abundance of available gene expression data to identify driver regulators from case-control studies. Accuracy of gene regulatory network inference is increased by combining multiome single-cell and atlas-scale bulk data.},
  archive      = {J_NBT},
  author       = {Yuan, Qiuyue and Duren, Zhana},
  doi          = {10.1038/s41587-024-02182-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {247-257},
  shortjournal = {Nature Biotech.},
  title        = {Inferring gene regulatory networks from single-cell multiome data using atlas-scale external data},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Therapeutic application of circular RNA aptamers in a mouse model of psoriasis. <em>NBT</em>, <em>43</em>(2), 236-246. (<a href='https://doi.org/10.1038/s41587-024-02204-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Efforts to advance RNA aptamers as a new therapeutic modality have been limited by their susceptibility to degradation and immunogenicity. In a previous study, we demonstrated synthesized short double-stranded region-containing circular RNAs (ds-cRNAs) with minimal immunogenicity targeted to dsRNA-activated protein kinase R (PKR). Here we test the therapeutic potential of ds-cRNAs in a mouse model of imiquimod-induced psoriasis. We find that genetic supplementation of ds-cRNAs leads to inhibition of PKR, resulting in alleviation of downstream interferon-α and dsRNA signals and attenuation of psoriasis phenotypes. Delivery of ds-cRNAs by lipid nanoparticles to the spleen attenuates PKR activity in examined splenocytes, resulting in reduced epidermal thickness. These findings suggest that ds-cRNAs represent a promising approach to mitigate excessive PKR activation for therapeutic purposes. A circular RNA aptamer shows therapeutic efficacy in vivo in a psoriasis model.},
  archive      = {J_NBT},
  author       = {Guo, Si-Kun and Liu, Chu-Xiao and Xu, Yi-Feng and Wang, Xiao and Nan, Fang and Huang, Youkui and Li, Siqi and Nan, Shan and Li, Ling and Kon, Edo and Li, Chen and Wei, Meng-Yuan and Su, Rina and Wei, Jia and Peng, Shiguang and Ad-El, Nitay and Liu, Jiaquan and Peer, Dan and Chen, Ting and Yang, Li and Chen, Ling-Ling},
  doi          = {10.1038/s41587-024-02204-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {236-246},
  shortjournal = {Nature Biotech.},
  title        = {Therapeutic application of circular RNA aptamers in a mouse model of psoriasis},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Starfysh integrates spatial transcriptomic and histologic data to reveal heterogeneous tumor–immune hubs. <em>NBT</em>, <em>43</em>(2), 223-235. (<a href='https://doi.org/10.1038/s41587-024-02173-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatially resolved gene expression profiling provides insight into tissue organization and cell–cell crosstalk; however, sequencing-based spatial transcriptomics (ST) lacks single-cell resolution. Current ST analysis methods require single-cell RNA sequencing data as a reference for rigorous interpretation of cell states, mostly do not use associated histology images and are not capable of inferring shared neighborhoods across multiple tissues. Here we present Starfysh, a computational toolbox using a deep generative model that incorporates archetypal analysis and any known cell type markers to characterize known or new tissue-specific cell states without a single-cell reference. Starfysh improves the characterization of spatial dynamics in complex tissues using histology images and enables the comparison of niches as spatial hubs across tissues. Integrative analysis of primary estrogen receptor (ER)-positive breast cancer, triple-negative breast cancer (TNBC) and metaplastic breast cancer (MBC) tissues led to the identification of spatial hubs with patient- and disease-specific cell type compositions and revealed metabolic reprogramming shaping immunosuppressive hubs in aggressive MBC. Reference-free integrative analysis of spatial transcriptomics data is achieved with archetype analysis.},
  archive      = {J_NBT},
  author       = {He, Siyu and Jin, Yinuo and Nazaret, Achille and Shi, Lingting and Chen, Xueer and Rampersaud, Sham and Dhillon, Bahawar S. and Valdez, Izabella and Friend, Lauren E. and Fan, Joy Linyue and Park, Cameron Y. and Mintz, Rachel L. and Lao, Yeh-Hsing and Carrera, David and Fang, Kaylee W. and Mehdi, Kaleem and Rohde, Madeline and McFaline-Figueroa, José L. and Blei, David and Leong, Kam W. and Rudensky, Alexander Y. and Plitas, George and Azizi, Elham},
  doi          = {10.1038/s41587-024-02173-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {223-235},
  shortjournal = {Nature Biotech.},
  title        = {Starfysh integrates spatial transcriptomic and histologic data to reveal heterogeneous tumor–immune hubs},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Discovery of tumor-reactive t cell receptors by massively parallel library synthesis and screening. <em>NBT</em>, <em>43</em>(2), 214-222. (<a href='https://doi.org/10.1038/s41587-024-02210-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cell receptor (TCR) gene therapy is a potent form of cellular immunotherapy in which patient T cells are genetically engineered to express TCRs with defined tumor reactivity. However, the isolation of therapeutic TCRs is complicated by both the general scarcity of tumor-specific T cells among patient T cell repertoires and the patient-specific nature of T cell epitopes expressed on tumors. Here we describe a high-throughput, personalized TCR discovery pipeline that enables the assembly of complex synthetic TCR libraries in a one-pot reaction, followed by pooled expression in reporter T cells and functional genetic screening against patient-derived tumor or antigen-presenting cells. We applied the method to screen thousands of tumor-infiltrating lymphocyte (TIL)-derived TCRs from multiple patients and identified dozens of CD4+ and CD8+ T-cell-derived TCRs with potent tumor reactivity, including TCRs that recognized patient-specific neoantigens. Tumor-specific T cell receptors (TCRs) are identified by functional screening of synthetic TCR libraries.},
  archive      = {J_NBT},
  author       = {Moravec, Ziva and Zhao, Yue and Voogd, Rhianne and Cook, Danielle R. and Kinrot, Seon and Capra, Benjamin and Yang, Haiyan and Raud, Brenda and Ou, Jiayu and Xuan, Jiekun and Wei, Teng and Ren, Lili and Hu, Dandan and Wang, Jun and Haanen, John B.A.G. and Schumacher, Ton N. and Chen, Xi and Porter, Ely and Scheper, Wouter},
  doi          = {10.1038/s41587-024-02210-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {214-222},
  shortjournal = {Nature Biotech.},
  title        = {Discovery of tumor-reactive t cell receptors by massively parallel library synthesis and screening},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Genome engineering with cas9 and AAV repair templates generates frequent concatemeric insertions of viral vectors. <em>NBT</em>, <em>43</em>(2), 204-213. (<a href='https://doi.org/10.1038/s41587-024-02171-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR–Cas9 paired with adeno-associated virus serotype 6 (AAV6) is among the most efficient tools for producing targeted gene knockins. Here, we report that this system can lead to frequent concatemeric insertions of the viral vector genome at the target site that are difficult to detect. Such errors can cause adverse and unreliable phenotypes that are antithetical to the goal of precision genome engineering. The concatemeric knockins occurred regardless of locus, vector concentration, cell line or cell type, including human pluripotent and hematopoietic stem cells. Although these highly abundant errors were found in more than half of the edited cells, they could not be readily detected by common analytical methods. We describe strategies to detect and thoroughly characterize the concatemeric viral vector insertions, and we highlight analytical pitfalls that mask their prevalence. We then describe strategies to prevent the concatemeric inserts by cutting the vector genome after transduction. This approach is compatible with established gene editing pipelines, enabling robust genetic knockins that are safer, more reliable and more reproducible. AAV vectors form difficult-to-detect concatemers at Cas9 target sites.},
  archive      = {J_NBT},
  author       = {Suchy, Fabian P. and Karigane, Daiki and Nakauchi, Yusuke and Higuchi, Maimi and Zhang, Jinyu and Pekrun, Katja and Hsu, Ian and Fan, Amy C. and Nishimura, Toshinobu and Charlesworth, Carsten T. and Bhadury, Joydeep and Nishimura, Toshiya and Wilkinson, Adam C. and Kay, Mark A. and Majeti, Ravindra and Nakauchi, Hiromitsu},
  doi          = {10.1038/s41587-024-02171-w},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {204-213},
  shortjournal = {Nature Biotech.},
  title        = {Genome engineering with cas9 and AAV repair templates generates frequent concatemeric insertions of viral vectors},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Branched chemically modified poly(A) tails enhance the translation capacity of mRNA. <em>NBT</em>, <em>43</em>(2), 194-203. (<a href='https://doi.org/10.1038/s41587-024-02174-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although messenger RNA (mRNA) has proved effective as a vaccine, its potential as a general therapeutic modality is limited by its instability and low translation capacity. To increase the duration and level of protein expression from mRNA, we designed and synthesized topologically and chemically modified mRNAs with multiple synthetic poly(A) tails. Here we demonstrate that the optimized multitailed mRNA yielded ~4.7–19.5-fold higher luminescence signals than the control mRNA from 24 to 72 h post transfection in cellulo and 14 days detectable signal versus &lt;7 days signal from the control in vivo. We further achieve efficient multiplexed genome editing of the clinically relevant genes Pcsk9 and Angptl3 in mouse liver at a minimal mRNA dosage. Taken together, these results provide a generalizable approach to synthesize capped branched mRNA with markedly enhanced translation capacity. mRNA with engineered poly(A) tails produces prolonged higher levels of protein.},
  archive      = {J_NBT},
  author       = {Chen, Hongyu and Liu, Dangliang and Guo, Jianting and Aditham, Abhishek and Zhou, Yiming and Tian, Jiakun and Luo, Shuchen and Ren, Jingyi and Hsu, Alvin and Huang, Jiahao and Kostas, Franklin and Wu, Mingrui and Liu, David R. and Wang, Xiao},
  doi          = {10.1038/s41587-024-02174-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {194-203},
  shortjournal = {Nature Biotech.},
  title        = {Branched chemically modified poly(A) tails enhance the translation capacity of mRNA},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Development of supramolecular anticoagulants with on-demand reversibility. <em>NBT</em>, <em>43</em>(2), 186-193. (<a href='https://doi.org/10.1038/s41587-024-02209-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drugs are administered at a dosing schedule set by their therapeutic index, and termination of action is achieved by clearance and metabolism of the drug. In some cases, such as anticoagulant drugs or immunotherapeutics, it is important to be able to quickly reverse the drug’s action. Here, we report a general strategy to achieve on-demand reversibility by designing a supramolecular drug (a noncovalent assembly of two cooperatively interacting drug fragments held together by transient hybridization of peptide nucleic acid (PNA)) that can be reversed with a PNA antidote that outcompetes the hybridization between the fragments. We demonstrate the approach with thrombin-inhibiting anticoagulants, creating very potent and reversible bivalent direct thrombin inhibitors (Ki = 74 pM). The supramolecular inhibitor effectively inhibited thrombus formation in mice in a needle injury thrombosis model, and this activity could be reversed by administration of the PNA antidote. This design is applicable to therapeutic targets where two binding sites can be identified. A supramolecular drug design yields anticoagulants that are rapidly reversible.},
  archive      = {J_NBT},
  author       = {Dockerill, Millicent and Ford, Daniel J. and Angerani, Simona and Alwis, Imala and Dowman, Luke J. and Ripoll-Rozada, Jorge and Smythe, Rhyll E. and Liu, Joanna S. T. and Pereira, Pedro José Barbosa and Jackson, Shaun P. and Payne, Richard J. and Winssinger, Nicolas},
  doi          = {10.1038/s41587-024-02209-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {186-193},
  shortjournal = {Nature Biotech.},
  title        = {Development of supramolecular anticoagulants with on-demand reversibility},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). De novo and somatic structural variant discovery with SVision-pro. <em>NBT</em>, <em>43</em>(2), 181-185. (<a href='https://doi.org/10.1038/s41587-024-02190-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read-based de novo and somatic structural variant (SV) discovery remains challenging, necessitating genomic comparison between samples. We developed SVision-pro, a neural-network-based instance segmentation framework that represents genome-to-genome-level sequencing differences visually and discovers SV comparatively between genomes without any prerequisite for inference models. SVision-pro outperforms state-of-the-art approaches, in particular, the resolving of complex SVs is improved, with low Mendelian error rates, high sensitivity of low-frequency SVs and reduced false-positive rates compared with SV merging approaches. SVision-pro enables pairwise genome comparison and improves de novo and somatic structural variant calling.},
  archive      = {J_NBT},
  author       = {Wang, Songbo and Lin, Jiadong and Jia, Peng and Xu, Tun and Li, Xiujuan and Liu, Yuezhuangnan and Xu, Dan and Bush, Stephen J. and Meng, Deyu and Ye, Kai},
  doi          = {10.1038/s41587-024-02190-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {181-185},
  shortjournal = {Nature Biotech.},
  title        = {De novo and somatic structural variant discovery with SVision-pro},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Capturing and modeling cellular niches from dissociated single-cell and spatial data. <em>NBT</em>, <em>43</em>(2), 179-180. (<a href='https://doi.org/10.1038/s41587-024-02207-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells interact with their local environment to enact global tissue function. By harnessing gene–gene covariation in cellular neighborhoods from spatial transcriptomics data, the covariance environment (COVET) niche representation and the environmental variational inference (ENVI) data integration method model phenotype–microenvironment interplay and reconstruct the spatial context of dissociated single-cell RNA sequencing datasets.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02207-1},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {179-180},
  shortjournal = {Nature Biotech.},
  title        = {Capturing and modeling cellular niches from dissociated single-cell and spatial data},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Designing drugs with reversible activity. <em>NBT</em>, <em>43</em>(2), 177-178. (<a href='https://doi.org/10.1038/s41587-024-02242-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A strategy for creating drugs that can be quickly neutralized is demonstrated for anticoagulants.},
  archive      = {J_NBT},
  author       = {Neri, Dario},
  doi          = {10.1038/s41587-024-02242-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {177-178},
  shortjournal = {Nature Biotech.},
  title        = {Designing drugs with reversible activity},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A chromatin-based model for deciphering gene interactions. <em>NBT</em>, <em>43</em>(2), 176. (<a href='https://doi.org/10.1038/s41587-025-02573-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Li, Rong},
  doi          = {10.1038/s41587-025-02573-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {176},
  shortjournal = {Nature Biotech.},
  title        = {A chromatin-based model for deciphering gene interactions},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Heat-resistant rice without yield compromise. <em>NBT</em>, <em>43</em>(2), 176. (<a href='https://doi.org/10.1038/s41587-025-02574-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Despang, Alexandra},
  doi          = {10.1038/s41587-025-02574-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {176},
  shortjournal = {Nature Biotech.},
  title        = {Heat-resistant rice without yield compromise},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). De novo-designed protein binders neutralize snake toxins. <em>NBT</em>, <em>43</em>(2), 176. (<a href='https://doi.org/10.1038/s41587-025-02575-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Bishop, Anahita},
  doi          = {10.1038/s41587-025-02575-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {176},
  shortjournal = {Nature Biotech.},
  title        = {De novo-designed protein binders neutralize snake toxins},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Antibodies. <em>NBT</em>, <em>43</em>(2), 175. (<a href='https://doi.org/10.1038/s41587-024-02547-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to antibody compositions and methods of use.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02547-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {175},
  shortjournal = {Nature Biotech.},
  title        = {Antibodies},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Redefining antibody patent protection using paratope mapping and CDR-scanning. <em>NBT</em>, <em>43</em>(2), 170-174. (<a href='https://doi.org/10.1038/s41587-024-02549-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Patent protection for therapeutic monoclonal antibodies requires a fundamental shift in strategy following Amgen v. Sanofi, which invalidated broad functional claims for antibody genera in the USA.},
  archive      = {J_NBT},
  author       = {Banik, Soma S. R. and Deng, Xiaoxiang and Davidson, Edgar and Storz, Ulrich and Doranz, Benjamin J.},
  doi          = {10.1038/s41587-024-02549-w},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {170-174},
  shortjournal = {Nature Biotech.},
  title        = {Redefining antibody patent protection using paratope mapping and CDR-scanning},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A platform for the biomedical application of large language models. <em>NBT</em>, <em>43</em>(2), 166-169. (<a href='https://doi.org/10.1038/s41587-024-02534-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Lobentanzer, Sebastian and Feng, Shaohong and Bruderer, Noah and Maier, Andreas and Wang, Cankun and Baumbach, Jan and Abreu-Vicente, Jorge and Krehl, Nils and Ma, Qin and Lemberger, Thomas and Saez-Rodriguez, Julio},
  doi          = {10.1038/s41587-024-02534-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {166-169},
  shortjournal = {Nature Biotech.},
  title        = {A platform for the biomedical application of large language models},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(2), 158. (<a href='https://doi.org/10.1038/s41587-025-02571-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02571-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {158},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Startup grows egg proteins in potato fields. <em>NBT</em>, <em>43</em>(2), 157. (<a href='https://doi.org/10.1038/s41587-025-02563-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-025-02563-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {157},
  shortjournal = {Nature Biotech.},
  title        = {Startup grows egg proteins in potato fields},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gene and cell therapies for parkinson’s make headway. <em>NBT</em>, <em>43</em>(2), 156. (<a href='https://doi.org/10.1038/s41587-025-02581-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02581-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {156},
  shortjournal = {Nature Biotech.},
  title        = {Gene and cell therapies for parkinson’s make headway},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Colossal raises $200 million to bring back the dodo. <em>NBT</em>, <em>43</em>(2), 155. (<a href='https://doi.org/10.1038/s41587-025-02578-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02578-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {155},
  shortjournal = {Nature Biotech.},
  title        = {Colossal raises $200 million to bring back the dodo},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). AI biotechs launch bioprospecting expeditions with indigenous groups, agree to share benefits. <em>NBT</em>, <em>43</em>(2), 154-156. (<a href='https://doi.org/10.1038/s41587-025-02564-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies are striking unconventional benefit-sharing agreements with national governments to scour planet hotspots, gathering thousands of new DNA and protein sequences to develop into new, commercially useful molecules.},
  archive      = {J_NBT},
  author       = {Johnson, Ben},
  doi          = {10.1038/s41587-025-02564-5},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {154-156},
  shortjournal = {Nature Biotech.},
  title        = {AI biotechs launch bioprospecting expeditions with indigenous groups, agree to share benefits},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Largest proteome study enlists 14 biopharmas. <em>NBT</em>, <em>43</em>(2), 153. (<a href='https://doi.org/10.1038/s41587-025-02577-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02577-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {153},
  shortjournal = {Nature Biotech.},
  title        = {Largest proteome study enlists 14 biopharmas},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Gains in early-onset dementia with progranulin open new paths for drug discovery. <em>NBT</em>, <em>43</em>(2), 151-153. (<a href='https://doi.org/10.1038/s41587-025-02562-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The first clinical results in patients with a genetic form of frontotemporal dementia (FTD) show that enhancing progranulin in the brain may halt disease progression. If successful, this potentially disease-modifying approach may uncover new avenues for treating other neurodegenerative diseases.},
  archive      = {J_NBT},
  author       = {Gravitz, Lauren},
  doi          = {10.1038/s41587-025-02562-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {151-153},
  shortjournal = {Nature Biotech.},
  title        = {Gains in early-onset dementia with progranulin open new paths for drug discovery},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Sound healing and beyond. <em>NBT</em>, <em>43</em>(2), 149-150. (<a href='https://doi.org/10.1038/s41587-025-02566-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultrasound neurotechnologies are moving quickly into clinical trials in a wide variety of applications, and initiatives to open-source their manufacture will make them more accessible.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-025-02566-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {149-150},
  shortjournal = {Nature Biotech.},
  title        = {Sound healing and beyond},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Fresh from the biotech pipeline: FDA approvals settle in 2024, but what next?. <em>NBT</em>, <em>43</em>(2), 159-165. (<a href='https://doi.org/10.1038/s41587-025-02555-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After two volatile years, FDA approvals in 2024 settled closer to their 10-year average. Will nominated commissioner Marty Makary shake things up, or maintain a steady ship?},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-025-02555-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {2},
  pages        = {159-165},
  shortjournal = {Nature Biotech.},
  title        = {Fresh from the biotech pipeline: FDA approvals settle in 2024, but what next?},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). People. <em>NBT</em>, <em>43</em>(1), 148. (<a href='https://doi.org/10.1038/s41587-024-02527-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02527-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {148},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Five questions with anastassia vorobieva. <em>NBT</em>, <em>43</em>(1), 147. (<a href='https://doi.org/10.1038/s41587-024-02522-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A biophysicist interested in understanding the molecular principles of membrane protein folding and applying them to protein design discusses her nonlinear academic path and the potential societal impact of her research.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-024-02522-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {147},
  shortjournal = {Nature Biotech.},
  title        = {Five questions with anastassia vorobieva},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). As new cell and gene therapies emerge from academia, we must RISE to the opportunity. <em>NBT</em>, <em>43</em>(1), 143-146. (<a href='https://doi.org/10.1038/s41587-024-02520-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Academic medical centers must play a crucial part in overcoming cell and gene therapy development hurdles by establishing institutional frameworks that offer comprehensive support extending beyond financial assistance.},
  archive      = {J_NBT},
  author       = {Uma Naresh, Nandhitha and Hammill, Dana and Kessel, Mark and Hajjar, Roger J. and Yozwiak, Nathan L.},
  doi          = {10.1038/s41587-024-02520-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {143-146},
  shortjournal = {Nature Biotech.},
  title        = {As new cell and gene therapies emerge from academia, we must RISE to the opportunity},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Prediction of tumor-reactive t cell receptors from scRNA-seq data for personalized t cell therapy. <em>NBT</em>, <em>43</em>(1), 134-142. (<a href='https://doi.org/10.1038/s41587-024-02161-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The identification of patient-derived, tumor-reactive T cell receptors (TCRs) as a basis for personalized transgenic T cell therapies remains a time- and cost-intensive endeavor. Current approaches to identify tumor-reactive TCRs analyze tumor mutations to predict T cell activating (neo)antigens and use these to either enrich tumor infiltrating lymphocyte (TIL) cultures or validate individual TCRs for transgenic autologous therapies. Here we combined high-throughput TCR cloning and reactivity validation to train predicTCR, a machine learning classifier that identifies individual tumor-reactive TILs in an antigen-agnostic manner based on single-TIL RNA sequencing. PredicTCR identifies tumor-reactive TCRs in TILs from diverse cancers better than previous gene set enrichment-based approaches, increasing specificity and sensitivity (geometric mean) from 0.38 to 0.74. By predicting tumor-reactive TCRs in a matter of days, TCR clonotypes can be prioritized to accelerate the manufacture of personalized T cell therapies. Tumor-reactive T cell receptors are rapidly identified from lymphocyte scRNA-seq data.},
  archive      = {J_NBT},
  author       = {Tan, C. L. and Lindner, K. and Boschert, T. and Meng, Z. and Rodriguez Ehrenfried, A. and De Roia, A. and Haltenhof, G. and Faenza, A. and Imperatore, F. and Bunse, L. and Lindner, J. M. and Harbottle, R. P. and Ratliff, M. and Offringa, R. and Poschke, I. and Platten, M. and Green, E. W.},
  doi          = {10.1038/s41587-024-02161-y},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {134-142},
  shortjournal = {Nature Biotech.},
  title        = {Prediction of tumor-reactive t cell receptors from scRNA-seq data for personalized t cell therapy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Quality assessment of gene repertoire annotations with OMArk. <em>NBT</em>, <em>43</em>(1), 124-133. (<a href='https://doi.org/10.1038/s41587-024-02147-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the era of biodiversity genomics, it is crucial to ensure that annotations of protein-coding gene repertoires are accurate. State-of-the-art tools to assess genome annotations measure the completeness of a gene repertoire but are blind to other errors, such as gene overprediction or contamination. We introduce OMArk, a software package that relies on fast, alignment-free sequence comparisons between a query proteome and precomputed gene families across the tree of life. OMArk assesses not only the completeness but also the consistency of the gene repertoire as a whole relative to closely related species and reports likely contamination events. Analysis of 1,805 UniProt Eukaryotic Reference Proteomes with OMArk demonstrated strong evidence of contamination in 73 proteomes and identified error propagation in avian gene annotation resulting from the use of a fragmented zebra finch proteome as a reference. This study illustrates the importance of comparing and prioritizing proteomes based on their quality measures. A new tool checks the quality of gene annotations in genome sequences.},
  archive      = {J_NBT},
  author       = {Nevers, Yannis and Warwick Vesztrocy, Alex and Rossier, Victor and Train, Clément-Marie and Altenhoff, Adrian and Dessimoz, Christophe and Glover, Natasha M.},
  doi          = {10.1038/s41587-024-02147-w},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {124-133},
  shortjournal = {Nature Biotech.},
  title        = {Quality assessment of gene repertoire annotations with OMArk},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Near-cognate tRNAs increase the efficiency and precision of pseudouridine-mediated readthrough of premature termination codons. <em>NBT</em>, <em>43</em>(1), 114-123. (<a href='https://doi.org/10.1038/s41587-024-02165-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Programmable RNA pseudouridylation has emerged as a new type of RNA base editor to suppress premature termination codons (PTCs) that can lead to truncated and nonfunctional proteins. However, current methods to correct disease-associated PTCs suffer from low efficiency and limited precision. Here we develop RESTART v3, which uses near-cognate tRNAs to improve the readthrough efficiency of pseudouridine-modified PTCs. We show an average of ~5-fold (range: 2.1- to 9.5-fold) higher editing efficiency than RESTART v2 in cultured cells and achieve functional PTC readthrough in disease cell models of cystic fibrosis and Hurler syndrome. Furthermore, RESTART v3 enables accurate incorporation of the original amino acid for nearly half of the PTC sites, considering the naturally occurring frequencies of sense-to-nonsense codons, without affecting normal termination codons. Although off-target sites were detected, we did not observe changes to the coding information or the expression level of transcripts, and the overall natural tRNA abundance remained constant. Disease-related premature stop codons are efficiently corrected by harnessing near-cognate tRNAs.},
  archive      = {J_NBT},
  author       = {Luo, Nan and Huang, Qiang and Dong, Liting and Liu, Wenqing and Song, Jinghui and Sun, Hanxiao and Wu, Hao and Gao, Yuan and Yi, Chengqi},
  doi          = {10.1038/s41587-024-02165-8},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {114-123},
  shortjournal = {Nature Biotech.},
  title        = {Near-cognate tRNAs increase the efficiency and precision of pseudouridine-mediated readthrough of premature termination codons},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Protein-adaptive differential scanning fluorimetry using conformationally responsive dyes. <em>NBT</em>, <em>43</em>(1), 106-113. (<a href='https://doi.org/10.1038/s41587-024-02158-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Differential scanning fluorimetry (DSF) is a technique that reports protein thermal stability via the selective recognition of unfolded states by fluorogenic dyes. However, DSF applications remain limited by protein incompatibilities with existing DSF dyes. Here we overcome this obstacle with the development of a protein-adaptive DSF platform (paDSF) that combines a dye library ‘Aurora’ with a streamlined procedure to identify protein–dye pairs on demand. paDSF was successfully applied to 94% (66 of 70) of proteins, tripling the previous compatibility and delivering assays for 66 functionally and biochemically diverse proteins, including 10 from severe acute respiratory syndrome coronavirus 2. We find that paDSF can be used to monitor biological processes that were previously inaccessible, demonstrated for the interdomain allostery of O-GlcNAc transferase. The chemical diversity and varied selectivities of Aurora dyes suggest that paDSF functionality may be readily extended. paDSF is a generalizable tool to interrogate protein stability, dynamics and ligand binding. A library of fluorogenic dyes enables the measurement of protein unfolding for a large range of proteins.},
  archive      = {J_NBT},
  author       = {Wu, Taiasean and Yu, Joshua C. and Suresh, Arundhati and Gale-Day, Zachary J. and Alteen, Matthew G. and Woo, Amanda S. and Millbern, Zoe and Johnson, Oleta T. and Carroll, Emma C. and Partch, Carrie L. and Fourches, Denis and Vinueza, Nelson R. and Vocadlo, David J. and Gestwicki, Jason E.},
  doi          = {10.1038/s41587-024-02158-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {106-113},
  shortjournal = {Nature Biotech.},
  title        = {Protein-adaptive differential scanning fluorimetry using conformationally responsive dyes},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). All-atom RNA structure determination from cryo-EM maps. <em>NBT</em>, <em>43</em>(1), 97-105. (<a href='https://doi.org/10.1038/s41587-024-02149-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many methods exist for determining protein structures from cryogenic electron microscopy maps, but this remains challenging for RNA structures. Here we developed EMRNA, a method for accurate, automated determination of full-length all-atom RNA structures from cryogenic electron microscopy maps. EMRNA integrates deep learning-based detection of nucleotides, three-dimensional backbone tracing and scoring with consideration of sequence and secondary structure information, and full-atom construction of the RNA structure. We validated EMRNA on 140 diverse RNA maps ranging from 37 to 423 nt at 2.0–6.0 Å resolutions, and compared EMRNA with auto-DRRAFTER, phenix.map_to_model and CryoREAD on a set of 71 cases. EMRNA achieves a median accuracy of 2.36 Å root mean square deviation and 0.86 TM-score for full-length RNA structures, compared with 6.66 Å and 0.58 for auto-DRRAFTER. EMRNA also obtains a high residue coverage and sequence match of 93.30% and 95.30% in the built models, compared with 58.20% and 42.20% for phenix.map_to_model and 56.45% and 52.3% for CryoREAD. EMRNA is fast and can build an RNA structure of 100 nt within 3 min. RNA structures are built from cryogenic electron microscopy maps using deep learning and backbone tracing.},
  archive      = {J_NBT},
  author       = {Li, Tao and He, Jiahua and Cao, Hong and Zhang, Yi and Chen, Ji and Xiao, Yi and Huang, Sheng-You},
  doi          = {10.1038/s41587-024-02149-8},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {97-105},
  shortjournal = {Nature Biotech.},
  title        = {All-atom RNA structure determination from cryo-EM maps},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Simultaneous single-cell analysis of 5mC and 5hmC with SIMPLE-seq. <em>NBT</em>, <em>43</em>(1), 85-96. (<a href='https://doi.org/10.1038/s41587-024-02148-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dynamic 5-methylcytosine (5mC) and 5-hydroxymethylcytosine (5hmC) modifications to DNA regulate gene expression in a cell-type-specific manner and are associated with various biological processes, but the two modalities have not yet been measured simultaneously from the same genome at the single-cell level. Here we present SIMPLE-seq, a scalable, base resolution method for joint analysis of 5mC and 5hmC from thousands of single cells. Based on orthogonal labeling and recording of ‘C-to-T’ mutational signals from 5mC and 5hmC sites, SIMPLE-seq detects these two modifications from the same molecules in single cells and enables unbiased DNA methylation dynamics analysis of heterogeneous biological samples. We applied this method to mouse embryonic stem cells, human peripheral blood mononuclear cells and mouse brain to give joint epigenome maps at single-cell and single-molecule resolution. Integrated analysis of these two cytosine modifications reveals distinct epigenetic patterns associated with divergent regulatory programs in different cell types as well as cell states. SIMPLE-seq measures both 5-methylcytosine and 5-hydroxymethylcytosine at base resolution in single cells.},
  archive      = {J_NBT},
  author       = {Bai, Dongsheng and Zhang, Xiaoting and Xiang, Huifen and Guo, Zijian and Zhu, Chenxu and Yi, Chengqi},
  doi          = {10.1038/s41587-024-02148-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {85-96},
  shortjournal = {Nature Biotech.},
  title        = {Simultaneous single-cell analysis of 5mC and 5hmC with SIMPLE-seq},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Improving microbial phylogeny with citizen science within a mass-market video game. <em>NBT</em>, <em>43</em>(1), 76-84. (<a href='https://doi.org/10.1038/s41587-024-02175-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Citizen science video games are designed primarily for users already inclined to contribute to science, which severely limits their accessibility for an estimated community of 3 billion gamers worldwide. We created Borderlands Science (BLS), a citizen science activity that is seamlessly integrated within a popular commercial video game played by tens of millions of gamers. This integration is facilitated by a novel game-first design of citizen science games, in which the game design aspect has the highest priority, and a suitable task is then mapped to the game design. BLS crowdsources a multiple alignment task of 1 million 16S ribosomal RNA sequences obtained from human microbiome studies. Since its initial release on 7 April 2020, over 4 million players have solved more than 135 million science puzzles, a task unsolvable by a single individual. Leveraging these results, we show that our multiple sequence alignment simultaneously improves microbial phylogeny estimations and UniFrac effect sizes compared to state-of-the-art computational methods. This achievement demonstrates that hyper-gamified scientific tasks attract massive crowds of contributors and offers invaluable resources to the scientific community. Gamification of the multiple sequence alignment problem improves microbial phylogeny estimates.},
  archive      = {J_NBT},
  author       = {Sarrazin-Gendron, Roman and Ghasemloo Gheidari, Parham and Butyaev, Alexander and Keding, Timothy and Cai, Eddie and Zheng, Jiayue and Mutalova, Renata and Mounthanyvong, Julien and Zhu, Yuxue and Nazarova, Elena and Drogaris, Chrisostomos and Erhart, Kornél and Brouillette, Amélie and Richard, Gabriel and Pitchford, Randy and Caisse, Sébastien and Blanchette, Mathieu and McDonald, Daniel and Knight, Rob and Szantner, Attila and Waldispühl, Jérôme},
  doi          = {10.1038/s41587-024-02175-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {76-84},
  shortjournal = {Nature Biotech.},
  title        = {Improving microbial phylogeny with citizen science within a mass-market video game},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A small-molecule TNIK inhibitor targets fibrosis in preclinical and clinical models. <em>NBT</em>, <em>43</em>(1), 63-75. (<a href='https://doi.org/10.1038/s41587-024-02143-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Idiopathic pulmonary fibrosis (IPF) is an aggressive interstitial lung disease with a high mortality rate. Putative drug targets in IPF have failed to translate into effective therapies at the clinical level. We identify TRAF2- and NCK-interacting kinase (TNIK) as an anti-fibrotic target using a predictive artificial intelligence (AI) approach. Using AI-driven methodology, we generated INS018_055, a small-molecule TNIK inhibitor, which exhibits desirable drug-like properties and anti-fibrotic activity across different organs in vivo through oral, inhaled or topical administration. INS018_055 possesses anti-inflammatory effects in addition to its anti-fibrotic profile, validated in multiple in vivo studies. Its safety and tolerability as well as pharmacokinetics were validated in a randomized, double-blinded, placebo-controlled phase I clinical trial (NCT05154240) involving 78 healthy participants. A separate phase I trial in China, CTR20221542, also demonstrated comparable safety and pharmacokinetic profiles. This work was completed in roughly 18 months from target discovery to preclinical candidate nomination and demonstrates the capabilities of our generative AI-driven drug-discovery pipeline. An AI-generated small-molecule inhibitor treats fibrosis in vivo and in phase I clinical trials.},
  archive      = {J_NBT},
  author       = {Ren, Feng and Aliper, Alex and Chen, Jian and Zhao, Heng and Rao, Sujata and Kuppe, Christoph and Ozerov, Ivan V. and Zhang, Man and Witte, Klaus and Kruse, Chris and Aladinskiy, Vladimir and Ivanenkov, Yan and Polykovskiy, Daniil and Fu, Yanyun and Babin, Eugene and Qiao, Junwen and Liang, Xing and Mou, Zhenzhen and Wang, Hui and Pun, Frank W. and Torres-Ayuso, Pedro and Veviorskiy, Alexander and Song, Dandan and Liu, Sang and Zhang, Bei and Naumov, Vladimir and Ding, Xiaoqiang and Kukharenko, Andrey and Izumchenko, Evgeny and Zhavoronkov, Alex},
  doi          = {10.1038/s41587-024-02143-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {63-75},
  shortjournal = {Nature Biotech.},
  title        = {A small-molecule TNIK inhibitor targets fibrosis in preclinical and clinical models},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Morphological diversification and functional maturation of human astrocytes in glia-enriched cortical organoid transplanted in mouse brain. <em>NBT</em>, <em>43</em>(1), 52-62. (<a href='https://doi.org/10.1038/s41587-024-02157-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Astrocytes, the most abundant glial cell type in the brain, are underrepresented in traditional cortical organoid models due to the delayed onset of cortical gliogenesis. Here we introduce a new glia-enriched cortical organoid model that exhibits accelerated astrogliogenesis. We demonstrated that induction of a gliogenic switch in a subset of progenitors enabled the rapid derivation of astroglial cells, which account for 25–31% of the cell population within 8–10 weeks of differentiation. Intracerebral transplantation of these organoids reliably generated a diverse repertoire of cortical neurons and anatomical subclasses of human astrocytes. Spatial transcriptome profiling identified layer-specific expression patterns among distinct subclasses of astrocytes within organoid transplants. Using an in vivo acute neuroinflammation model, we identified a subpopulation of astrocytes that rapidly activates pro-inflammatory pathways upon cytokine stimulation. Additionally, we demonstrated that CD38 signaling has a crucial role in mediating metabolic and mitochondrial stress in reactive astrocytes. This model provides a robust platform for investigating human astrocyte function. Transplantation of glia-enriched organoids into the mouse brain enables the study of astrocyte subtypes.},
  archive      = {J_NBT},
  author       = {Wang, Meiyan and Zhang, Lei and Novak, Sammy Weiser and Yu, Jingting and Gallina, Iryna S. and Xu, Lynne L. and Lim, Christina K. and Fernandes, Sarah and Shokhirev, Maxim N. and Williams, April E. and Saxena, Monisha D. and Coorapati, Shashank and Parylak, Sarah L. and Quintero, Cristian and Molina, Elsa and Andrade, Leonardo R. and Manor, Uri and Gage, Fred H.},
  doi          = {10.1038/s41587-024-02157-8},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {52-62},
  shortjournal = {Nature Biotech.},
  title        = {Morphological diversification and functional maturation of human astrocytes in glia-enriched cortical organoid transplanted in mouse brain},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Harnessing eukaryotic retroelement proteins for transgene insertion into human safe-harbor loci. <em>NBT</em>, <em>43</em>(1), 42-51. (<a href='https://doi.org/10.1038/s41587-024-02137-y'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current approaches for inserting autonomous transgenes into the genome, such as CRISPR–Cas9 or virus-based strategies, have limitations including low efficiency and high risk of untargeted genome mutagenesis. Here, we describe precise RNA-mediated insertion of transgenes (PRINT), an approach for site-specifically primed reverse transcription that directs transgene synthesis directly into the genome at a multicopy safe-harbor locus. PRINT uses delivery of two in vitro transcribed RNAs: messenger RNA encoding avian R2 retroelement-protein and template RNA encoding a transgene of length validated up to 4 kb. The R2 protein coordinately recognizes the target site, nicks one strand at a precise location and primes complementary DNA synthesis for stable transgene insertion. With a cultured human primary cell line, over 50% of cells can gain several 2 kb transgenes, of which more than 50% are full-length. PRINT advantages include no extragenomic DNA, limiting risk of deleterious mutagenesis and innate immune responses, and the relatively low cost, rapid production and scalability of RNA-only delivery. Transgenes are inserted into human cells by 2-RNA delivery of a retroelement protein and template.},
  archive      = {J_NBT},
  author       = {Zhang, Xiaozhu and Van Treeck, Briana and Horton, Connor A. and McIntyre, Jeremy J. R. and Palm, Sarah M. and Shumate, Justin L. and Collins, Kathleen},
  doi          = {10.1038/s41587-024-02137-y},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {42-51},
  shortjournal = {Nature Biotech.},
  title        = {Harnessing eukaryotic retroelement proteins for transgene insertion into human safe-harbor loci},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). OMArk, a tool for gene annotation quality control, reveals erroneous gene inference. <em>NBT</em>, <em>43</em>(1), 40-41. (<a href='https://doi.org/10.1038/s41587-024-02155-w'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed the OMArk software package for evaluating protein-coding gene annotation quality. In addition to assessing the completeness of a proteome, OMArk estimates the overall quality of the gene set’s content, a feature that will help to improve public protein sequence data.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02155-w},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {40-41},
  shortjournal = {Nature Biotech.},
  title        = {OMArk, a tool for gene annotation quality control, reveals erroneous gene inference},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A deep learning-based method for modeling of RNA structures from cryo-EM maps. <em>NBT</em>, <em>43</em>(1), 38-39. (<a href='https://doi.org/10.1038/s41587-024-02162-x'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a deep learning-based method, EMRNA, to automatically model RNA structures from cryo-electron microscopy maps. Evaluation of EMRNA on diverse test sets of RNA maps shows that it builds RNA models with high accuracy and efficiency.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02162-x},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {38-39},
  shortjournal = {Nature Biotech.},
  title        = {A deep learning-based method for modeling of RNA structures from cryo-EM maps},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Video game unleashes millions of citizen scientists on microbiome research. <em>NBT</em>, <em>43</em>(1), 36-37. (<a href='https://doi.org/10.1038/s41587-024-02203-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Borderlands Science is a casual mini-game released within a mass-market video game that crowdsources the alignment of one million RNA sequences from the human microbiome. In 3 years, 4 million participants generated over 135 million puzzle solutions that were used to build a reference alignment and improve microbial phylogeny.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02203-5},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {36-37},
  shortjournal = {Nature Biotech.},
  title        = {Video game unleashes millions of citizen scientists on microbiome research},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Glia-enriched cortical organoids implanted in mice capture astrocyte diversity. <em>NBT</em>, <em>43</em>(1), 34-35. (<a href='https://doi.org/10.1038/s41587-024-02198-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The underrepresentation of functional glial cells is a major challenge in brain organoid models. We developed an astroglia-enriched cortical organoid model that allows efficient generation of functional astrocytes and enables the formation of astroglial morphological subclasses with layer-specific gene expression profiles upon transplantation into the mouse brain.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02198-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {34-35},
  shortjournal = {Nature Biotech.},
  title        = {Glia-enriched cortical organoids implanted in mice capture astrocyte diversity},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A retrotransposon for site-specific gene transfer. <em>NBT</em>, <em>43</em>(1), 31-33. (<a href='https://doi.org/10.1038/s41587-024-02180-9'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An engineered retrotransposon achieves targeted gene transfer into the human genome.},
  archive      = {J_NBT},
  author       = {Dyda, Fred and Hickman, Alison B.},
  doi          = {10.1038/s41587-024-02180-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {31-33},
  shortjournal = {Nature Biotech.},
  title        = {A retrotransposon for site-specific gene transfer},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Synthetic GPCRs for controlling cellular responses. <em>NBT</em>, <em>43</em>(1), 30. (<a href='https://doi.org/10.1038/s41587-024-02536-1'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02536-1},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {30},
  shortjournal = {Nature Biotech.},
  title        = {Synthetic GPCRs for controlling cellular responses},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). An organoid model with all three pancreatic lineages resembles fetal pancreas. <em>NBT</em>, <em>43</em>(1), 30. (<a href='https://doi.org/10.1038/s41587-024-02537-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02537-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {30},
  shortjournal = {Nature Biotech.},
  title        = {An organoid model with all three pancreatic lineages resembles fetal pancreas},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Therapeutic t cells with synthetic gene circuits act locally. <em>NBT</em>, <em>43</em>(1), 30. (<a href='https://doi.org/10.1038/s41587-024-02541-4'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marchal, Iris},
  doi          = {10.1038/s41587-024-02541-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {30},
  shortjournal = {Nature Biotech.},
  title        = {Therapeutic t cells with synthetic gene circuits act locally},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Cancer vaccines. <em>NBT</em>, <em>43</em>(1), 29. (<a href='https://doi.org/10.1038/s41587-024-02521-8'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to compositions and methods for producing and using cancer vaccines.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02521-8},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {29},
  shortjournal = {Nature Biotech.},
  title        = {Cancer vaccines},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). How will AI affect patent disclosures?. <em>NBT</em>, <em>43</em>(1), 26-28. (<a href='https://doi.org/10.1038/s41587-024-02515-6'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence tools for drafting patents will exacerbate challenges with the disclosure of useful technical information in patent documents.},
  archive      = {J_NBT},
  author       = {Ouellette, Lisa Larrimore and Fang, Victoria and Ouellette, Nicholas T.},
  doi          = {10.1038/s41587-024-02515-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {26-28},
  shortjournal = {Nature Biotech.},
  title        = {How will AI affect patent disclosures?},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Toward a safer and more secure US bioeconomy. <em>NBT</em>, <em>43</em>(1), 23-25. (<a href='https://doi.org/10.1038/s41587-024-02519-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To enhance the safety and security of the US bioeconomy, a new public–private partnership should be established to facilitate information sharing and threat analysis among industry, government and academia, and to develop and deploy safeguards.},
  archive      = {J_NBT},
  author       = {Watson, Matthew C. and Rambhia, Kunal J. and Seltzer, Meghan J. and Carter, Sarah R. and Moritz, Rebecca L. and Attal-Juncqua, Aurelia and Diggans, James and Dileo, John},
  doi          = {10.1038/s41587-024-02519-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {23-25},
  shortjournal = {Nature Biotech.},
  title        = {Toward a safer and more secure US bioeconomy},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The devolution of biosimilars regulations. <em>NBT</em>, <em>43</em>(1), 19-22. (<a href='https://doi.org/10.1038/s41587-024-02497-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After two decades of experience with biosimilars, physicochemical and in vitro biological comparison with their reference products appear sufficient to guarantee clinical safety and efficacy. Hence, the regulation of biosimilars has become redundant, and biopharmaceuticals should now be regulated through the generic pathway available for small molecules.},
  archive      = {J_NBT},
  author       = {Doevendans, Erik and van Meer, Peter and Schellekens, Huub},
  doi          = {10.1038/s41587-024-02497-5},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {19-22},
  shortjournal = {Nature Biotech.},
  title        = {The devolution of biosimilars regulations},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biotech news from around the world. <em>NBT</em>, <em>43</em>(1), 10. (<a href='https://doi.org/10.1038/s41587-024-02538-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02538-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {10},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mushroom makes tasty eco-asphalt. <em>NBT</em>, <em>43</em>(1), 9. (<a href='https://doi.org/10.1038/s41587-024-02530-7'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Turrell, Claire},
  doi          = {10.1038/s41587-024-02530-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {9},
  shortjournal = {Nature Biotech.},
  title        = {Mushroom makes tasty eco-asphalt},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Pricey sickle cell gene therapies primed for change. <em>NBT</em>, <em>43</em>(1), 8. (<a href='https://doi.org/10.1038/s41587-024-02542-3'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02542-3},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {8},
  shortjournal = {Nature Biotech.},
  title        = {Pricey sickle cell gene therapies primed for change},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Liver-avoiding capsids draw $1.1 billion. <em>NBT</em>, <em>43</em>(1), 7. (<a href='https://doi.org/10.1038/s41587-024-02545-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02545-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {7},
  shortjournal = {Nature Biotech.},
  title        = {Liver-avoiding capsids draw $1.1 billion},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Agriculture to flourish on precision breeding: Who will benefit?. <em>NBT</em>, <em>43</em>(1), 6-8. (<a href='https://doi.org/10.1038/s41587-024-02532-5'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {From corn to bananas, farmers stand to gain from cultivating edited crops that are resilient and sustainable, paired with precision insecticides and microbe engineering. But reaching those with the greatest need remains a challenge.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-024-02532-5},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {6-8},
  shortjournal = {Nature Biotech.},
  title        = {Agriculture to flourish on precision breeding: Who will benefit?},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Ionis wins FDA nod for triglyceride-lowering rare disease drug. <em>NBT</em>, <em>43</em>(1), 5. (<a href='https://doi.org/10.1038/s41587-024-02546-z'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02546-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {5},
  shortjournal = {Nature Biotech.},
  title        = {Ionis wins FDA nod for triglyceride-lowering rare disease drug},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A cure for HIV? TCR agents seek to wipe out viral reservoirs. <em>NBT</em>, <em>43</em>(1), 3-5. (<a href='https://doi.org/10.1038/s41587-024-02529-0'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bispecific T cell receptor molecules designed to wipe out HIV-infected T cells offer glimpses of a cure.},
  archive      = {J_NBT},
  author       = {Hoyos-Flight, Monica},
  doi          = {10.1038/s41587-024-02529-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {3-5},
  shortjournal = {Nature Biotech.},
  title        = {A cure for HIV? TCR agents seek to wipe out viral reservoirs},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Making space for spatial biology in the clinic. <em>NBT</em>, <em>43</em>(1), 1-2. (<a href='https://doi.org/10.1038/s41587-024-02543-2'>www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics methods have been slow to move into clinical practice, but spatial proteomics are cheaper and more scalable, and could progress faster.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-024-02543-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {1-2},
  shortjournal = {Nature Biotech.},
  title        = {Making space for spatial biology in the clinic},
  volume       = {43},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
